<!DOCTYPE HTML>
<html>
<head>
<title>JabRef references</title>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<script type="text/javascript">
<!--
// QuickSearch script for JabRef HTML export 
// Version: 3.0
//
// Copyright (c) 2006-2011, Mark Schenk
//
// This software is distributed under a Creative Commons Attribution 3.0 License
// http://creativecommons.org/licenses/by/3.0/
//
// Features:
// - intuitive find-as-you-type searching
//    ~ case insensitive
//    ~ ignore diacritics (optional)
//
// - search with/without Regular Expressions
// - match BibTeX key
//

// Search settings
var searchAbstract = true;	// search in abstract
var searchReview = true;	// search in review

var noSquiggles = true; 	// ignore diacritics when searching
var searchRegExp = false; 	// enable RegExp searches


if (window.addEventListener) {
	window.addEventListener("load",initSearch,false); }
else if (window.attachEvent) {
	window.attachEvent("onload", initSearch); }

function initSearch() {
	// check for quick search table and searchfield
	if (!document.getElementById('qs_table')||!document.getElementById('quicksearch')) { return; }

	// load all the rows and sort into arrays
	loadTableData();
	
	//find the query field
	qsfield = document.getElementById('qs_field');

	// previous search term; used for speed optimisation
	prevSearch = '';

	//find statistics location
	stats = document.getElementById('stat');
	setStatistics(-1);
	
	// set up preferences
	initPreferences();

	// shows the searchfield
	document.getElementById('quicksearch').style.display = 'block';
	document.getElementById('qs_field').onkeyup = quickSearch;
}

function loadTableData() {
	// find table and appropriate rows
	searchTable = document.getElementById('qs_table');
	var allRows = searchTable.getElementsByTagName('tbody')[0].getElementsByTagName('tr');

	// split all rows into entryRows and infoRows (e.g. abstract, review, bibtex)
	entryRows = new Array(); infoRows = new Array(); absRows = new Array(); revRows = new Array();

	// get data from each row
	entryRowsData = new Array(); absRowsData = new Array(); revRowsData = new Array(); 
	
	BibTeXKeys = new Array();
	
	for (var i=0, k=0, j=0; i<allRows.length;i++) {
		if (allRows[i].className.match(/entry/)) {
			entryRows[j] = allRows[i];
			entryRowsData[j] = stripDiacritics(getTextContent(allRows[i]));
			allRows[i].id ? BibTeXKeys[j] = allRows[i].id : allRows[i].id = 'autokey_'+j;
			j ++;
		} else {
			infoRows[k++] = allRows[i];
			// check for abstract/review
			if (allRows[i].className.match(/abstract/)) {
				absRows.push(allRows[i]);
				absRowsData[j-1] = stripDiacritics(getTextContent(allRows[i]));
			} else if (allRows[i].className.match(/review/)) {
				revRows.push(allRows[i]);
				revRowsData[j-1] = stripDiacritics(getTextContent(allRows[i]));
			}
		}
	}
	//number of entries and rows
	numEntries = entryRows.length;
	numInfo = infoRows.length;
	numAbs = absRows.length;
	numRev = revRows.length;
}

function quickSearch(){
	
	tInput = qsfield;

	if (tInput.value.length == 0) {
		showAll();
		setStatistics(-1);
		qsfield.className = '';
		return;
	} else {
		t = stripDiacritics(tInput.value);

		if(!searchRegExp) { t = escapeRegExp(t); }
			
		// only search for valid RegExp
		try {
			textRegExp = new RegExp(t,"i");
			closeAllInfo();
			qsfield.className = '';
		}
			catch(err) {
			prevSearch = tInput.value;
			qsfield.className = 'invalidsearch';
			return;
		}
	}
	
	// count number of hits
	var hits = 0;

	// start looping through all entry rows
	for (var i = 0; cRow = entryRows[i]; i++){

		// only show search the cells if it isn't already hidden OR if the search term is getting shorter, then search all
		if(cRow.className.indexOf('noshow')==-1 || tInput.value.length <= prevSearch.length){
			var found = false; 

			if (entryRowsData[i].search(textRegExp) != -1 || BibTeXKeys[i].search(textRegExp) != -1){ 
				found = true;
			} else {
				if(searchAbstract && absRowsData[i]!=undefined) {
					if (absRowsData[i].search(textRegExp) != -1){ found=true; } 
				}
				if(searchReview && revRowsData[i]!=undefined) {
					if (revRowsData[i].search(textRegExp) != -1){ found=true; } 
				}
			}
			
			if (found){
				cRow.className = 'entry show';
				hits++;
			} else {
				cRow.className = 'entry noshow';
			}
		}
	}

	// update statistics
	setStatistics(hits)
	
	// set previous search value
	prevSearch = tInput.value;
}


// Strip Diacritics from text
// http://stackoverflow.com/questions/990904/javascript-remove-accents-in-strings

// String containing replacement characters for stripping accents 
var stripstring = 
    'AAAAAAACEEEEIIII'+
    'DNOOOOO.OUUUUY..'+
    'aaaaaaaceeeeiiii'+
    'dnooooo.ouuuuy.y'+
    'AaAaAaCcCcCcCcDd'+
    'DdEeEeEeEeEeGgGg'+
    'GgGgHhHhIiIiIiIi'+
    'IiIiJjKkkLlLlLlL'+
    'lJlNnNnNnnNnOoOo'+
    'OoOoRrRrRrSsSsSs'+
    'SsTtTtTtUuUuUuUu'+
    'UuUuWwYyYZzZzZz.';

function stripDiacritics(str){

    if(noSquiggles==false){
        return str;
    }

    var answer='';
    for(var i=0;i<str.length;i++){
        var ch=str[i];
        var chindex=ch.charCodeAt(0)-192;   // Index of character code in the strip string
        if(chindex>=0 && chindex<stripstring.length){
            // Character is within our table, so we can strip the accent...
            var outch=stripstring.charAt(chindex);
            // ...unless it was shown as a '.'
            if(outch!='.')ch=outch;
        }
        answer+=ch;
    }
    return answer;
}

// http://stackoverflow.com/questions/3446170/escape-string-for-use-in-javascript-regex
// NOTE: must escape every \ in the export code because of the JabRef Export...
function escapeRegExp(str) {
  return str.replace(/[-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "\\$&");
}

function toggleInfo(articleid,info) {

	var entry = document.getElementById(articleid);
	var abs = document.getElementById('abs_'+articleid);
	var rev = document.getElementById('rev_'+articleid);
	var bib = document.getElementById('bib_'+articleid);
	
	if (abs && info == 'abstract') {
		abs.className.indexOf('noshow') == -1?abs.className = 'abstract noshow':abs.className = 'abstract show';
	} else if (rev && info == 'review') {
		rev.className.indexOf('noshow') == -1?rev.className = 'review noshow':rev.className = 'review show';
	} else if (bib && info == 'bibtex') {
		bib.className.indexOf('noshow') == -1?bib.className = 'bibtex noshow':bib.className = 'bibtex show';
	} else { 
		return;
	}

	// check if one or the other is available
	var revshow; var absshow; var bibshow;
	(abs && abs.className.indexOf('noshow') == -1)? absshow = true: absshow = false;
	(rev && rev.className.indexOf('noshow') == -1)? revshow = true: revshow = false;	
	(bib && bib.className.indexOf('noshow') == -1)? bibshow = true: bibshow = false;
	
	// highlight original entry
	if(entry) {
		if (revshow || absshow || bibshow) {
		entry.className = 'entry highlight show';
		} else {
		entry.className = 'entry show';
		}
	}
	
	// When there's a combination of abstract/review/bibtex showing, need to add class for correct styling
	if(absshow) {
		(revshow||bibshow)?abs.className = 'abstract nextshow':abs.className = 'abstract';
	} 
	if (revshow) {
		bibshow?rev.className = 'review nextshow': rev.className = 'review';
	}	
	
}

function setStatistics (hits) {
	if(hits < 0) { hits=numEntries; }
	if(stats) { stats.firstChild.data = hits + '/' + numEntries}
}

function getTextContent(node) {
	// Function written by Arve Bersvendsen
	// http://www.virtuelvis.com
	
	if (node.nodeType == 3) {
	return node.nodeValue;
	} // text node
	if (node.nodeType == 1 && node.className != "infolinks") { // element node
	var text = [];
	for (var chld = node.firstChild;chld;chld=chld.nextSibling) {
		text.push(getTextContent(chld));
	}
	return text.join("");
	} return ""; // some other node, won't contain text nodes.
}

function showAll(){
	closeAllInfo();
	for (var i = 0; i < numEntries; i++){ entryRows[i].className = 'entry show'; }
}

function closeAllInfo(){
	for (var i=0; i < numInfo; i++){
		if (infoRows[i].className.indexOf('noshow') ==-1) {
			infoRows[i].className = infoRows[i].className + ' noshow';
		}
	}
}

function clearQS() {
	qsfield.value = '';
	showAll();
}

function redoQS(){
	showAll();
	quickSearch(qsfield);
}

function updateSetting(obj){
	var option = obj.id;
	var checked = obj.value;

	switch(option)
	 {
	 case "opt_searchAbs":
	   searchAbstract=!searchAbstract;
	   redoQS();
	   break;
	 case "opt_searchRev":
	   searchReview=!searchReview;
	   redoQS();
	   break;
	 case "opt_useRegExp":
	   searchRegExp=!searchRegExp;
	   redoQS();
	   break;
	 case "opt_noAccents":
	   noSquiggles=!noSquiggles;
	   loadTableData();
	   redoQS();
	   break;
	 }
}

function initPreferences(){
	if(searchAbstract){document.getElementById("opt_searchAbs").checked = true;}
	if(searchReview){document.getElementById("opt_searchRev").checked = true;}
	if(noSquiggles){document.getElementById("opt_noAccents").checked = true;}
	if(searchRegExp){document.getElementById("opt_useRegExp").checked = true;}
	
	if(numAbs==0) {document.getElementById("opt_searchAbs").parentNode.style.display = 'none';}
	if(numRev==0) {document.getElementById("opt_searchRev").parentNode.style.display = 'none';}
}

function toggleSettings(){
	var togglebutton = document.getElementById('showsettings');
	var settings = document.getElementById('settings');
	
	if(settings.className == "hidden"){
		settings.className = "show";
		togglebutton.innerText = "close settings";
		togglebutton.textContent = "close settings";
	}else{
		settings.className = "hidden";
		togglebutton.innerText = "settings...";		
		togglebutton.textContent = "settings...";
	}
}

-->
</script>
<style type="text/css">
body { background-color: white; font-family: Arial, sans-serif; font-size: 13px; line-height: 1.2; padding: 1em; color: #2E2E2E; margin: auto 2em; }

form#quicksearch { width: auto; border-style: solid; border-color: gray; border-width: 1px 0px; padding: 0.7em 0.5em; display:none; position:relative; }
span#searchstat {padding-left: 1em;}

div#settings { margin-top:0.7em; /* border-bottom: 1px transparent solid; background-color: #efefef; border: 1px grey solid; */ }
div#settings ul {margin: 0; padding: 0; }
div#settings li {margin: 0; padding: 0 1em 0 0; display: inline; list-style: none; }
div#settings li + li { border-left: 2px #efefef solid; padding-left: 0.5em;}
div#settings input { margin-bottom: 0px;}

div#settings.hidden {display:none;}

#showsettings { border: 1px grey solid; padding: 0 0.5em; float:right; line-height: 1.6em; text-align: right; }
#showsettings:hover { cursor: pointer; }

.invalidsearch { background-color: red; }
input[type="button"] { background-color: #efefef; border: 1px #2E2E2E solid;}

table { width: 100%; empty-cells: show; border-spacing: 0em 0.2em; margin: 1em 0em; border-style: none; }
th, td { border: 1px gray solid; border-width: 1px 1px; padding: 0.5em; vertical-align: top; text-align: left; }
th { background-color: #efefef; }
td + td, th + th { border-left: none; }

td a { color: navy; text-decoration: none; }
td a:hover  { text-decoration: underline; }

tr.noshow { display: none;}
tr.highlight td { background-color: #EFEFEF; border-top: 2px #2E2E2E solid; font-weight: bold; }
tr.abstract td, tr.review td, tr.bibtex td { background-color: #EFEFEF; text-align: justify; border-bottom: 2px #2E2E2E solid; }
tr.nextshow td { border-bottom: 1px gray solid; }

tr.bibtex pre { width: 100%; overflow: auto; white-space: pre-wrap;}
p.infolinks { margin: 0.3em 0em 0em 0em; padding: 0px; }

@media print {
	p.infolinks, #qs_settings, #quicksearch, t.bibtex { display: none !important; }
	tr { page-break-inside: avoid; }
}
</style>
</head>
<body>

<form action="" id="quicksearch">
<input type="text" id="qs_field" autocomplete="off" placeholder="Type to search..." /> <input type="button" onclick="clearQS()" value="clear" />
<span id="searchstat">Matching entries: <span id="stat">0</span></span>
<div id="showsettings" onclick="toggleSettings()">settings...</div>
<div id="settings" class="hidden">
<ul>
<li><input type="checkbox" class="search_setting" id="opt_searchAbs" onchange="updateSetting(this)"><label for="opt_searchAbs"> include abstract</label></li>
<li><input type="checkbox" class="search_setting" id="opt_searchRev" onchange="updateSetting(this)"><label for="opt_searchRev"> include review</label></li>
<li><input type="checkbox" class="search_setting" id="opt_useRegExp" onchange="updateSetting(this)"><label for="opt_useRegExp"> use RegExp</label></li>
<li><input type="checkbox" class="search_setting" id="opt_noAccents" onchange="updateSetting(this)"><label for="opt_noAccents"> ignore accents</label></li>
</ul>
</div>
</form>
<table id="qs_table" border="1">
<thead><tr><th width="20%">Author</th><th width="30%">Title</th><th width="5%">Year</th><th width="30%">Journal/Proceedings</th><th width="10%">Reftype</th><th width="5%">DOI/URL</th></tr></thead>
<tbody><tr id="Aalten1997" class="entry">
	<td>Aalten, V.D., Groot, D.B., Findlay, J., Berendsen, H. and Amadei, A.</td>
	<td>A comparison of techniques for calculating protein essential dynamics <p class="infolinks">[<a href="javascript:toggleInfo('Aalten1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Journal of Computational Chemistry<br/>Vol. 18(2), pp. 169 - 181&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/(SICI)1096-987X(19970130)18">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Aalten1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Aalten1997,
  author = {Aalten, Van DMF and Groot, De BL and Findlay, JBC and Berendsen, HJC and Amadei, A},
  title = {A comparison of techniques for calculating protein essential dynamics},
  journal = {Journal of Computational Chemistry},
  year = {1997},
  volume = {18},
  number = {2},
  pages = {169 - 181},
  note = {Quick discussion of using ED in the dihedral angle subspace},
  doi = {https://doi.org/10.1002/(SICI)1096-987X(19970130)18}
}
</pre></td>
</tr>
<tr id="Abad-Zapatero1987" class="entry">
	<td>Abad-Zapatero, C., Griffith, J., Sussman, J. and Rossmann, M.</td>
	<td>Refined crystal structure of dogfish M4 apo-lactate dehydrogenase. <p class="infolinks">[<a href="javascript:toggleInfo('Abad-Zapatero1987','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Abad-Zapatero1987','bibtex')">BibTeX</a>]</p></td>
	<td>1987</td>
	<td>Journal of Molecular Biology<br/>Vol. 198(3), pp. 445-67&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Abad-Zapatero1987" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The crystal structure of M4 apo-lactate dehydrogenase from the spiny dogfish (Squalus acanthius) was initially refined by a constrained-restrained, and subsequently restrained, least-squares technique. The final structure contained 286 water molecules and two sulfate ions per subunit and gave an R-factor of 0.202 for difraction data between 8.0 and 2.0 A resolution. The upper limit for the co-ordinate accuracy of the atoms was estimated to be 0.25 A. The elements of secondary structure of the refined protein have not changed from those described previously, except for the appearance of a one-and-a-half turn 3(10) helix immediately after beta J. There is also a short segment of 3(10) helix between beta C and beta D in the part of the chain that connects the two beta alpha beta alpha beta units of the six-stranded parallel sheet (residues Tyr83 to Ala87). Examination of the interactions among the different elements of secondary structure by means of a surface accessibility algorithm supports the four structural clusters in the subunit. The first of the two sulfate ions is in the active site and occupies a cavity near the essential His195. Its nearest protein ligands are Arg171, Asp168 and Asn140. The second sulfate ion is located near the P-axis subunit interface. It is liganded by His188 and Arg173. These two residues are conserved in bacterial lactate dehydrogenase and form part of the fructose 1,6-bisphosphate effector binding site. Two other data sets in which one (collected at pH 7.8) or both (collected at pH 6.0) sulfate ions were replaced by citrate ions were also analyzed. Five cycles of refinement with respect to the pH 6.0 data (25 to 2.8 A resolution) resulted in an R value of 0.191. Only water molecules occupy the subunit boundary anion binding site at pH 7.8. The amino acid sequence was found to be in poor agreement with (2Fobs-Fcalc) electron density maps for the peptide between residues 207 and 211. The original sequence WNALKE was replaced by NVASIK. The essential His195 is hydrogen bonded to Asp168 on one side and Asn140 on the other. The latter residue is part of a turn that contains the only cis peptide bond of the structure at Pro141. The "flexible loop" (residues 97 to 123), which folds down over the active center in ternary complexes of the enzyme with substrate and coenzyme, has a well-defined structure. Analysis of the environment of Tyr237 suggests how its chemical modification inhibits the enzyme.</td>
</tr>
<tr id="bib_Abad-Zapatero1987" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Abad-Zapatero1987,
  author = {Abad-Zapatero, C and Griffith, JP and Sussman, JL and Rossmann, MG},
  title = {Refined crystal structure of dogfish M4 apo-lactate dehydrogenase.},
  journal = {Journal of Molecular Biology},
  year = {1987},
  volume = {198},
  number = {3},
  pages = {445-67}
}
</pre></td>
</tr>
<tr id="Adamczyk2011" class="entry">
	<td>Adamczyk, A.J., Cao, J., Kamerlin, S.C. and Warshel, A.</td>
	<td>Catalysis by dihydrofolate reductase and other enzymes arises from electrostatic preorganization, not conformational motions. <p class="infolinks">[<a href="javascript:toggleInfo('Adamczyk2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Adamczyk2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Proceedings of the National Academy of Science<br/>Vol. 108(34), pp. 14115 - 14120&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.1111252108">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Adamczyk2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The proposal that enzymatic catalysis is due to conformational fluctuations has been previously promoted by means of indirect considerations. However, recent works have focused on cases where the relevant motions have components toward distinct conformational regions, whose population could be manipulated by mutations. In particular, a recent work has claimed to provide direct experimental evidence for a dynamical contribution to catalysis in dihydrofolate reductase, where blocking a relevant conformational coordinate was related to the suppression of the motion toward the occluded conformation. The present work utilizes computer simulations to elucidate the true molecular basis for the experimentally observed effect. We start by reproducing the trend in the measured change in catalysis upon mutations (which was assumed to arise as a result of a "dynamical knockout" caused by the mutations). This analysis is performed by calculating the change in the corresponding activation barriers without the need to invoke dynamical effects. We then generate the catalytic landscape of the enzyme and demonstrate that motions in the conformational space do not help drive catalysis. We also discuss the role of flexibility and conformational dynamics in catalysis, once again demonstrating that their role is negligible and that the largest contribution to catalysis arises from electrostatic preorganization. Finally, we point out that the changes in the reaction potential surface modify the reorganization free energy (which includes entropic effects), and such changes in the surface also alter the corresponding motion. However, this motion is never the reason for catalysis, but rather simply a reflection of the shape of the reaction potential surface.</td>
</tr>
<tr id="bib_Adamczyk2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Adamczyk2011,
  author = {Adamczyk, Andrew J and Cao, Jie and Kamerlin, Shina C and Warshel, Arieh},
  title = {Catalysis by dihydrofolate reductase and other enzymes arises from electrostatic preorganization, not conformational motions.},
  journal = {Proceedings of the National Academy of Science},
  year = {2011},
  volume = {108},
  number = {34},
  pages = {14115 - 14120},
  doi = {https://doi.org/10.1073/pnas.1111252108}
}
</pre></td>
</tr>
<tr id="Adamic2001" class="entry">
	<td>Adamic, LA., Lukose, RM., Puniyani, AR. and Huberman, BA.</td>
	<td>Search in power-law networks <p class="infolinks">[<a href="javascript:toggleInfo('Adamic2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Adamic2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Physical Review E<br/>Vol. 64(4), pp. 046135 - 046135&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1103/PhysRevE.64.046135">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Adamic2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Many communication and social networks have power-law link distributions, containing a few nodes that have a very high degree and many with low degree. The high connectivity nodes play the important role of hubs in communication and networking, a fact that can be exploited when designing efficient search algo- rithms. We introduce a number of local search strategies that utilize high degree nodes in power-law graphs and that have costs scaling sublinearly with the size of the graph. We also demonstrate the utility of these strategies on the GNUTELLA peer-to-peer network.</td>
</tr>
<tr id="bib_Adamic2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Adamic2001,
  author = {Adamic, LA and Lukose, RM and Puniyani, AR and Huberman, BA},
  title = {Search in power-law networks},
  journal = {Physical Review E},
  year = {2001},
  volume = {64},
  number = {4},
  pages = {046135 - 046135},
  doi = {https://doi.org/10.1103/PhysRevE.64.046135}
}
</pre></td>
</tr>
<tr id="Adams1973" class="entry">
	<td>Adams, M.J., Buehner, M., Chandrasekhar, K., Ford, G.C., Hackert, M.L., Liljas, A., Rossmann, M.G., Smiley, I.E., Allison, W.S., Everse, J., Kaplan, NO. and Taylor, S.S.</td>
	<td>Structure-function relationships in lactate dehydrogenase. <p class="infolinks">[<a href="javascript:toggleInfo('Adams1973','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Adams1973','bibtex')">BibTeX</a>]</p></td>
	<td>1973</td>
	<td>Proceedings of the National Academy of Sciences<br/>Vol. 70(7), pp. 1968 - 1972&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.70.7.1968">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Adams1973" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The binding of coenzyme and substrate are considered in relation to the known primary and tertiary structure of lactate dehydrogenase (EC 1.1.1.27). The adenine binds in a hydrophobic crevice, and the two coenzyme phosphates are oriented by interactions with the protein. The positively charged guanidinium group of arginine 101 then folds over the negatively charged phosphates, collapsing the loop region over the active center and positioning the unreactive B side of the nicotinamide in a hydrophobic protein environment. Collapse of the loop also introduces various charged groups into the vicinity of the substrate binding site. The substrate is situated between histidine 195 and the C4 position on the nicotinamide ring, and is partially oriented by interactions between its carboxyl group and arginine 171. The spatial arrangements of these groups may provide the specificity for the L-isomer of lactate.</td>
</tr>
<tr id="bib_Adams1973" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Adams1973,
  author = {Adams, Margaret J and Buehner, Manfred and Chandrasekhar, K and Ford, Geoffrey C and Hackert, Marvin L and Liljas, Anders and Rossmann, Michael G and Smiley, Ira E and Allison, William S and Everse, J and Kaplan, NO and Taylor, Susan S},
  title = {Structure-function relationships in lactate dehydrogenase.},
  journal = {Proceedings of the National Academy of Sciences},
  year = {1973},
  volume = {70},
  number = {7},
  pages = {1968 - 1972},
  note = {NULL},
  doi = {https://doi.org/10.1073/pnas.70.7.1968}
}
</pre></td>
</tr>
<tr id="Adler2018" class="entry">
	<td>Adler, M., Mayo, A., Zhou, X., Franklin, R.A., Jacox, J.B., Medzhitov, R. and Alon, U.</td>
	<td>Endocytosis as a stabilizing mechanism for tissue homeostasis <p class="infolinks">[<a href="javascript:toggleInfo('Adler2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Proceedings of the National Academy of Sciences, pp. 201714377&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Adler2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Adler2018,
  author = {Adler, Miri and Mayo, Avi and Zhou, Xu and Franklin, Ruth A and Jacox, Jeremy B and Medzhitov, Ruslan and Alon, Uri},
  title = {Endocytosis as a stabilizing mechanism for tissue homeostasis},
  journal = {Proceedings of the National Academy of Sciences},
  publisher = {National Acad Sciences},
  year = {2018},
  pages = {201714377}
}
</pre></td>
</tr>
<tr id="Agarwal2004" class="entry">
	<td>Agarwal, P.K., Geist, A. and Gorin, A.</td>
	<td>Protein dynamics and enzymatic catalysis: Investigating the peptidyl-prolyl cis-trans isomerization activity of cyclophilin A <p class="infolinks">[<a href="javascript:toggleInfo('Agarwal2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Agarwal2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Biochemistry<br/>Vol. 43(33), pp. 10605 - 10618&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi0495228">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Agarwal2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A growing body of evidence suggests a connection between protein dynamics and enzymatic catalysis. In this paper, we present a variety of computational studies designed to investigate the role of protein dynamics in the detailed mechanism of peptidyl-prolyl cis-trans isomerization catalyzed by human cyclophilin A. The results identify a network of protein vibrations, extending from surface regions of the enzyme to the active site and coupled to substrate turnover. Indications are that this network may have a role in promoting catalysis. Crucial parts of this network are found to be conserved in 10 cyclophilin structures from six different species. Experimental evidence for the existence of this network comes from previous NMR relaxation studies, where motions in several residues, forming parts of this network, were detected only during substrate turnover. The high temperature factors (from X-ray crystal structures) associated with the network residues provide further evidence of these vibrations. Along with the knowledge of enzyme structure, this type of network could provide new insights into enzymatic catalysis and the effect of distant ligand binding on protein function. The procedure outlined in this paper is general and can be applied to other enzymatic systems as well. This presents an interesting opportunity; collaborative experimental and theoretical investigations designed to characterize in detail the nature and function of this type of network could enhance the understanding of protein dynamics in enzymatic catalysis.</td>
</tr>
<tr id="bib_Agarwal2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Agarwal2004,
  author = {Agarwal, Pratul K and Geist, Al and Gorin, Andrey},
  title = {Protein dynamics and enzymatic catalysis: Investigating the peptidyl-prolyl cis-trans isomerization activity of cyclophilin A},
  journal = {Biochemistry},
  year = {2004},
  volume = {43},
  number = {33},
  pages = {10605 - 10618},
  doi = {https://doi.org/10.1021/bi0495228}
}
</pre></td>
</tr>
<tr id="Agrawal2010" class="entry">
	<td>Agrawal, N.J., Nukpezah, J. and Radhakrishnan, R.</td>
	<td>Minimal mesoscale model for protein-mediated vesiculation in clathrin-dependent endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Agrawal2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Agrawal2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>PLoS computational biology<br/>Vol. 6&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pcbi.1000926">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Agrawal2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In eukaryotic cells, the internalization of extracellular cargo via the endocytic machinery is an important regulatory process required for many essential cellular functions. The role of cooperative protein-protein and protein-membrane interactions in the ubiquitous endocytic pathway in mammalian cells, namely the clathrin-dependent endocytosis, remains unresolved. We employ the Helfrich membrane Hamiltonian together with surface evolution methodology to address how the shapes and energetics of vesicular-bud formation in a planar membrane are stabilized by presence of the clathrin-coat assembly. Our results identify a unique dual role for the tubulating protein epsin: multiple epsins localized spatially and orientationally collectively play the role of a curvature inducing capsid; in addition, epsin serves the role of an adapter in binding the clathrin coat to the membrane. Our results also suggest an important role for the clathrin lattice, namely in the spatial- and orientational-templating of epsins. We suggest that there exists a critical size of the coat above which a vesicular bud with a constricted neck resembling a mature vesicle is stabilized. Based on the observed strong dependence of the vesicle diameter on the bending rigidity, we suggest that the variability in bending stiffness due to variations in membrane composition with cell type can explain the experimentally observed variability on the size of clathrin-coated vesicles, which typically range 50-100 nm. Our model also provides estimates for the number of epsins involved in stabilizing a coated vesicle, and without any direct fitting reproduces the experimentally observed shapes of vesicular intermediates as well as their probability distributions quantitatively, in wildtype as well as CLAP IgG injected neuronal cell experiments. We have presented a minimal mesoscale model which quantitatively explains several experimental observations on the process of vesicle nucleation induced by the clathrin-coated assembly prior to vesicle scission in clathrin dependent endocytosis.</td>
</tr>
<tr id="bib_Agrawal2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Agrawal2010,
  author = {Agrawal, Neeraj J and Nukpezah, Jonathan and Radhakrishnan, Ravi},
  title = {Minimal mesoscale model for protein-mediated vesiculation in clathrin-dependent endocytosis.},
  journal = {PLoS computational biology},
  year = {2010},
  volume = {6},
  doi = {https://doi.org/10.1371/journal.pcbi.1000926}
}
</pre></td>
</tr>
<tr id="Aguet2013" class="entry">
	<td>Aguet, F., Antonescu, C.N., Mettlen, M., Schmid, S.L. and Danuser, G.</td>
	<td>Advances in analysis of low signal-to-noise images link dynamin and AP2 to the functions of an endocytic checkpoint. <p class="infolinks">[<a href="javascript:toggleInfo('Aguet2013','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Aguet2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Developmental cell<br/>Vol. 26, pp. 279-291&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.devcel.2013.06.019">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Aguet2013" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Numerous endocytic accessory proteins (EAPs) mediate assembly and maturation of clathrin-coated pits (CCPs) into cargo-containing vesicles. Analysis of EAP function through bulk measurement of cargo uptake has been hampered due to potential redundancy among EAPs and, as we show here, the plasticity and resilience of clathrin-mediated endocytosis (CME). Instead, EAP function is best studied by uncovering the correlation between variations in EAP association to individual CCPs and the resulting variations in maturation. However, most EAPs bind to CCPs in low numbers, making the measurement of EAP association via fused fluorescent reporters highly susceptible to detection errors. Here, we present a framework for unbiased measurement of EAP recruitment to CCPs and their direct effects on CCP dynamics. We identify dynamin and the EAP-binding α-adaptin appendage domain of the AP2 adaptor as switches in a regulated, multistep maturation process and provide direct evidence for a molecular checkpoint in CME.</td>
</tr>
<tr id="bib_Aguet2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Aguet2013,
  author = {Aguet, François and Antonescu, Costin N and Mettlen, Marcel and Schmid, Sandra L and Danuser, Gaudenz},
  title = {Advances in analysis of low signal-to-noise images link dynamin and AP2 to the functions of an endocytic checkpoint.},
  journal = {Developmental cell},
  year = {2013},
  volume = {26},
  pages = {279--291},
  doi = {https://doi.org/10.1016/j.devcel.2013.06.019}
}
</pre></td>
</tr>
<tr id="Akhmedov2017" class="entry">
	<td>Akhmedov, M., Kedaigle, A., Chong, R., Montemanni, R., Bertoni, F., Fraenkel, E., Kwee, I., Ma'ayan, A., Epstein, D.M.., Haley, J.D., Jensen, L.J.. and Mering, C.</td>
	<td>PCSF: An R-package for network-based interpretation of high-throughput data <p class="infolinks">[<a href="javascript:toggleInfo('Akhmedov2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Akhmedov2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>PLoS Computational Biology<br/>Vol. 13(7), pp. e1005694 - e1005694&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pcbi.1005694">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Akhmedov2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: With the recent technological developments a vast amount of high-throughput data has been profiled to understand the mechanism of complex diseases. The current bioinformatics challenge is to interpret the data and underlying biology, where efficient algorithms for analyzing heterogeneous high-throughput data using biological networks are becoming increasingly valuable. In this paper, we propose a software package based on the Prize-collecting Steiner Forest graph optimization approach. The PCSF package performs fast and user-friendly network analysis of high-throughput data by mapping the data onto a biological networks such as protein-protein interaction, gene-gene interaction or any other correlation or coexpression based networks. Using the interaction networks as a template, it determines high-confidence subnetworks relevant to the data, which potentially leads to predictions of functional units. It also interactively visualizes the resulting subnetwork with functional enrichment analysis.</td>
</tr>
<tr id="bib_Akhmedov2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Akhmedov2017,
  author = {Akhmedov, Murodzhon and Kedaigle, Amanda and Chong, Renan and Montemanni, Roberto and Bertoni, Francesco and Fraenkel, Ernest and Kwee, Ivo and Ma'ayan, A and Epstein, D.M. and Haley, J.D and Jensen, L.J. and Mering, C.v.},
  title = {PCSF: An R-package for network-based interpretation of high-throughput data},
  journal = {PLoS Computational Biology},
  year = {2017},
  volume = {13},
  number = {7},
  pages = {e1005694 - e1005694},
  doi = {https://doi.org/10.1371/journal.pcbi.1005694}
}
</pre></td>
</tr>
<tr id="Alford2017" class="entry">
	<td>Alford, RF., Leaver-Fay,, A, Jeliazkov, JR., O'Meara,, MJ, DiMaio,, FP, Park, H., Shapovalov, MV., Renfrew, PD., Mulligan, VK., Kappel, K., Labonte, JW., Pacella, MS., Bonneau, R., Bradley, P., Dunbrack, J.RL., Das, R., Baker, D., Kulhman, B., Kortemme, T. and Gray, JG.</td>
	<td>The Rosetta All-Atom Energy Function for Macromolecular Modeling and Design <p class="infolinks">[<a href="javascript:toggleInfo('Alford2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Alford2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of Chemical Theory and Computation<br/>Vol. 13(6), pp. 3031-3048&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jctc.7b00125">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Alford2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In this article, we report only 10 atomic layer thick, high mobility, transparent thin film transistors (TFTs) with ambipolar device characteristics fabricated on both a conventional silicon platform as well as on a flexible substrate. Monolayer graphene was used as metal electrodes, 3−4 atomic layers of h-BN were used as the gate dielectric, and finally bilayers of WSe2 were used as the semiconducting channel material for the TFTs. The field effect carrier mobility was extracted to be 45 cm2/(V s), which exceeds the mobility values of state of the art amorphous silicon based TFTs by ∼100 times. The active device stack of WSe2−hBN− graphene was found to be more than 88&#37; transparent over the entire visible spectrum and the device characteristics were unaltered for in-plane mechanical strain of up to 2%. The device demonstrated remarkable temperature stability over 77−400 K. Low contact resistance value of 1.4 kΩ-μm, subthreshold slope of 90 mv/decade, current ON−OFF ratio of 107, and presence of both electron and hole conduction were observed in our all two-dimensional (2D) TFTs, which are extremely desirable but rarely reported characteristics of most of the organic and inorganic TFTs. To the best of our knowledge, this is the first</td>
</tr>
<tr id="bib_Alford2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Alford2017,
  author = {Alford, RF and A, Leaver-Fay, and Jeliazkov, JR and MJ, O'Meara, and FP, DiMaio, and Park, H and Shapovalov, MV and Renfrew, PD and Mulligan, VK and Kappel, K and Labonte, JW and Pacella, MS and Bonneau, R and Bradley, P and Dunbrack, Jr. RL and Das, R and Baker, D and Kulhman, B and Kortemme, T and Gray, JG},
  title = {The Rosetta All-Atom Energy Function for Macromolecular Modeling and Design},
  journal = {Journal of Chemical Theory and Computation},
  year = {2017},
  volume = {13},
  number = {6},
  pages = {3031-3048},
  doi = {https://doi.org/10.1021/acs.jctc.7b00125}
}
</pre></td>
</tr>
<tr id="Alhambra2001" class="entry">
	<td>Alhambra, C., Corchado, J., S&aacute;nchez, M., Mireia, Garcia-Viloca,., Gao, J. and Truhlar, D.G.</td>
	<td>Canonical variational theory for enzyme kinetics with the protein mean force and multidimensional quantum mechanical tunneling dynamics. Theory and application to liver alcohol dehydrogenase <p class="infolinks">[<a href="javascript:toggleInfo('Alhambra2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Alhambra2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 105(45), pp. 11326 - 11340&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp0120312">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Alhambra2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present a theoretical framework for the calculation of rate constants of enzyme-catalyzed reactions that combines variational optimization of the dynamical bottleneck for overbarrier reactive events and multidimensional quantum mechanical tunneling dynamics for through-barrier reactive events, both in the presence of the protein environment. The theory features a two-zone, three-stage procedure called ensemble-averaged variational transition state theory with multidimensional tunneling (EA-VTST/MT) with the transmission coefficient based on the equilibrium secondary-zone (ESZ) approximation for including the effects of the protein on a catalytic reaction center, called the primary zone. The dynamics is calculated by canonical variational theory with optimized multidimensional tunneling contributions, and the formalism allows for Boltzmann averaging over an ensemble of reactant and transition state conformations. In the first stage of the calculations, we assume that the generalized transition states can be well described by a single progress coordinate expressed in primary-zone internal coordinates; in subsequent steps, the transmission coefficient is averaged over a set of primary-zone reaction paths that depend on the protein configuration, and each reaction path has its own reaction coordinate and optimized tunneling path. We also present a simpler approximation to the transmission coefficient that is called the static secondary-zone (SSZ) approximation. We illustrate both versions of this method by carrying out calculations of the reaction rate constants and kinetic isotope effects for oxidation of benzyl alcoholate to benzaldehyde by horse liver alcohol dehydrogenase. The potential energy surface is modeled by a combined generalized hybrid orbital/quantum mechanical/molecular mechanical/semiempirical valence bond (GHO-QM/MM/SEVB) method. The multidimensional tunneling calculations are microcanonically optimized by employing both the small-curvature tunneling approximation and version 4 of the large-curvature tunneling approximation. We find that the variation of the protein mean force as a function of reaction coordinate is quantitatively significant, but it does not change the qualitative conclusions for the present reaction. We obtain good agreement with experiment for both kinetic isotope effects and Swain?Schaad exponents.</td>
</tr>
<tr id="bib_Alhambra2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Alhambra2001,
  author = {Alhambra, Crist&oacute;bal and Corchado, Jos&eacute; and S&aacute;nchez, Maria and Mireia, Garcia-Viloca, and Gao, Jiali and Truhlar, Donald G},
  title = {Canonical variational theory for enzyme kinetics with the protein mean force and multidimensional quantum mechanical tunneling dynamics. Theory and application to liver alcohol dehydrogenase},
  journal = {Journal of Physical Chemistry B},
  year = {2001},
  volume = {105},
  number = {45},
  pages = {11326 - 11340},
  doi = {https://doi.org/10.1021/jp0120312}
}
</pre></td>
</tr>
<tr id="Alhambra2000" class="entry">
	<td>Alhambra, C., Corchado, J.C., Sanchez, M., Gao, J. and Truhlar, D.G.</td>
	<td>Quantum dynamics of hydride transfer in enzyme catalysis <p class="infolinks">[<a href="javascript:toggleInfo('Alhambra2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Alhambra2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Journal of the American Chemical Society<br/>Vol. 122(34), pp. 8197 - 8203&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ja001476l">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Alhambra2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: One of the strongest experimental indications of hydrogen tunneling in biology has been the elevated Swain-Schaad exponent for the secondary kinetic isotope effect in the hydride-transfer step catalyzed by liver alcohol dehydrogenase. This process has been simulated using canonical variational transition-state theory for overbarrier dynamics and optimized multidimensional paths for tunneling. Semiclassical quantum effects on the dynamics are included on a 21-atom substrate-enzyme-coenzyme primary zone embedded in the potential of a substrate-enzyme-coenzyme-solvent secondary zone. The potential energy surface is calculated by treating 54 atoms by quantum mechanical electronic structure methods and 5506 protein, coenzyme, and solvent atoms by molecular mechanical force fields. We find an elevated Swain-Schaad exponent for the secondary kinetic isotope effect and generally good agreement with other experimental observables. Quantum mechanical tunneling is calculated to account for ∼60&#37; of the reactive flux, confirming the dominance of tunneling that was inferred from the Swain-Schaad exponent. The calculations provide a detailed picture of the origin of the kinetic isotope effect and the nature of the tunneling process.</td>
</tr>
<tr id="bib_Alhambra2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Alhambra2000,
  author = {Alhambra, Cristobal and Corchado, Jose C and Sanchez, Maria and Gao, Jiali and Truhlar, Donald G},
  title = {Quantum dynamics of hydride transfer in enzyme catalysis},
  journal = {Journal of the American Chemical Society},
  year = {2000},
  volume = {122},
  number = {34},
  pages = {8197 - 8203},
  doi = {https://doi.org/10.1021/ja001476l}
}
</pre></td>
</tr>
<tr id="Alhambra1999" class="entry">
	<td>Alhambra, C., Gao, J., Corchado, J., Villa, J. and Truhlar, D.</td>
	<td>Quantum Mechanical Dynamical Effects in an Enzyme-Catalyzed Proton Transfer Reaction <p class="infolinks">[<a href="javascript:toggleInfo('Alhambra1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Alhambra1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Journal of the American Chemical Society<br/>Vol. 121(10), pp. 2253 - 2258&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/JA9831655">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Alhambra1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We have calculated the reaction rate and kinetic isotope effects for conversion of 2-phospho-d-glycerate to phosphoenolpyruvate by yeast enolase. The potential energy surface is modeled by a combined quantum mechanical/molecular mechanical method with generalized hybrid orbitals. The dynamics calculations are carried out by semiclassical variational transition state theory with multidimensional tunneling contributions. Quantum effects are included for a 25-atom cluster consisting of the substrate and part of the protein embedded in a rigid framework consisting of the rest of the protein and water. Quantum effects are important for calculating the absolute rate constant, and variational optimization of the dynamical bottleneck location is important for calculating the kinetic isotope effects. This provides the first evidence that transition state geometries are isotope dependent for enzyme reactions.</td>
</tr>
<tr id="bib_Alhambra1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Alhambra1999,
  author = {Alhambra, C and Gao, J and Corchado, J and Villa, J and Truhlar, DG},
  title = {Quantum Mechanical Dynamical Effects in an Enzyme-Catalyzed Proton Transfer Reaction},
  journal = {Journal of the American Chemical Society},
  year = {1999},
  volume = {121},
  number = {10},
  pages = {2253 - 2258},
  doi = {https://doi.org/10.1021/JA9831655}
}
</pre></td>
</tr>
<tr id="Aloy2006" class="entry">
	<td>Aloy, P. and Russell, R.B.</td>
	<td>Structural systems biology: modelling protein interactions <p class="infolinks">[<a href="javascript:toggleInfo('Aloy2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Aloy2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Nature(3), pp. 188-97&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nrm1859">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Aloy2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Much of systems biology aims to predict the behaviour of biological systems on the basis of the set of molecules involved. Understanding the interactions between these molecules is therefore crucial to such efforts. Although many thousands of interactions are known, precise molecular details are available for only a tiny fraction of them. The difficulties that are involved in experimentally determining atomic structures for interacting proteins make predictive methods essential for progress. Structural details can ultimately turn abstract system representations into models that more accurately reflect biological reality.</td>
</tr>
<tr id="bib_Aloy2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Aloy2006,
  author = {Aloy, Patrick and Russell, Robert B},
  title = {Structural systems biology: modelling protein interactions},
  journal = {Nature},
  year = {2006},
  number = {3},
  pages = {188-97},
  doi = {https://doi.org/10.1038/nrm1859}
}
</pre></td>
</tr>
<tr id="Al-Qodah2006" class="entry">
	<td>Al-Qodah, Z.</td>
	<td>Production and characterization of thermostable α-amylase by thermophilic Geobacillus stearothermophilus <p class="infolinks">[<a href="javascript:toggleInfo('Al-Qodah2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Biotechnology Journal<br/>Vol. 1, pp. 850 - 857&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Al-Qodah2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Al-Qodah2006,
  author = {Al-Qodah, Zakaria},
  title = {Production and characterization of thermostable α-amylase by thermophilic Geobacillus stearothermophilus},
  journal = {Biotechnology Journal},
  year = {2006},
  volume = {1},
  pages = {850 - 857}
}
</pre></td>
</tr>
<tr id="Altan2018" class="entry">
	<td>Altan, I., Fusco, D., Afonine, P.V. and Charbonneau, P.</td>
	<td>Learning about Biomolecular Solvation from Water in Protein Crystals <p class="infolinks">[<a href="javascript:toggleInfo('Altan2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>The Journal of Physical Chemistry B&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jpcb.7b09898">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Altan2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Altan2018,
  author = {Irem Altan and Diana Fusco and Pavel V. Afonine and Patrick Charbonneau},
  title = {Learning about Biomolecular Solvation from Water in Protein Crystals},
  journal = {The Journal of Physical Chemistry B},
  publisher = {American Chemical Society (ACS)},
  year = {2018},
  doi = {https://doi.org/10.1021/acs.jpcb.7b09898}
}
</pre></td>
</tr>
<tr id="Altmann1989" class="entry">
	<td>Altmann, S.L.</td>
	<td>Hamilton, Rodrigues, and the quaternion scandal <p class="infolinks">[<a href="javascript:toggleInfo('Altmann1989','bibtex')">BibTeX</a>]</p></td>
	<td>1989</td>
	<td>Mathematics Magazine<br/>Vol. 62(5), pp. 291-308&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Altmann1989" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Altmann1989,
  author = {Altmann, Simon L},
  title = {Hamilton, Rodrigues, and the quaternion scandal},
  journal = {Mathematics Magazine},
  publisher = {JSTOR},
  year = {1989},
  volume = {62},
  number = {5},
  pages = {291--308}
}
</pre></td>
</tr>
<tr id="Amadei1993" class="entry">
	<td>Amadei, A., Linssen, A.BM. and Berendsen, H.JC.</td>
	<td>Essential Dyanmics of Proteins <p class="infolinks">[<a href="javascript:toggleInfo('Amadei1993','bibtex')">BibTeX</a>]</p></td>
	<td>1993</td>
	<td>Proteins<br/>Vol. 17, pp. 412 - 425&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Amadei1993" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Amadei1993,
  author = {Amadei, Andrea and Linssen, Antonius BM and Berendsen, Herman JC},
  title = {Essential Dyanmics of Proteins},
  journal = {Proteins},
  year = {1993},
  volume = {17},
  pages = {412 - 425}
}
</pre></td>
</tr>
<tr id="Amzel1997" class="entry">
	<td>Amzel, M.L.</td>
	<td>Loss of translational entropy in binding, folding, and catalysis <p class="infolinks">[<a href="javascript:toggleInfo('Amzel1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Amzel1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Proteins: Structure, Function and Genetics<br/>Vol. 28(2), pp. 144 - 149&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/(SICI)1097-0134(199706)28">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Amzel1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: There is a loss of translational entropy associated with the formation of a complex between two molecules in solution. Estimation of this contribution is essential for understanding binding, protein-protein association, and catalysis. Based on the cell model of liquids, it is possible to estimate the loss of translational entropy in all these cases. The resulting formulas are straightforward, and the calculations are easy to perform. Comparison of the results with experimental data suggests that the proposed method provides estimates that are much more accurate than those obtained with existing methods.</td>
</tr>
<tr id="bib_Amzel1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Amzel1997,
  author = {Amzel, Mario L},
  title = {Loss of translational entropy in binding, folding, and catalysis},
  journal = {Proteins: Structure, Function and Genetics},
  year = {1997},
  volume = {28},
  number = {2},
  pages = {144 - 149},
  doi = {https://doi.org/10.1002/(SICI)1097-0134(199706)28}
}
</pre></td>
</tr>
<tr id="An2009" class="entry">
	<td>An, G.</td>
	<td>Dynamic knowledge representation using agent-based modeling: Ontology instantiation and verification of conceptual models <p class="infolinks">[<a href="javascript:toggleInfo('An2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('An2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td><br/>Vol. 500(1), pp. 445&nbsp;</td>
	<td>book</td>
	<td><a href="https://doi.org/10.1007/978-1-59745-525-1_15">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_An2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The sheer volume of biomedical research threatens to overwhelm the capacity of individuals to effectively process this information. Adding to this challenge is the multiscale nature of both biological systems and the research community as a whole. Given this volume and rate of generation of biomedical information, the research community must develop methods for robust representation of knowledge in order for individuals, and the community as a whole, to "know what they know." Despite increasing emphasis on "data-driven" research, the fact remains that researchers guide their research using intuitively constructed conceptual models derived from knowledge extracted from publications, knowledge that is generally qualitatively expressed using natural language. Agent-based modeling (ABM) is a computational modeling method that is suited to translating the knowledge expressed in biomedical texts into dynamic representations of the conceptual models generated by researchers. The hierarchical object-class orientation of ABM maps well to biomedical ontological structures, facilitating the translation of ontologies into instantiated models. Furthermore, ABM is suited to producing the nonintuitive behaviors that often "break" conceptual models. Verification in this context is focused at determining the plausibility of a particular conceptual model, and qualitative knowledge representation is often sufficient for this goal. Thus, utilized in this fashion, ABM can provide a powerful adjunct to other computational methods within the research process, as well as providing a metamodeling framework to enhance the evolution of biomedical ontologies.</td>
</tr>
<tr id="bib_An2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{An2009,
  author = {An, Gary},
  title = {Dynamic knowledge representation using agent-based modeling: Ontology instantiation and verification of conceptual models},
  publisher = {springer},
  year = {2009},
  volume = {500},
  number = {1},
  pages = {445},
  doi = {https://doi.org/10.1007/978-1-59745-525-1_15}
}
</pre></td>
</tr>
<tr id="Ander2004" class="entry">
	<td>Ander, M., Beltrao, P., Di Ventura, B., Ferkinghoff-Borg, J., Foglierini, M., Kaplan, A., Lemerle, C., Tomás-Oliveira, I. and Serrano, L.</td>
	<td>SmartCell, a framework to simulate cellular processes that combines stochastic approximation with diffusion and localisation: analysis of simple networks. <p class="infolinks">[<a href="javascript:toggleInfo('Ander2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ander2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Systems biology<br/>Vol. 1, pp. 129-138&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Ander2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: SmartCell has been developed to be a general framework for modelling and simulation of diffusion-reaction networks in a whole-cell context. It supports localisation and diffusion by using a mesoscopic stochastic reaction model. The SmartCell package can handle any cell geometry, considers different cell compartments, allows localisation of species, supports DNA transcription and translation, membrane diffusion and multistep reactions, as well as cell growth. Moreover, different temporal and spatial constraints can be applied to the model. A GUI interface that facilitates model making is also available. In this work we discuss limitations and advantages arising from the approach used in SmartCell and determine the impact of localisation on the behaviour of simple well-defined networks, previously analysed with differential equations. Our results show that this factor might play an important role in the response of networks and cannot be neglected in cell simulations.</td>
</tr>
<tr id="bib_Ander2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ander2004,
  author = {Ander, M and Beltrao, P and Di Ventura, B and Ferkinghoff-Borg, J and Foglierini, M and Kaplan, A and Lemerle, C and Tomás-Oliveira, I and Serrano, L},
  title = {SmartCell, a framework to simulate cellular processes that combines stochastic approximation with diffusion and localisation: analysis of simple networks.},
  journal = {Systems biology},
  year = {2004},
  volume = {1},
  pages = {129--138}
}
</pre></td>
</tr>
<tr id="Andersen2002" class="entry">
	<td>Andersen, C., Palmer, A.G., Brunak, S. and Rost, B.</td>
	<td>Continuum Secondary Structure Captures Protein Flexibility <p class="infolinks">[<a href="javascript:toggleInfo('Andersen2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Andersen2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Structure<br/>Vol. 10(2), pp. 175 - 184&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S0969-2126(02)00700-1">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Andersen2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The DSSP program assigns protein secondary structure to one of eight states. This discrete assignment cannot describe the continuum of thermal fluctuations. Hence, a continuous assignment is proposed. Technically, the continuum results from averaging over ten discrete DSSP assignments with different hydrogen bond thresholds. The final continuous assignment for a single NMR model successfully reflected the structural variations observed between all NMR models in the ensemble. The structural variations between NMR models were verified to correlate with thermal motion; these variations were captured by the continuous assignments. Because the continuous assignment reproduces the structural variation between many NMR models from one single model, functionally important variation can be extracted from a single X-ray structure. Thus, continuous assignments of secondary structure may affect future protein structure analysis, comparison, and prediction.</td>
</tr>
<tr id="bib_Andersen2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Andersen2002,
  author = {Andersen, Claus and Palmer, Arthur G and Brunak, S&oslash;ren and Rost, Burkhard},
  title = {Continuum Secondary Structure Captures Protein Flexibility},
  journal = {Structure},
  year = {2002},
  volume = {10},
  number = {2},
  pages = {175 - 184},
  note = {Continuum Secondary Structure paper (continuous assignment of secondary structure features). Enhancement of DSSP},
  doi = {https://doi.org/10.1016/S0969-2126(02)00700-1}
}
</pre></td>
</tr>
<tr id="Andres1994" class="entry">
	<td>Andres, J., Moliner, V. and Safont, VS.</td>
	<td>Theoretical kinetic isotope effects for the hydride-transfer step in lactate dehydrogenase <p class="infolinks">[<a href="javascript:toggleInfo('Andres1994','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Andres1994','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td>Journal of the American Chemical Society<br/>Vol. 90(12), pp. 1703 - 1707&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1039/FT9949001703">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Andres1994" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The transition-state (TS) structure for the hydride transfer in lactate dehydrogenase (LDH) enzyme has been calculated with analytical gradients at MNDO, AM1 and PM3 semiempirical levels. The TS is a first-order saddle point on the hypersurface. The geometrical arrangement of the TS results in optimal frontier orbital interaction. Information extracted from these structures is used to calculate theoretical kinetic isotope effects (KIE). The results are independent of the semiempirical procedure. The transition vector suggests that primary and secondary KIEs must be strongly coupled. The relative orientation imposed by the active-site constraints on the reactants in LDH (endo configuration) is optimal for the hydride transfer and situates the system in the neighbourhood of the saddle point.By comparison of this TS with other first-order saddle points associated with the hydride-transfer process in another related enzyme's active site, the structure of the moieties and dynamical fluctuations in the TS appear to be fairly transferable and invariant for the dehydrogenase enzyme family.When the reactants are moulded into the quadratic zone of the TS structure, the KIE can be rendered in fairly good qualitative agreement. This result suggests that the active site moulds the reactants into the structure and fluctuation patterns of the calculated TS.</td>
</tr>
<tr id="bib_Andres1994" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Andres1994,
  author = {Andres, J and Moliner, V and Safont, VS},
  title = {Theoretical kinetic isotope effects for the hydride-transfer step in lactate dehydrogenase},
  journal = {Journal of the American Chemical Society},
  year = {1994},
  volume = {90},
  number = {12},
  pages = {1703 - 1707},
  note = {NULL},
  doi = {https://doi.org/10.1039/FT9949001703}
}
</pre></td>
</tr>
<tr id="Andrews2010" class="entry">
	<td>Andrews, S.S., Addy, N.J., Brent, R. and Arkin, A.P.</td>
	<td>Detailed Simulations of Cell Biology with Smoldyn 2.1 <p class="infolinks">[<a href="javascript:toggleInfo('Andrews2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Andrews2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>PLoS Computational Biology<br/>Vol. 6(3), pp. e1000705&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pcbi.1000705">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Andrews2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Most cellular processes depend on intracellular locations and random collisions of individual protein molecules. To model these processes, we developed algorithms to simulate the diffusion, membrane interactions, and reactions of individual molecules, and implemented these in the Smoldyn program. Compared to the popular MCell and ChemCell simulators, we found that Smoldyn was in many cases more accurate, more computationally efficient, and easier to use. Using Smoldyn, we modeled pheromone response system signaling among yeast cells of opposite mating type. This model showed that secreted Bar1 protease might help a cell identify the fittest mating partner by sharpening the pheromone concentration gradient. This model involved about 200,000 protein molecules, about 7000 cubic microns of volume, and about 75 minutes of simulated time; it took about 10 hours to run. Over the next several years, as faster computers become available, Smoldyn will allow researchers to model and explore systems the size of entire bacterial and smaller eukaryotic cells.</td>
</tr>
<tr id="bib_Andrews2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Andrews2010,
  author = {Andrews, Steven S and Addy, Nathan J and Brent, Roger and Arkin, Adam P},
  title = {Detailed Simulations of Cell Biology with Smoldyn 2.1},
  journal = {PLoS Computational Biology},
  year = {2010},
  volume = {6},
  number = {3},
  pages = {e1000705},
  doi = {https://doi.org/10.1371/journal.pcbi.1000705}
}
</pre></td>
</tr>
<tr id="Antonescu2011" class="entry">
	<td>Antonescu, C.N., Aguet, F., Danuser, G. and Schmid, S.L.</td>
	<td>Phosphatidylinositol-(4,5)-bisphosphate regulates clathrin-coated pit initiation, stabilization, and size. <p class="infolinks">[<a href="javascript:toggleInfo('Antonescu2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Antonescu2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Molecular biology of the cell<br/>Vol. 22, pp. 2588-2600&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1091/mbc.E11-04-0362">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Antonescu2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis (CME) is the major mechanism for internalization in mammalian cells. CME initiates by recruitment of adaptors and clathrin to form clathrin-coated pits (CCPs). Nearly half of nascent CCPs abort, whereas others are stabilized by unknown mechanisms and undergo further maturation before pinching off to form clathrin-coated vesicles (CCVs). Phosphatidylinositol-(4,5)-bisphosphate (PIP(2)), the main lipid binding partner of endocytic proteins, is required for CCP assembly, but little is currently known about its contribution(s) to later events in CCV formation. Using small interfering RNA (siRNA) knockdown and overexpression, we have analyzed the effects of manipulating PIP(2) synthesis and turnover on CME by quantitative total internal reflection fluorescence microscopy and computational analysis. Phosphatidylinositol-4-phosphate-5-kinase cannot be detected within CCPs but functions in initiation and controls the rate and extent of CCP growth. In contrast, the 5'-inositol phosphatase synaptojanin 1 localizes to CCPs and controls early stabilization and maturation efficiency. Together these results suggest that the balance of PIP(2) synthesis in the bulk plasma membrane and its local turnover within CCPs control multiple stages of CCV formation.</td>
</tr>
<tr id="bib_Antonescu2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Antonescu2011,
  author = {Antonescu, Costin N and Aguet, François and Danuser, Gaudenz and Schmid, Sandra L},
  title = {Phosphatidylinositol-(4,5)-bisphosphate regulates clathrin-coated pit initiation, stabilization, and size.},
  journal = {Molecular biology of the cell},
  year = {2011},
  volume = {22},
  pages = {2588--2600},
  doi = {https://doi.org/10.1091/mbc.E11-04-0362}
}
</pre></td>
</tr>
<tr id="Antoniou2006" class="entry">
	<td>Antoniou, D., Basner, J., N&uacute;&ntilde;ez, S. and Schwartz, S.D.</td>
	<td>Computational and theoretical methods to explore the relation between enzyme dynamics and catalysis. <p class="infolinks">[<a href="javascript:toggleInfo('Antoniou2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Chemical reviews<br/>Vol. 106(8), pp. 3170 - 87&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/cr0503052">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Antoniou2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Antoniou2006,
  author = {Antoniou, Dimitri and Basner, Jodi and N&uacute;&ntilde;ez, Sara and Schwartz, Steven D},
  title = {Computational and theoretical methods to explore the relation between enzyme dynamics and catalysis.},
  journal = {Chemical reviews},
  year = {2006},
  volume = {106},
  number = {8},
  pages = {3170 - 87},
  doi = {https://doi.org/10.1021/cr0503052}
}
</pre></td>
</tr>
<tr id="Antoniou1998" class="entry">
	<td>Antoniou, D. and Schwartz, S.D.</td>
	<td>Activated chemistry in the presence of a strongly symmetrically coupled vibration <p class="infolinks">[<a href="javascript:toggleInfo('Antoniou1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Antoniou1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>The Journal of Chemical Physics<br/>Vol. 108(9), pp. 3620 - 3625&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.475756">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Antoniou1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In the gas phase, tunneling reaction rates can be significantly enhanced if the reaction coordinate is symmetrically coupled to a harmonic oscillation, as has been emphasized by Benderskii and co-workers [Adv. Chem. Phys. 88, 1 (1994)]. This is due to the fact that the symmetric coupling leads to modulation of the barrier height. Similar effects have been observed in reactions in model condensed phase studies, as in the Hamiltonians that have been studied by Borgis and Hynes [J. Chem. Phys. 94, 3619 (1991)] and Suarez and Silbey [J. Chem. Phys. 94, 4809 (1991)]. All of these works assume that tunneling proceeds from the ground state. In this paper, we use the exponential resummation technique that we used in our recent work on the quantum Kramers problem, to study the case when there can be excitations to higher states and activated transmission over a barrier. We present a general methodology to exactly include direct coupling between the reaction coordinate and the symmetrically coupled promoting vibrat...</td>
</tr>
<tr id="bib_Antoniou1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Antoniou1998,
  author = {Antoniou, Dimitri and Schwartz, Steven D},
  title = {Activated chemistry in the presence of a strongly symmetrically coupled vibration},
  journal = {The Journal of Chemical Physics},
  year = {1998},
  volume = {108},
  number = {9},
  pages = {3620 - 3625},
  doi = {https://doi.org/10.1063/1.475756}
}
</pre></td>
</tr>
<tr id="Antoniou2009" class="entry">
	<td>Antoniou, D. and Schwartz, S.D.</td>
	<td>Approximate inclusion of quantum effects in transition path sampling <p class="infolinks">[<a href="javascript:toggleInfo('Antoniou2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Antoniou2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>The Journal of Chemical Physics<br/>Vol. 131(22), pp. 224111 - 224111&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.3272793">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Antoniou2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We propose a method for incorporating nuclear quantum effects in transition path sampling studies of systems that consist of a few degrees of freedom that must be treated quantum mechanically, while the rest are classical-like. We used the normal mode centroid method to describe the quantum subsystem, which is a method that is not CPU intensive but still reasonably accurate. We applied this mixed centroid/classical transition path sampling method to a model system that has nontrivial quantum behavior, and showed that it can capture the correct quantum dynamical features.</td>
</tr>
<tr id="bib_Antoniou2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Antoniou2009,
  author = {Antoniou, Dimitri and Schwartz, Steven D},
  title = {Approximate inclusion of quantum effects in transition path sampling},
  journal = {The Journal of Chemical Physics},
  year = {2009},
  volume = {131},
  number = {22},
  pages = {224111 - 224111},
  doi = {https://doi.org/10.1063/1.3272793}
}
</pre></td>
</tr>
<tr id="Antoniou2009a" class="entry">
	<td>Antoniou, D. and Schwartz, S.D.</td>
	<td>The stochastic separatrix and the reaction coordinate for complex systems <p class="infolinks">[<a href="javascript:toggleInfo('Antoniou2009a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Antoniou2009a','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Journal of Chemical Physics<br/>Vol. 130(15), pp. 151103&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.3123162">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Antoniou2009a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present a new approach to the identification of degrees of freedom which comprise a reaction coordinate in a complex system. The method begins with the generation of an ensemble of reactive trajectories. Each trajectory is analyzed for its equicommittor position or transition state; then the transition state ensemble is identified as the stochastic separatrix. Numerical analysis of the points along the separatrix for variability of coordinate location correctly identifies the components of the reaction coordinate in a test system of a double well coupled to a promoting vibration and a bath of linearly coupled oscillators.</td>
</tr>
<tr id="bib_Antoniou2009a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Antoniou2009a,
  author = {Antoniou, Dimitri and Schwartz, Steven D},
  title = {The stochastic separatrix and the reaction coordinate for complex systems},
  journal = {Journal of Chemical Physics},
  year = {2009},
  volume = {130},
  number = {15},
  pages = {151103},
  doi = {https://doi.org/10.1063/1.3123162}
}
</pre></td>
</tr>
<tr id="Antoniou2011" class="entry">
	<td>Antoniou, D. and Schwartz, S.D.</td>
	<td>Protein dynamics and enzymatic chemical barrier passage <p class="infolinks">[<a href="javascript:toggleInfo('Antoniou2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 115, pp. 15147 - 15158&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Antoniou2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Antoniou2011,
  author = {Antoniou, Dimitri and Schwartz, Steven D},
  title = {Protein dynamics and enzymatic chemical barrier passage},
  journal = {Journal of Physical Chemistry B},
  year = {2011},
  volume = {115},
  pages = {15147 - 15158},
  note = {NULL}
}
</pre></td>
</tr>
<tr id="Antoniou2016" class="entry">
	<td>Antoniou, D. and Schwartz, S.D.</td>
	<td>Phase Space Bottlenecks in Enzymatic Reactions <p class="infolinks">[<a href="javascript:toggleInfo('Antoniou2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Antoniou2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 120(3), pp. 433 - 439&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jpcb.5b11157">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Antoniou2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The definition of a transition state on an individual reactive trajectory is made via a committor analysis. In the past, the bottleneck definition has often been applied in configuration space. This is an approximation, and in order to expand this definition, we are revisiting an enzyme in which we had identified a fast subpicosecond motion that makes the reaction possible. First we used a time-series analysis method to identify the exact time when this motion initiates donor−acceptor compression. Then we modified the standard committor analysis of transition path sampling to identify events in phase space and found that there is a dividing surface in phase space significantly earlier than the configurationally defined transition-state crossing.</td>
</tr>
<tr id="bib_Antoniou2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Antoniou2016,
  author = {Antoniou, Dimitri and Schwartz, Steven D},
  title = {Phase Space Bottlenecks in Enzymatic Reactions},
  journal = {Journal of Physical Chemistry B},
  year = {2016},
  volume = {120},
  number = {3},
  pages = {433 - 439},
  doi = {https://doi.org/10.1021/acs.jpcb.5b11157}
}
</pre></td>
</tr>
<tr id="Antown2018" class="entry">
	<td>Antown, F., Dragi&ccaron;evi&cacute;, D. and Froyland, G.</td>
	<td>Optimal linear responses for Markov chains and stochastically perturbed dynamical systems <p class="infolinks">[<a href="javascript:toggleInfo('Antown2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Journal of Statistical Physics&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1007/s10955-018-1985-1">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Antown2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Antown2018,
  author = {Antown, Fadi and Dragi&ccaron;evi&cacute;, Davor and Froyland, Gary},
  title = {Optimal linear responses for Markov chains and stochastically perturbed dynamical systems},
  journal = {Journal of Statistical Physics},
  year = {2018},
  doi = {https://doi.org/10.1007/s10955-018-1985-1}
}
</pre></td>
</tr>
<tr id="Apic2001" class="entry">
	<td>Apic, G., Gough, J. and a Teichmann, S.</td>
	<td>An insight into domain combinations. <p class="infolinks">[<a href="javascript:toggleInfo('Apic2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Apic2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Bioinformatics<br/>Vol. 17 Suppl 1(suppl\_1), pp. S83 - S89&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/bioinformatics/17.suppl_1.S83">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Apic2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Domains are the building blocks of all globular proteins, and are units of compact three-dimensional structure as well as evolutionary units. There is a limited repertoire of domain families, so that these domain families are duplicated and combined in different ways to form the set of proteins in a genome. Proteins are gene products. The processes that produce new genes are duplication and recombination as well as gene fusion and fission. We attempt to gain an overview of these processes by studying the structural domains in the proteins of seven genomes from the three kingdoms of life: Eubacteria, Archaea and Eukaryota. We use here the domain and superfamily definitions in Structural Classification of Proteins Database (SCOP) in order to map pairs of adjacent domains in genome sequences in terms of their superfamily combinations. We find 624 out of the 764 superfamilies in SCOP in these genomes, and the 624 families occur in 585 pairwise combinations. Most families are observed in combination with one or two other families, while a few families are very versatile in their combinatorial behaviour. This type of pattern can be described by a scale-free network. Finally, we study domain repeats and we compare the set of the domain combinations in the genomes to those in PDB, and discuss the implications for structural genomics.</td>
</tr>
<tr id="bib_Apic2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Apic2001,
  author = {Apic, G and Gough, J and a Teichmann, S},
  title = {An insight into domain combinations.},
  journal = {Bioinformatics},
  year = {2001},
  volume = {17 Suppl 1},
  number = {suppl1},
  pages = {S83 - S89},
  doi = {https://doi.org/10.1093/bioinformatics/17.suppl_1.S83}
}
</pre></td>
</tr>
<tr id="Arcangeli2001" class="entry">
	<td>Arcangeli, C., Bizzarri, A. and Cannistraro, S.</td>
	<td>Concerted motions in copper plastocyanin and azurin: an essential dynamics study <p class="infolinks">[<a href="javascript:toggleInfo('Arcangeli2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Arcangeli2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Biophysical Chemistry<br/>Vol. 90(1), pp. 45 - 56&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S0301-4622(01)00128-4">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Arcangeli2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Essential dynamics analysis of molecular dynamics simulation trajectories (1.1 ns) of two copper containing electron transfer proteins, plastocyanin and azurin, has been performed. The protein essential modes have been analysed in order to identify large concerted motions which could be relevant for the electron transfer function exerted by these proteins. The analysis, conducted for temporal windows of different lengths along the protein trajectories, shows a rapid convergence and indicates that for both the proteins the predominant internal motions occur in a subspace of only a few degrees of freedom. Moreover, it is found that for both the proteins the likely binding sites (i.e. the hydrophobic and negative patches) with the reaction partners move in a concerted fashion with a few structural regions far from the active site. Such results are discussed in connection with the possible involvement of large concerted motions in the recognition and binding interaction with physiological electron transfer partners.</td>
</tr>
<tr id="bib_Arcangeli2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Arcangeli2001,
  author = {Arcangeli, Caterina and Bizzarri, Anna and Cannistraro, Salvatore},
  title = {Concerted motions in copper plastocyanin and azurin: an essential dynamics study},
  journal = {Biophysical Chemistry},
  year = {2001},
  volume = {90},
  number = {1},
  pages = {45 - 56},
  note = {Application of essential dynamics analysis. Eigenvectors will converge on the timescales of our runs},
  doi = {https://doi.org/10.1016/S0301-4622(01)00128-4}
}
</pre></td>
</tr>
<tr id="Arntsen2017" class="entry">
	<td>Arntsen, C., Chen, C. and Voth, G.A.</td>
	<td>Reactive molecular dynamics models from ab initio molecular dynamics data using relative entropy minimization <p class="infolinks">[<a href="javascript:toggleInfo('Arntsen2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Arntsen2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Chemical Physics Letters<br/>Vol. 683, pp. 573-578&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cplett.2017.04.064">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Arntsen2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present two new multiscale molecular dynamics (MS-RMD) models for the hydrated excess proton in water developed directly from ab initio molecular dynamics (AIMD) simulation data of the same system. The potential of mean force along the proton transfer reaction coordinate and radial distribution functions for the MS-RMD models are shown to faithfully reproduce those of AIMD. The models are developed using an algorithm based on relative entropy minimization, thus demonstrating the ability of the method to rapidly generate accurate and highly efficient reactive MD force fields.</td>
</tr>
<tr id="bib_Arntsen2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Arntsen2017,
  author = {Arntsen, Christopher and Chen, Chen and Voth, Gregory A},
  title = {Reactive molecular dynamics models from ab initio molecular dynamics data using relative entropy minimization},
  journal = {Chemical Physics Letters},
  year = {2017},
  volume = {683},
  pages = {573-578},
  doi = {https://doi.org/10.1016/j.cplett.2017.04.064}
}
</pre></td>
</tr>
<tr id="Bachman2011" class="entry">
	<td>Bachman, J. and Sorger, P.</td>
	<td>New approaches to modeling complex biochemistry <p class="infolinks">[<a href="javascript:toggleInfo('Bachman2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bachman2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Nature Methods<br/>Vol. 8(2), pp. 130 - 131&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nmeth0211-130">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Bachman2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Combining rule-based descriptions of biochemical reactions with agent-based computer simulation opens new avenues for exploring complex cellular processes.</td>
</tr>
<tr id="bib_Bachman2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bachman2011,
  author = {Bachman, John and Sorger, Peter},
  title = {New approaches to modeling complex biochemistry},
  journal = {Nature Methods},
  year = {2011},
  volume = {8},
  number = {2},
  pages = {130 - 131},
  doi = {https://doi.org/10.1038/nmeth0211-130}
}
</pre></td>
</tr>
<tr id="Baker2010" class="entry">
	<td>Baker, D.</td>
	<td>An exciting but challenging road ahead for computational enzyme design. <p class="infolinks">[<a href="javascript:toggleInfo('Baker2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Protein Science<br/>Vol. 19(10), pp. 1817 - 9&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/pro.481">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Baker2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Baker2010,
  author = {Baker, David},
  title = {An exciting but challenging road ahead for computational enzyme design.},
  journal = {Protein Science},
  year = {2010},
  volume = {19},
  number = {10},
  pages = {1817 - 9},
  doi = {https://doi.org/10.1002/pro.481}
}
</pre></td>
</tr>
<tr id="Bakowies1996" class="entry">
	<td>Bakowies, D. and Thiel, W.</td>
	<td>Hybrid Models for Combined Quantum Mechanical and Molecular Mechanical Approaches <p class="infolinks">[<a href="javascript:toggleInfo('Bakowies1996','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bakowies1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>Journal of Physical Chemistry<br/>Vol. 100(25), pp. 10580 - 10594&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp9536514">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Bakowies1996" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A hierarchy of three models for combined quantum mechanical (QM) and molecular mechanical (MM) approaches is presented. They simplify the QM description of large molecules by reducing it to the electronically important fragment which interacts with the molecular mechanically treated remainder of the molecule. In the simplest model A, the QM fragments are only mechanically embedded in their MM environment. The more refined models B and C include a quantum mechanical treatment of electrostatic interactions between the fragments and a semiclassical description of polarization. The implementation of models A-C for MNDO type wavefunctions and the MM3 force field is outlined. Selected applications in organic chemistry are discussed, addressing the ability of the proposed models to reproduce substituent effects (MM) on chemical structure and reactivity (QM). These applications include protonations, deprotonations, hydride transfer reactions, nucleophilic additions, and nucleophilic ring cleavage reactions.</td>
</tr>
<tr id="bib_Bakowies1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bakowies1996,
  author = {Bakowies, Dirk and Thiel, Walter},
  title = {Hybrid Models for Combined Quantum Mechanical and Molecular Mechanical Approaches},
  journal = {Journal of Physical Chemistry},
  year = {1996},
  volume = {100},
  number = {25},
  pages = {10580 - 10594},
  doi = {https://doi.org/10.1021/jp9536514}
}
</pre></td>
</tr>
<tr id="Ball1995" class="entry">
	<td>Ball, CL., Hunt, SP. and Robinson, MS.</td>
	<td>Expression and localization of alpha-adaptin isoforms. <p class="infolinks">[<a href="javascript:toggleInfo('Ball1995','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ball1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>Journal of Cell Science<br/>Vol. 108 ( Pt 8, pp. 2865 - 75&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Ball1995" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: There are two alpha-adaptin genes, alpha A and alpha C, which in brain encode proteins of of M(r) 108 x 10(3) and 104 x 10(3), respectively. Although both mRNAs can be detected on northern blots of brain and liver, the higher molecular mass polypeptide can only be detected on western blots of brain. Here we explain these observations by showing that alpha A is alternatively spliced and that the protein product in most tissues is different from the one expressed in brain in that it is missing 21 amino acids within the hinge region, giving it a similar mobility to that of alpha C. Monospecific antibodies were raised against the various alpha-adaptin isoforms and used to compare their distribution in cells and tissues. Both alpha A and alpha c are co-assembled into the same coated pits, and the larger isoform of alpha A is co-assembled with the smaller isoforms of alpha-adaptin, both in cells that naturally express it an in transfected cells. Examination of brain and spinal cord sections, labelled either for the larger isoform of alpha A or for alpha C, reveals that that the two are to some extent differentially distributed, consistent with previous in situ hybridisation studies. This finding, combined with the observation that there is considerable variability in the relative expression of the two isoforms in different tissues, indicates that the two genes are switched on in response to different stimuli. Moreover, the larger isoform of alpha A appears to be more efficiently concentrated in the nerve terminals than alpha C, which is found not only at the terminals but also diffusely distributed in the cell bodies and dendrites. This suggests that alpha C may play more of a role in the recycling of membrane components throughout the cell.</td>
</tr>
<tr id="bib_Ball1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ball1995,
  author = {Ball, CL and Hunt, SP and Robinson, MS},
  title = {Expression and localization of alpha-adaptin isoforms.},
  journal = {Journal of Cell Science},
  year = {1995},
  volume = {108 ( Pt 8},
  pages = {2865 - 75},
  note = {Two isoforms of AP2 alpha subunit - may be functionally identical}
}
</pre></td>
</tr>
<tr id="Banerjee2012" class="entry">
	<td>Banerjee, A., Berezhkovskii, A. and Nossal, R.</td>
	<td>Stochastic model of clathrin-coated pit assembly. <p class="infolinks">[<a href="javascript:toggleInfo('Banerjee2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Banerjee2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Biophysical journal<br/>Vol. 102, pp. 2725-2730&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.bpj.2012.05.010">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Banerjee2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In recent years, fluorescence microscopy has enabled researchers to observe the dynamics of clathrin-coated pit (CCP) assembly in real time. The assembly dynamics of CCPs shows striking heterogeneity. Some CCPs are long-lived (productive CCPs); they bind cargo and grow in size to form clathrin-coated vesicles. In contrast, other CCPs (abortive CCPs) are relatively short-lived and disassemble well before reaching vesicle size. Within both populations there is significant variance in CCP lifetime. We propose a stochastic biophysical model that links these observations with the energetics of CCPs and kinetics of their assembly. We show that without cargo, CCP assembly faces a high energy barrier that is difficult to overcome. As a consequence, CCPs without cargo are almost always abortive. We suggest a mechanism by which cargo binding stabilizes CCPs and facilitates their growth. The lifetime distribution of abortive pits calculated from our model agrees well with published experimental data. We also estimate the lifetimes of productive CCPs and show that the stochastic nature of CCP assembly plays a crucial role in causing their observed wide distribution.</td>
</tr>
<tr id="bib_Banerjee2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Banerjee2012,
  author = {Banerjee, Anand and Berezhkovskii, Alexander and Nossal, Ralph},
  title = {Stochastic model of clathrin-coated pit assembly.},
  journal = {Biophysical journal},
  year = {2012},
  volume = {102},
  pages = {2725--2730},
  doi = {https://doi.org/10.1016/j.bpj.2012.05.010}
}
</pre></td>
</tr>
<tr id="Barabasi2004" class="entry">
	<td>Barabasi, A. and Oltvai, Z.N.</td>
	<td>Network biology: understanding the cell's functional organization. <p class="infolinks">[<a href="javascript:toggleInfo('Barabasi2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Barabasi2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Nature Reviews Genetics<br/>Vol. 5(2), pp. 101 - 13&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nrg1272">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Barabasi2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A key aim of postgenomic biomedical research is to systematically catalogue all molecules and their interactions within a living cell. There is a clear need to understand how these molecules and the interactions between them determine the function of this enormously complex machinery, both in isolation and when surrounded by other cells. Rapid advances in network biology indicate that cellular networks are governed by universal laws and offer a new conceptual framework that could potentially revolutionize our view of biology and disease pathologies in the twenty-first century.</td>
</tr>
<tr id="bib_Barabasi2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Barabasi2004,
  author = {Barabasi, A and Oltvai, Zolt&aacute;n N},
  title = {Network biology: understanding the cell's functional organization.},
  journal = {Nature Reviews Genetics},
  year = {2004},
  volume = {5},
  number = {2},
  pages = {101 - 13},
  doi = {https://doi.org/10.1038/nrg1272}
}
</pre></td>
</tr>
<tr id="Barabasi2011" class="entry">
	<td>Barabási, A.-L., Gulbahce, N. and Loscalzo, J.</td>
	<td>Network medicine: a network-based approach to human disease. <p class="infolinks">[<a href="javascript:toggleInfo('Barabasi2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Barabasi2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Nature reviews. Genetics<br/>Vol. 12, pp. 56-68&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nrg2918">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Barabasi2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Given the functional interdependencies between the molecular components in a human cell, a disease is rarely a consequence of an abnormality in a single gene, but reflects the perturbations of the complex intracellular and intercellular network that links tissue and organ systems. The emerging tools of network medicine offer a platform to explore systematically not only the molecular complexity of a particular disease, leading to the identification of disease modules and pathways, but also the molecular relationships among apparently distinct (patho)phenotypes. Advances in this direction are essential for identifying new disease genes, for uncovering the biological significance of disease-associated mutations identified by genome-wide association studies and full-genome sequencing, and for identifying drug targets and biomarkers for complex diseases.</td>
</tr>
<tr id="bib_Barabasi2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Barabasi2011,
  author = {Barabási, Albert-László and Gulbahce, Natali and Loscalzo, Joseph},
  title = {Network medicine: a network-based approach to human disease.},
  journal = {Nature reviews. Genetics},
  year = {2011},
  volume = {12},
  pages = {56--68},
  doi = {https://doi.org/10.1038/nrg2918}
}
</pre></td>
</tr>
<tr id="Barkai1997" class="entry">
	<td>Barkai, N. and Leibler, S.</td>
	<td>Robustness in simple biochemical networks. <p class="infolinks">[<a href="javascript:toggleInfo('Barkai1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Barkai1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Nature<br/>Vol. 387(6636), pp. 913 - 7&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/43199">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Barkai1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Cells use complex networks of interacting molecular components to transfer and process information. These "computational devices of living cells" are responsible for many important cellular processes, including cell-cycle regulation and signal transduction. Here we address the issue of the sensitivity of the networks to variations in their biochemical parameters. We propose a mechanism for robust adaptation in simple signal transduction networks. We show that this mechanism applies in particular to bacterial chemotaxis. This is demonstrated within a quantitative model which explains, in a unified way, many aspects of chemotaxis, including proper responses to chemical gradients. The adaptation property is a consequence of the network's connectivity and does not require the 'fine-tuning' of parameters. We argue that the key properties of biochemical networks should be robust in order to ensure their proper functioning.</td>
</tr>
<tr id="bib_Barkai1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Barkai1997,
  author = {Barkai, N and Leibler, S},
  title = {Robustness in simple biochemical networks.},
  journal = {Nature},
  year = {1997},
  volume = {387},
  number = {6636},
  pages = {913 - 7},
  doi = {https://doi.org/10.1038/43199}
}
</pre></td>
</tr>
<tr id="Bateman2004" class="entry">
	<td>Bateman, A., Coin, L., Durbin, R., Finn, R.D., Hollich, V., Grif, S., Khanna, A., Marshall, M., Moxon, S., Sonnhammer, E.LL., Studholme, D.J., Yeats, C. and Eddy, S.R.</td>
	<td>The Pfam protein families database. <p class="infolinks">[<a href="javascript:toggleInfo('Bateman2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bateman2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Nucleic Acids Research<br/>Vol. 32(Database issue), pp. 138D - 41&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/nar/gkh121">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Bateman2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Pfam is a large collection of protein families and domains. Over the past 2 years the number of families in Pfam has doubled and now stands at 6190 (version 10.0). Methodology improvements for searching the Pfam collection locally as well as via the web are described. Other recent innovations include modelling of discontinuous domains allowing Pfam domain definitions to be closer to those found in structure databases. Pfam is available on the web in the UK (http://www.sanger.ac.uk/Software/Pfam/), the USA (http://pfam.wustl.edu/), France (http://pfam.jouy.inra.fr/) and Sweden (http://Pfam.cgb.ki.se/).</td>
</tr>
<tr id="bib_Bateman2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bateman2004,
  author = {Bateman, Alex and Coin, Lachlan and Durbin, Richard and Finn, Robert D and Hollich, Volker and Grif, Sam and Khanna, Ajay and Marshall, Mhairi and Moxon, Simon and Sonnhammer, Erik LL and Studholme, David J and Yeats, Corin and Eddy, Sean R},
  title = {The Pfam protein families database.},
  journal = {Nucleic Acids Research},
  year = {2004},
  volume = {32},
  number = {Database issue},
  pages = {138D - 41},
  doi = {https://doi.org/10.1093/nar/gkh121}
}
</pre></td>
</tr>
<tr id="Baweja2018" class="entry">
	<td>Baweja, L.K. and Roche, J.</td>
	<td>Pushing the Limits of Structure-Based Models: Prediction of Non-Globular Protein Folding and Fibrils Formation with Go-Model Simulations <p class="infolinks">[<a href="javascript:toggleInfo('Baweja2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Baweja2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Baweja2018,
  author = {Baweja, Lokesh Kumar and Roche, Julien},
  title = {Pushing the Limits of Structure-Based Models: Prediction of Non-Globular Protein Folding and Fibrils Formation with Go-Model Simulations},
  publisher = {ACS Publications},
  year = {2018}
}
</pre></td>
</tr>
<tr id="Bean2000" class="entry">
	<td>Bean, A.J., Davanger, S., Chou, M.F., Gerhardt, B., Tsujimoto, S. and Chang, Y.C.</td>
	<td>Hrs-2 regulates receptor-mediated endocytosis via interactions with Eps15 <p class="infolinks">[<a href="javascript:toggleInfo('Bean2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bean2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Journal of Biological Chemistry<br/>Vol. 275(20), pp. 15271 - 15278&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.275.20.15271">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Bean2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Hrs-2, via interactions with SNAP-25, plays a regulatory role on the exocytic machinery. We now show that Hrs-2 physically interacts with Eps15, a protein required for receptor-mediated endocytosis. The Hrs-2/Eps15 interaction is calcium dependent, inhibited by SNAP-25 and alpha-adaptin, and results in the inhibition of receptor-mediated endocytosis. Immunoelectron microscopy reveals Hrs-2 localization on the limiting membrane of multivesicular bodies, organelles in the endosomal pathway. These data show that Hrs-2 regulates endocytosis, delineate a biochemical pathway (Hrs-2-Eps15-AP2) in which Hrs-2 functions, and suggest that Hrs-2 acts to provide communication between endo- and exocytic processes.</td>
</tr>
<tr id="bib_Bean2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bean2000,
  author = {Bean, Andrew J and Davanger, Svend and Chou, Marian F and Gerhardt, Brenda and Tsujimoto, Susan and Chang, Yu Chieh},
  title = {Hrs-2 regulates receptor-mediated endocytosis via interactions with Eps15},
  journal = {Journal of Biological Chemistry},
  year = {2000},
  volume = {275},
  number = {20},
  pages = {15271 - 15278},
  note = {HeLa cells. Some calcium dependent interactions of hrs2 and eps15. interaction between eps15 and alpha-adaptin. doesn't say alpha-a or alpha-c.},
  doi = {https://doi.org/10.1074/jbc.275.20.15271}
}
</pre></td>
</tr>
<tr id="Becker1997" class="entry">
	<td>Becker, OM.</td>
	<td>Geometric versus topological clustering: an insight into conformation mapping. <p class="infolinks">[<a href="javascript:toggleInfo('Becker1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Becker1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Proteins<br/>Vol. 27(2), pp. 213 - 26&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Becker1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clustering molecular conformations into "families" is a common procedure in conformational analysis of molecular systems. An implicit assumption which often underlies this clustering approach is that the resulting geometric families reflect the energetic structure of the system's potential energy surface. In a broader context we address the question whether structural similarity is correlated with energy basins, i.e., whether conformations that belong to the same energy basin are also geometrically similar. 'Topological mapping' and principal coordinate projections are used here to address this question and to assess the quality of the 'family clustering' procedure. Applying the analysis to a small tetrapeptide it was found that the general correlation that exists between energy basins and structural similarity is not absolute. Clusters generated by the geometric 'family clustering' procedure do not always reflect the underlying energy basins. In particular it was found that the 'family tree' that is generated by the 'family clustering' procedure is completely inconsistent with its real topological counterpart, the 'disconnectivity' graph of this system. It is also demonstrated that principal coordinate analysis is a powerful visualization technique which, at least for this system, works better when distances are measured in dihedral angle space rather than in cartesian space.</td>
</tr>
<tr id="bib_Becker1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Becker1997,
  author = {Becker, OM},
  title = {Geometric versus topological clustering: an insight into conformation mapping.},
  journal = {Proteins},
  year = {1997},
  volume = {27},
  number = {2},
  pages = {213 - 26}
}
</pre></td>
</tr>
<tr id="Becker2001" class="entry">
	<td>Becker, O.M.</td>
	<td>Conformational Analysis <p class="infolinks">[<a href="javascript:toggleInfo('Becker2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Computational Biochemistry and Biophysics&nbsp;</td>
	<td>inbook</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Becker2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inbook{Becker2001,
  author = {Becker, Oren M},
  title = {Conformational Analysis},
  booktitle = {Computational Biochemistry and Biophysics},
  publisher = {Marcel Dekker, Inc.},
  year = {2001}
}
</pre></td>
</tr>
<tr id="Becker1997a" class="entry">
	<td>Becker, O.M. and Karplus, M.</td>
	<td>The topology of multidimensional potential energy surfaces: Theory and application to peptide structure and kinetics <p class="infolinks">[<a href="javascript:toggleInfo('Becker1997a','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Journal of Chemical Physics<br/>Vol. 106(4), pp. 1495 - 1517&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Becker1997a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Becker1997a,
  author = {Becker, Oren M and Karplus, Martin},
  title = {The topology of multidimensional potential energy surfaces: Theory and application to peptide structure and kinetics},
  journal = {Journal of Chemical Physics},
  year = {1997},
  volume = {106},
  number = {4},
  pages = {1495 - 1517}
}
</pre></td>
</tr>
<tr id="Beer1998" class="entry">
	<td>de Beer, T., Carter, R.E., Lobel-Rice, K.E., Sorkin, A. and Overduin, M.</td>
	<td>Structure and Asn-Pro-Phe binding pocket of the Eps15 homology domain. <p class="infolinks">[<a href="javascript:toggleInfo('Beer1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Beer1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Science (New York, N.Y.)<br/>Vol. 281, pp. 1357-1360&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Beer1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Eps15 homology (EH) domains are eukaryotic signaling modules that recognize proteins containing Asn-Pro-Phe (NPF) sequences. The structure of the central EH domain of Eps15 has been solved by heteronuclear magnetic resonance spectroscopy. The fold consists of a pair of EF hand motifs, the second of which binds tightly to calcium. The NPF peptide is bound in a hydrophobic pocket between two alpha helices, and binding is mediated by a critical aromatic interaction as revealed by structure-based mutagenesis. The fold is predicted to be highly conserved among 30 identified EH domains and provides a structural basis for defining EH-mediated events in protein trafficking and growth factor signaling.</td>
</tr>
<tr id="bib_Beer1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Beer1998,
  author = {de Beer, T and Carter, R E and Lobel-Rice, K E and Sorkin, A and Overduin, M},
  title = {Structure and Asn-Pro-Phe binding pocket of the Eps15 homology domain.},
  journal = {Science (New York, N.Y.)},
  year = {1998},
  volume = {281},
  pages = {1357--1360}
}
</pre></td>
</tr>
<tr id="Beer2000" class="entry">
	<td>de Beer, T., Hoofnagle, A.N., Enmon, J.L., Bowers, R.C., Yamabhai, M., Kay, B.K. and Overduin, M.</td>
	<td>Molecular mechanism of NPF recognition by EH domains. <p class="infolinks">[<a href="javascript:toggleInfo('Beer2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Beer2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Nature structural biology<br/>Vol. 7, pp. 1018-1022&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/80924">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Beer2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Eps15 homology (EH) domains are protein interaction modules that recognize Asn-Pro-Phe (NPF) motifs in their biological ligands to mediate critical events during endocytosis and signal transduction. To elucidate the structural basis of the EH-NPF interaction, the solution structures of two EH-NPF complexes were solved using NMR spectroscopy. The first complex contains a peptide representing the Hrb C-terminal NPFL motif; the second contains a peptide in which an Arg residue substitutes the C-terminal Leu. The NPF residues are almost completely embedded in a hydrophobic pocket on the EH domain surface and the backbone of NPFX adopts a conformation reminiscent of the Asx-Pro type I beta-turn motif. The residue directly following NPF is crucial for recognition and is required to complete the beta-turn. Five amino acids on the EH surface mediate specific recognition of this residue through hydrophobic and electrostatic contacts. The complexes explain the selectivity of the second EH domain of Eps15 for NPF over DPF motifs and reveal a critical aromatic interaction that provides a conserved anchor for the recognition of FW, WW, SWG and HTF ligands by other EH domains.</td>
</tr>
<tr id="bib_Beer2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Beer2000,
  author = {de Beer, T and Hoofnagle, A N and Enmon, J L and Bowers, R C and Yamabhai, M and Kay, B K and Overduin, M},
  title = {Molecular mechanism of NPF recognition by EH domains.},
  journal = {Nature structural biology},
  year = {2000},
  volume = {7},
  pages = {1018--1022},
  doi = {https://doi.org/10.1038/80924}
}
</pre></td>
</tr>
<tr id="Benmerah1996" class="entry">
	<td>Benmerah, A., B&egrave;gue, B. and Dautry-Varsat,, A</td>
	<td>The ear of-adaptin interacts with the COOH-terminal domain of the Eps15 protein <p class="infolinks">[<a href="javascript:toggleInfo('Benmerah1996','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Benmerah1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>Journal of Biological Chemistry&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.271.20.12111">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Benmerah1996" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Abstract The role of Eps15 in clathrin-mediated endocytosis is supported by two observations. First, it interacts specifically and constitutively with the plasma membrane adaptor AP-2. Second, its NH Graphic terminus shows significant homology to the NH Graphic terminus of yeast End3p, necessary for endocytosis of α-factor. To gain further insight into the role of Eps15-AP-2 association, we have now delineated their sites of  ...</td>
</tr>
<tr id="bib_Benmerah1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Benmerah1996,
  author = {Benmerah, A and B&egrave;gue, B and A, Dautry-Varsat,},
  title = {The ear of-adaptin interacts with the COOH-terminal domain of the Eps15 protein},
  journal = {Journal of Biological Chemistry},
  year = {1996},
  doi = {https://doi.org/10.1074/jbc.271.20.12111}
}
</pre></td>
</tr>
<tr id="Benmerah1995" class="entry">
	<td>Benmerah, A., Gagnon, J. and B&egrave;gue, B.</td>
	<td>The tyrosine kinase substrate eps15 is constitutively associated with the plasma membrane adaptor AP-2. <p class="infolinks">[<a href="javascript:toggleInfo('Benmerah1995','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Benmerah1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>The Journal of Cell Biology&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1083/jcb.131.6.1831">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Benmerah1995" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Abstract The ubiquitous eps15 protein was initially described as a substrate of the EGF receptor kinase. Its functions are not yet delineated and this work provides evidence for its possible role in endocytosis. A novel anti-eps15 antibody, 6G4, coimmunoprecipitated proteins of molecular mass 102 kD. In human cells, these proteins were identified as the alpha-and beta-adaptins of the AP-2 complex on the basis of their NH2-terminal sequence  ...</td>
</tr>
<tr id="bib_Benmerah1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Benmerah1995,
  author = {Benmerah, A and Gagnon, J and B&egrave;gue, B},
  title = {The tyrosine kinase substrate eps15 is constitutively associated with the plasma membrane adaptor AP-2.},
  journal = {The Journal of Cell Biology},
  year = {1995},
  doi = {https://doi.org/10.1083/jcb.131.6.1831}
}
</pre></td>
</tr>
<tr id="Benmerah1998" class="entry">
	<td>Benmerah, A., Lamaze, C., Begue, B., Schmid, S.L., Dautry-Varsat, A. and Cerf-Bensussan, N.</td>
	<td>AP-2/Eps15 interaction is required for receptor-mediated endocytosis <p class="infolinks">[<a href="javascript:toggleInfo('Benmerah1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Benmerah1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>The Journal of Cell Biology<br/>Vol. 140(5), pp. 1055-1062&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1083/jcb.140.5.1055">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Benmerah1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We have previously shown that the protein Eps15 is constitutively associated with the plasma membrane adaptor complex, AP-2, suggesting its possible role in endocytosis. To explore the role of Eps15 and the function of AP-2/Eps15 association in endocytosis, the Eps15 binding domain for AP-2 was precisely delineated. The entire COOH-terminal domain of Eps15 or a mutant form lacking all the AP-2--binding sites was fused to the green  ...</td>
</tr>
<tr id="bib_Benmerah1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Benmerah1998,
  author = {Benmerah, Alexandre and Lamaze, Christophe and Begue, Bernadette and Schmid, Sandra L and Dautry-Varsat, Alice and Cerf-Bensussan, Nadine},
  title = {AP-2/Eps15 interaction is required for receptor-mediated endocytosis},
  journal = {The Journal of Cell Biology},
  year = {1998},
  volume = {140},
  number = {5},
  pages = {1055-1062},
  doi = {https://doi.org/10.1083/jcb.140.5.1055}
}
</pre></td>
</tr>
<tr id="Bennett1977" class="entry">
	<td>Bennett, C.H..</td>
	<td>Algorithms for chemical computations <p class="infolinks">[<a href="javascript:toggleInfo('Bennett1977','bibtex')">BibTeX</a>]</p></td>
	<td>1977</td>
	<td>ACS Symposium Series No. 46, pp. 63&nbsp;</td>
	<td>inbook</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Bennett1977" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inbook{Bennett1977,
  author = {Bennett, C.H.},
  title = {Algorithms for chemical computations},
  booktitle = {ACS Symposium Series No. 46},
  publisher = {American Chemical Society},
  year = {1977},
  pages = {63}
}
</pre></td>
</tr>
<tr id="Bennett1976" class="entry">
	<td>Bennett, C.H.</td>
	<td>Efficient estimation of free energy differences from Monte Carlo data <p class="infolinks">[<a href="javascript:toggleInfo('Bennett1976','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bennett1976','bibtex')">BibTeX</a>]</p></td>
	<td>1976</td>
	<td>Journal of Computational Physics<br/>Vol. 22(2), pp. 245 - 268&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/0021-9991(76)90078-4">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Bennett1976" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Near-optimal strategies are developed for estimating the free energy difference between two canonical ensembles, given a Metropolis-type Monte Carlo program for sampling each one. The estimation strategy depends on the extent of overlap between the two ensembles, on the smoothness of the density-of-states as a function of the difference potential, and on the relative Monte Carlo sampling costs, per statistically independent data point. The best estimate of the free energy difference is usually obtained by dividing the available computer time approximately equally between the two ensembles; its efficiency (variance x computer time)-1 is never less, and may be several orders of magnitude greater, than that obtained by sampling only one ensemble, as is done in perturbation theory. ?? 1976.</td>
</tr>
<tr id="bib_Bennett1976" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bennett1976,
  author = {Bennett, Charles H},
  title = {Efficient estimation of free energy differences from Monte Carlo data},
  journal = {Journal of Computational Physics},
  year = {1976},
  volume = {22},
  number = {2},
  pages = {245 - 268},
  doi = {https://doi.org/10.1016/0021-9991(76)90078-4}
}
</pre></td>
</tr>
<tr id="Berezhkovskii2009" class="entry">
	<td>Berezhkovskii, A., Hummer, G. and Szabo, A.</td>
	<td>Reactive flux and folding pathways in network models of coarse-grained protein dynamics <p class="infolinks">[<a href="javascript:toggleInfo('Berezhkovskii2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Berezhkovskii2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Journal of Chemical Physics<br/>Vol. 130(20), pp. 205102 - 205102&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.3139063">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Berezhkovskii2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The reactive flux between folded and unfolded states of a two-state protein, whose coarse-grained dynamics is described by a master equation, is expressed in terms of the commitment or splitting probabilities of the microstates in the bottleneck region. This allows one to determine how much each transition through a dividing surface contributes to the reactive flux. By repeating the analysis for a series of dividing surfaces or, alternatively, by partitioning the reactive flux into contributions of unidirectional pathways that connect reactants and products, insight can be gained into the mechanism of protein folding. Our results for the flux in a network with complex connectivity, obtained using the discrete counterpart of Kramers' theory of activated rate processes, show that the number of reactive transitions is typically much smaller than the total number of transitions that cross a dividing surface at equilibrium.</td>
</tr>
<tr id="bib_Berezhkovskii2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Berezhkovskii2009,
  author = {Berezhkovskii, Alexander and Hummer, Gerhard and Szabo, Attila},
  title = {Reactive flux and folding pathways in network models of coarse-grained protein dynamics},
  journal = {Journal of Chemical Physics},
  year = {2009},
  volume = {130},
  number = {20},
  pages = {205102 - 205102},
  doi = {https://doi.org/10.1063/1.3139063}
}
</pre></td>
</tr>
<tr id="Berezhkovskii2013" class="entry">
	<td>Berezhkovskii, A.M. and Szabo, A.</td>
	<td>Diffusion along the splitting/commitment probability reaction coordinate <p class="infolinks">[<a href="javascript:toggleInfo('Berezhkovskii2013','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Berezhkovskii2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 117(42), pp. 13115 - 13119&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp403043a">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Berezhkovskii2013" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The splitting or commitment probabilities of states in the region of configuration space that separates reactants and products play an important role in the theory of chemical reactions. Assuming that the splitting probability changes more slowly than any other coordinate, we project multidimensional diffusive dynamics onto it. The resulting one-dimensional diffusion equation is not exact because the assumed separation of time scales does not hold in general. Nevertheless, this equation has the remarkable property that it always predicts the exact value of the number of transitions between reactants and products per unit time at equilibrium and hence the exact reaction rate. In the special case of two deep basins separated by a harmonic saddle, this equation is equivalent to the one that describes diffusion along a coordinate perpendicular to the transition state, defined as the surface starting from which reactants and products are reached with equal probability.</td>
</tr>
<tr id="bib_Berezhkovskii2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Berezhkovskii2013,
  author = {Berezhkovskii, Alexander M and Szabo, Attila},
  title = {Diffusion along the splitting/commitment probability reaction coordinate},
  journal = {Journal of Physical Chemistry B},
  year = {2013},
  volume = {117},
  number = {42},
  pages = {13115 - 13119},
  doi = {https://doi.org/10.1021/jp403043a}
}
</pre></td>
</tr>
<tr id="BergenEnHenegouwen2009" class="entry">
	<td>van Bergen En Henegouwen, P.</td>
	<td>Eps15: a multifunctional adaptor protein regulating intracellular trafficking. <p class="infolinks">[<a href="javascript:toggleInfo('BergenEnHenegouwen2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('BergenEnHenegouwen2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Cell communication and signaling : CCS<br/>Vol. 7, pp. 24&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1186/1478-811X-7-24">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_BergenEnHenegouwen2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Over expression of receptor tyrosine kinases is responsible for the development of a wide variety of malignancies. Termination of growth factor signaling is primarily determined by the down regulation of active growth factor/receptor complexes. In recent years, considerable insight has been gained in the endocytosis and degradation of growth factor receptors. A crucial player in this process is the EGFR Protein tyrosine kinase Substrate #15, or Eps15. This protein functions as a scaffolding adaptor protein and is involved both in secretion and endocytosis. Eps15 has been shown to bind to AP-1 and AP-2 complexes, to bind to inositol lipids and to several other proteins involved in the regulation of intracellular trafficking. In addition, Eps15 has been detected in the nucleus of mammalian cells. Activation of growth factor receptors induces tyrosine phosphorylation and mono-ubiquitination of Eps15. The role of these post translational modifications of Eps15 is still a mystery. It is proposed that Eps15 and its family members Eps15R and Eps15b are involved in the regulation of membrane morphology, which is required for intracellular vesicle formation and trafficking.</td>
</tr>
<tr id="bib_BergenEnHenegouwen2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{BergenEnHenegouwen2009,
  author = {van Bergen En Henegouwen, Paul},
  title = {Eps15: a multifunctional adaptor protein regulating intracellular trafficking.},
  journal = {Cell communication and signaling : CCS},
  year = {2009},
  volume = {7},
  pages = {24},
  note = {nice table of eps15 interactions and refs.},
  doi = {https://doi.org/10.1186/1478-811X-7-24}
}
</pre></td>
</tr>
<tr id="Bernal1933" class="entry">
	<td>Bernal, J. and Fowler, R.</td>
	<td>A Theory of Water and Ionic Solution, with Particular Reference to Hydrogen and Hydroxyl Ions <p class="infolinks">[<a href="javascript:toggleInfo('Bernal1933','bibtex')">BibTeX</a>]</p></td>
	<td>1933</td>
	<td>The Journal of Chemical Physics<br/>Vol. 1(10), pp. 599-615&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.1749541">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Bernal1933" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bernal1933,
  author = {Bernal, JD and Fowler, RH},
  title = {A Theory of Water and Ionic Solution, with Particular Reference to Hydrogen and Hydroxyl Ions},
  journal = {The Journal of Chemical Physics},
  year = {1933},
  volume = {1},
  number = {10},
  pages = {599-615},
  doi = {https://doi.org/10.1063/1.1749541}
}
</pre></td>
</tr>
<tr id="Bhabha2011" class="entry">
	<td>Bhabha, G., Lee, J., Ekiert, D.C., Gam, J., a Wilson, I., Dyson, J.H., Benkovic, S.J. and Wright, P.E.</td>
	<td>The chemical step of enzyme catalysis <p class="infolinks">[<a href="javascript:toggleInfo('Bhabha2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bhabha2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Science<br/>Vol. 332(6026), pp. 234 - 238&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1126/science.1198542.A">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Bhabha2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Conformational dynamics play a key role in enzyme catalysis. Although protein motions have clear implications for ligand flux, a role for dynamics in the chemical step of enzyme catalysis has not been clearly established. We generated a mutant of Escherichia coli dihydrofolate reductase that abrogates millisecond-time-scale fluctuations in the enzyme active site without perturbing its structural and electrostatic preorganization. This dynamic knockout severely impairs hydride transfer. Thus, we have found a link between conformational fluctuations on the millisecond time scale and the chemical step of an enzymatic reaction, with broad implications for our understanding of enzyme mechanisms and for design of novel protein catalysts.</td>
</tr>
<tr id="bib_Bhabha2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bhabha2011,
  author = {Bhabha, Gira and Lee, Jeeyeon and Ekiert, Damian C and Gam, Jongsik and a Wilson, Ian and Dyson, Jane H and Benkovic, Stephen J and Wright, Peter E},
  title = {The chemical step of enzyme catalysis},
  journal = {Science},
  year = {2011},
  volume = {332},
  number = {6026},
  pages = {234 - 238},
  note = {NULL},
  doi = {https://doi.org/10.1126/science.1198542.A}
}
</pre></td>
</tr>
<tr id="Bhowmick2015" class="entry">
	<td>Bhowmick, A. and Head-Gordon, T.</td>
	<td>A Monte Carlo method for generating side chain structural ensembles <p class="infolinks">[<a href="javascript:toggleInfo('Bhowmick2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bhowmick2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Structure<br/>Vol. 23(1), pp. 44 - 55&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.str.2014.10.011">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Bhowmick2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We report a Monte Carlo side chain entropy (MC-SCE) method that uses a physical energy function inclusive of long-range electrostatics and hydrophobic potential of mean force, coupled with both backbone variations and a backbone dependent side chain rotamer library, to describe protein conformational ensembles. Using the MC-SCE method in conjunction with backbone variability, we can reliably determine the side chain rotamer populations derived from both room temperature and cryogenically cooled X-ray crystallographic structures for CypA and H-Ras and NMR J-coupling constants for CypA, Eglin-C, and the DHFR product binary complexes E:THF and E:FOL. Furthermore, we obtain near perfect discrimination between a protein's native state ensemble and ensembles of misfolded structures for 55 different proteins, thereby generating far more competitive side chain packings for all of these proteins and their misfolded states.</td>
</tr>
<tr id="bib_Bhowmick2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bhowmick2015,
  author = {Bhowmick, Asmit and Head-Gordon, Teresa},
  title = {A Monte Carlo method for generating side chain structural ensembles},
  journal = {Structure},
  year = {2015},
  volume = {23},
  number = {1},
  pages = {44 - 55},
  doi = {https://doi.org/10.1016/j.str.2014.10.011}
}
</pre></td>
</tr>
<tr id="Bhowmick2016" class="entry">
	<td>Bhowmick, A., Sharma, S.C., Honma, H. and Head-gordon, T.</td>
	<td>The role of side chain entropy and mutual information for improving the de novo design of Kemp eliminases KE07 and KE70 <p class="infolinks">[<a href="javascript:toggleInfo('Bhowmick2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bhowmick2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Physical Chemistry Chemical Physics<br/>Vol. 18(28), pp. 19386 - 19396&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1039/C6CP03622H">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Bhowmick2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Side chain entropy and mutual entropy information between residue pairs have been calculated for two de novo designed Kemp eliminase enzymes, KE07 and KE70, and for their most improved versions at the end of laboratory directed evolution (LDE). We find that entropy, not just enthalpy, helped to destabilize the preference for the reactant state complex of the designed enzyme as well as favoring stabilization of the transition state complex for the best LDE enzymes. Furthermore, residues with the highest side chain couplings as measured by mutual information, when experimentally mutated, were found to diminish or annihilate catalytic activity, some of which were far from the active site. In summary, our findings demonstrate how side chain fluctuations and their coupling can be an important design feature for de novo enzymes, and furthermore could be utilized in the computational steps in lieu of or in addition to the LDE steps in future enzyme design projects.</td>
</tr>
<tr id="bib_Bhowmick2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bhowmick2016,
  author = {Bhowmick, Asmit and Sharma, Sudhir C and Honma, Hallie and Head-gordon, Teresa},
  title = {The role of side chain entropy and mutual information for improving the de novo design of Kemp eliminases KE07 and KE70},
  journal = {Physical Chemistry Chemical Physics},
  year = {2016},
  volume = {18},
  number = {28},
  pages = {19386 - 19396},
  doi = {https://doi.org/10.1039/C6CP03622H}
}
</pre></td>
</tr>
<tr id="Bianco2017" class="entry">
	<td>Bianco, V., Franzese, G., Dellago, C. and Coluzza, I.</td>
	<td>Role of Water in the Selection of Stable Proteins at Ambient and Extreme Thermodynamic Conditions <p class="infolinks">[<a href="javascript:toggleInfo('Bianco2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bianco2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Physical Reviews X<br/>Vol. 021047(2), pp. 1 - 15&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1103/PhysRevX.7.021047">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Bianco2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Proteins that are functional at ambient conditions do not necessarily work at extreme conditions of temperature T and pressure P. Furthermore, there are limits of T and P above which no protein has a stable functional state. Here, we show that these limits and the selection mechanisms for working proteins depend on how the properties of the surrounding water change with T and P. We find that proteins selected at high T are superstable and are characterized by a nonextreme segregation of a hydrophilic surface and a hydrophobic core. Surprisingly, a larger segregation reduces the stability range in T and P. Our computer simulations, based on a new protein design protocol, explain the hydropathy profile of proteins as a consequence of a selection process influenced by water. Our results, potentially useful for engineering proteins and drugs working far from ambient conditions, offer an alternative rationale to the evolutionary action exerted by the environment in extreme conditions.</td>
</tr>
<tr id="bib_Bianco2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bianco2017,
  author = {Bianco, Valentino and Franzese, Giancarlo and Dellago, Christoph and Coluzza, Ivan},
  title = {Role of Water in the Selection of Stable Proteins at Ambient and Extreme Thermodynamic Conditions},
  journal = {Physical Reviews X},
  year = {2017},
  volume = {021047},
  number = {2},
  pages = {1 - 15},
  doi = {https://doi.org/10.1103/PhysRevX.7.021047}
}
</pre></td>
</tr>
<tr id="Billeter2001a" class="entry">
	<td>Billeter, S.R., Webb, S.P., Agarwal, P.K., Iordanov, T. and Hammes-Schiffer, S.</td>
	<td>Hydride transfer in liver alcohol dehydrogenase: Quantum dynamics, kinetic isotope effects, and role of enzyme motion <p class="infolinks">[<a href="javascript:toggleInfo('Billeter2001a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Billeter2001a','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Journal of the American Chemical Society<br/>Vol. 123(45), pp. 11262 - 11272&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ja011384b">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Billeter2001a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The quantum dynamics of the hydride transfer reaction catalyzed by liver alcohol dehydrogenase (LADH) are studied with real-time dynamical simulations including the motion of the entire solvated enzyme. The electronic quantum effects are incorporated with an empirical valence bond potential, and the nuclear quantum effects of the transferring hydrogen are incorporated with a mixed quantum/classical molecular dynamics method in which the transferring hydrogen nucleus is represented by a three-dimensional vibrational wave function. The equilibrium transition state theory rate constants are determined from the adiabatic quantum free energy profiles, which include the free energy of the zero point motion for the transferring nucleus. The nonequilibrium dynamical effects are determined by calculating the transmission coefficients with a reactive flux scheme based on real-time molecular dynamics with quantum transitions (MDQT) surface hopping trajectories. The values of nearly unity for these transmission coefficients imply that nonequilibrium dynamical effects such as barrier recrossings are not dominant for this reaction. The calculated deuterium and tritium kinetic isotope effects for the overall rate agree with experimental results. These simulations elucidate the fundamental nature of the nuclear quantum effects and provide evidence of hydrogen tunneling in the direction along the donoracceptor axis. An analysis of the geometrical parameters during the equilibrium and nonequilibrium simulations provides insight into the relation between specific enzyme motions and enzyme activity. The donoracceptor distance, the catalytic zincsubstrate oxygen distance, and the coenzyme (NAD+/NADH) ring angles are found to strongly impact the activation free energy barrier, while the donoracceptor distance and one of the coenzyme ring angles are found to be correlated to the degree of barrier recrossing. The distance between VAL-203 and the reactive center is found to significantly impact the activation free energy but not the degree of barrier recrossing. This result indicates that the experimentally observed effect of mutating VAL-203 on the enzyme activity is due to the alteration of the equilibrium free energy difference between the transition state and the reactant rather than nonequilibrium dynamical factors. The promoting motion of VAL-203 is characterized in terms of steric interactions involving THR-178 and the coenzyme. The quantum dynamics of the hydride transfer reaction catalyzed by liver alcohol dehydrogenase (LADH) are studied with real-time dynamical simulations including the motion of the entire solvated enzyme. The electronic quantum effects are incorporated with an empirical valence bond potential, and the nuclear quantum effects of the transferring hydrogen are incorporated with a mixed quantum/classical molecular dynamics method in which the transferring hydrogen nucleus is represented by a three-dimensional vibrational wave function. The equilibrium transition state theory rate constants are determined from the adiabatic quantum free energy profiles, which include the free energy of the zero point motion for the transferring nucleus. The nonequilibrium dynamical effects are determined by calculating the transmission coefficients with a reactive flux scheme based on real-time molecular dynamics with quantum transitions (MDQT) surface hopping trajectories. The values of nearly unity for these transmission coefficients imply that nonequilibrium dynamical effects such as barrier recrossings are not dominant for this reaction. The calculated deuterium and tritium kinetic isotope effects for the overall rate agree with experimental results. These simulations elucidate the fundamental nature of the nuclear quantum effects and provide evidence of hydrogen tunneling in the direction along the donoracceptor axis. An analysis of the geometrical parameters during the equilibrium and nonequilibrium simulations provides insight into the relation between specific enzyme motions and enzyme activity. The donoracceptor distance, the catalytic zincsubstrate oxygen distance, and the coenzyme (NAD+/NADH) ring angles are found to strongly impact the activation free energy barrier, while the donoracceptor distance and one of the coenzyme ring angles are found to be correlated to the degree of barrier recrossing. The distance between VAL-203 and the reactive center is found to significantly impact the activation free energy but not the degree of barrier recrossing. This result indicates that the experimentally observed effect of mutating VAL-203 on the enzyme activity is due to the alteration of the equilibrium free energy difference between the transition state and the reactant rather than nonequilibrium dynamical factors. The promoting motion of VAL-203 is characterized in terms of steric interactions involving THR-178 and the coenzyme.</td>
</tr>
<tr id="bib_Billeter2001a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Billeter2001a,
  author = {Billeter, Salomon R and Webb, Simon P and Agarwal, Pratul K and Iordanov, Tzvetelin and Hammes-Schiffer, Sharon},
  title = {Hydride transfer in liver alcohol dehydrogenase: Quantum dynamics, kinetic isotope effects, and role of enzyme motion},
  journal = {Journal of the American Chemical Society},
  year = {2001},
  volume = {123},
  number = {45},
  pages = {11262 - 11272},
  note = {second wavefunction method paper},
  doi = {https://doi.org/10.1021/ja011384b}
}
</pre></td>
</tr>
<tr id="Billeter2001" class="entry">
	<td>Billeter, S.R., Webb, S.P., Iordanov, T., Agarwal, P.K. and Hammes-Schiffer, S.</td>
	<td>Hybrid approach for including electronic and nuclear quantum effects in molecular dynamics simulations of hydrogen transfer reactions in enzymes <p class="infolinks">[<a href="javascript:toggleInfo('Billeter2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Billeter2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>The Journal of Chemical Physics<br/>Vol. 114(15), pp. 6925 - 6925&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.1356441">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Billeter2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A hybrid approach for simulating proton and hydride transfer reactions in enzymes is presented. The electronic quantum effects are incorporated with an empirical valence bond approach. The nuclear quantum effects of the transferring hydrogen are included with a mixed quantum/classical molecular dynamics method in which the hydrogen nucleus is described as a multidimensional vibrational wave function. The free energy profiles are obtained as functions of a collective reaction coordinate. A perturbation formula is derived to incorporate the vibrationally adiabatic nuclear quantum effects into the free energy profiles. The dynamical effects are studied with the molecular dynamics with quantum transitions (MDQT) surface hopping method, which incorporates nonadiabatic transitions among the adiabatic hydrogen vibrational states. The MDQT method is combined with a reactive flux approach to calculate the transmission coefficient and to investigate the real-time dynamics of reactive trajectories. This hybrid approach includes nuclear quantum effects such as zero point energy, hydrogen tunneling, and excited vibrational states, as well as the dynamics of the complete enzyme and solvent. The nuclear quantum effects are incorporated during the generation of the free energy profiles and dynamical trajectories rather than subsequently added as corrections. Moreover, this methodology provides detailed mechanistic information at the molecular level and allows the calculation of rates and kinetic isotope effects. An initial application of this approach to the enzyme liver alcohol dehydrogenase is also presented.</td>
</tr>
<tr id="bib_Billeter2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Billeter2001,
  author = {Billeter, Salomon R and Webb, Simon P and Iordanov, Tzvetelin and Agarwal, Pratul K and Hammes-Schiffer, Sharon},
  title = {Hybrid approach for including electronic and nuclear quantum effects in molecular dynamics simulations of hydrogen transfer reactions in enzymes},
  journal = {The Journal of Chemical Physics},
  year = {2001},
  volume = {114},
  number = {15},
  pages = {6925 - 6925},
  note = {first wavefunction method paper},
  doi = {https://doi.org/10.1063/1.1356441}
}
</pre></td>
</tr>
<tr id="Bjoerkholm2009" class="entry">
	<td>Bjorkholm, P. and Sonnhammer, E.</td>
	<td>Comparative analysis and unification of domain -- domain interaction networks <p class="infolinks">[<a href="javascript:toggleInfo('Bjoerkholm2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bjoerkholm2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Bioinformatics<br/>Vol. 25(22), pp. 3020 - 3025&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/bioinformatics/btp522">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Bjoerkholm2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Motivation: Certain protein domains are known to preferentially interact with other domains. Several approaches have been proposed to predict domain--domain interactions, and over nine datasets are available. Our aim is to analyse the coverage and quality of the existing resources, as well as the extent of their overlap. With this knowledge, we have the opportunity to merge individual domain interaction networks to construct a comprehensive and reliable database. Results: In this article we introduce a new approach towards comparing domain--domain interaction networks. This approach is used to compare nine predicted domain and protein interaction networks. The networks were used to generate a database of unified domain interactions, UniDomInt. Each interaction in the dataset is scored according to the benchmarked reliability of the sources. The performance of UniDomInt is an improvement compared to the underlying source networks and to another composite resource, Domine. Availability:http://sonnhammer.sbc.su.se/download/UniDomInt/ Contact:Erik.Sonnhammer@sbc.su.se</td>
</tr>
<tr id="bib_Bjoerkholm2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bjoerkholm2009,
  author = {Bjorkholm, Patrik and Sonnhammer, Erik},
  title = {Comparative analysis and unification of domain -- domain interaction networks},
  journal = {Bioinformatics},
  year = {2009},
  volume = {25},
  number = {22},
  pages = {3020 - 3025},
  doi = {https://doi.org/10.1093/bioinformatics/btp522}
}
</pre></td>
</tr>
<tr id="Blank1986" class="entry">
	<td>Blank, G.S. and Brodsky, F.M.</td>
	<td>Site-specific disruption of clathrin assembly produces novel structures. <p class="infolinks">[<a href="javascript:toggleInfo('Blank1986','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Blank1986','bibtex')">BibTeX</a>]</p></td>
	<td>1986</td>
	<td>The EMBO journal<br/>Vol. 5, pp. 2087-2095&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Blank1986" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin assembly in vitro produces a highly ordered polyhedral structure (basket). This resembles clathrin assembled in situ on coated pits and vesicles which form during receptor-mediated endocytosis. Sites on clathrin involved in assembly were identified by assembling clathrin in the presence of anti-clathrin monoclonal antibodies. Three of the antibodies, as IgG, prevented the assembly of normal baskets, and their Fab fragments induced formation of two types of novel clathrin structures. Antibody effects on assembly and competitive binding data indicate these antibodies bind to two sites, critical for clathrin interactions, located in the same region of the clathrin heavy chain. Analysis of novel structures formed, suggested that nucleation but not further assembly was occurring, implying an ordered sequence of clathrin interactions during assembly.</td>
</tr>
<tr id="bib_Blank1986" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Blank1986,
  author = {Blank, G S and Brodsky, F M},
  title = {Site-specific disruption of clathrin assembly produces novel structures.},
  journal = {The EMBO journal},
  year = {1986},
  volume = {5},
  pages = {2087--2095}
}
</pre></td>
</tr>
<tr id="Blank1987" class="entry">
	<td>Blank, G.S. and Brodsky, F.M.</td>
	<td>Clathrin assembly involves a light chain-binding region. <p class="infolinks">[<a href="javascript:toggleInfo('Blank1987','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Blank1987','bibtex')">BibTeX</a>]</p></td>
	<td>1987</td>
	<td>The Journal of cell biology<br/>Vol. 105, pp. 2011-2019&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Blank1987" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Two regions on the clathrin heavy chain that are involved in triskelion interactions during assembly have been localized on the triskelion structure. These regions were previously identified with anti-heavy chain monoclonal antibodies X19 and X35, which disrupt clathrin assembly (Blank, G. S., and F. M. Brodsky, 1986, EMBO (Eur. Mol. Biol. Organ.) J., 5:2087-2095). Antibody-binding sites were determined based on their reactivity with truncated triskelions, and were mapped to an 8-kD region in the middle of the proximal portion of the triskelion arm (X19) and a 6-kD region at the triskelion elbow (X35). The elbow site implicated in triskelion assembly was also shown to be included within a heavy chain region involved in binding the light chains and to constitute part of the light chain-binding site. We postulate that this region of the heavy chain binds to the interaction site identified on the light chains that has homology to intermediate filament proteins (Brodsky, F. M., C. J. Galloway, G. S. Blank, A. P. Jackson, H.-F. Seow, K. Drickamer, and P. Parham, 1987, Nature (Lond.), 326:203-205). These findings suggest the existence of a heavy chain site, near the triskelion elbow, which is involved in both intramolecular and intermolecular interactions during clathrin assembly.</td>
</tr>
<tr id="bib_Blank1987" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Blank1987,
  author = {Blank, G S and Brodsky, F M},
  title = {Clathrin assembly involves a light chain-binding region.},
  journal = {The Journal of cell biology},
  year = {1987},
  volume = {105},
  pages = {2011--2019}
}
</pre></td>
</tr>
<tr id="Blinov2006" class="entry">
	<td>Blinov, M., Yang, J., Faeder, J. and Hlavacek, W.</td>
	<td>Graph theory for rule-based modeling of biochemical networks <p class="infolinks">[<a href="javascript:toggleInfo('Blinov2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Blinov2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Transactions on Computational Systems Biology VII<br/>Vol. 4230, pp. 89 - 106&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1007/11905455_5">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Blinov2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We introduce a graph-theoretic formalism suitable for modeling biochemical networks marked by combinatorial complexity, such as signal-transduction systems, in which protein-protein interactions play a prominent role. This development extends earlier work by allowing for explicit representation of the connectivity of a protein complex. Within the formalism, typed attributed graphs are used to represent proteins and their functional components, complexes, conformations, and states of post-translational covalent modification. Graph transformation rules are used to represent protein-protein interactions and their effects. Each rule defines a generalized reaction, i.e., a class of potential reactions that are logically consistent with knowledge or assumptions about the represented biomolecular interaction. A model is specified by defining 1) molecular-entity graphs, which delimit the molecular entities and material components of a system and their possible states, 2) graph transformation rules, and 3) a seed set of graphs representing chemical species, such as the initial species present before introduction of a signal. A reaction network is generated iteratively through application of the graph transformation rules. The rules are first applied to the seed graphs and then to any and all new graphs that subsequently arise as a result of graph transformation. This procedure continues until no new graphs are generated or a specified termination condition is satisfied. The formalism supports the generation of a list of reactions in a system, which can be used to derive different types of physicochemical models, which can be simulated and analyzed in different ways. The processes of generating and simulating the network may be combined so that species are generated only as needed.</td>
</tr>
<tr id="bib_Blinov2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Blinov2006,
  author = {Blinov, ML and Yang, J and Faeder, JR and Hlavacek, WS},
  title = {Graph theory for rule-based modeling of biochemical networks},
  journal = {Transactions on Computational Systems Biology VII},
  year = {2006},
  volume = {4230},
  pages = {89 - 106},
  doi = {https://doi.org/10.1007/11905455_5}
}
</pre></td>
</tr>
<tr id="Blinov2004" class="entry">
	<td>Blinov, M.L., Faeder, J.R., Goldstein, B. and Hlavacek, W.S.</td>
	<td>BioNetGen: Software for rule-based modeling of signal transduction based on the interactions of molecular domains <p class="infolinks">[<a href="javascript:toggleInfo('Blinov2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Blinov2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Bioinformatics<br/>Vol. 20(17), pp. 3289 - 3291&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/bioinformatics/bth378">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Blinov2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: BioNetGen allows a user to create a computational model that characterizes the dynamics of a signal transduction system, and that accounts comprehensively and precisely for specified enzymatic activities, potential post-translational modifications and interactions of the domains of signaling molecules. The output defines and parameterizes the network of molecular species that can arise during signaling and provides functions that relate model variables to experimental readouts of interest. Models that can be generated are relevant for rational drug discovery, analysis of proteomic data and mechanistic studies of signal transduction.</td>
</tr>
<tr id="bib_Blinov2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Blinov2004,
  author = {Blinov, Michael L and Faeder, James R and Goldstein, Byron and Hlavacek, William S},
  title = {BioNetGen: Software for rule-based modeling of signal transduction based on the interactions of molecular domains},
  journal = {Bioinformatics},
  year = {2004},
  volume = {20},
  number = {17},
  pages = {3289 - 3291},
  doi = {https://doi.org/10.1093/bioinformatics/bth378}
}
</pre></td>
</tr>
<tr id="Blinov2017" class="entry">
	<td>Blinov, M.L., Schaff, J.C., Vasilescu, D., Moraru, I.I., Bloom, J.E. and Loew, L.M.</td>
	<td>Compartmental and spatial rule-based modeling with Virtual cell (VCell) <p class="infolinks">[<a href="javascript:toggleInfo('Blinov2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Biophysical Journal<br/>Vol. 113, pp. 1365-1372&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Blinov2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Blinov2017,
  author = {Blinov, Mikhail L and Schaff, James C and Vasilescu, Dan and Moraru, Ion I and Bloom, Judy E and Loew, Leslie M},
  title = {Compartmental and spatial rule-based modeling with Virtual cell (VCell)},
  journal = {Biophysical Journal},
  publisher = {Cold Spring Harbor Labs Journals},
  year = {2017},
  volume = {113},
  pages = {1365-1372}
}
</pre></td>
</tr>
<tr id="Blondeau2004" class="entry">
	<td>Blondeau, F., Ritter, B., Allaire, P.D., Wasiak, S., Girard, M., Hussain, N.K., Angers, A., Legendre-Guillemin, V., Roy, L., Boismenu, D., Kearney, R.E., Bell, A.W., Bergeron, J.J.M. and McPherson, P.S.</td>
	<td>Tandem MS analysis of brain clathrin-coated vesicles reveals their critical involvement in synaptic vesicle recycling. <p class="infolinks">[<a href="javascript:toggleInfo('Blondeau2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Blondeau2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Proceedings of the National Academy of Sciences of the United States of America<br/>Vol. 101, pp. 3833-3838&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.0308186101">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Blondeau2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Tandem MS has identified 209 proteins of clathrin-coated vesicles (CCVs) isolated from rat brain. An overwhelming abundance of peptides were assigned to the clathrin coat with a 1:1 stoichiometry observed for clathrin heavy and light chains and a 2:1 stoichiometry of clathrin heavy chain with clathrin adaptor protein heterotetramers. Thirty-two proteins representing many of the known components of synaptic vesicles (SVs) were identified, supporting that a main function for brain CCVs is to recapture SVs after exocytosis. A ratio of vesicle-N-ethylmaleimide-sensitive factor attachment protein receptors to target-N-ethylmaleimide-sensitive factor attachment protein receptors, similar to that previously detected on SVs, supports a single-step model for SV sorting during CCV-mediated recycling of SVs. The uncovering of eight previously undescribed proteins, four of which have to date been linked to clathrin-mediated trafficking, further attests to the value of the current organelle-based proteomics strategy.</td>
</tr>
<tr id="bib_Blondeau2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Blondeau2004,
  author = {Blondeau, Francois and Ritter, Brigitte and Allaire, Patrick D and Wasiak, Sylwia and Girard, Martine and Hussain, Natasha K and Angers, Annie and Legendre-Guillemin, Valerie and Roy, Line and Boismenu, Daniel and Kearney, Robert E and Bell, Alexander W and Bergeron, John J M and McPherson, Peter S},
  title = {Tandem MS analysis of brain clathrin-coated vesicles reveals their critical involvement in synaptic vesicle recycling.},
  journal = {Proceedings of the National Academy of Sciences of the United States of America},
  year = {2004},
  volume = {101},
  pages = {3833--3838},
  doi = {https://doi.org/10.1073/pnas.0308186101}
}
</pre></td>
</tr>
<tr id="Boecking2014" class="entry">
	<td>Böcking, T., Aguet, F., Rapoport, I., Banzhaf, M., Yu, A., Zeeh, J. and Kirchhausen, T.</td>
	<td>Key Interactions for Clathrin Coat Stability <p class="infolinks">[<a href="javascript:toggleInfo('Boecking2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Structure<br/>Vol. 22(6), pp. 819 - 829&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.str.2014.04.002">DOI</a> <a href="http://www.sciencedirect.com/science/article/pii/S0969212614001063">URL</a>&nbsp;</td>
</tr>
<tr id="bib_Boecking2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Boecking2014,
  author = {Till Böcking and François Aguet and Iris Rapoport and Manuel Banzhaf and Anan Yu and Jean Christophe Zeeh and Tom Kirchhausen},
  title = {Key Interactions for Clathrin Coat Stability},
  journal = {Structure},
  year = {2014},
  volume = {22},
  number = {6},
  pages = {819 - 829},
  url = {http://www.sciencedirect.com/science/article/pii/S0969212614001063},
  doi = {https://doi.org/10.1016/j.str.2014.04.002}
}
</pre></td>
</tr>
<tr id="Boehm2002" class="entry">
	<td>Boehm, M. and Bonifacino, J.S.</td>
	<td>Genetic analyses of adaptin function from yeast to mammals <p class="infolinks">[<a href="javascript:toggleInfo('Boehm2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Boehm2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Gene<br/>Vol. 286(2), pp. 175 - 186&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S0378-1119(02)00422-5">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Boehm2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Adaptor protein (AP) complexes are heterotetrameric assemblies of subunits named adaptins. Four AP complexes, termed AP-1, AP-2, AP-3, and AP-4, have been described in various eukaryotic organisms. Biochemical and morphological evidence indicates that AP complexes play roles in the formation of vesicular transport intermediates and the selection of cargo molecules for inclusion into these intermediates. This understanding is being expanded by the application of genetic interference procedures. Here, we review recent progress in the genetic analysis of the function of AP complexes, focusing on studies that make use of targeted interference or naturally-occurring mutations in various model organisms. textcopyright 2002 Elsevier Science B.V. All rights reserved.</td>
</tr>
<tr id="bib_Boehm2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Boehm2002,
  author = {Boehm, Markus and Bonifacino, Juan S},
  title = {Genetic analyses of adaptin function from yeast to mammals},
  journal = {Gene},
  year = {2002},
  volume = {286},
  number = {2},
  pages = {175 - 186},
  doi = {https://doi.org/10.1016/S0378-1119(02)00422-5}
}
</pre></td>
</tr>
<tr id="Boehr2006a" class="entry">
	<td>Boehr, D., McElheny, D., Dyson, H. and Wright, P.</td>
	<td>The dynamic energy landscape of dihydrofolate reductase catalysis <p class="infolinks">[<a href="javascript:toggleInfo('Boehr2006a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Boehr2006a','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Science<br/>Vol. 313(5793), pp. 1638 - 1642&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1126/science.1130258">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Boehr2006a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We used nuclear magnetic resonance relaxation dispersion to characterize higher energy conformational substates of Escherichia coli dihydrofolate reductase. Each intermediate in the catalytic cycle samples low-lying excited states whose conformations resemble the ground-state structures of preceding and following intermediates. Substrate and cofactor exchange occurs through these excited substates. The maximum hydride transfer and steady-state turnover rates are governed by the dynamics of transitions between ground and excited states of the intermediates. Thus, the modulation of the energy landscape by the bound ligands funnels the enzyme through its reaction cycle along a preferred kinetic path.</td>
</tr>
<tr id="bib_Boehr2006a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Boehr2006a,
  author = {Boehr, David and McElheny, Dan and Dyson, H and Wright, Peter},
  title = {The dynamic energy landscape of dihydrofolate reductase catalysis},
  journal = {Science},
  year = {2006},
  volume = {313},
  number = {5793},
  pages = {1638 - 1642},
  note = {NULL},
  doi = {https://doi.org/10.1126/science.1130258}
}
</pre></td>
</tr>
<tr id="Boehr2006" class="entry">
	<td>Boehr, D.D., Dyson, J.H. and Wright, P.E.</td>
	<td>An NMR perspective on enzyme dynamics <p class="infolinks">[<a href="javascript:toggleInfo('Boehr2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Boehr2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Chemical Reviews<br/>Vol. 106(8), pp. 3055 - 3079&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/cr050312q">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Boehr2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: NA</td>
</tr>
<tr id="bib_Boehr2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Boehr2006,
  author = {Boehr, David D and Dyson, Jane H and Wright, Peter E},
  title = {An NMR perspective on enzyme dynamics},
  journal = {Chemical Reviews},
  year = {2006},
  volume = {106},
  number = {8},
  pages = {3055 - 3079},
  doi = {https://doi.org/10.1021/cr050312q}
}
</pre></td>
</tr>
<tr id="Boeke2014" class="entry">
	<td>Boeke, D., Trautmann, S., Meurer, M., Wachsmuth, M., Godlee, C., Knop, M. and Kaksonen, M.</td>
	<td>Quantification of cytosolic interactions identifies Ede1 oligomers as key organizers of endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Boeke2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Boeke2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Molecular systems biology<br/>Vol. 10, pp. 756&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.15252/msb.20145422">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Boeke2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis is a highly conserved intracellular trafficking pathway that depends on dynamic protein-protein interactions between up to 60 different proteins. However, little is known about the spatio-temporal regulation of these interactions. Using fluorescence (cross)-correlation spectroscopy in yeast, we tested 41 previously reported interactions in vivo and found 16 to exist in the cytoplasm. These detected cytoplasmic interactions included the self-interaction of Ede1, homolog of mammalian Eps15. Ede1 is the crucial scaffold for the organization of the early stages of endocytosis. We show that oligomerization of Ede1 through its central coiled coil domain is necessary for its localization to the endocytic site and we link the oligomerization of Ede1 to its function in locally concentrating endocytic adaptors and organizing the endocytic machinery. Our study sheds light on the importance of the regulation of protein-protein interactions in the cytoplasm for the assembly of the endocytic machinery in vivo.</td>
</tr>
<tr id="bib_Boeke2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Boeke2014,
  author = {Boeke, Dominik and Trautmann, Susanne and Meurer, Matthias and Wachsmuth, Malte and Godlee, Camilla and Knop, Michael and Kaksonen, Marko},
  title = {Quantification of cytosolic interactions identifies Ede1 oligomers as key organizers of endocytosis.},
  journal = {Molecular systems biology},
  year = {2014},
  volume = {10},
  pages = {756},
  note = {eps15 dimerization (through homology with ede1) dissociation constant},
  doi = {https://doi.org/10.15252/msb.20145422}
}
</pre></td>
</tr>
<tr id="Boldt2016" class="entry">
	<td>Boldt, K., van Reeuwijk, J., Lu, Q., Koutroumpas, K., Thanh-Minh, N., Texier, Y., van Beersum, S., Horn, N., Willer, J.R., Mans, D.A., Dougherty, G., Lamers, I., Coene, K., Arts, H.H., Betts, M.J., Beyer, T., Bolat, E., Gloeckner, C., Haidari, K., Hetterschijt, L., Iaconis, D., Jenkins, D., Klose, F., Knapp, B., Latour, B., Letteboer, S., Marcelis, C.L., Mitic, D., Morleo, M., Oud, M.M., Riemersma, M., Rix, S., Terhal, P.A., Toedt, G., van Dam, T., de Vrieze, E., Wissinger, Y., Wu, K., Apic, G., Beales, P.L., Blacque, O.E., Gibson, T.J., Huynen, M.A., Katsanis, N., Kremer, H., Omran, H., van Wijk, E., Wolfrum, U., Kepes, F., Davis, E.E., Franco, B., Giles, R.H., Ueffing, M., Russell, R.B. and Roepman, R.</td>
	<td>An organelle-specific protein landscape identifies novel diseases and molecular mechanisms. <p class="infolinks">[<a href="javascript:toggleInfo('Boldt2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Boldt2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Nature Communications<br/>Vol. 7(May), pp. 11491 - 11491&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/ncomms11491">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Boldt2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Cellular organelles provide opportunities to relate biological mechanisms to disease. Here we use affinity proteomics, genetics and cell biology to interrogate cilia: poorly understood organelles, where defects cause genetic diseases. Two hundred and seventeen tagged human ciliary proteins create a final landscape of 1,319 proteins, 4,905 interactions and 52 complexes. Reverse tagging, repetition of purifications and statistical analyses, produce a high-resolution network that reveals organelle-specific interactions and complexes not apparent in larger studies, and links vesicle transport, the cytoskeleton, signalling and ubiquitination to ciliary signalling and proteostasis. We observe sub-complexes in exocyst and intraflagellar transport complexes, which we validate biochemically, and by probing structurally predicted, disruptive, genetic variants from ciliary disease patients. The landscape suggests other genetic diseases could be ciliary including 3M syndrome. We show that 3M genes are involved in ciliogenesis, and that patient fibroblasts lack cilia. Overall, this organelle-specific targeting strategy shows considerable promise for Systems Medicine.</td>
</tr>
<tr id="bib_Boldt2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Boldt2016,
  author = {Boldt, Karsten and van Reeuwijk, Jeroen and Lu, Qianhao and Koutroumpas, Konstantinos and Thanh-Minh, Nguyen and Texier, Yves and van Beersum, Sylvia and Horn, Nicola and Willer, Jason R and Mans, Dorus A and Dougherty, Gerard and Lamers, Ideke and Coene, Karlien and Arts, Heleen H and Betts, Matthew J and Beyer, Tina and Bolat, Emine and Gloeckner, Christian and Haidari, Khatera and Hetterschijt, Lisette and Iaconis, Daniela and Jenkins, Dagan and Klose, Franziska and Knapp, Barbara and Latour, Brooke and Letteboer, Stef and Marcelis, Carlo L and Mitic, Dragana and Morleo, Manuela and Oud, Machteld M and Riemersma, Moniek and Rix, Susan and Terhal, Paulien A and Toedt, Grischa and van Dam, Teunis and de Vrieze, Erik and Wissinger, Yasmin and Wu, Ka and Apic, Gordana and Beales, Philip L and Blacque, Oliver E and Gibson, Toby J and Huynen, Martijn A and Katsanis, Nicholas and Kremer, Hannie and Omran, Heymut and van Wijk, Erwin and Wolfrum, Uwe and Kepes, Francois and Davis, Erica E and Franco, Brunella and Giles, Rachel H and Ueffing, Marius and Russell, Robert B and Roepman, Ronald},
  title = {An organelle-specific protein landscape identifies novel diseases and molecular mechanisms.},
  journal = {Nature Communications},
  year = {2016},
  volume = {7},
  number = {May},
  pages = {11491 - 11491},
  doi = {https://doi.org/10.1038/ncomms11491}
}
</pre></td>
</tr>
<tr id="Bolhuis2000" class="entry">
	<td>Bolhuis, P., Dellago, C. and Chandler, D.</td>
	<td>Reaction coordinates of biomolecular isomerization <p class="infolinks">[<a href="javascript:toggleInfo('Bolhuis2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bolhuis2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Proceedings of the National Academy of Sciences<br/>Vol. 97(11), pp. 5877 - 5882&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.100127697">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Bolhuis2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Transition path sampling has been applied to the molecular dynamics of the alanine dipeptide in vacuum and in aqueous solution. The analysis shows that more degrees of freedom than the traditional dihedral angles, varphi\ and psi\, are necessary to describe the reaction coordinates for isomerization of this molecule. In vacuum, an additional dihedral angle is identified as significant. In solution, solvent variables are shown to play a significant role, and this role appears to be more specific than can be captured by friction models. Implications for larger molecules are discussed.</td>
</tr>
<tr id="bib_Bolhuis2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bolhuis2000,
  author = {Bolhuis, PG and Dellago, C and Chandler, D},
  title = {Reaction coordinates of biomolecular isomerization},
  journal = {Proceedings of the National Academy of Sciences},
  year = {2000},
  volume = {97},
  number = {11},
  pages = {5877 - 5882},
  doi = {https://doi.org/10.1073/pnas.100127697}
}
</pre></td>
</tr>
<tr id="Bolhuis2000a" class="entry">
	<td>Bolhuis, P.G. and Chandler, D.</td>
	<td>Transition path sampling of cavitation between molecular scale solvophobic surfaces <p class="infolinks">[<a href="javascript:toggleInfo('Bolhuis2000a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bolhuis2000a','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Journal of Chemical Physics<br/>Vol. 113(18), pp. 8154 - 8160&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.1315997">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Bolhuis2000a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The dynamics of a cavitation transition between repulsive plates in a Lennard-Jones system is studied using transition path sampling. It is found that the critical nucleus for the transition coincides with the formation of a vapor tube connecting the two plates. The number of particles between the plates and the tube radius are relevant order parameters. In the transition state ensemble, the distributions of these parameters have widths that are roughly 20&#37; of their respective means. Committor distributions of constrained ensembles show that of these two important parameters, the tube radius is the most significant component in the reaction coordinate for the cavitation transition.</td>
</tr>
<tr id="bib_Bolhuis2000a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bolhuis2000a,
  author = {Bolhuis, Peter G and Chandler, David},
  title = {Transition path sampling of cavitation between molecular scale solvophobic surfaces},
  journal = {Journal of Chemical Physics},
  year = {2000},
  volume = {113},
  number = {18},
  pages = {8154 - 8160},
  doi = {https://doi.org/10.1063/1.1315997}
}
</pre></td>
</tr>
<tr id="Bolhuis2002" class="entry">
	<td>Bolhuis, P.G., Chandler, D., Dellago, C. and Geissler, P.L.</td>
	<td>Transition path sampling : Throwing ropes Over rough mountain passes, in the dark <p class="infolinks">[<a href="javascript:toggleInfo('Bolhuis2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bolhuis2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Annual Review of Physical Chemistry<br/>Vol. 53(1), pp. 291 - 318&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1146/annurev.physchem.53.082301.113146">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Bolhuis2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Abstract  This article reviews the concepts and methods of transition path sampling. These methods allow computational studies of rare events without requiring prior knowledge of mechanisms, reaction coordinates, and transition states. Based upon a statistical mechanics of trajectory space, they provide a perspective with which time dependent phenomena, even for systems driven far from equilibrium, can be examined with the same types of importance sampling tools that in the past have been applied so successfully to static equilibrium properties.</td>
</tr>
<tr id="bib_Bolhuis2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bolhuis2002,
  author = {Bolhuis, Peter G and Chandler, David and Dellago, Christoph and Geissler, Phillip L},
  title = {Transition path sampling : Throwing ropes Over rough mountain passes, in the dark},
  journal = {Annual Review of Physical Chemistry},
  year = {2002},
  volume = {53},
  number = {1},
  pages = {291 - 318},
  note = {NULL},
  doi = {https://doi.org/10.1146/annurev.physchem.53.082301.113146}
}
</pre></td>
</tr>
<tr id="Bolhuis1998" class="entry">
	<td>Bolhuis, P.G., Dellago, C. and Chandler, D.</td>
	<td>Sampling ensembles of deterministic transition pathways <p class="infolinks">[<a href="javascript:toggleInfo('Bolhuis1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bolhuis1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Faraday Discussions<br/>Vol. 110(0), pp. 421 - 436&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1039/a801266k">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Bolhuis1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We extend the method of transition-path sampling to the case of deterministic dynamics. This method is a Monte Carlo procedure for sampling the ensemble of trajectories that carry a many-particle system from one set of stable or metastable states to another. It requires no preconceived notions of transition mechanisms or transition states. Rather, it is from the resulting set of suitably weighted dynamical transition paths that one identifies transition mechanisms, determines relevant transition states and calculates transition rate constants. In earlier work, transition-path sampling was considered in the context of stochastic dynamics. Here, the necessary modifications that make it applicable to deterministic dynamics are discussed and the modifications illustrated with microcanonical simulations of isomerization events in two-dimensional seven-atom Lennard-Jones clusters.</td>
</tr>
<tr id="bib_Bolhuis1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bolhuis1998,
  author = {Bolhuis, Peter G and Dellago, Christoph and Chandler, David},
  title = {Sampling ensembles of deterministic transition pathways},
  journal = {Faraday Discussions},
  year = {1998},
  volume = {110},
  number = {0},
  pages = {421 - 436},
  note = {NULL},
  doi = {https://doi.org/10.1039/a801266k}
}
</pre></td>
</tr>
<tr id="Boll1995" class="entry">
	<td>Boll, W., Gallusser, A. and Kirchhausen, T.</td>
	<td>Role of the regulatory domain of the EGF-receptor cytoplasmic tail in selective binding of the clathrin-associated complex AP-2. <p class="infolinks">[<a href="javascript:toggleInfo('Boll1995','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Boll1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>Current biology : CB<br/>Vol. 5, pp. 1168-1178&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Boll1995" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: After stimulation of a cell by the mitogenic epidermal growth factor (EGF), the EGF receptor (EGF-R) is cleared from the cell surface in order to turn off receptor signaling. This internalization is mediated via clathrin-coated pits and coated vesicles, and ultimately the receptors are delivered to the lysosome and destroyed. It is believed that clathrin-associated protein complexes or adaptors (APs) link the entrapment of EGF-R and other nutrient and growth-factor receptors to the formation of the clathrin-coated pit. Two classes of APs are known--AP-2, found at the plasma membrane, and AP-1, found in the trans-Golgi network. Activated EGF-R associates with AP-2s at the plasma membrane, but the mechanism responsible for this association is not known. Here, we investigate, in vivo and in vitro, three aspects of the interaction between APs and EGF-R: firstly, we ask whether EGF-R at the plasma membrane distinguishes between AP-1 and AP-2; secondly, we ask which part of the receptor's cytoplasmic tail is responsible for binding; finally, we ask whether autophosphorylation by EGF-R is essential for the interaction. We demonstrate that EGF-R displays a selective association for AP-2 over AP-1 in vivo, and that this preferential interaction can also be detected using surface plasmon resonance in vitro. Using a truncated mutant and a kinase-dead mutant of EGF-R, we show that the regulatory domain of the cytoplasmic tail is essential for the recruitment of AP-2 in vivo and that this domain is required for association between purified AP-2 and EGF-R in vitro. Finally, we demonstrate, in vivo and in vitro, that tyrosine auto-phosphorylation by the receptor is not an essential pre-condition for the recruitment of AP-2. EGF-R binds selectively to AP-2s, and the regulatory domain of its cytoplasmic tail is required for this interaction. The lack of correlation between receptor autophosphorylation and AP-2 recruitment suggests that activation of the EGF-R kinase stimulates endocytosis by the phosphorylation of a factor distinct from EGF-R itself, as also proposed by others based on experiments measuring receptor traffic and entrapment.</td>
</tr>
<tr id="bib_Boll1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Boll1995,
  author = {Boll, W and Gallusser, A and Kirchhausen, T},
  title = {Role of the regulatory domain of the EGF-receptor cytoplasmic tail in selective binding of the clathrin-associated complex AP-2.},
  journal = {Current biology : CB},
  year = {1995},
  volume = {5},
  pages = {1168--1178}
}
</pre></td>
</tr>
<tr id="Borgonovo2012" class="entry">
	<td>Borgonovo, J., Capella, P., Seltzer, A. and Sosa, M.A.</td>
	<td>Expression of coat proteins changes during postnatal development in selected areas of the rat brain. <p class="infolinks">[<a href="javascript:toggleInfo('Borgonovo2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Borgonovo2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>International Journal of Developmental Neuroscience<br/>Vol. 30(4), pp. 333 - 41&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.ijdevneu.2012.01.005">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Borgonovo2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: It is well known that clathrin-mediated endocytosis is crucial for the normal functioning and integrity of neurons in the central nervous system. In this study we attempted to correlate the expression of coat proteins with development in different areas of rat brain. By Western blot, we studied the expression of AP-2, GGA1 and GGA2 in striatum, cerebellum, brain stem, cerebral cortex and hippocampus of newborn rats and during post-natal development; 5, 15, 30, 60, 90 or 150 days after birth. We observed that the expression of the α2 subunit of AP-2 increased substantially between the 15th and 30th day after birth in all areas studied, excepting the cerebellum and cortex. On the other hand, the expression of the α1 subunit does not change significantly during the development in any of the areas under study. We also noted that the expression of the μ2 subunit did not follow the pattern of α2 during development. In general terms, the expression of GGA1 and GGA2 followed a similar pattern to that of AP-2, although these proteins increased significantly in the cerebral cortex from the 15th day after birth. Moreover, presenilin-1, a protein associated with aging and neurodegeneration, shows an expression pattern similar to coat proteins in the striatum and cortex. These results suggest that proteins that conform the intracellular transport machinery in the brain cells seems to accompany development, according to the maturation of the different brain areas.</td>
</tr>
<tr id="bib_Borgonovo2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Borgonovo2012,
  author = {Borgonovo, Janina and Capella, Pablo and Seltzer, Alicia and Sosa, Miguel A},
  title = {Expression of coat proteins changes during postnatal development in selected areas of the rat brain.},
  journal = {International Journal of Developmental Neuroscience},
  year = {2012},
  volume = {30},
  number = {4},
  pages = {333 - 41},
  doi = {https://doi.org/10.1016/j.ijdevneu.2012.01.005}
}
</pre></td>
</tr>
<tr id="Borner2012" class="entry">
	<td>Borner, G.H., Antrobus, R., Hirst, J., Bhumbra, G.S., Kozik, P., Jackson, L.P., Sahlender, D.A. and Robinson, M.S.</td>
	<td>Multivariate proteomic profiling identifies novel accessory proteins of coated vesicles <p class="infolinks">[<a href="javascript:toggleInfo('Borner2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>J Cell Biol<br/>Vol. 197(1), pp. 141-160&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Borner2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Borner2012,
  author = {Borner, Georg HH and Antrobus, Robin and Hirst, Jennifer and Bhumbra, Gary S and Kozik, Patrycja and Jackson, Lauren P and Sahlender, Daniela A and Robinson, Margaret S},
  title = {Multivariate proteomic profiling identifies novel accessory proteins of coated vesicles},
  journal = {J Cell Biol},
  publisher = {Rockefeller University Press},
  year = {2012},
  volume = {197},
  number = {1},
  pages = {141--160}
}
</pre></td>
</tr>
<tr id="Boucher2014" class="entry">
	<td>Boucher, J.I., Jacobowitz, J.R., Beckett, B.C., Classen, S. and Theobald, D.L.</td>
	<td>An atomic-resolution view of neofunctionalization in the evolution of apicomplexan lactate dehydrogenases. <p class="infolinks">[<a href="javascript:toggleInfo('Boucher2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Boucher2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>eLife<br/>Vol. 3(0), pp. e02304 - e02304&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.7554/eLife.02304">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Boucher2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Malate and lactate dehydrogenases (MDH and LDH) are homologous, core metabolic enzymes that share a fold and catalytic mechanism yet possess strict specificity for their substrates. In the Apicomplexa, convergent evolution of an unusual LDH from MDH produced a difference in specificity exceeding 12 orders of magnitude. The mechanisms responsible for this extraordinary functional shift are currently unknown. Using ancestral protein resurrection, we find that specificity evolved in apicomplexan LDHs by classic neofunctionalization characterized by long-range epistasis, a promiscuous intermediate, and few gain-of-function mutations of large effect. In canonical MDHs and LDHs, a single residue in the active-site loop governs substrate specificity: Arg102 in MDHs and Gln102 in LDHs. During the evolution of the apicomplexan LDH, however, specificity switched via an insertion that shifted the position and identity of this 'specificity residue' to Trp107f. Residues far from the active site also determine specificity, as shown by the crystal structures of three ancestral proteins bracketing the key duplication event. This work provides an unprecedented atomic-resolution view of evolutionary trajectories creating a nascent enzymatic function.</td>
</tr>
<tr id="bib_Boucher2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Boucher2014,
  author = {Boucher, Jeffrey I and Jacobowitz, Joseph R and Beckett, Brian C and Classen, Scott and Theobald, Douglas L},
  title = {An atomic-resolution view of neofunctionalization in the evolution of apicomplexan lactate dehydrogenases.},
  journal = {eLife},
  year = {2014},
  volume = {3},
  number = {0},
  pages = {e02304 - e02304},
  note = {NULL},
  doi = {https://doi.org/10.7554/eLife.02304}
}
</pre></td>
</tr>
<tr id="Boucrot2012" class="entry">
	<td>Boucrot, E., Pick, A., Çamdere, G., Liska, N., Evergren, E., McMahon, H.T. and Kozlov, M.M.</td>
	<td>Membrane fission is promoted by insertion of amphipathic helices and is restricted by crescent BAR domains. <p class="infolinks">[<a href="javascript:toggleInfo('Boucrot2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Boucrot2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Cell<br/>Vol. 149, pp. 124-136&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cell.2012.01.047">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Boucrot2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Shallow hydrophobic insertions and crescent-shaped BAR scaffolds promote membrane curvature. Here, we investigate membrane fission by shallow hydrophobic insertions quantitatively and mechanistically. We provide evidence that membrane insertion of the ENTH domain of epsin leads to liposome vesiculation, and that epsin is required for clathrin-coated vesicle budding in cells. We also show that BAR-domain scaffolds from endophilin, amphiphysin, GRAF, and β2-centaurin limit membrane fission driven by hydrophobic insertions. A quantitative assay for vesiculation reveals an antagonistic relationship between amphipathic helices and scaffolds of N-BAR domains in fission. The extent of vesiculation by these proteins and vesicle size depend on the number and length of amphipathic helices per BAR domain, in accord with theoretical considerations. This fission mechanism gives a new framework for understanding membrane scission in the absence of mechanoenzymes such as dynamin and suggests how Arf and Sar proteins work in vesicle scission.</td>
</tr>
<tr id="bib_Boucrot2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Boucrot2012,
  author = {Boucrot, Emmanuel and Pick, Adi and Çamdere, Gamze and Liska, Nicole and Evergren, Emma and McMahon, Harvey T and Kozlov, Michael M},
  title = {Membrane fission is promoted by insertion of amphipathic helices and is restricted by crescent BAR domains.},
  journal = {Cell},
  year = {2012},
  volume = {149},
  pages = {124--136},
  note = {membrane curvature},
  doi = {https://doi.org/10.1016/j.cell.2012.01.047}
}
</pre></td>
</tr>
<tr id="Boulant2011" class="entry">
	<td>Boulant, S., Kural, C., Zeeh, Jean-Christophe., Ubelmann, F. and Kirchhausen, T.</td>
	<td>Actin dynamics counteract membrane tension during clathrin-mediated endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Boulant2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Boulant2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Nature Cell Biology<br/>Vol. 13(9), pp. 1124 - 1131&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/ncb2307">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Boulant2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis is independent of actin dynamics in many circumstances but requires actin polymerization in others. We show that membrane tension determines the actin dependence of clathrin-coat assembly. As found previously, clathrin assembly supports formation of mature coated pits in the absence of actin polymerization on both dorsal and ventral surfaces of non-polarized mammalian cells, and also on basolateral surfaces of polarized cells. Actin engagement is necessary, however, to complete membrane deformation into a coated pit on apical surfaces of polarized cells and, more generally, on the surface of any cell in which the plasma membrane is under tension from osmotic swelling or mechanical stretching. We use these observations to alter actin dependence experimentally and show that resistance of the membrane to propagation of the clathrin lattice determines the distinction between 'actin dependent and 'actin independent'. We also find that light-chain-bound Hip1R mediates actin engagement. These data thus provide a unifying explanation for the role of actin dynamics in coated-pit budding.</td>
</tr>
<tr id="bib_Boulant2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Boulant2011,
  author = {Boulant, Steeve and Kural, Comert and Zeeh, Jean-Christophe and Ubelmann, Florent and Kirchhausen, Tomas},
  title = {Actin dynamics counteract membrane tension during clathrin-mediated endocytosis.},
  journal = {Nature Cell Biology},
  year = {2011},
  volume = {13},
  number = {9},
  pages = {1124 - 1131},
  doi = {https://doi.org/10.1038/ncb2307}
}
</pre></td>
</tr>
<tr id="Boulfelfel2015" class="entry">
	<td>Boulfelfel, S., Ravikovitch, P.I. and Sholl, D.S.</td>
	<td>Modeling Diffusion of Linear Hydrocarbons in Silica Zeolite LTA Using Transition Path Sampling <p class="infolinks">[<a href="javascript:toggleInfo('Boulfelfel2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Boulfelfel2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>The Journal of Physical Chemistry C<br/>Vol. 119(27), pp. 15643 - 15653&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jpcc.5b01633">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Boulfelfel2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The diffusivities of linear hydrocarbons (CH4, C2H6, C2H4, C3H8, C3H6, and C4H10) in pure silica zeolite LTA (ITQ-29) are computed at 300 K and infinite dilution. To overcome the time scale problem arising from the slow diffusion process at room temperature, we used transition path sampling (TPS). The influence of framework flexibility on diffusion is investigated by combining TPS simulations with fully flexible molecular dynamics performed in the NpT ensemble. The ensemble of the collected reactive trajectories was used to characterize sets of transition states, and the corresponding configurations were analyzed to construct window size distributions during the molecular hopping events. The diffusion process is affected by framework flexibility, and the influence of framework flexibility on diffusion of propane and butane is much larger than for methane and ethane. The diffusivities of linear hydrocarbons (CH4, C2H6, C2H4, C3H8, C3H6, and C4H10) in pure silica zeolite LTA (ITQ-29) are computed at 300 K and infinite dilution. To overcome the time scale problem arising from the slow diffusion process at room temperature, we used transition path sampling (TPS). The influence of framework flexibility on diffusion is investigated by combining TPS simulations with fully flexible molecular dynamics performed in the NpT ensemble. The ensemble of the collected reactive trajectories was used to characterize sets of transition states, and the corresponding configurations were analyzed to construct window size distributions during the molecular hopping events. The diffusion process is affected by framework flexibility, and the influence of framework flexibility on diffusion of propane and butane is much larger than for methane and ethane.</td>
</tr>
<tr id="bib_Boulfelfel2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Boulfelfel2015,
  author = {Boulfelfel, Salah and Ravikovitch, Peter I and Sholl, David S},
  title = {Modeling Diffusion of Linear Hydrocarbons in Silica Zeolite LTA Using Transition Path Sampling},
  journal = {The Journal of Physical Chemistry C},
  year = {2015},
  volume = {119},
  number = {27},
  pages = {15643 - 15653},
  doi = {https://doi.org/10.1021/acs.jpcc.5b01633}
}
</pre></td>
</tr>
<tr id="Braams2006" class="entry">
	<td>Braams, B.J. and Manolopoulos, D.E.</td>
	<td>On the short-time limit of ring polymer molecular dynamics <p class="infolinks">[<a href="javascript:toggleInfo('Braams2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Braams2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>The Journal of Chemical Physics<br/>Vol. 125(12), pp. 124105 - 124105&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.2357599">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Braams2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We examine the short-time accuracy of a class of approximate quantum dynamical techniques that includes the centroid molecular dynamics (CMD) and ring polymermolecular dynamics (RPMD) methods. Both of these methods are based on the path integral molecular dynamics (PIMD) technique for calculating the exact static equilibrium properties of quantum mechanical systems. For Kubo-transformed real-time correlation functions involving operators that are linear functions of positions or momenta, the RPMD and (adiabatic) CMD approximations differ only in the choice of the artificial mass matrix of the system of ring polymer beads that is employed in PIMD. The obvious ansatz for a general method of this type is therefore to regard the elements of the PIMD (or Parrinello-Rahman) mass matrix as an adjustable set of parameters that can be chosen to improve the accuracy of the resulting approximation. We show here that this ansatz leads uniquely to the RPMD approximation when the criterion that is used to select the mass matrix is the short-time accuracy of the Kubo-transformed correlation function. In particular, we show that the leading error in the RPMD position autocorrelation function is O ( t 8 ) and the error in the velocity autocorrelation function is O ( t 6 ) , for a general anharmonic potential. The corresponding errors in the CMD approximation are O ( t 6 ) and O ( t 4 ) , respectively.</td>
</tr>
<tr id="bib_Braams2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Braams2006,
  author = {Braams, Bastiaan J and Manolopoulos, David E},
  title = {On the short-time limit of ring polymer molecular dynamics},
  journal = {The Journal of Chemical Physics},
  year = {2006},
  volume = {125},
  number = {12},
  pages = {124105 - 124105},
  note = {NULL},
  doi = {https://doi.org/10.1063/1.2357599}
}
</pre></td>
</tr>
<tr id="Brazhkin2011" class="entry">
	<td>Brazhkin, V., Fomin, Y.D., a Lyapin, G., Ryzhov, V. and Tsiok, E.</td>
	<td>Widom line for the liquid-gas transition in Lennard-Jones system. <p class="infolinks">[<a href="javascript:toggleInfo('Brazhkin2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Brazhkin2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 115(48), pp. 14112 - 5&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp2039898">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Brazhkin2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The locus of extrema (ridges) for heat capacity, thermal expansion coefficient, compressibility, and density fluctuations for model particle systems with Lennard-Jones (LJ) potential in the supercritical region have been obtained. It was found that the ridges for different thermodynamic values virtually merge into a single Widom line at T \textless\ 1.1T(c) and P \textless\ 1.5P(c) and become practically completely smeared at T \textless\ 2.5T(c) and P \textless\ 10P(c), where T(c) and P(c) are the critical temperature and pressure. The ridge for heat capacity approaches close to critical isochore, whereas the lines of extrema for other values correspond to density decrease. The lines corresponding to the supercritical maxima for argon and neon are in good agreement with the computer simulation data for LJ fluid. The behavior of the ridges for LJ fluid, in turn, is close to that for the supercritical van der Waals fluid, which is indicative of a fairly universal behavior of the Widom line for a liquid-gas transition.</td>
</tr>
<tr id="bib_Brazhkin2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Brazhkin2011,
  author = {Brazhkin, VV and Fomin, Yu D and a Lyapin, G and Ryzhov, VN and Tsiok, EN},
  title = {Widom line for the liquid-gas transition in Lennard-Jones system.},
  journal = {Journal of Physical Chemistry B},
  year = {2011},
  volume = {115},
  number = {48},
  pages = {14112 - 5},
  doi = {https://doi.org/10.1021/jp2039898}
}
</pre></td>
</tr>
<tr id="Brehme2009" class="entry">
	<td>Brehme, M., Hantschel, O., Colinge, J., Kaupe, I., Planyavsky, M., K&ouml;cher, T., Mechtler, K., Bennett, K.L. and Superti-Furga, G.</td>
	<td>Charting the molecular network of the drug target Bcr-Abl <p class="infolinks">[<a href="javascript:toggleInfo('Brehme2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Brehme2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Proceedings of the National Academy of Science<br/>Vol. 106(18), pp. 7414 - 9&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.0900653106">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Brehme2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The tyrosine kinase Bcr-Abl causes chronic myeloid leukemia and is the cognate target of tyrosine kinase inhibitors like imatinib. We have charted the protein-protein interaction network of Bcr-Abl by a 2-pronged approach. Using a monoclonal antibody we have first purified endogenous Bcr-Abl protein complexes from the CML K562 cell line and characterized the set of most tightly-associated interactors by MS. Nine interactors were subsequently subjected to tandem affinity purifications/MS analysis to obtain a molecular interaction network of some hundred cellular proteins. The resulting network revealed a high degree of interconnection of 7 "core" components around Bcr-Abl (Grb2, Shc1, Crk-I, c-Cbl, p85, Sts-1, and SHIP-2), and their links to different signaling pathways. Quantitative proteomics analysis showed that tyrosine kinase inhibitors lead to a disruption of this network. Certain components still appear to interact with Bcr-Abl in a phosphotyrosine-independent manner. We propose that Bcr-Abl and other drug targets, rather than being considered as single polypeptides, can be considered as complex protein assemblies that remodel upon drug action.</td>
</tr>
<tr id="bib_Brehme2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Brehme2009,
  author = {Brehme, Marc and Hantschel, Oliver and Colinge, Jacques and Kaupe, Ines and Planyavsky, Melanie and K&ouml;cher, Thomas and Mechtler, Karl and Bennett, Keiryn L and Superti-Furga, Giulio},
  title = {Charting the molecular network of the drug target Bcr-Abl},
  journal = {Proceedings of the National Academy of Science},
  year = {2009},
  volume = {106},
  number = {18},
  pages = {7414 - 9},
  doi = {https://doi.org/10.1073/pnas.0900653106}
}
</pre></td>
</tr>
<tr id="Breitling2008" class="entry">
	<td>Breitling, R., Gilbert, D., Heiner, M. and Orton, R.</td>
	<td>A structured approach for the engineering of biochemical network models, illustrated for signalling pathways <p class="infolinks">[<a href="javascript:toggleInfo('Breitling2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Breitling2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Briefings in Bioinformatics<br/>Vol. 9(5), pp. 404 - 421&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/bib/bbn026">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Breitling2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Quantitative models of biochemical networks (signal transduction cascades, metabolic pathways, gene regulatory circuits) are a central component of modern systems biology. Building and managing these complex models is a major challenge that can benefit from the application of formal methods adopted from theoretical computing science. Here we provide a general introduction to the field of formal modelling, which emphasizes the intuitive biochemical basis of the modelling process, but is also accessible for an audience with a background in computing science and/or model engineering. We show how signal transduction cascades can be modelled in a modular fashion, using both a qualitative approach--qualitative Petri nets, and quantitative approaches--continuous Petri nets and ordinary differential equations (ODEs). We review the major elementary building blocks of a cellular signalling model, discuss which critical design decisions have to be made during model building, and present a number of novel computational tools that can help to explore alternative modular models in an easy and intuitive manner. These tools, which are based on Petri net theory, offer convenient ways of composing hierarchical ODE models, and permit a qualitative analysis of their behaviour. We illustrate the central concepts using signal transduction as our main example. The ultimate aim is to introduce a general approach that provides the foundations for a structured formal engineering of large-scale models of biochemical networks.</td>
</tr>
<tr id="bib_Breitling2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Breitling2008,
  author = {Breitling, Rainer and Gilbert, David and Heiner, Monika and Orton, Richard},
  title = {A structured approach for the engineering of biochemical network models, illustrated for signalling pathways},
  journal = {Briefings in Bioinformatics},
  year = {2008},
  volume = {9},
  number = {5},
  pages = {404 - 421},
  doi = {https://doi.org/10.1093/bib/bbn026}
}
</pre></td>
</tr>
<tr id="Brett2002" class="entry">
	<td>Brett, T.J., Traub, L.M. and Fremont, D.H.</td>
	<td>Accessory protein recruitment motifs in clathrin-mediated endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Brett2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Brett2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Structure<br/>Vol. 10(6), pp. 797-809&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Brett2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis depends upon the interaction of accessory proteins with the alpha-ear of the AP-2 adaptor. We present structural characterization of these regulatory interactions. DPF and DPW motif peptides derived from eps15 and epsin bind in type I beta turn conformations to a conserved pocket on the alpha-ear platform. We show evidence for a second binding site that is DPW motif specific. The structure of a complex with an AP-2 binding segment from amphiphysin reveals a novel binding motif that we term FxDxF, which is engaged in an extended conformation by a unique surface of the platform domain. The FxDxF motif is also used by AP180 and the 170 kDa isoform of synaptojanin and can be found in several potential endocytic proteins, including HIP1, CD2AP, and PLAP. A mechanism of clathrin assembly regulation is suggested by three different AP-2 engagement modes.</td>
</tr>
<tr id="bib_Brett2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Brett2002,
  author = {Brett, Tom J and Traub, Linton M and Fremont, Daved H},
  title = {Accessory protein recruitment motifs in clathrin-mediated endocytosis.},
  journal = {Structure},
  year = {2002},
  volume = {10},
  number = {6},
  pages = {797-809},
  note = {mouse. domain information. AP2A? - EPN1, AMPH, AP180/CALM EPN1 competes with AP180/CALM, AMPH, and EPS15. good figure of alpha adaptin binding sites. dissociation constant.}
}
</pre></td>
</tr>
<tr id="Brodsky2001" class="entry">
	<td>Brodsky, F.M., Chen, C.Y., Knuehl, C., Towler, M.C. and Wakeham, D.E.</td>
	<td>Biological basket weaving: formation and function of clathrin-coated vesicles <p class="infolinks">[<a href="javascript:toggleInfo('Brodsky2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Brodsky2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Annual review of cell and developmental biology<br/>Vol. 17, pp. 517-568&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1146/annurev.cellbio.17.1.517">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Brodsky2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: There has recently been considerable progress in understanding the regulation of clathrin-coated vesicle (CCV) formation and function. These advances are due to the determination of the structure of a number of CCV coat components at molecular resolution and the identification of novel regulatory proteins that control CCV formation in the cell. In addition, pathways of (a) phosphorylation, (b) receptor signaling, and (c) lipid modification that influence CCV formation, as well as the interaction between the cytoskeleton and CCV transport pathways are becoming better defined. It is evident that although clathrin coat assembly drives CCV formation, this fundamental reaction is modified by different regulatory proteins, depending on where CCVs are forming in the cell. This regulatory difference likely reflects the distinct biological roles of CCVs at the plasma membrane and trans-Golgi network, as well as the distinct properties of these membranes themselves. Tissue-specific functions of CCVs require even more-specialized regulation and defects in these pathways can now be correlated with human diseases.</td>
</tr>
<tr id="bib_Brodsky2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Brodsky2001,
  author = {Brodsky, F M and Chen, C Y and Knuehl, C and Towler, M C and Wakeham, D E},
  title = {Biological basket weaving: formation and function of clathrin-coated vesicles},
  journal = {Annual review of cell and developmental biology},
  year = {2001},
  volume = {17},
  pages = {517--568},
  doi = {https://doi.org/10.1146/annurev.cellbio.17.1.517}
}
</pre></td>
</tr>
<tr id="Brooks2009" class="entry">
	<td>Brooks, B., Brooks, C., Mackerell, A., Nilsson, L., Petrella, R., Roux, B., Won, Y., Archontis, G., Bartels, C., Boresch, S., Caflisch, A., Caves, L., Cui, Q., Dinner, A., Feig, M., Fischer, S., Gao, J., Hodoscek, M., Im, W., Kuczera, K., Lazaridis, T., Ma, J., Ovchinnikov, V., Paci, E., Pastor, R., Post, C., Pu, J., Schaefer, M., Tidor, B., Venable, R., Woodcock, H., Wu, X., Yang, W., York, D. and Karplus, M.</td>
	<td>CHARMM: The biomolecular simulation program <p class="infolinks">[<a href="javascript:toggleInfo('Brooks2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Brooks2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Journal of Computational Chemistry<br/>Vol. 30(10), pp. 1545 - 614&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/jcc.21287">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Brooks2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: CHARMM (Chemistry at HARvard Molecular Mechanics) is a highly versatile and widely used molecular simulation program. It has been developed over the last three decades with a primary focus on molecules of biological interest, including proteins, peptides, lipids, nucleic acids, carbohydrates, and small molecule ligands, as they occur in solution, crystals, and membrane environments. For the study of such systems, the program provides a large suite of computational tools that include numerous conformational and path sampling methods, free energy estimators, molecular minimization, dynamics, and analysis techniques, and model-building capabilities. The CHARMM program is applicable to problems involving a much broader class of many-particle systems. Calculations with CHARMM can be performed using a number of different energy functions and models, from mixed quantum mechanical-molecular mechanical force fields, to all-atom classical potential energy functions with explicit solvent and various boundary conditions, to implicit solvent and membrane models. The program has been ported to numerous platforms in both serial and parallel architectures. This article provides an overview of the program as it exists today with an emphasis on developments since the publication of the original CHARMM article in 1983. textcopyright 2009 Wiley Periodicals, Inc.J Comput Chem, 2009.</td>
</tr>
<tr id="bib_Brooks2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Brooks2009,
  author = {Brooks, BR and Brooks, CL and Mackerell, AD and Nilsson, L and Petrella, RJ and Roux, B and Won, Y and Archontis, G and Bartels, C and Boresch, S and Caflisch, A and Caves, L and Cui, Q and Dinner, AR and Feig, M and Fischer, S and Gao, J and Hodoscek, M and Im, W and Kuczera, K and Lazaridis, T and Ma, J and Ovchinnikov, V and Paci, E and Pastor, RW and Post, CB and Pu, JZ and Schaefer, M and Tidor, B and Venable, RM and Woodcock, HL and Wu, X and Yang, W and York, DM and Karplus, M},
  title = {CHARMM: The biomolecular simulation program},
  journal = {Journal of Computational Chemistry},
  year = {2009},
  volume = {30},
  number = {10},
  pages = {1545 - 614},
  note = {NULL},
  doi = {https://doi.org/10.1002/jcc.21287}
}
</pre></td>
</tr>
<tr id="Brooks1983" class="entry">
	<td>Brooks, B.R., Bruccoleri, R.E., Olafson, B.D., States, D.J., Swaminathan, S. and Karplus, M.</td>
	<td>CHARMM: A program for macromolecular energy, minimization, and dynamics calculations <p class="infolinks">[<a href="javascript:toggleInfo('Brooks1983','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Brooks1983','bibtex')">BibTeX</a>]</p></td>
	<td>1983</td>
	<td>Journal of Computational Chemistry<br/>Vol. 4(2), pp. 187 - 217&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/jcc.540040211">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Brooks1983" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: CHARMM (Chemistry at HARvard Macromolecular Mechanics) is a highly flexible computer program which uses empirical energy functions to model macromolecular systems. The program can read or model build structures, energy minimize them by first- or second-derivative techniques, perform a normal mode or molecular dynamics simulation, and analyze the structural, equilibrium, and dynamic properties determined in these calculations. The operations that CHARMM can perform are described, and some implementation details are given. A set of parameters for the empirical energy function and a sample run are included.</td>
</tr>
<tr id="bib_Brooks1983" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Brooks1983,
  author = {Brooks, Bernard R and Bruccoleri, Robert E and Olafson, Barry D and States, David J and Swaminathan, S and Karplus, Martin},
  title = {CHARMM: A program for macromolecular energy, minimization, and dynamics calculations},
  journal = {Journal of Computational Chemistry},
  year = {1983},
  volume = {4},
  number = {2},
  pages = {187 - 217},
  note = {NULL},
  doi = {https://doi.org/10.1002/jcc.540040211}
}
</pre></td>
</tr>
<tr id="Brothers2004" class="entry">
	<td>Brothers, E.N., Suarez, D., Deerfield, D.W. and Merz, K.M.</td>
	<td>PM3-compatible zinc parameters optimized for metalloenzyme active sites. <p class="infolinks">[<a href="javascript:toggleInfo('Brothers2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Brothers2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of Computational Chemistry<br/>Vol. 25(14), pp. 1677 - 92&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/jcc.20086">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Brothers2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Recent studies have shown that semiempirical methods (e.g., PM3 and AM1) for zinc-containing compounds are unreliable for modeling structures containing zinc ions with ligand environments similar to those observed in zinc metalloenzymes. To correct these deficiencies a reparameterization of zinc at the PM3 level was undertaken. In this effort we included frequency corrected B3LYP/6-311G* zinc metalloenzyme ligand environments along with previously utilized experimental data. Average errors for the heats of formation have been reduced from 46.9 kcal/mol (PM3) to 14.2 kcal/mol for this new parameter set, termed ZnB for "Zinc, Biological." In addition, the new parameter sets predict geometries for the Bacillus fragilis active site model and other zinc metalloenzyme mimics that are qualitatively in agreement with high-level ab initio results, something existing parameter sets failed to do.</td>
</tr>
<tr id="bib_Brothers2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Brothers2004,
  author = {Brothers, Edward N and Suarez, Dimas and Deerfield, David W and Merz, Kenneth M},
  title = {PM3-compatible zinc parameters optimized for metalloenzyme active sites.},
  journal = {Journal of Computational Chemistry},
  year = {2004},
  volume = {25},
  number = {14},
  pages = {1677 - 92},
  note = {parameters used for the active site quantum calculations},
  doi = {https://doi.org/10.1002/jcc.20086}
}
</pre></td>
</tr>
<tr id="Brown2004" class="entry">
	<td>Brown, M.W., Yowell, C.A., Hoard, A., Jagt, T.A., Hunsaker, L.A., Deck, L.M., Royer, R.E., Piper, R.C., Dame, J.B., Makler, M.T. and Jagt, D.L.</td>
	<td>Comparative structural analysis and kinetic properties of lactate dehydrogenases from the four species of human malarial parasites <p class="infolinks">[<a href="javascript:toggleInfo('Brown2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Brown2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Biochemistry<br/>Vol. 43(20), pp. 6219 - 6229&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi049892w">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Brown2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Parasite lactate dehydrogenase (pLDH) is a potential drug target for new antimalarials owing to parasite dependence on glycolysis for ATP production. The pLDH from all four species of human malarial parasites were cloned, expressed, and analyzed for structural and kinetic properties that might be exploited for drug development. pLDH from Plasmodium vivax, malariae, and ovale exhibit 90-92&#37; identity to pLDH from Plasmodium falciparum. Catalytic residues are identical. Resides I250 and T246, conserved in most LDH, are replaced by proline in all pLDH. The pLDH contain the same five-amino acid insert (DKEWN) in the substrate specificity loops. Within the cofactor site, pLDH from P. falciparum and P. malariae are identical, while pLDH from P. vivax and P. ovale have one substitution. Homology modeling of pLDH from P. vivax, ovale, and malariae with the crystal structure of pLDH from P. falciparum gave nearly identical structures. Nevertheless, the kinetic properties and sensitivities to inhibitors targeted to the cofactor binding site differ significantly. Michaelis constants for pyruvate and lactate differ 8-9-fold; Michaelis constants for NADH, NAD(+), and the NAD(+) analogue 3-acetylpyridine adenine dinucleotide differ up to 4-fold. Dissociation constants for the inhibitors differ up to 21-fold. Molecular docking studies of the binding of the inhibitors to the cofactor sites of all four pLDH predict similar orientations, with the docked ligands positioned at the nicotinamide end of the cofactor site. pH studies indicate that inhibitor binding is independent of pH in the pH 6-8 range, suggesting that differences in dissociation constants for a specific inhibitor are not due to altered active site pK values among the four pLDH.</td>
</tr>
<tr id="bib_Brown2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Brown2004,
  author = {Brown, Michael W and Yowell, Charles A and Hoard, Anna and Jagt, Thomas A and Hunsaker, Lucy A and Deck, Lorraine M and Royer, Robert E and Piper, Robert C and Dame, John B and Makler, Michael T and Jagt, David L},
  title = {Comparative structural analysis and kinetic properties of lactate dehydrogenases from the four species of human malarial parasites},
  journal = {Biochemistry},
  year = {2004},
  volume = {43},
  number = {20},
  pages = {6219 - 6229},
  note = {NULL},
  doi = {https://doi.org/10.1021/bi049892w}
}
</pre></td>
</tr>
<tr id="Bryant2000" class="entry">
	<td>Bryant, Z., Pande, V.S. and Rokhsar, D.S.</td>
	<td>Mechanical unfolding of a beta-hairpin using molecular dynamics. <p class="infolinks">[<a href="javascript:toggleInfo('Bryant2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bryant2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Biophysical Journal<br/>Vol. 78(February), pp. 584 - 589&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S0006-3495(00)76618-5">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Bryant2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Single-molecule mechanical unfolding experiments have the potential to provide insights into the details of protein folding pathways. To investigate the relationship between force-extension unfolding curves and microscopic events, we performed molecular dynamics simulations of the mechanical unfolding of the C-terminal hairpin of protein G. We have studied the dependence of the unfolding pathway on pulling speed, cantilever stiffness, and attachment points. Under conditions that generate low forces, the unfolding trajectory mimics the untethered, thermally accessible pathway previously proposed based on high-temperature studies. In this stepwise pathway, complete breakdown of backbone hydrogen bonds precedes dissociation of the hydrophobic cluster. Under more extreme conditions, the cluster and hydrogen bonds break simultaneously. Transitions between folding intermediates can be identified in our simulations as features of the calculated force-extension curves.</td>
</tr>
<tr id="bib_Bryant2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bryant2000,
  author = {Bryant, Zev and Pande, Vijay S and Rokhsar, Daniel S},
  title = {Mechanical unfolding of a beta-hairpin using molecular dynamics.},
  journal = {Biophysical Journal},
  year = {2000},
  volume = {78},
  number = {February},
  pages = {584 - 589},
  doi = {https://doi.org/10.1016/S0006-3495(00)76618-5}
}
</pre></td>
</tr>
<tr id="Bryson1995" class="entry">
	<td>Bryson, J., Betz, S., Lu, H., Suich, D., Zhou, H., O'Neil, K. and DeGrado, W.</td>
	<td>Protein Design: A Hierarchic Approach <p class="infolinks">[<a href="javascript:toggleInfo('Bryson1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>Science<br/>Vol. 270(5238), pp. 935 - 941&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1126/science.270.5238.935">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Bryson1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bryson1995,
  author = {Bryson, JW and Betz, SF and Lu, HS and Suich, DJ and Zhou, HX and O'Neil, KT and DeGrado, WF},
  title = {Protein Design: A Hierarchic Approach},
  journal = {Science},
  year = {1995},
  volume = {270},
  number = {5238},
  pages = {935 - 941},
  doi = {https://doi.org/10.1126/science.270.5238.935}
}
</pre></td>
</tr>
<tr id="Burgess2010" class="entry">
	<td>Burgess, S.A., Lindsay, D. and Flint, S.H.</td>
	<td>Thermophilic bacilli and their importance in dairy processing. <p class="infolinks">[<a href="javascript:toggleInfo('Burgess2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Burgess2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>International journal of food microbiology<br/>Vol. 144(2), pp. 215 - 25&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.ijfoodmicro.2010.09.027">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Burgess2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The thermophilic bacilli, such as Anoxybacillus flavithermus and Geobacillus spp., are an important group of contaminants in the dairy industry. Although these bacilli are generally not pathogenic, their presence in dairy products is an indicator of poor hygiene and high numbers are unacceptable to customers. In addition, their growth may result in milk product defects caused by the production of acids or enzymes, potentially leading to off-flavours. Dairy thermophiles are usually selected for by the conditions during dairy manufacture. These bacteria are able to grow in sections of dairy manufacturing plants where temperatures reach 40-65textdegreeC. Furthermore, because they are spore formers, they are difficult to eliminate. In addition, they exhibit a wide temperature growth range, exhibit a fast growth rate (generation time of approximately 15-20 min) and tend to readily form biofilms. Many strategies have been tested to remove, prevent and/or delay the formation of thermophilic bacilli biofilms in dairy manufacture, but with limited success. This is, in part, because little is known about the structure and composition of thermophilic bacilli biofilms in general and, more specifically, in milk processing environments. Therefore, new cleaning regimes often do not target the problem optimally. A greater understanding of the structure of thermophilic biofilms within the context of the milk processing environment and their link with spore formation is needed to develop better control measures. This review discusses the characteristics and food spoilage potential, enumeration and identification methods for the thermophilic bacilli, as well as their importance to dairy manufacture, with an emphasis on biofilm development and spore formation.</td>
</tr>
<tr id="bib_Burgess2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Burgess2010,
  author = {Burgess, Sara A and Lindsay, Denise and Flint, Steve H},
  title = {Thermophilic bacilli and their importance in dairy processing.},
  journal = {International journal of food microbiology},
  year = {2010},
  volume = {144},
  number = {2},
  pages = {215 - 25},
  note = {Temperature ranges for G. stearothermophilus},
  doi = {https://doi.org/10.1016/j.ijfoodmicro.2010.09.027}
}
</pre></td>
</tr>
<tr id="Burgner1984" class="entry">
	<td>Burgner, J.W. and Ray, W.J.</td>
	<td>On the origin of the lactate dehydrogenase induced rate effect. <p class="infolinks">[<a href="javascript:toggleInfo('Burgner1984','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Burgner1984','bibtex')">BibTeX</a>]</p></td>
	<td>1984</td>
	<td>Biochemistry<br/>Vol. 23(16), pp. 3636 - 48&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi00311a010">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Burgner1984" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: To evaluate the ability of lactate dehydrogenase to facilitate the bond making/breaking steps for both the addition of pyruvate enol to NAD (pyruvate adduct reaction) and the normal redox reaction, the ability of the enzyme to facilitate the tautomerization of bound pyruvate is assessed. In addition, the equilibrium constants for the adduct reaction are obtained for both bound and free reactants from the ratio of the rate constants in the forward and reverse reactions (at pH 7). The latter comparison indicates that the enzyme facilitates bond making/breaking in the (forward) pyruvate adduct reaction by a factor of about 10(11) M. Similar comparisons suggest that reactant immobilization accounts for about 1000 M of this 10(11) M rate effect. Since the (pH-independent) rate constant for the ketonization of bound pyruvate enol assisted by the external buffer, imidazolium ion, is 2 X 10(7) M-1 s-1 and the corresponding rate constant for free pyruvate enol, again assisted by imidazolium ion, is 35 M-1 s-1 [Burger, J. W., II, &amp; Ray, W. J., Jr. (1978) Biochemistry 17, 1664], the enzyme facilitates the bond making/breaking steps associated with the conversion of bound HO-C less than to bound O = C less than by a factor of about 10(6)-fold. The product of the above two rate enhancement factors and the rate factor suggested previously for the environmental effect on NAD produced by its binding to lactate dehydrogenase, 100-fold, is 10(11) M, and it accounts for the bond making/breaking effects exerted by the enzyme in the pyruvate adduct reaction. The rate constant for oxidation of ethanol (a model for lactate) by 1-methylnicotinamide (a model for NAD) is about 5 X 10(-12) M-1 s-1 at 25 degrees C in pure ethanol (delta H for this reaction is about 30 kcal/mol). The ratio of the rate constants for E X NAD X Lac----E X NADH X Pyr and the above model reaction is estimated as about 10(14) M in water; i.e., the LDH-induced rate effect is about 10(14) M. The product of the values for the above rate factors for the normal redox reaction is about 10(12) M. Although the value of this product is less certain than that for the adduct reaction, these rate factors do account for much of the LDH-induced rate effect.</td>
</tr>
<tr id="bib_Burgner1984" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Burgner1984,
  author = {Burgner, John W and Ray, William J},
  title = {On the origin of the lactate dehydrogenase induced rate effect.},
  journal = {Biochemistry},
  year = {1984},
  volume = {23},
  number = {16},
  pages = {3636 - 48},
  doi = {https://doi.org/10.1021/bi00311a010}
}
</pre></td>
</tr>
<tr id="Bzik1993" class="entry">
	<td>Bzik, D.J., Fox, B.A. and Gonyer, K.</td>
	<td>Expression of Plasmodium falciparum lactate dehydrogenase in Escherichia coli <p class="infolinks">[<a href="javascript:toggleInfo('Bzik1993','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bzik1993','bibtex')">BibTeX</a>]</p></td>
	<td>1993</td>
	<td>Molecular and Biochemical Parasitology<br/>Vol. 59(1), pp. 155 - 166&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/0166-6851(93)90016-Q">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Bzik1993" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A Plasmodium falciparum gene is described which encodes lactate dehydrogenase activity P. falciparum LDH). The P. falciparum LDH gene contains no introns and is present in a single copy on chromosome 13. P. falciparum LDH was expressed in all asexual blood stages as a 1.6-kb mRNA. The predicted 316 amino acid protein coding region of P. falciparum LDH was inserted into the prokaryotic expression vector pKK223-3 and a 33-kDa protein having LDH activity was synthesized in Escherichia coli. P. falciparumLDH primary structure displays high amino acid similarity (50--57 to vertebrate and bacterial LDH, but lacks the amino terminal extension observed in all vertebrate LDH. The majority of amino acid residues implicated in substrate and coenzyme binding and catalysis of other LDH are well conserved in P. falciparum LDH. However, several notable differences in amino acid composition were observed. P. falciparum LDH contained several distinctive single amino acid insertions and deletions compared to other LDH enzymes, and most remarkably, it contained a novel insertion of 5 amino acids within the conserved mobile loop region near arginine residue 109, a residue which is known to make contact with pyruvate in the ternary complex of other LDH. These results suggest that novel features of P. falciparum LDH primary structure may be correlated with previously characterized and distinctive kinetic, biochemical, immunochemical, and electrophoretic properties of P. falciparum LDH.</td>
</tr>
<tr id="bib_Bzik1993" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Bzik1993,
  author = {Bzik, David J and Fox, Barbara A and Gonyer, Kenneth},
  title = {Expression of Plasmodium falciparum lactate dehydrogenase in Escherichia coli},
  journal = {Molecular and Biochemical Parasitology},
  year = {1993},
  volume = {59},
  number = {1},
  pages = {155 - 166},
  note = {first paper showing five residue insertion into the substrate-specificity loop of pfldh},
  doi = {https://doi.org/10.1016/0166-6851(93)90016-Q}
}
</pre></td>
</tr>
<tr id="Cameron2004" class="entry">
	<td>Cameron, A., Read, J., Tranter, R., Winter, V.J., Sessions, R.B., Brady, L., Vivas, L., Easton, A., Kendrick, H., Croft, S.L., Barros, D., Lavandera, J., Martin, J., Risco, F., Garcia-Ochoa, S., Gamo, F., Sanz, L., Leon, L., Ruiz, J.R., Gabarro, R., Mallon, A. and Heras, F.</td>
	<td>Identification and activity of a series of azole-based compounds with lactate dehydrogenase-directed anti-malarial activity <p class="infolinks">[<a href="javascript:toggleInfo('Cameron2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Cameron2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of Biological Chemistry<br/>Vol. 279(30), pp. 31429 - 31439&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M402433200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Cameron2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Plasmodium falciparum, the causative agent of malaria, relies extensively on glycolysis coupled with homolactic fermentation during its blood-borne stages for energy production. Selective inhibitors of the parasite lactate dehydrogenase (LDH), central to NAD(+) regeneration, therefore potentially provide a route to new antimalarial drugs directed against a novel molecular target. A series of heterocyclic, azole-based compounds are described that preferentially inhibit P. falciparum LDH at sub-micromolar concentrations, typically at concentrations about 100-fold lower than required for human lactate dehydrogenase inhibition. Crystal structures show these competitive inhibitors form a network of interactions with amino acids within the active site of the enzyme, stacking alongside the nicotinamide ring of the NAD(+) cofactor. These compounds display modest activity against parasitized erythrocytes, including parasite strains with known resistance to existing anti-malarials and against Plasmodium berghei in BALB/c mice. Initial toxicity data suggest the azole derivatives have generally low cytotoxicity, and preliminary pharmoco-kinetic data show favorable bioavailability and circulation times. These encouraging results suggest that further enhancement of these structures may yield candidates suitable for consideration as new therapeutics for the treatment of malaria. In combination these studies also provide strong support for the validity of targeting the Plasmodium glycolytic pathway and, in particular, LDH in the search for novel anti-malarials.</td>
</tr>
<tr id="bib_Cameron2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cameron2004,
  author = {Cameron, Angus and Read, Jon and Tranter, Rebecca and Winter, Victoria J and Sessions, Richard B and Brady, Leo and Vivas, Livia and Easton, Anna and Kendrick, Howard and Croft, Simon L and Barros, David and Lavandera, Jose and Martin, Jose and Risco, Felix and Garcia-Ochoa, Silvestre and Gamo, Fracisco and Sanz, Laura and Leon, Luisa and Ruiz, Jose R and Gabarro, Raquel and Mallon, Araceli and Heras, Federico},
  title = {Identification and activity of a series of azole-based compounds with lactate dehydrogenase-directed anti-malarial activity},
  journal = {Journal of Biological Chemistry},
  year = {2004},
  volume = {279},
  number = {30},
  pages = {31429 - 31439},
  doi = {https://doi.org/10.1074/jbc.M402433200}
}
</pre></td>
</tr>
<tr id="Cao1998" class="entry">
	<td>Cao, H., Garcia, F. and McNiven, M.A.</td>
	<td>Differential distribution of dynamin isoforms in mammalian cells. <p class="infolinks">[<a href="javascript:toggleInfo('Cao1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Cao1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Molecular biology of the cell<br/>Vol. 9, pp. 2595-2609&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Cao1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Dynamins are 100-kDa GTPases that are essential for clathrin-coated vesicle formation during receptor-mediated endocytosis. To date, three different dynamin genes have been identified, with each gene expressing at least four different alternatively spliced forms. Currently, it is unclear whether these different dynamin gene products perform distinct or redundant cellular functions. Therefore, the focus of this study was to identify additional spliced variants of dynamin from rat tissues and to define the distribution of the dynamin family members in a cultured rat epithelial cell model (Clone 9 cells). After long-distance reverse transcription (RT)-PCR of mRNA from different rat tissues, the full-length cDNAs encoding the different dynamin isoforms were sequenced and revealed four additional spliced variants for dynamin I and nine for dynamin III. Thus, in rat tissues there are a total of at least 25 different mRNAs produced from the three dynamin genes. Subsequently, we generated stably transfected Clone 9 cells expressing full-length cDNAs of six different spliced forms tagged with green fluorescent protein. Confocal or fluorescence microscopy of these transfected cells revealed that many of the dynamin proteins associate with distinct membrane compartments, which include clathrin-coated pits at the plasma membrane and the Golgi apparatus, and several undefined vesicle populations. These results indicate that the dynamin family is more extensive than was originally predicted and suggest that the different dynamin proteins are localized to distinct cytoplasmic or membrane compartments.</td>
</tr>
<tr id="bib_Cao1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cao1998,
  author = {Cao, H and Garcia, F and McNiven, M A},
  title = {Differential distribution of dynamin isoforms in mammalian cells.},
  journal = {Molecular biology of the cell},
  year = {1998},
  volume = {9},
  pages = {2595--2609},
  note = {dyna1 is brain only, dyna2 is ubiquitously expressed}
}
</pre></td>
</tr>
<tr id="Cao1994a" class="entry">
	<td>Cao, J. and Voth, G.A.</td>
	<td>The formulation of quantum statistical mechanics based on the Feynman path centroid density. I. Equilibrium properties <p class="infolinks">[<a href="javascript:toggleInfo('Cao1994a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Cao1994a','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td>The Journal of Chemical Physics<br/>Vol. 101(7), pp. 6184 - 6192&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.468400">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Cao1994a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The concept of instantaneous normal modes in liquids is extended into the quantum regime using the Feynman path centroid perspective in quantum statistical mechanics. To accomplish this goal, the variational quadratic approximation for the effective centroid potential is recast in a general multidimensional phase space form. In the context of the effective quadratic approximation, the velocity autocorrelation functions of liquids can then be predicted based on a set of instantaneous quantumnormal modes. Representative applications are presented for quantum Lennard‐Jones liquids and a quantum particle solvated in a classical fluid. The quantum effective phonon spectrum leads to some revealing observations and interpretations for these systems.</td>
</tr>
<tr id="bib_Cao1994a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cao1994a,
  author = {Cao, Jianshu and Voth, Gregory A},
  title = {The formulation of quantum statistical mechanics based on the Feynman path centroid density. I. Equilibrium properties},
  journal = {The Journal of Chemical Physics},
  year = {1994},
  volume = {101},
  number = {7},
  pages = {6184 - 6192},
  note = {NULL},
  doi = {https://doi.org/10.1063/1.468400}
}
</pre></td>
</tr>
<tr id="Cao1994b" class="entry">
	<td>Cao, J. and Voth, G.A.</td>
	<td>The formulation of quantum statistical mechanics based on the Feynman path centroid density. II. Dynamical properties <p class="infolinks">[<a href="javascript:toggleInfo('Cao1994b','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Cao1994b','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td>The Journal of Chemical Physics<br/>Vol. 101(7), pp. 6184 - 6192&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.468400">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Cao1994b" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The concept of instantaneous normal modes in liquids is extended into the quantum regime using the Feynman path centroid perspective in quantum statistical mechanics. To accomplish this goal, the variational quadratic approximation for the effective centroid potential is recast in a general multidimensional phase space form. In the context of the effective quadratic approximation, the velocity autocorrelation functions of liquids can then be predicted based on a set of instantaneous quantumnormal modes. Representative applications are presented for quantum Lennard‐Jones liquids and a quantum particle solvated in a classical fluid. The quantum effective phonon spectrum leads to some revealing observations and interpretations for these systems.</td>
</tr>
<tr id="bib_Cao1994b" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cao1994b,
  author = {Cao, Jianshu and Voth, Gregory A},
  title = {The formulation of quantum statistical mechanics based on the Feynman path centroid density. II. Dynamical properties},
  journal = {The Journal of Chemical Physics},
  year = {1994},
  volume = {101},
  number = {7},
  pages = {6184 - 6192},
  note = {NULL},
  doi = {https://doi.org/10.1063/1.468400}
}
</pre></td>
</tr>
<tr id="Cao1994c" class="entry">
	<td>Cao, J. and Voth, G.A.</td>
	<td>The formulation of quantum statistical mechanics based on the Feynman path centroid density. III. Phase space formalism and analysis of centroid molecular dynamics <p class="infolinks">[<a href="javascript:toggleInfo('Cao1994c','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Cao1994c','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td>The Journal of Chemical Physics<br/>Vol. 101(7), pp. 6184 - 6192&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.468400">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Cao1994c" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The concept of instantaneous normal modes in liquids is extended into the quantum regime using the Feynman path centroid perspective in quantum statistical mechanics. To accomplish this goal, the variational quadratic approximation for the effective centroid potential is recast in a general multidimensional phase space form. In the context of the effective quadratic approximation, the velocity autocorrelation functions of liquids can then be predicted based on a set of instantaneous quantumnormal modes. Representative applications are presented for quantum Lennard‐Jones liquids and a quantum particle solvated in a classical fluid. The quantum effective phonon spectrum leads to some revealing observations and interpretations for these systems.</td>
</tr>
<tr id="bib_Cao1994c" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cao1994c,
  author = {Cao, Jianshu and Voth, Gregory A},
  title = {The formulation of quantum statistical mechanics based on the Feynman path centroid density. III. Phase space formalism and analysis of centroid molecular dynamics},
  journal = {The Journal of Chemical Physics},
  year = {1994},
  volume = {101},
  number = {7},
  pages = {6184 - 6192},
  note = {NULL},
  doi = {https://doi.org/10.1063/1.468400}
}
</pre></td>
</tr>
<tr id="Cao1994d" class="entry">
	<td>Cao, J. and Voth, G.A.</td>
	<td>The formulation of quantum statistical mechanics based on the Feynman path centroid density. IV. Algorithms for centroid molecular dynamics <p class="infolinks">[<a href="javascript:toggleInfo('Cao1994d','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Cao1994d','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td>The Journal of Chemical Physics<br/>Vol. 101(7), pp. 6184 - 6192&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.468400">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Cao1994d" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The concept of instantaneous normal modes in liquids is extended into the quantum regime using the Feynman path centroid perspective in quantum statistical mechanics. To accomplish this goal, the variational quadratic approximation for the effective centroid potential is recast in a general multidimensional phase space form. In the context of the effective quadratic approximation, the velocity autocorrelation functions of liquids can then be predicted based on a set of instantaneous quantumnormal modes. Representative applications are presented for quantum Lennard‐Jones liquids and a quantum particle solvated in a classical fluid. The quantum effective phonon spectrum leads to some revealing observations and interpretations for these systems.</td>
</tr>
<tr id="bib_Cao1994d" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cao1994d,
  author = {Cao, Jianshu and Voth, Gregory A},
  title = {The formulation of quantum statistical mechanics based on the Feynman path centroid density. IV. Algorithms for centroid molecular dynamics},
  journal = {The Journal of Chemical Physics},
  year = {1994},
  volume = {101},
  number = {7},
  pages = {6184 - 6192},
  note = {NULL},
  doi = {https://doi.org/10.1063/1.468400}
}
</pre></td>
</tr>
<tr id="Cao1994e" class="entry">
	<td>Cao, J. and Voth, G.A.</td>
	<td>The formulation of quantum statistical mechanics based on the Feynman path centroid density. V. Quantum instantaneous normal mode theory of liquids <p class="infolinks">[<a href="javascript:toggleInfo('Cao1994e','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Cao1994e','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td>The Journal of Chemical Physics<br/>Vol. 101(7), pp. 6184 - 6192&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.468400">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Cao1994e" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The concept of instantaneous normal modes in liquids is extended into the quantum regime using the Feynman path centroid perspective in quantum statistical mechanics. To accomplish this goal, the variational quadratic approximation for the effective centroid potential is recast in a general multidimensional phase space form. In the context of the effective quadratic approximation, the velocity autocorrelation functions of liquids can then be predicted based on a set of instantaneous quantumnormal modes. Representative applications are presented for quantum Lennard‐Jones liquids and a quantum particle solvated in a classical fluid. The quantum effective phonon spectrum leads to some revealing observations and interpretations for these systems.</td>
</tr>
<tr id="bib_Cao1994e" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cao1994e,
  author = {Cao, Jianshu and Voth, Gregory A},
  title = {The formulation of quantum statistical mechanics based on the Feynman path centroid density. V. Quantum instantaneous normal mode theory of liquids},
  journal = {The Journal of Chemical Physics},
  year = {1994},
  volume = {101},
  number = {7},
  pages = {6184 - 6192},
  note = {NULL},
  doi = {https://doi.org/10.1063/1.468400}
}
</pre></td>
</tr>
<tr id="Caratzoulas2002" class="entry">
	<td>Caratzoulas, S., Mincer, J.S. and Schwartz, S.D.</td>
	<td>Identification of a protein-promoting vibration in the reaction catalyzed by horse liver alcohol dehydrogenase. <p class="infolinks">[<a href="javascript:toggleInfo('Caratzoulas2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Caratzoulas2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Journal of the American Chemical Society<br/>Vol. 124(13), pp. 3270 - 3276&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ja017146y">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Caratzoulas2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In this article we present computational studies of horse liver alcohol dehydrogenase (HLADH). The computations identify a rate-promoting vibration that is symmetrically coupled to the reaction coordinate. In HLADH a bulky amino acid (Val203) is positioned at the face of the nicotinamide adenine dinucleotide (NAD(+)) cofactor distal to alcohol substrate to restrict the separation of reactants and control the stereochemistry. Molecular dynamics simulations were performed on the dimeric HLADH, including the NAD cofactor, the substrate, and the crystallographic waters, for three different configurations, reactants, products, and transition state. From the spectral density for the substrate-NAD relative motion, and that for the NAD-Val203 relative motion, we find that the two motions are in resonance. By computing the associated spectrum, we find that the reaction coordinate is coupled with the substrate-NAD motion, and from the fact that the coupling vanishes at or near the transition state (demonstrated by the disappearance of strong features in the spectral density), we conclude that the substrate-NAD motion plays the role of a promoting vibration symmetrically coupled to the reaction coordinate.</td>
</tr>
<tr id="bib_Caratzoulas2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Caratzoulas2002,
  author = {Caratzoulas, Stavros and Mincer, Joshua S and Schwartz, Steven D},
  title = {Identification of a protein-promoting vibration in the reaction catalyzed by horse liver alcohol dehydrogenase.},
  journal = {Journal of the American Chemical Society},
  year = {2002},
  volume = {124},
  number = {13},
  pages = {3270 - 3276},
  note = {NULL},
  doi = {https://doi.org/10.1021/ja017146y}
}
</pre></td>
</tr>
<tr id="Carballo-Pacheco2016" class="entry">
	<td>Carballo-Pacheco, M. and Strodel, B.</td>
	<td>Advances in the Simulation of Protein Aggregation at the Atomistic Scale. <p class="infolinks">[<a href="javascript:toggleInfo('Carballo-Pacheco2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Carballo-Pacheco2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 120(12), pp. 2991-2999&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jpcb.6b00059">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Carballo-Pacheco2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Protein aggregation into highly-structured amyloids fibrils is associated with various diseases including Alzheimer's disease, Parkinson's disease and type II diabetes. Amyloids can also have normal biological functions and, in the future, could be used as the basis for novel nanoscale materials. However, a full understanding of the physicochemical forces that drive protein aggregation is still lacking. Such understanding is crucial for the development of drugs that can effectively inhibit aberrant amyloid aggregation and for the directed design of functional amyloids. Atomistic simulations can help understand protein aggregation. In particular, atomistic simulations can be used to study the initial formation of toxic oligomers which are hard to characterize experimentally, and to understand the difference in aggregation behaviour between different amyloidogenic peptides. Here, we review the latest atomistic simulations of protein aggregation, concentrating on amyloidogenic protein fragments, and provide an outlook for the future in this field.</td>
</tr>
<tr id="bib_Carballo-Pacheco2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Carballo-Pacheco2016,
  author = {Carballo-Pacheco, Martin and Strodel, Birgit},
  title = {Advances in the Simulation of Protein Aggregation at the Atomistic Scale.},
  journal = {Journal of Physical Chemistry B},
  year = {2016},
  volume = {120},
  number = {12},
  pages = {2991-2999},
  doi = {https://doi.org/10.1021/acs.jpcb.6b00059}
}
</pre></td>
</tr>
<tr id="Caves1998" class="entry">
	<td>Caves, L.S., Evanseck, J.D. and Karplus, M.</td>
	<td>Locally accessible conformations of proteins: Multiple molecular dynamics simulations of crambin <p class="infolinks">[<a href="javascript:toggleInfo('Caves1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Protein Science<br/>Vol. 7, pp. 649 - 666&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Caves1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Caves1998,
  author = {Caves, Leo SD and Evanseck, Jeffrey D and Karplus, Martin},
  title = {Locally accessible conformations of proteins: Multiple molecular dynamics simulations of crambin},
  journal = {Protein Science},
  year = {1998},
  volume = {7},
  pages = {649 - 666}
}
</pre></td>
</tr>
<tr id="Cha1989" class="entry">
	<td>Cha, Y., Murray, C. and Klinman, J.</td>
	<td>Hydrogen tunneling in enzyme reactions. <p class="infolinks">[<a href="javascript:toggleInfo('Cha1989','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Cha1989','bibtex')">BibTeX</a>]</p></td>
	<td>1989</td>
	<td>Science (New York, N.Y.)<br/>Vol. 243(4896), pp. 1325 - 30&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Cha1989" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Primary and secondary protium-to-tritium (H/T) and deuterium-to-tritium (D/T) kinetic isotope effects for the catalytic oxidation of benzyl alcohol to benzaldehyde by yeast alcohol dehydrogenase (YADH) at 25 degrees Celsius have been determined. Previous studies showed that this reaction is nearly or fully rate limited by the hydrogen-transfer step. Semiclassical mass considerations that do not include tunneling effects would predict that kH/kT = (kD/kT)3.26, where kH, kD, and kT are the rate constants for the reaction of protium, deuterium, and tritium derivatives, respectively. Significant deviations from this relation have now been observed for both primary and especially secondary effects, such that experimental H/T ratios are much greater than those calculated from the above expression. These deviations also hold in the temperature range from 0 to 40 degrees Celsius. Such deviations were previously predicted to result from a reaction coordinate containing a significant contribution from hydrogen tunneling.</td>
</tr>
<tr id="bib_Cha1989" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cha1989,
  author = {Cha, Y and Murray, CJ and Klinman, JP},
  title = {Hydrogen tunneling in enzyme reactions.},
  journal = {Science (New York, N.Y.)},
  year = {1989},
  volume = {243},
  number = {4896},
  pages = {1325 - 30},
  note = {KIE study of YADH with benzyl alcohol as substrate}
}
</pre></td>
</tr>
<tr id="Chaikuad2005" class="entry">
	<td>Chaikuad, A., Fairweather, V., Conners, R., Joseph-Horne, T., Turgut-Balik, D. and Brady, L.R.</td>
	<td>Structure of lactate dehydrogenase from Plasmodium vivax: Complexes with NADH and APADH <p class="infolinks">[<a href="javascript:toggleInfo('Chaikuad2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Chaikuad2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Biochemistry<br/>Vol. 44(49), pp. 16221 - 16228&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi051416y">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Chaikuad2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Malaria caused by Plasmodium vivax is a major cause of global morbidity and, in rare cases, mortality. Lactate dehydrogenase is an essential Plasmodium protein and, therefore, a potential antimalarial drug target. Ideally, drugs directed against this target would be effective against both major species of Plasmodium, P. falciparum and P. vivax. In this study, the crystal structure of the lactate dehydrogenase protein from P. vivax has been solved and is compared to the equivalent structure from the P. falciparum enzyme. The active sites and cofactor binding pockets of both enzymes are found to be highly similar and differentiate these enzymes from their human counterparts. These structures suggest effective inhibition of both enzymes should be readily achievable with a common inhibitor. The crystal structures of both enzymes have also been solved in complex with the synthetic cofactor APADH. The unusual cofactor binding site in these Plasmodium enzymes is found to readily accommodate both NADH and APADH, explaining why the Plasmodium enzymes retain enzymatic activity in the presence of this synthetic cofactor.</td>
</tr>
<tr id="bib_Chaikuad2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Chaikuad2005,
  author = {Chaikuad, Apirat and Fairweather, Victoria and Conners, Rebecca and Joseph-Horne, Tim and Turgut-Balik, Dilek and Brady, Leo R},
  title = {Structure of lactate dehydrogenase from Plasmodium vivax: Complexes with NADH and APADH},
  journal = {Biochemistry},
  year = {2005},
  volume = {44},
  number = {49},
  pages = {16221 - 16228},
  note = {NULL},
  doi = {https://doi.org/10.1021/bi051416y}
}
</pre></td>
</tr>
<tr id="Chakravorty2018" class="entry">
	<td>Chakravorty, A., Jia, Z., Li, L., Zhao, S. and Alexov, E.</td>
	<td>Reproducing the Ensemble Average Polar Solvation Energy of a Protein from a Single Structure: Gaussian-Based Smooth Dielectric Function for Macromolecular Modeling <p class="infolinks">[<a href="javascript:toggleInfo('Chakravorty2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Journal of chemical theory and computation&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Chakravorty2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Chakravorty2018,
  author = {Chakravorty, Arghya and Jia, Zhe and Li, Lin and Zhao, Shan and Alexov, Emil},
  title = {Reproducing the Ensemble Average Polar Solvation Energy of a Protein from a Single Structure: Gaussian-Based Smooth Dielectric Function for Macromolecular Modeling},
  journal = {Journal of chemical theory and computation},
  publisher = {ACS Publications},
  year = {2018}
}
</pre></td>
</tr>
<tr id="Chandler1978" class="entry">
	<td>Chandler, D.</td>
	<td>Statistical mechanics of isomerization dynamics in liquids and the transition state approximation <p class="infolinks">[<a href="javascript:toggleInfo('Chandler1978','bibtex')">BibTeX</a>]</p></td>
	<td>1978</td>
	<td>J. Chem. Phys.<br/>Vol. 68(6), pp. 2959 - 2959&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.436049">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Chandler1978" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Chandler1978,
  author = {Chandler, David},
  title = {Statistical mechanics of isomerization dynamics in liquids and the transition state approximation},
  journal = {J. Chem. Phys.},
  year = {1978},
  volume = {68},
  number = {6},
  pages = {2959 - 2959},
  doi = {https://doi.org/10.1063/1.436049}
}
</pre></td>
</tr>
<tr id="Chandler1981" class="entry">
	<td>Chandler, D.</td>
	<td>Exploiting the isomorphism between quantum theory and classical statistical mechanics of polyatomic fluids <p class="infolinks">[<a href="javascript:toggleInfo('Chandler1981','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Chandler1981','bibtex')">BibTeX</a>]</p></td>
	<td>1981</td>
	<td>The Journal of Chemical Physics<br/>Vol. 74(7), pp. 4078 - 4078&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.441588">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Chandler1981" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: From a discretization of the path integral formulation of quantum mechanics, it is possible to relate equilibrium quantum many body theory to classical statistical mechanics. In this paper, we significantly extend and analyze this well known isomorphism in terms of the equilibrium theory of classical molecular fluids composed of flexible polyatomic species. We show how quantum influence functionals are isomorphic to classical cavity distribution functions. The former describe the influence of surrounding media on the dynamics of quantal degrees of freedom, and the latter describe environmental effects for classical models of flexible molecules and chemical equilibria. The connection allows the use of classical theories to perform nonperturbative calculations of influence functionals which treat the influence functionals and many body correlation functions in a self‐consistent fashion. We illustrate the computational advantages of the method by studying its predictions for a hard sphere model of liquid helium above the l transition. The nature of quantum indistinguishability of identical particles (i.e., quantum exchange) is treated in our theory in terms of an exact isomorphism with chemical equilibria. This connection allows the treatment of exchange in condensed phases in terms of the classical law of mass action, and provides a computational advance over existing methods for interacting systems. By picturing exchange in terms of classical association equilibrium, we arrive at a view (hinted at long ago by Feynman and by Penrose and Onsager) in which the Bose condensation is related to an equilibrium polymeric sol--gel transition. Thus, below the l transition, the correlations in liquid helium are equivalent to those in a classical fluid containing a finite concentration of macroscopic polymers. We stress how the path integral aspect of the isomorphism leads to useful geometrical interpretations of quantum phenomena. For example, tunneling phenomena can be viewed in terms of solitonic (or instantonic) configurations or flexible chain molecules. With this picture, we show how the isomorphism can be employed to understand both adiabatic and nonadiabatic solvent effects on chemical bonding. For concreteness, we provide a detailed analysis of a particular model of the chemical bond for which a partial summation over intermediate quantum paths leads to an Ising model problem in the isomorphism. While applications like this are presented in the form of qualitative illustrations, a variety of methods can be employed to produce quantitative results. We sketch how these calculations can be performed for various problems, making connections with methods like the renormalization group (RG) technique. The classical isomorphism together with the modern theory of classical polyatomic systems provides a powerful framework for quantitative solutions of condensed matter quantum mechanical problems.</td>
</tr>
<tr id="bib_Chandler1981" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Chandler1981,
  author = {Chandler, David},
  title = {Exploiting the isomorphism between quantum theory and classical statistical mechanics of polyatomic fluids},
  journal = {The Journal of Chemical Physics},
  year = {1981},
  volume = {74},
  number = {7},
  pages = {4078 - 4078},
  doi = {https://doi.org/10.1063/1.441588}
}
</pre></td>
</tr>
<tr id="Chandler1987" class="entry">
	<td>Chandler, D.</td>
	<td>Introduction to modern statistical mechanics <p class="infolinks">[<a href="javascript:toggleInfo('Chandler1987','bibtex')">BibTeX</a>]</p></td>
	<td>1987</td>
	<td>, pp. 274&nbsp;</td>
	<td>book</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Chandler1987" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{Chandler1987,
  author = {Chandler, David},
  title = {Introduction to modern statistical mechanics},
  publisher = {Oxford University Press},
  year = {1987},
  pages = {274}
}
</pre></td>
</tr>
<tr id="Chang1993" class="entry">
	<td>Chang, M.P., Mallet, W.G., Mostov, K.E. and Brodsky, F.M.</td>
	<td>Adaptor self-aggregation, adaptor-receptor recognition and binding of alpha-adaptin subunits to the plasma membrane contribute to recruitment of adaptor (AP2) components of clathrin-coated pits. <p class="infolinks">[<a href="javascript:toggleInfo('Chang1993','bibtex')">BibTeX</a>]</p></td>
	<td>1993</td>
	<td>The EMBO journal<br/>Vol. 12(5), pp. 2169&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Chang1993" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Chang1993,
  author = {Chang, Michael P and Mallet, William G and Mostov, Keith E and Brodsky, Frances M},
  title = {Adaptor self-aggregation, adaptor-receptor recognition and binding of alpha-adaptin subunits to the plasma membrane contribute to recruitment of adaptor (AP2) components of clathrin-coated pits.},
  journal = {The EMBO journal},
  publisher = {European Molecular Biology Organization},
  year = {1993},
  volume = {12},
  number = {5},
  pages = {2169}
}
</pre></td>
</tr>
<tr id="Chen2005" class="entry">
	<td>Chen, C.-Y. and Brodsky, F.M.</td>
	<td>Huntingtin-interacting protein 1 (Hip1) and Hip1-related protein (Hip1R) bind the conserved sequence of clathrin light chains and thereby influence clathrin assembly in vitro and actin distribution in vivo. <p class="infolinks">[<a href="javascript:toggleInfo('Chen2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Chen2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>The Journal of biological chemistry<br/>Vol. 280, pp. 6109-6117&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M408454200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Chen2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin heavy and light chains form triskelia, which assemble into polyhedral coats of membrane vesicles that mediate transport for endocytosis and organelle biogenesis. Light chain subunits regulate clathrin assembly in vitro by suppressing spontaneous self-assembly of the heavy chains. The residues that play this regulatory role are at the N terminus of a conserved 22-amino acid sequence that is shared by all vertebrate light chains. Here we show that these regulatory residues and others in the conserved sequence mediate light chain interaction with Hip1 and Hip1R. These related proteins were previously found to be enriched in clathrin-coated vesicles and to promote clathrin assembly in vitro. We demonstrate Hip1R binding preference for light chains associated with clathrin heavy chain and show that Hip1R stimulation of clathrin assembly in vitro is blocked by mutations in the conserved sequence of light chains that abolish interaction with Hip1 and Hip1R. In vivo overexpression of a fragment of clathrin light chain comprising the Hip1R-binding region affected cellular actin distribution. Together these results suggest that the roles of Hip1 and Hip1R in affecting clathrin assembly and actin distribution are mediated by their interaction with the conserved sequence of clathrin light chains.</td>
</tr>
<tr id="bib_Chen2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Chen2005,
  author = {Chen, Chih-Ying and Brodsky, Frances M},
  title = {Huntingtin-interacting protein 1 (Hip1) and Hip1-related protein (Hip1R) bind the conserved sequence of clathrin light chains and thereby influence clathrin assembly in vitro and actin distribution in vivo.},
  journal = {The Journal of biological chemistry},
  year = {2005},
  volume = {280},
  pages = {6109--6117},
  note = {domain information. CLTB - HIP1/R, CLTC},
  doi = {https://doi.org/10.1074/jbc.M408454200}
}
</pre></td>
</tr>
<tr id="Chen2002" class="entry">
	<td>Chen, C.-Y., Reese, M.L., Hwang, P.K., Ota, N., Agard, D. and Brodsky, F.M.</td>
	<td>Clathrin light and heavy chain interface: α-helix binding superhelix loops via critical tryptophans <p class="infolinks">[<a href="javascript:toggleInfo('Chen2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>The EMBO Journal<br/>Vol. 21(22), pp. 6072-6082&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/emboj/cdf594">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Chen2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Chen2002,
  author = {Chen, Chih-Ying and Reese, Michael L. and Hwang, Peter K. and Ota, Nobuyuki and Agard, David and Brodsky, Frances M.},
  title = {Clathrin light and heavy chain interface: α-helix binding superhelix loops via critical tryptophans},
  journal = {The EMBO Journal},
  publisher = {John Wiley &amp; Sons, Ltd},
  year = {2002},
  volume = {21},
  number = {22},
  pages = {6072--6082},
  note = {domain information. CLTC - CLTB. CLTC.PROX - CLTC.PROX},
  doi = {https://doi.org/10.1093/emboj/cdf594}
}
</pre></td>
</tr>
<tr id="Chen2007" class="entry">
	<td>Chen, D., Wilkinson, C.R., Watt, S., Penkett, C.J., Toone, M.W., Jones, N. and Bahler, J.</td>
	<td>High-Resolution Crystal Structure and In Vivo Function of a Kinesin-2 Homologue in Giardia intestinalis <p class="infolinks">[<a href="javascript:toggleInfo('Chen2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Chen2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Molecular biology of the cell<br/>Vol. 19(1), pp. 308 - 317&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1091/mbc.E07">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Chen2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: YY1 is a transcription factor that can repress or activate the transcription of a variety of genes. Here, we show that the function of YY1 as a repressor in cardiac myocytes is tightly dependent on its ability to interact with histone deacetylase 5 (HDAC5). YY1 interacts with HDAC5, and overexpression of YY1 prevents HDAC5 nuclear export in response to hypertrophic stimuli and the increase in cell size and re-expression of fetal genes that accompany pathological cardiac hypertrophy. Knockdown of YY1 results in up-regulation of all genes present during fetal development and increases the cell size of neonatal cardiac myocytes. Moreover, overexpression of a YY1 deletion construct that does not interact with HDAC5 results in transcription activation, suggesting that HDAC5 is necessary for YY1 function as a transcription repressor. In support of this relationship, we show that knockdown of HDAC5 results in transcription activation by YY1. Finally, we show that YY1 interaction with HDAC5 is dependent on the HDAC5 phosphorylation domain and that overexpression of YY1 reduces HDAC5 phosphorylation in response to hypertrophic stimuli. Our results strongly suggest that YY1 functions as an antihypertrophic factor by preventing HDAC5 nuclear export and that up-regulation of YY1 in human heart failure may be a protective mechanism against pathological hypertrophy</td>
</tr>
<tr id="bib_Chen2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Chen2007,
  author = {Chen, Dongrong and Wilkinson, Caroline RM and Watt, Stephen and Penkett, Christopher J and Toone, Mark W and Jones, Nic and Bahler, Jurg},
  title = {High-Resolution Crystal Structure and In Vivo Function of a Kinesin-2 Homologue in Giardia intestinalis},
  journal = {Molecular biology of the cell},
  year = {2007},
  volume = {19},
  number = {1},
  pages = {308 - 317},
  doi = {https://doi.org/10.1091/mbc.E07}
}
</pre></td>
</tr>
<tr id="Chen1998" class="entry">
	<td>Chen, H., Fre, S., Slepnev, V., Capua, M., Takei, K., Butler, M., Di Fiore, P. and De Camilli, P.</td>
	<td>Epsin is an EH-domain-binding protein implicated in clathrin-mediated endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Chen1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Chen1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Nature<br/>Vol. 394(6695), pp. 793 - 797&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/29555">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Chen1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: During endocytosis, clathrin and the clathrin adaptor protein AP-2, assisted by a variety of accessory factors, help to generate an invaginated bud at the cell membrane. One of these factors is Eps15, a clathrin-coat-associated protein that binds the alpha-adaptin subunit of AP-2. Here we investigate the function of Eps15 by characterizing an important binding partner for its region containing EH domains; this protein, epsin, is closely related to the Xenopus mitotic phosphoprotein MP90 and has a ubiquitous tissue distribution. It is concentrated together with Eps15 in presynaptic nerve terminals, which are sites specialized for the clathrin-mediated endocytosis of synaptic vesicles. The central region of epsin binds AP-2 and its carboxy-terminal region binds Eps15. Epsin is associated with clathrin coats in situ, can be co-precipitated with AP-2 and Eps15 from brain extracts, but does not co-purify with clathrin coat components in a clathrin-coated vesicle fraction. When epsin function is disrupted, clathrin-mediated endocytosis is blocked. We propose that epsin may participate, together with Eps15, in the molecular rearrangement of the clathrin coats that are required for coated-pit invagination and vesicle fission.</td>
</tr>
<tr id="bib_Chen1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Chen1998,
  author = {Chen, H and Fre, S and Slepnev, VI and Capua, MR and Takei, K and Butler, MH and Di Fiore, Pier and De Camilli, Pietro},
  title = {Epsin is an EH-domain-binding protein implicated in clathrin-mediated endocytosis.},
  journal = {Nature},
  year = {1998},
  volume = {394},
  number = {6695},
  pages = {793 - 797},
  note = {rat cells. interactions of epn1 with eps15 and alpha-adaptin. domain information.},
  doi = {https://doi.org/10.1038/29555}
}
</pre></td>
</tr>
<tr id="Cheng2007" class="entry">
	<td>Cheng, Y., Boll, W., Kirchhausen, T., Harrison, S.C. and Walz, T.</td>
	<td>Cryo-electron tomography of clathrin-coated vesicles: structural implications for coat assembly <p class="infolinks">[<a href="javascript:toggleInfo('Cheng2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Journal of molecular biology<br/>Vol. 365(3), pp. 892-899&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Cheng2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cheng2007,
  author = {Cheng, Yifan and Boll, Werner and Kirchhausen, Tomas and Harrison, Stephen C and Walz, Thomas},
  title = {Cryo-electron tomography of clathrin-coated vesicles: structural implications for coat assembly},
  journal = {Journal of molecular biology},
  publisher = {Elsevier},
  year = {2007},
  volume = {365},
  number = {3},
  pages = {892--899}
}
</pre></td>
</tr>
<tr id="Lai2017" class="entry">
	<td>Cheng-Tsung, L., Rosi, N.L. and Schatz, G.C.</td>
	<td>All-Atom Molecular Dynamics Simulations of Peptide Amphiphile Assemblies That Spontaneously Form Twisted and Helical Ribbon Structures <p class="infolinks">[<a href="javascript:toggleInfo('Lai2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Lai2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of Physical Chemistry Letters<br/>Vol. 8, pp. 2170 - 2174&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jpclett.7b00745">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Lai2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Self-assembly of peptide amphiphiles (PAs) has been an active research area as the assemblies can be programmed into various shaped nanostructures. Although cylindrical micelles are common structures, gold-binding peptide conjugates PEPAu can self-assemble into chiral nanofibers with single or double helices. When gold nanoparticles bind to the helices, the resulting chiral nanoparticle assemblies have a collective plasmonic circular dichroism signal that can serve as nanoscale circular polarizers or chiroptical sensors. A better atomic level understanding of the factors which lead to helical PA assemblies is therefore of significant importance. In this study we show that all-atom molecular dynamics simulations can describe the spontaneous structural transformation from a planar assembly of PAs to a twisted assembly or to a helical ribbon. The twist angle and the helical diameter calculated from the simulations closely match the experimental results, with the oxidation of a single Met residue in each PA l...</td>
</tr>
<tr id="bib_Lai2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Lai2017,
  author = {Cheng-Tsung, Lai and Rosi, Nathaniel L and Schatz, George C},
  title = {All-Atom Molecular Dynamics Simulations of Peptide Amphiphile Assemblies That Spontaneously Form Twisted and Helical Ribbon Structures},
  journal = {Journal of Physical Chemistry Letters},
  year = {2017},
  volume = {8},
  pages = {2170 - 2174},
  doi = {https://doi.org/10.1021/acs.jpclett.7b00745}
}
</pre></td>
</tr>
<tr id="Chi2008" class="entry">
	<td>Chi, S., Cao, H., Chen, J. and McNiven, M.A.</td>
	<td>Eps15 mediates vesicle trafficking from the trans-Golgi network via an interaction with the clathrin adaptor AP-1. <p class="infolinks">[<a href="javascript:toggleInfo('Chi2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Chi2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Molecular Biology of the Cell<br/>Vol. 19, pp. 3564-3575&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1091/mbc.E07-10-0997">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Chi2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Eps15 (EGFR pathway substrate clone 15) is well known for its role in clathrin-coated vesicle formation at the plasma membrane through interactions with other clathrin adaptor proteins such as AP-2. Interestingly, we observed that in addition to its plasma membrane localization, Eps15 is also present at the trans-Golgi network (TGN). Therefore, we predicted that Eps15 might associate with clathrin adaptor proteins at the TGN and thereby mediate the formation of Golgi-derived vesicles. Indeed, we have found that Eps15 and the TGN clathrin adaptor AP-1 coimmunoprecipitate from rat liver Golgi fractions. Furthermore, we have identified a 14-amino acid motif near the AP-2-binding domain of Eps15 that is required for binding to AP-1, but not AP-2. Disruption of the Eps15-AP-1 interaction via siRNA knockdown of AP-1 or expression of mutant Eps15 protein, which lacks a 14-amino acid motif representing the AP-1 binding site of Eps15, significantly reduced the exit of secretory proteins from the TGN. Together, these findings indicate that Eps15 plays an important role in clathrin-coated vesicle formation not only at the plasma membrane but also at the TGN during the secretory process.</td>
</tr>
<tr id="bib_Chi2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Chi2008,
  author = {Chi, Susan and Cao, Hong and Chen, Jing and McNiven, Mark A},
  title = {Eps15 mediates vesicle trafficking from the trans-Golgi network via an interaction with the clathrin adaptor AP-1.},
  journal = {Molecular Biology of the Cell},
  year = {2008},
  volume = {19},
  pages = {3564--3575},
  note = {citations to domain information for EPS15. human. EPS15 - CLTC through colocalization at a Golgi.},
  doi = {https://doi.org/10.1091/mbc.E07-10-0997}
}
</pre></td>
</tr>
<tr id="Chung2002" class="entry">
	<td>Chung, F. and Lu, L.</td>
	<td>The average distances in random graphs with given expected degrees <p class="infolinks">[<a href="javascript:toggleInfo('Chung2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Chung2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Proceedings of the National Academy of Sciences of the United States of America<br/>Vol. 99(25), pp. 15879 - 15882&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.252631999">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Chung2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Random graph theory is used to examine the "small-world phenomenon"; any two strangers are connected through a short chain of mutual acquaintances. We will show that for certain families of random graphs with given expected degrees the average distance is almost surely of order log nlog d, where d is the weighted average of the sum of squares of the expected degrees. Of particular interest are power law random graphs in which the number of vertices of degree k is proportional to 1kbeta for some fixed exponent beta. For the case of beta \textgreater\ 3, we prove that the average distance of the power law graphs is almost surely of order log nlog d. However, many Internet, social, and citation networks are power law graphs with exponents in the range 2 \textless\ beta \textless\ 3 for which the power law random graphs have average distance almost surely of order log log n, but have diameter of order log n (provided having some mild constraints for the average distance and maximum degree). In particular, these graphs contain a dense subgraph, which we call the core, having n(clog log n) vertices. Almost all vertices are within distance log log n of the core although there are vertices at distance log n from the core.</td>
</tr>
<tr id="bib_Chung2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Chung2002,
  author = {Chung, Fan and Lu, Linyuan},
  title = {The average distances in random graphs with given expected degrees},
  journal = {Proceedings of the National Academy of Sciences of the United States of America},
  year = {2002},
  volume = {99},
  number = {25},
  pages = {15879 - 15882},
  doi = {https://doi.org/10.1073/pnas.252631999}
}
</pre></td>
</tr>
<tr id="Chylek2014" class="entry">
	<td>Chylek, L.A., Harris, L.A., Tung, C., Faeder, J.R., Lopez, C.F. and Hlavacek, W.S.</td>
	<td>Rule-based modeling: A computational approach for studying biomolecular site dynamics in cell signaling systems <p class="infolinks">[<a href="javascript:toggleInfo('Chylek2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Chylek2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Wiley Interdisciplinary Reviews: Systems Biology and Medicine<br/>Vol. 6(1), pp. 13 - 36&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/wsbm.1245">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Chylek2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Rule-based modeling was developed to address the limitations of traditional approaches for modeling chemical kinetics in cell signaling systems. These systems consist of multiple interacting biomolecules (e.g., proteins), which themselves consist of multiple parts (e.g., domains, linear motifs, and sites of phosphorylation). Consequently, biomolecules that mediate information processing generally have the potential to interact in multiple ways, with the number of possible complexes and posttranslational modification states tending to grow exponentially with the number of binary interactions considered. As a result, only large reaction networks capture all possible consequences of the molecular interactions that occur in a cell signaling system, which is problematic because traditional modeling approaches for chemical kinetics (e.g., ordinary differential equations) require explicit network specification. This problem is circumvented through representation of interactions in terms of local rules. With this approach, network specification is implicit and model specification is concise. Concise representation results in a coarse graining of chemical kinetics, which is introduced because all reactions implied by a rule inherit the rate law associated with that rule. Coarse graining can be appropriate if interactions are modular, and the coarseness of a model can be adjusted as needed. Rules can be specified using specialized model-specification languages, and recently developed tools designed for specification of rule-based models allow one to leverage powerful software engineering capabilities. A rule-based model comprises a set of rules, which can be processed by general-purpose simulation and analysis tools to achieve different objectives (e.g., to perform either a deterministic or stochastic simulation).</td>
</tr>
<tr id="bib_Chylek2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Chylek2014,
  author = {Chylek, Lily A and Harris, Leonard A and Tung, Chang and Faeder, James R and Lopez, Carlos F and Hlavacek, William S},
  title = {Rule-based modeling: A computational approach for studying biomolecular site dynamics in cell signaling systems},
  journal = {Wiley Interdisciplinary Reviews: Systems Biology and Medicine},
  year = {2014},
  volume = {6},
  number = {1},
  pages = {13 - 36},
  doi = {https://doi.org/10.1002/wsbm.1245}
}
</pre></td>
</tr>
<tr id="Chylek2011" class="entry">
	<td>Chylek, L., Hu, B., Blinov, M.L., Emonet, T., Faeder, J.R., Goldstein, B., Gutenkunst, R.N., Haugh, J.M., Lipniacki, T., Posner, R.G., Yang, J. and Hlavacek, W.S.</td>
	<td>Guidelines for visualizing and annotating rule-based models <p class="infolinks">[<a href="javascript:toggleInfo('Chylek2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Chylek2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Molecular bioSystems<br/>Vol. 7(10), pp. 2779 - 2795&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1039/c1mb05077j">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Chylek2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Rule-based modeling provides a means to represent cell signaling systems in a way that captures site-specific details of molecular interactions. For rule-based models to be more widely understood and (re)used, conventions for model visualization and annotation are needed. We have developed the concepts of an extended contact map and a model guide for illustrating and annotating rule-based models. An extended contact map represents the scope of a model by providing an illustration of each molecule, molecular component, direct physical interaction, post-translational modification, and enzyme-substrate relationship considered in a model. A map can also illustrate allosteric effects, structural relationships among molecular components, and compartmental locations of molecules. A model guide associates elements of a contact map with annotation and elements of an underlying model, which may be fully or partially specified. A guide can also serve to document the biological knowledge upon which a model is based. We provide examples of a map and guide for a published rule-based model that characterizes early events in IgE receptor (FcεRI) signaling. We also provide examples of how to visualize a variety of processes that are common in cell signaling systems but not considered in the example model, such as ubiquitination. An extended contact map and an associated guide can document knowledge of a cell signaling system in a form that is visual as well as executable. As a tool for model annotation, a map and guide can communicate the content of a model clearly and with precision, even for large models.</td>
</tr>
<tr id="bib_Chylek2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Chylek2011,
  author = {Chylek, Lily and Hu, Bin and Blinov, Michael L and Emonet, Thierry and Faeder, James R and Goldstein, Byron and Gutenkunst, Ryan N and Haugh, Jason M and Lipniacki, Tomasz and Posner, Richard G and Yang, Jin and Hlavacek, William S},
  title = {Guidelines for visualizing and annotating rule-based models},
  journal = {Molecular bioSystems},
  year = {2011},
  volume = {7},
  number = {10},
  pages = {2779 - 2795},
  doi = {https://doi.org/10.1039/c1mb05077j}
}
</pre></td>
</tr>
<tr id="Cichonska2017" class="entry">
	<td>Cichonska, A., Ravikumar, B., Parri, E., Timonen, S., Pahikkala, T., Airola, A., Wennerberg, K., Rousu, J. and Aittokallio, T.</td>
	<td>Computational-experimental approach to drug-target interaction mapping: A case study on kinase inhibitors <p class="infolinks">[<a href="javascript:toggleInfo('Cichonska2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Cichonska2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>PLOS Computational Biology<br/>Vol. 13(8), pp. e1005678 - e1005678&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pcbi.1005678">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Cichonska2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Due to relatively high costs and labor required for experimental profiling of the full target space of chemical compounds, various machine learning models have been proposed as cost-effective means to advance this process in terms of predicting the most potent compound-target interactions for subsequent verification. However, most of the model predictions lack direct experimental validation in the laboratory, making their practical benefits for drug discovery or repurposing applications largely unknown. Here, we therefore introduce and carefully test a systematic computational-experimental framework for the prediction and pre-clinical verification of drug-target interactions using a well-established kernel-based regression algorithm as the prediction model. To evaluate its performance, we first predicted unmeasured binding affinities in a large-scale kinase inhibitor profiling study, and then experimentally tested 100 compound-kinase pairs. The relatively high correlation of 0.77 (p \textless\ 0.0001) between the predicted and measured bioactivities supports the potential of the model for filling the experimental gaps in existing compound-target interaction maps. Further, we subjected the model to a more challenging task of predicting target interactions for such a new candidate drug compound that lacks prior binding profile information. As a specific case study, we used tivozanib, an investigational VEGF receptor inhibitor with currently unknown off-target profile. Among 7 kinases with high predicted affinity, we experimentally validated 4 new off-targets of tivozanib, namely the Src-family kinases FRK and FYN A, the non-receptor tyrosine kinase ABL1, and the serine/threonine kinase SLK. Our sub-sequent experimental validation protocol effectively avoids any possible information leakage between the training and validation data, and therefore enables rigorous model validation for practical applications. These results demonstrate that the kernel-based modeling approach offers practical benefits for probing novel insights into the mode of action of investigational compounds, and for the identification of new target selectivities for drug repurposing applications.</td>
</tr>
<tr id="bib_Cichonska2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cichonska2017,
  author = {Cichonska, Anna and Ravikumar, Balaguru and Parri, Elina and Timonen, Sanna and Pahikkala, Tapio and Airola, Antti and Wennerberg, Krister and Rousu, Juho and Aittokallio, Tero},
  title = {Computational-experimental approach to drug-target interaction mapping: A case study on kinase inhibitors},
  journal = {PLOS Computational Biology},
  year = {2017},
  volume = {13},
  number = {8},
  pages = {e1005678 - e1005678},
  doi = {https://doi.org/10.1371/journal.pcbi.1005678}
}
</pre></td>
</tr>
<tr id="Cleland1963a" class="entry">
	<td>Cleland, W.</td>
	<td>The kinetics of enzyme catalyzed reactions with two or more substrates or products: I: Nomenclature and rate equations <p class="infolinks">[<a href="javascript:toggleInfo('Cleland1963a','bibtex')">BibTeX</a>]</p></td>
	<td>1963</td>
	<td>Biochim. Biophys. Acta<br/>Vol. 67, pp. 173 - 187&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Cleland1963a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cleland1963a,
  author = {Cleland, WW},
  title = {The kinetics of enzyme catalyzed reactions with two or more substrates or products: I: Nomenclature and rate equations},
  journal = {Biochim. Biophys. Acta},
  year = {1963},
  volume = {67},
  pages = {173 - 187}
}
</pre></td>
</tr>
<tr id="Cleland1963b" class="entry">
	<td>Cleland, W.</td>
	<td>The kinetics of enzyme catalyzed reactions with two or more substrates or products: II. Inhibition: Nomenclature and theory <p class="infolinks">[<a href="javascript:toggleInfo('Cleland1963b','bibtex')">BibTeX</a>]</p></td>
	<td>1963</td>
	<td>Biochim. Biophys. Acta<br/>Vol. 67, pp. 173 - 187&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Cleland1963b" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cleland1963b,
  author = {Cleland, WW},
  title = {The kinetics of enzyme catalyzed reactions with two or more substrates or products: II. Inhibition: Nomenclature and theory},
  journal = {Biochim. Biophys. Acta},
  year = {1963},
  volume = {67},
  pages = {173 - 187}
}
</pre></td>
</tr>
<tr id="Cleland1963c" class="entry">
	<td>Cleland, W.</td>
	<td>The kinetics of enzyme-catalyzed reactions with two or more substrates or products. III. Prediction of initial velocity and inhibition patterns by inspection. <p class="infolinks">[<a href="javascript:toggleInfo('Cleland1963c','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Cleland1963c','bibtex')">BibTeX</a>]</p></td>
	<td>1963</td>
	<td>Biochim. Biophys. Acta<br/>Vol. 67, pp. 188 - 196&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/0926-6569(63)90227-X">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Cleland1963c" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A nomenclature is proposed to describe different types of inhibitions of enzyme-catalyzed reactions, particularly for reactions with more than one substrate and product. The rate equations for such inhibitions are discussed, as are methods for distinguishing between the various types of inhibition and obtaining inhibition and kinetic constants from experimental data. Several examples are given of the type of information that can be obtained from inhibition studies.</td>
</tr>
<tr id="bib_Cleland1963c" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cleland1963c,
  author = {Cleland, WW},
  title = {The kinetics of enzyme-catalyzed reactions with two or more substrates or products. III. Prediction of initial velocity and inhibition patterns by inspection.},
  journal = {Biochim. Biophys. Acta},
  year = {1963},
  volume = {67},
  pages = {188 - 196},
  doi = {https://doi.org/10.1016/0926-6569(63)90227-X}
}
</pre></td>
</tr>
<tr id="Cleland1995" class="entry">
	<td>Cleland, W.</td>
	<td>Isotope effects: Determination of enzyme transition state structure <p class="infolinks">[<a href="javascript:toggleInfo('Cleland1995','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Cleland1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>Methods in Enzymology<br/>Vol. 249(1982), pp. 341 - 373&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/0076-6879(95)49041-8">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Cleland1995" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A review, with 46 refs., on some basic definitions and the more modern methods of measuring isotope effects and detg. intrinsic isotope effects. Then examples were given of the detn. of transition state structure and what these studies tell about enzymic mechanisms. The appendix presented the computer programs for ISOCALC4 and ISOCALC5. [on SciFinder(R)]</td>
</tr>
<tr id="bib_Cleland1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cleland1995,
  author = {Cleland, WW},
  title = {Isotope effects: Determination of enzyme transition state structure},
  journal = {Methods in Enzymology},
  year = {1995},
  volume = {249},
  number = {1982},
  pages = {341 - 373},
  doi = {https://doi.org/10.1016/0076-6879(95)49041-8}
}
</pre></td>
</tr>
<tr id="Cleland2013" class="entry">
	<td>Cleland, W.</td>
	<td>Reflections: The Use of Isotope Effects to Determine Enzyme Mechanisms W. Wallace Cleland <p class="infolinks">[<a href="javascript:toggleInfo('Cleland2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Journal of Biological Chemistry<br/>Vol. 278(52), pp. 51975 - 51984&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.X300005200">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Cleland2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cleland2013,
  author = {Cleland, WW},
  title = {Reflections: The Use of Isotope Effects to Determine Enzyme Mechanisms W. Wallace Cleland},
  journal = {Journal of Biological Chemistry},
  year = {2013},
  volume = {278},
  number = {52},
  pages = {51975 - 51984},
  doi = {https://doi.org/10.1074/jbc.X300005200}
}
</pre></td>
</tr>
<tr id="Cocucci2012" class="entry">
	<td>Cocucci, E., Aguet, F., Boulant, S. and Kirchhausen, T.</td>
	<td>The first five seconds in the life of a clathrin-coated pit <p class="infolinks">[<a href="javascript:toggleInfo('Cocucci2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Cocucci2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Cell&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Cocucci2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Coated pits assemble by growth of a clathrin lattice, which is linked by adaptors to the underlying membrane. How does this process start? We used live-cell TIRF imaging with single-molecule EGFP sensitivity and high temporal resolution to detect arrival of the clathrin triskelions and AP2 adaptors that initiate coat assembly. Unbiased object identification and trajectory tracking, together with a statistical model, yield the arrival times and numbers of  ...</td>
</tr>
<tr id="bib_Cocucci2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cocucci2012,
  author = {Cocucci, E and Aguet, F and Boulant, S and Kirchhausen, T},
  title = {The first five seconds in the life of a clathrin-coated pit},
  journal = {Cell},
  year = {2012}
}
</pre></td>
</tr>
<tr id="Cohen2003" class="entry">
	<td>Cohen, R. and Havlin, S.</td>
	<td>Scale-free networks are ultrasmall. <p class="infolinks">[<a href="javascript:toggleInfo('Cohen2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Cohen2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Physical Review Letters<br/>Vol. 90(5), pp. 058701 - 058701&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1103/PhysRevLett.90.058701">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Cohen2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We study the diameter, or the mean distance between sites, in a scale-free network, having N sites and degree distribution p(k) proportional, variant k(-lambda), i.e., the probability of having k links outgoing from a site. In contrast to the diameter of regular random networks or small-world networks, which is known to be d approximately ln(N, we show, using analytical arguments, that scale-free networks with 2\textless\lambda\textless\3 have a much smaller diameter, behaving as d approximately ln(ln(N. For lambda=3, our analysis yields d approximately ln(N/ln(ln(N, as obtained by Bollobas and Riordan, while for lambda\textgreater\3, d approximately ln(N. We also show that, for any lambda\textgreater\2, one can construct a deterministic scale-free network with d approximately ln(ln(N, which is the lowest possible diameter.</td>
</tr>
<tr id="bib_Cohen2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cohen2003,
  author = {Cohen, Reuven and Havlin, Shlomo},
  title = {Scale-free networks are ultrasmall.},
  journal = {Physical Review Letters},
  year = {2003},
  volume = {90},
  number = {5},
  pages = {058701 - 058701},
  doi = {https://doi.org/10.1103/PhysRevLett.90.058701}
}
</pre></td>
</tr>
<tr id="Collette2009" class="entry">
	<td>Collette, J.R., Chi, R.J., Boettner, D.R., Fernandez-Golbano, I.M., Plemel, R., Merz, A.J., Geli, M.I., Traub, L.M. and Lemmon, S.K.</td>
	<td>Clathrin functions in the absence of the terminal domain binding site for adaptor-associated clathrin-box motifs. <p class="infolinks">[<a href="javascript:toggleInfo('Collette2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Collette2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Molecular biology of the cell<br/>Vol. 20, pp. 3401-3413&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1091/mbc.E08-10-1082">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Collette2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin is involved in vesicle formation in the trans-Golgi network (TGN)/endosomal system and during endocytosis. Clathrin recruitment to membranes is mediated by the clathrin heavy chain (HC) N-terminal domain (TD), which forms a seven-bladed beta-propeller. TD binds membrane-associated adaptors, which have short peptide motifs, either the clathrin-box (CBM) and/or the W-box; however, the importance of the TD binding sites for these motifs has not been tested in vivo. We investigated the importance of the TD in clathrin function by generating 1) mutations in the yeast HC gene (CHC1) to disrupt the binding sites for the CBM and W-box (chc1-box), and 2) four TD-specific temperature-sensitive alleles of CHC1. We found that TD is important for the retention of resident TGN enzymes and endocytosis of alpha-factor; however, the known adaptor binding sites are not necessary, because chc1-box caused little to no effect on trafficking pathways involving clathrin. The Chc1-box TD was able to interact with the endocytic adaptor Ent2 in a CBM-dependent manner, and HCs encoded by chc1-box formed clathrin-coated vesicles. These data suggest that additional or alternative binding sites exist on the TD propeller to help facilitate the recruitment of clathrin to sites of vesicle formation.</td>
</tr>
<tr id="bib_Collette2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Collette2009,
  author = {Collette, John R and Chi, Richard J and Boettner, Douglas R and Fernandez-Golbano, Isabel M and Plemel, Rachael and Merz, Alex J and Geli, Maria Isabel and Traub, Linton M and Lemmon, Sandra K},
  title = {Clathrin functions in the absence of the terminal domain binding site for adaptor-associated clathrin-box motifs.},
  journal = {Molecular biology of the cell},
  year = {2009},
  volume = {20},
  pages = {3401--3413},
  doi = {https://doi.org/10.1091/mbc.E08-10-1082}
}
</pre></td>
</tr>
<tr id="Collins2002" class="entry">
	<td>Collins, B.M., McCoy, A., Kent, H.M., Evans, P.R. and Owen, D.J.</td>
	<td>Molecular architecture and functional model of the endocytic AP2 complex <p class="infolinks">[<a href="javascript:toggleInfo('Collins2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Collins2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Cell<br/>Vol. 109(4), pp. 523 - 535&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S0092-8674(02)00735-3">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Collins2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: AP2 is the best-characterized member of the family of heterotetrameric clathrin adaptor complexes that play pivotal roles in many vesicle trafficking pathways within the cell. AP2 functions in clathrin-mediated endocytosis, the process whereby cargo enters the endosomal system from the plasma membrane. We describe the structure of the 200 kDa AP2 "core" (α trunk, β2 trunk, μ2, and σ2) complexed with the polyphosphatidylinositol headgroup mimic inositolhexakisphosphate at 2.6 &Aring; resolution. Two potential polyphosphatidylinositide binding sites are observed, one on α and one on μ2. The binding site for Yxxφ endocytic motifs is buried, indicating that a conformational change, probably triggered by phosphorylation in the disordered μ2 linker, is necessary to allow Yxxφ motif binding. A model for AP2 recruitment and activation is proposed.</td>
</tr>
<tr id="bib_Collins2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Collins2002,
  author = {Collins, Brett M and McCoy, Airlie and Kent, Helen M and Evans, Philip R and Owen, David J},
  title = {Molecular architecture and functional model of the endocytic AP2 complex},
  journal = {Cell},
  year = {2002},
  volume = {109},
  number = {4},
  pages = {523 - 535},
  note = {AP2A2, AP2M1 - PIP2. phosphorylation information. domain information. all AP2 subunits interact with each other, and if one is deleted, the others become insoluble due to large hydrophobic patches.},
  doi = {https://doi.org/10.1016/S0092-8674(02)00735-3}
}
</pre></td>
</tr>
<tr id="Confalonieri2000" class="entry">
	<td>Confalonieri, S., Salcini, A., Puri, C., Tacchetti, C. and Fiore, P.</td>
	<td>Tyrosine phosphorylation of Eps15 is required for ligand-regulated, but not constitutive, endocytosis <p class="infolinks">[<a href="javascript:toggleInfo('Confalonieri2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Confalonieri2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Journal of Cell Biology<br/>Vol. 150(4), pp. 905 - 911&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1083/jcb.150.4.905">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Confalonieri2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Membrane receptors are internalized either constitutively or upon ligand engagement. Whereas there is evidence for differential regulation of the two processes, little is known about the molecular machinery involved. Previous studies have shown that an unidentified kinase substrate is required for endocytosis of the epidermal growth factor receptor (EGFR), the prototypical ligand-inducible receptor, but not of the transferrin receptor (TfR), the prototypical constitutively internalized receptor. Eps15, an endocytic protein that is tyrosine phosphorylated by EGFR, is a candidate for such a function. Here, we show that tyrosine phosphorylation of Eps15 is necessary for internalization of the EGFR, but not of the TfR. We mapped Tyr 850 as the major in vivo tyrosine phosphorylation site of Eps15. A phosphorylation-negative mutant of Eps15 acted as a dominant negative on the internalization of the EGFR, but not of the TfR. A phosphopeptide, corresponding to the phosphorylated sequence of Eps15, inhibited EGFR endocytosis, suggesting that phosphotyrosine in Eps15 serves as a docking site for a phosphotyrosine binding protein. Thus, tyrosine phosphorylation of Eps15 represents the first molecular determinant, other than those contained in the receptors themselves, which is involved in the differential regulation of constitutive vs. regulated endocytosis.</td>
</tr>
<tr id="bib_Confalonieri2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Confalonieri2000,
  author = {Confalonieri, Stefano and Salcini, Anna and Puri, Claudia and Tacchetti, Carlo and Fiore, Pier},
  title = {Tyrosine phosphorylation of Eps15 is required for ligand-regulated, but not constitutive, endocytosis},
  journal = {Journal of Cell Biology},
  year = {2000},
  volume = {150},
  number = {4},
  pages = {905 - 911},
  note = {Mouse EPS15},
  doi = {https://doi.org/10.1083/jcb.150.4.905}
}
</pre></td>
</tr>
<tr id="Conner2002" class="entry">
	<td>Conner, S.D. and Schmid, S.L.</td>
	<td>Identification of an adaptor-associated kinase, AAK1, as a regulator of clathrin-mediated endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Conner2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Conner2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>The Journal of cell biology<br/>Vol. 156, pp. 921-929&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1083/jcb.200108123">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Conner2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The mu 2 subunit of the AP2 complex is known to be phosphorylated in vitro by a copurifying kinase, and it has been demonstrated recently that mu 2 phosphorylation is required for transferrin endocytosis (Olusanya, O., P.D. Andrews, J.R. Swedlow, and E. Smythe. 2001. Curr. Biol. 11:896-900). However, the identity of the endogenous kinase responsible for this phosphorylation is unknown. Here we identify and characterize a novel member of the Prk/Ark family of serine/threonine kinases, adaptor-associated kinase (AAK)1. We find that AAK1 copurifies with adaptor protein (AP)2 and that it directly binds the ear domain of alpha-adaptin in vivo and in vitro. In neuronal cells, AAK1 is enriched at presynaptic terminals, whereas in nonneuronal cells it colocalizes with clathrin and AP2 in clathrin-coated pits and at the leading edge of migrating cells. AAK1 specifically phosphorylates the mu subunit in vitro, and stage-specific assays for endocytosis show that mu phosphorylation by AAK1 results in a decrease in AP2-stimulated transferrin internalization. Together, these results provide strong evidence that AAK1 is the endogenous mu 2 kinase and plays a regulatory role in clathrin-mediated endocytosis. These results also lend support to the idea that clathrin-mediated endocytosis is controlled by cycles of phosphorylation/desphosphorylation.</td>
</tr>
<tr id="bib_Conner2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Conner2002,
  author = {Conner, Sean D and Schmid, Sandra L},
  title = {Identification of an adaptor-associated kinase, AAK1, as a regulator of clathrin-mediated endocytosis.},
  journal = {The Journal of cell biology},
  year = {2002},
  volume = {156},
  pages = {921--929},
  doi = {https://doi.org/10.1083/jcb.200108123}
}
</pre></td>
</tr>
<tr id="Conner2003" class="entry">
	<td>Conner, S.D. and Schmid, S.L.</td>
	<td>Regulated portals of entry into the cell. <p class="infolinks">[<a href="javascript:toggleInfo('Conner2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Conner2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Nature<br/>Vol. 422, pp. 37-44&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nature01451">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Conner2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The plasma membrane is the interface between cells and their harsh environment. Uptake of nutrients and all communication among cells and between cells and their environment occurs through this interface. 'Endocytosis' encompasses several diverse mechanisms by which cells internalize macromolecules and particles into transport vesicles derived from the plasma membrane. It controls entry into the cell and has a crucial role in development, the immune response, neurotransmission, intercellular communication, signal transduction, and cellular and organismal homeostasis. As the complexity of molecular interactions governing endocytosis are revealed, it has become increasingly clear that it is tightly coordinated and coupled with overall cell physiology and thus, must be viewed in a broader context than simple vesicular trafficking.</td>
</tr>
<tr id="bib_Conner2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Conner2003,
  author = {Conner, Sean D and Schmid, Sandra L},
  title = {Regulated portals of entry into the cell.},
  journal = {Nature},
  year = {2003},
  volume = {422},
  pages = {37--44},
  doi = {https://doi.org/10.1038/nature01451}
}
</pre></td>
</tr>
<tr id="Conners2005" class="entry">
	<td>Conners, R., Schambach, F., Read, J., Cameron, A., Sessions, R.B., Vivas, L., Easton, A., Croft, S.L. and Brady, L.R.</td>
	<td>Mapping the binding site for gossypol-like inhibitors of Plasmodium falciparum lactate dehydrogenase <p class="infolinks">[<a href="javascript:toggleInfo('Conners2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Conners2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Molecular and Biochemical Parasitology<br/>Vol. 142(2), pp. 137 - 148&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.molbiopara.2005.03.015">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Conners2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Gossypol is a di-sesquiterpene natural-product in the form of a functionalised binaphthyl and is isolated from cotton plants. The compound has long been known to exhibit anti-malarial and other biological activities. Previous studies have indicated that compounds of this type target Plasmodium falciparum lactate dehydrogenase (pfLDH), an essential enzyme for energy generation within the parasite. In this study, we report that simple naphthalene-based compounds, the core of the gossypol structure, exhibit weak inhibition of the parasite lactate dehydrogenase. Crystal structures of the complexes formed by binding of these naphthalene-based compounds to their target enzyme have been used to delineate the molecular features likely to form the gossypol binding site. Two modes of binding are observed: one overlapping the pyruvate but not the co-factor site, the other bridging the binding sites for the co-factor nicontinamide group and pyruvate substrate. This latter site encompasses molecular features unique to Plasmodium forms of LDH and is likely to represent the mode of binding for gossypol derivatives that show selectivity for the parasite enzymes. We also report a substrate analogue that unexpectedly binds within the adenine pocket of the co-factor groove. Although these core pharmacophore-like molecules only exhibit low levels of inhibitory activity, these molecular snapshots provide a rational basis for renewed structure-based development of naphthalene-based compounds as anti-malarial agents. ?? 2005 Elsevier B.V. All rights reserved.</td>
</tr>
<tr id="bib_Conners2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Conners2005,
  author = {Conners, Rebecca and Schambach, Felix and Read, Jon and Cameron, Angus and Sessions, Richard B and Vivas, Livia and Easton, Anna and Croft, Simon L and Brady, Leo R},
  title = {Mapping the binding site for gossypol-like inhibitors of Plasmodium falciparum lactate dehydrogenase},
  journal = {Molecular and Biochemical Parasitology},
  year = {2005},
  volume = {142},
  number = {2},
  pages = {137 - 148},
  doi = {https://doi.org/10.1016/j.molbiopara.2005.03.015}
}
</pre></td>
</tr>
<tr id="Cook1980" class="entry">
	<td>Cook, P.F., Blanchard, J.S. and Cleland, W.</td>
	<td>Primary and secondary deuterium isotope effects on equilibrium constants for enzyme-catalyzed reactions <p class="infolinks">[<a href="javascript:toggleInfo('Cook1980','bibtex')">BibTeX</a>]</p></td>
	<td>1980</td>
	<td>Biochemistry<br/>Vol. 19(21), pp. 4853 - 4858&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi00562a023">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Cook1980" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cook1980,
  author = {Cook, Paul F and Blanchard, John S and Cleland, WW},
  title = {Primary and secondary deuterium isotope effects on equilibrium constants for enzyme-catalyzed reactions},
  journal = {Biochemistry},
  year = {1980},
  volume = {19},
  number = {21},
  pages = {4853 - 4858},
  note = {Kinetic complexity masks KIEs},
  doi = {https://doi.org/10.1021/bi00562a023}
}
</pre></td>
</tr>
<tr id="Cook1981" class="entry">
	<td>Cook, P.F. and Cleland, W.</td>
	<td>pH Variation of isotope effects in enzyme-catalyzed reactions. 2. Isotope-dependent step not pH dependent. Kinetic mechanism of alcohol dehydrogenase <p class="infolinks">[<a href="javascript:toggleInfo('Cook1981','bibtex')">BibTeX</a>]</p></td>
	<td>1981</td>
	<td>Biochemistry<br/>Vol. 20(7), pp. 1805 - 1816&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi00510a015">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Cook1981" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cook1981,
  author = {Cook, Paul F and Cleland, WW},
  title = {pH Variation of isotope effects in enzyme-catalyzed reactions. 2. Isotope-dependent step not pH dependent. Kinetic mechanism of alcohol dehydrogenase},
  journal = {Biochemistry},
  year = {1981},
  volume = {20},
  number = {7},
  pages = {1805 - 1816},
  note = {Kinetic complexity masks KIEs},
  doi = {https://doi.org/10.1021/bi00510a015}
}
</pre></td>
</tr>
<tr id="Cook2007" class="entry">
	<td>Cook, P.F. and Cleland, W.</td>
	<td>Enzyme Kinetics and Mechanism <p class="infolinks">[<a href="javascript:toggleInfo('Cook2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>&nbsp;</td>
	<td>book</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Cook2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{Cook2007,
  author = {Cook, Paul F and Cleland, WW},
  title = {Enzyme Kinetics and Mechanism},
  publisher = {Taylor &amp; Francis},
  year = {2007}
}
</pre></td>
</tr>
<tr id="Cook2015" class="entry">
	<td>Cook, W.J., Senkovich, O., Hernandez, A., Speed, H. and Chattopadhyay, D.</td>
	<td>Biochemical and structural characterization of Cryptosporidium parvum lactate dehydrogenase <p class="infolinks">[<a href="javascript:toggleInfo('Cook2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Cook2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>International Journal of Biological Macromolecules<br/>Vol. 74, pp. 608 - 619&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.ijbiomac.2014.12.019">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Cook2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The protozoan parasite Cryptosporidium parvum causes waterborne diseases worldwide. There is no effective therapy for C. parvum infection. The parasite depends mainly on glycolysis for energy production. Lactate dehydrogenase is a major regulator of glycolysis. This paper describes the biochemical characterization of C. parvum lactate dehydrogenase and high resolution crystal structures of the apo-enzyme and four ternary complexes. The ternary complexes capture the enzyme bound to NAD/NADH or its 3-acetylpyridine analog in the cofactor binding pocket, while the substrate binding site is occupied by one of the following ligands: lactate, pyruvate or oxamate. The results reveal distinctive features of the parasitic enzyme. For example, C. parvum lactate dehydrogenase prefers the acetylpyridine analog of NADH as a cofactor. Moreover, it is slightly less sensitive to gossypol inhibition compared with mammalian lactate dehydrogenases and not inhibited by excess pyruvate. The active site loop and the antigenic loop in C. parvum lactate dehydrogenase are considerably different from those in the human counterpart. Structural features and enzymatic properties of C. parvum lactate dehydrogenase are similar to enzymes from related parasites. Structural comparison with malate dehydrogenase supports a common ancestry for the two genes.</td>
</tr>
<tr id="bib_Cook2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cook2015,
  author = {Cook, William J and Senkovich, Olga and Hernandez, Agustin and Speed, Haley and Chattopadhyay, Debasish},
  title = {Biochemical and structural characterization of Cryptosporidium parvum lactate dehydrogenase},
  journal = {International Journal of Biological Macromolecules},
  year = {2015},
  volume = {74},
  pages = {608 - 619},
  note = {NULL},
  doi = {https://doi.org/10.1016/j.ijbiomac.2014.12.019}
}
</pre></td>
</tr>
<tr id="Copie1988" class="entry">
	<td>Copie, V., Faraci, S.W., Walsh, C.T. and Griffin, R.G.</td>
	<td>Inhibition of alanine racemase by alanine phosphonate: detection of an imine linkage to pyridoxal 5'-phosphate in the enzyme-inhibitor complex by solid-state nitrogen-15 nuclear magnetic resonance <p class="infolinks">[<a href="javascript:toggleInfo('Copie1988','bibtex')">BibTeX</a>]</p></td>
	<td>1988</td>
	<td>Biochemistry<br/>Vol. 27(14), pp. 4966 - 4970&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi00414a002">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Copie1988" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Copie1988,
  author = {Copie, Valerie and Faraci, Stephen W and Walsh, Christopher T and Griffin, Robert G},
  title = {Inhibition of alanine racemase by alanine phosphonate: detection of an imine linkage to pyridoxal 5'-phosphate in the enzyme-inhibitor complex by solid-state nitrogen-15 nuclear magnetic resonance},
  journal = {Biochemistry},
  year = {1988},
  volume = {27},
  number = {14},
  pages = {4966 - 4970},
  doi = {https://doi.org/10.1021/bi00414a002}
}
</pre></td>
</tr>
<tr id="Craig2004" class="entry">
	<td>Craig, I.R. and Manolopoulos, D.E.</td>
	<td>Quantum statistics and classical mechanics: Real time correlation functions from ring polymer molecular dynamics <p class="infolinks">[<a href="javascript:toggleInfo('Craig2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Craig2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of Chemical Physics<br/>Vol. 121(8), pp. 3368 - 3373&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.1777575">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Craig2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We propose an approximate method for calculating Kubo-transformed real-time correlation functions involving position-dependent operators, based on path integral (Parrinello-Rahman) molecular dynamics. The method gives the exact quantum mechanical correlation function at time zero, exactly satisfies the quantum mechanical detailed balance condition, and for correlation functions of the form C(Ax)(t) and C(xB)(t) it gives the exact result for a harmonic potential. It also works reasonably well at short times for more general potentials and correlation functions, as we illustrate with some example calculations. The method provides a consistent improvement over purely classical molecular dynamics that is most apparent in the low-temperature regime.</td>
</tr>
<tr id="bib_Craig2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Craig2004,
  author = {Craig, Ian R and Manolopoulos, David E},
  title = {Quantum statistics and classical mechanics: Real time correlation functions from ring polymer molecular dynamics},
  journal = {Journal of Chemical Physics},
  year = {2004},
  volume = {121},
  number = {8},
  pages = {3368 - 3373},
  note = {NULL},
  doi = {https://doi.org/10.1063/1.1777575}
}
</pre></td>
</tr>
<tr id="Creighton1993" class="entry">
	<td>Creighton, T.E.</td>
	<td>Proteins: Structures and Molecular Properties <p class="infolinks">[<a href="javascript:toggleInfo('Creighton1993','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Creighton1993','bibtex')">BibTeX</a>]</p></td>
	<td>1993</td>
	<td>W. H. Freeman, New York<br/>Vol. 2nd, pp. 507 - 507&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Creighton1993" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: 2nd ed.</td>
</tr>
<tr id="bib_Creighton1993" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Creighton1993,
  author = {Creighton, Thomas E},
  title = {Proteins: Structures and Molecular Properties},
  journal = {W. H. Freeman, New York},
  year = {1993},
  volume = {2nd},
  pages = {507 - 507}
}
</pre></td>
</tr>
<tr id="Crowther1976" class="entry">
	<td>Crowther, R., Pinch, J. and Pearse, B.</td>
	<td>On the structure of coated vesicles <p class="infolinks">[<a href="javascript:toggleInfo('Crowther1976','bibtex')">BibTeX</a>]</p></td>
	<td>1976</td>
	<td>Journal of molecular biology<br/>Vol. 103(4), pp. 785-798&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Crowther1976" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Crowther1976,
  author = {Crowther, RA and Pinch, JT and Pearse, BMF},
  title = {On the structure of coated vesicles},
  journal = {Journal of molecular biology},
  publisher = {Elsevier},
  year = {1976},
  volume = {103},
  number = {4},
  pages = {785--798},
  note = {108 molecules of clathrin -> one of the shapes of CCVs}
}
</pre></td>
</tr>
<tr id="Cui2001" class="entry">
	<td>Cui, Q., Elstner, M., Kaxiras, E., Frauenheim, T. and Karplus, M.</td>
	<td>A QM/MM Implementation of the Self-Consistent Charge Density Functional Tight Binding (SCC-DFTB) Method <p class="infolinks">[<a href="javascript:toggleInfo('Cui2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Cui2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 105(2), pp. 569 - 585&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp0029109">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Cui2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A quantum mechanical/molecular mechanical (QM/MM) approach based on an approximate density functional theory, the so-called self-consistent charge density functional tight binding (SCC-DFTB) method, has been implemented in the CHARMM program and tested on a number of systems of biological interest. In the gas phase, SCC-DFTB gives reliable energetics for models of the triosephosphate isomerase (TIM) catalyzed reactions. The rms errors in the energetics compared to B3LYP/6-31+G(d,p) are about 2?4 kcal/mol; this is to be contrasted with AM1, where the corresponding errors are 9?11 kcal/mol. The method also gives accurate vibrational frequencies. For the TIM reactions in the presence of the enzyme, the overall SCC-DFTB/CHARMM results are in somewhat worse agreement with the B3LYP/6-31+G(d,p)/CHARMM values; the rms error in the energies is 5.4 kcal/mol. Single-point B3LYP/CHARMM energies at the SCC-DFTB/CHARMM optimized structures were found to be very similar to the full B3LYP/CHARMM values. The relative stabilities of the αR and 310 conformations of penta- and octaalanine peptides were studied with minimization and molecular dynamics simulations in vacuum and in solution. Although CHARMM and SCC-DFTB give qualitative different results in the gas phase (the latter is in approximate agreement with previous B3LYP calculations), similar behavior was found in aqueous solution simulations with CHARMM and SCC-DFTB/CHARMM. The 310 conformation was not found to be stable, and converted to the αR form in about 15 ps. The αR conformation was stable in the simulation with both SCC-DFTB/CHARMM and CHARMM. The i,i+3 COtextperiodcenteredtextperiodcenteredtextperiodcenteredHN distances in the αR conformation were shorter with the SCC-DFTB method (2.58 &Aring;) than with CHARMM (3.13 &Aring;). With SCC-DFTB/CHARMM, significant populations with i,i+3 COtextperiodcenteredtextperiodcenteredtextperiodcenteredHN distances near 2.25 &Aring;, particularly for the residues at the termini, were found. This can be related to the conclusion from NMR spectroscopy that the 310 configuration contributes for alanine-rich peptides, especially at the termini. A quantum mechanical/molecular mechanical (QM/MM) approach based on an approximate density functional theory, the so-called self-consistent charge density functional tight binding (SCC-DFTB) method, has been implemented in the CHARMM program and tested on a number of systems of biological interest. In the gas phase, SCC-DFTB gives reliable energetics for models of the triosephosphate isomerase (TIM) catalyzed reactions. The rms errors in the energetics compared to B3LYP/6-31+G(d,p) are about 2?4 kcal/mol; this is to be contrasted with AM1, where the corresponding errors are 9?11 kcal/mol. The method also gives accurate vibrational frequencies. For the TIM reactions in the presence of the enzyme, the overall SCC-DFTB/CHARMM results are in somewhat worse agreement with the B3LYP/6-31+G(d,p)/CHARMM values; the rms error in the energies is 5.4 kcal/mol. Single-point B3LYP/CHARMM energies at the SCC-DFTB/CHARMM optimized structures were found to be very similar to the full B3LYP/CHARMM values. The relative stabilities of the αR and 310 conformations of penta- and octaalanine peptides were studied with minimization and molecular dynamics simulations in vacuum and in solution. Although CHARMM and SCC-DFTB give qualitative different results in the gas phase (the latter is in approximate agreement with previous B3LYP calculations), similar behavior was found in aqueous solution simulations with CHARMM and SCC-DFTB/CHARMM. The 310 conformation was not found to be stable, and converted to the αR form in about 15 ps. The αR conformation was stable in the simulation with both SCC-DFTB/CHARMM and CHARMM. The i,i+3 COtextperiodcenteredtextperiodcenteredtextperiodcenteredHN distances in the αR conformation were shorter with the SCC-DFTB method (2.58 &Aring;) than with CHARMM (3.13 &Aring;). With SCC-DFTB/CHARMM, significant populations with i,i+3 COtextperiodcenteredtextperiodcenteredtextperiodcenteredHN distances near 2.25 &Aring;, particularly for the residues at the termini, were found. This can be related to the conclusion from NMR spectroscopy that the 310 configuration contributes for alanine-rich peptides, especially at the termini.</td>
</tr>
<tr id="bib_Cui2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cui2001,
  author = {Cui, Qiang and Elstner, Marcus and Kaxiras, Efthimios and Frauenheim, Thomas and Karplus, Martin},
  title = {A QM/MM Implementation of the Self-Consistent Charge Density Functional Tight Binding (SCC-DFTB) Method},
  journal = {Journal of Physical Chemistry B},
  year = {2001},
  volume = {105},
  number = {2},
  pages = {569 - 585},
  doi = {https://doi.org/10.1021/jp0029109}
}
</pre></td>
</tr>
<tr id="Cupers1997" class="entry">
	<td>Cupers, P., Haar, E., Boll, W. and Kirchhausen, T.</td>
	<td>Parallel dimers and anti-parallel tetramers formed by epidermal growth factor receptor pathway substrate clone 15 (EPS15) <p class="infolinks">[<a href="javascript:toggleInfo('Cupers1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Cupers1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Journal of Biological Chemistry<br/>Vol. 272(52), pp. 33430 - 33434&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.272.52.33430">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Cupers1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The recently discovered localization of epidermal growth factor receptor pathway substrate clone 15 (Eps15) to plasma membrane clathrin-coated pits and its constitutive association with the endocytic clathrin adaptor protein complex, AP-2, strongly suggest that Eps15 has an important role in the pathway of clathrin-dependent endocytic traffic. We report here that Eps15 forms dimers and tetramers of distinct shape. The Eps15 dimer is an elongated molecule, 32 nm in length. There is a globular "head" at one end of the molecule and an extended "stalk" of 25 nm which is kinked at about 17 nm away from the head. In the Eps15 dimer, two subunits are arranged parallel to each other, so that the head corresponds to two side by side copies of the N-terminal region I, which contains the three Eps15 homology domains. The proximal part of the stalk is the coiled-coil central region II containing 20 heptad repeats. The kink is at the boundary between region II and the C-terminal region III, which contains the AP-2 binding site, 15 aspartic-proline-phenylalanine repeats, and proline-rich Src homology domain ligand sites. The Eps15 tetramer has a "dumbbell" shape, [\textasciitilde\]31 nm in length; it is formed by the anti-parallel association of two Eps15 dimers. Formation of these Eps15 tetramers appears to require contacts between regions I of one dimer and regions III of a second apposing dimer. The extended shapes of the Eps15 dimers and tetramers suggest how Eps15 oligomers are located in the clathrin coat. We discuss the implications for accessibility to partners and for proposed functions of Eps15.</td>
</tr>
<tr id="bib_Cupers1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Cupers1997,
  author = {Cupers, Philippe and Haar, Ernst and Boll, Werner and Kirchhausen, Tomas},
  title = {Parallel dimers and anti-parallel tetramers formed by epidermal growth factor receptor pathway substrate clone 15 (EPS15)},
  journal = {Journal of Biological Chemistry},
  year = {1997},
  volume = {272},
  number = {52},
  pages = {33430 - 33434},
  note = {EPS15 forms dimers and tetramers.},
  doi = {https://doi.org/10.1074/jbc.272.52.33430}
}
</pre></td>
</tr>
<tr id="Daecke2005" class="entry">
	<td>Daecke, J., Fackler, O., Dittmar, M. and Krausslich, H.</td>
	<td>Involvement of clathrin-mediated endocytosis in human immunodeficiency virus type 1 entry <p class="infolinks">[<a href="javascript:toggleInfo('Daecke2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Daecke2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Journal of Virology<br/>Vol. 79(3), pp. 1581 - 1581&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1128/JVI.79.3.1581">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Daecke2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Productive entry of human immunodeficiency virus (HIV) is believed to occur by direct fusion at the plasma membrane. Endocytic uptake of HIV particles has been observed in several studies but is considered to be nonproductive, leading to virus degradation in the lysosome. We show here that endocytosis contributes significantly to productive HIV entry in HeLa cells by using trans dominant-negative mutants of dynamin and Eps15. Inducible expression of a dominant-negative mutant of dynamin in a CD4-positive HeLa cell line reduced HIV infection by 40 to 80%. This effect was independent of the infectious dose and was observed for three different isolates. Analysis of reverse transcription products by real-time PCR and of virus entry by delivery of a virion-associated Vpr---lactamase fusion protein revealed a similar reduction, indicating that the block occurred at the entry stage. A strong reduction of HIV entry was also observed upon transient trans- fection of a different trans dominant-negative variant of dynamin, and this reduction correlated with the relative inhibition of transferrin endocytosis. Expression of a dominant-negative variant of Eps15, which is specific for clathrin-dependent endocytosis, reduced HIV entry in HeLa cells by ca 95 confirming the role of endocytosis for productive infection. In contrast, no effect was observed for a dominant-negative variant of caveolin. We conclude that dynamin-dependent, clathrin-mediated endocytosis can lead to productive entry of HIV in HeLa cells, suggesting this pathway as an alternative route of virus entry</td>
</tr>
<tr id="bib_Daecke2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Daecke2005,
  author = {Daecke, Jessica and Fackler, O.T. and Dittmar, M.T. and Krausslich, H.G.},
  title = {Involvement of clathrin-mediated endocytosis in human immunodeficiency virus type 1 entry},
  journal = {Journal of Virology},
  year = {2005},
  volume = {79},
  number = {3},
  pages = {1581 - 1581},
  doi = {https://doi.org/10.1128/JVI.79.3.1581}
}
</pre></td>
</tr>
<tr id="Dametto2012" class="entry">
	<td>Dametto, M., Antoniou, D. and Schwartz, S.D.</td>
	<td>Barrier crossing in dihydrofolate reductase does not involve a rate-promoting vibration <p class="infolinks">[<a href="javascript:toggleInfo('Dametto2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Dametto2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Molecular Physics<br/>Vol. 110(9-10), pp. 531 - 536&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1080/00268976.2012.655337">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Dametto2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We have studied atomic motions during the chemical reaction catalyzed by the enzyme dihydrofolate reductase of Escherichia coli (EcDHFR), an important enzyme for nucleic acid synthesis. In our earlier work on the enzymes human lactate dehydrogenase and purine nucleoside phosphorylase, we had identified fast sub-ps motions that are part of the reaction coordinate. We employed Transition Path Sampling (TPS) and our recently developed reaction coordinate identification methodology to investigate if such fast motions couple to the reaction in DHFR on the barrier-crossing timescale. While we identified some protein motions near the barrier crossing event, these motions do not constitute a compressive promoting vibration, and do not appear as a clearly identifiable protein component in reaction.</td>
</tr>
<tr id="bib_Dametto2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dametto2012,
  author = {Dametto, Mariangela and Antoniou, Dimitri and Schwartz, Steven D},
  title = {Barrier crossing in dihydrofolate reductase does not involve a rate-promoting vibration},
  journal = {Molecular Physics},
  year = {2012},
  volume = {110},
  number = {9-10},
  pages = {531 - 536},
  note = {NULL},
  doi = {https://doi.org/10.1080/00268976.2012.655337}
}
</pre></td>
</tr>
<tr id="Daniel2015" class="entry">
	<td>Daniel, J.A., Chau, N., Abdel-Hamid, M.K., Hu, L., von Kleist, L., Whiting, A., Krishnan, S., Maamary, P., Joseph, S.R., Simpson, F., Haucke, V., McCluskey, A. and Robinson, P.J.</td>
	<td>Phenothiazine-derived antipsychotic drugs inhibit dynamin and clathrin-mediated endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Daniel2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Daniel2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Traffic (Copenhagen, Denmark)<br/>Vol. 16, pp. 635-654&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1111/tra.12272">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Daniel2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Chlorpromazine is a phenothiazine-derived antipsychotic drug (APD) that inhibits clathrin-mediated endocytosis (CME) in cells by an unknown mechanism. We examined whether its action and that of other APDs might be mediated by the GTPase activity of dynamin. Eight of eight phenothiazine-derived APDs inhibited dynamin I (dynI) in the 2-12 µm range, the most potent being trifluoperazine (IC50 2.6 ± 0.7 µm). They also inhibited dynamin II (dynII) at similar concentrations. Typical and atypical APDs not based on the phenothiazine scaffold were 8- to 10-fold less potent (haloperidol and clozapine) or were inactive (droperidol, olanzapine and risperidone). Kinetic analysis showed that phenothiazine-derived APDs were lipid competitive, while haloperidol was uncompetitive with lipid. Accordingly, phenothiazine-derived APDs inhibited dynI GTPase activity stimulated by lipids but not by various SH3 domains. All dynamin-active APDs also inhibited transferrin (Tfn) CME in cells at related potencies. Structure-activity relationships (SAR) revealed dynamin inhibition to be conferred by a substituent group containing a terminal tertiary amino group at the N2 position. Chlorpromazine was previously proposed to target AP-2 recruitment in the formation of clathrin-coated vesicles (CCV). However, neither chlorpromazine nor thioridazine affected AP-2 interaction with amphiphysin or clathrin. Super-resolution microscopy revealed that chlorpromazine blocks neither clathrin recruitment by AP-2, nor AP-2 recruitment, showing that CME inhibition occurs downstream of CCV formation. Overall, potent dynamin inhibition is a shared characteristic of phenothiazine-derived APDs, but not other typical or atypical APDs, and the data indicate that dynamin is their likely in-cell target in endocytosis.</td>
</tr>
<tr id="bib_Daniel2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Daniel2015,
  author = {Daniel, James A and Chau, Ngoc and Abdel-Hamid, Mohammed K and Hu, Lingbo and von Kleist, Lisa and Whiting, Ainslie and Krishnan, Sai and Maamary, Peter and Joseph, Shannon R and Simpson, Fiona and Haucke, Volker and McCluskey, Adam and Robinson, Phillip J},
  title = {Phenothiazine-derived antipsychotic drugs inhibit dynamin and clathrin-mediated endocytosis.},
  journal = {Traffic (Copenhagen, Denmark)},
  year = {2015},
  volume = {16},
  pages = {635--654},
  note = {drug interactions with CME},
  doi = {https://doi.org/10.1111/tra.12272}
}
</pre></td>
</tr>
<tr id="Dannhauser2012" class="entry">
	<td>Dannhauser, P. and Ungewickell, E.</td>
	<td>Reconstitution of clathrin-coated bud and vesicle formation with minimal components <p class="infolinks">[<a href="javascript:toggleInfo('Dannhauser2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Dannhauser2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Nature cell biology&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Dannhauser2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Abstract During the process of clathrin-mediated endocytosis an essentially planar area of membrane has to undergo a gross deformation to form a spherical bud. Three ways have been recognized by which membranes can be induced to transform themselves locally from a planar state to one of high curvature: a change in lipid distribution between the leaflets, insertion of a protein into one leaflet and formation of a protein scaffold over the surface.  ...</td>
</tr>
<tr id="bib_Dannhauser2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dannhauser2012,
  author = {Dannhauser, PN and Ungewickell, EJ},
  title = {Reconstitution of clathrin-coated bud and vesicle formation with minimal components},
  journal = {Nature cell biology},
  year = {2012}
}
</pre></td>
</tr>
<tr id="Daumke2014" class="entry">
	<td>Daumke, O., Roux, A. and Haucke, V.</td>
	<td>BAR domain scaffolds in dynamin-mediated membrane fission. <p class="infolinks">[<a href="javascript:toggleInfo('Daumke2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Daumke2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Cell<br/>Vol. 156, pp. 882-892&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cell.2014.02.017">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Daumke2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Biological membranes undergo constant remodeling by membrane fission and fusion to change their shape and to exchange material between subcellular compartments. During clathrin-mediated endocytosis, the dynamic assembly and disassembly of protein scaffolds comprising members of the bin-amphiphysin-rvs (BAR) domain protein superfamily constrain the membrane into distinct shapes as the pathway progresses toward fission by the GTPase dynamin. In this Review, we discuss how BAR domain protein assembly and disassembly are controlled in space and time and which structural and biochemical features allow the tight regulation of their shape and function to enable dynamin-mediated membrane fission.</td>
</tr>
<tr id="bib_Daumke2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Daumke2014,
  author = {Daumke, Oliver and Roux, Aurélien and Haucke, Volker},
  title = {BAR domain scaffolds in dynamin-mediated membrane fission.},
  journal = {Cell},
  year = {2014},
  volume = {156},
  pages = {882--892},
  doi = {https://doi.org/10.1016/j.cell.2014.02.017}
}
</pre></td>
</tr>
<tr id="Davarifar2011" class="entry">
	<td>Davarifar, A., Antoniou, D. and Schwartz, S.D.</td>
	<td>The promoting vibration in human heart lactate dehydrogenase is a preferred vibrational channel <p class="infolinks">[<a href="javascript:toggleInfo('Davarifar2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Davarifar2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 115(51), pp. 15439 - 15444&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp210347h">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Davarifar2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We examine whether the rate-promoting vibration of lactate dehydrogenase is a preferred axis of thermal energy transfer. While it seems plausible that such a mechanistically important motion is also a favored direction of energy transfer, none of the previous studies of rate-promoting vibrations in enzymatic catalysis have addressed this question. It is equally likely that the promoting vibration, though catalytically important, has no different properties than any other axis in the protein. Resolution of this issue is important for two reasons: First, if energy is transferred along this axis in a preferred fashion, it shows that the protein is engineered in a way that transfers thermal energy into a motion that is coupled to the chemical step. Second, the discovery of a preferred direction of thermal transfer provides a potential route to experimental verification of the promoting vibration concept. Our computational experiments are specifically designed to mimic potential laser experiment with the deposition of thermal energy in an active-site chromophore with subsequent measurement of temperature at various points in the protein. Our results indicate that the promoting vibration is indeed a preferred channel of energy transfer. In addition, we study the vibrational structure of the protein via the dynamical structure factor to show preferred vibrational motion along the promoting vibration axis is an inherent property of the protein structure via thermal fluctuations.</td>
</tr>
<tr id="bib_Davarifar2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Davarifar2011,
  author = {Davarifar, Ardy and Antoniou, Dimitri and Schwartz, Steven D},
  title = {The promoting vibration in human heart lactate dehydrogenase is a preferred vibrational channel},
  journal = {Journal of Physical Chemistry B},
  year = {2011},
  volume = {115},
  number = {51},
  pages = {15439 - 15444},
  doi = {https://doi.org/10.1021/jp210347h}
}
</pre></td>
</tr>
<tr id="David1996" class="entry">
	<td>David, C., McPherson, P.S., Mundigl, O. and de Camilli, P.</td>
	<td>A role of amphiphysin in synaptic vesicle endocytosis suggested by its binding to dynamin in nerve terminals <p class="infolinks">[<a href="javascript:toggleInfo('David1996','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('David1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>Proceedings of the National academy of Sciences<br/>Vol. 93, pp. 331-335&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_David1996" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Amphiphysin, a major autoantigen in paraneoplastic Stiff-Man syndrome, is an SH3 domain-containing neuronal protein, concentrated in nerve terminals. Here, we demonstrate a specific, SH3 domain-mediated, interaction between amphiphysin and dynamin by gel overlay and affinity chromatography. In addition, we show that the two proteins are colocalized in nerve terminals and are coprecipitated from brain extracts consistent with their interactions in situ. We also report that a region of amphiphysin distinct from its SH3 domain mediates its binding to the alpha c subunit of AP2 adaptin, which is also concentrated in nerve terminals. These findings support a role of amphiphysin in synaptic vesicle endocytosis.</td>
</tr>
<tr id="bib_David1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{David1996,
  author = {David, C and McPherson, P S and Mundigl, O and de Camilli, P},
  title = {A role of amphiphysin in synaptic vesicle endocytosis suggested by its binding to dynamin in nerve terminals},
  journal = {Proceedings of the National academy of Sciences},
  year = {1996},
  volume = {93},
  pages = {331--335},
  note = {Rabbit/rat brain. AP2A2 specifically binds to AMPH. direct, but not through the AMPH.SH3 domain.}
}
</pre></td>
</tr>
<tr id="DeCamilli2002" class="entry">
	<td>De Camilli, P., Chen, H., Hyman, J., Panepucci, E., Bateman, A. and Brunger, A.T.</td>
	<td>The ENTH domain. <p class="infolinks">[<a href="javascript:toggleInfo('DeCamilli2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('DeCamilli2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>FEBS letters<br/>Vol. 513, pp. 11-18&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_DeCamilli2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The epsin NH2-terminal homology (ENTH) domain is a membrane interacting module composed by a superhelix of alpha-helices. It is present at the NH2-terminus of proteins that often contain consensus sequences for binding to clathrin coat components and their accessory factors, and therefore function as endocytic adaptors. ENTH domain containing proteins have additional roles in signaling and actin regulation and may have yet other actions in the nucleus. The ENTH domain is structurally similar to the VHS domain. These domains define two families of adaptor proteins which function in membrane traffic and whose interaction with membranes is regulated, in part, by phosphoinositides.</td>
</tr>
<tr id="bib_DeCamilli2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{DeCamilli2002,
  author = {De Camilli, Pietro and Chen, Hong and Hyman, Joel and Panepucci, Ezequiel and Bateman, Alex and Brunger, Axel T},
  title = {The ENTH domain.},
  journal = {FEBS letters},
  year = {2002},
  volume = {513},
  pages = {11--18}
}
</pre></td>
</tr>
<tr id="Deck1998" class="entry">
	<td>Deck, L., Royer, R., Chamblee, B., Hernandez, V., Malone, R., Torres, J., Hunsaker, L., Piper, R., Makler, M. and Jagt, V.D.</td>
	<td>Selective inhibitors of human lactate dehydrogenases and lactate dehydrogenase from the malarial parasite Plasmodium falciparum <p class="infolinks">[<a href="javascript:toggleInfo('Deck1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Deck1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>J Med Chem<br/>Vol. 41(20), pp. 3879 - 3887&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jm980334n">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Deck1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Derivatives of the sesquiterpene 8-deoxyhemigossylic acid (2, 3-dihydroxy-6-methyl-4-(1-methylethyl)-1-naphthoic acid) were synthesized that contained altered alkyl groups in the 4-position and contained alkyl or aralkyl groups in the 7-position. These substituted dihydroxynaphthoic acids are selective inhibitors of human lactate dehydrogenase-H (LDH-H) and LDH-M and of lactate dehydrogenase from the malarial parasite Plasmodium falciparum (pLDH). All inhibitors are competitive with the binding of NADH. Selectivity for LDH-H, LDH-M, or pLDH is strongly dependent upon the groups that are in the 4- and 7-positions of the dihydroxynaphthoic acid backbone. Dissociation constants as low as 50 nM were observed, with selectivity as high as 400-fold.</td>
</tr>
<tr id="bib_Deck1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Deck1998,
  author = {Deck, LM and Royer, RE and Chamblee, BB and Hernandez, VM and Malone, RR and Torres, JE and Hunsaker, LA and Piper, RC and Makler, MT and Jagt, Vander DL},
  title = {Selective inhibitors of human lactate dehydrogenases and lactate dehydrogenase from the malarial parasite Plasmodium falciparum},
  journal = {J Med Chem},
  year = {1998},
  volume = {41},
  number = {20},
  pages = {3879 - 3887},
  doi = {https://doi.org/10.1021/jm980334n}
}
</pre></td>
</tr>
<tr id="Delft1997" class="entry">
	<td>Delft, S., Schumacher, C., Hage, W., Verkleij, A.J. and en Henegouwen, P.M.</td>
	<td>Association and colocalization of Eps15 with adaptor protein-2 and clathrin <p class="infolinks">[<a href="javascript:toggleInfo('Delft1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Delft1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Journal of Cell Biology<br/>Vol. 136(4), pp. 811 - 821&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1083/jcb.136.4.811">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Delft1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Eps15 has been identified as a substrate of the EGF receptor tyrosine kinase. In this report, we show that activation of the EGF receptor by either EGF or TGF-alpha results in phosphorylation of Eps15. Stimulation of cells with PDGF or insulin did not lead to Eps15 phosphorylation, suggesting that phosphorylation of Eps15 is a receptor-specific process. We demonstrate that Eps15 is constitutively associated with both alpha-adaptin and clathrin. Upon EGF stimulation, Eps15 and alpha-adaptin are recruited to the EGF receptor. Using a truncated EGF receptor mutant, we demonstrate that the regulatory domain of the cytoplasmic tail of the EGF receptor is essential for the binding of Eps15. Fractionation studies reveal that Eps15 is present in cell fractions enriched for plasma membrane and endosomal membranes. Immunofluorescence studies show that Eps15 colocalizes with adaptor protein-2 (AP-2) and partially with clathrin. No colocalization of Eps15 was observed with the early endosomal markers rab4 and rab5. These observations indicate that Eps15 is present in coated pits and coated vesicles of the clathrin-mediated endocytic pathway, but not in early endosomes. Neither AP-2 nor clathrin are required for the binding of Eps15 to coated pits or coated vesicles, since in membranes lacking AP-2 and clathrin, Eps15 still shows the same staining pattern. These findings suggest that Eps15 may play a critical role in the recruitment of active EGF receptors into coated pit regions before endocytosis of ligand-occupied EGF receptors.</td>
</tr>
<tr id="bib_Delft1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Delft1997,
  author = {Delft, Sanne and Schumacher, Christopher and Hage, Willem and Verkleij, Arie J and en Henegouwen, Paul MP},
  title = {Association and colocalization of Eps15 with adaptor protein-2 and clathrin},
  journal = {Journal of Cell Biology},
  year = {1997},
  volume = {136},
  number = {4},
  pages = {811 - 821},
  doi = {https://doi.org/10.1083/jcb.136.4.811}
}
</pre></td>
</tr>
<tr id="DellAngelica1998" class="entry">
	<td>Dell'Angelica, E.C., Klumperman, J., Stoorvogel, W. and Bonifacino, J.S.</td>
	<td>Association of the AP-3 adaptor complex with clathrin <p class="infolinks">[<a href="javascript:toggleInfo('DellAngelica1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Science<br/>Vol. 280(5362), pp. 431-434&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_DellAngelica1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{DellAngelica1998,
  author = {Dell'Angelica, Esteban C and Klumperman, Judith and Stoorvogel, Willem and Bonifacino, Juan S},
  title = {Association of the AP-3 adaptor complex with clathrin},
  journal = {Science},
  publisher = {American Association for the Advancement of Science},
  year = {1998},
  volume = {280},
  number = {5362},
  pages = {431--434}
}
</pre></td>
</tr>
<tr id="Dellago1999" class="entry">
	<td>Dellago, C., Bolhuis, P.G. and Chandler, D.</td>
	<td>On the calculation of reaction rate constants in the transition path ensemble <p class="infolinks">[<a href="javascript:toggleInfo('Dellago1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Dellago1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>The Journal of Chemical Physics<br/>Vol. 110(14), pp. 6617 - 6625&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.478569">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Dellago1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present improved formulas for the calculation of transition rate constants in the transition path ensemble. In this method transition paths between stable states are generated by sampling the distribution of paths with a Monte Carlo procedure. With the new expressions the computational cost for the calculation of transition rate constants can be reduced considerably compared to our original formulation. We demonstrate the method by studying the isomerization of a diatomic molecule immersed in a Weeks--Chandler--Andersen fluid. The paper is concluded by an efficiency analysis of the path sampling algorithm.</td>
</tr>
<tr id="bib_Dellago1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dellago1999,
  author = {Dellago, Christoph and Bolhuis, Peter G and Chandler, David},
  title = {On the calculation of reaction rate constants in the transition path ensemble},
  journal = {The Journal of Chemical Physics},
  year = {1999},
  volume = {110},
  number = {14},
  pages = {6617 - 6625},
  note = {Book of Dellago II},
  doi = {https://doi.org/10.1063/1.478569}
}
</pre></td>
</tr>
<tr id="Dellago1998" class="entry">
	<td>Dellago, C., Bolhuis, P.G., Csajka, F.S. and Chandler, D.</td>
	<td>Transition path sampling and the calculation of rate constants <p class="infolinks">[<a href="javascript:toggleInfo('Dellago1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Dellago1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>The Journal of Chemical Physics<br/>Vol. 108(5), pp. 1964 - 1977&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.475562">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Dellago1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We have developed a method to study transition pathways for rare events in complex systems. The method can be used to determine rate constants for transitions between stable states by turning the calculation of reactive flux correlation functions into the computation of an isomorphic reversible work. In contrast to previous dynamical approaches, the method relies neither on prior knowledge nor on explicit specification of transition states. Rather, it provides an importance sampling from which transition states can be characterized statistically. A simple model is analyzed to illustrate the methodology.</td>
</tr>
<tr id="bib_Dellago1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dellago1998,
  author = {Dellago, Christoph and Bolhuis, Peter G and Csajka, Felix S and Chandler, David},
  title = {Transition path sampling and the calculation of rate constants},
  journal = {The Journal of Chemical Physics},
  year = {1998},
  volume = {108},
  number = {5},
  pages = {1964 - 1977},
  note = {Book of Dellago I},
  doi = {https://doi.org/10.1063/1.475562}
}
</pre></td>
</tr>
<tr id="Dellago2002" class="entry">
	<td>Dellago, C., Bolhuis, P.G. and Geissler, P.L.</td>
	<td>Transition Path Sampling <p class="infolinks">[<a href="javascript:toggleInfo('Dellago2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Dellago2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td><br/>Vol. 123, pp. 1&nbsp;</td>
	<td>book</td>
	<td><a href="https://doi.org/10.1002/0471231509.ch1">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Dellago2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In this chapter we present the foundations and methodology of transition path sampling comprehensively, including details important for its implementation. We first discuss the theoretical basis of transition path sampling, namely, a statistical mechanics of trajectories. We then describe how reactive trajectories may be efficiently sampled, and subsequently analyzed. The practical simplicity of the method is emphasized by outlining essential algorithms in separate schemes.</td>
</tr>
<tr id="bib_Dellago2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{Dellago2002,
  author = {Dellago, Christoph and Bolhuis, Peter G and Geissler, Phillip L},
  title = {Transition Path Sampling},
  publisher = {John Wiley &amp; Sons, Inc.},
  year = {2002},
  volume = {123},
  pages = {1},
  note = {NULL},
  doi = {https://doi.org/10.1002/0471231509.ch1}
}
</pre></td>
</tr>
<tr id="Dematteis2018" class="entry">
	<td>Dematteis, G., Grafke, T. and Vanden-Eijnden, E.</td>
	<td>Rogue waves and large deviations in deep sea <p class="infolinks">[<a href="javascript:toggleInfo('Dematteis2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Proceedings of the National Academy of Sciences, pp. 201710670&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Dematteis2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dematteis2018,
  author = {Dematteis, Giovanni and Grafke, Tobias and Vanden-Eijnden, Eric},
  title = {Rogue waves and large deviations in deep sea},
  journal = {Proceedings of the National Academy of Sciences},
  publisher = {National Acad Sciences},
  year = {2018},
  pages = {201710670}
}
</pre></td>
</tr>
<tr id="Demerdash2009" class="entry">
	<td>Demerdash, O.N., Daily, M.D. and Mitchell, J.C.</td>
	<td>Structure-based predictive models for allosteric hot spots <p class="infolinks">[<a href="javascript:toggleInfo('Demerdash2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>PLoS computational biology<br/>Vol. 5(10), pp. e1000531&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Demerdash2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Demerdash2009,
  author = {Demerdash, Omar NA and Daily, Michael D and Mitchell, Julie C},
  title = {Structure-based predictive models for allosteric hot spots},
  journal = {PLoS computational biology},
  publisher = {Public Library of Science},
  year = {2009},
  volume = {5},
  number = {10},
  pages = {e1000531}
}
</pre></td>
</tr>
<tr id="DePristo2005" class="entry">
	<td>DePristo, M., Weinreich, D.M. and Hartl, D.L.</td>
	<td>Missense meanderings in sequence space: a biophysical view of protein evolution. <p class="infolinks">[<a href="javascript:toggleInfo('DePristo2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('DePristo2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Nature Reviews Genetics<br/>Vol. 6(9), pp. 678 - 687&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nrg1672">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_DePristo2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Proteins are finicky molecules; they are barely stable and are prone to aggregate, but they must function in a crowded environment that is full of degradative enzymes bent on their destruction. It is no surprise that many common diseases are due to missense mutations that affect protein stability and aggregation. Here we review the literature on biophysics as it relates to molecular evolution, focusing on how protein stability and aggregation affect organismal fitness. We then advance a biophysical model of protein evolution that helps us to understand phenomena that range from the dynamics of molecular adaptation to the clock-like rate of protein evolution.</td>
</tr>
<tr id="bib_DePristo2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{DePristo2005,
  author = {DePristo, Mark and Weinreich, Daniel M and Hartl, Daniel L},
  title = {Missense meanderings in sequence space: a biophysical view of protein evolution.},
  journal = {Nature Reviews Genetics},
  year = {2005},
  volume = {6},
  number = {9},
  pages = {678 - 687},
  doi = {https://doi.org/10.1038/nrg1672}
}
</pre></td>
</tr>
<tr id="Dergai2016" class="entry">
	<td>Dergai, M., Iershov, A., Novokhatska, O., Pankivskyi, S. and Rynditch, A.</td>
	<td>Evolutionary Changes on the Way to Clathrin-Mediated Endocytosis in Animals <p class="infolinks">[<a href="javascript:toggleInfo('Dergai2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Genome Biology and Evolution<br/>Vol. 8(3), pp. 588-606&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Dergai2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dergai2016,
  author = {Dergai, Mykola and Iershov, Anton and Novokhatska, Olga and Pankivskyi, Serhii and Rynditch, Alla},
  title = {Evolutionary Changes on the Way to Clathrin-Mediated Endocytosis in Animals},
  journal = {Genome Biology and Evolution},
  publisher = {Oxford University Press},
  year = {2016},
  volume = {8},
  number = {3},
  pages = {588--606}
}
</pre></td>
</tr>
<tr id="Dergai2011" class="entry">
	<td>Dergai, M., Skrypkina, I., Dergai, O., Tsyba, L., Novokhatska, O., Filonenko, V., Drobot, L. and Rynditch, A.</td>
	<td>Identification and characterization of a novel mammalian isoform of the endocytic adaptor ITSN1 <p class="infolinks">[<a href="javascript:toggleInfo('Dergai2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Dergai2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Gene<br/>Vol. 485(2), pp. 120 - 129&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.gene.2011.06.021">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Dergai2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Intersectin 1 (ITSN1) is an evolutionarily conserved adaptor protein engaged in clathrin-mediated endocytosis, cell signaling and actin cytoskeleton rearrangements. Two major ITSN1 isoforms were initially described, the ubiquitous short isoform (ITSN1-s) and the long isoform (ITSN1-l) expressed predominantly in neurons. Numerous alternative splicing events for ITSN1 pre-mRNA were later identified. Here we describe a novel isoform ITSN1-22a with an alternative C-terminus encoded by exon 22a. This exon is only found in placental mammals. The transcript of ITSN1-22a is detected in a wide range of human and mouse tissues. We show here that two alternative splicing events affect the coding sequence of the ITSN1-22a isoform. Moreover, alternative polyadenylation of these transcripts was demonstrated in human tissues. The protein encoded by the ITSN1-22a transcript possesses two EH domains, a coiled-coil region, an SH3A domain and a specific C-terminal domain (CTD) but lacks four SH3 domains in comparison with ITSN1-s. The level of ITSN1-22a protein varies in different mouse tissues and human cell lines. The highest amounts of this isoform occur in mouse brain, spleen, lung and the human B cell line DG75. ITSN1-22a binds via its CTD to the SH3 domain of the endocytic protein amphiphysin 1 and the SH3A domain of ITSN1. Furthermore association in vivo and codistribution of ITSN1-22a and ITSN1-s were demonstrated suggesting that these isoforms could function in concert. We have revealed differential binding of ITSN1-s and ITSN1-22a to the ubiquitin ligase Cbl. Both isoforms possess the SH3A domain capable of binding to Cbl; however ITSN1-22a in contrast to ITSN1-s did not interact with Cbl in vivo. In vitro binding experiments demonstrated that the CTD of ITSN1-22a negatively regulated its binding to Cbl; at the same time interaction with another partner, dynamin 1 was not affected by the presence of the CTD. These data suggest that intramolecular interaction within ITSN1-22a could specifically regulate its binding to protein partners. Thus, this novel mammalian ITSN1 isoform possesses a significantly altered domain structure and performs specific protein-protein interactions. ?? 2011 Elsevier B.V.</td>
</tr>
<tr id="bib_Dergai2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dergai2011,
  author = {Dergai, Mykola and Skrypkina, Inessa and Dergai, Oleksandr and Tsyba, Liudmyla and Novokhatska, Olga and Filonenko, Valeriy and Drobot, Lyudmyla and Rynditch, Alla},
  title = {Identification and characterization of a novel mammalian isoform of the endocytic adaptor ITSN1},
  journal = {Gene},
  year = {2011},
  volume = {485},
  number = {2},
  pages = {120 - 129},
  note = {HEK and MCF-7 cells. ITSN1-s ubiquitous, ITSN1-l mainly neural. interactions are specific to ITSN1-22a, a novel isoform.},
  doi = {https://doi.org/10.1016/j.gene.2011.06.021}
}
</pre></td>
</tr>
<tr id="Dergai2010" class="entry">
	<td>Dergai, O., Novokhatska, O., Dergai, M., Skrypkina, I., Tsyba, L., Moreau, J. and Rynditch, A.</td>
	<td>Intersectin 1 forms complexes with SGIP1 and Reps1 in clathrin-coated pits. <p class="infolinks">[<a href="javascript:toggleInfo('Dergai2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Dergai2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Biochemical and biophysical research communications<br/>Vol. 402, pp. 408-413&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.bbrc.2010.10.045">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Dergai2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Intersectin 1 (ITSN1) is an evolutionarily conserved adaptor protein involved in clathrin-mediated endocytosis, cellular signaling and cytoskeleton rearrangement. ITSN1 gene is located on human chromosome 21 in Down syndrome critical region. Several studies confirmed role of ITSN1 in Down syndrome phenotype. Here we report the identification of novel interconnections in the interaction network of this endocytic adaptor. We show that the membrane-deforming protein SGIP1 (Src homology 3-domain growth factor receptor-bound 2-like (endophilin) interacting protein 1) and the signaling adaptor Reps1 (RalBP associated Eps15-homology domain protein) interact with ITSN1 in vivo. Both interactions are mediated by the SH3 domains of ITSN1 and proline-rich motifs of protein partners. Moreover complexes comprising SGIP1, Reps1 and ITSN1 have been identified. We also identified new interactions between SGIP1, Reps1 and the BAR (Bin/amphiphysin/Rvs) domain-containing protein amphiphysin 1. Immunofluorescent data have demonstrated colocalization of ITSN1 with the newly identified protein partners in clathrin-coated pits. These findings expand the role of ITSN1 as a scaffolding molecule bringing together components of endocytic complexes.</td>
</tr>
<tr id="bib_Dergai2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dergai2010,
  author = {Dergai, Oleksandr and Novokhatska, Olga and Dergai, Mykola and Skrypkina, Inessa and Tsyba, Liudmyla and Moreau, Jacques and Rynditch, Alla},
  title = {Intersectin 1 forms complexes with SGIP1 and Reps1 in clathrin-coated pits.},
  journal = {Biochemical and biophysical research communications},
  year = {2010},
  volume = {402},
  pages = {408--413},
  doi = {https://doi.org/10.1016/j.bbrc.2010.10.045}
}
</pre></td>
</tr>
<tr id="Dewer1985" class="entry">
	<td>Dewer, M., Zoebisch, E.G., Healy, E.F. and Stewart, J.J.</td>
	<td>AM1: a new general purpose quantum mechanical molecular model <p class="infolinks">[<a href="javascript:toggleInfo('Dewer1985','bibtex')">BibTeX</a>]</p></td>
	<td>1985</td>
	<td>Journal of the American Chemical Society<br/>Vol. 107(13), pp. 3902 - 3909&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Dewer1985" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dewer1985,
  author = {Dewer, Michael and Zoebisch, Eve G and Healy, Eamonn F and Stewart, James JP},
  title = {AM1: a new general purpose quantum mechanical molecular model},
  journal = {Journal of the American Chemical Society},
  year = {1985},
  volume = {107},
  number = {13},
  pages = {3902 - 3909},
  note = {NULL}
}
</pre></td>
</tr>
<tr id="Dilek2001" class="entry">
	<td>Dilek, Turgut-Balik,., Shoemark, D.K., Sessions, R.B., Moreton, K.M. and Holbrook, J.J.</td>
	<td>Mutagenic exploration of the active site of lactate dehydrogenase from Plasmodium falciparum <p class="infolinks">[<a href="javascript:toggleInfo('Dilek2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Dilek2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Biotechnology Letters<br/>Vol. 23(11), pp. 923 - 927&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Dilek2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Several site-directed mutations of residues around the active site of the lactate dehydrogenase from Plasmodium falciparum are described. These include changes to three highly, but not completely, conserved residues in the pocket of the active site and also three changes (including deletions) to the active site loop. Changes to residues in the active-site pocket resulted in little or no over-production of protein and no enzymic activity. Likewise, a five residue deletion from the active site loop gave no over-produced protein, while a two residue deletion and changes of residue type in this loop were tolerated. The results are discussed in the light of this protein being a suitable target for novel anti-malarials.</td>
</tr>
<tr id="bib_Dilek2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dilek2001,
  author = {Dilek, Turgut-Balik, and Shoemark, Debbie K and Sessions, Richard B and Moreton, Kathleen M and Holbrook, John J},
  title = {Mutagenic exploration of the active site of lactate dehydrogenase from Plasmodium falciparum},
  journal = {Biotechnology Letters},
  year = {2001},
  volume = {23},
  number = {11},
  pages = {923 - 927},
  note = {NULL}
}
</pre></td>
</tr>
<tr id="Doshi2012" class="entry">
	<td>Doshi, U. and Hamelberg, D.</td>
	<td>Improved Statistical Sampling and Accuracy with Accelerated Molecular Dynamics on Rotatable Torsions <p class="infolinks">[<a href="javascript:toggleInfo('Doshi2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Journal of Chemical Theory and Computation<br/>Vol. 8, pp. 4004 - 4012&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Doshi2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Doshi2012,
  author = {Doshi, Urmi and Hamelberg, Donald},
  title = {Improved Statistical Sampling and Accuracy with Accelerated Molecular Dynamics on Rotatable Torsions},
  journal = {Journal of Chemical Theory and Computation},
  year = {2012},
  volume = {8},
  pages = {4004 - 4012}
}
</pre></td>
</tr>
<tr id="Doshi2014" class="entry">
	<td>Doshi, U. and Hamelberg, D.</td>
	<td>Towards fast, rigorous and efficient conformational sampling of biomolecules: Advances in accelerated molecular dynamics <p class="infolinks">[<a href="javascript:toggleInfo('Doshi2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Biochimica et Biophysica Acta&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Doshi2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Doshi2014,
  author = {Doshi, Urmi and Hamelberg, Donald},
  title = {Towards fast, rigorous and efficient conformational sampling of biomolecules: Advances in accelerated molecular dynamics},
  journal = {Biochimica et Biophysica Acta},
  year = {2014}
}
</pre></td>
</tr>
<tr id="Dougoud2018" class="entry">
	<td>Dougoud, M., Vinckenbosch, L., Rohr, R., Bersier, L.-F. and Mazza, C.</td>
	<td>The feasibility of equilibria in large ecosystems: a primary but neglected concept in the complexity-stability debate <p class="infolinks">[<a href="javascript:toggleInfo('Dougoud2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>PLoS Computational Biology<br/>Vol. 14(2), pp. e1005988&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pcbi.1005988">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Dougoud2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dougoud2018,
  author = {Dougoud, Michael and Vinckenbosch, Laura and Rohr, Rudolf and Bersier, Louis-F&eacute;lix and Mazza, Christian},
  title = {The feasibility of equilibria in large ecosystems: a primary but neglected concept in the complexity-stability debate},
  journal = {PLoS Computational Biology},
  year = {2018},
  volume = {14},
  number = {2},
  pages = {e1005988},
  doi = {https://doi.org/10.1371/journal.pcbi.1005988}
}
</pre></td>
</tr>
<tr id="Doyon2011" class="entry">
	<td>Doyon, J.B., Zeitler, B., Cheng, J., Cheng, A.T., Cherone, J.M., Santiago, Y., Lee, A.H., Vo, T.D., Doyon, Y., Miller, J.C., Paschon, D.E., Zhang, L., Rebar, E.J., Gregory, P.D., Urnov, F.D. and Drubin, D.G.</td>
	<td>Rapid and efficient clathrin-mediated endocytosis revealed in genome-edited mammalian cells. <p class="infolinks">[<a href="javascript:toggleInfo('Doyon2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Doyon2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Nature cell biology<br/>Vol. 13(3), pp. 331 - 337&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/ncb2175">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Doyon2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis (CME) is the best-studied pathway by which cells selectively internalize molecules from the plasma membrane and surrounding environment. Previous live-cell imaging studies using ectopically overexpressed fluorescent fusions of endocytic proteins indicated that mammalian CME is a highly dynamic but inefficient and heterogeneous process. In contrast, studies of endocytosis in budding yeast using fluorescent protein fusions expressed at physiological levels from native genomic loci have revealed a process that is very regular and efficient. To analyse endocytic dynamics in mammalian cells in which endogenous protein stoichiometry is preserved, we targeted zinc finger nucleases (ZFNs) to the clathrin light chain A and dynamin-2 genomic loci and generated cell lines expressing fluorescent protein fusions from each locus. The genome-edited cells exhibited enhanced endocytic function, dynamics and efficiency when compared with previously studied cells, indicating that CME is highly sensitive to the levels of its protein components. Our study establishes that ZFN-mediated genome editing is a robust tool for expressing protein fusions at endogenous levels to faithfully report subcellular localization and dynamics.</td>
</tr>
<tr id="bib_Doyon2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Doyon2011,
  author = {Doyon, Jeffrey B and Zeitler, Bryan and Cheng, Jackie and Cheng, Aaron T and Cherone, Jennifer M and Santiago, Yolanda and Lee, Andrew H and Vo, Thuy D and Doyon, Yannick and Miller, Jeffrey C and Paschon, David E and Zhang, Lei and Rebar, Edward J and Gregory, Philip D and Urnov, Fyodor D and Drubin, David G},
  title = {Rapid and efficient clathrin-mediated endocytosis revealed in genome-edited mammalian cells.},
  journal = {Nature cell biology},
  year = {2011},
  volume = {13},
  number = {3},
  pages = {331 - 337},
  doi = {https://doi.org/10.1038/ncb2175}
}
</pre></td>
</tr>
<tr id="Drake2000" class="entry">
	<td>Drake, M.T., Downs, M. and Traub, L.M.</td>
	<td>Epsin Binds to Clathrin by Associating Directly with the Clathrin-terminal Domain <p class="infolinks">[<a href="javascript:toggleInfo('Drake2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Drake2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Journal of Biological Chemistry<br/>Vol. 275(9), pp. 6479 - 6489&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.275.9.6479">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Drake2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Epsin is a recently identified protein that appears to play an important role in clathrin-mediated endocytosis. The central region of epsin 1, the so-called DPW domain, binds to the heterotetrameric AP-2 adaptor complex by associating directly with the globular appendage of the α subunit. We have found that this central portion of epsin 1 also associates with clathrin. The interaction with clathrin is direct and not mediated by epsin-bound AP-2. Alanine scanning mutagenesis shows that clathrin binding depends on the sequence 257LMDLADV located within the epsin 1 DPW domain. This sequence, related to the known clathrin-binding sequences in the adaptor β subunits, amphiphysin, and β-arrestin, facilitates the association of epsin 1 with the terminal domain of the clathrin heavy chain. Unexpectedly, inhibiting the binding of AP-2 to the GST-epsin DPW fusion protein by progressively deleting DPW triplets but leaving the LMDLADV sequence intact, diminishes the association of clathrin in parallel with AP-2. Because the β subunit of the AP-2 complex also contains a clathrin-binding site, optimal association with soluble clathrin appears to depend on the presence of at least two distinct clathrin-binding sites, and we show that a second clathrin-binding sequence 480LVDLD, located within the carboxyl-terminal segment of epsin 1, also interacts with clathrin directly. The LMDLADV and LVDLD sequences act cooperatively in clathrin recruitment assays, suggesting that they bind to different sites on the clathrin-terminal domain. The evolutionary conservation of similar clathrin-binding sequences in several metazoan epsin-like molecules suggests that the ability to establish multiple protein-protein contacts within a developing clathrin-coated bud is an important aspect of epsin function.</td>
</tr>
<tr id="bib_Drake2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Drake2000,
  author = {Drake, Matthew T and Downs, Maureen and Traub, Linton M},
  title = {Epsin Binds to Clathrin by Associating Directly with the Clathrin-terminal Domain},
  journal = {Journal of Biological Chemistry},
  year = {2000},
  volume = {275},
  number = {9},
  pages = {6479 - 6489},
  note = {domain information for EPN1 and CLTC. rat epn1. some binding site information for other CLTC binding proteins, too, through citations. EPN1- CLTC},
  doi = {https://doi.org/10.1074/jbc.275.9.6479}
}
</pre></td>
</tr>
<tr id="Drake2001" class="entry">
	<td>Drake, M.T. and Traub, L.M.</td>
	<td>Interaction of two structurally distinct sequence types with the clathrin terminal domain beta-propeller. <p class="infolinks">[<a href="javascript:toggleInfo('Drake2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Drake2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>The Journal of biological chemistry<br/>Vol. 276, pp. 28700-28709&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M104226200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Drake2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The amino-terminal domain of the clathrin heavy chain, which folds into a seven-bladed beta-propeller, binds directly to several endocytic proteins via short sequences based on the consensus residues LLDLD. In addition to a single LLDLD-based, type I clathrin-binding sequence, both amphiphysin and epsin contain a second, distinct sequence that is also capable of binding to clathrin directly. Here, we analyzed these sequences, which we term type II sequences, and show that the (257)LMDLA sequence in rat epsin 1 appears to be a weak clathrin-binding variant of the sequence (417)PWDLW originally found in human amphiphysin II. The structural features of the type II sequence required for association with clathrin are distinct from the LLDLD-based sequence. In the central segment of amphiphysin, the type I and type II sequences cooperate to effect optimal clathrin binding and the formation of sedimentable assemblies. Together, the data provide evidence for two interaction surfaces upon certain endocytic accessory proteins that could cooperate with other coat components to enhance clathrin bud formation at the cell surface.</td>
</tr>
<tr id="bib_Drake2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Drake2001,
  author = {Drake, M T and Traub, L M},
  title = {Interaction of two structurally distinct sequence types with the clathrin terminal domain beta-propeller.},
  journal = {The Journal of biological chemistry},
  year = {2001},
  volume = {276},
  pages = {28700--28709},
  doi = {https://doi.org/10.1074/jbc.M104226200}
}
</pre></td>
</tr>
<tr id="Drew2017" class="entry">
	<td>Drew, K., Müller, C.L., Bonneau, R. and Marcotte, E.M.</td>
	<td>Identifying direct contacts between protein complex subunits from their conditional dependence in proteomics datasets. <p class="infolinks">[<a href="javascript:toggleInfo('Drew2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Drew2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>PLoS computational biology<br/>Vol. 13, pp. e1005625&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pcbi.1005625">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Drew2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Determining the three dimensional arrangement of proteins in a complex is highly beneficial for uncovering mechanistic function and interpreting genetic variation in coding genes comprising protein complexes. There are several methods for determining co-complex interactions between proteins, among them co-fractionation / mass spectrometry (CF-MS), but it remains difficult to identify directly contacting subunits within a multi-protein complex. Correlation analysis of CF-MS profiles shows promise in detecting protein complexes as a whole but is limited in its ability to infer direct physical contacts among proteins in sub-complexes. To identify direct protein-protein contacts within human protein complexes we learn a sparse conditional dependency graph from approximately 3,000 CF-MS experiments on human cell lines. We show substantial performance gains in estimating direct interactions compared to correlation analysis on a benchmark of large protein complexes with solved three-dimensional structures. We demonstrate the method's value in determining the three dimensional arrangement of proteins by making predictions for complexes without known structure (the exocyst and tRNA multi-synthetase complex) and by establishing evidence for the structural position of a recently discovered component of the core human EKC/KEOPS complex, GON7/C14ORF142, providing a more complete 3D model of the complex. Direct contact prediction provides easily calculable additional structural information for large-scale protein complex mapping studies and should be broadly applicable across organisms as more CF-MS datasets become available.</td>
</tr>
<tr id="bib_Drew2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Drew2017,
  author = {Drew, Kevin and Müller, Christian L and Bonneau, Richard and Marcotte, Edward M},
  title = {Identifying direct contacts between protein complex subunits from their conditional dependence in proteomics datasets.},
  journal = {PLoS computational biology},
  year = {2017},
  volume = {13},
  pages = {e1005625},
  doi = {https://doi.org/10.1371/journal.pcbi.1005625}
}
</pre></td>
</tr>
<tr id="Du1998" class="entry">
	<td>Du, R., Pande, V.S., Grosberg, AY., Tanaka, T. and Shakhnovich, E.S.</td>
	<td>On the transition coordinate for protein folding <p class="infolinks">[<a href="javascript:toggleInfo('Du1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Du1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>J Chem Phys<br/>Vol. 108(1), pp. 334 - 350&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.475393">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Du1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: To understand the kinetics of protein folding, we introduce the concept of a "transition coordinate" which is defined to be the coordinate along which the system progresses most slowly. As a practical implementation of this concept, we define the transmission coefficient for any conformation to be the probability for a chain with the given conformation to fold before it unfolds. Since the transmission coefficient can serve as the best possible measure of kinetic distance for a system, we present two methods by which we can determine how closely any parameter of the system approximates the transmission coefficient. As we determine that the transmission coefficient for a short-chain heteropolymer system is dominated by entropic factors, we have chosen to illustrate the methods mentioned by applying them to geometrical properties of the system such as the number of native contacts and the looplength distribution. We find that these coordinates are not good approximations of the transmission coefficient and therefore, cannot adequately describe the kinetics of protein folding. (C) 1998 American Institute of Physics.</td>
</tr>
<tr id="bib_Du1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Du1998,
  author = {Du, Rose and Pande, Vijay S and Grosberg, AY and Tanaka, Toyoichi and Shakhnovich, Eugene S},
  title = {On the transition coordinate for protein folding},
  journal = {J Chem Phys},
  year = {1998},
  volume = {108},
  number = {1},
  pages = {334 - 350},
  doi = {https://doi.org/10.1063/1.475393}
}
</pre></td>
</tr>
<tr id="Duin2001" class="entry">
	<td>Duin, A.CT., Dasgupta, S., Lorant, F. and Goddard, W.A.</td>
	<td>ReaxFF: A reactive force field for hydrocarbons <p class="infolinks">[<a href="javascript:toggleInfo('Duin2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Duin2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Journal of Physical Chemistry A<br/>Vol. 105(41), pp. 9396 - 9409&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp004368u">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Duin2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: To make practical the molecular dynamics simulation of large scale reactive chemical systems (1000s of atoms), we developed ReaxFF, a force field for reactive systems. ReaxFF uses a general relationship between bond distance and bond order on one hand and between bond order and bond energy on the other hand that leads to proper dissociation of bonds to separated atoms. Other valence terms present in the force field (angle and torsion) are defined in terms of the same bond orders so that all these terms go to zero smoothly as bonds break. In addition, ReaxFF has Coulomb and Morse (van der Waals) potentials to describe nonbond interactions between all atoms (no exclusions). These nonbond interactions are shielded at short range so that the Coulomb and van der Waals interactions become constant as Rij ? 0. We report here the ReaxFF for hydrocarbons. The parameters were derived from quantum chemical calculations on bond dissociation and reactions of small molecules plus heat of formation and geometry data for a number of stable hydrocarbon compounds. We find that the ReaxFF provides a good description of these data. Generally, the results are of an accuracy similar or better than PM3, while ReaxFF is about 100 times faster. In turn, the PM3 is about 100 times faster than the QC calculations. Thus, with ReaxFF we hope to be able to study complex reactions in hydrocarbons. To make practical the molecular dynamics simulation of large scale reactive chemical systems (1000s of atoms), we developed ReaxFF, a force field for reactive systems. ReaxFF uses a general relationship between bond distance and bond order on one hand and between bond order and bond energy on the other hand that leads to proper dissociation of bonds to separated atoms. Other valence terms present in the force field (angle and torsion) are defined in terms of the same bond orders so that all these terms go to zero smoothly as bonds break. In addition, ReaxFF has Coulomb and Morse (van der Waals) potentials to describe nonbond interactions between all atoms (no exclusions). These nonbond interactions are shielded at short range so that the Coulomb and van der Waals interactions become constant as Rij ? 0. We report here the ReaxFF for hydrocarbons. The parameters were derived from quantum chemical calculations on bond dissociation and reactions of small molecules plus heat of formation and geometry data for a number of stable hydrocarbon compounds. We find that the ReaxFF provides a good description of these data. Generally, the results are of an accuracy similar or better than PM3, while ReaxFF is about 100 times faster. In turn, the PM3 is about 100 times faster than the QC calculations. Thus, with ReaxFF we hope to be able to study complex reactions in hydrocarbons.</td>
</tr>
<tr id="bib_Duin2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Duin2001,
  author = {Duin, Adri CT and Dasgupta, Siddharth and Lorant, Francois and Goddard, William A},
  title = {ReaxFF: A reactive force field for hydrocarbons},
  journal = {Journal of Physical Chemistry A},
  year = {2001},
  volume = {105},
  number = {41},
  pages = {9396 - 9409},
  doi = {https://doi.org/10.1021/jp004368u}
}
</pre></td>
</tr>
<tr id="Dunn1996" class="entry">
	<td>Dunn, C.R., Banfield, M.J., Barker, J.J., Higham, C.W., Moreton, K.M., Dilek, Turgut-Balik,., Brady, L.R. and Holbrook, J.J.</td>
	<td>The structure of lactate dehydrogenase from Plasmodium falciparum reveals a new target for anti-malarial design <p class="infolinks">[<a href="javascript:toggleInfo('Dunn1996','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Dunn1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>Nature Structural Biology<br/>Vol. 3(11), pp. 912 - 915&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nsb1196-912">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Dunn1996" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The crystal structure of Plasmodium falciparum lactate dehydrogenase reveals a surprising shift in the position of the NADH cofactor that explains the unusual biochemical properties of this enzyme. There is also a distinctive surface cleft adjacent to the NADH binding pocket that forms an attractive target for inhibitor design.</td>
</tr>
<tr id="bib_Dunn1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dunn1996,
  author = {Dunn, Cameron R and Banfield, Mark J and Barker, John J and Higham, Christopher W and Moreton, Kathleen M and Dilek, Turgut-Balik, and Brady, Leo R and Holbrook, John J},
  title = {The structure of lactate dehydrogenase from Plasmodium falciparum reveals a new target for anti-malarial design},
  journal = {Nature Structural Biology},
  year = {1996},
  volume = {3},
  number = {11},
  pages = {912 - 915},
  note = {NULL},
  doi = {https://doi.org/10.1038/nsb1196-912}
}
</pre></td>
</tr>
<tr id="Dynkin1965" class="entry">
	<td>Dynkin, E.B.</td>
	<td>Markov Processes <p class="infolinks">[<a href="javascript:toggleInfo('Dynkin1965','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Dynkin1965','bibtex')">BibTeX</a>]</p></td>
	<td>1965</td>
	<td><br/>Vol. 29&nbsp;</td>
	<td>book</td>
	<td><a href="https://doi.org/10.1016/S0079-8169(08)61835-8">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Dynkin1965" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Markov process theory is basically an extension of ordinary calculus to accommodate functions whos time evolutions are not entirely deterministic. It is a subject that is becoming increasingly important for many fields of science. This book develops the single-variable theory of both continuous and jump Markov processes in a way that should appeal especially to physicists and chemists at the senior and graduate level.</td>
</tr>
<tr id="bib_Dynkin1965" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{Dynkin1965,
  author = {Dynkin, E.B},
  title = {Markov Processes},
  publisher = {Academic Press Inc.},
  year = {1965},
  volume = {29},
  doi = {https://doi.org/10.1016/S0079-8169(08)61835-8}
}
</pre></td>
</tr>
<tr id="Dzierlenga2015" class="entry">
	<td>Dzierlenga, M.W., Antoniou, D. and Schwartz, S.D.</td>
	<td>Another look at the mechanisms of hydride transfer enzymes with quantum and classical transition path sampling <p class="infolinks">[<a href="javascript:toggleInfo('Dzierlenga2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Dzierlenga2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Journal of Physical Chemistry Letters<br/>Vol. 6(7), pp. 1177 - 1181&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jpclett.5b00346">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Dzierlenga2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The mechanisms involved in enzymatic hydride transfer have been studied for years, but questions remain due, in part, to the difficulty of probing the effects of protein motion and hydrogen tunneling. In this study, we use transition path sampling (TPS) with normal mode centroid molecular dynamics (CMD) to calculate the barrier to hydride transfer in yeast alcohol dehydrogenase (YADH) and human heart lactate dehydrogenase (LDH). Calculation of the work applied to the hydride allowed for observation of the change in barrier height upon inclusion of quantum dynamics. Similar calculations were performed using deuterium as the transferring particle in order to approximate kinetic isotope effects (KIEs). The change in barrier height in YADH is indicative of a zero-point energy (ZPE) contribution and is evidence that catalysis occurs via a protein compression that mediates a near-barrierless hydride transfer. Calculation of the KIE using the difference in barrier height between the hydride and deuteride agreed well with experimental results.</td>
</tr>
<tr id="bib_Dzierlenga2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dzierlenga2015,
  author = {Dzierlenga, Michael W and Antoniou, Dimitri and Schwartz, Steven D},
  title = {Another look at the mechanisms of hydride transfer enzymes with quantum and classical transition path sampling},
  journal = {Journal of Physical Chemistry Letters},
  year = {2015},
  volume = {6},
  number = {7},
  pages = {1177 - 1181},
  note = {NULL},
  doi = {https://doi.org/10.1021/acs.jpclett.5b00346}
}
</pre></td>
</tr>
<tr id="Dzierlenga2016" class="entry">
	<td>Dzierlenga, M.W., Varga, M.J. and Schwartz, S.D.</td>
	<td>Chapter Two -- Path sampling methods for enzymatic quantum particle transfer reactions <p class="infolinks">[<a href="javascript:toggleInfo('Dzierlenga2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Dzierlenga2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Methods in Enzymology<br/>Vol. 578, pp. 21 - 43&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/bs.mie.2016.05.028">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Dzierlenga2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The mechanisms of enzymatic reactions are studied via a host of computational techniques. While previous methods have been used successfully, many fail to incorporate the full dynamical properties of enzymatic systems. This can lead to misleading results in cases where enzyme motion plays a significant role in the reaction coordinate, which is especially relevant in particle transfer reactions where nuclear tunneling may occur. In this chapter, we outline previous methods, as well as discuss newly developed dynamical methods to interrogate mechanisms of enzymatic particle transfer reactions. These new methods allow for the calculation of free energy barriers and kinetic isotope effects (KIEs) with the incorporation of quantum effects through centroid molecular dynamics (CMD) and the full complement of enzyme dynamics through transition path sampling (TPS). Recent work, summarized in this chapter, applied the method for calculation of free energy barriers to reaction in lactate dehydrogenase (LDH) and yeast alcohol dehydrogenase (YADH). We found that tunneling plays an insignificant role in YADH but plays a more significant role in LDH, though not dominant over classical transfer. Additionally, we summarize the application of a TPS algorithm for the calculation of reaction rates in tandem with CMD to calculate the primary H/D KIE of YADH from first principles. We found that the computationally obtained KIE is within the margin of error of experimentally determined KIEs and corresponds to the KIE of particle transfer in the enzyme. These methods provide new ways to investigate enzyme mechanism with the inclusion of protein and quantum dynamics.</td>
</tr>
<tr id="bib_Dzierlenga2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dzierlenga2016,
  author = {Dzierlenga, Michael W and Varga, Matthew J and Schwartz, Steven D},
  title = {Chapter Two -- Path sampling methods for enzymatic quantum particle transfer reactions},
  journal = {Methods in Enzymology},
  year = {2016},
  volume = {578},
  pages = {21 - 43},
  note = {NULL},
  doi = {https://doi.org/10.1016/bs.mie.2016.05.028}
}
</pre></td>
</tr>
<tr id="Edeling2006" class="entry">
	<td>Edeling, M.A., Mishra, S.K., Keyel, P.A., Steinhauser, A.L., Collins, B.M., Roth, R., Heuser, J.E., Owen, D.J. and Traub, L.M.</td>
	<td>Molecular switches involving the AP-2 &beta;2 appendage regulate endocytic cargo selection and clathrin coat assembly. <p class="infolinks">[<a href="javascript:toggleInfo('Edeling2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Edeling2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Developmental cell<br/>Vol. 10, pp. 329-342&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.devcel.2006.01.016">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Edeling2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-associated sorting proteins (CLASPs) expand the repertoire of endocytic cargo sorted into clathrin-coated vesicles beyond the transmembrane proteins that bind physically to the AP-2 adaptor. LDL and GPCRs are internalized by ARH and beta-arrestin, respectively. We show that these two CLASPs bind selectively to the AP-2 beta2 appendage platform via an alpha-helical [DE](n)X(1-2)FXX[FL]XXXR motif, and that this motif also occurs and is functional in the epsins. In beta-arrestin, this motif maintains the endocytosis-incompetent state by binding back on the folded core of the protein in a beta strand conformation. Triggered via a beta-arrestin/GPCR interaction, the motif must be displaced and must undergo a strand to helix transition to enable the beta2 appendage binding that drives GPCR-beta-arrestin complexes into clathrin coats. Another interaction surface on the beta2 appendage sandwich is identified for proteins such as eps15 and clathrin, suggesting a mechanism by which clathrin displaces eps15 to lattice edges during assembly.</td>
</tr>
<tr id="bib_Edeling2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Edeling2006,
  author = {Edeling, Melissa A and Mishra, Sanjay K and Keyel, Peter A and Steinhauser, Amie L and Collins, Brett M and Roth, Robyn and Heuser, John E and Owen, David J and Traub, Linton M},
  title = {Molecular switches involving the AP-2 &beta;2 appendage regulate endocytic cargo selection and clathrin coat assembly.},
  journal = {Developmental cell},
  year = {2006},
  volume = {10},
  pages = {329--342},
  note = {beta2 interacts with eps15, amph I and II, ap180, calm, cltc. uses brain cells. domain interaction information.},
  doi = {https://doi.org/10.1016/j.devcel.2006.01.016}
}
</pre></td>
</tr>
<tr id="Edeling2006a" class="entry">
	<td>Edeling, M.A., Smith, C. and Owen, D.</td>
	<td>Life of a clathrin coat: insights from clathrin and AP structures <p class="infolinks">[<a href="javascript:toggleInfo('Edeling2006a','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Nature reviews. Molecular cell biology<br/>Vol. 7(1), pp. 32&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Edeling2006a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Edeling2006a,
  author = {Edeling, Melissa A and Smith, Corinne and Owen, David},
  title = {Life of a clathrin coat: insights from clathrin and AP structures},
  journal = {Nature reviews. Molecular cell biology},
  publisher = {Nature Publishing Group},
  year = {2006},
  volume = {7},
  number = {1},
  pages = {32}
}
</pre></td>
</tr>
<tr id="Efron1993" class="entry">
	<td>Efron, B. and Tibshirani, R.</td>
	<td>An Introduction to the Bootstrap <p class="infolinks">[<a href="javascript:toggleInfo('Efron1993','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Efron1993','bibtex')">BibTeX</a>]</p></td>
	<td>1993</td>
	<td>Teach Statistics<br/>Vol. 57(2), pp. 436 - 436&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1111/1467-9639.00050">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Efron1993" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Statistics is a subject of many uses and surprisingly few effective practitioners. The traditional road to statistical knowledge is blocked, for most, by a formidable wall of mathematics. The approach in An Introduction to the Bootstrap avoids that wall. It arms scientists and engineers, as well as statisticians, with the computational techniques they need to analyze and understand complicated data sets.</td>
</tr>
<tr id="bib_Efron1993" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Efron1993,
  author = {Efron, Bradley. and Tibshirani, RJ},
  title = {An Introduction to the Bootstrap},
  journal = {Teach Statistics},
  year = {1993},
  volume = {57},
  number = {2},
  pages = {436 - 436},
  doi = {https://doi.org/10.1111/1467-9639.00050}
}
</pre></td>
</tr>
<tr id="Ehrlich2004" class="entry">
	<td>Ehrlich, M., Boll, W., Van Oijen, A., Hariharan, R., Chandran, K., Nibert, M.L. and Kirchhausen, T.</td>
	<td>Endocytosis by random initiation and stabilization of clathrin-coated pits. <p class="infolinks">[<a href="javascript:toggleInfo('Ehrlich2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ehrlich2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Cell<br/>Vol. 118, pp. 591-605&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cell.2004.08.017">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ehrlich2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-coated vesicles carry traffic from the plasma membrane to endosomes. We report here the real-time visualization of cargo sorting and endocytosis by clathrin-coated pits in living cells. We have detected the formation of coats by monitoring incorporation of fluorescently tagged clathrin or its adaptor AP-2; we have also followed clathrin-mediated uptake of transferrin and of single LDL or reovirus particles. The intensity of a cargo-loaded clathrin cluster grows steadily during its lifetime, and the time required to complete assembly is proportional to the size of the cargo particle. These results are consistent with a nucleation-growth mechanism and an approximately constant growth rate. There are no strongly preferred nucleation sites. A proportion of the nucleation events are weak and short lived. Cargo incorporation occurs primarily or exclusively in a newly formed coated pit. Our data lead to a model in which coated pits initiate randomly but collapse unless stabilized, perhaps by cargo capture.</td>
</tr>
<tr id="bib_Ehrlich2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ehrlich2004,
  author = {Ehrlich, Marcelo and Boll, Werner and Van Oijen, Antoine and Hariharan, Ramesh and Chandran, Kartik and Nibert, Max L and Kirchhausen, Tomas},
  title = {Endocytosis by random initiation and stabilization of clathrin-coated pits.},
  journal = {Cell},
  year = {2004},
  volume = {118},
  pages = {591--605},
  note = {mean lifetime of a coated vesicle is &nbsp;32s},
  doi = {https://doi.org/10.1016/j.cell.2004.08.017}
}
</pre></td>
</tr>
<tr id="Eichinger1999" class="entry">
	<td>Eichinger, M., Tavan, P., Hutter, J. and Parrinello, M.</td>
	<td>A hybrid method for solutes in complex solvents: Density functional theory combined with empirical force fields <p class="infolinks">[<a href="javascript:toggleInfo('Eichinger1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Eichinger1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>The Journal of Chemical Physics<br/>Vol. 110(21), pp. 10452 - 10452&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.479049">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Eichinger1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present a hybrid method for molecular dynamics simulations of solutes in complex solvents as represented, for example, by substrates within enzymes. The method combines a quantum mechanical (QM) description of the solute with a molecular mechanics (MM) approach for the solvent. The QM fragment of a simulation system is treated by ab initiodensity functional theory(DFT) based on plane-wave expansions. Long-range Coulomb interactions within the MM fragment and between the QM and the MM fragment are treated by a computationally efficient fast multipole method. For the description of covalent bonds between the two fragments, we introduce the scaled position link atom method (SPLAM), which removes the shortcomings of related procedures. The various aspects of the hybrid method are scrutinized through test calculations on liquid water, the water dimer, ethane and a small molecule related to the retinal Schiff base. In particular, the extent to which vibrational spectra obtained by DFT for the solute can be spoiled by the lower quality force field of the solvent is checked, including cases in which the two fragments are covalently joined. The results demonstrate that our QM/MM hybrid method is especially well suited for the vibrational analysis of molecules in condensed phase.</td>
</tr>
<tr id="bib_Eichinger1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Eichinger1999,
  author = {Eichinger, M and Tavan, P and Hutter, J and Parrinello, M},
  title = {A hybrid method for solutes in complex solvents: Density functional theory combined with empirical force fields},
  journal = {The Journal of Chemical Physics},
  year = {1999},
  volume = {110},
  number = {21},
  pages = {10452 - 10452},
  note = {Parinello method for DFT in CHARMM},
  doi = {https://doi.org/10.1063/1.479049}
}
</pre></td>
</tr>
<tr id="Eisenberg2007" class="entry">
	<td>Eisenberg, E. and Greene, L.E.</td>
	<td>Multiple Roles of Auxilin and Hsc70 in Clathrin-Mediated Endocytosis <p class="infolinks">[<a href="javascript:toggleInfo('Eisenberg2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Traffic<br/>Vol. 8, pp. 640-646&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1111/j.1600-0854.2007.00568.x">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Eisenberg2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Eisenberg2007,
  author = {Eisenberg, Evan and Greene, Lois E.},
  title = {Multiple Roles of Auxilin and Hsc70 in Clathrin-Mediated Endocytosis},
  journal = {Traffic},
  year = {2007},
  volume = {8},
  pages = {640-646},
  note = {Hsc70 and auxilin may be involved in clathrin exchange during invagination.},
  doi = {https://doi.org/10.1111/j.1600-0854.2007.00568.x}
}
</pre></td>
</tr>
<tr id="Elber1990" class="entry">
	<td>Elber, R. and Karplus, M.</td>
	<td>Enhanced sampling in molecular dynamics: use of the time-dependent Hartree approximation for a simulation of carbon monoxide diffusion through myoglobin <p class="infolinks">[<a href="javascript:toggleInfo('Elber1990','bibtex')">BibTeX</a>]</p></td>
	<td>1990</td>
	<td>Journal of the American Chemical Society<br/>Vol. 112(25), pp. 9161 - 9175&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ja00181a020">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Elber1990" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Elber1990,
  author = {Elber, R and Karplus, M},
  title = {Enhanced sampling in molecular dynamics: use of the time-dependent Hartree approximation for a simulation of carbon monoxide diffusion through myoglobin},
  journal = {Journal of the American Chemical Society},
  year = {1990},
  volume = {112},
  number = {25},
  pages = {9161 - 9175},
  note = {NULL},
  doi = {https://doi.org/10.1021/ja00181a020}
}
</pre></td>
</tr>
<tr id="Elf2004" class="entry">
	<td>Elf, J. and Ehrenberg, M.</td>
	<td>Spontaneous separation of bi-stable biochemical systems into spatial domains of opposite phases. <p class="infolinks">[<a href="javascript:toggleInfo('Elf2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Elf2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Systems biology<br/>Vol. 1, pp. 230-236&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Elf2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Bi-stable chemical systems are the basic building blocks for intracellular memory and cell fate decision circuits. These circuits are built from molecules, which are present at low copy numbers and are slowly diffusing in complex intracellular geometries. The stochastic reaction-diffusion kinetics of a double-negative feedback system and a MAPK phosphorylation-dephosphorylation system is analysed with Monte-Carlo simulations of the reaction-diffusion master equation. The results show the geometry of intracellular reaction compartments to be important both for the duration and the locality of biochemical memory. Rules for when the systems lose global hysteresis by spontaneous separation into spatial domains in opposite phases are formulated in terms of geometrical constraints, diffusion rates and attractor escape times. The analysis is facilitated by a new efficient algorithm for exact sampling of the Markov process corresponding to the reaction-diffusion master equation.</td>
</tr>
<tr id="bib_Elf2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Elf2004,
  author = {Elf, J and Ehrenberg, M},
  title = {Spontaneous separation of bi-stable biochemical systems into spatial domains of opposite phases.},
  journal = {Systems biology},
  year = {2004},
  volume = {1},
  pages = {230--236}
}
</pre></td>
</tr>
<tr id="Elstner1998" class="entry">
	<td>Elstner, M., Porezag, D., Jungnickel, G., Elsner, J., Haugk, M., Frauenheim, T., Suhai, S. and Seifert, G.</td>
	<td>Self-consistent-charge density-functional tight-binding method for simulations of complex materials properties <p class="infolinks">[<a href="javascript:toggleInfo('Elstner1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Physical Review B<br/>Vol. 58(11), pp. 7260 - 7268&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1103/PhysRevB.58.7260">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Elstner1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Elstner1998,
  author = {Elstner, M and Porezag, D and Jungnickel, G and Elsner, J and Haugk, M and Frauenheim, Th. and Suhai, S and Seifert, G},
  title = {Self-consistent-charge density-functional tight-binding method for simulations of complex materials properties},
  journal = {Physical Review B},
  year = {1998},
  volume = {58},
  number = {11},
  pages = {7260 - 7268},
  note = {First SCC-DFT Paper},
  doi = {https://doi.org/10.1103/PhysRevB.58.7260}
}
</pre></td>
</tr>
<tr id="Endoh2018" class="entry">
	<td>Endoh, K.S., Kawakatsu, T. and M&uuml;ller-Plathe, F.</td>
	<td>A Coarse-Grained Molecular Simulation Model for Gecko-Feet Keratin <p class="infolinks">[<a href="javascript:toggleInfo('Endoh2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>The Journal of Physical Chemistry B&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Endoh2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Endoh2018,
  author = {Endoh, Kenkoh S and Kawakatsu, Toshihiro and M&uuml;ller-Plathe, Florian},
  title = {A Coarse-Grained Molecular Simulation Model for Gecko-Feet Keratin},
  journal = {The Journal of Physical Chemistry B},
  publisher = {ACS Publications},
  year = {2018}
}
</pre></td>
</tr>
<tr id="Engh1991" class="entry">
	<td>Engh, R. and Huber, R.</td>
	<td>Accurate bond and angle parameters for X-ray protein structure refinement <p class="infolinks">[<a href="javascript:toggleInfo('Engh1991','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Engh1991','bibtex')">BibTeX</a>]</p></td>
	<td>1991</td>
	<td>Acta Crystallographica Section A Foundations of Crystallography<br/>Vol. 47(4), pp. 392 - 400&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1107/S0108767391001071">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Engh1991" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Bond-length and bond-angle parameters are derived from a statistical survey of X-ray structures of small compounds from the Cambridge Structural Database. The side chains of the common amino acids and the polypeptide backbone were represented by appropriate chemical fragments taken from the Database. Average bond lengths and bond angles are determined from the resulting samples and the sample standard deviations provide information regarding the expected variability of the average values which can be parametrized as force constants. These parameters are ideally suited for the refinement of protein structures determined by X-ray crystallography since they are derived from X-ray structures, are accurate to within the deviations from target values suggested for X-ray structure refinement and use force constants which directly reflect the variability or uncertainty of the average values. Tests of refinement of the structures of BPTI and phycocyanin demonstrate the integrity of the parameters and comparisons of equivalent refinements with XPLOR parameters show improvement in R-factors and geometry statistics.</td>
</tr>
<tr id="bib_Engh1991" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Engh1991,
  author = {Engh, RA and Huber, R},
  title = {Accurate bond and angle parameters for X-ray protein structure refinement},
  journal = {Acta Crystallographica Section A Foundations of Crystallography},
  year = {1991},
  volume = {47},
  number = {4},
  pages = {392 - 400},
  note = {Standard bond lengths and angles from x-ray crystallography for SASA},
  doi = {https://doi.org/10.1107/S0108767391001071}
}
</pre></td>
</tr>
<tr id="Erdemir2017" class="entry">
	<td>Erdemir, A. and Mutlu, O.</td>
	<td>Functional and structural characterization of the pentapeptide insertion of Theileria annulata lactate dehydrogenase by site-directed mutagenesis, comparative modeling and molecular dynamics simulations <p class="infolinks">[<a href="javascript:toggleInfo('Erdemir2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Erdemir2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of Molecular Graphics and Modelling<br/>Vol. 74, pp. 344 - 351&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.jmgm.2017.04.010">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Erdemir2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: a b s t r a c t Lactate dehydrogenase (LDH) is an important metabolic enzyme in glycolysis and it has been considered as the main energy source in many organisms including apicomplexan parasites. Differences at the active site loop of the host and parasite LDH's makes this enzyme an attractive target for drug inhibitors. In this study, five amino acid insertions in the active site pocket of Theileria annulata LDH (TaLDH) were deleted by PCR-based site-directed mutagenesis, expression and activity analysis of mutant and wild type TaLDH enzymes were performed. Removal of the insertion at the active site loop caused production of an inactive enzyme. Furthermore, structures of wild and mutant enzymes were predicted by comparative modeling and the importance of the insertions at the active site loop were also assigned by molecular docking and dynamics simulations in order to evaluate essential role of this loop for the enzymatic activity. Pentapeptide insertion removal resulted in loss of LDH activity due to deletion of Trp96 and conformational change of Arg98 because of loop instability. Analysis of wild type and mutant enzymes with comparative molecular dynamics simulations showed that the fluctuations of the loop residues increase in mutant enzyme. Together with in silico studies, in vitro results revealed that active site loop has a vital role in the enzyme activity and our findings promise hope for the further drug design studies against theileriosis and other apicomplexan parasite diseases.</td>
</tr>
<tr id="bib_Erdemir2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Erdemir2017,
  author = {Erdemir, Aysegul and Mutlu, Ozal},
  title = {Functional and structural characterization of the pentapeptide insertion of Theileria annulata lactate dehydrogenase by site-directed mutagenesis, comparative modeling and molecular dynamics simulations},
  journal = {Journal of Molecular Graphics and Modelling},
  year = {2017},
  volume = {74},
  pages = {344 - 351},
  doi = {https://doi.org/10.1016/j.jmgm.2017.04.010}
}
</pre></td>
</tr>
<tr id="Erp2016" class="entry">
	<td>Erp, T.S., Moqadam, M., Riccardi, E. and Lervik, A.</td>
	<td>Analyzing Complex Reaction Mechanisms Using Path Sampling <p class="infolinks">[<a href="javascript:toggleInfo('Erp2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Erp2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Journal of Chemical Theory and Computation<br/>Vol. 12(11), pp. 5398 - 5410&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jctc.6b00642">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Erp2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We introduce an approach to analyze collective variables (CVs) regarding their predictive power for a reaction. The method is based on already available path sampling data produced by, for instance, transition interface sampling or forward flux sampling, which are path sampling methods used for efficient computation of reaction rates. By a search in CV space, a measure of predictiveness can be optimized and, in addition, the number of CVs can be reduced using projection operations which keep this measure invariant. The approach allows testing hypotheses on the reaction mechanism but could, in principle, also be used to construct the phase-space committor surfaces without the need of additional trajectory sampling. The procedure is illustrated for a one-dimensional double-well potential, a theoretical model for an ion-transfer reaction in which the solvent structure can lower the barrier, and an ab initio molecular dynamics study of water auto-ionization. The analysis technique enhances the quantitative interpretation of path sampling data which can provide clues on how chemical reactions can be steered in desired directions.</td>
</tr>
<tr id="bib_Erp2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Erp2016,
  author = {Erp, Titus S and Moqadam, Mahmoud and Riccardi, Enrico and Lervik, Anders},
  title = {Analyzing Complex Reaction Mechanisms Using Path Sampling},
  journal = {Journal of Chemical Theory and Computation},
  year = {2016},
  volume = {12},
  number = {11},
  pages = {5398 - 5410},
  doi = {https://doi.org/10.1021/acs.jctc.6b00642}
}
</pre></td>
</tr>
<tr id="Eurenius1996" class="entry">
	<td>Eurenius, K.P., Chatfield, D.C., Brooks, B.R. and Hodoscek, M.</td>
	<td>Enzyme mechanisms with hybrid quantum and molecular mechanical potentials. I. Theoretical considerations <p class="infolinks">[<a href="javascript:toggleInfo('Eurenius1996','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Eurenius1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>Int. J. Quant. Chem.<br/>Vol. 60(6), pp. 1189 - 1200&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/(SICI)1097-461X(1996)60">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Eurenius1996" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The application of hybrid quantum mechanical and molecular mechanical (QM/MM) potentials to the study of chemical reactions in enzymes is outlined. The discussion is general and addresses the difficulties encountered in an enzyme QM/MM study. First, general criteria for determining whether a particular enzyme is an appropriate candidate for a QM/MM approach are outlined. Methods for obtaining starting structures are detailed. The importance of choosing appropriate levels of ab initio or semiempirical theory is emphasized. Approaches for interfacing the QM and MM regions are briefly discussed, with greater detail given to describing our CHARMM-GAMESS interface. Techniques for partitioning the system into QM and MM regions are explored. Link atom placement, as distant from reacting atoms as possible within the confines of computational efficiency, is examined in some detail. Methods for determining reaction paths are also discussed. textcopyright 1996 John Wiley &amp; Sons, Inc.</td>
</tr>
<tr id="bib_Eurenius1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Eurenius1996,
  author = {Eurenius, Kirsten P and Chatfield, David C and Brooks, Bernard R and Hodoscek, Milan},
  title = {Enzyme mechanisms with hybrid quantum and molecular mechanical potentials. I. Theoretical considerations},
  journal = {Int. J. Quant. Chem.},
  year = {1996},
  volume = {60},
  number = {6},
  pages = {1189 - 1200},
  doi = {https://doi.org/10.1002/(SICI)1097-461X(1996)60}
}
</pre></td>
</tr>
<tr id="Eventoff1977" class="entry">
	<td>Eventoff, W., Rossmann, M.G., Taylor, S.S., Torff, Hans-Joachim., Meyer, H., Keil, W. and Hans-Hermann, K.</td>
	<td>Structural adaptations of lactate dehydrogenase isozymes <p class="infolinks">[<a href="javascript:toggleInfo('Eventoff1977','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Eventoff1977','bibtex')">BibTeX</a>]</p></td>
	<td>1977</td>
	<td>Proceedings of the National Acadamy of Sciences<br/>Vol. 74(7), pp. 2677 - 2681&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cell.2012.10.020">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Eventoff1977" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Glucocorticoids act via the glucocorticoid receptor (GR). Upon ligand binding, GR translocates to the nucleus and binds directly to glucocorticoid responsive elements (GREs) to regulate transcriptional output. Glucocorticoid secretion increases in response to stress to affect transcriptional output within specific areas of the brain including the hippocampus (HC). Therefore, here we assess changes in genome-wide GR chromatin binding profiles in the rat HC in response to a glucocorticoid rise, with or without an acute stress in order to identify significant motifs at the sites of GR binding. Male ADX Sprague Dawley rats were sacrificed after a 30 min glucocorticoid infusion with/without a 30 min restraint stress and HC dissected for chromatin immunoprecipitation assays (ChIP) using a GR antibody cocktail. GR binding sites and DNA motifs were identified, including the previously characterised Per1 regulatory sites. Novel sites were identified proximal to Bdnf, SGK1 and Camk2a. Interestingly, there was no significant difference in peak tag density between basal and stress groups. HOMER analysis identified motifs for GRE, NF1 and NeuroD1 as top recurring motifs at 53, 62 and 18&#37; respectively. The motifs identified in this study represent targets for further study. NeuroD1 in particular is thought to be involved in learning and memory. Therefore NeuroD1 is an interesting target for future analysis of GC-regulated HC function. BDNF and Camk2a mRNA levels have previously been described as regulated in response to stress. Interestingly our findings have shown no change at the point of GR binding to these GREs and little change to the genome-wide chromatin binding profile in response to acute stress compared to increased glucocorticoids in the absence of a concomitant stressor. These findings suggests that glucocorticoid dependent stress regulation in the HC is primarily regulated by the glucocorticoid rise, and not further modulated other stress associated factors in our ADX rat model.</td>
</tr>
<tr id="bib_Eventoff1977" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Eventoff1977,
  author = {Eventoff, William and Rossmann, Michael G and Taylor, Susan S and Torff, Hans-Joachim and Meyer, Helmut and Keil, Walter and Hans-Hermann, Kiltz},
  title = {Structural adaptations of lactate dehydrogenase isozymes},
  journal = {Proceedings of the National Acadamy of Sciences},
  year = {1977},
  volume = {74},
  number = {7},
  pages = {2677 - 2681},
  doi = {https://doi.org/10.1016/j.cell.2012.10.020}
}
</pre></td>
</tr>
<tr id="Eyring1935" class="entry">
	<td>Eyring, H.</td>
	<td>The Activated Complex in Chemical Reactions <p class="infolinks">[<a href="javascript:toggleInfo('Eyring1935','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Eyring1935','bibtex')">BibTeX</a>]</p></td>
	<td>1935</td>
	<td>The Journal of Chemical Physics<br/>Vol. 3(2), pp. 107 - 115&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.1749604">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Eyring1935" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The calculation of absolute reaction rates is formulated in terms of quantities which are available from the potential surfaces which can be constructed at the present time. The probability of the activated state is calculated using ordinary statistical mechanics. This probability multiplied by the rate of decomposition gives the specific rate of reaction. The occurrence of quantized vibrations in the activated complex, in degrees of freedom which are unquantized in the original molecules, leads to relative reaction rates for isotopes quite different from the rates predicted using simple kinetic theory. The necessary conditions for the general statistical treatment to reduce to the usual kinetic treatment are given.</td>
</tr>
<tr id="bib_Eyring1935" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Eyring1935,
  author = {Eyring, Henry},
  title = {The Activated Complex in Chemical Reactions},
  journal = {The Journal of Chemical Physics},
  year = {1935},
  volume = {3},
  number = {2},
  pages = {107 - 115},
  doi = {https://doi.org/10.1063/1.1749604}
}
</pre></td>
</tr>
<tr id="Faeder2005" class="entry">
	<td>Faeder, J.R., Blinov, M.L., Goldstein, B. and Hlavacek, W.S.</td>
	<td>Rule-based modeling of biochemical networks <p class="infolinks">[<a href="javascript:toggleInfo('Faeder2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Faeder2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Complexity<br/>Vol. 10(4), pp. 22 - 41&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/cplx.20074">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Faeder2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present a method for generating a biochemical reaction network from a description of the interactions of components of biomolecules. The interactions are specified in the form of reaction rules, each of which defines a class of reaction associated with a type of interaction. Reactants within a class have shared properties, which are specified in the rule defining the class. A rule also provides a rate law, which governs each reaction in a class, and a template for transforming reactants into products. A set of reaction rules can be applied to a seed set of chemical species and, subsequently, any new species that are found as products of reactions to generate a list of reactions and a list of the chemical species that participate in these reactions, i.e., a reaction network, which can be translated into a mathematical model. textcopyright 2005 Wiley Periodicals, Inc. Complexity 10: 22--41, 2005</td>
</tr>
<tr id="bib_Faeder2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Faeder2005,
  author = {Faeder, James R and Blinov, Michael L and Goldstein, Byron and Hlavacek, William S},
  title = {Rule-based modeling of biochemical networks},
  journal = {Complexity},
  year = {2005},
  volume = {10},
  number = {4},
  pages = {22 - 41},
  doi = {https://doi.org/10.1002/cplx.20074}
}
</pre></td>
</tr>
<tr id="Fagerberg2014" class="entry">
	<td>Fagerberg, L., Hallström, B.M., Oksvold, P., Kampf, C., Djureinovic, D., Odeberg, J., Habuka, M., Tahmasebpoor, S., Danielsson, A., Edlund, K., Asplund, A., Sjöstedt, E., Lundberg, E., Szigyarto, C.A.-K., Skogs, M., Takanen, J.O., Berling, H., Tegel, H., Mulder, J., Nilsson, P., Schwenk, J.M., Lindskog, C., Danielsson, F., Mardinoglu, A., Sivertsson, A., von Feilitzen, K., Forsberg, M., Zwahlen, M., Olsson, I., Navani, S., Huss, M., Nielsen, J., Ponten, F. and Uhlén, M.</td>
	<td>Analysis of the human tissue-specific expression by genome-wide integration of transcriptomics and antibody-based proteomics. <p class="infolinks">[<a href="javascript:toggleInfo('Fagerberg2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Fagerberg2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Molecular &amp; cellular proteomics : MCP<br/>Vol. 13, pp. 397-406&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/mcp.M113.035600">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Fagerberg2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Global classification of the human proteins with regards to spatial expression patterns across organs and tissues is important for studies of human biology and disease. Here, we used a quantitative transcriptomics analysis (RNA-Seq) to classify the tissue-specific expression of genes across a representative set of all major human organs and tissues and combined this analysis with antibody-based profiling of the same tissues. To present the data, we launch a new version of the Human Protein Atlas that integrates RNA and protein expression data corresponding to ∼80% of the human protein-coding genes with access to the primary data for both the RNA and the protein analysis on an individual gene level. We present a classification of all human protein-coding genes with regards to tissue-specificity and spatial expression pattern. The integrative human expression map can be used as a starting point to explore the molecular constituents of the human body.</td>
</tr>
<tr id="bib_Fagerberg2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Fagerberg2014,
  author = {Fagerberg, Linn and Hallström, Björn M and Oksvold, Per and Kampf, Caroline and Djureinovic, Dijana and Odeberg, Jacob and Habuka, Masato and Tahmasebpoor, Simin and Danielsson, Angelika and Edlund, Karolina and Asplund, Anna and Sjöstedt, Evelina and Lundberg, Emma and Szigyarto, Cristina Al-Khalili and Skogs, Marie and Takanen, Jenny Ottosson and Berling, Holger and Tegel, Hanna and Mulder, Jan and Nilsson, Peter and Schwenk, Jochen M and Lindskog, Cecilia and Danielsson, Frida and Mardinoglu, Adil and Sivertsson, Asa and von Feilitzen, Kalle and Forsberg, Mattias and Zwahlen, Martin and Olsson, IngMarie and Navani, Sanjay and Huss, Mikael and Nielsen, Jens and Ponten, Fredrik and Uhlén, Mathias},
  title = {Analysis of the human tissue-specific expression by genome-wide integration of transcriptomics and antibody-based proteomics.},
  journal = {Molecular &amp; cellular proteomics : MCP},
  year = {2014},
  volume = {13},
  pages = {397--406},
  note = {expression levels of genes by tissue type. AMPH highly enriched in brain over ovaries (hela)},
  doi = {https://doi.org/10.1074/mcp.M113.035600}
}
</pre></td>
</tr>
<tr id="Fan1991" class="entry">
	<td>Fan, F., a Lorenzen, J. and Plapp, B.</td>
	<td>An aspartate residue in yeast alcohol dehydrogenase I determines the specificity for coenzyme. <p class="infolinks">[<a href="javascript:toggleInfo('Fan1991','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Fan1991','bibtex')">BibTeX</a>]</p></td>
	<td>1991</td>
	<td>Biochemistry<br/>Vol. 30(26), pp. 6397 - 401&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Fan1991" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In the three-dimensional structures of enzymes that bind NAD or FAD, there is an acidic residue that interacts with the 2'- and 3'-hydroxyl groups of the adenosine ribose of the coenzyme. The size and charge of the carboxylate might repel the binding of the 2'-phosphate group of NADP and explain the specificity for NAD. In the NAD-dependent alcohol dehydrogenases, Asp-223 (horse liver alcohol dehydrogenase sequence) appears to have this role. The homologous residue in yeast alcohol dehydrogenase I (residue 201 in the protein sequence) was substituted with Gly, and the D223G enzyme was expressed in yeast, purified, and characterized. The wild-type enzyme is specific for NAD. In contrast, the D223G enzyme bound and reduced NAD+ and NADP+ equally well, but, relative to wild-type enzyme, the dissociation constant for NAD+ was increased 17-fold, and the reactivity (V/K) on ethanol was decreased to 1%. Even though catalytic efficiency was reduced, yeast expressing the altered or wild-type enzyme grew at comparable rates, suggesting that equilibration of NAD and NADP pools is not lethal. Asp-223 participates in binding NAD and in excluding NADP, but it is not the only residue important for determining specificity for coenzyme.</td>
</tr>
<tr id="bib_Fan1991" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Fan1991,
  author = {Fan, F and a Lorenzen, J and Plapp, BV},
  title = {An aspartate residue in yeast alcohol dehydrogenase I determines the specificity for coenzyme.},
  journal = {Biochemistry},
  year = {1991},
  volume = {30},
  number = {26},
  pages = {6397 - 401}
}
</pre></td>
</tr>
<tr id="Fay2003" class="entry">
	<td>Fay, J.C. and Wu, C.</td>
	<td>Sequence divergence, functional constraint, and selection in protein evolution <p class="infolinks">[<a href="javascript:toggleInfo('Fay2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Fay2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Annual Review of Genomics and Human Genetics<br/>Vol. 4(1), pp. 213 - 235&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1146/annurev.genom.4.020303.162528">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Fay2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The genome sequences of multiple species has enabled functional inferences from comparative genomics. A primary objective is to infer biological functions from the conservation of homologous DNA sequences between species. A second, more difficult, objective is to understand what functional DNA sequences have changed over time and are responsible for species' phenotypic differences. The neutral theory of molecular evolution provides a theoretical framework in which both objectives can be explicitly tested. Development of statistical tests within this framework has provided insight into the evolutionary forces that constrain and in some cases change DNA sequences and the resulting patterns that emerge. In this article, we review recent work on how functional constraint and changes in protein function are inferred from protein polymorphism and divergence data. We relate these studies to our understanding of the neutral theory and adaptive evolution.</td>
</tr>
<tr id="bib_Fay2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Fay2003,
  author = {Fay, Justin C and Wu, CI},
  title = {Sequence divergence, functional constraint, and selection in protein evolution},
  journal = {Annual Review of Genomics and Human Genetics},
  year = {2003},
  volume = {4},
  number = {1},
  pages = {213 - 235},
  doi = {https://doi.org/10.1146/annurev.genom.4.020303.162528}
}
</pre></td>
</tr>
<tr id="Feng2018" class="entry">
	<td>Feng, J. and Shukla, D.</td>
	<td>Characterizing Conformational Dynamics of Proteins Using Evolutionary Couplings <p class="infolinks">[<a href="javascript:toggleInfo('Feng2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>The Journal of Physical Chemistry B&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Feng2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Feng2018,
  author = {Feng, Jiangyan and Shukla, Diwakar},
  title = {Characterizing Conformational Dynamics of Proteins Using Evolutionary Couplings},
  journal = {The Journal of Physical Chemistry B},
  publisher = {ACS Publications},
  year = {2018}
}
</pre></td>
</tr>
<tr id="Ferrenberg1989" class="entry">
	<td>Ferrenberg, A. and Swendsen, R.</td>
	<td>Optimised monte carlo data analysis <p class="infolinks">[<a href="javascript:toggleInfo('Ferrenberg1989','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ferrenberg1989','bibtex')">BibTeX</a>]</p></td>
	<td>1989</td>
	<td>Physical Review Letters<br/>Vol. 63(12), pp. 1195 - 1198&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1103/PhysRevLett.63.1195">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ferrenberg1989" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present a new method for optimizing the analysis of data from multiple Monte Carlo computer simulations over wide ranges of parameter values. Explicit error estimates allow objective planning of the lengths of runs and the parameter values to be simulated. The method is applicable to simulations in lattice gauge theories, chemistry, and biology, as well as statistical mechanics. PACS numbers: 05.50,+q, 64.60.Fr, 75.10.Hk Recently, we showed that histograms can be used to greatly increase the amount of information obtained from a single computer simulation in the neighborhood of a critical point. ' In particular, we demonstrated that the location and height of maxima and minima can be determined with higher accuracy and much less comput-er time than previously obtainable. We also noted that the region of validity of the single histogram method coincides with the finite-size scaling region, so that infor-mation about the critical region does not deteriorate with increasing system size. On the other hand, for more general problems it is often desired to investigate the behavior of the system over a wider range of parameter values. In this situa-tion, it is necessary to perform simulations at more than one value of the parameters of interest. This paper presents an optimized method for combin-ing the data from an arbitrary number of simulations to obtain information over a wide range of parameter values in the form of continuous functions. The method goes beyond earlier methods in that it provides an opti-mized combination of data from different sources, and can be applied to an arbitrary number of simulations. Errors can be calculated and provide a clear and simple guide to optimizing the length and location of additional simulations to provide maximum accuracy. It is possible to extend the temperature range to gen-erate the free energy and entropy from zero to infinite temperature. Finally, our method can be used with any simulation method that provides data for a system in equilibrium, and it requires a negligible amount of additional comput-er time for its implementation. The method is therefore applicable to simulations in lattice gauge theories, chemistry, and biology, as well as statistical mechan-1cs. We will first describe the method and then demon-strate its e%ciency by calculating the properties of an I. 16, d 2 Ising model for the full range of tempera-tures. Consider the general Hamiltonian where S(o) is an operator (energy, magnetization, etc.) defined on the spins [a;J, and factors of --- I/kttT have been absorbed. We are interested in the behavior of the system as a function of K. For simplicity, we will consid-er only one parameter EC, but the generalization to an ar-bitrary number of parameters is straightforward. The partition function is given by Z(K) g exp [H(cr) ] g W(S) [KS], where W(S) is the density of states. Consider R Monte Carlo simulations. We perform the nth simulation at EC " and store the data as histograms, + " (S)f,with total numbers of values [n " j. Errors are given by where we have used a bar over an expression to indicate the expectation value with respect to all Monte Carlo 1989 The American Physical Society 1195</td>
</tr>
<tr id="bib_Ferrenberg1989" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ferrenberg1989,
  author = {Ferrenberg, A.M. and Swendsen, R.H.},
  title = {Optimised monte carlo data analysis},
  journal = {Physical Review Letters},
  year = {1989},
  volume = {63},
  number = {12},
  pages = {1195 - 1198},
  doi = {https://doi.org/10.1103/PhysRevLett.63.1195}
}
</pre></td>
</tr>
<tr id="Ferrer2005" class="entry">
	<td>Ferrer, S., Ruiz-Pernia, J., Tunon, I., Moliner, V., Garcia-Viloca, M., Gonzalez-Lafont, A. and Lluch, J.M.</td>
	<td>A QM/MM exploration of the potential energy surface of pyruvate to lactate transformation catalyzed by ldh. Improving the accuracy of semiempirical descriptions <p class="infolinks">[<a href="javascript:toggleInfo('Ferrer2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ferrer2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Journal of Chemical Theory and Computation<br/>Vol. 1(4), pp. 750 - 761&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ct050016l">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ferrer2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present a QM/MM study of the potential energy surface of the pyruvate to lactate transformation catalyzed by L-lactate dehydrogenase. The transformation involves a hydride and a proton transfer which are followed by means of the corresponding antisymmetric combination of the distances from the hydrogen atom to the donor and the acceptor atoms. To discriminate among the possible reaction mechanisms we have considered different improvements of the AM1/MM description: reoptimization of the van der Waals parameters and inclusion of corrections to the QM energy associated with both transfer coordinates. The QM subsystem has been also enlarged to include charge-transfer effects from the substrate to some specific residues. In our best treatment, the transformation is described as a concerted process through a single transition structure in which the hydride transfer is more advanced than the proton transfer. From the methodological point of view, the correction schemes tested here improve the quality of the semiempirical potential energy surface although they also present deficiencies attributed to consideration of the proton and hydride transfer corrections as separable ones.</td>
</tr>
<tr id="bib_Ferrer2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ferrer2005,
  author = {Ferrer, Silvia and Ruiz-Pernia, Javier and Tunon, Inaki and Moliner, Vicente and Garcia-Viloca, Mireia and Gonzalez-Lafont, Angels and Lluch, Jose M},
  title = {A QM/MM exploration of the potential energy surface of pyruvate to lactate transformation catalyzed by ldh. Improving the accuracy of semiempirical descriptions},
  journal = {Journal of Chemical Theory and Computation},
  year = {2005},
  volume = {1},
  number = {4},
  pages = {750 - 761},
  note = {NULL},
  doi = {https://doi.org/10.1021/ct050016l}
}
</pre></td>
</tr>
<tr id="Feynman1986" class="entry">
	<td>Feynman, R. and Kleinert, H.</td>
	<td>Effective classical partition functions <p class="infolinks">[<a href="javascript:toggleInfo('Feynman1986','bibtex')">BibTeX</a>]</p></td>
	<td>1986</td>
	<td>Physical Review A<br/>Vol. 34(6), pp. 5080 - 5084&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Feynman1986" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Feynman1986,
  author = {Feynman, RP and Kleinert, H},
  title = {Effective classical partition functions},
  journal = {Physical Review A},
  year = {1986},
  volume = {34},
  number = {6},
  pages = {5080 - 5084}
}
</pre></td>
</tr>
<tr id="Feynman1965" class="entry">
	<td>Feynman, R.P. and Hibbs, A.R.</td>
	<td>Quantum Mechanics and Path Integrals <p class="infolinks">[<a href="javascript:toggleInfo('Feynman1965','bibtex')">BibTeX</a>]</p></td>
	<td>1965</td>
	<td>&nbsp;</td>
	<td>book</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Feynman1965" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{Feynman1965,
  author = {Feynman, Richard P and Hibbs, Albert R},
  title = {Quantum Mechanics and Path Integrals},
  publisher = {McGraw Hill},
  year = {1965}
}
</pre></td>
</tr>
<tr id="Field1990" class="entry">
	<td>Field, M.J., Bash, P.A. and Karplus, M.</td>
	<td>A combined quantum mechanical and molecular mechanical potential for molecular dynamics simulations <p class="infolinks">[<a href="javascript:toggleInfo('Field1990','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Field1990','bibtex')">BibTeX</a>]</p></td>
	<td>1990</td>
	<td>Journal of Computational Chemistry<br/>Vol. 11(6), pp. 700 - 733&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/jcc.540110605">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Field1990" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A combined quantum mechanical (QM) and molecular mechanical (MM) potential has been developed for the study of reactions in condensed phases. For the quantum mechanical calculations semiempirical methods of the MNDO and AM1 type are used, while the molecular mechanics part is treated with the CHARMM force field. Specific prescriptions are given for the interactions between the QM and MM portions of the system; cases in which the QM and MM methodology is applied to parts of the same molecule or to different molecules are considered. The details of the method and a range of test calculations, including comparisons with ab initio and experimental results, are given. It is found that in many cases satisfactory results are obtained. However, there are limitations to this type of approach, some of which arise from the AM1 or MNDO methods themselves and others from the present QM/MM implementation. This suggests that it is important to test the applicability of the method to each particular case prior to its use. Possible areas of improvement in the methodology are discussed.</td>
</tr>
<tr id="bib_Field1990" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Field1990,
  author = {Field, Martin J and Bash, Paul A and Karplus, Martin},
  title = {A combined quantum mechanical and molecular mechanical potential for molecular dynamics simulations},
  journal = {Journal of Computational Chemistry},
  year = {1990},
  volume = {11},
  number = {6},
  pages = {700 - 733},
  doi = {https://doi.org/10.1002/jcc.540110605}
}
</pre></td>
</tr>
<tr id="Fischer1894" class="entry">
	<td>Fischer, E.</td>
	<td>Einfluss der Configuration auf die Wirkung der Enzyme <p class="infolinks">[<a href="javascript:toggleInfo('Fischer1894','bibtex')">BibTeX</a>]</p></td>
	<td>1894</td>
	<td>Berichte der deutschen chemischen Gesellschaft<br/>Vol. 27(3), pp. 2985 - 2993&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/cber.18940270364">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Fischer1894" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Fischer1894,
  author = {Fischer, Emil},
  title = {Einfluss der Configuration auf die Wirkung der Enzyme},
  journal = {Berichte der deutschen chemischen Gesellschaft},
  year = {1894},
  volume = {27},
  number = {3},
  pages = {2985 - 2993},
  doi = {https://doi.org/10.1002/cber.18940270364}
}
</pre></td>
</tr>
<tr id="Ford2001" class="entry">
	<td>Ford, M., Pearse, B., Higgins, M., Vallis, Y., Owen, D., Gibson, A., Hopkins, C., Evans, P. and McMahon, H.</td>
	<td>Simultaneous binding of PtdIns(4,5)P2 and clathrin by AP180 in the nucleation of clathrin lattices on membranes <p class="infolinks">[<a href="javascript:toggleInfo('Ford2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ford2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Science<br/>Vol. 291(5506), pp. 1051 - 1055&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1126/science.291.5506.1051">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ford2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Adaptor protein 180 (AP180) and its homolog, clathrin assembly lymphoid myeloid leukemia protein (CALM), are closely related proteins that play important roles in clathrin-mediated endocytosis. Here, we present the structure of the NH2-terminal domain of CALM bound to phosphatidylinositol-4,5- bisphosphate [PtdIns(4,5)P2] via a lysine-rich motif. This motif is found in other proteins predicted to have domains of similar structure (for example, Huntingtin interacting protein 1). The structure is in part similar to the epsin NH2-terminal (ENTH) domain, but epsin lacks the PtdIns(4,5)P2-binding site. Because AP180 could bind to PtdIns(4,5)P2 and clathrin simultaneously, it may serve to tether clathrin to the membrane. This was shown by using purified components and a budding assay on preformed lipid monolayers. In the presence of AP180, clathrin lattices formed on the monolayer. When AP2 was also present, coated pits were formed.</td>
</tr>
<tr id="bib_Ford2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ford2001,
  author = {Ford, MG and Pearse, BM and Higgins, MK and Vallis, Y and Owen, DJ and Gibson, A and Hopkins, CR and Evans, PR and McMahon, HT},
  title = {Simultaneous binding of PtdIns(4,5)P2 and clathrin by AP180 in the nucleation of clathrin lattices on membranes},
  journal = {Science},
  year = {2001},
  volume = {291},
  number = {5506},
  pages = {1051 - 1055},
  note = {dissociation constant information for AP180 - PIP2 and CALM - PIP2.},
  doi = {https://doi.org/10.1126/science.291.5506.1051}
}
</pre></td>
</tr>
<tr id="Ford2002" class="entry">
	<td>Ford, M.G., Mills, I.G., Peter, B.J., Vallis, Y., Praefcke, G.J., Evans, P.R. and McMahon, H.T.</td>
	<td>Curvature of clathrin-coated pits driven by epsin <p class="infolinks">[<a href="javascript:toggleInfo('Ford2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ford2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Nature<br/>Vol. 419(6905), pp. 361 - 366&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nature01020">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ford2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis involves cargo selection and membrane budding into vesicles with the aid of a protein coat. Formation of invaginated pits on the plasma membrane and subsequent budding of vesicles is an energetically demanding process that involves the cooperation of clathrin with many different proteins. Here we investigate the role of the brain-enriched protein epsin 1 in this process. Epsin is targeted to areas of endocytosis by binding the membrane lipid phosphatidylinositol-4,5-bisphosphate (PtdIns(4,5)P(2)). We show here that epsin 1 directly modifies membrane curvature on binding to PtdIns(4,5)P(2) in conjunction with clathrin polymerization. We have discovered that formation of an amphipathic alpha-helix in epsin is coupled to PtdIns(4,5)P(2) binding. Mutation of residues on the hydrophobic region of this helix abolishes the ability to curve membranes. We propose that this helix is inserted into one leaflet of the lipid bilayer, inducing curvature. On lipid monolayers epsin alone is sufficient to facilitate the formation of clathrin-coated invaginations.</td>
</tr>
<tr id="bib_Ford2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ford2002,
  author = {Ford, Marijn GJ and Mills, Ian G and Peter, Brian J and Vallis, Yvonne and Praefcke, Gerrit JK and Evans, Philip R and McMahon, Harvey T},
  title = {Curvature of clathrin-coated pits driven by epsin},
  journal = {Nature},
  year = {2002},
  volume = {419},
  number = {6905},
  pages = {361 - 366},
  note = {rat brain. domain information. AP180, EPN1 - PIP2, CLTC},
  doi = {https://doi.org/10.1038/nature01020}
}
</pre></td>
</tr>
<tr id="Fortunato2017" class="entry">
	<td>Fortunato, M.E. and Colina, C.M.</td>
	<td>pysimm: A python package for simulation of molecular systems <p class="infolinks">[<a href="javascript:toggleInfo('Fortunato2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>SoftwareX<br/>Vol. 6, pp. 7-12&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Fortunato2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Fortunato2017,
  author = {Fortunato, Michael E and Colina, Coray M},
  title = {pysimm: A python package for simulation of molecular systems},
  journal = {SoftwareX},
  publisher = {Elsevier},
  year = {2017},
  volume = {6},
  pages = {7--12}
}
</pre></td>
</tr>
<tr id="Fotin2004" class="entry">
	<td>Fotin, A., Cheng, Y., Sliz, P., Grigorieff, N., Harrison, S.C., Kirchhausen, T. and Walz, T.</td>
	<td>Molecular model for a complete clathrin lattice from electron cryomicroscopy <p class="infolinks">[<a href="javascript:toggleInfo('Fotin2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Nature<br/>Vol. 432(7017), pp. 573-579&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nature03079">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Fotin2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Fotin2004,
  author = {Fotin, Alexander and Cheng, Yifan and Sliz, Piotr and Grigorieff, Nikolaus and Harrison, Stephen C. and Kirchhausen, Tomas and Walz, Thomas},
  title = {Molecular model for a complete clathrin lattice from electron cryomicroscopy},
  journal = {Nature},
  year = {2004},
  volume = {432},
  number = {7017},
  pages = {573--579},
  doi = {https://doi.org/10.1038/nature03079}
}
</pre></td>
</tr>
<tr id="Fraaije2017" class="entry">
	<td>Fraaije, J., van Male, J., Becherer, P. and Serral Gracia, R.</td>
	<td>Calculation of diffusion coefficients through coarse-grained simulations using the Automated-Fragmentation-Parametrization method and the recovery of Wilke-Chang statistical correlation <p class="infolinks">[<a href="javascript:toggleInfo('Fraaije2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of chemical theory and computation&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Fraaije2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Fraaije2017,
  author = {Fraaije, Johannes and van Male, Jan and Becherer, Paul and Serral Gracia, Ruben},
  title = {Calculation of diffusion coefficients through coarse-grained simulations using the Automated-Fragmentation-Parametrization method and the recovery of Wilke-Chang statistical correlation},
  journal = {Journal of chemical theory and computation},
  publisher = {ACS Publications},
  year = {2017}
}
</pre></td>
</tr>
<tr id="Freddolino2006" class="entry">
	<td>Freddolino, P.L., Arkhipov, A.S., Larson, S.B., McPherson, A. and Schulten, K.</td>
	<td>Molecular dynamics simulations of the complete satellite tobacco mosaic virus <p class="infolinks">[<a href="javascript:toggleInfo('Freddolino2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Freddolino2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Structure<br/>Vol. 14(3), pp. 437 - 449&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.str.2005.11.014">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Freddolino2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: This work presents an all-atom molecular dynamics simulation of a complete virus, the satellite tobacco mosaic virus. Simulations with up to 1 million atoms for over 50 ns demonstrate the stability of the entire virion and of the RNA core alone, while the capsid without RNA exhibits a pronounced instability. Physical properties of the simulated virus particle including electrostatic potential, radial distribution of viral components, and patterns of correlated motion are analyzed, and the implications for the assembly and infection mechanism of the virus are discussed. ??2006 Elsevier Ltd. All rights reserved.</td>
</tr>
<tr id="bib_Freddolino2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Freddolino2006,
  author = {Freddolino, Peter L and Arkhipov, Anton S and Larson, Steven B and McPherson, Alexander and Schulten, Klaus},
  title = {Molecular dynamics simulations of the complete satellite tobacco mosaic virus},
  journal = {Structure},
  year = {2006},
  volume = {14},
  number = {3},
  pages = {437 - 449},
  doi = {https://doi.org/10.1016/j.str.2005.11.014}
}
</pre></td>
</tr>
<tr id="Freidlin2016" class="entry">
	<td>Freidlin, M. and Koralov, L.</td>
	<td>Metastable distributions of Markov chains with rare transitions <p class="infolinks">[<a href="javascript:toggleInfo('Freidlin2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Freidlin2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Journal of Statistical Physics<br/>Vol. 167(6), pp. 1 - 18&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1007/s10955-017-1777-z">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Freidlin2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In this paper we consider Markov chains with transition rates that depend on a small parameter &dollar;\textbackslash\varepsilon&dollar;. Under a mild assumption on the asymptotics of these transition rates, we describe the behavior of the chain at various &dollar;\textbackslash\varepsilon&dollar;-dependent time scales, i.e., we calculate the metastable distributions.</td>
</tr>
<tr id="bib_Freidlin2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Freidlin2016,
  author = {Freidlin, Mark and Koralov, Leonid},
  title = {Metastable distributions of Markov chains with rare transitions},
  journal = {Journal of Statistical Physics},
  year = {2016},
  volume = {167},
  number = {6},
  pages = {1 - 18},
  doi = {https://doi.org/10.1007/s10955-017-1777-z}
}
</pre></td>
</tr>
<tr id="Fried2017" class="entry">
	<td>Fried, S.D. and Boxer, S.G.</td>
	<td>Electric Fields and Enzyme Catalysis <p class="infolinks">[<a href="javascript:toggleInfo('Fried2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Fried2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Annual Review of Biochemistry<br/>Vol. 86(1), pp. annurev - biochem-061516-044432&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1146/annurev-biochem-061516-044432">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Fried2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: What happens inside an enzyme's active site to allow slow and difficult chemical reactions to occur so rapidly? This question has occupied biochemists' attention for a long time. Computer models of increasing sophistication have predicted an important role for electrostatic interactions in enzymatic reactions, yet this hypothesis has proved vexingly difficult to test experimentally. Recent experiments utilizing the vibrational Stark effect make it possible to measure the electric field a substrate molecule experiences when bound inside its enzyme's active site. These experiments have provided compelling evidence supporting a major electrostatic contribution to enzymatic catalysis. Here, we review these results and develop a simple model for electrostatic catalysis that enables us to incorporate disparate concepts introduced by many investigators to describe how enzymes work into a more unified framework stressing the importance of electric fields at the active site. Expected final online publication date ...</td>
</tr>
<tr id="bib_Fried2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Fried2017,
  author = {Fried, Stephen D and Boxer, Steven G},
  title = {Electric Fields and Enzyme Catalysis},
  journal = {Annual Review of Biochemistry},
  year = {2017},
  volume = {86},
  number = {1},
  pages = {annurev - biochem-061516-044432},
  doi = {https://doi.org/10.1146/annurev-biochem-061516-044432}
}
</pre></td>
</tr>
<tr id="Frost2008" class="entry">
	<td>Frost, A., Perera, R., Roux, A., Spasov, K., Destaing, O., Egelman, E.H., De Camilli, P. and Unger, V.M.</td>
	<td>Structural basis of membrane invagination by F-BAR domains. <p class="infolinks">[<a href="javascript:toggleInfo('Frost2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Frost2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Cell<br/>Vol. 132, pp. 807-817&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cell.2007.12.041">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Frost2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: BAR superfamily domains shape membranes through poorly understood mechanisms. We solved structures of F-BAR modules bound to flat and curved bilayers using electron (cryo)microscopy. We show that membrane tubules form when F-BARs polymerize into helical coats that are held together by lateral and tip-to-tip interactions. On gel-state membranes or after mutation of residues along the lateral interaction surface, F-BARs adsorb onto bilayers via surfaces other than their concave face. We conclude that membrane binding is separable from membrane bending, and that imposition of the module's concave surface forces fluid-phase bilayers to bend locally. Furthermore, exposure of the domain's lateral interaction surface through a change in orientation serves as the crucial trigger for assembly of the helical coat and propagation of bilayer bending. The geometric constraints and sequential assembly of the helical lattice explain how F-BAR and classical BAR domains segregate into distinct microdomains, and provide insight into the spatial regulation of membrane invagination.</td>
</tr>
<tr id="bib_Frost2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Frost2008,
  author = {Frost, Adam and Perera, Rushika and Roux, Aurélien and Spasov, Krasimir and Destaing, Olivier and Egelman, Edward H and De Camilli, Pietro and Unger, Vinzenz M},
  title = {Structural basis of membrane invagination by F-BAR domains.},
  journal = {Cell},
  year = {2008},
  volume = {132},
  pages = {807--817},
  doi = {https://doi.org/10.1016/j.cell.2007.12.041}
}
</pre></td>
</tr>
<tr id="Frye1998" class="entry">
	<td>Frye, K.J. and Royer, C.A.</td>
	<td>Probing the contribution of internal cavities to the volume change of protein unfolding under pressure <p class="infolinks">[<a href="javascript:toggleInfo('Frye1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Protein Science<br/>Vol. 7(10), pp. 2217-2222&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Frye1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Frye1998,
  author = {Frye, Kelly J and Royer, Catherine A},
  title = {Probing the contribution of internal cavities to the volume change of protein unfolding under pressure},
  journal = {Protein Science},
  publisher = {Wiley Online Library},
  year = {1998},
  volume = {7},
  number = {10},
  pages = {2217--2222}
}
</pre></td>
</tr>
<tr id="Gaidarov1996" class="entry">
	<td>Gaidarov, I., Chen, Q., Falck, J.R., Reddy, K.K. and Keen, J.H.</td>
	<td>A functional phosphatidylinositol 3,4,5-trisphosphate/phosphoinositide binding domain in the clathrin adaptor AP-2 alpha subunit. Implications for the endocytic pathway. <p class="infolinks">[<a href="javascript:toggleInfo('Gaidarov1996','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gaidarov1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>The Journal of biological chemistry<br/>Vol. 271, pp. 20922-20929&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Gaidarov1996" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-coated pits are sites of concentration of ligand-bound signaling receptors. Several such receptors are known to recruit, bind, and activate the heterodimeric phosphatidylinositol-3-kinase, resulting in the generation of phosphatidylinositol 3,4, 5-trisphosphate. We report here that dioctanoyl-phosphatidylinositol-3,4,5-P3 binds specifically and saturably to soluble AP-2 and with greater affinity to AP-2 within assembled coat structures. Soluble -myo-inositol hexakisphosphate shows converse behavior. Binding to bovine brain clathrin-coated vesicles is evident only after detergent extraction. These observations and evidence for recognition of the diacylglyceryl backbone as well as the inositol phosphate headgroup are consistent with AP-2 interaction with membrane phosphoinositides in coated vesicles and with soluble inositol phosphates in cytoplasm. A discrete binding domain is identified near the N terminus of the AP-2 alpha subunit, and an expressed fusion protein containing this sequence exhibits specific, high affinity binding that is virtually identical to the parent protein. This region of the AP-2 alpha sequence also shows the greatest conservation between a Caenorhabditis elegans homolog and mammalian alpha, consistent with a function in recognition of an evolutionarily unchanging low molecular weight ligand. Binding of phosphatidylinositol 3,4, 5-trisphosphate to AP-2 inhibits the protein's clathrin binding and assembly activities. These findings are discussed in the context of the potential roles of phosphoinositides and AP-2 in the internalization and trafficking of cell surface receptors.</td>
</tr>
<tr id="bib_Gaidarov1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gaidarov1996,
  author = {Gaidarov, I and Chen, Q and Falck, J R and Reddy, K K and Keen, J H},
  title = {A functional phosphatidylinositol 3,4,5-trisphosphate/phosphoinositide binding domain in the clathrin adaptor AP-2 alpha subunit. Implications for the endocytic pathway.},
  journal = {The Journal of biological chemistry},
  year = {1996},
  volume = {271},
  pages = {20922--20929},
  note = {ap2 interactions with various inositols, but not 4,5-pip2}
}
</pre></td>
</tr>
<tr id="Gaidarov1999" class="entry">
	<td>Gaidarov, I. and Keen, J.H.</td>
	<td>Phosphoinositide-AP-2 interactions required for targeting to plasma membrane clathrin-coated pits. <p class="infolinks">[<a href="javascript:toggleInfo('Gaidarov1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gaidarov1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>The Journal of cell biology<br/>Vol. 146, pp. 755-764&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Gaidarov1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The clathrin-associated AP-2 adaptor protein is a major polyphosphoinositide-binding protein in mammalian cells. A high affinity binding site has previously been localized to the NH(2)-terminal region of the AP-2 alpha subunit (Gaidarov et al. 1996. J. Biol. Chem. 271:20922-20929). Here we used deletion and site- directed mutagenesis to determine that alpha residues 21-80 comprise a discrete folding and inositide-binding domain. Further, positively charged residues located within this region are involved in binding, with a lysine triad at positions 55-57 particularly critical. Mutant peptides and protein in which these residues were changed to glutamine retained wild-type structural and functional characteristics by several criteria including circular dichroism spectra, resistance to limited proteolysis, and clathrin binding activity. When expressed in intact cells, mutated alpha subunit showed defective localization to clathrin-coated pits; at high expression levels, the appearance of endogenous AP-2 in coated pits was also blocked consistent with a dominant-negative phenotype. These results, together with recent work indicating that phosphoinositides are also critical to ligand-dependent recruitment of arrestin-receptor complexes to coated pits (Gaidarov et al. 1999. EMBO (Eur. Mol. Biol. Organ.) J. 18:871-881), suggest that phosphoinositides play a critical and general role in adaptor incorporation into plasma membrane clathrin-coated pits.</td>
</tr>
<tr id="bib_Gaidarov1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gaidarov1999,
  author = {Gaidarov, I and Keen, J H},
  title = {Phosphoinositide-AP-2 interactions required for targeting to plasma membrane clathrin-coated pits.},
  journal = {The Journal of cell biology},
  year = {1999},
  volume = {146},
  pages = {755--764},
  note = {IP6 interactions with alpha adaptin. NOT PIP2.}
}
</pre></td>
</tr>
<tr id="Gaidarov1999a" class="entry">
	<td>Gaidarov, I., Santini, F., Warren, R.A. and Keen, J.H.</td>
	<td>Spatial control of coated-pit dynamics in living cells. <p class="infolinks">[<a href="javascript:toggleInfo('Gaidarov1999a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gaidarov1999a','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Nature cell biology<br/>Vol. 1, pp. 1-7&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/8971">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Gaidarov1999a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Here we visualize new aspects of the dynamics of endocytotic clathrin-coated pits and vesicles in mammalian cells by using a fusion protein consisting of green fluorescent protein and clathrin light chain a. Clathrin-coated pits invaginating from the plasma membrane show definite, but highly limited, mobility within the membrane that is relaxed upon treatment with latrunculin B, an inhibitor of actin assembly, indicating that an actin-based framework may be involved in the mobility of these pits. Transient, motile coated vesicles that originate from coated pits can be detected, with multiple vesicles occasionally appearing to emanate from a single pit. Despite their seemingly random distribution, coated pits tend to form repeatedly at defined sites while excluding other regions. This spatial regulation of coated-pit assembly and function is attributable to the attachment of the coated pits to the membrane skeleton.</td>
</tr>
<tr id="bib_Gaidarov1999a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gaidarov1999a,
  author = {Gaidarov, I and Santini, F and Warren, R A and Keen, J H},
  title = {Spatial control of coated-pit dynamics in living cells.},
  journal = {Nature cell biology},
  year = {1999},
  volume = {1},
  pages = {1--7},
  doi = {https://doi.org/10.1038/8971}
}
</pre></td>
</tr>
<tr id="Galli2017" class="entry">
	<td>Galli, V., Sebastian, R., Moutel, S., Ecard, J., Perex, F. and Roux, A.</td>
	<td>Uncoupling of dynamin polymerization and GTPase activity revealed by the conformation-specific nanobody Dynab <p class="infolinks">[<a href="javascript:toggleInfo('Galli2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>eLife&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Galli2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Galli2017,
  author = {Galli, Valentina and Sebastian, Rafael and Moutel, Sandrine and Ecard, Jason and Perex, Franck and Roux, Aurélien},
  title = {Uncoupling of dynamin polymerization and GTPase activity revealed by the conformation-specific nanobody Dynab},
  journal = {eLife},
  year = {2017}
}
</pre></td>
</tr>
<tr id="Galperin1998" class="entry">
	<td>Galperin, M., Walker, D. and Koonin, E.</td>
	<td>Analogous enzymes: independent inventions in enzyme evolution. <p class="infolinks">[<a href="javascript:toggleInfo('Galperin1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Galperin1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Genome research<br/>Vol. 8(8), pp. 779 - 90&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1101/GR.8.8.779">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Galperin1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: It is known that the same reaction may be catalyzed by structurally unrelated enzymes. We performed a systematic search for such analogous (as opposed to homologous) enzymes by evaluating sequence conservation among enzymes with the same enzyme classification (EC) number using sensitive, iterative sequence database search methods. Enzymes without detectable sequence similarity to each other were found for 105 EC numbers (a total of 243 distinct proteins). In 34 cases, independent evolutionary origin of the suspected analogous enzymes was corroborated by showing that they possess different structural folds. Analogous enzymes were found in each class of enzymes, but their overall distribution on the map of biochemical pathways is patchy, suggesting multiple events of gene transfer and selective loss in evolution, rather than acquisition of entire pathways catalyzed by a set of unrelated enzymes. Recruitment of enzymes that catalyze a similar but distinct reaction seems to be a major scenario for the evolution of analogous enzymes, which should be taken into account for functional annotation of genomes. For many analogous enzymes, the bacterial form of the enzyme is different from the eukaryotic one; such enzymes may be promising targets for the development of new antibacterial drugs.</td>
</tr>
<tr id="bib_Galperin1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Galperin1998,
  author = {Galperin, MY and Walker, DR and Koonin, EV},
  title = {Analogous enzymes: independent inventions in enzyme evolution.},
  journal = {Genome research},
  year = {1998},
  volume = {8},
  number = {8},
  pages = {779 - 90},
  note = {NULL},
  doi = {https://doi.org/10.1101/GR.8.8.779}
}
</pre></td>
</tr>
<tr id="Ganzhorn1987" class="entry">
	<td>Ganzhorn, A.J., Green, D.W., Hershey, A.D., Gould, R.M. and Plapp, B.V.</td>
	<td>Kinetic characterization of yeast alcohol dehydrogenases: Amino acid residue 294 and substrate specificity. <p class="infolinks">[<a href="javascript:toggleInfo('Ganzhorn1987','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ganzhorn1987','bibtex')">BibTeX</a>]</p></td>
	<td>1987</td>
	<td>Journal of Biological Chemistry<br/>Vol. 262(8), pp. 3754 - 3761&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Ganzhorn1987" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A three-dimensional model of yeast alcohol dehydrogenase, based on the homologous horse liver enzyme, was used to compare the substrate binding pockets of the three isozymes (I, II, and III) from Saccharomyces cerevisiae and the enzyme from Schizosaccharomyces pombe. Isozyme I and the S. pombe enzyme have methionine at position 294 (numbered as in the liver enzyme, corresponding to 270 in yeast), whereas isozymes II and III have leucine. Otherwise the active sites of the S. cerevisiae enzymes are the same. All four wild-type enzymes were produced from the cloned genes. In addition, oligonucleotide-directed mutagenesis was used to change Met-294 in alcohol dehydrogenase I to leucine. The mechanisms for all five enzymes were predominantly ordered with ethanol (but partially random with butanol) at pH 7.3 and 30 degrees C. The wild-type alcohol dehydrogenases and the leucine mutant had similar kinetic constants, except that isozyme II had 10-20-fold smaller Michaelis and inhibition constants for ethanol. Thus, residue 294 is not responsible for this difference. Apparently, substitutions outside of the substrate binding pocket indirectly affect the interactions of the alcohol dehydrogenases with ethanol. Nevertheless, the substitution of methionine with leucine in the substrate binding site of alcohol dehydrogenase I produced a 7-10-fold increase in reactivity (V/Km) with butanol, pentanol, and hexanol. The higher activity is due to tighter binding of the longer chain alcohols and to more rapid hydrogen transfer.</td>
</tr>
<tr id="bib_Ganzhorn1987" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ganzhorn1987,
  author = {Ganzhorn, Axel J and Green, David W and Hershey, Andrew D and Gould, Robert M and Plapp, Bryce V},
  title = {Kinetic characterization of yeast alcohol dehydrogenases: Amino acid residue 294 and substrate specificity.},
  journal = {Journal of Biological Chemistry},
  year = {1987},
  volume = {262},
  number = {8},
  pages = {3754 - 3761}
}
</pre></td>
</tr>
<tr id="Gao1998" class="entry">
	<td>Gao, J., Amara, P., Alhambra, C. and Field, M.J.</td>
	<td>A Generalized Hybrid Orbital (GHO) method for the treatment of boundary atoms in combined QM/MM calculations <p class="infolinks">[<a href="javascript:toggleInfo('Gao1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gao1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Journal of Physical Chemistry A<br/>Vol. 102(24), pp. 4714 - 4721&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp9809890">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Gao1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A generalized hybrid orbital (GHO) method has been developed at the semiempirical level in combined quantum mechanical and molecular mechanical (QM/MM) calculations. In this method, a set of hybrid orbitals is placed on the boundary atom between the QM and MM fragments, and one of the hybrid orbitals participates in the SCF calculation for the atoms in the QM region. The GHO method provides a well-defined potential energy surface for a hybrid QM/MM system and is a significant improvement over the ?link-atom? approach by saturating the QM valencies with hydrogen atoms. The method has been tested on small molecules and yields reasonable structural, energetic, and electronic results in comparison with the results of the corresponding QM and MM approximations. The GHO method will greatly increase the applicability of combined QM/MM methods to systems comprising large molecules, such as proteins. A generalized hybrid orbital (GHO) method has been developed at the semiempirical level in combined quantum mechanical and molecular mechanical (QM/MM) calculations. In this method, a set of hybrid orbitals is placed on the boundary atom between the QM and MM fragments, and one of the hybrid orbitals participates in the SCF calculation for the atoms in the QM region. The GHO method provides a well-defined potential energy surface for a hybrid QM/MM system and is a significant improvement over the ?link-atom? approach by saturating the QM valencies with hydrogen atoms. The method has been tested on small molecules and yields reasonable structural, energetic, and electronic results in comparison with the results of the corresponding QM and MM approximations. The GHO method will greatly increase the applicability of combined QM/MM methods to systems comprising large molecules, such as proteins.</td>
</tr>
<tr id="bib_Gao1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gao1998,
  author = {Gao, Jiali and Amara, Patricia and Alhambra, Cristobal and Field, Martin J},
  title = {A Generalized Hybrid Orbital (GHO) method for the treatment of boundary atoms in combined QM/MM calculations},
  journal = {Journal of Physical Chemistry A},
  year = {1998},
  volume = {102},
  number = {24},
  pages = {4714 - 4721},
  note = {NULL},
  doi = {https://doi.org/10.1021/jp9809890}
}
</pre></td>
</tr>
<tr id="Gao2002" class="entry">
	<td>Gao, J. and Truhlar, D.G.</td>
	<td>Quantum mechanical methods for enzyme kinetics <p class="infolinks">[<a href="javascript:toggleInfo('Gao2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gao2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Annual Review of Physical Chemistry<br/>Vol. 53(1), pp. 467 - 505&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1146/annurev.physchem.53.091301.150114">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Gao2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Abstract  This review discusses methods for the incorporation of quantum mechanical effects into enzyme kinetics simulations in which the enzyme is an explicit part of the model. We emphasize three aspects: (a) use of quantum mechanical electronic structure methods such as molecular orbital theory and density functional theory, usually in conjunction with molecular mechanics; (b) treating vibrational motions quantum mechanically, either in an instantaneous harmonic approximation, or by path integrals, or by a three-dimensional wave function coupled to classical nuclear motion; (c) incorporation of multidimensional tunneling approximations into reaction rate calculations.</td>
</tr>
<tr id="bib_Gao2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gao2002,
  author = {Gao, Jiali and Truhlar, Donald G},
  title = {Quantum mechanical methods for enzyme kinetics},
  journal = {Annual Review of Physical Chemistry},
  year = {2002},
  volume = {53},
  number = {1},
  pages = {467 - 505},
  doi = {https://doi.org/10.1146/annurev.physchem.53.091301.150114}
}
</pre></td>
</tr>
<tr id="Garcia1992" class="entry">
	<td>Garcia, A.E.</td>
	<td>Large-Amplitude Nonlinear Motions in Proteins <p class="infolinks">[<a href="javascript:toggleInfo('Garcia1992','bibtex')">BibTeX</a>]</p></td>
	<td>1992</td>
	<td>Physical Review Letters<br/>Vol. 68(17), pp. 2696 - 2699&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Garcia1992" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Garcia1992,
  author = {Garcia, Angel E},
  title = {Large-Amplitude Nonlinear Motions in Proteins},
  journal = {Physical Review Letters},
  year = {1992},
  volume = {68},
  number = {17},
  pages = {2696 - 2699}
}
</pre></td>
</tr>
<tr id="Mireia2003" class="entry">
	<td>Garcia-Viloca, M., Truhlar, D.G. and Gao, J.</td>
	<td>Reaction-path energetics and kinetics of the hydride transfer reaction catalyzed by dihydrofolate reductase. <p class="infolinks">[<a href="javascript:toggleInfo('Mireia2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Mireia2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Biochemistry<br/>Vol. 42(46), pp. 13558 - 75&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi034824f">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Mireia2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We have studied the hydride transfer reaction catalyzed by the enzyme dihydrofolate reductase (DHFR) and the coenzyme nicotinamide adenine dinucleotide phosphate (NADPH); the substrate is 5-protonated 7,8-dihydrofolate, and the product is tetrahydrofolate. The potential energy surface is modeled by a combined quantum mechanical-molecular mechanical (QM/MM) method employing Austin model 1 (AM1) and a simple valence bond potential for 69 QM atoms and employing the CHARMM22 and TIP3P molecular mechanics force fields for the other 21 399 atoms; the QM and MM regions are joined by two boundary atoms treated by the generalized hybrid orbital (GHO) method. All simulations are carried out using periodic boundary conditions at neutral pH and 298 K. In stage 1, a reaction coordinate is defined as the difference between the breaking and forming bond distances to the hydride ion, and a quasithermodynamic free energy profile is calculated along this reaction coordinate. This calculation includes quantization effects on bound vibrations but not on the reaction coordinate, and it is used to locate the variational transition state that defines a transition state ensemble. Then, the key interactions at the reactant, variational transition state, and product are analyzed in terms of both bond distances and electrostatic energies. The results of both analyses support the conclusion derived from previous mutational studies that the M20 loop of DHFR makes an important contribution to the electrostatic stabilization of the hydride transfer transition state. Third, transmission coefficients (including recrossing factors and multidimensional tunneling) are calculated and averaged over the transition state ensemble. These averaged transmission coefficients, combined with the quasithermodynamic free energy profile determined in stage 1, allow us to calculate rate constants, phenomenological free energies of activation, and primary and secondary kinetic isotope effects. A primary kinetic isotope effect (KIE) of 2.8 has been obtained, in good agreement with the experimentally determined value of 3.0 and with the value 3.2 calculated previously. The primary KIE is mainly a consequence of the quantization of bound vibrations. In contrast, the secondary KIE, with a value of 1.13, is almost entirely due to dynamical effects on the reaction coordinate, especially tunneling.</td>
</tr>
<tr id="bib_Mireia2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mireia2003,
  author = {Garcia-Viloca, Mireia and Truhlar, Donald G and Gao, Jiali},
  title = {Reaction-path energetics and kinetics of the hydride transfer reaction catalyzed by dihydrofolate reductase.},
  journal = {Biochemistry},
  year = {2003},
  volume = {42},
  number = {46},
  pages = {13558 - 75},
  doi = {https://doi.org/10.1021/bi034824f}
}
</pre></td>
</tr>
<tr id="Gardner2002" class="entry">
	<td>Gardner, M., Hall, N., Fung, E., White, O., Berriman, M., Hyman, R., Carlton, J., Pain, A., Nelson, K., Bowman, S., Paulsen, I., James, K., Eisen, J., Rutherford, K., Salzberg, S., Craig, A., Kyes, S., Chan, M.-S., Nene, V., Shallom, S., Suh, B., Peterson, J., Angiuoli, S., Pertea, M., Allen, J., Selengut, J., Haft, D., Mather, M., Vaidya, A., Martin, D., Fairlamb, A., Fraunholz, M., Roos, D., Ralph, S., McFadden, G., Cummings, L., Subramanian, G., Mungall, C., Venter, J., Carucci, D., Hoffman, S., Newbold, C., Davis, R., Fraser, C. and Barrell, B.</td>
	<td>Genome sequence of the human malaria parasite Plasmodium falciparum <p class="infolinks">[<a href="javascript:toggleInfo('Gardner2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gardner2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Nature<br/>Vol. 419(6906), pp. 498 - 511&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nature01097">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Gardner2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The parasite Plasmodium falciparum is responsible for hundreds of millions of cases of malaria, and kills more than one million African children annually. Here we report an analysis of the genome sequence of P. falciparum clone 3D7. The 23-megabase nuclear genome consists of 14 chromosomes, encodes about 5,300 genes, and is the most (A + T)-rich genome sequenced to date. Genes involved in antigenic variation are concentrated in the subtelomeric regions of the chromosomes. Compared to the genomes of free-living eukaryotic microbes, the genome of this intracellular parasite encodes fewer enzymes and transporters, but a large proportion of genes are devoted to immune evasion and host-parasite interactions. Many nuclear-encoded proteins are targeted to the apicoplast, an organelle involved in fatty-acid and isoprenoid metabolism. The genome sequence provides the foundation for future studies of this organism, and is being exploited in the search for new drugs and vaccines to fight malaria.</td>
</tr>
<tr id="bib_Gardner2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gardner2002,
  author = {Gardner, M.J. and Hall, Neil and Fung, Eula and White, Owen and Berriman, Matthew and Hyman, R.W. and Carlton, J.M. and Pain, Arnab and Nelson, K.E. and Bowman, Sharen and Paulsen, I.T. and James, Keith and Eisen, J.A. and Rutherford, Kim and Salzberg, S.L. and Craig, Alister and Kyes, Sue and Chan, M.-S. and Nene, Vishvanath and Shallom, S.J. and Suh, Bernard and Peterson, Jeremy and Angiuoli, Sam and Pertea, Mihaela and Allen, Jonathan and Selengut, Jeremy and Haft, Daniel and Mather, M.W. and Vaidya, A.B. and Martin, D.M.A. and Fairlamb, A.H. and Fraunholz, M.J. and Roos, D.S. and Ralph, S.A. and McFadden, G.I. and Cummings, L.M. and Subramanian, G.M. and Mungall, Chris and Venter, J.C. and Carucci, D.J. and Hoffman, S.L. and Newbold, Chris and Davis, R.W. and Fraser, C.M. and Barrell, Bart},
  title = {Genome sequence of the human malaria parasite Plasmodium falciparum},
  journal = {Nature},
  year = {2002},
  volume = {419},
  number = {6906},
  pages = {498 - 511},
  doi = {https://doi.org/10.1038/nature01097}
}
</pre></td>
</tr>
<tr id="Gasparotto2018" class="entry">
	<td>Gasparotto, P., Meissner, R. and Ceriotti, M.</td>
	<td>Recognizing Local and Global Structural Motifs At the Atomic Scale <p class="infolinks">[<a href="javascript:toggleInfo('Gasparotto2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Journal of chemical theory and computation&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Gasparotto2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gasparotto2018,
  author = {Gasparotto, Piero and Meissner, Robert and Ceriotti, Michele},
  title = {Recognizing Local and Global Structural Motifs At the Atomic Scale},
  journal = {Journal of chemical theory and computation},
  publisher = {ACS Publications},
  year = {2018}
}
</pre></td>
</tr>
<tr id="Geissler1999" class="entry">
	<td>Geissler, P.L., Dellago, C. and Chandler, D.</td>
	<td>Kinetic Pathways of Ion Pair Dissociation in Water <p class="infolinks">[<a href="javascript:toggleInfo('Geissler1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Geissler1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 103(18), pp. 3706 - 3710&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp984837g">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Geissler1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We have successfully applied the transition path sampling method to the deterministic dynamics of a many-body system with long-ranged interactions. The process we investigate, dissociation of a model Na+Cl- ion pair in water, involves a wide range of transition pathways in which collective solvent motions play an important role. Transition states along these pathways encompass a broad distribution of ionic separations. Ion pairs in contact remain associated for ?20 ps on average, a time scale considerably longer than the ?3ps predicted by transition state theory with ionic separation as the order parameter. In contrast to theories of barrier crossing based upon frictional effects (with or without memory), we find that the discrepancy between these time scales arises from neglected solvent free energy barriers rather than stochastic buffeting of the ion pair. We have successfully applied the transition path sampling method to the deterministic dynamics of a many-body system with long-ranged interactions. The process we investigate, dissociation of a model Na+Cl- ion pair in water, involves a wide range of transition pathways in which collective solvent motions play an important role. Transition states along these pathways encompass a broad distribution of ionic separations. Ion pairs in contact remain associated for ?20 ps on average, a time scale considerably longer than the ?3ps predicted by transition state theory with ionic separation as the order parameter. In contrast to theories of barrier crossing based upon frictional effects (with or without memory), we find that the discrepancy between these time scales arises from neglected solvent free energy barriers rather than stochastic buffeting of the ion pair.</td>
</tr>
<tr id="bib_Geissler1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Geissler1999,
  author = {Geissler, Phillip L and Dellago, Christoph and Chandler, David},
  title = {Kinetic Pathways of Ion Pair Dissociation in Water},
  journal = {Journal of Physical Chemistry B},
  year = {1999},
  volume = {103},
  number = {18},
  pages = {3706 - 3710},
  doi = {https://doi.org/10.1021/jp984837g}
}
</pre></td>
</tr>
<tr id="Geissler2000" class="entry">
	<td>Geissler, P.L., Dellago, C., Chandler, D., Hutter, J. and Parrinello, M.</td>
	<td>Ab initio analysis of proton transfer dynamics in (H2O)3H+ <p class="infolinks">[<a href="javascript:toggleInfo('Geissler2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Geissler2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Chemical Physics Letters<br/>Vol. 321(3-4), pp. 225 - 230&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S0009-2614(00)00381-X">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Geissler2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We have harvested ab initio trajectories of proton transfer in (H2O)3H+ by combining Car--Parrinello molecular dynamics (CPMD) with the transition path sampling method. Two transition state regions contribute to these dynamics, with saddle points similar to those identified by Geissler, Dellago, and Chandler for an empirical model of the same cluster [Phys. Chem. Chem. Phys. 1 (1999) 1317]. As in that model, the location of a transition state along a finite-temperature trajectory indicates that proton transfer is driven by reorganization of the oxygen ring. From vibrational properties it is estimated that the characteristic time for proton transfer is ∼1 ns at a temperature of 300 K.</td>
</tr>
<tr id="bib_Geissler2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Geissler2000,
  author = {Geissler, Phillip L and Dellago, Christoph and Chandler, David and Hutter, Jurg and Parrinello, Michele},
  title = {Ab initio analysis of proton transfer dynamics in (H2O)3H+},
  journal = {Chemical Physics Letters},
  year = {2000},
  volume = {321},
  number = {3-4},
  pages = {225 - 230},
  doi = {https://doi.org/10.1016/S0009-2614(00)00381-X}
}
</pre></td>
</tr>
<tr id="Gerlt2017" class="entry">
	<td>Gerlt, J.A.</td>
	<td>Genomic Enzymology: Web Tools for Leveraging Protein Family Sequence − Function Space and Genome Context to Discover Novel Functions <p class="infolinks">[<a href="javascript:toggleInfo('Gerlt2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Biochemistry<br/>Vol. 56(33), pp. 4293-4308&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.biochem.7b00614">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Gerlt2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gerlt2017,
  author = {Gerlt, John A},
  title = {Genomic Enzymology: Web Tools for Leveraging Protein Family Sequence − Function Space and Genome Context to Discover Novel Functions},
  journal = {Biochemistry},
  year = {2017},
  volume = {56},
  number = {33},
  pages = {4293-4308},
  doi = {https://doi.org/10.1021/acs.biochem.7b00614}
}
</pre></td>
</tr>
<tr id="Gerlt1993" class="entry">
	<td>Gerlt, J.A. and Gassman, P.G.</td>
	<td>Understanding the rates of certain enzyme-catalyzed reactions: Proton abstraction from carbon acids, acyl transfer reactions, and displacement reactions of phosphodiesters <p class="infolinks">[<a href="javascript:toggleInfo('Gerlt1993','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gerlt1993','bibtex')">BibTeX</a>]</p></td>
	<td>1993</td>
	<td>Biochemistry<br/>Vol. 32(45), pp. 11943 - 11952&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi00096a001">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Gerlt1993" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The AG*int for abstraction of the a-proton of a carbon acid is - 12 kcal/mol ( &amp;\ , 1993); the AG*int for abstraction of the proton from a normal acid is 1 3 kcal/ mol (Kresge, 1975). the carbonyl oxygen as the a-proton is abstracted ( &amp;\ , 1993).</td>
</tr>
<tr id="bib_Gerlt1993" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gerlt1993,
  author = {Gerlt, John A and Gassman, Paul G},
  title = {Understanding the rates of certain enzyme-catalyzed reactions: Proton abstraction from carbon acids, acyl transfer reactions, and displacement reactions of phosphodiesters},
  journal = {Biochemistry},
  year = {1993},
  volume = {32},
  number = {45},
  pages = {11943 - 11952},
  doi = {https://doi.org/10.1021/bi00096a001}
}
</pre></td>
</tr>
<tr id="Ghadie2017" class="entry">
	<td>Ghadie, M., Lambourne, L., Vidal, M. and Xia, Y.</td>
	<td>Domain-based prediction of the human isoform interactome provides insights into the functional impact of alternative splicing <p class="infolinks">[<a href="javascript:toggleInfo('Ghadie2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>PLoS Computational Biology<br/>Vol. 13(8), pp. e1005717&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pcbi.1005717">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Ghadie2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ghadie2017,
  author = {Ghadie, Mohamed and Lambourne, Luke and Vidal, Marc and Xia, Yu},
  title = {Domain-based prediction of the human isoform interactome provides insights into the functional impact of alternative splicing},
  journal = {PLoS Computational Biology},
  year = {2017},
  volume = {13},
  number = {8},
  pages = {e1005717},
  doi = {https://doi.org/10.1371/journal.pcbi.1005717}
}
</pre></td>
</tr>
<tr id="Ghaemi0" class="entry">
	<td>Ghaemi, Z., Guzman, I., Gnutt, D., Luthey-Schulten, Z. and Gruebele, M.</td>
	<td>Role of Electrostatics in Protein–RNA Binding: The Global vs the Local Energy Landscape <p class="infolinks">[<a href="javascript:toggleInfo('Ghaemi0','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>The Journal of Physical Chemistry B<br/>Vol. 0(0), pp. null&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jpcb.7b04318">DOI</a> <a href=" 
        http://dx.doi.org/10.1021/acs.jpcb.7b04318
    
">URL</a>&nbsp;</td>
</tr>
<tr id="bib_Ghaemi0" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ghaemi0,
  author = {Ghaemi, Zhaleh and Guzman, Irisbel and Gnutt, David and Luthey-Schulten, Zaida and Gruebele, Martin},
  title = {Role of Electrostatics in Protein–RNA Binding: The Global vs the Local Energy Landscape},
  journal = {The Journal of Physical Chemistry B},
  year = {2017},
  volume = {0},
  number = {0},
  pages = {null},
  url = { <br>        http://dx.doi.org/10.1021/acs.jpcb.7b04318<br>    <br>},
  doi = {https://doi.org/10.1021/acs.jpcb.7b04318}
}
</pre></td>
</tr>
<tr id="Giani2017" class="entry">
	<td>Giani, M., den Otter, W.K. and Briels, W.J.</td>
	<td>Early stages of clathrin aggregation at a membrane in coarse-grained simulations. <p class="infolinks">[<a href="javascript:toggleInfo('Giani2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Giani2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>The Journal of chemical physics<br/>Vol. 146, pp. 155102&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.4979985">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Giani2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The self-assembly process of clathrin coated pits during endocytosis has been simulated by combining and extending coarse grained models of the clathrin triskelion, the adaptor protein AP2, and a flexible network membrane. The AP2's core, upon binding to membrane and cargo, releases a motif that can bind clathrin. In conditions where the core-membrane-cargo binding is weak, the binding of this motif to clathrin can result in a stable complex. We characterize the conditions and mechanisms resulting in the formation of clathrin lattices that curve the membrane, i.e., clathrin coated pits. The mechanical properties of the AP2 β linker appear crucial to the orientation of the curved clathrin lattice relative to the membrane, with wild-type short linkers giving rise to the inward curving buds enabling endocytosis while long linkers produce upside-down cages and outward curving bulges.</td>
</tr>
<tr id="bib_Giani2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Giani2017,
  author = {Giani, M and den Otter, W K and Briels, W J},
  title = {Early stages of clathrin aggregation at a membrane in coarse-grained simulations.},
  journal = {The Journal of chemical physics},
  year = {2017},
  volume = {146},
  pages = {155102},
  doi = {https://doi.org/10.1063/1.4979985}
}
</pre></td>
</tr>
<tr id="Gillespie1976" class="entry">
	<td>Gillespie, D.T.</td>
	<td>A general method for numerically simulating the stochastic time evolution of coupled chemical reactions <p class="infolinks">[<a href="javascript:toggleInfo('Gillespie1976','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gillespie1976','bibtex')">BibTeX</a>]</p></td>
	<td>1976</td>
	<td>Journal of Computational Physics<br/>Vol. 22(4), pp. 403 - 434&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/0021-9991(76)90041-3">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Gillespie1976" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: An exact method is presented for numerically calculating, within the framework of the stochastic formulation of chemical kinetics, the time evolution of any spatially homogeneous mixture of molecular species which interreact through a specified set of coupled chemical reaction channels. The method is a compact, computer-oriented, Monte Carlo simulation procedure. It should be particularly useful for modeling the transient behavior of well-mixed gas-phase systems in which many molecular species participate in many highly coupled chemical reactions. For "ordinary" chemical systems in which fluctuations and correlations play no significant role, the method stands as an alternative to the traditional procedure of numerically solving the deterministic reaction rate equations. For nonlinear systems near chemical instabilities, where fluctuations and correlations may invalidate the deterministic equations, the method constitutes an efficient way of numerically examining the predictions of the stochastic master equation. Although fully equivalent to the spatially homogeneous master equation, the numerical simulation algorithm presented here is more directly based on a newly defined entity called "the reaction probability density function." The purpose of this article is to describe the mechanics of the simulation algorithm, and to establish in a rigorous, a priori manner its physical and mathematical validity; numerical applications to specific chemical systems will be presented in subsequent publications. ?? 1976.</td>
</tr>
<tr id="bib_Gillespie1976" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gillespie1976,
  author = {Gillespie, Daniel T},
  title = {A general method for numerically simulating the stochastic time evolution of coupled chemical reactions},
  journal = {Journal of Computational Physics},
  year = {1976},
  volume = {22},
  number = {4},
  pages = {403 - 434},
  doi = {https://doi.org/10.1016/0021-9991(76)90041-3}
}
</pre></td>
</tr>
<tr id="Gillespie1992" class="entry">
	<td>Gillespie, D.T.</td>
	<td>A rigorous derivation of the chemical master equation <p class="infolinks">[<a href="javascript:toggleInfo('Gillespie1992','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gillespie1992','bibtex')">BibTeX</a>]</p></td>
	<td>1992</td>
	<td>Physica A: Statistical Mechanics and its Applications<br/>Vol. 188(1-3), pp. 404 - 425&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/0378-4371(92)90283-V">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Gillespie1992" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: It is widely believed that the chemical master equation has no rigorous microphysical basis, and hence no a priori claim to validity. This view is challenged here through arguments purporting to show that the chemical master equation is exact for any gas-phase chemical system that is kept well stirred and thermally equilibrated. textcopyright 1992.</td>
</tr>
<tr id="bib_Gillespie1992" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gillespie1992,
  author = {Gillespie, Daniel T},
  title = {A rigorous derivation of the chemical master equation},
  journal = {Physica A: Statistical Mechanics and its Applications},
  year = {1992},
  volume = {188},
  number = {1-3},
  pages = {404 - 425},
  doi = {https://doi.org/10.1016/0378-4371(92)90283-V}
}
</pre></td>
</tr>
<tr id="Gillespie2007" class="entry">
	<td>Gillespie, D.T.</td>
	<td>Stochastic simulation of chemical kinetics <p class="infolinks">[<a href="javascript:toggleInfo('Gillespie2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gillespie2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Annual Review of Physical Chemistry<br/>Vol. 58(1), pp. 35 - 55&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1146/annurev.physchem.58.032806.104637">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Gillespie2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Stochastic chemical kinetics describes the time evolution of a well-stirred chemically reacting system in a way that takes into account the fact that molecules come in whole numbers and exhibit some degree of randomness in their dynamical behavior. Researchers are increasingly using this approach to chemical kinetics in the analysis of cellular systems in biology, where the small molecular populations of only a few reactant species can lead to deviations from the predictions of the deterministic differential equations of classical chemical kinetics. After reviewing the supporting theory of stochastic chemical kinetics, I discuss some recent advances in methods for using that theory to make numerical simulations. These include improvements to the exact stochastic simulation algorithm (SSA) and the approximate explicit tau-leaping procedure, as well as the development of two approximate strategies for simulating systems that are dynamically stiff: implicit tau-leaping and the slow-scale SSA.</td>
</tr>
<tr id="bib_Gillespie2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gillespie2007,
  author = {Gillespie, Daniel T},
  title = {Stochastic simulation of chemical kinetics},
  journal = {Annual Review of Physical Chemistry},
  year = {2007},
  volume = {58},
  number = {1},
  pages = {35 - 55},
  doi = {https://doi.org/10.1146/annurev.physchem.58.032806.104637}
}
</pre></td>
</tr>
<tr id="Gillespie2013" class="entry">
	<td>Gillespie, D.T., Hellander, A. and Petzold, L.R.</td>
	<td>Perspective: Stochastic algorithms for chemical kinetics <p class="infolinks">[<a href="javascript:toggleInfo('Gillespie2013','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gillespie2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Journal of Chemical Physics<br/>Vol. 138(17), pp. 170901&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.4801941">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Gillespie2013" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We outline our perspective on stochastic chemical kinetics, paying particular attention to numerical simulation algorithms. We first focus on dilute, well-mixed systems, whose description using ordinary differential equations has served as the basis for traditional chemical kinetics for the past 150 years. For such systems, we review the physical and mathematical rationale for a discrete-stochastic approach, and for the approximations that need to be made in order to regain the traditional continuous-deterministic description. We next take note of some of the more promising strategies for dealing stochastically with stiff systems, rare events, and sensitivity analysis. Finally, we review some recent efforts to adapt and extend the discrete-stochastic approach to systems that are not well-mixed. In that currently developing area, we focus mainly on the strategy of subdividing the system into well-mixed subvolumes, and then simulating diffusional transfers of reactant molecules between adjacent subvolumes together with chemical reactions inside the subvolumes.</td>
</tr>
<tr id="bib_Gillespie2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gillespie2013,
  author = {Gillespie, Daniel T and Hellander, Andreas and Petzold, Linda R},
  title = {Perspective: Stochastic algorithms for chemical kinetics},
  journal = {Journal of Chemical Physics},
  year = {2013},
  volume = {138},
  number = {17},
  pages = {170901},
  doi = {https://doi.org/10.1063/1.4801941}
}
</pre></td>
</tr>
<tr id="Gillesple1977" class="entry">
	<td>Gillesple, D.T.</td>
	<td>Exact Stochastic Simulation of Coupled Chemical Reactions <p class="infolinks">[<a href="javascript:toggleInfo('Gillesple1977','bibtex')">BibTeX</a>]</p></td>
	<td>1977</td>
	<td>Journal of Physical Chemistry<br/>Vol. 81(25), pp. 2340 - 2361&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Gillesple1977" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gillesple1977,
  author = {Gillesple, Daniel T},
  title = {Exact Stochastic Simulation of Coupled Chemical Reactions},
  journal = {Journal of Physical Chemistry},
  year = {1977},
  volume = {81},
  number = {25},
  pages = {2340 - 2361}
}
</pre></td>
</tr>
<tr id="Girard2005" class="entry">
	<td>Girard, M., Allaire, P.D., McPherson, P.S. and Blondeau, F.</td>
	<td>Non-stoichiometric relationship between clathrin heavy and light chains revealed by quantitative comparative proteomics of clathrin-coated vesicles from brain and liver. <p class="infolinks">[<a href="javascript:toggleInfo('Girard2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Girard2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Molecular &amp; cellular proteomics : MCP<br/>Vol. 4, pp. 1145-1154&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/mcp.M500043-MCP200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Girard2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We used tandem mass spectrometry with peptide counts to identify and to determine the relative levels of expression of abundant protein components of highly enriched clathrin-coated vesicles (CCVs) from rat liver. The stoichiometry of stable protein complexes including clathrin heavy chain and clathrin light chain dimers and adaptor protein (AP) heterotetramers was assessed. We detected a deficit of clathrin light chain compared with clathrin heavy chain in non-brain tissues, suggesting a level of regulation of clathrin cage formation specific to brain. The high ratio of AP-1 to AP-2 in liver CCVs is reversed compared with brain where there is more AP-2 than AP-1. Despite this, general endocytic cargo proteins were readily detected in liver but not in brain CCVs, consistent with the previous demonstration that a major function for brain CCVs is recycling synaptic vesicles. Finally we identified 21 CCV-associated proteins in liver not yet characterized in mammals. Our results further validate the peptide accounting approach, reveal new information on the properties of CCVs, and allow for the use of quantitative proteomics to compare abundant components of organelles under different experimental and pathological conditions.</td>
</tr>
<tr id="bib_Girard2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Girard2005,
  author = {Girard, Martine and Allaire, Patrick D and McPherson, Peter S and Blondeau, Francois},
  title = {Non-stoichiometric relationship between clathrin heavy and light chains revealed by quantitative comparative proteomics of clathrin-coated vesicles from brain and liver.},
  journal = {Molecular &amp; cellular proteomics : MCP},
  year = {2005},
  volume = {4},
  pages = {1145--1154},
  doi = {https://doi.org/10.1074/mcp.M500043-MCP200}
}
</pre></td>
</tr>
<tr id="Gizynski2017" class="entry">
	<td>Gizynski, K. and Gorecki, J.</td>
	<td>Chemical memory with states coded in light controlled oscillations of interacting Belousov--Zhabotinsky droplets <p class="infolinks">[<a href="javascript:toggleInfo('Gizynski2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gizynski2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Phys. Chem. Chem. Phys.<br/>Vol. 19(19), pp. 6519 - 6531&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1039/c6cp07492h">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Gizynski2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The information storing potential of droplets, in which an oscillatory, photosensitive Belousov--Zhabotinsky (BZ) reaction proceeds, is investigated experimentally. We consider coupled oscillations in pairs and triplets of droplets. Droplets are surrounded by a solution of lipids in decane. Oscillations synchronize via diffusion of an activator through a lipid bilayer. The reaction in each droplet can be individually controlled by illumi-nation with blue light through an optical fiber. We found that in pairs of BZ droplets, only the in-phase and the forcing oscillation modes are stable, however switching between these modes is not reliable. In triplets of droplets, switching between two different, stable rotational modes (clockwise and anticlockwise) can be easily implemented. Therefore, such a system is an excellent candidate for a light controlled, reliable, one bit chemical memory unit.</td>
</tr>
<tr id="bib_Gizynski2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gizynski2017,
  author = {Gizynski, Konrad and Gorecki, Jerzy},
  title = {Chemical memory with states coded in light controlled oscillations of interacting Belousov--Zhabotinsky droplets},
  journal = {Phys. Chem. Chem. Phys.},
  year = {2017},
  volume = {19},
  number = {19},
  pages = {6519 - 6531},
  doi = {https://doi.org/10.1039/c6cp07492h}
}
</pre></td>
</tr>
<tr id="Glasner2006" class="entry">
	<td>Glasner, M., Gerlt, J. and Babbitt, P.</td>
	<td>Evolution of enzyme superfamilies <p class="infolinks">[<a href="javascript:toggleInfo('Glasner2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Glasner2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Current Opinion in Chemical Biology<br/>Vol. 10(5), pp. 492 - 497&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cbpa.2006.08.012">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Glasner2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Enzyme evolution is often constrained by aspects of catalysis. Sets of homologous proteins that catalyze different overall reactions but share an aspect of catalysis, such as a common partial reaction, are called mechanistically diverse superfamilies. The common mechanistic steps and structural characteristics of several of these superfamilies, including the enolase, Nudix, amidohydrolase, and haloacid dehalogenase superfamilies have been characterized. In addition, studies of mechanistically diverse superfamilies are helping to elucidate mechanisms of functional diversification, such as catalytic promiscuity. Understanding how enzyme superfamilies evolve is vital for accurate genome annotation, predicting protein functions, and protein engineering.</td>
</tr>
<tr id="bib_Glasner2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Glasner2006,
  author = {Glasner, M and Gerlt, J and Babbitt, P},
  title = {Evolution of enzyme superfamilies},
  journal = {Current Opinion in Chemical Biology},
  year = {2006},
  volume = {10},
  number = {5},
  pages = {492 - 497},
  note = {NULL},
  doi = {https://doi.org/10.1016/j.cbpa.2006.08.012}
}
</pre></td>
</tr>
<tr id="Goh2001" class="entry">
	<td>Goh, K., Kahng, B. and Kim, D.</td>
	<td>Universal behavior of load distribution in scale-free networks <p class="infolinks">[<a href="javascript:toggleInfo('Goh2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Goh2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Physical review letters<br/>Vol. 87(27 Pt 1), pp. 278701 - 278701&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1103/PhysRevLett.91.189804">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Goh2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We study a problem of data packet transport in scale-free networks whose degree distribution follows a power law with the exponent gamma. Load, or "betweenness centrality," of a vertex is the accumulated total number of data packets passing through that vertex when every pair of vertices sends and receives a data packet along the shortest path connecting the pair. It is found that the load distribution follows a power law with the exponent delta approximately 2.2(1), insensitive to different values of gamma in the range, 2 \textless\ gamma \textless\ or = 3, and different mean degrees, which is valid for both undirected and directed cases. Thus, we conjecture that the load exponent is a universal quantity to characterize scale-free networks.</td>
</tr>
<tr id="bib_Goh2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Goh2001,
  author = {Goh, KI and Kahng, B and Kim, D},
  title = {Universal behavior of load distribution in scale-free networks},
  journal = {Physical review letters},
  year = {2001},
  volume = {87},
  number = {27 Pt 1},
  pages = {278701 - 278701},
  doi = {https://doi.org/10.1103/PhysRevLett.91.189804}
}
</pre></td>
</tr>
<tr id="Gohlke2017" class="entry">
	<td>Gohlke, H., Ben-Shalom, I.Y., Kopitz, H., Pfeiffer-Marek, S. and Baringhaus, K.-H.</td>
	<td>Rigidity Theory-Based Approximation of Vibrational Entropy Changes upon Binding to Biomolecules <p class="infolinks">[<a href="javascript:toggleInfo('Gohlke2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gohlke2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of Chemical Theory and Computation<br/>Vol. 13(4), pp. 1495 - 1502&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jctc.7b00014">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Gohlke2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We introduce a computationally efficient approximation of vibrational entropy changes (ΔSvib) upon binding to biomolecules based on rigidity theory. From constraint network representations of the binding partners, ΔSvib is estimated from changes in the number of low frequency (``spongy'') modes with respect to changes in the networks' coordination number. Compared to ΔSvib computed by normal-mode analysis (NMA), our approach yields significant and good to fair correlations for data sets of protein--protein and protein--ligand complexes. Our approach could be a valuable alternative to NMA-based ΔSvib computation in end-point (free) energy methods.</td>
</tr>
<tr id="bib_Gohlke2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gohlke2017,
  author = {Gohlke, Holger and Ben-Shalom, Ido Y. and Kopitz, Hannes and Pfeiffer-Marek, Stefania and Baringhaus, Karl-Heinz},
  title = {Rigidity Theory-Based Approximation of Vibrational Entropy Changes upon Binding to Biomolecules},
  journal = {Journal of Chemical Theory and Computation},
  year = {2017},
  volume = {13},
  number = {4},
  pages = {1495 - 1502},
  doi = {https://doi.org/10.1021/acs.jctc.7b00014}
}
</pre></td>
</tr>
<tr id="Gomez1997" class="entry">
	<td>Gomez, M.S., Piper, R.C., Hunsaker, L.A., Royer, R.E., Deck, L.M., Makler, M.T. and Jagt, D.L.</td>
	<td>Substrate and cofactor specificity and selective inhibition of lactate dehydrogenase from the malarial parasite P. falciparum <p class="infolinks">[<a href="javascript:toggleInfo('Gomez1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gomez1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Molecular and Biochemical Parasitology<br/>Vol. 90(1), pp. 235 - 246&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S0166-6851(97)00140-0">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Gomez1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Lactate dehydrogenase from the malarial parasite Plasmodium falciparum has many amino acid residues that are unique compared to any other known lactate dehydrogenase. This includes residues that define the substrate and cofactor binding sites. Nevertheless, parasite lactate dehydrogenase exhibits high specificity for pyruvic acid, even more restricted than the specificity of human lactate dehydrogenases M4 and H4. Parasite lactate dehydrogenase exhibits high catalytic efficiency in the reduction of pyruvate, k(cat)/K(m) = 9.0 x 108 min-1 M-1. Parasite lactate dehydrogenase also exhibits similar cofactor specificity to the human isoforms in the oxidation of L-lactate with NAD+ and with a series of NAD+ analogs, suggesting a similar cofactor binding environment in spite of the numerous amino acid differences. Parasite lactate dehydrogenase exhibits an enhanced k(cat) with the analog 3-acetylpyridine adenine dinucleotide (APAD+) whereas the human isoforms exhibit a lower k(cat). This differential response to APAD+ provides the kinetic basis for the enzyme-based detection of malarial parasites. A series of inhibitors structurally related to the natural product gossypol were shown to be competitive inhibitors of the binding of NADH. Slight changes in structure produced marked changes in selectivity of inhibition of lactate dehydrogenase. 7-p-Trifluoromethylbenzyl-8-deoxyhemigossylic acid inhibited parasite lactate dehydrogenase, K(i) = 0.2 ??M, which was 65- and 400-fold tighter binding compared to the M4 and H4 isoforms of human lactate dehydrogenase. The results suggest that the cofactor site of parasite lactate dehydrogenase may be a potential target for structure-based drug design.</td>
</tr>
<tr id="bib_Gomez1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gomez1997,
  author = {Gomez, Manuel S and Piper, Robert C and Hunsaker, Lucy A and Royer, Robert E and Deck, Lorraine M and Makler, Michael T and Jagt, David L},
  title = {Substrate and cofactor specificity and selective inhibition of lactate dehydrogenase from the malarial parasite P. falciparum},
  journal = {Molecular and Biochemical Parasitology},
  year = {1997},
  volume = {90},
  number = {1},
  pages = {235 - 246},
  note = {NULL},
  doi = {https://doi.org/10.1016/S0166-6851(97)00140-0}
}
</pre></td>
</tr>
<tr id="Gottfried2009" class="entry">
	<td>Gottfried, I., Ehrlich, M. and Ashery, U.</td>
	<td>HIP1 exhibits an early recruitment and a late stage function in the maturation of coated pits. <p class="infolinks">[<a href="javascript:toggleInfo('Gottfried2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gottfried2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Cellular and molecular life sciences : CMLS<br/>Vol. 66, pp. 2897-2911&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1007/s00018-009-0077-4">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Gottfried2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Huntingtin interacting protein 1 (HIP1) is an accessory protein of the clathrin-mediated endocytosis (CME) pathway, yet its precise role and the step at which it becomes involved are unclear. We employed live-cell imaging techniques to focus on the early steps of CME and characterize HIP1 dynamics. We show that HIP1 is highly colocalized with clathrin at the plasma membrane and shares similar dynamics with a subpopulation of clathrin assemblies. Employing transferrin receptor fused to pHluorin, we distinguished between open pits to which HIP1 localizes and newly internalized vesicles that are devoid of HIP1. Moreover, shRNA knockdown of clathrin compromised HIP1 membranal localization, unlike the reported behavior of Sla2p. HIP1 fragment, lacking its ANTH and Talin-like domains, inhibits internalization of transferrin, but retains colocalization with membranal clathrin assemblies. These data demonstrate HIP1's role in pits maturation and formation of the coated vesicle, and its strong dependence on clathrin for membranal localization.</td>
</tr>
<tr id="bib_Gottfried2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gottfried2009,
  author = {Gottfried, Irit and Ehrlich, Marcelo and Ashery, Uri},
  title = {HIP1 exhibits an early recruitment and a late stage function in the maturation of coated pits.},
  journal = {Cellular and molecular life sciences : CMLS},
  year = {2009},
  volume = {66},
  pages = {2897--2911},
  note = {though HIP1 has an ENTH domain, its localization on the PM is not dependent on it, it is dependent on clathrin - ergo, HIP1 does not play a role in recruiting clathrin},
  doi = {https://doi.org/10.1007/s00018-009-0077-4}
}
</pre></td>
</tr>
<tr id="Grassein2018" class="entry">
	<td>Grassein, P., Delarue, P., Scheraga, H., Maisuradze, G. and Senet, P.</td>
	<td>A Statistical Model to Decipher Protein Folding/unfolding at a Local Scale. <p class="infolinks">[<a href="javascript:toggleInfo('Grassein2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>The journal of physical chemistry. B&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Grassein2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Grassein2018,
  author = {Grassein, P and Delarue, P and Scheraga, HA and Maisuradze, GG and Senet, P},
  title = {A Statistical Model to Decipher Protein Folding/unfolding at a Local Scale.},
  journal = {The journal of physical chemistry. B},
  year = {2018}
}
</pre></td>
</tr>
<tr id="Greaves2017" class="entry">
	<td>Greaves, R.B., Dietmann, S., Smith, A., Stepney, S. and Halley, J.D.</td>
	<td>A conceptual and computational framework for modelling and understanding the non-equilibrium gene regulatory networks of mouse embryonic stem cells <p class="infolinks">[<a href="javascript:toggleInfo('Greaves2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Greaves2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>PLOS Computational Biology<br/>Vol. 13(9), pp. 1-24&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pcbi.1005713">DOI</a> <a href="https://doi.org/10.1371/journal.pcbi.1005713">URL</a>&nbsp;</td>
</tr>
<tr id="abs_Greaves2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Author summary Pluripotent stem cells possess the capacity both to renew themselves indefinitely and to differentiate to any cell type in the body. Thus the ability to direct stem cell differentiation would have immense potential in regenerative medicine. There is a massive amount of biological data relevant to stem cells; here we exploit data relating to stem cell differentiation to help understand cell behaviour and complexity. These cells contain a dynamic, non-equilibrium network of genes regulated in part by transcription factors expressed by the network itself. Here we take an existing theoretical framework, Transcription Factor Branching Processes, which explains how these genetic networks can have critical behaviour, and can tip between low and full expression. We use this theory as the basis for the design and implementation of a computational simulation platform, which we then use to run a variety of simulation experiments, to gain a better understanding how these various transcription factors can combine, interact, and influence each other. The simulation parameters are derived from experimental data relating to the core factors in pluripotent stem cell differentiation. The simulation results determine the critical values of branching process parameters, and how these are modulated by the various interacting transcription factors.</td>
</tr>
<tr id="bib_Greaves2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Greaves2017,
  author = {Greaves, Richard B. AND Dietmann, Sabine AND Smith, Austin AND Stepney, Susan AND Halley, Julianne D.},
  title = {A conceptual and computational framework for modelling and understanding the non-equilibrium gene regulatory networks of mouse embryonic stem cells},
  journal = {PLOS Computational Biology},
  publisher = {Public Library of Science},
  year = {2017},
  volume = {13},
  number = {9},
  pages = {1-24},
  url = {https://doi.org/10.1371/journal.pcbi.1005713},
  doi = {https://doi.org/10.1371/journal.pcbi.1005713}
}
</pre></td>
</tr>
<tr id="Green1993" class="entry">
	<td>Green, D.W., Sun, H. and Plapp, B.V.</td>
	<td>Inversion of the substrate specificity of yeast alcohol dehydrogenase <p class="infolinks">[<a href="javascript:toggleInfo('Green1993','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Green1993','bibtex')">BibTeX</a>]</p></td>
	<td>1993</td>
	<td>Journal of Biological Chemistry<br/>Vol. 268(11), pp. 7792 - 7798&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Green1993" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The relationship between the size of the substrate binding pocket and the catalytic reactivities with varied alcohols was studied with the Saccharomyces cerevisiae alcohol dehydrogenase I (ScADH) and compared with the liver enzymes from horse (EqADH, EE isoenzyme) and monkey (MmADH alpha, alpha-isoenzyme). The yeast enzyme is most active with ethanol, and its activity decreases as the size of the alcohol is increased, whereas the activities of the liver enzymes increase with larger alcohols. The substrate pocket in ScADH was enlarged by single substitutions of Thr-48 to Ser (T48S), Trp-57 to Met (W57M), and Trp-93 to Ala (W93A), and a double change, T48S:W93A, and a triple, T48S:W57M:W93A. The T48S enzyme has the same pattern of activity (V/K) as wild-type ScADH for linear primary alcohols. The W57M enzymes have lowered reactivity with primary and secondary alcohols. The W93A and T48S:W93A enzymes resemble MmADH alpha in having an inverted specificity pattern for primary alcohols, being 3- and 10-fold more active on hexanol and 350- and 540-fold less active on ethanol, and are as reactive as the liver enzymes with long chain primary alcohols. The three Ala-93 enzymes also acquired weak activity on branched chain alcohols and cyclohexanol.</td>
</tr>
<tr id="bib_Green1993" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Green1993,
  author = {Green, David W and Sun, Hong and Plapp, Bryce V},
  title = {Inversion of the substrate specificity of yeast alcohol dehydrogenase},
  journal = {Journal of Biological Chemistry},
  year = {1993},
  volume = {268},
  number = {11},
  pages = {7792 - 7798}
}
</pre></td>
</tr>
<tr id="Groot1996" class="entry">
	<td>de Groot, B., Amadei, A., Scheek, R., van Nuland, N. and Berendsen, H.</td>
	<td>An extended sampling of the configurational space of HPr from E. coli. <p class="infolinks">[<a href="javascript:toggleInfo('Groot1996','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Groot1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>Proteins<br/>Vol. 26(3), pp. 314 - 22&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/(SICI)1097-0134(199611)26">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Groot1996" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Recently, we developed a method (Amadei et al., J. Biomol. Str. Dyn. 13: 615-626; de Groot et al., J. Biomol. Str. Dyn. 13: 741-751, 1996) to obtain an extended sampling of the configurational space of proteins, using an adapted form of molecular dynamics (MD) simulations, based on the essential dynamics (ED) (Amadei et al., Proteins 17:412-425, 1993) method. In the present study, this ED sampling technique is applied to the histidine-containing phosphocarrier protein HPr from Escherichia coli. We find a cluster of conformations that is an order of magnitude larger than that found for a usual MD simulation of comparable length. The structures in this cluster are geometrically and energetically comparable to NMR structures. Moreover, on average, this large cluster satisfies nearly all NMR-derived distance restraints.</td>
</tr>
<tr id="bib_Groot1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Groot1996,
  author = {de Groot, BL and Amadei, A and Scheek, RM and van Nuland, NA and Berendsen, HJ},
  title = {An extended sampling of the configurational space of HPr from E. coli.},
  journal = {Proteins},
  year = {1996},
  volume = {26},
  number = {3},
  pages = {314 - 22},
  note = {Application of essential dynamics analysis by original authors.},
  doi = {https://doi.org/10.1002/(SICI)1097-0134(199611)26}
}
</pre></td>
</tr>
<tr id="Grote1980" class="entry">
	<td>Grote, R.F. and Hynes, J.T.</td>
	<td>The stable states picture of chemical reactions. II. Rate constants for condensed and gas phase reaction models <p class="infolinks">[<a href="javascript:toggleInfo('Grote1980','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Grote1980','bibtex')">BibTeX</a>]</p></td>
	<td>1980</td>
	<td>The Journal of Chemical Physics<br/>Vol. 73(6), pp. 2715 - 2715&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.440485">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Grote1980" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The time correlation function (tcf) formulas for rate constants κ derived via the stable states picture (SSP) of chemical reactions are applied to a wide variety (a--d) of gas and solution phase reaction models. (a) For gas phase bimolecular reactions, we show that the flux tcf governing κ corresponds to standard numerical trajectory calculation methods. Alternate formulas for κ are derived which focus on saddle point surfaces, thus increasing computational efficiency. Advantages of the SSP formulas for κ are discussed. (b) For gas phase unimolecular reactions, simple results for κ are found in both the strong and weak coupling collision limits; the often ignored role of product stabilization is exposed for reversible isomerizations. The SSP results correct some standard weak coupling rate constant results by as much as 50%. (c) For barrier crossing reactions in solution, we evaluate κ for a generalized (non‐Markovian) Langevin description of the dynamics. For several realistic models of time dependent friction, κ differs dramatically from the popular Kramers constant friction predictions; this has important implications for the validity of transition state theory. (d) For solution reactions heavily influenced by spatial diffusion, we show that the SSP isolates short range reaction dynamics of interest and includes important barrier region effects in structural isomerizations often missed in standard descriptions.</td>
</tr>
<tr id="bib_Grote1980" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Grote1980,
  author = {Grote, Richard F and Hynes, James T},
  title = {The stable states picture of chemical reactions. II. Rate constants for condensed and gas phase reaction models},
  journal = {The Journal of Chemical Physics},
  year = {1980},
  volume = {73},
  number = {6},
  pages = {2715 - 2715},
  doi = {https://doi.org/10.1063/1.440485}
}
</pre></td>
</tr>
<tr id="Gu2013" class="entry">
	<td>Gu, H. and Zhang, S.</td>
	<td>Advances in kinetic isotope effect measurement techniques for enzyme mechanism study. <p class="infolinks">[<a href="javascript:toggleInfo('Gu2013','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gu2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Molecules (Basel, Switzerland)<br/>Vol. 18(8), pp. 9278 - 92&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.3390/molecules18089278">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Gu2013" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Kinetic isotope effects (KIEs) are a very powerful tool for investigating enzyme mechanisms. Precision of measurement is the most important factor for KIE determinations, especially for small heavy atom KIEs. Internal competition is commonly used to measure small KIEs on V/K. Several methods, including such as liquid scintillation counting, mass spectrometry, nuclear magnetic resonance spectroscopy and polarimetry have been used to determine KIEs. In this paper, which does not aspire to be an exhaustive review, we briefly review different experimental approaches for the measurement of KIEs on enzymatic reaction with an emphasis on newer techniques employing mass spectrometry and nuclear magnetic resonance spectrometry as well as some corresponding examples.</td>
</tr>
<tr id="bib_Gu2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gu2013,
  author = {Gu, Hong and Zhang, Shuming},
  title = {Advances in kinetic isotope effect measurement techniques for enzyme mechanism study.},
  journal = {Molecules (Basel, Switzerland)},
  year = {2013},
  volume = {18},
  number = {8},
  pages = {9278 - 92},
  doi = {https://doi.org/10.3390/molecules18089278}
}
</pre></td>
</tr>
<tr id="Guillaume-Gentil2016" class="entry">
	<td>Guillaume-Gentil, O., Grindberg, R.V., Kooger, R., Dorwling-Carter, L., Martinez, V., Ossola, D., Pilhofer, M., Zambelli, T. and Vorholt, J.A.</td>
	<td>Tunable Single-Cell Extraction for Molecular Analyses. <p class="infolinks">[<a href="javascript:toggleInfo('Guillaume-Gentil2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Guillaume-Gentil2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Cell<br/>Vol. 166, pp. 506-516&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cell.2016.06.025">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Guillaume-Gentil2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Because of cellular heterogeneity, the analysis of endogenous molecules from single cells is of significant interest and has major implications. While micromanipulation or cell sorting followed by cell lysis is already used for subsequent molecular examinations, approaches to directly extract the content of living cells remain a challenging but promising alternative to achieving non-destructive sampling and cell-context preservation. Here, we demonstrate the quantitative extraction from single cells with spatiotemporal control using fluidic force microscopy. We further present a comprehensive analysis of the soluble molecules withdrawn from the cytoplasm or the nucleus, including the detection of enzyme activities and transcript abundances. This approach has uncovered the ability of cells to withstand extraction of up to several picoliters and opens opportunities to study cellular dynamics and cell-cell communication under physiological conditions at the single-cell level.</td>
</tr>
<tr id="bib_Guillaume-Gentil2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Guillaume-Gentil2016,
  author = {Guillaume-Gentil, Orane and Grindberg, Rashel V and Kooger, Romain and Dorwling-Carter, Livie and Martinez, Vincent and Ossola, Dario and Pilhofer, Martin and Zambelli, Tomaso and Vorholt, Julia A},
  title = {Tunable Single-Cell Extraction for Molecular Analyses.},
  journal = {Cell},
  year = {2016},
  volume = {166},
  pages = {506--516},
  note = {HeLa cells have a median cytoplasmic volume of 1.6 pL},
  doi = {https://doi.org/10.1016/j.cell.2016.06.025}
}
</pre></td>
</tr>
<tr id="Gunsteren1988" class="entry">
	<td>Gunsteren, V.W. and Berendsen, H.</td>
	<td>A Leap-frog Algorithm for Stochastic Dynamics <p class="infolinks">[<a href="javascript:toggleInfo('Gunsteren1988','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Gunsteren1988','bibtex')">BibTeX</a>]</p></td>
	<td>1988</td>
	<td>Molecular Simulation<br/>Vol. 1(3), pp. 173 - 185&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1080/08927028808080941">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Gunsteren1988" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A third-order algorithm for stochastic dynamics (SD) simulations is\textbackslash\nproposed, identical to the powerful molecular dynamics leap-frog\textbackslash\nalgorithm in the limit of infinitely small friction coefficient &Icirc;textthreesuperior.\textbackslash\nIt belongs to the class of SD algorithms, in which the integration\textbackslash\ntime step &Icirc;t is not limited by the condition &Icirc;t &acirc;¤\textbackslash\n&Icirc;textthreesuperior-1, but only by the properties of the systematic force.\textbackslash\nIt is shown how constraints, such as bond length or bond angle constraints,\textbackslash\ncan be incorporated in the computational scheme. It is argued that\textbackslash\nthe third-order Verlet-type SD algorithm proposed earlier may be\textbackslash\nsimplified without loosing its third-order accuracy. The leap-frog\textbackslash\nSD algorithm is proven to be equivalent to the verlet-type SD algorithm.\textbackslash\nBoth these SD algorithms are slightly more economical on computer\textbackslash\nstorage than the Beeman-type SD algorithm.</td>
</tr>
<tr id="bib_Gunsteren1988" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Gunsteren1988,
  author = {Gunsteren, Van WF and Berendsen, HJC},
  title = {A Leap-frog Algorithm for Stochastic Dynamics},
  journal = {Molecular Simulation},
  year = {1988},
  volume = {1},
  number = {3},
  pages = {173 - 185},
  doi = {https://doi.org/10.1080/08927028808080941}
}
</pre></td>
</tr>
<tr id="Haar2000" class="entry">
	<td>ter Haar, E., Harrison, S.C. and Kirchhausen, T.</td>
	<td>Peptide-in-groove interactions link target proteins to the beta-propeller of clathrin. <p class="infolinks">[<a href="javascript:toggleInfo('Haar2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Haar2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Proceedings of the National Academy of Sciences of the United States of America<br/>Vol. 97, pp. 1096-1100&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Haar2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The "WD40" domain is a widespread recognition module for linking partner proteins in intracellular networks of signaling and sorting. The clathrin amino-terminal domain, which directs incorporation of cargo into coated pits, is a beta-propeller closely related in structure to WD40 modules. The crystallographically determined structures of complexes of the clathrin-terminal domain with peptides derived from two different cargo adaptors, beta-arrestin 2 and the beta-subunit of the AP-3 complex, reveal strikingly similar peptide-in-groove interactions. The two peptides in our structures contain related, five-residue motifs, which form the core of their contact with clathrin. A number of other proteins involved in endocytosis have similar "clathrin-box" motifs, and it therefore is likely that they all bind the terminal domain in the same way. We propose that a peptide-in-groove interaction is an important general mode by which beta-propellers recognize specific target proteins.</td>
</tr>
<tr id="bib_Haar2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Haar2000,
  author = {ter Haar, E and Harrison, S C and Kirchhausen, T},
  title = {Peptide-in-groove interactions link target proteins to the beta-propeller of clathrin.},
  journal = {Proceedings of the National Academy of Sciences of the United States of America},
  year = {2000},
  volume = {97},
  pages = {1096--1100},
  note = {AP2B1, AMPH, EPN1, AP180 have clathrin boxes. focuses on beta-arrestin}
}
</pre></td>
</tr>
<tr id="Haar1998" class="entry">
	<td>ter Haar, E., Musacchio, A., Harrison, S.C. and Kirchhausen, T.</td>
	<td>Atomic Structure of Clathrin: A β Propeller Terminal Domain Joins an α Zigzag Linker <p class="infolinks">[<a href="javascript:toggleInfo('Haar1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Cell<br/>Vol. 95(4), pp. 563 - 573&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S0092-8674(00)81623-2">DOI</a> <a href="http://www.sciencedirect.com/science/article/pii/S0092867400816232">URL</a>&nbsp;</td>
</tr>
<tr id="bib_Haar1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Haar1998,
  author = {Ernst ter Haar and Andrea Musacchio and Stephen C Harrison and Tomas Kirchhausen},
  title = {Atomic Structure of Clathrin: A β Propeller Terminal Domain Joins an α Zigzag Linker},
  journal = {Cell},
  year = {1998},
  volume = {95},
  number = {4},
  pages = {563 - 573},
  url = {http://www.sciencedirect.com/science/article/pii/S0092867400816232},
  doi = {https://doi.org/10.1016/S0092-8674(00)81623-2}
}
</pre></td>
</tr>
<tr id="Habermann2004" class="entry">
	<td>Habermann, B.</td>
	<td>The BAR-domain family of proteins: a case of bending and binding? <p class="infolinks">[<a href="javascript:toggleInfo('Habermann2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Habermann2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>EMBO reports<br/>Vol. 5, pp. 250-255&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/sj.embor.7400105">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Habermann2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: BAR-domains recently took centre stage in science through a report on the crystal structure of this domain in Drosophila Amphiphysin. Though only weakly conserved at the sequence level, the structure of the BAR domain shows striking similarity to the GTPase-binding domain of Arfaptin 2, an effector of Rho- and Arf- GTPases. On the basis of this sequence and structural similarity, these two proteins have been classified as belonging to the same family, the BAR-domain family, and they probably also have similar functional characteristics. Presented here are the results of a database search for the sequence of the BAR domain of Amphiphysin and Arfaptin 2. This search identified a variety of related proteins, most of which are involved in intracellular transport and especially in endocytosis. For example, the BAR-domain family includes Endophilins, GTPase-activating proteins of the Centaurinbeta family and Oligophrenins, the adaptor proteins APPL1 and APPL2 that were recently shown to interact with the small GTPase Rab5, as well as members of the Sorting nexin family. On the basis of the structures of Amphiphysin and Arfaptin 2 and the cellular role of Amphiphysins in the early steps of endocytosis, the functions of the BAR domain have been defined as a dimerization motif and as sensing and inducing membrane curvature. However, data on Arfaptin 2 and now also on the Adaptor proteins APPL1 and 2 suggest that another function of the BAR domain is to bind to small GTPases.</td>
</tr>
<tr id="bib_Habermann2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Habermann2004,
  author = {Habermann, Bianca},
  title = {The BAR-domain family of proteins: a case of bending and binding?},
  journal = {EMBO reports},
  year = {2004},
  volume = {5},
  pages = {250--255},
  doi = {https://doi.org/10.1038/sj.embor.7400105}
}
</pre></td>
</tr>
<tr id="Haldane2017" class="entry">
	<td>Haldane, A., Flynn, W.F., He, P. and Levy, R.M.</td>
	<td>Coevolutionary Landscape of Kinase Family Proteins: Sequence Probabilities and Functional Motifs <p class="infolinks">[<a href="javascript:toggleInfo('Haldane2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Biophysical Journal&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Haldane2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Haldane2017,
  author = {Haldane, Allan and Flynn, William F and He, Peng and Levy, Ronald M},
  title = {Coevolutionary Landscape of Kinase Family Proteins: Sequence Probabilities and Functional Motifs},
  journal = {Biophysical Journal},
  year = {2017}
}
</pre></td>
</tr>
<tr id="Hamelberg2004" class="entry">
	<td>Hamelberg, D., Mongan, J. and McCammon, A.J.</td>
	<td>Accelerated molecular dynamics: A promising and efficient simulation method for biomolecules <p class="infolinks">[<a href="javascript:toggleInfo('Hamelberg2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of Chemical Physics<br/>Vol. 120(24), pp. 11919 - 11929&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Hamelberg2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hamelberg2004,
  author = {Hamelberg, Donald and Mongan, John and McCammon, Andrew J.},
  title = {Accelerated molecular dynamics: A promising and efficient simulation method for biomolecules},
  journal = {Journal of Chemical Physics},
  year = {2004},
  volume = {120},
  number = {24},
  pages = {11919 - 11929}
}
</pre></td>
</tr>
<tr id="Hamelberg2007" class="entry">
	<td>Hamelberg, D., de Oliveira, C.F. and McCammon, A.J.</td>
	<td>Sampling of slow diffusive conformational transitions with accelerated molecular dynamics. <p class="infolinks">[<a href="javascript:toggleInfo('Hamelberg2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hamelberg2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>The Journal of chemical physics<br/>Vol. 127(15), pp. 155102 - 155102&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.2789432">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hamelberg2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Slow diffusive conformational transitions play key functional roles in biomolecular systems. Our ability to sample these motions with molecular dynamics simulation in explicit solvent is limited by the slow diffusion of the solvent molecules around the biomolecules. Previously, we proposed an accelerated molecular dynamics method that has been shown to efficiently sample the torsional degrees of freedom of biomolecules beyond the millisecond timescale. However, in our previous approach, large-amplitude displacements of biomolecules are still slowed by the diffusion of the solvent. Here we present a unified approach of efficiently sampling both the torsional degrees of freedom and the diffusive motions concurrently. We show that this approach samples the configuration space more efficiently than normal molecular dynamics and that ensemble averages converge faster to the correct values.</td>
</tr>
<tr id="bib_Hamelberg2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hamelberg2007,
  author = {Hamelberg, Donald and de Oliveira, Cesar F and McCammon, Andrew J.},
  title = {Sampling of slow diffusive conformational transitions with accelerated molecular dynamics.},
  journal = {The Journal of chemical physics},
  year = {2007},
  volume = {127},
  number = {15},
  pages = {155102 - 155102},
  note = {Use dual boost to do aMD in explicit solvent.},
  doi = {https://doi.org/10.1063/1.2789432}
}
</pre></td>
</tr>
<tr id="Hammes-Schiffer2002" class="entry">
	<td>Hammes-Schiffer, S.</td>
	<td>Comparison of hydride, hydrogen atom, and proton-coupled electron transfer reactions. <p class="infolinks">[<a href="javascript:toggleInfo('Hammes-Schiffer2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hammes-Schiffer2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Chemphyschem : a European journal of chemical physics and physical chemistry<br/>Vol. 3(1), pp. 33 - 42&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/1439-7641(20020118)3">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hammes-Schiffer2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A comparison of hydride, hydrogen atom, and proton-coupled electron transfer reactions is presented. Herein, hydride and hydrogen atom transfer refer to reactions in which the electrons and protons transfer between the same donor and acceptor, while proton-coupled electron transfer (PCET) refers to reactions in which the electrons and protons transfer between different centers. Within these definitions, hydride and hydrogen atom transfer reactions are typically electronically adiabatic, hence evolving on a single electronic surface. In contrast, PCET reactions are often electronically nonadiabatic since the electron transfers a longer distance through a proton transfer interface. For all three types of reactions, solute reorganization is important, particularly the hydrogen donor--acceptor mode. Solvent reorganization is critical for hydride transfer and PCET, which involve significant solute charge redistribution, but not for hydrogen atom transfer. Theoretical descriptions and simulation methodology for all three types of reactions are presented, as well as experimentally relevant applications to hydride transfer in enzymes and PCET in solution.</td>
</tr>
<tr id="bib_Hammes-Schiffer2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hammes-Schiffer2002,
  author = {Hammes-Schiffer, Sharon},
  title = {Comparison of hydride, hydrogen atom, and proton-coupled electron transfer reactions.},
  journal = {Chemphyschem : a European journal of chemical physics and physical chemistry},
  year = {2002},
  volume = {3},
  number = {1},
  pages = {33 - 42},
  doi = {https://doi.org/10.1002/1439-7641(20020118)3}
}
</pre></td>
</tr>
<tr id="Hammes-Schiffer2006" class="entry">
	<td>Hammes-Schiffer, S.</td>
	<td>Hydrogen tunneling and protein motion in enzyme reactions <p class="infolinks">[<a href="javascript:toggleInfo('Hammes-Schiffer2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hammes-Schiffer2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Acc. Chem. Res.<br/>Vol. 39(2), pp. 93 - 100&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ar040199a">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hammes-Schiffer2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Tunneling important Dynamical effects not important</td>
</tr>
<tr id="bib_Hammes-Schiffer2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hammes-Schiffer2006,
  author = {Hammes-Schiffer, Sharon},
  title = {Hydrogen tunneling and protein motion in enzyme reactions},
  journal = {Acc. Chem. Res.},
  year = {2006},
  volume = {39},
  number = {2},
  pages = {93 - 100},
  doi = {https://doi.org/10.1021/ar040199a}
}
</pre></td>
</tr>
<tr id="Han2004" class="entry">
	<td>Han, J.-D.J., Bertin, N., Hao, T., Goldberg, D.S., Berriz, G.F., Zhang, L.V., Dupuy, D., Walhout, A.J., Cusick, M.E., Roth, F.P. and Vidal, M.</td>
	<td>Evidence for dynamically organized modularity in the yeast protein-protein interaction network. <p class="infolinks">[<a href="javascript:toggleInfo('Han2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Han2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Nature<br/>Vol. 430(6995), pp. 88 - 93&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nature02795">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Han2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In apparently scale-free protein-protein interaction networks, or 'interactome' networks, most proteins interact with few partners, whereas a small but significant proportion of proteins, the 'hubs', interact with many partners. Both biological and non-biological scale-free networks are particularly resistant to random node removal but are extremely sensitive to the targeted removal of hubs. A link between the potential scale-free topology of interactome networks and genetic robustness seems to exist, because knockouts of yeast genes encoding hubs are approximately threefold more likely to confer lethality than those of non-hubs. Here we investigate how hubs might contribute to robustness and other cellular properties for protein-protein interactions dynamically regulated both in time and in space. We uncovered two types of hub: 'party' hubs, which interact with most of their partners simultaneously, and 'date' hubs, which bind their different partners at different times or locations. Both in silico studies of network connectivity and genetic interactions described in vivo support a model of organized modularity in which date hubs organize the proteome, connecting biological processes--or modules--to each other, whereas party hubs function inside modules.</td>
</tr>
<tr id="bib_Han2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Han2004,
  author = {Han, Jing-Dong J and Bertin, Nicolas and Hao, Tong and Goldberg, Debra S and Berriz, Gabriel F and Zhang, Lan V and Dupuy, Denis and Walhout, Albertha JM and Cusick, Michael E and Roth, Frederick P and Vidal, Marc},
  title = {Evidence for dynamically organized modularity in the yeast protein-protein interaction network.},
  journal = {Nature},
  year = {2004},
  volume = {430},
  number = {6995},
  pages = {88 - 93},
  doi = {https://doi.org/10.1038/nature02795}
}
</pre></td>
</tr>
<tr id="Handel2017" class="entry">
	<td>Handel, A.</td>
	<td>Learning infectious disease epidemiology in a modern framework <p class="infolinks">[<a href="javascript:toggleInfo('Handel2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>PLOS Computational Biology<br/>Vol. 13(10), pp. e1005642&nbsp;</td>
	<td>misc</td>
	<td><a href="https://doi.org/10.1371/journal.pcbi.1005642">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Handel2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@misc{Handel2017,
  author = {Andreas Handel},
  title = {Learning infectious disease epidemiology in a modern framework},
  journal = {PLOS Computational Biology},
  publisher = {Public Library of Science (PLoS)},
  year = {2017},
  volume = {13},
  number = {10},
  pages = {e1005642},
  doi = {https://doi.org/10.1371/journal.pcbi.1005642}
}
</pre></td>
</tr>
<tr id="Hao1999" class="entry">
	<td>Hao, W., Luo, Z., Zheng, L., Prasad, K. and Lafer, E.</td>
	<td>AP180 and AP-2 interact directly in a complex that cooperatively assembles clathrin <p class="infolinks">[<a href="javascript:toggleInfo('Hao1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hao1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Journal of Biological &hellip;&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.274.32.22785">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hao1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Abstract Clathrin-coated vesicles are involved in protein and lipid trafficking between intracellular compartments in eukaryotic cells. AP-2 and AP180 are the resident coat proteins of clathrin-coated vesicles in nerve terminals, and interactions between these proteins could be important in vesicle dynamics. AP180 and AP-2 each assemble clathrin efficiently under acidic conditions, but neither protein will assemble clathrin efficiently at  ...</td>
</tr>
<tr id="bib_Hao1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hao1999,
  author = {Hao, W and Luo, Z and Zheng, L and Prasad, K and Lafer, EM},
  title = {AP180 and AP-2 interact directly in a complex that cooperatively assembles clathrin},
  journal = {Journal of Biological &hellip;},
  year = {1999},
  note = {domain information. AP2A2, AP2B1 - AP180. phosphorylation},
  doi = {https://doi.org/10.1074/jbc.274.32.22785}
}
</pre></td>
</tr>
<tr id="Harlan1994" class="entry">
	<td>Harlan, J.E., Hajduk, P.J., Yoon, H.S. and Fesik, S.W.</td>
	<td>Pleckstrin homology domains bind to phosphatidylinositol-4,5-bisphosphate. <p class="infolinks">[<a href="javascript:toggleInfo('Harlan1994','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Harlan1994','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td>Nature<br/>Vol. 371, pp. 168-170&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/371168a0">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Harlan1994" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The pleckstrin homology (PH) domain is a new protein module of around 100 amino acids found in several proteins involved in signal transduction. Although its specific function has yet to be elucidated, the carboxy-terminal regions of many PH domains bind to the beta gamma subunits of G proteins. On the basis of structural similarities between PH domains and lipid-binding proteins, we have proposed that PH domains may be binding to lipophilic molecules. Indeed, many of the proteins that contain this domain associate with phospholipid membranes, and disruption of this domain can interfere with membrane association. Here we report that PH domains bind to phosphatidylinositol-4,5-bisphosphate and show that the lipid-binding site is located at the lip of the beta-barrel. This suggests that PH domains may be important for membrane localization of proteins through interactions with phosphatidylinositol-4,5-bisphosphate.</td>
</tr>
<tr id="bib_Harlan1994" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Harlan1994,
  author = {Harlan, J E and Hajduk, P J and Yoon, H S and Fesik, S W},
  title = {Pleckstrin homology domains bind to phosphatidylinositol-4,5-bisphosphate.},
  journal = {Nature},
  year = {1994},
  volume = {371},
  pages = {168--170},
  doi = {https://doi.org/10.1038/371168a0}
}
</pre></td>
</tr>
<tr id="Hashizume1976" class="entry">
	<td>Hashizume, S., Sekiguchi, T. and Nosoh, Y.</td>
	<td>Effect of temperature on the viability of Bacillus stearothermophilus <p class="infolinks">[<a href="javascript:toggleInfo('Hashizume1976','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hashizume1976','bibtex')">BibTeX</a>]</p></td>
	<td>1976</td>
	<td>Archives of Microbiology<br/>Vol. 107(1), pp. 75 - 80&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1007/BF00427870">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hashizume1976" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: One of the obligate thermophilic bacteria, Bacillus stearothermophilus, was unable to grow at temperatures below 35textdegree C. About 80&#37; of the population in the bacterial culture died at the temperatures, and the same extent of loss in either of the activities of oxygen consumption or synthesis of protein or nucleic acid of the organisms was observed. With the progress of death of the organisms, reduced nicotinamide-adenine dinucleotide came to be oxidized by the organisms, enzymes such as fructose-1,6-diphosphate aldolase, when the organisms were washed with phosphate buffer, were leaked out of the organisms, and an increasing amount of ribonucleoprotein was released into the culture medium. The change of the membrane state was then suggested to be one of the possible causes for the death of the organisms at the temperatures.</td>
</tr>
<tr id="bib_Hashizume1976" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hashizume1976,
  author = {Hashizume, Shuichi and Sekiguchi, Takeshi and Nosoh, Yoshiaki},
  title = {Effect of temperature on the viability of Bacillus stearothermophilus},
  journal = {Archives of Microbiology},
  year = {1976},
  volume = {107},
  number = {1},
  pages = {75 - 80},
  note = {AlaR cannot grow below 35 C.},
  doi = {https://doi.org/10.1007/BF00427870}
}
</pre></td>
</tr>
<tr id="Havugimana2012" class="entry">
	<td>Havugimana, P.C., Hart, T.G., Nepusz, T., Yang, H., Turinsky, A.L., Li, Z., Wang, P.I., Boutz, D.R., Fong, V., Phanse, S., Babu, M., Craig, S.A., Hu, P., Wan, C., Vlasblom, J., Dar, V., Bezginov, A., Clark, G.W., Wu, G.C., Wodak, S.J., Tillier, E., Paccanaro, A., Marcotte, E.M. and Emili, A.</td>
	<td>A census of human soluble protein complexes <p class="infolinks">[<a href="javascript:toggleInfo('Havugimana2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Havugimana2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Cell<br/>Vol. 150(5), pp. 1068 - 1081&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cell.2012.08.011">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Havugimana2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Cellular processes often depend on stable physical associations between proteins. Despite recent progress, knowledge of the composition of human protein complexes remains limited. To close this gap, we applied an integrative global proteomic profiling approach, based on chromatographic separation of cultured human cell extracts into more than one thousand biochemical fractions that were subsequently analyzed by quantitative tandem mass spectrometry, to systematically identify a network of 13,993 high-confidence physical interactions among 3,006 stably associated soluble human proteins. Most of the 622 putative protein complexes we report are linked to core biological processes and encompass both candidate disease genes and unannotated proteins to inform on mechanism. Strikingly, whereas larger multiprotein assemblies tend to be more extensively annotated and evolutionarily conserved, human protein complexes with five or fewer subunits are far more likely to be functionally unannotated or restricted to vertebrates, suggesting more recent functional innovations. textcopyright 2012 Elsevier Inc.</td>
</tr>
<tr id="bib_Havugimana2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Havugimana2012,
  author = {Havugimana, Pierre C and Hart, Traver G and Nepusz, Tam&aacute;s and Yang, Haixuan and Turinsky, Andrei L and Li, Zhihua and Wang, Peggy I and Boutz, Daniel R and Fong, Vincent and Phanse, Sadhna and Babu, Mohan and Craig, Stephanie A and Hu, Pingzhao and Wan, Cuihong and Vlasblom, James and Dar, Vaqaar and Bezginov, Alexandr and Clark, Gregory W and Wu, Gabriel C and Wodak, Shoshana J and Tillier, Elisabeth and Paccanaro, Alberto and Marcotte, Edward M and Emili, Andrew},
  title = {A census of human soluble protein complexes},
  journal = {Cell},
  year = {2012},
  volume = {150},
  number = {5},
  pages = {1068 - 1081},
  doi = {https://doi.org/10.1016/j.cell.2012.08.011}
}
</pre></td>
</tr>
<tr id="Hein2015" class="entry">
	<td>Hein, M.Y., Hubner, N.C., Poser, I., Cox, J., Nagaraj, N., Toyoda, Y., Gak, I.A., Weisswange, I., Mansfeld, J., Buchholz, F., Hyman, A.A. and Mann, M.</td>
	<td>A Human Interactome in Three Quantitative Dimensions Organized by Stoichiometries and Abundances <p class="infolinks">[<a href="javascript:toggleInfo('Hein2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hein2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Cell<br/>Vol. 163(3), pp. 712 - 723&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cell.2015.09.053">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hein2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Summary The organization of a cell emerges from the interactions in protein networks. The interactome is critically dependent on the strengths of interactions and the cellular abundances of the connected proteins, both of which span orders of magnitude. However, these aspects have not yet been analyzed globally. Here, we have generated a library of HeLa cell lines expressing 1,125 GFP-tagged proteins under near-endogenous control, which we used as input for a next-generation interaction survey. Using quantitative proteomics, we detect specific interactions, estimate interaction stoichiometries, and measure cellular abundances of interacting proteins. These three quantitative dimensions reveal that the protein network is dominated by weak, substoichiometric interactions that play a pivotal role in defining network topology. The minority of stable complexes can be identified by their unique stoichiometry signature. This study provides a rich interaction dataset connecting thousands of proteins and introduces a framework for quantitative network analysis.</td>
</tr>
<tr id="bib_Hein2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hein2015,
  author = {Hein, Marco Y and Hubner, Nina C and Poser, Ina and Cox, J&uuml;rgen and Nagaraj, Nagarjuna and Toyoda, Yusuke and Gak, Igor A and Weisswange, Ina and Mansfeld, Jorg and Buchholz, Frank and Hyman, Anthony A and Mann, Matthias},
  title = {A Human Interactome in Three Quantitative Dimensions Organized by Stoichiometries and Abundances},
  journal = {Cell},
  year = {2015},
  volume = {163},
  number = {3},
  pages = {712 - 723},
  doi = {https://doi.org/10.1016/j.cell.2015.09.053}
}
</pre></td>
</tr>
<tr id="Henne2010" class="entry">
	<td>Henne, W., Boucrot, E., Meinecke, M., Evergren, E., Vallis, Y., Mittal, R. and McMahon, H.T.</td>
	<td>FCHo proteins are nucleators of clathrin-mediated endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Henne2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Henne2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Science<br/>Vol. 328(5983), pp. 1281 - 4&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1126/science.1188462">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Henne2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis, the major pathway for ligand internalization into eukaryotic cells, is thought to be initiated by the clustering of clathrin and adaptors around receptors destined for internalization. However, here we report that the membrane-sculpting F-BAR domain-containing Fer/Cip4 homology domain-only proteins 1 and 2 (FCHo1/2) were required for plasma membrane clathrin-coated vesicle (CCV) budding and marked sites of CCV formation. Changes in FCHo1/2 expression levels correlated directly with numbers of CCV budding events, ligand endocytosis, and synaptic vesicle marker recycling. FCHo1/2 proteins bound specifically to the plasma membrane and recruited the scaffold proteins eps15 and intersectin, which in turn engaged the adaptor complex AP2. The FCHo F-BAR membrane-bending activity was required, leading to the proposal that FCHo1/2 sculpt the initial bud site and recruit the clathrin machinery for CCV formation.</td>
</tr>
<tr id="bib_Henne2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Henne2010,
  author = {Henne, William and Boucrot, Emmanuel and Meinecke, Michael and Evergren, Emma and Vallis, Yvonne and Mittal, Rohit and McMahon, Harvey T.},
  title = {FCHo proteins are nucleators of clathrin-mediated endocytosis.},
  journal = {Science},
  year = {2010},
  volume = {328},
  number = {5983},
  pages = {1281 - 4},
  note = {FCHO2.MUH interactions with EPS15 and ITSN1-L domain information. expression information},
  doi = {https://doi.org/10.1126/science.1188462}
}
</pre></td>
</tr>
<tr id="Henne2007" class="entry">
	<td>Henne, W.M., Kent, H.M., Ford, M.G.J., Hegde, B.G., Daumke, O., Butler, P.J.G., Mittal, R., Langen, R., Evans, P.R. and McMahon, H.T.</td>
	<td>Structure and analysis of FCHo2 F-BAR domain: a dimerizing and membrane recruitment module that effects membrane curvature. <p class="infolinks">[<a href="javascript:toggleInfo('Henne2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Henne2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Structure (London, England : 1993)<br/>Vol. 15, pp. 839-852&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.str.2007.05.002">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Henne2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A spectrum of membrane curvatures exists within cells, and proteins have evolved different modules to detect, create, and maintain these curvatures. Here we present the crystal structure of one such module found within human FCHo2. This F-BAR (extended FCH) module consists of two F-BAR domains, forming an intrinsically curved all-helical antiparallel dimer with a Kd of 2.5 microM. The module binds liposomes via a concave face, deforming them into tubules with variable diameters of up to 130 nm. Pulse EPR studies showed the membrane-bound dimer is the same as the crystal dimer, although the N-terminal helix changed conformation on membrane binding. Mutation of a phenylalanine on this helix partially attenuated narrow tubule formation, and resulted in a gain of curvature sensitivity. This structure shows a distant relationship to curvature-sensing BAR modules, and suggests how similar coiled-coil architectures in the BAR superfamily have evolved to expand the repertoire of membrane-sculpting possibilities.</td>
</tr>
<tr id="bib_Henne2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Henne2007,
  author = {Henne, William Mike and Kent, Helen M and Ford, Marijn G J and Hegde, Balachandra G and Daumke, Oliver and Butler, P Jonathan G and Mittal, Rohit and Langen, Ralf and Evans, Philip R and McMahon, Harvey T},
  title = {Structure and analysis of FCHo2 F-BAR domain: a dimerizing and membrane recruitment module that effects membrane curvature.},
  journal = {Structure (London, England : 1993)},
  year = {2007},
  volume = {15},
  pages = {839--852},
  note = {dissociation constant. domain information. citations for amph, epn1 interactions with pip2. FCHO2 can interact with the membrane as a dimer.},
  doi = {https://doi.org/10.1016/j.str.2007.05.002}
}
</pre></td>
</tr>
<tr id="Henry2010" class="entry">
	<td>Henry, G.D., Corrigan, D.J., Dineen, J.V. and Baleja, J.D.</td>
	<td>Charge effects in the selection of NPF motifs by the EH domain of EHD1. <p class="infolinks">[<a href="javascript:toggleInfo('Henry2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Henry2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Biochemistry<br/>Vol. 49, pp. 3381-3392&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi100065r">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Henry2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The Eps15 homology (EH) domain is found in proteins associated with endocytosis and vesicle trafficking. EH domains bind to their target proteins through an asparagine-proline-phenylalanine (NPF) motif. We have measured the interaction energetics of the EH domain from EHD1 with peptides derived from two of its binding partners: Rabenosyn-5 (Ac-GPSLNPFDEED-NH(2)) and Rab11-Fip2 (Ac-YESTNPFTAK-NH(2)). Heteronuclear single quantum coherence (HSQC) spectroscopy shows that both peptides bind in the canonical binding pocket of EHD1 EH and induce identical structural changes, yet the affinity of the negatively charged Ac-GPSLNPFDEED-NH(2) (K(a) = 8 x 10(5) M(-1)) is tighter by 2 orders of magnitude. The thermodynamic profiles (DeltaG, DeltaH, DeltaS) were measured for both peptides as a function of temperature. The enthalpies of binding are essentially identical, and the difference in affinity is a consequence of the difference in entropic cost. Ac-GPSLNPFDEED-NH(2) binding is salt-dependent, demonstrating an electrostatic component to the interaction, whereas Ac-YESTNPFTAK-NH(2) binding is independent of salt. Successive replacement of acidic residues in Ac-GPSLNPFDEED-NH(2) with neutral residues showed that all are important. Lysine side chains in EHD1 EH create a region of strong positive surface potential near the NPF binding pocket. Contributions by lysine epsilon-amino groups to complex formation with Ac-GPSLNPFDEED-NH(2) was shown using direct-observe (15)N NMR spectroscopy. These experiments have enabled us to define a new extended interaction motif for EHD proteins, N-P-F-[DE]-[DE]-[DE], which we have used to predict new interaction partners and hence broaden the range of cellular activities involving the EHD proteins.</td>
</tr>
<tr id="bib_Henry2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Henry2010,
  author = {Henry, Gillian D and Corrigan, Daniel J and Dineen, Joseph V and Baleja, James D},
  title = {Charge effects in the selection of NPF motifs by the EH domain of EHD1.},
  journal = {Biochemistry},
  year = {2010},
  volume = {49},
  pages = {3381--3392},
  doi = {https://doi.org/10.1021/bi100065r}
}
</pre></td>
</tr>
<tr id="Heo2011" class="entry">
	<td>Heo, M., Maslov, S. and Shakhnovich, E.</td>
	<td>Topology of protein interaction network shapes protein abundances and strengths of their functional and nonspecific interactions. <p class="infolinks">[<a href="javascript:toggleInfo('Heo2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Heo2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Proceedings of the National Academy of Sciences<br/>Vol. 108(10), pp. 4258 - 63&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.1009392108">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Heo2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: How do living cells achieve sufficient abundances of functional protein complexes while minimizing promiscuous nonfunctional interactions? Here we study this problem using a first-principle model of the cell whose phenotypic traits are directly determined from its genome through biophysical properties of protein structures and binding interactions in a crowded cellular environment. The model cell includes three independent prototypical pathways, whose topologies of protein-protein interaction (PPI) subnetworks are different, but whose contributions to the cell fitness are equal. Model cells evolve through genotypic mutations and phenotypic protein copy number variations. We found a strong relationship between evolved physical-chemical properties of protein interactions and their abundances due to a "frustration" effect: Strengthening of functional interactions brings about hydrophobic interfaces, which make proteins prone to promiscuous binding. The balancing act is achieved by lowering concentrations of hub proteins while raising solubilities and abundances of functional monomers. On the basis of these principles we generated and analyzed a possible realization of the proteome-wide PPI network in yeast. In this simulation we found that high-throughput affinity capture-mass spectroscopy experiments can detect functional interactions with high fidelity only for high-abundance proteins while missing most interactions for low-abundance proteins.</td>
</tr>
<tr id="bib_Heo2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Heo2011,
  author = {Heo, Muyoung and Maslov, Sergei and Shakhnovich, Eugene},
  title = {Topology of protein interaction network shapes protein abundances and strengths of their functional and nonspecific interactions.},
  journal = {Proceedings of the National Academy of Sciences},
  year = {2011},
  volume = {108},
  number = {10},
  pages = {4258 - 63},
  doi = {https://doi.org/10.1073/pnas.1009392108}
}
</pre></td>
</tr>
<tr id="Herda2018" class="entry">
	<td>Herda, M. and Rodrigues, L.M.</td>
	<td>Large-time behavior of solutions to Vlasov-Poisson-Fokker-Planck equations: from evanescent collisions to diffusive limit <p class="infolinks">[<a href="javascript:toggleInfo('Herda2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Journal of Statistical Physics&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Herda2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Herda2018,
  author = {Herda, Maxime and Rodrigues, Luis Miguel},
  title = {Large-time behavior of solutions to Vlasov-Poisson-Fokker-Planck equations: from evanescent collisions to diffusive limit},
  journal = {Journal of Statistical Physics},
  year = {2018}
}
</pre></td>
</tr>
<tr id="Hewitt1999" class="entry">
	<td>Hewitt, C., Eszes, C., Sessions, R., Moreton, K., Dafforn, T., Takei, J., Dempsey, C., Clarke, A. and Holbrook, J.</td>
	<td>A general method for relieving substrate inhibition in lactate dehydrogenases <p class="infolinks">[<a href="javascript:toggleInfo('Hewitt1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hewitt1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Protein Engineering<br/>Vol. 12(6), pp. 491 - 496&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/protein/12.6.491">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hewitt1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The mutation S163L in human heart lactate dehydrogenase removes substrate inhibition while only modestly reducing the turnover rate for pyruvate. Since this is the third enzyme to show this behaviour, we suggest that the S163L mutation is a general method for the removal of substrate inhibition in L-LDH enzymes. Engineering such enzymatic properties has clear industrial applications in the use of these enzymes to produce enantiomerically pure α-hydroxy acids. The mutation leads to two principal effects. (1) Sub-strate inhibition is caused by the formation of a covalent adduct between pyruvate and the oxidized form of the cofactor. The inability of S163L mutants to catalyse the formation of this inhibitory adduct is demonstrated. How-ever, NMR experiments show that the orientation of the nicotinamide ring in the mutant NAD ⍣ binary complex is not perturbed. (2) The mutation also leads to a large increase in the K M for pyruvate. The kinetic and binding properties of S163L LDH mutants are accounted for by a mechanism which invokes a non-productive, bound form of the cofactor. Molecular modelling suggests a structure for this non-productive enzyme--NADH complex. Keywords: lactate dehydrogenase/mutagenesis/protein engineer-ing/substrate inhibition Introduction Mammalian and bacterial lactate dehydrogenases show inhibi-tion by high concentrations of substrate during the reduction of pyruvate to lactate. This inhibition is a consequence of the formation of a covalent adduct between pyruvate and the oxidized cofactor, NAD ϩ , before it is released from the enzyme (Fromm, 1961; Gutfreund et al., 1968; Coulson and Rabin, 1969). The enzyme catalyses the formation of this adduct which then binds tightly to the protein, inhibiting further redox catalysis (Everse et al., 1971) (Figure 1). Apart from purely scientific interest, the ability to remove substrate inhibition in enzymes by appropriate mutagenesis has applications in the industrial context, allowing the production of large quantities of enantiomerically pure pharmaceutical intermediates. We have shown previously that the mutation S163L in the lactate dehydrogenases from Bacillus stearothermophilus (bsLDH) and human muscle (hM 4 LDH) removes substrate inhibition (Eszes et al., 1996; Hewitt et al., 1997). In this paper, we show that this mutation is also effective in the human heart isoform (hH 4 LDH), the most strongly</td>
</tr>
<tr id="bib_Hewitt1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hewitt1999,
  author = {Hewitt, CO and Eszes, CM and Sessions, RB and Moreton, KM and Dafforn, TR and Takei, J and Dempsey, CE and Clarke, AR and Holbrook, JJ},
  title = {A general method for relieving substrate inhibition in lactate dehydrogenases},
  journal = {Protein Engineering},
  year = {1999},
  volume = {12},
  number = {6},
  pages = {491 - 496},
  note = {NULL},
  doi = {https://doi.org/10.1093/protein/12.6.491}
}
</pre></td>
</tr>
<tr id="Hewitt1997" class="entry">
	<td>Hewitt, C., Sessions, R., Dafforn, T. and Holbrook, J.</td>
	<td>Protein engineering tests of a homology model of Plasmodium falciparum lactate dehydrogenase <p class="infolinks">[<a href="javascript:toggleInfo('Hewitt1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hewitt1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Protein Engineering Design and Selection<br/>Vol. 10(1), pp. 39 - 44&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/protein/10.1.39">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hewitt1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: This paper describes the testing of a homology model of Plasmodium falciparum lactate dehydrogenase (pfLDH) by protein engineering. The model had been validated in structural terms. It suggests explanations of the unusual properties of pfLDH (compared with all other LDHs). These unusual features are a lack of substrate inhibition, high activity with the synthetic coenzyme 3-acetylpyridine adenine dinucleotide (APAD+) and changes in residues at previously conserved positions. pfLDH shows several amino acid insertions and deletions in an alignment with protein sequences from all other known LDHs. The most notable is a five amino acid insertion into the active-site loop. In addition, a conserved serine at position 163 is replaced by leucine. The results showed that when the unique pfLDH structural features were engineered into Bacillus stearothermophilus lactate dehydrogenase, the thermophilic enzyme acquired the properties previously uniquely associated with the malarial enzyme. We conclude that the homology model of the malarial enzyme is adequate for the prediction of successful redesigns and, in the regions tested, is accurate.</td>
</tr>
<tr id="bib_Hewitt1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hewitt1997,
  author = {Hewitt, CO and Sessions, RB and Dafforn, TR and Holbrook, JJ},
  title = {Protein engineering tests of a homology model of Plasmodium falciparum lactate dehydrogenase},
  journal = {Protein Engineering Design and Selection},
  year = {1997},
  volume = {10},
  number = {1},
  pages = {39 - 44},
  note = {NULL},
  doi = {https://doi.org/10.1093/protein/10.1.39}
}
</pre></td>
</tr>
<tr id="Hilvert2000" class="entry">
	<td>Hilvert, D.</td>
	<td>Critical analysis of antibody catalysis <p class="infolinks">[<a href="javascript:toggleInfo('Hilvert2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hilvert2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Annual review of biochemistry<br/>Vol. 69(1), pp. 751 - 793&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1146/annurev.biochem.69.1.751">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hilvert2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Antibody molecules elicited with rationally designed transition-state analogs catalyze numerous reactions, including many that cannot be achieved by standard chemical methods. Although relatively primitive when compared with natural enzymes, these catalysts are valuable tools for probing the origins and evolution of biological catalysis. Mechanistic and structural analyses of representative antibody catalysts, generated with a variety of strategies for several different reaction types, suggest that their modest efficiency is a consequence of imperfect hapten design and indirect selection. Development of improved transition-state analogs, refinements in immunization and screening protocols, and elaboration of general strategies for augmenting the efficiency of first-generation catalytic antibodies are identified as evident, but difficult, challenges for this field. Rising to these challenges and more successfully integrating programmable design with the selective forces of biology will enhance our understanding of enzymatic catalysis. Further, it should yield useful prot</td>
</tr>
<tr id="bib_Hilvert2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hilvert2000,
  author = {Hilvert, Donald},
  title = {Critical analysis of antibody catalysis},
  journal = {Annual review of biochemistry},
  year = {2000},
  volume = {69},
  number = {1},
  pages = {751 - 793},
  doi = {https://doi.org/10.1146/annurev.biochem.69.1.751}
}
</pre></td>
</tr>
<tr id="Hinrichsen2006" class="entry">
	<td>Hinrichsen, L., Meyerholz, A., Groos, S. and Ungewickell, E.J.</td>
	<td>Bending a membrane: how clathrin affects budding. <p class="infolinks">[<a href="javascript:toggleInfo('Hinrichsen2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hinrichsen2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Proceedings of the National Academy of Sciences of the United States of America<br/>Vol. 103, pp. 8715-8720&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.0600312103">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hinrichsen2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Receptor-mediated endocytosis of ligands, such as transferrin and LDL, is suppressed when clathrin synthesis is blocked by RNA interference in HeLa cells. We have found that domains containing the adapter complex 2 (AP2)-coated vesicle adapter and the endocytic accessory proteins CALM (clathrin assembly lymphoid myeloid leukemia protein), epsin, and eps15/eps15R (EGF receptor pathway substrate 15-related) nevertheless persist at the plasma membrane. They are similar in size and number to those seen in clathrin-expressing cells. Here we characterize these membrane domains by fluorescence and electron microscopy in detail. Fluorescence recovery after photobleaching measurements suggest that the exchange between membrane-bound and free cytosolic AP2 molecules is not significantly influenced by the depletion of clathrin. The AP2 membrane domains are dispersed upon interfering with protein-protein interactions that involve the alpha appendage domain of AP2. Electron microscopy of cellular cortices revealed that the AP2 membrane domains lack any curvature, suggesting that clathrin is essential for driving coated pit invagination. A model for coated vesicle formation, incorporating a mechanism commonly referred to as a "Brownian ratchet," is consistent with our observations.</td>
</tr>
<tr id="bib_Hinrichsen2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hinrichsen2006,
  author = {Hinrichsen, Lars and Meyerholz, Anika and Groos, Stephanie and Ungewickell, Ernst J},
  title = {Bending a membrane: how clathrin affects budding.},
  journal = {Proceedings of the National Academy of Sciences of the United States of America},
  year = {2006},
  volume = {103},
  pages = {8715--8720},
  doi = {https://doi.org/10.1073/pnas.0600312103}
}
</pre></td>
</tr>
<tr id="Hlavacek2006" class="entry">
	<td>Hlavacek, W.S., Faeder, J.R., Blinov, M.L., Posner, R.G., Hucka, M. and Fontana, W.</td>
	<td>Rules for Modeling Signal-Transduction Systems <p class="infolinks">[<a href="javascript:toggleInfo('Hlavacek2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hlavacek2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Science STKE<br/>Vol. 7(344), pp. 1 - 18&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1126/stke.3442006re6">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hlavacek2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Formalized rules for protein-protein interactions have recently been introduced to represent the binding and enzymatic activities of proteins in cellular signaling. Rules encode an understanding of how a system works in terms of the biomolecules in the system and their possible states and interactions. A set of rules can be as easy to read as a diagrammatic interaction map, but unlike most such maps, rules have precise interpretations. Rules can be processed to automatically generate a mathematical or computational model for a system, which enables explanatory and predictive insights into the system's behavior. Rules are independent units of a model specification that facilitate model revision. Instead of changing a large number of equations or lines of code, as may be required in the case of a conventional mathematical model, a protein interaction can be introduced or modified simply by adding or changing a single rule that represents the interaction of interest. Rules can be defined and visualized by using graphs, so no specialized training in mathematics or computer science is necessary to create models or to take advantage of the representational precision of rules. Rules can be encoded in a machine-readable format to enable electronic storage and exchange of models, as well as basic knowledge about protein-protein interactions. Here, we review the motivation for rule-based modeling; applications of the approach; and issues that arise in model specification, simulation, and testing. We also discuss rule visualization and exchange and the software available for rule-based modeling.</td>
</tr>
<tr id="bib_Hlavacek2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hlavacek2006,
  author = {Hlavacek, William S and Faeder, James R and Blinov, Michael L and Posner, Richard G and Hucka, Michael and Fontana, Walter},
  title = {Rules for Modeling Signal-Transduction Systems},
  journal = {Science STKE},
  year = {2006},
  volume = {7},
  number = {344},
  pages = {1 - 18},
  doi = {https://doi.org/10.1126/stke.3442006re6}
}
</pre></td>
</tr>
<tr id="Hocky2017" class="entry">
	<td>Hocky, G.M., Dannenhoffer-Lafage, T. and Voth, G.A.</td>
	<td>Coarse-Grained Directed Simulation <p class="infolinks">[<a href="javascript:toggleInfo('Hocky2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of Chemical Theory and Computation<br/>Vol. 0(0), pp. null&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jctc.7b00690">DOI</a> <a href=" 
        http://dx.doi.org/10.1021/acs.jctc.7b00690
    
">URL</a>&nbsp;</td>
</tr>
<tr id="bib_Hocky2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hocky2017,
  author = {Hocky, Glen M. and Dannenhoffer-Lafage, Thomas and Voth, Gregory A.},
  title = {Coarse-Grained Directed Simulation},
  journal = {Journal of Chemical Theory and Computation},
  year = {2017},
  volume = {0},
  number = {0},
  pages = {null},
  url = { <br>        http://dx.doi.org/10.1021/acs.jctc.7b00690<br>    <br>},
  doi = {https://doi.org/10.1021/acs.jctc.7b00690}
}
</pre></td>
</tr>
<tr id="Hoeller2006" class="entry">
	<td>Hoeller, D., Crosetto, N., Blagoev, B., Raiborg, C., Tikkanen, R., Wagner, S., Kowanetz, K., Breitling, R., Mann, M., Stenmark, H. and Dikic, I.</td>
	<td>Regulation of ubiquitin-binding proteins by monoubiquitination. <p class="infolinks">[<a href="javascript:toggleInfo('Hoeller2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hoeller2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Nature cell biology<br/>Vol. 8(2), pp. 163 - 9&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/ncb1354">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hoeller2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Proteins containing ubiquitin-binding domains (UBDs) interact with ubiquitinated targets and regulate diverse biological processes, including endocytosis, signal transduction, transcription and DNA repair. Many of the UBD-containing proteins are also themselves monoubiquitinated, but the functional role and the mechanisms that underlie this modification are less well understood. Here, we demonstrate that monoubiquitination of the endocytic proteins Sts1, Sts2, Eps15 and Hrs results in intramolecular interactions between ubiquitin and their UBDs, thereby preventing them from binding in trans to ubiquitinated targets. Permanent monoubiquitination of these proteins, mimicked by the fusion of ubiquitin to their carboxyl termini, impairs their ability to regulate trafficking of ubiquitinated receptors. Moreover, we mapped the in vivo monoubiquitination site in Sts2 and demonstrated that its mutation enhances the Sts2-mediated effects of epidermal-growth-factor-receptor downregulation. We propose that monoubiquitination of ubiquitin-binding proteins inhibits their capacity to bind to and control the functions of ubiquitinated targets in vivo.</td>
</tr>
<tr id="bib_Hoeller2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hoeller2006,
  author = {Hoeller, Daniela and Crosetto, Nicola and Blagoev, Blagoy and Raiborg, Camilla and Tikkanen, Ritva and Wagner, Sebastian and Kowanetz, Katarzyna and Breitling, Rainer and Mann, Matthias and Stenmark, Harald and Dikic, Ivan},
  title = {Regulation of ubiquitin-binding proteins by monoubiquitination.},
  journal = {Nature cell biology},
  year = {2006},
  volume = {8},
  number = {2},
  pages = {163 - 9},
  note = {HEK and HeLa cells. eps15-epn1.},
  doi = {https://doi.org/10.1038/ncb1354}
}
</pre></td>
</tr>
<tr id="Hofstad2017" class="entry">
	<td>van der Hofstad, R. and Komj&aacute;thy, J.</td>
	<td>When is a Scale-Free Graph Ultra-Small? <p class="infolinks">[<a href="javascript:toggleInfo('Hofstad2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hofstad2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of Statistical Physics&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1007/s10955-017-1864-1">DOI</a> <a href="https://doi.org/10.1007/s10955-017-1864-1">URL</a>&nbsp;</td>
</tr>
<tr id="abs_Hofstad2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In this paper we study typical distances in the configuration model, when the degrees have asymptotically infinite variance. We assume that the empirical degree distribution follows a power law with exponent                                                                   &dollar;&dollar;backslashtau backslashin (2,3)&dollar;&dollar;                                                                            &tau;                      ∈                      (                      2                      ,                      3                      )                                                                    , up to value                                                                   &dollar;&dollar;n^backslashbeta n&dollar;&dollar;                                                                            n                                              &beta;                        n                                                                                           for some                                                                   &dollar;&dollar;backslashbeta nbackslashgg (backslashlog n)^-backslashgamma &dollar;&dollar;                                                                                                    &beta;                        n                                            ≫                                                                        (                          log                          n                          )                                                                          -                          &gamma;                                                                                                                   and                                                                   &dollar;&dollar;backslashgamma backslashin (0,1)&dollar;&dollar;                                                                            &gamma;                      ∈                      (                      0                      ,                      1                      )                                                                    . This assumption is satisfied for power law i.i.d. degrees, and also includes truncated power-law empirical degree distributions where the (possibly exponential) truncation happens at                                                                   &dollar;&dollar;n^backslashbeta n&dollar;&dollar;                                                                            n                                              &beta;                        n                                                                                          . These examples are commonly observed in many real-life networks. We show that the graph distance between two uniformly chosen vertices centers around                                                                   &dollar;&dollar;2 backslashlog backslashlog (n^backslashbeta n) / |backslashlog (backslashtau -2)| + 1/(backslashbeta n(3-backslashtau ))&dollar;&dollar;                                                                            2                      log                      log                                              (                                                  n                                                      &beta;                            n                                                                          )                                            /                                              |                        log                                                  (                          &tau;                          -                          2                          )                                                |                                            +                      1                      /                                              (                                                  &beta;                          n                                                                          (                          3                          -                          &tau;                          )                                                )                                                                                          , with tight fluctuations. Thus, the graph is an ultrasmall world whenever                                                                   &dollar;&dollar;1/backslashbeta n=o(backslashlog backslashlog n)&dollar;&dollar;                                                                            1                      /                                              &beta;                        n                                            =                      o                                              (                        log                        log                        n                        )                                                                                          . We determine the distribution of the fluctuations around this value, in particular we prove these form a sequence of tight random variables with distributions that show                                                                   &dollar;&dollar;backslashlog backslashlog &dollar;&dollar;                                                                            log                      log                                                                    -periodicity, and as a result it is non-converging. We describe the topology and number of shortest paths: We show that the number of shortest paths is of order                                                                   &dollar;&dollar;n^fnbackslashbeta n&dollar;&dollar;                                                                            n                                                                        f                          n                                                                          &beta;                          n                                                                                                                  , where                                                                   &dollar;&dollar;fn backslashin (0,1)&dollar;&dollar;                                                                                                    f                        n                                            ∈                                              (                        0                        ,                        1                        )                                                                                           is a random variable that oscillates with n. We decompose shortest paths into three segments, two `end-segments' starting at each of the two uniformly chosen vertices, and a middle segment. The two end-segments of any shortest path have length                                                                   &dollar;&dollar;backslashlog backslashlog (n^backslashbeta n) / |backslashlog (backslashtau -2)|&dollar;&dollar;                                                                            log                      log                                              (                                                  n                                                      &beta;                            n                                                                          )                                            /                                              |                        log                                                  (                          &tau;                          -                          2                          )                                                |                                                                                          +tight, and the total degree is increasing towards the middle of the path on these segments. The connecting middle segment has length                                                                   &dollar;&dollar;1/(backslashbeta n(3-backslashtau ))&dollar;&dollar;                                                                            1                      /                      (                                              &beta;                        n                                                                    (                        3                        -                        &tau;                        )                                            )                                                                    +tight, and it contains only vertices with degree at least of order                                                                   &dollar;&dollar;n^(1-fn)backslashbeta n&dollar;&dollar;                                                                            n                                                                        (                          1                          -                                                      f                            n                                                    )                                                                          &beta;                          n                                                                                                                  , thus all the degrees on this segment are comparable to the maximal degree. Our theorems also apply when instead of truncating the degrees, we start with a configuration model and we remove every vertex with degree at least                                                                   &dollar;&dollar;n^backslashbeta n&dollar;&dollar;                                                                            n                                              &beta;                        n                                                                                          , and the edges attached to these vertices. This sheds light on the attack vulnerability of the configuration model with infinite variance degrees.</td>
</tr>
<tr id="bib_Hofstad2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hofstad2017,
  author = {van der Hofstad, Remco and Komj&aacute;thy, J&uacute;lia},
  title = {When is a Scale-Free Graph Ultra-Small?},
  journal = {Journal of Statistical Physics},
  year = {2017},
  url = {https://doi.org/10.1007/s10955-017-1864-1},
  doi = {https://doi.org/10.1007/s10955-017-1864-1}
}
</pre></td>
</tr>
<tr id="Holland2017" class="entry">
	<td>Holland, D.O., Shapiro, B.H., Xue, P. and Johnson, M.E.</td>
	<td>Protein-protein binding selectivity and network topology constrain global and local properties of interface binding networks <p class="infolinks">[<a href="javascript:toggleInfo('Holland2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Holland2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Scientific Reports<br/>Vol. 7(1), pp. 1 - 14&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/s41598-017-05686-2">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Holland2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Protein-protein interactions networks (PPINs) are known to share a highly conserved structure across all organisms. What is poorly understood, however, is the structure of the child interface interaction networks (IINs), which map the binding sites proteins use for each interaction. In this study we analyze four independently constructed IINs from yeast and humans and find a conserved structure of these networks with a unique topology distinct from the parent PPIN. Using an IIN sampling algorithm and a fitness function trained on the manually curated PPINs, we show that IIN topology can be mostly explained as a balance between limits on interface diversity and a need for physico-chemical binding complementarity. This complementarity must be optimized both for functional interactions and against mis-interactions, and this selectivity is encoded in the IIN motifs. To test whether the parent PPIN shapes IINs, we compared optimal IINs in biological PPINs versus random PPINs. We found that the hubs in biological networks allow for selective binding with minimal interfaces, suggesting that binding specificity is an additional pressure for a scale-free-like PPIN. We confirm through phylogenetic analysis that hub interfaces are strongly conserved and rewiring of interactions between proteins involved in endocytosis preserves interface binding selectivity.</td>
</tr>
<tr id="bib_Holland2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Holland2017,
  author = {Holland, David O and Shapiro, Benjamin H and Xue, Pei and Johnson, Margaret E},
  title = {Protein-protein binding selectivity and network topology constrain global and local properties of interface binding networks},
  journal = {Scientific Reports},
  year = {2017},
  volume = {7},
  number = {1},
  pages = {1 - 14},
  doi = {https://doi.org/10.1038/s41598-017-05686-2}
}
</pre></td>
</tr>
<tr id="Hollopeter2014" class="entry">
	<td>Hollopeter, G., Lange, J.J., Zhang, Y., Vu, T.N., Gu, M., Ailion, M., Lambie, E.J., Slaughter, B.D., Unruh, J.R., Florens, L. and Jorgensen, E.M.</td>
	<td>The membrane-associated proteins FCHo and SGIP are allosteric activators of the AP2 clathrin adaptor complex. <p class="infolinks">[<a href="javascript:toggleInfo('Hollopeter2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hollopeter2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>eLife<br/>Vol. 3&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.7554/eLife.03648">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hollopeter2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The AP2 clathrin adaptor complex links protein cargo to the endocytic machinery but it is unclear how AP2 is activated on the plasma membrane. Here we demonstrate that the membrane-associated proteins FCHo and SGIP1 convert AP2 into an open, active conformation. We screened for Caenorhabditis elegans mutants that phenocopy the loss of AP2 subunits and found that AP2 remains inactive in fcho-1 mutants. A subsequent screen for bypass suppressors of fcho-1 nulls identified 71 compensatory mutations in all four AP2 subunits. Using a protease-sensitivity assay we show that these mutations restore the open conformation in vivo. The domain of FCHo that induces this rearrangement is not the F-BAR domain or the µ-homology domain, but rather is an uncharacterized 90 amino acid motif, found in both FCHo and SGIP proteins, that directly binds AP2. Thus, these proteins stabilize nascent endocytic pits by exposing membrane and cargo binding sites on AP2.</td>
</tr>
<tr id="bib_Hollopeter2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hollopeter2014,
  author = {Hollopeter, Gunther and Lange, Jeffrey J and Zhang, Ying and Vu, Thien N and Gu, Mingyu and Ailion, Michael and Lambie, Eric J and Slaughter, Brian D and Unruh, Jay R and Florens, Laurence and Jorgensen, Erik M},
  title = {The membrane-associated proteins FCHo and SGIP are allosteric activators of the AP2 clathrin adaptor complex.},
  journal = {eLife},
  year = {2014},
  volume = {3},
  note = {C. elegans},
  doi = {https://doi.org/10.7554/eLife.03648}
}
</pre></td>
</tr>
<tr id="Holmberg1999" class="entry">
	<td>Holmberg, N., Ryde, U. and Bulow, L.</td>
	<td>Redesign of the coenzyme specificity in L-lactate dehydrogenase from bacillus stearothermophilus using site-directed mutagenesis and media engineering. <p class="infolinks">[<a href="javascript:toggleInfo('Holmberg1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Holmberg1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Protein engineering<br/>Vol. 12(10), pp. 851 - 856&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/protein/12.10.851">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Holmberg1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: L-lactate dehydrogenase (LDH) from Bacillus stearothermophilus is a redox enzyme which has a strong preference for NADH over NADPH as coenzyme. To exclude NADPH from the coenzyme-binding pocket, LDH contains a conserved aspartate residue at position 52. However, this residue is probably not solely responsible for the NADH specificity. In this report we examine the possibilities of altering the coenzyme specificity of LDH by introducing a range of different point mutations in the coenzyme-binding domain. Furthermore, after choosing the mutant with the highest selectivity for NADPH, we also investigated the possibility of further altering the coenzyme specificity by adding an organic solvent to the reaction mixture. The LDH mutant, I51K:D52S, exhibited a 56-fold increased specificity to NADPH over the wild-type LDH in a reaction mixture containing 15&#37; methanol. Furthermore, the NADPH turnover number of this mutant was increased almost fourfold as compared with wild-type LDH. To explain the altered coenzyme specificity exhibited by the D52SI51K double mutant, molecular dynamics simulations were performed.</td>
</tr>
<tr id="bib_Holmberg1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Holmberg1999,
  author = {Holmberg, N and Ryde, U and Bulow, L},
  title = {Redesign of the coenzyme specificity in L-lactate dehydrogenase from bacillus stearothermophilus using site-directed mutagenesis and media engineering.},
  journal = {Protein engineering},
  year = {1999},
  volume = {12},
  number = {10},
  pages = {851 - 856},
  doi = {https://doi.org/10.1093/protein/12.10.851}
}
</pre></td>
</tr>
<tr id="Holt2009" class="entry">
	<td>Holt, L.J., Tuch, B.B., Villen, J., Johnson, A.D., Gygi, S.P. and Morgan, D.O.</td>
	<td>Global analysis of Cdk1 substrate phosphorylation sites provides insights into evolution. <p class="infolinks">[<a href="javascript:toggleInfo('Holt2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Holt2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Science<br/>Vol. 325(5948), pp. 1682 - 6&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1126/science.1172867">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Holt2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: To explore the mechanisms and evolution of cell-cycle control, we analyzed the position and conservation of large numbers of phosphorylation sites for the cyclin-dependent kinase Cdk1 in the budding yeast Saccharomyces cerevisiae. We combined specific chemical inhibition of Cdk1 with quantitative mass spectrometry to identify the positions of 547 phosphorylation sites on 308 Cdk1 substrates in vivo. Comparisons of these substrates with orthologs throughout the ascomycete lineage revealed that the position of most phosphorylation sites is not conserved in evolution; instead, clusters of sites shift position in rapidly evolving disordered regions. We propose that the regulation of protein function by phosphorylation often depends on simple nonspecific mechanisms that disrupt or enhance protein-protein interactions. The gain or loss of phosphorylation sites in rapidly evolving regions could facilitate the evolution of kinase-signaling circuits.</td>
</tr>
<tr id="bib_Holt2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Holt2009,
  author = {Holt, Liam J and Tuch, Brian B and Villen, Judit and Johnson, Alexander D and Gygi, Steven P and Morgan, David O},
  title = {Global analysis of Cdk1 substrate phosphorylation sites provides insights into evolution.},
  journal = {Science},
  year = {2009},
  volume = {325},
  number = {5948},
  pages = {1682 - 6},
  doi = {https://doi.org/10.1126/science.1172867}
}
</pre></td>
</tr>
<tr id="Hom2007" class="entry">
	<td>Hom, R.A., Vora, M., Regner, M., Subach, O.M., Cho, W., Verkhusha, V.V., Stahelin, R.V. and Kutateladze, T.G.</td>
	<td>pH-dependent binding of the Epsin ENTH domain and the AP180 ANTH domain to PI(4,5)P2-containing bilayers. <p class="infolinks">[<a href="javascript:toggleInfo('Hom2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hom2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Journal of molecular biology<br/>Vol. 373, pp. 412-423&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.jmb.2007.08.016">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hom2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Epsin and AP180 are essential components of the endocytotic machinery, which controls internalization of protein receptors and other macromolecules at the cell surface. Epsin and AP180 are recruited to the plasma membrane by their structurally and functionally related N-terminal ENTH and ANTH domains that specifically recognize PtdIns(4,5)P2. Here, we show that membrane anchoring of the ENTH and ANTH domains is regulated by the acidic environment. Lowering the pH enhances PtdIns(4,5)P2 affinity of the ENTH and ANTH domains reinforcing their association with lipid vesicles and monolayers. The pH dependency is due to the conserved histidine residues of the ENTH and ANTH domains, protonation of which is necessary for the strong PtdIns(4,5)P2 recognition, as revealed by liposome binding, surface plasmon resonance, NMR, monolayer surface tension and mutagenesis experiments. The pH sensitivity of the ENTH and ANTH domains is reminiscent to the pH dependency of the FYVE domain suggesting a common regulatory mechanism of membrane anchoring by a subset of the PI-binding domains.</td>
</tr>
<tr id="bib_Hom2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hom2007,
  author = {Hom, Robert A and Vora, Mohsin and Regner, Maryann and Subach, Oksana M and Cho, Wonhwa and Verkhusha, Vladislav V and Stahelin, Robert V and Kutateladze, Tatiana G},
  title = {pH-dependent binding of the Epsin ENTH domain and the AP180 ANTH domain to PI(4,5)P2-containing bilayers.},
  journal = {Journal of molecular biology},
  year = {2007},
  volume = {373},
  pages = {412--423},
  note = {dissociation constant information. PIP2 (2%) - AP180 (440 uM), EPN1 (80 uM)},
  doi = {https://doi.org/10.1016/j.jmb.2007.08.016}
}
</pre></td>
</tr>
<tr id="Hoening2005" class="entry">
	<td>H&ouml;ning, S., Ricotta, D., Krauss, M., Sp&auml;te, K., Spolaore, B., Motley, A., Robinson, M., Robinson, C., Haucke, V. and Owen, D.J.</td>
	<td>Phosphatidylinositol-(4, 5)-bisphosphate regulates sorting signal recognition by the clathrin-associated adaptor complex AP2 <p class="infolinks">[<a href="javascript:toggleInfo('Hoening2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Molecular cell<br/>Vol. 18(5), pp. 519-531&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.molcel.2005.08.001">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Hoening2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hoening2005,
  author = {H&ouml;ning, Stefan and Ricotta, Doris and Krauss, Michael and Sp&auml;te, Kira and Spolaore, Barbara and Motley, Alison and Robinson, Margaret and Robinson, Carol and Haucke, Volker and Owen, David J},
  title = {Phosphatidylinositol-(4, 5)-bisphosphate regulates sorting signal recognition by the clathrin-associated adaptor complex AP2},
  journal = {Molecular cell},
  publisher = {Elsevier},
  year = {2005},
  volume = {18},
  number = {5},
  pages = {519--531},
  note = {"It has been variously suggested that AP2 binds D/ExxxLL acidic sorting motifs through its β2 subunit (Rapoport et al., 1998), its 2 subunit (Rodionov and Bakke, 1998), and also that AP2 does not bind to acidic dileucine motifs at all (Janvier et al., 2003)." one of the most frustrating sentences.},
  doi = {https://doi.org/10.1016/j.molcel.2005.08.001}
}
</pre></td>
</tr>
<tr id="Horiuti1938" class="entry">
	<td>Horiuti, J.</td>
	<td>On the statistical mechanical treatment of the absolute rate of chemical reaction <p class="infolinks">[<a href="javascript:toggleInfo('Horiuti1938','bibtex')">BibTeX</a>]</p></td>
	<td>1938</td>
	<td>Bulletin of the Chemical Society of Japan<br/>Vol. 13(1), pp. 210 - 216&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1246/bcsj.13.210">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Horiuti1938" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Horiuti1938,
  author = {Horiuti, Juro},
  title = {On the statistical mechanical treatment of the absolute rate of chemical reaction},
  journal = {Bulletin of the Chemical Society of Japan},
  year = {1938},
  volume = {13},
  number = {1},
  pages = {210 - 216},
  doi = {https://doi.org/10.1246/bcsj.13.210}
}
</pre></td>
</tr>
<tr id="Hove1954" class="entry">
	<td>Hove, L.</td>
	<td>Correlations in space and time and born approximation scattering in systems of interacting particles <p class="infolinks">[<a href="javascript:toggleInfo('Hove1954','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hove1954','bibtex')">BibTeX</a>]</p></td>
	<td>1954</td>
	<td>Physical Review<br/>Vol. 95(1), pp. 249 - 262&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1103/PhysRev.95.249">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hove1954" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A natural time-dependent generalization is given for the well-known pair distribution function g(r) of systems of interacting particles. The pair distribution in space and time thus defined, denoted by G(r, t), gives rise to a very simple and entirely general expression for the angular and energy distribution of Born approximation scattering by the system. This expression is the natural extension of the familiar Zernike-Prins formula to scattering in which the energy transfers are not negligible compared to the energy of the scattered particle. It is therefore of particular interest for scattering of slow neutrons by general systems of interacting particles: G is then the proper function in terms of which to analyze the scattering data.</td>
</tr>
<tr id="bib_Hove1954" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hove1954,
  author = {Hove, Leon},
  title = {Correlations in space and time and born approximation scattering in systems of interacting particles},
  journal = {Physical Review},
  year = {1954},
  volume = {95},
  number = {1},
  pages = {249 - 262},
  doi = {https://doi.org/10.1103/PhysRev.95.249}
}
</pre></td>
</tr>
<tr id="Hu2017" class="entry">
	<td>Hu, S., Soudackov, A.V., Hammes-Schiffer, S. and Klinman, J.P.</td>
	<td>Enhanced Rigidification within a Double Mutant of Soybean Lipoxygenase Provides Experimental Support for Vibronically Nonadiabatic Proton-Coupled Electron Transfer Models <p class="infolinks">[<a href="javascript:toggleInfo('Hu2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hu2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>ACS Catalysis, pp. 3569 - 3574&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acscatal.7b00688">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hu2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Soybean lipoxygenase (SLO) is a prototype for nonadiabatic hydrogen tunneling reactions and, as such, has served as the subject of numerous theoretical studies. In this work, we report a nearly temperature-independent kinetic isotope effect (KIE) with an average KIE value of 661 textpm 27 for a double mutant (DM) of SLO at six temperatures. The data are well-reproduced within a vibronically nonadiabatic proton-coupled electron transfer model in which the active site has become rigidified compared to wild-type enzyme and single-site mutants. A combined temperature--pressure perturbation further shows that temperature-dependent global motions within DM-SLO are more resistant to perturbation by elevated pressure. These findings provide strong experimental support for the model of hydrogen tunneling in SLO, where optimization of both local protein and ligand motions and distal conformational rearrangements is a prerequisite for effective proton vibrational wave function overlap between the substrate and the active-...</td>
</tr>
<tr id="bib_Hu2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hu2017,
  author = {Hu, Shenshen and Soudackov, Alexander V and Hammes-Schiffer, Sharon and Klinman, Judith P},
  title = {Enhanced Rigidification within a Double Mutant of Soybean Lipoxygenase Provides Experimental Support for Vibronically Nonadiabatic Proton-Coupled Electron Transfer Models},
  journal = {ACS Catalysis},
  year = {2017},
  pages = {3569 - 3574},
  doi = {https://doi.org/10.1021/acscatal.7b00688}
}
</pre></td>
</tr>
<tr id="Huang2004" class="entry">
	<td>Huang, F., Khvorova, A., Marshall, W. and Sorkin, A.</td>
	<td>Analysis of Clathrin-mediated Endocytosis of Epidermal Growth Factor Receptor by RNA Interference <p class="infolinks">[<a href="javascript:toggleInfo('Huang2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Huang2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of Biological Chemistry<br/>Vol. 279(16), pp. 16657 - 16661&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.C400046200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Huang2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: To identify proteins that participate in clathrin-mediated endocytosis of the epidermal growth factor receptor (EGFR), 13 endocytic proteins were depleted in HeLa cells using highly efficient small interfering RNAs that were designed using a novel selection algorithm. The effects of small interfering RNAs on the ligand-induced endocytosis of EGFR were compared with those effects on the constitutive internalization of the transferrin receptor. The knock-downs of clathrin heavy chain and dynamin produced maximal inhibitory effects on the internalization of both receptors. Depletion of alpha, beta2, or micro2 subunits of AP-2 reduced EGF and transferrin internalization rates by 40-60%. Down-regulation of several accessory proteins individually had no effect on endocytosis but caused significant inhibition of EGF and transferrin endocytosis when the homologous proteins were depleted simultaneously. Surprisingly, knockdown of clathrin-assembly lymphoid myeloid leukemia protein, CALM, did not influence transferrin endocytosis but considerably affected EGFR internalization. Thus, CALM is the second protein besides Grb2 that appears to play a specific role in EGFR endocytosis. This study demonstrates that the efficient gene silencing by rationally designed small interfering RNA can be used as an approach to functionally analyze the entire cellular machineries, such as the clathrin-coated pits and vesicles.</td>
</tr>
<tr id="bib_Huang2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Huang2004,
  author = {Huang, Fangtian and Khvorova, Anastasia and Marshall, William and Sorkin, Alexander},
  title = {Analysis of Clathrin-mediated Endocytosis of Epidermal Growth Factor Receptor by RNA Interference},
  journal = {Journal of Biological Chemistry},
  year = {2004},
  volume = {279},
  number = {16},
  pages = {16657 - 16661},
  doi = {https://doi.org/10.1074/jbc.C400046200}
}
</pre></td>
</tr>
<tr id="Huang2008" class="entry">
	<td>Huang, J., Qi, Z., Wu, F., Kotula, L., Jiang, S. and Chen, Y.</td>
	<td>Interaction of HIV-1 gp41 core with NPF motif in epsin: Implication in endocytosis of HIV <p class="infolinks">[<a href="javascript:toggleInfo('Huang2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Huang2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Journal of Biological Chemistry<br/>Vol. 283(22), pp. 14994 - 15002&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M800525200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Huang2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The human immunodeficiency virus, type 1 (HIV-1), gp41 core plays an important role in fusion between viral and target cell membranes. We previously identified an HIV-1 gp41 core-binding motif HXXNPF (where X is any amino acid residue). In this study, we found that Asn, Pro, and Phe were the key residues for gp41 core binding. There are two NPF motifs in Epsin-1-(470-499), a fragment of Epsin, which is an essential accessory factor of endocytosis that can dock to the plasma membrane by interacting with the lipid. Epsin-1-(470-499) bound significantly to the gp41 core formed by the polypeptide N36(L8)C34 and interacted with the recombinant soluble gp41 containing the core structure. A synthetic peptide containing the Epsin-1-(470-499) sequence could effectively block entry of HIV-1 virions into SupT1 T cells via the endocytosis pathway. These results suggest that interaction between Epsin and the gp41 core, which may be present in the target cell membrane, is probably essential for endocytosis of HIV-1, an alternative pathway of HIV-1 entry into the target cell.</td>
</tr>
<tr id="bib_Huang2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Huang2008,
  author = {Huang, Jing and Qi, Zhi and Wu, Fan and Kotula, Leszek and Jiang, Shibo and Chen, Ying},
  title = {Interaction of HIV-1 gp41 core with NPF motif in epsin: Implication in endocytosis of HIV},
  journal = {Journal of Biological Chemistry},
  year = {2008},
  volume = {283},
  number = {22},
  pages = {14994 - 15002},
  note = {HEK cells. eps15-epn1. domain information.},
  doi = {https://doi.org/10.1074/jbc.M800525200}
}
</pre></td>
</tr>
<tr id="Huang1997" class="entry">
	<td>Huang, K.M., Gullberg, L., Nelson, K.K., Stefan, C.J., Blumer, K. and Lemmon, S.K.</td>
	<td>Novel functions of clathrin light chains: clathrin heavy chain trimerization is defective in light chain-deficient yeast. <p class="infolinks">[<a href="javascript:toggleInfo('Huang1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Huang1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Journal of cell science<br/>Vol. 110 ( Pt 7), pp. 899-910&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Huang1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin is a major coat protein involved in sorting and retention of proteins at the late Golgi and in endocytosis from the cell surface. The clathrin triskelion contains three heavy chains, which provide the structural backbone of the clathrin lattice and three light chains, which are thought to regulate the formation or disassembly of clathrin coats. To better understand the function of the clathrin light chain, we characterized yeast strains carrying a disruption of the clathrin light chain gene (CLC1). Light chain-deficient cells showed phenotypes similar to those displayed by yeast that have a disruption in the clathrin heavy chain gene (CHC1). In clc1-delta cells, the steady state level of the clathrin heavy chain was reduced to 20%-25% of wild-type levels and most of the heavy chain was not trimerized. If CHC1 was overexpressed in clc1-delta cells, heavy chain trimers were detected and several clc1-delta phenotypes were partially rescued. These results indicate that the light chain is important for heavy chain trimerization and the heavy chain still has some function in the absence of the light chain. In yeast, deletion of CHC1 is lethal in strains carrying the scd1-i allele, while strains carrying the scd1-v allele can survive without the heavy chain. In previous studies we isolated several multicopy suppressors of inviability of chc1-delta scd1-i cells. Surprisingly, one of these suppressors, SCD4, is identical to CLC1. Overexpression of CLC1 in viable chc1-delta scd1-v strains rescued some but not all of the phenotypes displayed by these cells. In the absence of the heavy chain, the light chain was not found in a high molecular mass complex, but still associated with membranes. These results suggest that the light chain can function independently of the clathrin heavy chain in yeast.</td>
</tr>
<tr id="bib_Huang1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Huang1997,
  author = {Huang, K M and Gullberg, L and Nelson, K K and Stefan, C J and Blumer, K and Lemmon, S K},
  title = {Novel functions of clathrin light chains: clathrin heavy chain trimerization is defective in light chain-deficient yeast.},
  journal = {Journal of cell science},
  year = {1997},
  volume = {110 ( Pt 7)},
  pages = {899--910},
  note = {HC trimerization without LC is defective.}
}
</pre></td>
</tr>
<tr id="Huang2016" class="entry">
	<td>Huang, Po-Ssu., Boyken, S.E. and Baker, D.</td>
	<td>The coming of age of de novo protein design. <p class="infolinks">[<a href="javascript:toggleInfo('Huang2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Huang2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Nature<br/>Vol. 537(7620), pp. 320 - 7&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nature19946">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Huang2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: There are 20(200) possible amino-acid sequences for a 200-residue protein, of which the natural evolutionary process has sampled only an infinitesimal subset. De novo protein design explores the full sequence space, guided by the physical principles that underlie protein folding. Computational methodology has advanced to the point that a wide range of structures can be designed from scratch with atomic-level accuracy. Almost all protein engineering so far has involved the modification of naturally occurring proteins; it should now be possible to design new functional proteins from the ground up to tackle current challenges in biomedicine and nanotechnology.</td>
</tr>
<tr id="bib_Huang2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Huang2016,
  author = {Huang, Po-Ssu and Boyken, Scott E and Baker, David},
  title = {The coming of age of de novo protein design.},
  journal = {Nature},
  year = {2016},
  volume = {537},
  number = {7620},
  pages = {320 - 7},
  doi = {https://doi.org/10.1038/nature19946}
}
</pre></td>
</tr>
<tr id="Huang2016a" class="entry">
	<td>Huang, Q., Tran, K., Rodgers, J., Bartlett, D., Hemley, R. and Ichiye, T.</td>
	<td>A molecular perspective on the limits of life: Enzymes under pressure <p class="infolinks">[<a href="javascript:toggleInfo('Huang2016a','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Condensed Matter Physics<br/>Vol. 19(2)&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Huang2016a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Huang2016a,
  author = {Huang, Q and Tran, KN and Rodgers, JM and Bartlett, DH and Hemley, RJ and Ichiye, T},
  title = {A molecular perspective on the limits of life: Enzymes under pressure},
  journal = {Condensed Matter Physics},
  year = {2016},
  volume = {19},
  number = {2}
}
</pre></td>
</tr>
<tr id="Hummer2004" class="entry">
	<td>Hummer, G.</td>
	<td>From transition paths to transition states and rate coefficients <p class="infolinks">[<a href="javascript:toggleInfo('Hummer2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hummer2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of Chemical Physics<br/>Vol. 120(2), pp. 516 - 523&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.1630572">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hummer2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Transition states are defined as points in configuration space with the highest probability that trajectories passing through them are reactive (i.e., form transition paths between reactants and products). In the high-friction (diffusive) limit of Langevin dynamics, the resulting ensemble of transition states is shown to coincide with the separatrix formed by points of equal commitment (or splitting) probabilities for reaching the product and reactant regions. Transition states according to the new criterion can be identified directly from equilibrium trajectories, or indirectly by calculating probability densities in the equilibrium and transition-path ensembles using umbrella and transition-path sampling, respectively. An algorithm is proposed to calculate rate coefficients from the transition-path and equilibrium ensembles by estimating the frequency of transitions between reactants and products.</td>
</tr>
<tr id="bib_Hummer2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hummer2004,
  author = {Hummer, Gerhard},
  title = {From transition paths to transition states and rate coefficients},
  journal = {Journal of Chemical Physics},
  year = {2004},
  volume = {120},
  number = {2},
  pages = {516 - 523},
  doi = {https://doi.org/10.1063/1.1630572}
}
</pre></td>
</tr>
<tr id="Husic2018" class="entry">
	<td>Husic, B.E. and Pande, V.S.</td>
	<td>Markov State Models: From an Art to a Science <p class="infolinks">[<a href="javascript:toggleInfo('Husic2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Journal of the American Chemical Society&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jacs.7b12191">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Husic2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Husic2018,
  author = {Brooke E. Husic and Vijay S. Pande},
  title = {Markov State Models: From an Art to a Science},
  journal = {Journal of the American Chemical Society},
  publisher = {American Chemical Society (ACS)},
  year = {2018},
  doi = {https://doi.org/10.1021/jacs.7b12191}
}
</pre></td>
</tr>
<tr id="Huskey1991" class="entry">
	<td>Huskey, P.W.</td>
	<td>Origin of apparent Swain--Schaad deviations in criteria for tunneling <p class="infolinks">[<a href="javascript:toggleInfo('Huskey1991','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Huskey1991','bibtex')">BibTeX</a>]</p></td>
	<td>1991</td>
	<td>Journal of Physical Organic Chemistry<br/>Vol. 4(6), pp. 361 - 366&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/poc.610040607">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Huskey1991" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Vibrational analysis calculations have been conducted in an effort to understand the origins of apparently anomalous relationships reported between kH/kT and kD/kT kinetic isotope effects. The actual isotopic substitutions used in previous work are more complex than a simple comparison between H/T and D/T isotope effects. The relationships between the actual isotope effects determined in these studies amount to tests of two standards of conventional isotope effect theory, the rule of the geometric mean (or the lack of isotope effects on isotope effects) and the Swain---Schaad rule. Model calculations illustrate the importance of violations of the rule of the geometric mean (over Swain---Schaad deviations) arising in models that incorporate both explicit reaction-coordinate coupling of two isotopic sites and reaction-coordinate tunneling. Implications for experimental studies of turneling using apparent Swain---Schaad exponents are discussed.</td>
</tr>
<tr id="bib_Huskey1991" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Huskey1991,
  author = {Huskey, Phillip W},
  title = {Origin of apparent Swain--Schaad deviations in criteria for tunneling},
  journal = {Journal of Physical Organic Chemistry},
  year = {1991},
  volume = {4},
  number = {6},
  pages = {361 - 366},
  doi = {https://doi.org/10.1002/poc.610040607}
}
</pre></td>
</tr>
<tr id="Hussain2001" class="entry">
	<td>Hussain, N.K., Jenna, S., Glogauer, M., Quinn, C.C., Wasiak, S., Guipponi, M., Antonarakis, S.E., Kay, B.K., Stossel, T.P., Lamarche-Vane, N. and McPherson, P.S.</td>
	<td>Endocytic protein intersectin-l regulates actin assembly via Cdc42 and N-WASP. <p class="infolinks">[<a href="javascript:toggleInfo('Hussain2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hussain2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Nature cell biology<br/>Vol. 3, pp. 927-932&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/ncb1001-927">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hussain2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Intersectin-s is a modular scaffolding protein regulating the formation of clathrin-coated vesicles. In addition to the Eps15 homology (EH) and Src homology 3 (SH3) domains of intersectin-s, the neuronal variant (intersectin-l) also has Dbl homology (DH), pleckstrin homology (PH) and C2 domains. We now show that intersectin-l functions through its DH domain as a guanine nucleotide exchange factor (GEF) for Cdc42. In cultured cells, expression of DH-domain-containing constructs cause actin rearrangements specific for Cdc42 activation. Moreover, in vivo studies reveal that stimulation of Cdc42 by intersectin-l accelerates actin assembly via N-WASP and the Arp2/3 complex. N-WASP binds directly to intersectin-l and upregulates its GEF activity, thereby generating GTP-bound Cdc42, a critical activator of N-WASP. These studies reveal a role for intersectin-l in a novel mechanism of N-WASP activation and in regulation of the actin cytoskeleton.</td>
</tr>
<tr id="bib_Hussain2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hussain2001,
  author = {Hussain, N K and Jenna, S and Glogauer, M and Quinn, C C and Wasiak, S and Guipponi, M and Antonarakis, S E and Kay, B K and Stossel, T P and Lamarche-Vane, N and McPherson, P S},
  title = {Endocytic protein intersectin-l regulates actin assembly via Cdc42 and N-WASP.},
  journal = {Nature cell biology},
  year = {2001},
  volume = {3},
  pages = {927--932},
  doi = {https://doi.org/10.1038/ncb1001-927}
}
</pre></td>
</tr>
<tr id="Hussain2003" class="entry">
	<td>Hussain, N.K., Yamabhai, M., Bhakar, A.L., Metzler, M., Ferguson, S.S.G., Hayden, M.R., McPherson, P.S. and Kay, B.K.</td>
	<td>A role for epsin N-terminal homology/AP180 N-terminal homology (ENTH/ANTH) domains in tubulin binding. <p class="infolinks">[<a href="javascript:toggleInfo('Hussain2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hussain2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>The Journal of biological chemistry<br/>Vol. 278, pp. 28823-28830&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M300995200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hussain2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The epsin N-terminal homology (ENTH) domain is a protein module of approximately 150 amino acids found at the N terminus of a variety of proteins identified in yeast, plants, nematode, frog, and mammals. ENTH domains comprise multiple alpha-helices folded upon each other to form a compact globular structure that has been implicated in interactions with lipids and proteins. In characterizing this evolutionarily conserved domain, we isolated and identified tubulin as an ENTH domain-binding partner. The interaction, which is direct and has a dissociation constant of approximately 1 microm, was observed with ENTH domains of proteins present in various species. Tubulin is co-immunoprecipitated from rat brain extracts with the ENTH domain-containing proteins, epsins 1 and 2, and punctate epsin staining is observed along the microtubule cytoskeleton of dissociated cortical neurons. Consistent with a role in microtubule processes, the over-expression of epsin ENTH domain in PC12 cells stimulates neurite outgrowth. These data demonstrate an evolutionarily conserved property of ENTH domains to interact with tubulin and microtubules.</td>
</tr>
<tr id="bib_Hussain2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hussain2003,
  author = {Hussain, Natasha K and Yamabhai, Montarop and Bhakar, Asha L and Metzler, Martina and Ferguson, Stephen S G and Hayden, Michael R and McPherson, Peter S and Kay, Brian K},
  title = {A role for epsin N-terminal homology/AP180 N-terminal homology (ENTH/ANTH) domains in tubulin binding.},
  journal = {The Journal of biological chemistry},
  year = {2003},
  volume = {278},
  pages = {28823--28830},
  doi = {https://doi.org/10.1074/jbc.M300995200}
}
</pre></td>
</tr>
<tr id="Hussain1999" class="entry">
	<td>Hussain, N.K., Yamabhai, M., Ramjaun, A.R., Guy, A.M., Baranes, D., O'Bryan, J.P., Der, C.J., Kay, B.K. and McPherson, P.S.</td>
	<td>Splice variants of intersectin are components of the endocytic machinery in neurons and nonneuronal cells. <p class="infolinks">[<a href="javascript:toggleInfo('Hussain1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hussain1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>The Journal of biological chemistry<br/>Vol. 274, pp. 15671-15677&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Hussain1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We recently identified and cloned intersectin, a protein containing two Eps15 homology (EH) domains and five Src homology 3 (SH3) domains. Using a newly developed intersectin antibody, we demonstrate that endogenous COS-7 cell intersectin localizes to clathrin-coated pits, and transfection studies suggest that the EH domains may direct this localization. Through alternative splicing in a stop codon, a long form of intersectin is generated with a C-terminal extension containing Dbl homology (DH), pleckstrin homology (PH), and C2 domains. Western blots reveal that the long form of intersectin is expressed specifically in neurons, whereas the short isoform is expressed at lower levels in glia and other nonneuronal cells. Immunofluorescence analysis of cultured hippocampal neurons reveals that intersectin is found at the plasma membrane where it is co-localized with clathrin. Ibp2, a protein identified based on its interactions with the EH domains of intersectin, binds to clathrin through the N terminus of the heavy chain, suggesting a mechanism for the localization of intersectin at clathrin-coated pits. Ibp2 also binds to the clathrin adaptor AP2, and antibodies against intersectin co-immunoprecipitate clathrin, AP2, and dynamin from brain extracts. These data suggest that the long and short forms of intersectin are components of the endocytic machinery in neurons and nonneuronal cells.</td>
</tr>
<tr id="bib_Hussain1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hussain1999,
  author = {Hussain, N K and Yamabhai, M and Ramjaun, A R and Guy, A M and Baranes, D and O'Bryan, J P and Der, C J and Kay, B K and McPherson, P S},
  title = {Splice variants of intersectin are components of the endocytic machinery in neurons and nonneuronal cells.},
  journal = {The Journal of biological chemistry},
  year = {1999},
  volume = {274},
  pages = {15671--15677}
}
</pre></td>
</tr>
<tr id="Huttlin2017" class="entry">
	<td>Huttlin, E.L., Bruckner, R.J., Paulo, J.A., Cannon, J.R., Ting, L., Baltier, K., Colby, G., Gebreab, F., Gygi, M.P., Parzen, H., Szpyt, J., Tam, S., Zarraga, G., Pontano-Vaites, L., Swarup, S., White, A.E., Schweppe, D.K., Rad, R., Erickson, B.K., Obar, R.A., Guruharsha, K., Li, K., Artavanis-Tsakonas, S., Gygi, S.P. and Harper, W.J.</td>
	<td>Architecture of the human interactome defines protein communities and disease networks. <p class="infolinks">[<a href="javascript:toggleInfo('Huttlin2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Huttlin2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Nature<br/>Vol. 545(7655), pp. 505 - 509&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nature22366">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Huttlin2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The physiology of a cell can be viewed as the product of thousands of proteins acting in concert to shape the cellular response. Coordination is achieved in part through networks of protein-protein interactions that assemble functionally related proteins into complexes, organelles, and signal transduction pathways. Understanding the architecture of the human proteome has the potential to inform cellular, structural, and evolutionary mechanisms and is critical to elucidating how genome variation contributes to disease. Here we present BioPlex 2.0 (Biophysical Interactions of ORFeome-derived complexes), which uses robust affinity purification-mass spectrometry methodology to elucidate protein interaction networks and co-complexes nucleated by more than 25&#37; of protein-coding genes from the human genome, and constitutes, to our knowledge, the largest such network so far. With more than 56,000 candidate interactions, BioPlex 2.0 contains more than 29,000 previously unknown co-associations and provides functional insights into hundreds of poorly characterized proteins while enhancing network-based analyses of domain associations, subcellular localization, and co-complex formation. Unsupervised Markov clustering of interacting proteins identified more than 1,300 protein communities representing diverse cellular activities. Genes essential for cell fitness are enriched within 53 communities representing central cellular functions. Moreover, we identified 442 communities associated with more than 2,000 disease annotations, placing numerous candidate disease genes into a cellular framework. BioPlex 2.0 exceeds previous experimentally derived interaction networks in depth and breadth, and will be a valuable resource for exploring the biology of incompletely characterized proteins and for elucidating larger-scale patterns of proteome organization.</td>
</tr>
<tr id="bib_Huttlin2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Huttlin2017,
  author = {Huttlin, Edward L and Bruckner, Raphael J and Paulo, Joao A and Cannon, Joe R and Ting, Lily and Baltier, Kurt and Colby, Greg and Gebreab, Fana and Gygi, Melanie P and Parzen, Hannah and Szpyt, John and Tam, Stanley and Zarraga, Gabriela and Pontano-Vaites, Laura and Swarup, Sharan and White, Anne E and Schweppe, Devin K and Rad, Ramin and Erickson, Brian K and Obar, Robert A and Guruharsha, KG and Li, Kejie and Artavanis-Tsakonas, Spyros and Gygi, Steven P and Harper, Wade J},
  title = {Architecture of the human interactome defines protein communities and disease networks.},
  journal = {Nature},
  year = {2017},
  volume = {545},
  number = {7655},
  pages = {505 - 509},
  doi = {https://doi.org/10.1038/nature22366}
}
</pre></td>
</tr>
<tr id="Huttlin2016" class="entry">
	<td>Huttlin, E.L., Ting, L., Bruckner, R.J., Gebreab, F., Gygi, M.P., Szpyt, J., Tam, S., Zarraga, G., Colby, G., Baltier, K., Dong, R., Guarani, V., Vaites, L., Ordureau, A., Rad, R., Brian, K., W&uuml;hr, M., Chick, J., Zhai, B., Kolippakkam, D., Mintseris, J., Obar, R.A., Harris, T., Artavanis-tsakonas, S., Sowa, M.E., Paulo, J.A., Harper, W.J. and Gygi, S.P.</td>
	<td>The BioPlex Network: A Systematic Exploration of the Human Interactome <p class="infolinks">[<a href="javascript:toggleInfo('Huttlin2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Huttlin2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Cell<br/>Vol. 162(2), pp. 425 - 440&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cell.2015.06.043.The">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Huttlin2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Protein interactions form a network whose structure drives cellular function and whose organization informs biological inquiry. Using high-throughput affinity-purification mass spectrometry, we identify interacting partners for 2,594 human proteins in HEK293T cells. The resulting network (BioPlex) contains 23,744 interactions among 7,668 proteins with 86&#37; previously undocumented. BioPlex accurately depicts known complexes, attaining 80-100&#37; coverage for most CORUM complexes. The network readily subdivides into communities that correspond to complexes or clusters of functionally related proteins. More generally, network architecture reflects cellular localization, biological process, and molecular function, enabling functional characterization of thousands of proteins. Network structure also reveals associations among thousands of protein domains, suggesting a basis for examining structurally related proteins. Finally, BioPlex, in combination with other approaches, can be used to reveal interactions of biological or clinical significance. For example, mutations in the membrane protein VAPB implicated in familial amyotrophic lateral sclerosis perturb a defined community of interactors.</td>
</tr>
<tr id="bib_Huttlin2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Huttlin2016,
  author = {Huttlin, Edward L and Ting, Lily and Bruckner, Raphael J and Gebreab, Fana and Gygi, Melanie P and Szpyt, John and Tam, Stanley and Zarraga, Gabriela and Colby, Greg and Baltier, Kurt and Dong, Rui and Guarani, Virginia and Vaites, Laura and Ordureau, Alban and Rad, Ramin and Brian, K and W&uuml;hr, Martin and Chick, Joel and Zhai, Bo and Kolippakkam, Deepak and Mintseris, Julian and Obar, Robert A and Harris, Tim and Artavanis-tsakonas, Spyros and Sowa, Mathew E and Paulo, Joao A and Harper, Wade J and Gygi, Steven P},
  title = {The BioPlex Network: A Systematic Exploration of the Human Interactome},
  journal = {Cell},
  year = {2016},
  volume = {162},
  number = {2},
  pages = {425 - 440},
  doi = {https://doi.org/10.1016/j.cell.2015.06.043.The}
}
</pre></td>
</tr>
<tr id="Hwang1991" class="entry">
	<td>Hwang, J., Chu, Z., Yadov, A. and Warshel, A.</td>
	<td>Simulations of quantum mechanical corrections for rate constants of hydride-transfer reactions in enzymes and solutions <p class="infolinks">[<a href="javascript:toggleInfo('Hwang1991','bibtex')">BibTeX</a>]</p></td>
	<td>1991</td>
	<td>Journal of Physical Chemistry<br/>Vol. 95(22), pp. 8445 - 8448&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Hwang1991" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hwang1991,
  author = {Hwang, J.K. and Chu, Z.T. and Yadov, A and Warshel, A},
  title = {Simulations of quantum mechanical corrections for rate constants of hydride-transfer reactions in enzymes and solutions},
  journal = {Journal of Physical Chemistry},
  year = {1991},
  volume = {95},
  number = {22},
  pages = {8445 - 8448}
}
</pre></td>
</tr>
<tr id="Hwang1993" class="entry">
	<td>Hwang, J. and Warshel, A.</td>
	<td>A quantized classical path approach for calculations of quantum mechanical rate constants <p class="infolinks">[<a href="javascript:toggleInfo('Hwang1993','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hwang1993','bibtex')">BibTeX</a>]</p></td>
	<td>1993</td>
	<td>Journal of Physical Chemistry<br/>Vol. 97(39), pp. 10053 - 10058&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/j100141a026">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hwang1993" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A practical method for calcns. of quantized rate consts. is examd., and illustrated. The method, which is referred to as the \textbackslash\"quantized classical path\textbackslash\" (QCP) approach, evaluates quantum-mech. activation energies by propagating trajectories of the classical particles to generate the centroid positions for the quantum-mech. calcns. This provides a stable and effective way for evaluating quantized activation energies of processes in multidimensional systems. In fact, the present QCP approach was used before in studies of proton-transfer and hydride-transfer reactions in soln. and proteins (H., et al., 1991), but its validity was not established in well-defined test cases. The present work compares the results of the QCP method to that of a quantum-mech. calcn. of the reactive flux for an Eckart potential coupled to harmonic oscillators (R. P. McRae, et al., 1992), and demonstrates that the QCP approach provides reliable results. The performance of the QCP method is also illustrated in evaluation of partition functions by considering the HCl system studied by R. Q. Topper, et al., (1992), demonstrating accuracy and fast convergence. Finally, the applicability of the QCP method to multidimensional problems is illustrated by calcg. isotope effects in a proton-transfer reaction in aq. soln. [on SciFinder (R)]</td>
</tr>
<tr id="bib_Hwang1993" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hwang1993,
  author = {Hwang, Jenn and Warshel, Arieh},
  title = {A quantized classical path approach for calculations of quantum mechanical rate constants},
  journal = {Journal of Physical Chemistry},
  year = {1993},
  volume = {97},
  number = {39},
  pages = {10053 - 10058},
  doi = {https://doi.org/10.1021/j100141a026}
}
</pre></td>
</tr>
<tr id="Hynes1985" class="entry">
	<td>Hynes, J.T.</td>
	<td>Chemical reaction dynamics in solution <p class="infolinks">[<a href="javascript:toggleInfo('Hynes1985','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hynes1985','bibtex')">BibTeX</a>]</p></td>
	<td>1985</td>
	<td>Annual Review of Physical Chemistry<br/>Vol. 36(1), pp. 573 - 597&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1146/annurev.pc.36.100185.003041">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hynes1985" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In contrast to the gas phase, where reactants (usually) follow their route to products in splendid isolation, the ubiquitous solvent molecules in solution continually perturb the reactants in their course. The first way in which a solvent medium can affect a reaction rate is via a modification of the activation (free) energy. The second way in which a solvent can influence a reaction rate is via frictional, collisional effects. This review focuses on key prototypical solution reaction types and the progress that has been made in their elucidation. These are radical recombinations, atom transfers, isomerizations, charge transfers, and tunneling reactions.</td>
</tr>
<tr id="bib_Hynes1985" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hynes1985,
  author = {Hynes, James T},
  title = {Chemical reaction dynamics in solution},
  journal = {Annual Review of Physical Chemistry},
  year = {1985},
  volume = {36},
  number = {1},
  pages = {573 - 597},
  doi = {https://doi.org/10.1146/annurev.pc.36.100185.003041}
}
</pre></td>
</tr>
<tr id="Hyun2004" class="entry">
	<td>Hyun, T.S., Rao, D.S., Saint-Dic, D., Michael, L.E., Kumar, P.D., Bradley, S.V., Mizukami, I.F., Oravecz-Wilson, K.I. and Ross, T.S.</td>
	<td>HIP1 and HIP1r stabilize receptor tyrosine kinases and bind 3-phosphoinositides via epsin N-terminal homology domains. <p class="infolinks">[<a href="javascript:toggleInfo('Hyun2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Hyun2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>The Journal of biological chemistry<br/>Vol. 279, pp. 14294-14306&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M312645200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Hyun2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Huntingtin-interacting protein 1-related (HIP1r) is the only known mammalian relative of huntingtin-interacting protein 1 (HIP1), a protein that transforms fibroblasts via undefined mechanisms. Here we demonstrate that both HIP1r and HIP1 bind inositol lipids via their epsin N-terminal homology (ENTH) domains. In contrast to other ENTH domain-containing proteins, lipid binding is preferential to the 3-phosphate-containing inositol lipids, phosphatidylinositol 3,4-bisphosphate and phosphatidylinositol 3,5-bisphosphate. Furthermore, the HIP1r ENTH domain, like that of HIP1, is necessary for lipid binding, and expression of an ENTH domain-deletion mutant, HIP1r/deltaE, induces apoptosis. Consistent with the ability of HIP1r and HIP1 to affect cell survival, full-length HIP1 and HIP1r stabilize pools of growth factor receptors by prolonging their half-life following ligand-induced endocytosis. Although HIP1r and HIP1 display only a partially overlapping pattern of protein interactions, these data suggest that both proteins share a functional homology by binding 3-phosphorylated inositol lipids and stabilizing receptor tyrosine kinases in a fashion that may contribute to their ability to alter cell growth and survival.</td>
</tr>
<tr id="bib_Hyun2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Hyun2004,
  author = {Hyun, Teresa S and Rao, Dinesh S and Saint-Dic, Djenann and Michael, L Evan and Kumar, Priti D and Bradley, Sarah V and Mizukami, Ikuko F and Oravecz-Wilson, Katherine I and Ross, Theodora S},
  title = {HIP1 and HIP1r stabilize receptor tyrosine kinases and bind 3-phosphoinositides via epsin N-terminal homology domains.},
  journal = {The Journal of biological chemistry},
  year = {2004},
  volume = {279},
  pages = {14294--14306},
  doi = {https://doi.org/10.1074/jbc.M312645200}
}
</pre></td>
</tr>
<tr id="Im2011" class="entry">
	<td>Im, H., Sharpe, M.L., Strych, U., Davlieva, M. and Krause, K.L.</td>
	<td>The crystal structure of alanine racemase from Streptococcus pneumoniae, a target for structure-based drug design. <p class="infolinks">[<a href="javascript:toggleInfo('Im2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Im2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>BMC microbiology<br/>Vol. 11(1), pp. 116 - 116&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1186/1471-2180-11-116">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Im2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Background Streptococcus pneumoniae is a globally important pathogen. The Gram-positive diplococcus is a leading cause of pneumonia, otitis media, bacteremia, and meningitis, and antibiotic resistant strains have become increasingly common over recent years.Alanine racemase is a ubiquitous enzyme among bacteria and provides the essential cell wall precursor, D-alanine. Since it is absent in humans, this enzyme is an attractive target for the development of drugs against S. pneumoniae and other bacterial pathogens. Results Here we report the crystal structure of alanine racemase from S. pneumoniae (AlrSP). Crystals diffracted to a resolution of 2.0 &Aring; and belong to the space group P3121 with the unit cell parameters a = b = 119.97 &Aring;, c = 118.10 &Aring;, α = β = 90textdegree and γ = 120textdegree. Structural comparisons show that AlrSP shares both an overall fold and key active site residues with other bacterial alanine racemases. The active site cavity is similar to other Gram positive alanine racemases, featuring a restricted but conserved entryway. Conclusions We have solved the structure of AlrSP, an essential step towards the development of an accurate pharmacophore model of the enzyme, and an important contribution towards our on-going alanine racemase structure-based drug design project. We have identified three regions on the enzyme that could be targeted for inhibitor design, the active site, the dimer interface, and the active site entryway.</td>
</tr>
<tr id="bib_Im2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Im2011,
  author = {Im, Hookang and Sharpe, Miriam L and Strych, Ulrich and Davlieva, Milya and Krause, Kurt L},
  title = {The crystal structure of alanine racemase from Streptococcus pneumoniae, a target for structure-based drug design.},
  journal = {BMC microbiology},
  year = {2011},
  volume = {11},
  number = {1},
  pages = {116 - 116},
  note = {Crystal structure of alanine racemase from Streptococcus pneumoniae},
  doi = {https://doi.org/10.1186/1471-2180-11-116}
}
</pre></td>
</tr>
<tr id="Inagaki1986" class="entry">
	<td>Inagaki, K., Tanizawa, K., Badet, B., Walsh, C., Tanaka, H. and Soda, K.</td>
	<td>Thermostable alanine racemase from Bacillus stearothermophilus: molecular cloning of the gene, enzyme purification, and characterization. <p class="infolinks">[<a href="javascript:toggleInfo('Inagaki1986','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Inagaki1986','bibtex')">BibTeX</a>]</p></td>
	<td>1986</td>
	<td>Biochemistry<br/>Vol. 25(11), pp. 3268 - 74&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Inagaki1986" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The alanine racemase (EC 5.1.1.1) gene of a thermophilic bacterium, Bacillus stearothermophilus, was cloned and expressed in Escherichia coli C600 with vector plasmid pICR301, which was constructed from pBR322 and the L-alanine dehydrogenase gene derived from B. stearothermophilus. A coupled assay method with L-alanine dehydrogenase and tetrazolium salts was used to detect visually the alanine racemase activity in the clones. Alanine racemase overproduced in a clone carrying the plasmid pICR4, 12 kilobases of DNA, was purified from cell extracts about 340-fold to homogeneity by five steps including heat treatment. The overproduced enzyme was confirmed to originate from B. stearothermophilus by an immunochemical cross-reaction with the enzyme of B. stearothermophilus. The purified enzyme has a molecular weight of about 78 000 and consists of two identical subunits of Mr of 39 000. At the optimum temperature (50 degrees C), the enzyme has a specific activity of 1800 units/mg (Vmax, D- to L-alanine). Resolution and reconstitution experiments together with the absorption spectrum of the enzyme clearly indicate that alanine racemase of B. stearothermophilus is a pyridoxal 5'-phosphate enzyme.</td>
</tr>
<tr id="bib_Inagaki1986" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Inagaki1986,
  author = {Inagaki, K and Tanizawa, K and Badet, B and Walsh, CT and Tanaka, H and Soda, K},
  title = {Thermostable alanine racemase from Bacillus stearothermophilus: molecular cloning of the gene, enzyme purification, and characterization.},
  journal = {Biochemistry},
  year = {1986},
  volume = {25},
  number = {11},
  pages = {3268 - 74},
  note = {Vmax of AlaR(Gst)}
}
</pre></td>
</tr>
<tr id="Irannejad2014" class="entry">
	<td>Irannejad, R. and von Zastrow, M.</td>
	<td>GPCR signaling along the endocytic pathway. <p class="infolinks">[<a href="javascript:toggleInfo('Irannejad2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Irannejad2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Current opinion in cell biology<br/>Vol. 27, pp. 109-116&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.ceb.2013.10.003">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Irannejad2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Many G protein-coupled receptors (GPCRs) internalize after agonist-induced activation. While endocytosis has long been associated with homeostatic attenuation of cellular responsiveness, accumulating evidence from study of a wide range of eukaryotes reveals that the endocytic pathway also contributes to generating receptor-initiated signals themselves. Here we review recent progress in this area, discussing primarily but not exclusively GPCR signaling in mammalian cells.</td>
</tr>
<tr id="bib_Irannejad2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Irannejad2014,
  author = {Irannejad, Roshanak and von Zastrow, Mark},
  title = {GPCR signaling along the endocytic pathway.},
  journal = {Current opinion in cell biology},
  year = {2014},
  volume = {27},
  pages = {109--116},
  doi = {https://doi.org/10.1016/j.ceb.2013.10.003}
}
</pre></td>
</tr>
<tr id="Itoh2006" class="entry">
	<td>Itoh, T. and De Camilli, P.</td>
	<td>BAR, F-BAR (EFC) and ENTH/ANTH domains in the regulation of membrane-cytosol interfaces and membrane curvature. <p class="infolinks">[<a href="javascript:toggleInfo('Itoh2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Itoh2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Biochimica et Biophysica Acta<br/>Vol. 1761, pp. 897-912&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.bbalip.2006.06.015">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Itoh2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: BAR and ENTH domains are families of alpha-helical lipid bilayer binding modules found in proteins that function in endocytosis, actin regulation and signaling. Several members of these families not only bind the bilayer, but also participate in the regulation of its curvature. These properties are thought to play physiological roles at sites of membrane budding and at other sites where narrow tubular membranes occur in vivo. Studies of BAR and ENTH domains and of their flanking regions have provided new insights into mechanisms of membrane deformation and curvature sensing, and have emphasized the importance of amphipathic helices, thought to intercalate in one of the leaflets of the lipid bilayer, in the generation of membrane curvature. Structural studies and database searches are rapidly expanding the BAR and ENTH domains families, with the identification of new related domains and subfamilies, such as F-BAR (also called EFC) domains and ANTH domains, respectively. Here we present a short overview of the properties of these domains based on evidence obtained from genetics, cell biology, biochemistry and structural biology.</td>
</tr>
<tr id="bib_Itoh2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Itoh2006,
  author = {Itoh, Toshiki and De Camilli, Pietro},
  title = {BAR, F-BAR (EFC) and ENTH/ANTH domains in the regulation of membrane-cytosol interfaces and membrane curvature.},
  journal = {Biochimica et Biophysica Acta},
  year = {2006},
  volume = {1761},
  pages = {897--912},
  doi = {https://doi.org/10.1016/j.bbalip.2006.06.015}
}
</pre></td>
</tr>
<tr id="Itoh2005" class="entry">
	<td>Itoh, T., Erdmann, K.S., Roux, A., Habermann, B., Werner, H. and De Camilli, P.</td>
	<td>Dynamin and the actin cytoskeleton cooperatively regulate plasma membrane invagination by BAR and F-BAR proteins. <p class="infolinks">[<a href="javascript:toggleInfo('Itoh2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Itoh2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Developmental cell<br/>Vol. 9, pp. 791-804&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.devcel.2005.11.005">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Itoh2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Cell membranes undergo continuous curvature changes as a result of membrane trafficking and cell motility. Deformations are achieved both by forces extrinsic to the membrane as well as by structural modifications in the bilayer or at the bilayer surface that favor the acquisition of curvature. We report here that a family of proteins previously implicated in the regulation of the actin cytoskeleton also have powerful lipid bilayer-deforming properties via an N-terminal module (F-BAR) similar to the BAR domain. Several such proteins, like a subset of BAR domain proteins, bind to dynamin, a GTPase implicated in endocytosis and actin dynamics, via SH3 domains. The ability of BAR and F-BAR domain proteins to induce tubular invaginations of the plasma membrane is enhanced by disruption of the actin cytoskeleton and is antagonized by dynamin. These results suggest a close interplay between the mechanisms that control actin dynamics and those that mediate plasma membrane invagination and fission.</td>
</tr>
<tr id="bib_Itoh2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Itoh2005,
  author = {Itoh, Toshiki and Erdmann, Kai S and Roux, Aurelien and Habermann, Bianca and Werner, Hauke and De Camilli, Pietro},
  title = {Dynamin and the actin cytoskeleton cooperatively regulate plasma membrane invagination by BAR and F-BAR proteins.},
  journal = {Developmental cell},
  year = {2005},
  volume = {9},
  pages = {791--804},
  doi = {https://doi.org/10.1016/j.devcel.2005.11.005}
}
</pre></td>
</tr>
<tr id="Itoh2001" class="entry">
	<td>Itoh, T., Koshiba, S., Kigawa, T., Kikuchi, A., Yokoyama, S. and Takenawa, T.</td>
	<td>Role of the ENTH domain in phosphatidylinositol-4,5-bisphosphate binding and endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Itoh2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Itoh2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Science (New York, N.Y.)<br/>Vol. 291, pp. 1047-1051&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1126/science.291.5506.1047">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Itoh2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Endocytic proteins such as epsin, AP180, and Hip1R (Sla2p) share a conserved modular region termed the epsin NH2-terminal homology (ENTH) domain, which plays a crucial role in clathrin-mediated endocytosis through an unknown target. Here, we demonstrate a strong affinity of the ENTH domain for phosphatidylinositol-4,5-bisphosphate [PtdIns(4,5)P2]. With nuclear magnetic resonance analysis of the epsin ENTH domain, we determined that a cleft formed with positively charged residues contributed to phosphoinositide binding. Overexpression of a mutant, epsin Lys76 --> Ala76, with an ENTH domain defective in phosphoinositide binding, blocked epidermal growth factor internalization in COS-7 cells. Thus, interaction between the ENTH domain and PtdIns(4,5)P2 is essential for endocytosis mediated by clathrin-coated pits.</td>
</tr>
<tr id="bib_Itoh2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Itoh2001,
  author = {Itoh, T and Koshiba, S and Kigawa, T and Kikuchi, A and Yokoyama, S and Takenawa, T},
  title = {Role of the ENTH domain in phosphatidylinositol-4,5-bisphosphate binding and endocytosis.},
  journal = {Science (New York, N.Y.)},
  year = {2001},
  volume = {291},
  pages = {1047--1051},
  note = {EPN1.ENTH - PIP2 (0.2%) with KD 0.37 uM. AP180.ANTH - PIP2 (0.2%) with KD weaker than 0.37 uM. dissociation constant based on fraction bound/unbound.},
  doi = {https://doi.org/10.1126/science.291.5506.1047}
}
</pre></td>
</tr>
<tr id="Jackson2003" class="entry">
	<td>Jackson, A.P., Flett, A., Smythe, C., Hufton, L., Wettey, F.R. and Smythe, E.</td>
	<td>Clathrin promotes incorporation of cargo into coated pits by activation of the AP2 adaptor &mu;2 kinase <p class="infolinks">[<a href="javascript:toggleInfo('Jackson2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>J Cell Biol<br/>Vol. 163(2), pp. 231-236&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Jackson2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Jackson2003,
  author = {Jackson, Antony P and Flett, Alexander and Smythe, Carl and Hufton, Lindsay and Wettey, Frank R and Smythe, Elizabeth},
  title = {Clathrin promotes incorporation of cargo into coated pits by activation of the AP2 adaptor &mu;2 kinase},
  journal = {J Cell Biol},
  publisher = {Rockefeller University Press},
  year = {2003},
  volume = {163},
  number = {2},
  pages = {231--236}
}
</pre></td>
</tr>
<tr id="Jackson2010" class="entry">
	<td>Jackson, L.P., Kelly, B.T., McCoy, A.J., Gaffry, T., James, L.C., Collins, B.M., Höning, S., Evans, P.R. and Owen, D.J.</td>
	<td>A large-scale conformational change couples membrane recruitment to cargo binding in the AP2 clathrin adaptor complex. <p class="infolinks">[<a href="javascript:toggleInfo('Jackson2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Jackson2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Cell<br/>Vol. 141, pp. 1220-1229&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cell.2010.05.006">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Jackson2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The AP2 adaptor complex (alpha, beta2, sigma2, and mu2 subunits) crosslinks the endocytic clathrin scaffold to PtdIns4,5P(2)-containing membranes and transmembrane protein cargo. In the "locked" cytosolic form, AP2's binding sites for the two endocytic motifs, YxxPhi on the C-terminal domain of mu2 (C-mu2) and [ED]xxxL[LI] on sigma2, are blocked by parts of beta2. Using protein crystallography, we show that AP2 undergoes a large conformational change in which C-mu2 relocates to an orthogonal face of the complex, simultaneously unblocking both cargo-binding sites; the previously unstructured mu2 linker becomes helical and binds back onto the complex. This structural rearrangement results in AP2's four PtdIns4,5P(2)- and two endocytic motif-binding sites becoming coplanar, facilitating their simultaneous interaction with PtdIns4,5P(2)/cargo-containing membranes. Using a range of biophysical techniques, we show that the endocytic cargo binding of AP2 is driven by its interaction with PtdIns4,5P(2)-containing membranes.</td>
</tr>
<tr id="bib_Jackson2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Jackson2010,
  author = {Jackson, Lauren P and Kelly, Bernard T and McCoy, Airlie J and Gaffry, Thomas and James, Leo C and Collins, Brett M and Höning, Stefan and Evans, Philip R and Owen, David J},
  title = {A large-scale conformational change couples membrane recruitment to cargo binding in the AP2 clathrin adaptor complex.},
  journal = {Cell},
  year = {2010},
  volume = {141},
  pages = {1220--1229},
  note = {ap2 open, closed. the conformational change is quite dramatic. claims that the mu-2 conformational change is due to electrostatic attaction to pip2.},
  doi = {https://doi.org/10.1016/j.cell.2010.05.006}
}
</pre></td>
</tr>
<tr id="Jagt1981" class="entry">
	<td>Jagt, D.L., Hunsaker, L.A. and Heidrich, J.E.</td>
	<td>Partial purification and characterization of lactate dehydrogenase from Plasmodium falciparum <p class="infolinks">[<a href="javascript:toggleInfo('Jagt1981','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Jagt1981','bibtex')">BibTeX</a>]</p></td>
	<td>1981</td>
	<td>Molecular and Biochemical Parasitology<br/>Vol. 4(5-6), pp. 255 - 264&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/0166-6851(81)90058-X">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Jagt1981" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Lactate dehydrogenase (LDH) from Plasmodium falciparum was partially purified by two different procedures. In the first procedure, parasitized erythrocytes (80&#37; parasitemia) were lysed, and the soluble fraction was purified on DEAE-Sephadex to separate the parasite LDH (LDH-P) from the LDH isozymes present in the human erythrocytes. LDH-P was then purified by high-performance liquid chromatography (HPLC) on a TSK-G-3000 SW protein column. This two-step procedure gave LDH-P with specific activity 85 ??mol/min/mg protein; this represented a 700-fold increase in specific activity relative to the starting lysate. Alternatively, parasites of P. falciparum were isolated by mechanical rupture of infected erythrocytes followed by differential centrifugation. The 100 000 ?? g supernatant obtained after lysis of these parasites showed LDH-P specific activity 3.6 ??mol/min/mg protein. This activity was free of contaminating erythrocyte LDH as determined by electrophoresis and specific staining for LDH. Further purification of LDH-P by HPLC, as before, gave material with specific activity 98 ??mol/min/mg protein. Recoveries of activity on HPLC were 90 demonstrating the usefulness of this procedure for the partial purification of small quantities of parasite protein. The kinetic properties of LDH-P were compared with those of two of the human isozymes, LDH-H4 and LDH-M4. LDH-P resembles LDH-H4 in its kinetic properties: KM (NADH) is 7, 8.3 and 1.3 ??M for LDH-P, LDH-H4 and LDH-M4, respectively; KM (pyruvate) is 30, 60 and 180 ??M for LDH-P, LDH-H4 and LDH-M4. LDH-P differs significantly from LDH-H4 and LDH-M4 in that LDH-P is not sensitive to inhibition by high pyruvate nor sensitive to inhibition by the complex between NAD+ and pyruvate. LDH-P is inactivated within seconds by sodium deoxycholate at concentrations that do not affect LDH-H4 and only slowly inactivate LDH-M4. ?? 1981.</td>
</tr>
<tr id="bib_Jagt1981" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Jagt1981,
  author = {Jagt, David L and Hunsaker, Lucy A and Heidrich, John E},
  title = {Partial purification and characterization of lactate dehydrogenase from Plasmodium falciparum},
  journal = {Molecular and Biochemical Parasitology},
  year = {1981},
  volume = {4},
  number = {5-6},
  pages = {255 - 264},
  doi = {https://doi.org/10.1016/0166-6851(81)90058-X}
}
</pre></td>
</tr>
<tr id="Jain2006" class="entry">
	<td>Jain, A.N.</td>
	<td>Scoring functions for protein-ligand docking. <p class="infolinks">[<a href="javascript:toggleInfo('Jain2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Jain2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Current Protein and Peptide Science<br/>Vol. 7(5), pp. 407 - 420&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.2174/138920306778559395">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Jain2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Virtual screening by molecular docking has become established as a method for drug lead discovery and optimization. All docking algorithms make use of a scoring function in combination with a method of search. Two theoretical aspects of scoring function performance dominate operational performance. The first is the degree to which a scoring function has a global extremum within the ligand pose landscape at the proper location. The second is the degree to which the magnitude of the function at the extremum is accurate. Presuming adequate search strategies, a scoring function's location performance will dominate behavior with respect to docking accuracy: the degree to which a predicted pose of a ligand matches experimental observation. A scoring function's magnitude performance will dominate behavior with respect to screening utility: enrichment of true ligands over non-ligands. Magnitude estimation also controls pure scoring accuracy: the degree to which bona fide ligands of a particular protein may be correctly ranked. Approaches to the development of scoring functions have varied widely, with a number of functions yielding similarly high levels of performance relating to the location issue. However, even among functions performing equally well on location, widely varying performance is observed on the question of magnitude. In many cases, performance is good enough to yield high enrichments of true ligands versus non-ligands in screening across a wide variety of protein types. Generally, performance is not good enough to correctly rank among true ligands. Strategies for improvement are discussed.</td>
</tr>
<tr id="bib_Jain2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Jain2006,
  author = {Jain, Ajay N},
  title = {Scoring functions for protein-ligand docking.},
  journal = {Current Protein and Peptide Science},
  year = {2006},
  volume = {7},
  number = {5},
  pages = {407 - 420},
  doi = {https://doi.org/10.2174/138920306778559395}
}
</pre></td>
</tr>
<tr id="Jensen2006" class="entry">
	<td>Jensen, L., Saric, J. and Bork, P.</td>
	<td>Literature mining for the biologist: from information retrieval to biological discovery. <p class="infolinks">[<a href="javascript:toggleInfo('Jensen2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Jensen2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Nature Reviews Genetics<br/>Vol. 7(2), pp. 119 - 29&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nrg1768">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Jensen2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: For the average biologist, hands-on literature mining currently means a keyword search in PubMed. However, methods for extracting biomedical facts from the scientific literature have improved considerably, and the associated tools will probably soon be used in many laboratories to automatically annotate and analyse the growing number of system-wide experimental data sets. Owing to the increasing body of text and the open-access policies of many journals, literature mining is also becoming useful for both hypothesis generation and biological discovery. However, the latter will require the integration of literature and high-throughput data, which should encourage close collaborations between biologists and computational linguists.</td>
</tr>
<tr id="bib_Jensen2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Jensen2006,
  author = {Jensen, Lars and Saric, Jasmin and Bork, Peer},
  title = {Literature mining for the biologist: from information retrieval to biological discovery.},
  journal = {Nature Reviews Genetics},
  year = {2006},
  volume = {7},
  number = {2},
  pages = {119 - 29},
  doi = {https://doi.org/10.1038/nrg1768}
}
</pre></td>
</tr>
<tr id="Jeong2000" class="entry">
	<td>Jeong, H., Tombor, B., Albert, R., Oltvai, Z. and Barabasi, A.-L.</td>
	<td>The large-scale organization of metabolic networks <p class="infolinks">[<a href="javascript:toggleInfo('Jeong2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Jeong2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Nature<br/>Vol. 407(6804), pp. 651 - 654&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/35036627">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Jeong2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In a cell or microorganism, the processes that generate mass, energy, information transfer and cell-fate specification are seamlessly integrated through a complex network of cellular constituents and reactions1. However, despite the key role of these networks in sustaining cellular functions, their large-scale structure is essentially unknown. Here we present a systematic comparative mathematical analysis of the metabolic networks of 43 organisms representing all three domains of life. We show that, despite significant variation in their individual constituents and pathways, these metabolic networks have the same topological scaling properties and show striking similarities to the inherent organization of complex non-biological systems2. This may indicate that metabolic organization is not only identical for all living organisms, but also complies with the design principles of robust and error-tolerant scale-free networks2, 3, 4, 5, and may represent a common blueprint for the large-scale organization of interactions among all cellular constituents.</td>
</tr>
<tr id="bib_Jeong2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Jeong2000,
  author = {Jeong, H and Tombor, B and Albert, R and Oltvai, ZN and Barabasi, A.-L.},
  title = {The large-scale organization of metabolic networks},
  journal = {Nature},
  year = {2000},
  volume = {407},
  number = {6804},
  pages = {651 - 654},
  doi = {https://doi.org/10.1038/35036627}
}
</pre></td>
</tr>
<tr id="Johnson2011" class="entry">
	<td>Johnson, M.E. and Hummer, G.</td>
	<td>Nonspecific binding limits the number of proteins in a cell and shapes their interaction networks. <p class="infolinks">[<a href="javascript:toggleInfo('Johnson2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Johnson2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Proceedings of the National Academy of Sciences of the United States of America<br/>Vol. 108(2), pp. 603 - 608&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.1010954108">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Johnson2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Multicellular organisms, from Caenorhabditis elegans to humans, have roughly the same number of protein encoding genes. We show that the need to prevent disease-causing nonspecific interactions between proteins provides a simple physical reason why organism complexity is not reflected in the number of distinct proteins. By collective evolution of the amino acid sequences of protein binding interfaces we estimate the degree of misbinding as a function of the number of distinct proteins. Protein interaction energies are calculated with an empirical, residue-specific energy function tuned for protein binding. We show that the achievable energy gap favoring specific over nonspecific binding decreases with protein number in a power-law fashion. From the fraction of proteins involved in nonspecific complexes as a function of increasing protein number and decreasing energy gap, we predict the limits these binding requirements place on the number of different proteins that can function effectively in a given cellular compartment. Remarkably, the optimization of binding interfaces favors networks in which a few proteins have many partners, and most proteins have few partners, consistent with a scale-free network topology. We conclude that nonspecific binding adds to the evolutionary pressure to develop scale-free protein-protein interaction networks.</td>
</tr>
<tr id="bib_Johnson2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Johnson2011,
  author = {Johnson, Margaret E and Hummer, Gerhard},
  title = {Nonspecific binding limits the number of proteins in a cell and shapes their interaction networks.},
  journal = {Proceedings of the National Academy of Sciences of the United States of America},
  year = {2011},
  volume = {108},
  number = {2},
  pages = {603 - 608},
  doi = {https://doi.org/10.1073/pnas.1010954108}
}
</pre></td>
</tr>
<tr id="Johnson2013" class="entry">
	<td>Johnson, M.E. and Hummer, G.</td>
	<td>Interface-Resolved Network of Protein-Protein Interactions <p class="infolinks">[<a href="javascript:toggleInfo('Johnson2013','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Johnson2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>PLoS Computational Biology<br/>Vol. 9(5), pp. e1003065 - e1003065&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pcbi.1003065">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Johnson2013" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We define an interface-interaction network (IIN) to capture the specificity and competition between protein-protein interactions (PPI). This new type of network represents interactions between individual interfaces used in functional protein binding and thereby contains the detail necessary to describe the competition and cooperation between any pair of binding partners. Here we establish a general framework for the construction of IINs that merges computational structure-based interface assignment with careful curation of available literature. To complement limited structural data, the inclusion of biochemical data is critical for achieving the accuracy and completeness necessary to analyze the specificity and competition between the protein interactions. Firstly, this procedure provides a means to clarify the information content of existing data on purported protein interactions and to remove indirect and spurious interactions. Secondly, the IIN we have constructed here for proteins involved in clathrin-mediated endocytosis (CME) exhibits distinctive topological properties. In contrast to PPI networks with their global and relatively dense connectivity, the fragmentation of the IIN into distinctive network modules suggests that different functional pressures act on the evolution of its topology. Large modules in the IIN are formed by interfaces sharing specificity for certain domain types, such as SH3 domains distributed across different proteins. The shared and distinct specificity of an interface is necessary for effective negative and positive design of highly selective binding targets. Lastly, the organization of detailed structural data in a network format allows one to identify pathways of specific binding interactions and thereby predict effects of mutations at specific surfaces on a protein and of specific binding inhibitors, as we explore in several examples. Overall, the endocytosis IIN is remarkably complex and rich in features masked in the coarser PPI, and collects relevant detail of protein association in a readily interpretable format.</td>
</tr>
<tr id="bib_Johnson2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Johnson2013,
  author = {Johnson, Margaret E and Hummer, Gerhard},
  title = {Interface-Resolved Network of Protein-Protein Interactions},
  journal = {PLoS Computational Biology},
  year = {2013},
  volume = {9},
  number = {5},
  pages = {e1003065 - e1003065},
  doi = {https://doi.org/10.1371/journal.pcbi.1003065}
}
</pre></td>
</tr>
<tr id="Johnson2013a" class="entry">
	<td>Johnson, M.E. and Hummer, G.</td>
	<td>Construction of Transition Pathways in Molecular Reactions <p class="infolinks">[<a href="javascript:toggleInfo('Johnson2013a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Johnson2013a','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 116(29), pp. 8573 - 8583&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp212611k.Characterization">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Johnson2013a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We explore the theoretical foundation of different string methods used to find dominant reaction pathways in high-dimensional configuration spaces. Pathways are assessed by the amount of reactive flux they carry and by their orientation relative to the committor function. By examining the effects of transforming between different collective coordinates that span the same underlying space, we unmask artificial coordinate dependences in strings optimized to follow the free energy gradient. In contrast, strings optimized to follow the drift vector produce reaction pathways that are significantly less sensitive to reparameterizations of the collective coordinates. The differences in these paths arise because the drift vector depends on both the free energy gradient and the diffusion tensor of the coarse collective variables. Anisotropy and position dependence of diffusion tensors arise commonly in spaces of coarse variables, whose generally slow dynamics are obtained by nonlinear projections of the strongly coupled atomic motions. We show here that transition paths constructed to account for dynamics by following the drift vector will (to a close approximation) carry the maximum reactive flux both in systems with isotropic position dependent diffusion, and in systems with constant but anisotropic diffusion. We derive a simple method for calculating the committor function along paths that follow the reactive flux. Lastly, we provide guidance for the practical implementation of the dynamic string method.</td>
</tr>
<tr id="bib_Johnson2013a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Johnson2013a,
  author = {Johnson, Margaret E and Hummer, Gerhard},
  title = {Construction of Transition Pathways in Molecular Reactions},
  journal = {Journal of Physical Chemistry B},
  year = {2013},
  volume = {116},
  number = {29},
  pages = {8573 - 8583},
  doi = {https://doi.org/10.1021/jp212611k.Characterization}
}
</pre></td>
</tr>
<tr id="Johnson2013b" class="entry">
	<td>Johnson, M.E. and Hummer, G.</td>
	<td>Evolutionary pressure on the topology of protein interface interaction networks <p class="infolinks">[<a href="javascript:toggleInfo('Johnson2013b','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Johnson2013b','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 117(42), pp. 13098 - 13106&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp402944e">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Johnson2013b" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The densely connected structure of protein-protein interaction (PPI) networks reflects the functional need of proteins to cooperate in cellular processes. However, PPI networks do not adequately capture the competition in protein binding. By contrast, the interface interaction network (IIN) studied here resolves the modular character of protein-protein binding and distinguishes between simultaneous and exclusive interactions that underlie both cooperation and competition. We show that the topology of the IIN is under evolutionary pressure, and we connect topological features of the IIN to specific biological functions. To reveal the forces shaping the network topology, we use a sequence-based computational model of interface binding along with network analysis. We find that the more fragmented structure of IINs, in contrast to the dense PPI networks, arises in large part from the competition between specific and nonspecific binding. The need to minimize nonspecific binding favors specific network motifs, including a minimal number of cliques (i.e., fully connected subgraphs) and many disconnected fragments. Validating the model, we find that these network characteristics are closely mirrored in the IIN of clathrin-mediated endocytosis. Features unexpected on the basis of our motif analysis are found to indicate either exceptional binding selectivity or important regulatory functions.</td>
</tr>
<tr id="bib_Johnson2013b" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Johnson2013b,
  author = {Johnson, Margaret E and Hummer, Gerhard},
  title = {Evolutionary pressure on the topology of protein interface interaction networks},
  journal = {Journal of Physical Chemistry B},
  year = {2013},
  volume = {117},
  number = {42},
  pages = {13098 - 13106},
  doi = {https://doi.org/10.1021/jp402944e}
}
</pre></td>
</tr>
<tr id="Johnson2014" class="entry">
	<td>Johnson, M.E. and Hummer, G.</td>
	<td>Free-Propagator Reweighting Integrator for Single-Particle Dynamics in Reaction-Diffusion Models of Heterogeneous Protein-Protein Interaction Systems <p class="infolinks">[<a href="javascript:toggleInfo('Johnson2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Johnson2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Physical Reviews X<br/>Vol. 4(3), pp. 031037&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1103/PhysRevX.4.031037.Free-Propagator">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Johnson2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present a new algorithm for simulating reaction-diffusion equations at single-particle resolution. Our algorithm is designed to be both accurate and simple to implement, and to be applicable to large and heterogeneous systems, including those arising in systems biology applications. We combine the use of the exact Green's function for a pair of reacting particles with the approximate free-diffusion propagator for position updates to particles. Trajectory reweighting in our free-propagator reweighting (FPR) method recovers the exact association rates for a pair of interacting particles at all times. FPR simulations of many-body systems accurately reproduce the theoretically known dynamic behavior for a variety of different reaction types. FPR does not suffer from the loss of efficiency common to other path-reweighting schemes, first, because corrections apply only in the immediate vicinity of reacting particles and, second, because by construction the average weight factor equals one upon leaving this reaction zone. FPR applications include the modeling of pathways and networks of protein-driven processes where reaction rates can vary widely and thousands of proteins may participate in the formation of large assemblies. With a limited amount of bookkeeping necessary to ensure proper association rates for each reactant pair, FPR can account for changes to reaction rates or diffusion constants as a result of reaction events. Importantly, FPR can also be extended to physical descriptions of protein interactions with long-range forces, as we demonstrate here for Coulombic interactions.</td>
</tr>
<tr id="bib_Johnson2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Johnson2014,
  author = {Johnson, Margaret E and Hummer, Gerhard},
  title = {Free-Propagator Reweighting Integrator for Single-Particle Dynamics in Reaction-Diffusion Models of Heterogeneous Protein-Protein Interaction Systems},
  journal = {Physical Reviews X},
  year = {2014},
  volume = {4},
  number = {3},
  pages = {031037},
  doi = {https://doi.org/10.1103/PhysRevX.4.031037.Free-Propagator}
}
</pre></td>
</tr>
<tr id="Johnston1961" class="entry">
	<td>Johnston, H.S.</td>
	<td>Large Tunnelling Corrections in Chemical Reaction Rates <p class="infolinks">[<a href="javascript:toggleInfo('Johnston1961','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Johnston1961','bibtex')">BibTeX</a>]</p></td>
	<td>1961</td>
	<td><br/>Vol. 83(1)Advances in Chemical Physics, pp. 131&nbsp;</td>
	<td>inbook</td>
	<td><a href="https://doi.org/10.1002/9780470143490.ch4">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Johnston1961" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Look for ebook for details</td>
</tr>
<tr id="bib_Johnston1961" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inbook{Johnston1961,
  author = {Johnston, Harold S},
  title = {Large Tunnelling Corrections in Chemical Reaction Rates},
  booktitle = {Advances in Chemical Physics},
  publisher = {American Chemical Society},
  year = {1961},
  volume = {83},
  number = {1},
  pages = {131},
  doi = {https://doi.org/10.1002/9780470143490.ch4}
}
</pre></td>
</tr>
<tr id="Jorgensen1983" class="entry">
	<td>Jorgensen, W.L., Chandrasekhar, J., Madura, J.D., Impey, R.W. and Klein, M.L.</td>
	<td>Comparison of simple potential functions for simulating liquid water <p class="infolinks">[<a href="javascript:toggleInfo('Jorgensen1983','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Jorgensen1983','bibtex')">BibTeX</a>]</p></td>
	<td>1983</td>
	<td>The Journal of Chemical Physics<br/>Vol. 79(2), pp. 926 - 926&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.445869">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Jorgensen1983" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Classical Monte Carlo simulations have been carried out for liquid water in the NPT ensemble at 25 textdegreeC and 1 atm using six of the simpler intermolecular potential functions for the water dimer: Bernal--Fowler (BF), SPC, ST2, TIPS2, TIP3P, and TIP4P. Comparisons are made with experimental thermodynamic and structural data including the recent neutron diffraction results of Thiessen and Narten. The computed densities and potential energies are in reasonable accord with experiment except for the original BF model, which yields an 18&#37; overestimate of the density and poor structural results. The TIPS2 and TIP4P potentials yield oxygen--oxygen partial structure functions in good agreement with the neutron diffraction results. The accord with the experimental OH and HH partial structure functions is poorer; however, the computed results for these functions are similar for all the potential functions. Consequently, the discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations. Comparisons are also made for self‐diffusion coefficients obtained from molecular dynamics simulations. Overall, the SPC, ST2, TIPS2, and TIP4P models give reasonable structural and thermodynamic descriptions of liquid water and they should be useful in simulations of aqueous solutions. The simplicity of the SPC, TIPS2, and TIP4P functions is also attractive from a computational standpoint.</td>
</tr>
<tr id="bib_Jorgensen1983" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Jorgensen1983,
  author = {Jorgensen, William L and Chandrasekhar, Jayaraman and Madura, Jeffry D and Impey, Roger W and Klein, Michael L},
  title = {Comparison of simple potential functions for simulating liquid water},
  journal = {The Journal of Chemical Physics},
  year = {1983},
  volume = {79},
  number = {2},
  pages = {926 - 926},
  note = {NULL},
  doi = {https://doi.org/10.1063/1.445869}
}
</pre></td>
</tr>
<tr id="Ju2011" class="entry">
	<td>Ju, J., Xu, S., Furukawa, Y., Zhang, Y., Misono, H., Minamino, T., Namba, K., Zhao, B. and Ohnishi, K.</td>
	<td>Correlation between catalytic activity and monomer-dimer equilibrium of bacterial alanine racemases. <p class="infolinks">[<a href="javascript:toggleInfo('Ju2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ju2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Journal of biochemistry<br/>Vol. 149(1), pp. 83 - 9&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/jb/mvq120">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ju2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: From the reaction mechanism and crystal structure analysis, a bacterial alanine racemase is believed to work as a homodimer with a substrate, l-alanine or d-alanine. We analysed oligomerization states of seven alanine racemases, biosynthetic and catabolic, from Escherichia coli, Salmonella typhimurium, Pseudomonas aeruginosa, P. putida and P. fluorescens, with three different methods, gel filtration chromatography, native PAGE and analytical ultracentrifugation. All alanine racemases were proved to be in a dynamic equilibrium between monomeric and dimeric form with every methods used in this study. In both biosynthetic and catabolic alanine racemases, association constants for dimerization were high for the enzymes with high V(max) values. The enzymes with low V(max) values gave the low association constants. We proposed that alanine racemases are classified into two types; the enzymes with low and high-equilibrium association constants for dimerization.</td>
</tr>
<tr id="bib_Ju2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ju2011,
  author = {Ju, Jiansong and Xu, Shujing and Furukawa, Yukio and Zhang, Yong and Misono, Haruo and Minamino, Tohru and Namba, Keiichi and Zhao, Baohua and Ohnishi, Kouhei},
  title = {Correlation between catalytic activity and monomer-dimer equilibrium of bacterial alanine racemases.},
  journal = {Journal of biochemistry},
  year = {2011},
  volume = {149},
  number = {1},
  pages = {83 - 9},
  note = {&amp;quot;n both biosynthetic and catabolic alanine racemases, association constants for dimerization were high for the enzymes with high Vmax values. The enzymes with low Vmax values gave the low association constants. &amp;quot;},
  doi = {https://doi.org/10.1093/jb/mvq120}
}
</pre></td>
</tr>
<tr id="Jung2017" class="entry">
	<td>Jung, H., Okazaki, K.-i. and Hummer, G.</td>
	<td>Transition path sampling of rare events by shooting from the top <p class="infolinks">[<a href="javascript:toggleInfo('Jung2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of Chemical Physics<br/>Vol. 147(15), pp. 152716&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.4997378">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Jung2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Jung2017,
  author = {Jung, Hendrik and Okazaki, Kei-ichi and Hummer, Gerhard},
  title = {Transition path sampling of rare events by shooting from the top},
  journal = {Journal of Chemical Physics},
  year = {2017},
  volume = {147},
  number = {15},
  pages = {152716},
  doi = {https://doi.org/10.1063/1.4997378}
}
</pre></td>
</tr>
<tr id="Kabsch1983" class="entry">
	<td>Kabsch, W. and Sander, C.</td>
	<td>Dictionary of protein secondary structure: pattern recognition of hydrogen-bonded and geometrical features. <p class="infolinks">[<a href="javascript:toggleInfo('Kabsch1983','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kabsch1983','bibtex')">BibTeX</a>]</p></td>
	<td>1983</td>
	<td>Biopolymers<br/>Vol. 22(12), pp. 2577 - 637&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/bip.360221211">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kabsch1983" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: For a successful analysis of the relation between amino acid sequence and protein structure, an unambiguous and physically meaningful definition of secondary structure is essential. We have developed a set of simple and physically motivated criteria for secondary structure, programmed as a pattern-recognition process of hydrogen-bonded and geometrical features extracted from x-ray coordinates. Cooperative secondary structure is recognized as repeats of the elementary hydrogen-bonding patterns ``turn'' and ``bridge.'' Repeating turns are ``helices,'' repeating bridges are ``ladders,'' connected ladders are ``sheets.'' Geometric structure is defined in terms of the concepts torsion and curvature of differential geometry. Local chain ``chirality'' is the torsional handedness of four consecutive Cα positions and is positive for right-handed helices and negative for ideal twisted β-sheets. Curved pieces are defined as ``bends.'' Solvent ``exposure'' is given as the number of water molecules in possible contact with a residue. The end result is a compilation of the primary structure, including SS bonds, secondary structure, and solvent exposure of 62 different globular proteins. The presentation is in linear form: strip graphs for an overall view and strip tables for the details of each of 10.925 residues. The dictionary is also available in computer-readable form for protein structure prediction work.</td>
</tr>
<tr id="bib_Kabsch1983" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kabsch1983,
  author = {Kabsch, W and Sander, C},
  title = {Dictionary of protein secondary structure: pattern recognition of hydrogen-bonded and geometrical features.},
  journal = {Biopolymers},
  year = {1983},
  volume = {22},
  number = {12},
  pages = {2577 - 637},
  note = {Original DSSP algorithm paper},
  doi = {https://doi.org/10.1002/bip.360221211}
}
</pre></td>
</tr>
<tr id="Kadlecova2017" class="entry">
	<td>Kadlecova, Z., Spielman, S.J., Loerke, D., Mohanakrishnan, A., Reed, D.K. and Schmid, S.L.</td>
	<td>Regulation of clathrin-mediated endocytosis by hierarchical allosteric activation of AP2. <p class="infolinks">[<a href="javascript:toggleInfo('Kadlecova2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kadlecova2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>The Journal of cell biology<br/>Vol. 216, pp. 167-179&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1083/jcb.201608071">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kadlecova2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The critical initiation phase of clathrin-mediated endocytosis (CME) determines where and when endocytosis occurs. Heterotetrameric adaptor protein 2 (AP2) complexes, which initiate clathrin-coated pit (CCP) assembly, are activated by conformational changes in response to phosphatidylinositol-4,5-bisphosphate (PIP2) and cargo binding at multiple sites. However, the functional hierarchy of interactions and how these conformational changes relate to distinct steps in CCP formation in living cells remains unknown. We used quantitative live-cell analyses to measure discrete early stages of CME and show how sequential, allosterically regulated conformational changes activate AP2 to drive both nucleation and subsequent stabilization of nascent CCPs. Our data establish that cargoes containing Yxxφ motif, but not dileucine motif, play a critical role in the earliest stages of AP2 activation and CCP nucleation. Interestingly, these cargo and PIP2 interactions are not conserved in yeast. Thus, we speculate that AP2 has evolved as a key regulatory node to coordinate CCP formation and cargo sorting and ensure high spatial and temporal regulation of CME.</td>
</tr>
<tr id="bib_Kadlecova2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kadlecova2017,
  author = {Kadlecova, Zuzana and Spielman, Stephanie J and Loerke, Dinah and Mohanakrishnan, Aparna and Reed, Dana Kim and Schmid, Sandra L},
  title = {Regulation of clathrin-mediated endocytosis by hierarchical allosteric activation of AP2.},
  journal = {The Journal of cell biology},
  year = {2017},
  volume = {216},
  pages = {167--179},
  doi = {https://doi.org/10.1083/jcb.201608071}
}
</pre></td>
</tr>
<tr id="Kalthoff2002" class="entry">
	<td>Kalthoff, C., Alves, J., Urbanke, C., Knorr, R. and Ungewickell, E.J.</td>
	<td>Unusual structural organization of the endocytic proteins AP180 and epsin 1. <p class="infolinks">[<a href="javascript:toggleInfo('Kalthoff2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kalthoff2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>The Journal of biological chemistry<br/>Vol. 277, pp. 8209-8216&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M111587200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kalthoff2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Epsin and AP180/CALM are important endocytic accessory proteins that are believed to be involved in the formation of clathrin coats. Both proteins associate with phosphorylated membrane inositol lipids through their epsin N-terminal homology domains and with other components of the endocytic machinery through short peptide motifs in their carboxyl-terminal segments. Using hydrodynamic and spectroscopic methods, we demonstrate that the parts of epsin 1 and AP180 that are involved in protein-protein interactions behave as poorly structured flexible polypeptide chains with little or no conventional secondary structure. The predominant cytosolic forms of both proteins are monomers. Furthermore, we show that recombinant epsin 1, like AP180, drives in vitro assembly of clathrin cages. We conclude that the epsin N-terminal homology domain-containing proteins AP180/CALM and epsin 1 have a very similar molecular architecture that is designed for the rapid and efficient recruitment of the principal coat components clathrin and AP-2 at the sites of coated pit assembly.</td>
</tr>
<tr id="bib_Kalthoff2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kalthoff2002,
  author = {Kalthoff, Christoph and Alves, Jürgen and Urbanke, Claus and Knorr, Ruth and Ungewickell, Ernst J},
  title = {Unusual structural organization of the endocytic proteins AP180 and epsin 1.},
  journal = {The Journal of biological chemistry},
  year = {2002},
  volume = {277},
  pages = {8209--8216},
  doi = {https://doi.org/10.1074/jbc.M111587200}
}
</pre></td>
</tr>
<tr id="Kamerlin2010" class="entry">
	<td>Kamerlin, S., Mavri, J. and Warshel, a.</td>
	<td>Examining the case for the effect of barrier compression on tunneling, vibrationally enhanced catalysis, catalytic entropy and related issues <p class="infolinks">[<a href="javascript:toggleInfo('Kamerlin2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kamerlin2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>FEBS Letters<br/>Vol. 584(13), pp. 2759 - 2766&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.febslet.2010.04.062">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kamerlin2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The idea that tunneling is enhanced by the compression of the donor-acceptor distance has attracted significant interest. In particular, recent studies argued that this proposal is consistent with pressure effects on enzymatic reactions, and that the observed pressure effects support the idea of vibrationally enhanced catalysis. However, a careful analysis of the current works reveals serious inconsistencies in the evidence presented to support these hypotheses. Apparently, tunneling decreases upon compression, and external pressure does not lead to the applicable compression of the free energy surface. Additionally, pressure experiments do not provide actual evidence for vibrationally enhanced catalysis. Finally, the temperature dependence of the entropy change in hydride transfer reactions is shown to reflect simple electrostatic effects. ?? 2010 Federation of European Biochemical Societies.</td>
</tr>
<tr id="bib_Kamerlin2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kamerlin2010,
  author = {Kamerlin, Shina and Mavri, Janez and Warshel, a.},
  title = {Examining the case for the effect of barrier compression on tunneling, vibrationally enhanced catalysis, catalytic entropy and related issues},
  journal = {FEBS Letters},
  year = {2010},
  volume = {584},
  number = {13},
  pages = {2759 - 2766},
  note = {From Duplicate 1 (Examining the case for the effect of barrier compression on tunneling, vibrationally enhanced catalysis, catalytic entropy and related issues. - Kamerlin, Shina Caroline Lynn; Mavri, Janez; Warshel, A) Barrier broadening as donor-acceptor compress},
  doi = {https://doi.org/10.1016/j.febslet.2010.04.062}
}
</pre></td>
</tr>
<tr id="Karamzadeh2017" class="entry">
	<td>Karamzadeh, R., Karimi-Jafari, M., Sharifi-Zarchi, A., Chitsaz, H., Salekdeh, G. and Moosavi-Movahedi, A.</td>
	<td>Machine Learning and Network Analysis of Molecular Dynamics Trajectories Reveal Two Chains of Red/Ox-specific Residue Interactions in Human&nbsp;Protein Disulfide Isomerase <p class="infolinks">[<a href="javascript:toggleInfo('Karamzadeh2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Karamzadeh2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Scientific Reports(1), pp. 3666&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/s41598-017-03966-5">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Karamzadeh2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The human protein disulfide isomerase (hPDI), is an essential four-domain multifunctional enzyme. As a result of disulfide shuffling in its terminal domains, hPDI exists in two oxidation states with different conformational preferences which are important for substrate binding and functional activities. Here, we address the redox-dependent conformational dynamics of hPDI through molecular dynamics (MD) simulations. Collective domain motions are identified by the principal component analysis of MD trajectories and redox-dependent opening-closing structure variations are highlighted on projected free energy landscapes. Then, important structural features that exhibit considerable differences in dynamics of redox states are extracted by statistical machine learning methods. Mapping the structural variations to time series of residue interaction networks also provides a holistic representation of the dynamical redox differences. With emphasizing on persistent long-lasting interactions, an approach is proposed that compiled these time series networks to a single dynamic residue interaction network (DRIN). Differential comparison of DRIN in oxidized and reduced states reveals chains of residue interactions that represent potential allosteric paths between catalytic and ligand binding sites of hPDI.</td>
</tr>
<tr id="bib_Karamzadeh2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Karamzadeh2017,
  author = {Karamzadeh, Razieh and Karimi-Jafari, Mohammad and Sharifi-Zarchi, Ali and Chitsaz, Hamidreza and Salekdeh, Ghasem and Moosavi-Movahedi, Ali},
  title = {Machine Learning and Network Analysis of Molecular Dynamics Trajectories Reveal Two Chains of Red/Ox-specific Residue Interactions in Human&nbsp;Protein Disulfide Isomerase},
  journal = {Scientific Reports},
  year = {2017},
  number = {1},
  pages = {3666},
  doi = {https://doi.org/10.1038/s41598-017-03966-5}
}
</pre></td>
</tr>
<tr id="Kariya2000" class="entry">
	<td>Kariya, K., Koyama, S., Nakashima, S., Oshiro, T., Morinaka, K. and Kikuchi, A.</td>
	<td>Regulation of complex formation of POB1/Epsin/adaptor protein complex 2 by mitotic phosphorylation <p class="infolinks">[<a href="javascript:toggleInfo('Kariya2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kariya2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Journal of Biological Chemistry<br/>Vol. 275(24), pp. 18399 - 18406&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M000521200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kariya2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: RalBP1 and POB1, the downstream molecules of small GTP-binding protein Ral, are involved in receptor-mediated endocytosis together with Epsin and Eps15. The regulation of assembly of the complex of these proteins was examined. RalBP1, POB1, Epsin, and Eps15 formed a complex with alpha-adaptin of AP-2 in Chinese hamster ovary cells, but the formation was reduced in mitotic phase. RalBP1, POB1, Epsin, and Eps15 were all phosphorylated in mitotic phase. The phosphorylated forms of POB1 and Epsin were recognized by the antibody MPM2, which is known to detect mitotic phosphoproteins. POB1 and Epsin were phosphorylated by p34(cdc2) kinase in vitro. Their phosphorylation sites (Ser(411) of POB1 and Ser(357) of Epsin) were determined. Phosphorylated Epsin and Epsin(S357D) formed a complex with alpha-adaptin less efficiently than wild type Epsin. Although the EH domain of POB1 bound directly to Epsin, phosphorylation of Epsin inhibited the binding. Furthermore, Epsin(S357D) but not Epsin(S357A) lost the effect of Epsin on the insulin-dependent endocytosis. These results suggest that phosphorylation of Epsin in mitotic phase inhibits receptor-mediated endocytosis by disassembly of its complex with POB1 and alpha-adaptin.</td>
</tr>
<tr id="bib_Kariya2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kariya2000,
  author = {Kariya, Kenji and Koyama, Shinya and Nakashima, Shintaro and Oshiro, Takafumi and Morinaka, Kenji and Kikuchi, Akira},
  title = {Regulation of complex formation of POB1/Epsin/adaptor protein complex 2 by mitotic phosphorylation},
  journal = {Journal of Biological Chemistry},
  year = {2000},
  volume = {275},
  number = {24},
  pages = {18399 - 18406},
  note = {Interactions of EPS15, EPN1, and POB1 to form a complex with AP2. Some notes on phosphorylation.},
  doi = {https://doi.org/10.1074/jbc.M000521200}
}
</pre></td>
</tr>
<tr id="Karlsson2017" class="entry">
	<td>Karlsson, B.C. and Friedman, R.</td>
	<td>Dilution of whisky -- the molecular perspective <p class="infolinks">[<a href="javascript:toggleInfo('Karlsson2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Karlsson2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Scientific Reports&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/s41598-017-06423-5">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Karlsson2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Whisky is distilled to around 70&#37; alcohol by volume (vol- then diluted to about 40 vol- and often drunk after further slight dilution to enhance its taste. The taste of whisky is primarily associated with amphipathic molecules, such as guaiacol, but why and how dilution enhances the taste is not well understood. We carried out computer simulations of water-ethanol mixtures in the presence of guaiacol, providing atomistic details on the structure of the liquid mixture. We found that guaiacol is preferentially associated with ethanol, and, therefore, primarily found at the liquid-air interface in mixtures that contain up to 45 vol-&#37; of ethanol. At ethanol concentrations of 59 vol-&#37; or higher, guaiacol is increasingly surrounded by ethanol molecules and is driven to the bulk. This indicates that the taste of guaiacol in the whisky would be enhanced upon dilution prior to bottling. Our findings may apply to other flavour-giving amphipathic molecules and could contribute to optimising the production of spirits for desired tastes. Furthermore, it sheds light on the molecular structure of water-alcohol mixtures that contain small solutes, and reveals that interactions with the water may be negligible already at 89 vol-&#37; of ethanol. Whisky is a spirit that is produced in an extended process consisting of distillation of fermented grains, ageing and dilution. It is through this process that the distinctive taste of whisky develops. Distilled malt whiskies typ-ically contain around 70&#37; alcohol by volume (vol- before it is aged in barrels for at least three years. Some alcohol evaporates during the maturation resulting in an alcohol content of 55--65 vol-&#37; of cask-strength whisky. Before bottling, the whisky is diluted to around 40 vol-&#37; by the addition of water 1</td>
</tr>
<tr id="bib_Karlsson2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Karlsson2017,
  author = {Karlsson, Bjorn CG and Friedman, Ran},
  title = {Dilution of whisky -- the molecular perspective},
  journal = {Scientific Reports},
  year = {2017},
  doi = {https://doi.org/10.1038/s41598-017-06423-5}
}
</pre></td>
</tr>
<tr id="Karney2007" class="entry">
	<td>Karney, C.F.</td>
	<td>Quaternions in molecular modeling <p class="infolinks">[<a href="javascript:toggleInfo('Karney2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Journal of Molecular Graphics and Modelling<br/>Vol. 25(5), pp. 595-604&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Karney2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Karney2007,
  author = {Karney, Charles FF},
  title = {Quaternions in molecular modeling},
  journal = {Journal of Molecular Graphics and Modelling},
  publisher = {Elsevier},
  year = {2007},
  volume = {25},
  number = {5},
  pages = {595--604}
}
</pre></td>
</tr>
<tr id="Karney2005" class="entry">
	<td>Karney, C.F., Ferrara, J.E. and Brunner, S.</td>
	<td>Method for computing protein binding affinity <p class="infolinks">[<a href="javascript:toggleInfo('Karney2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Journal of computational chemistry<br/>Vol. 26(3), pp. 243-251&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Karney2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Karney2005,
  author = {Karney, Charles FF and Ferrara, Jason E and Brunner, Stephan},
  title = {Method for computing protein binding affinity},
  journal = {Journal of computational chemistry},
  publisher = {Wiley Online Library},
  year = {2005},
  volume = {26},
  number = {3},
  pages = {243--251}
}
</pre></td>
</tr>
<tr id="Kavanagh2004" class="entry">
	<td>Kavanagh, K.L., Elling, R.A. and Wilson, D.K.</td>
	<td>Structure of Toxoplasma gondii LDH1: Active-Site Differences from Human Lactate Dehydrogenases and the Structural Basis for Efficient APAD+ Use <p class="infolinks">[<a href="javascript:toggleInfo('Kavanagh2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kavanagh2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Biochemistry-us<br/>Vol. 43(4), pp. 879-889&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/BI035108G">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kavanagh2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: While within a human host the opportunistic pathogen Toxoplasma gondii relies heavily on glycolysis for its energy needs. Lactate dehydrogenase (LDH), the terminal enzyme in anaerobic glycolysis necessary for NAD+ regeneration, therefore represents an attractive therapeutic target. The tachyzoite stage lactate dehydrogenase (LDH1) from the parasite T. gondii has been crystallized in apo form and in ternary complexes containing NAD+ or the NAD+-analogue 3-acetylpyridine adenine dinucleotide (APAD+) and sulfate or the inhibitor oxalate. Comparison of the apo and ternary models shows an active-site loop that becomes ordered upon substrate binding. This active-site loop is five residues longer than in most LDHs and necessarily adopts a different conformation. While loop isomerization is fully rate-limiting in prototypical LDHs, kinetic data suggest that LDH1's rate is limited by chemical steps. The importance of charge neutralization in ligand binding is supported by the complexes that have been crystallized ...</td>
</tr>
<tr id="bib_Kavanagh2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kavanagh2004,
  author = {Kavanagh, Kathryn L and and Elling, Robert A and Wilson, David K},
  title = {Structure of Toxoplasma gondii LDH1: Active-Site Differences from Human Lactate Dehydrogenases and the Structural Basis for Efficient APAD+ Use},
  journal = {Biochemistry-us},
  year = {2004},
  volume = {43},
  number = {4},
  pages = {879-889},
  note = {NULL},
  doi = {https://doi.org/10.1021/BI035108G}
}
</pre></td>
</tr>
<tr id="Kay1999" class="entry">
	<td>Kay, B.K., Yamabhai, M., Wendland, B. and Emr, S.D.</td>
	<td>Identification of a novel domain shared by putative components of the endocytic and cytoskeletal machinery. <p class="infolinks">[<a href="javascript:toggleInfo('Kay1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kay1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Protein science : a publication of the Protein Society<br/>Vol. 8, pp. 435-438&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1110/ps.8.2.435">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kay1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We have identified a approximately 140 amino acid domain that is shared by a variety of proteins in budding and fission yeast, nematode, rat, mouse, frog, oat, and man. Typically, this domain is located within 20 residues of the N-terminus of the various proteins. The percent identity among the domains in the 12 proteins ranges from 42 to 93%, with 16 absolutely conserved residues: N-x(11-13)-V-x2-A-T-x(34-36)-R-x(7-8)-W-R-x3-K-x12-G-x-E-x15 -L-x11-12-D-x-G-R-x11-D-x7-R. Even though these proteins share little beyond their segment of homology, data are emerging that several of the proteins are involved in endocytosis and or regulation of cytoskeletal organization. We have named this protein segment the ENTH domain, for Epsin N-terminal Homology domain, and hypothesize that it is a candidate for binding specific ligands and/or enzymatic activity in the cell.</td>
</tr>
<tr id="bib_Kay1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kay1999,
  author = {Kay, B K and Yamabhai, M and Wendland, B and Emr, S D},
  title = {Identification of a novel domain shared by putative components of the endocytic and cytoskeletal machinery.},
  journal = {Protein science : a publication of the Protein Society},
  year = {1999},
  volume = {8},
  pages = {435--438},
  doi = {https://doi.org/10.1110/ps.8.2.435}
}
</pre></td>
</tr>
<tr id="Keller2010" class="entry">
	<td>Keller, B., Daura, X. and van Gunsteren, W.F.</td>
	<td>Comparing geometric and kinetic cluster algorithms for molecular simulation data. <p class="infolinks">[<a href="javascript:toggleInfo('Keller2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Keller2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>The Journal of chemical physics<br/>Vol. 132(7), pp. 074110 - 074110&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.3301140">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Keller2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The identification of metastable states of a molecule plays an important role in the interpretation of molecular simulation data because the free-energy surface, the relative populations in this landscape, and ultimately also the dynamics of the molecule under study can be described in terms of these states. We compare the results of three different geometric cluster algorithms (neighbor algorithm, K-medoids algorithm, and common-nearest-neighbor algorithm) among each other and to the results of a kinetic cluster algorithm. First, we demonstrate the characteristics of each of the geometric cluster algorithms using five two-dimensional data sets. Second, we analyze the molecular dynamics data of a beta-heptapeptide in methanol--a molecule that exhibits a distinct folded state, a structurally diverse unfolded state, and a fast folding/unfolding equilibrium--using both geometric and kinetic cluster algorithms. We find that geometric clustering strongly depends on the algorithm used and that the density based common-nearest-neighbor algorithm is the most robust of the three geometric cluster algorithms with respect to variations in the input parameters and the distance metric. When comparing the geometric cluster results to the metastable states of the beta-heptapeptide as identified by kinetic clustering, we find that in most cases the folded state is identified correctly but the overlap of geometric clusters with further metastable states is often at best approximate.</td>
</tr>
<tr id="bib_Keller2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Keller2010,
  author = {Keller, Bettina and Daura, Xavier and van Gunsteren, Wilfred F},
  title = {Comparing geometric and kinetic cluster algorithms for molecular simulation data.},
  journal = {The Journal of chemical physics},
  year = {2010},
  volume = {132},
  number = {7},
  pages = {074110 - 074110},
  note = {Kinetic clustering requires that transition probabilities between all conformation pairs be converged.},
  doi = {https://doi.org/10.1063/1.3301140}
}
</pre></td>
</tr>
<tr id="Kelly2014" class="entry">
	<td>Kelly, B.T., Graham, S.C., Liska, N., Dannhauser, P.N., Höning, S., Ungewickell, E.J. and Owen, D.J.</td>
	<td>Clathrin adaptors. AP2 controls clathrin polymerization with a membrane-activated switch. <p class="infolinks">[<a href="javascript:toggleInfo('Kelly2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kelly2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Science<br/>Vol. 345, pp. 459-463&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1126/science.1254836">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kelly2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis (CME) is vital for the internalization of most cell-surface proteins. In CME, plasma membrane-binding clathrin adaptors recruit and polymerize clathrin to form clathrin-coated pits into which cargo is sorted. Assembly polypeptide 2 (AP2) is the most abundant adaptor and is pivotal to CME. Here, we determined a structure of AP2 that includes the clathrin-binding β2 hinge and developed an AP2-dependent budding assay. Our findings suggest that an autoinhibitory mechanism prevents clathrin recruitment by cytosolic AP2. A large-scale conformational change driven by the plasma membrane phosphoinositide phosphatidylinositol 4,5-bisphosphate and cargo relieves this autoinhibition, triggering clathrin recruitment and hence clathrin-coated bud formation. This molecular switching mechanism can couple AP2's membrane recruitment to its key functions of cargo and clathrin binding.</td>
</tr>
<tr id="bib_Kelly2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kelly2014,
  author = {Kelly, Bernard T and Graham, Stephen C and Liska, Nicole and Dannhauser, Philip N and Höning, Stefan and Ungewickell, Ernst J and Owen, David J},
  title = {Clathrin adaptors. AP2 controls clathrin polymerization with a membrane-activated switch.},
  journal = {Science},
  year = {2014},
  volume = {345},
  pages = {459--463},
  doi = {https://doi.org/10.1126/science.1254836}
}
</pre></td>
</tr>
<tr id="Kelly2008" class="entry">
	<td>Kelly, B.T., McCoy, A.J., Späte, K., Miller, S.E., Evans, P.R., Höning, S. and Owen, D.J.</td>
	<td>A structural explanation for the binding of endocytic dileucine motifs by the AP2 complex. <p class="infolinks">[<a href="javascript:toggleInfo('Kelly2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Nature<br/>Vol. 456, pp. 976-979&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nature07422">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Kelly2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kelly2008,
  author = {Kelly, Bernard T and McCoy, Airlie J and Späte, Kira and Miller, Sharon E and Evans, Philip R and Höning, Stefan and Owen, David J},
  title = {A structural explanation for the binding of endocytic dileucine motifs by the AP2 complex.},
  journal = {Nature},
  year = {2008},
  volume = {456},
  pages = {976--979},
  note = {Ap2 open, closed},
  doi = {https://doi.org/10.1038/nature07422}
}
</pre></td>
</tr>
<tr id="Keshavamurthy2018" class="entry">
	<td>Keshavamurthy, S. and Patra, S.</td>
	<td>Detecting reactive islands using Lagrangian descriptors and the relevance to transition path sampling <p class="infolinks">[<a href="javascript:toggleInfo('Keshavamurthy2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Physical Chemistry Chemical Physics&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Keshavamurthy2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Keshavamurthy2018,
  author = {Keshavamurthy, Srihari and Patra, Sarbani},
  title = {Detecting reactive islands using Lagrangian descriptors and the relevance to transition path sampling},
  journal = {Physical Chemistry Chemical Physics},
  publisher = {Royal Society of Chemistry},
  year = {2018}
}
</pre></td>
</tr>
<tr id="Keyel2008" class="entry">
	<td>Keyel, P.A., Thieman, J.R., Roth, R., Erkan, E., Everett, E.T., Watkins, S.C., Heuser, J.E. and Traub, L.M.</td>
	<td>The AP-2 adaptor beta2 appendage scaffolds alternate cargo endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Keyel2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Keyel2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Molecular biology of the cell<br/>Vol. 19, pp. 5309-5326&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1091/mbc.E08-07-0712">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Keyel2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The independently folded appendages of the large alpha and beta2 subunits of the endocytic adaptor protein (AP)-2 complex coordinate proper assembly and operation of endocytic components during clathrin-mediated endocytosis. The beta2 subunit appendage contains a common binding site for beta-arrestin or the autosomal recessive hypercholesterolemia (ARH) protein. To determine the importance of this interaction surface in living cells, we used small interfering RNA-based gene silencing. The effect of extinguishing beta2 subunit expression on the internalization of transferrin is considerably weaker than an AP-2 alpha subunit knockdown. We show the mild sorting defect is due to fortuitous substitution of the beta2 chain with the closely related endogenous beta1 subunit of the AP-1 adaptor complex. Simultaneous silencing of both beta1 and beta2 subunit transcripts recapitulates the strong alpha subunit RNA interference (RNAi) phenotype and results in loss of ARH from endocytic clathrin coats. An RNAi-insensitive beta2-yellow fluorescent protein (YFP) expressed in the beta1 + beta2-silenced background restores cellular AP-2 levels, robust transferrin internalization, and ARH colocalization with cell surface clathrin. The importance of the beta appendage platform subdomain over clathrin for precise deposition of ARH at clathrin assembly zones is revealed by a beta2-YFP with a disrupted ARH binding interface, which does not restore ARH colocalization with clathrin. We also show a beta-arrestin 1 mutant, which engages coated structures in the absence of any G protein-coupled receptor stimulation, colocalizes with beta2-YFP and clathrin even in the absence of an operational clathrin binding sequence. These findings argue against ARH and beta-arrestin binding to a site upon the beta2 appendage platform that is later obstructed by polymerized clathrin. We conclude that ARH and beta-arrestin depend on a privileged beta2 appendage site for proper cargo recruitment to clathrin bud sites.</td>
</tr>
<tr id="bib_Keyel2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Keyel2008,
  author = {Keyel, Peter A and Thieman, James R and Roth, Robyn and Erkan, Elif and Everett, Eric T and Watkins, Simon C and Heuser, John E and Traub, Linton M},
  title = {The AP-2 adaptor beta2 appendage scaffolds alternate cargo endocytosis.},
  journal = {Molecular biology of the cell},
  year = {2008},
  volume = {19},
  pages = {5309--5326},
  doi = {https://doi.org/10.1091/mbc.E08-07-0712}
}
</pre></td>
</tr>
<tr id="Kiel2013" class="entry">
	<td>Kiel, C., Verschueren, E., Yang, J.-S. and Serrano, L.</td>
	<td>Integration of protein abundance and structure data reveals competition in the ErbB signaling network. <p class="infolinks">[<a href="javascript:toggleInfo('Kiel2013','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kiel2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Science signaling<br/>Vol. 6(306), pp. ra109 - ra109&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1126/scisignal.2004560">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kiel2013" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The mechanisms of context-specific differences in signal transduction, such as those that occur among different cell types, are not fully understood. One possibility is that differences in the abundance of proteins change signaling outputs because these proteins compete for binding to hub proteins at critical network branch points. Focusing on the ErbB signaling, we created a protein interaction network that included information about protein domains and analyzed the role of competing protein interactions. By leveraging three-dimensional protein structures to infer steric interactions among binding partners for a common binding domain or linear motif (node) and including information about protein abundance and interaction affinities, we identified a large number of competitive, mutually exclusive (XOR) protein interactions. Modeling changes in protein abundance with different patterns of partner proteins and XOR nodes (XOR motifs) revealed that each motif conferred a different response. We experimentally investigated the XOR motif containing the hub protein Ras and its binding partners RIN1 (Ras and Rab interactor 1) and CRAF (v-raf-leukemia viral oncogene 1). Consistent with the computational prediction, overexpression of RIN1 in cultured cells decreased the phosphorylation of CRAF and its downstream targets. Thus, our analyses provide evidence that variation in the abundance of proteins that compete for binding to XOR nodes could contribute to context-specific signaling plasticity.</td>
</tr>
<tr id="bib_Kiel2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kiel2013,
  author = {Kiel, Christina and Verschueren, Erik and Jae-Seong Yang and Serrano, Luis},
  title = {Integration of protein abundance and structure data reveals competition in the ErbB signaling network.},
  journal = {Science signaling},
  year = {2013},
  volume = {6},
  number = {306},
  pages = {ra109 - ra109},
  doi = {https://doi.org/10.1126/scisignal.2004560}
}
</pre></td>
</tr>
<tr id="Kim2006" class="entry">
	<td>Kim, P., Long, L., Yu, X. and Mark, G.</td>
	<td>Relating Three-Dimensional Structure to Protein Network Provides Evolutionary Insights <p class="infolinks">[<a href="javascript:toggleInfo('Kim2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kim2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Science<br/>Vol. 314(5807), pp. 1938 - 1941&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1126/science.1136174">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kim2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Most studies of protein networks operate on a high level of abstraction, neglecting structural and chemical aspects of each interaction. Here, we characterize interactions by using atomic-resolution information from three-dimensional protein structures. We find that some previously recognized relationships between network topology and genomic features (e.g., hubs tending to be essential proteins) are actually more reflective of a structural quantity, the number of distinct binding interfaces. Subdividing hubs with respect to this quantity provides insight into their evolutionary rate and indicates that additional mechanisms of network growth are active in evolution (beyond effective preferential attachment through gene duplication).</td>
</tr>
<tr id="bib_Kim2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kim2006,
  author = {Kim, Phillip and Long, Lu and Yu, Xia and Mark, Gerstein},
  title = {Relating Three-Dimensional Structure to Protein Network Provides Evolutionary Insights},
  journal = {Science},
  year = {2006},
  volume = {314},
  number = {5807},
  pages = {1938 - 1941},
  doi = {https://doi.org/10.1126/science.1136174}
}
</pre></td>
</tr>
<tr id="Kim2017" class="entry">
	<td>Kim, S., Lee, J., Jo, S., Brooks, C.L., Lee, H. and Im, W.</td>
	<td>CHARMM-GUI ligand reader and modeler for CHARMM force field generation of small molecules <p class="infolinks">[<a href="javascript:toggleInfo('Kim2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kim2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of Computational Chemistry<br/>Vol. 38(21), pp. 1 - 8&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/jcc.24829">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kim2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Reading ligand structures into any simulation program is often nontrivial and time consuming, especially when the force field parameters and/or structure files of the corresponding molecules are not available. To address this problem, we have developed Ligand Reader &amp; Modeler in CHARMM-GUI. Users can upload ligand structure information in various forms (using PDB ID, ligand ID, SMILES, MOL/MOL2/SDF file, or PDB/mmCIF file), and the uploaded structure is displayed on a sketchpad for verification and further modification. Based on the displayed structure, Ligand Reader &amp; Modeler generates the ligand force field parameters and necessary structure files by searching for the ligand in the CHARMM force field library or using the CHARMM general force field (CGenFF). In addition, users can define chemical substitution sites and draw substituents in each site on the sketchpad to generate a set of combinatorial structure files and corresponding force field parameters for throughput or alchemical free energy simulations. Finally, the output from Ligand Reader &amp; Modeler can be used in other CHARMM-GUI modules to build a protein-ligand simulation system for all supported simulation programs, such as CHARMM, NAMD, GROMACS, AMBER, GENESIS, LAMMPS, Desmond, OpenMM, and CHARMM/OpenMM. Ligand Reader &amp; Modeler is available as a functional module of CHARMM-GUI at http://www.charmm-gui.org/input/ligandrm. textcopyright 2017 Wiley Periodicals, Inc.</td>
</tr>
<tr id="bib_Kim2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kim2017,
  author = {Kim, Seonghoon and Lee, Jumin and Jo, Sunhwan and Brooks, Charles L and Lee, Hui and Im, Wonpil},
  title = {CHARMM-GUI ligand reader and modeler for CHARMM force field generation of small molecules},
  journal = {Journal of Computational Chemistry},
  year = {2017},
  volume = {38},
  number = {21},
  pages = {1 - 8},
  doi = {https://doi.org/10.1002/jcc.24829}
}
</pre></td>
</tr>
<tr id="Kinugawa1997" class="entry">
	<td>Kinugawa, K., Moore, P.B. and Klein, M.L.</td>
	<td>Centroid path integral molecular dynamics simulation of lithium para-hydrogen clusters <p class="infolinks">[<a href="javascript:toggleInfo('Kinugawa1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kinugawa1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>The Journal of Chemical Physics<br/>Vol. 106(3), pp. 1154 - 1154&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.473211">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kinugawa1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The real-time quantum dynamics of a series of lithium para-hydrogen clusters, Li(p-H2) n (n=13, 55, and 180), has been investigated at 2.5 and 4.0 K by means of normal mode centroid path integral molecular dynamics (NMCMD) simulation, following the methodology originally proposed by Cao and Voth [J. Chem. Phys. 101, 6168 (1994)]. The Li(p-H2)34 and neat (p-H2)34clusters have also been simulated at 2.5 K to see the effect of doping of a Li atom on the cluster dynamics. We explicitly display both the microcanonical and the constant-temperature equations of motion for NMCMD simulations using the Nos&eacute;--Hoover chain thermostats and the reference system propagator algorithm (RESPA). In addition to the energetic and structural properties, the real-time semi-classical dynamics of the centroids of the Li atom and p-H2 molecules in the clusters has been explored to investigate the diffusive and vibrational properties. In general, quantization of the nuclear motion enhances the ease of melting and diffusion, and also causes the power spectra of the velocity autocorrelation functions of hydrogen molecules to be dramatically shifted to lower frequency than in the classical regime. The self-diffusion coefficient of p-H2 molecules in the Li(p-H2)13cluster, which is melted completely at both 2.5 and 4.0 K, is comparable to the experimental value of bulk liquidp-H2 just above the melting temperature (14 K). No significant differences have been found for the structural properties, molecular diffusion, and the quantum vibrational spectra of p-H2 molecules between the neat (p-H2)34 and Li(p-H2)34clusters at 2.5 K. The n=34, 55, and 180 clusters show a solidlike core and a tendency towards surface melting at 2.5 K, with diffusion restricted to the outer region. The core of the Li(p-H2)55cluster exhibits slow liquidlikediffusion at 4.0 K, whereas that of the Li(p-H2)180cluster remains solidlike at this temperature. The quantum power spectrum of the velocity autocorrelation function of the core molecules in the Li(p-H2)180cluster reveals vibrational frequencies in good accord with experimental neutron and Raman data on the bulk solid. The power spectrum of the Li atom on the clustersurface is peaked at low frequency (&amp;lt;20 cm−1).</td>
</tr>
<tr id="bib_Kinugawa1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kinugawa1997,
  author = {Kinugawa, Kenichi and Moore, Preston B and Klein, Michael L},
  title = {Centroid path integral molecular dynamics simulation of lithium para-hydrogen clusters},
  journal = {The Journal of Chemical Physics},
  year = {1997},
  volume = {106},
  number = {3},
  pages = {1154 - 1154},
  note = {From Duplicate 1 (Centroid path integral molecular dynamics simulation of lithium para-hydrogen clusters - Kinugawa, Kenichi; Moore, Preston B.; Klein, Michael L.) NULL},
  doi = {https://doi.org/10.1063/1.473211}
}
</pre></td>
</tr>
<tr id="Kirchhausen1999" class="entry">
	<td>Kirchhausen, T.</td>
	<td>Adaptors for clathrin-mediated traffic <p class="infolinks">[<a href="javascript:toggleInfo('Kirchhausen1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kirchhausen1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Annual Review of Cell and Developmental Biology<br/>Vol. 15(1), pp. 705 - 732&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1146/annurev.cellbio.15.1.705">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kirchhausen1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-based systems are responsible for a large portion of vesicular traffic originating from the plasma membrane and the trans-Golgi network that reaches the endosomal compartment. The assembly of cytosolic clathrin forms the scaffold required for the local deformation of the membrane and for the formation of coated pits and vesicles. In this process, clathrin interacts in a coordinated fashion with a large number of protein partners. A subset designated clathrin adaptors links integral membrane proteins to the clathrin coat, a process that results in the recruitment of specific cargo proteins to the budding vesicle. This review focuses on the most recent advances dealing with the molecular basis for sorting by clathrin adaptors.</td>
</tr>
<tr id="bib_Kirchhausen1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kirchhausen1999,
  author = {Kirchhausen, Tomas},
  title = {Adaptors for clathrin-mediated traffic},
  journal = {Annual Review of Cell and Developmental Biology},
  year = {1999},
  volume = {15},
  number = {1},
  pages = {705 - 732},
  doi = {https://doi.org/10.1146/annurev.cellbio.15.1.705}
}
</pre></td>
</tr>
<tr id="Kirchhausen2000" class="entry">
	<td>Kirchhausen, T.</td>
	<td>Clathrin <p class="infolinks">[<a href="javascript:toggleInfo('Kirchhausen2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kirchhausen2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Annual review of Biochemistry<br/>Vol. 69(1), pp. 699 - 727&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1146/annurev.biochem.69.1.699">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kirchhausen2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrinwas discovered nearly 25 years ago. Since then, a large number of other proteins that participate in the processbywhichclathrin-coatedvesicles retrieve synaptic membranes or take up endocytic receptors have been identified. The functional relationships among these disparate components remain, inmany cases, obscure. High- resolution structures of parts of clathrin, determined by X-ray crystallography, and lower-resolution images of assembled coats, determined by electron cryomicroscopy, now provide the information necessary to integrate various lines of evidence and to design experiments that test specific mechanistic notions. This reviewsummarizes and illustrates the recent structural results and outlines what is known about coated-vesicle assembly in the context of this information.</td>
</tr>
<tr id="bib_Kirchhausen2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kirchhausen2000,
  author = {Kirchhausen, Tomas},
  title = {Clathrin},
  journal = {Annual review of Biochemistry},
  year = {2000},
  volume = {69},
  number = {1},
  pages = {699 - 727},
  doi = {https://doi.org/10.1146/annurev.biochem.69.1.699}
}
</pre></td>
</tr>
<tr id="Kirchhausen2014" class="entry">
	<td>Kirchhausen, T., Owen, D. and Harrison, S.C.</td>
	<td>Molecular structure, function, and dynamics of clathrin-mediated membrane traffic <p class="infolinks">[<a href="javascript:toggleInfo('Kirchhausen2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kirchhausen2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Cold Spring Harbor Perspectives in Biology<br/>Vol. 6(5), pp. a016725 - a016725&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1101/cshperspect.a016725">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kirchhausen2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin is a molecular scaffold for vesicular uptake of cargo at the plasma membrane, where its assembly into cage-like lattices underlies the clathrin-coated pits of classical endocytosis. This review describes the structures of clathrin, major cargo adaptors, and other proteins that participate in forming a clathrin-coated pit, loading its contents, pinching off the membrane as a lattice-enclosed vesicle, and recycling the components. It integrates as much of the structural information as possible at the time of writing into a sketch of the principal steps in coated-pit and coated-vesicle formation.</td>
</tr>
<tr id="bib_Kirchhausen2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kirchhausen2014,
  author = {Kirchhausen, Tom and Owen, David and Harrison, Stephen C},
  title = {Molecular structure, function, and dynamics of clathrin-mediated membrane traffic},
  journal = {Cold Spring Harbor Perspectives in Biology},
  year = {2014},
  volume = {6},
  number = {5},
  pages = {a016725 - a016725},
  note = {good review of entire process, with some domain information.},
  doi = {https://doi.org/10.1101/cshperspect.a016725}
}
</pre></td>
</tr>
<tr id="Kirkpatrick1983" class="entry">
	<td>Kirkpatrick, S., Gelatt, C. and Vecchi, M.</td>
	<td>Optimization by simulated annealing. <p class="infolinks">[<a href="javascript:toggleInfo('Kirkpatrick1983','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kirkpatrick1983','bibtex')">BibTeX</a>]</p></td>
	<td>1983</td>
	<td>Science (New York, N.Y.)<br/>Vol. 220(4598), pp. 671 - 80&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1126/science.220.4598.671">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kirkpatrick1983" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: There is a deep and useful connection between statistical mechanics (the behavior of systems with many degrees of freedom in thermal equilibrium at a finite temperature) and multivariate or combinatorial optimization (finding the minimum of a given function depending on many parameters). A detailed analogy with annealing in solids provides a framework for optimization of the properties of very large and complex systems. This connection to statistical mechanics exposes new information and provides an unfamiliar perspective on traditional optimization problems and methods.</td>
</tr>
<tr id="bib_Kirkpatrick1983" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kirkpatrick1983,
  author = {Kirkpatrick, S and Gelatt, CD and Vecchi, MP},
  title = {Optimization by simulated annealing.},
  journal = {Science (New York, N.Y.)},
  year = {1983},
  volume = {220},
  number = {4598},
  pages = {671 - 80},
  doi = {https://doi.org/10.1126/science.220.4598.671}
}
</pre></td>
</tr>
<tr id="Kitano2002" class="entry">
	<td>Kitano, H.</td>
	<td>Computational Systems Biology <p class="infolinks">[<a href="javascript:toggleInfo('Kitano2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kitano2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Nature<br/>Vol. 420, pp. 229 - 247&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/B978-012088786-6/50031-9">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kitano2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: This chapter focuses on methods of constructing discrete dynamic models of biochemical networks from high throughput experimental data sets, also sometimes referred to as top-down modeling or reverse-engineering. Traditionally, models of molecular regulatory systems in cells have been created bottom-up, where the model is constructed piece-by-piece by adding new components and characterizing their interactions with other molecules in the model. Bottom-up modeling is essentially a process of synthesis by which models of isolated cellular components are merged to become part of a larger model. All top-down discrete modeling methods explored so far have some similarities. They all essentially take the view of a biochemical network as an information processing system. Each method settles on a particular modeling framework, such as graphical models, Boolean networks, and multistate models of a certain type. An important feature of the algorithm is that its performance is substantially improved by supplying as initialization the output of the exact data fitting algorithm described previously versus a random initialization. The key theoretical ingredient in the algorithm is a mathematical characterization of the evolution rules to guarantee that each mutation still satisfies the minimalist criterion imposed.</td>
</tr>
<tr id="bib_Kitano2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kitano2002,
  author = {Kitano, Hiroaki},
  title = {Computational Systems Biology},
  journal = {Nature},
  year = {2002},
  volume = {420},
  pages = {229 - 247},
  doi = {https://doi.org/10.1016/B978-012088786-6/50031-9}
}
</pre></td>
</tr>
<tr id="Kleinlogel2008" class="entry">
	<td>Kleinlogel, S. and White, A.G.</td>
	<td>The Secret World of Shrimps: Polarisation Vision at Its Best <p class="infolinks">[<a href="javascript:toggleInfo('Kleinlogel2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kleinlogel2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>PLoS ONE<br/>Vol. 3(5)&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pone.0002190">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kleinlogel2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Background: Animal vision spans a great range of complexity, with systems evolving to detect variations in light intensity, distribution, colour, and polarisation. Polarisation vision systems studied to date detect one to four channels of linear polarisation, combining them in opponent pairs to provide intensity-independent operation. Circular polarisation vision has never been seen, and is widely believed to play no part in animal vision. Methodology/Principal Findings: Polarisation is fully measured via Stokes' parameters---obtained by combined linear and circular polarisation measurements. Optimal polarisation vision is the ability to see Stokes' parameters: here we show that the crustacean Gonodactylus smithii measures the exact components required. Conclusions/Significance: This vision provides optimal contrast-enhancement and precise determination of polarisation with no confusion states or neutral points---significant advantages. Linear and circular polarisation each give partial information about the polarisation of light---but the combination of the two, as we will show here, results in optimal polarisation vision. We suggest that linear and circular polarisation vision not be regarded as different modalities, since both are necessary for optimal polarisation vision; their combination renders polarisation vision independent of strongly linearly or circularly polarised features in the animal's environment.</td>
</tr>
<tr id="bib_Kleinlogel2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kleinlogel2008,
  author = {Kleinlogel, Sonja and White, Andrew G},
  title = {The Secret World of Shrimps: Polarisation Vision at Its Best},
  journal = {PLoS ONE},
  year = {2008},
  volume = {3},
  number = {5},
  doi = {https://doi.org/10.1371/journal.pone.0002190}
}
</pre></td>
</tr>
<tr id="Kleist2011" class="entry">
	<td>von Kleist, L., Stahlschmidt, W., Bulut, H., Gromova, K., Puchkov, D., Robertson, M.J., MacGregor, K., Tomlin, N., Pechstein, A., Chau, N., Chircop, M., Sakoff, J., von Kries, J.P., Saenger, W., Kräusslich, H.-G., Shupliakov, O., Robinson, P.J., McCluskey, A. and Haucke, V.</td>
	<td>Role of the clathrin terminal domain in regulating coated pit dynamics revealed by small molecule inhibition. <p class="infolinks">[<a href="javascript:toggleInfo('Kleist2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kleist2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Cell<br/>Vol. 146, pp. 471-484&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cell.2011.06.025">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kleist2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis (CME) regulates many cell physiological processes such as the internalization of growth factors and receptors, entry of pathogens, and synaptic transmission. Within the endocytic network, clathrin functions as a central organizing platform for coated pit assembly and dissociation via its terminal domain (TD). We report the design and synthesis of two compounds named pitstops that selectively block endocytic ligand association with the clathrin TD as confirmed by X-ray crystallography. Pitstop-induced inhibition of clathrin TD function acutely interferes with receptor-mediated endocytosis, entry of HIV, and synaptic vesicle recycling. Endocytosis inhibition is caused by a dramatic increase in the lifetimes of clathrin coat components, including FCHo, clathrin, and dynamin, suggesting that the clathrin TD regulates coated pit dynamics. Pitstops provide new tools to address clathrin function in cell physiology with potential applications as inhibitors of virus and pathogen entry and as modulators of cell signaling.</td>
</tr>
<tr id="bib_Kleist2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kleist2011,
  author = {von Kleist, Lisa and Stahlschmidt, Wiebke and Bulut, Haydar and Gromova, Kira and Puchkov, Dmytro and Robertson, Mark J and MacGregor, Kylie and Tomlin, Nikolay and Pechstein, Arndt and Chau, Ngoc and Chircop, Megan and Sakoff, Jennette and von Kries, Jens Peter and Saenger, Wolfram and Kräusslich, Hans-Georg and Shupliakov, Oleg and Robinson, Phillip J and McCluskey, Adam and Haucke, Volker},
  title = {Role of the clathrin terminal domain in regulating coated pit dynamics revealed by small molecule inhibition.},
  journal = {Cell},
  year = {2011},
  volume = {146},
  pages = {471--484},
  doi = {https://doi.org/10.1016/j.cell.2011.06.025}
}
</pre></td>
</tr>
<tr id="Klinman1972" class="entry">
	<td>Klinman, J.</td>
	<td>The mechanism of enzyme-catalyzed reduced nicotinamide adenine dinucleotide-dependent reductions. Substituent and isotope effects in the yeast alcohol dehydrogenase reaction. <p class="infolinks">[<a href="javascript:toggleInfo('Klinman1972','bibtex')">BibTeX</a>]</p></td>
	<td>1972</td>
	<td>Journal of Biological Chemistry<br/>Vol. 247(24), pp. 7977 - 87&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Klinman1972" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Klinman1972,
  author = {Klinman, JP},
  title = {The mechanism of enzyme-catalyzed reduced nicotinamide adenine dinucleotide-dependent reductions. Substituent and isotope effects in the yeast alcohol dehydrogenase reaction.},
  journal = {Journal of Biological Chemistry},
  year = {1972},
  volume = {247},
  number = {24},
  pages = {7977 - 87}
}
</pre></td>
</tr>
<tr id="Klinman1976" class="entry">
	<td>Klinman, J.P.</td>
	<td>Isotope effects and structure-reactivity correlations in the yeast alcohol dehydrogenase reaction. A study of the enzyme-catalyzed oxidation of aromatic alcohols <p class="infolinks">[<a href="javascript:toggleInfo('Klinman1976','bibtex')">BibTeX</a>]</p></td>
	<td>1976</td>
	<td>Biochemistry<br/>Vol. 15(9), pp. 2018 - 2026&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi00654a032">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Klinman1976" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Klinman1976,
  author = {Klinman, Judith P},
  title = {Isotope effects and structure-reactivity correlations in the yeast alcohol dehydrogenase reaction. A study of the enzyme-catalyzed oxidation of aromatic alcohols},
  journal = {Biochemistry},
  year = {1976},
  volume = {15},
  number = {9},
  pages = {2018 - 2026},
  note = {Transition rate constant information},
  doi = {https://doi.org/10.1021/bi00654a032}
}
</pre></td>
</tr>
<tr id="Klosek1991" class="entry">
	<td>Klosek, M., Matkowsky, B. and Schuss, Z.</td>
	<td>The Kramers Problem in the Turnover Regime: The Role of the Stochastic Separatrix <p class="infolinks">[<a href="javascript:toggleInfo('Klosek1991','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Klosek1991','bibtex')">BibTeX</a>]</p></td>
	<td>1991</td>
	<td>Berichte der Bunsengesellschaft f&uuml;r physikalische Chemie<br/>Vol. 95(3), pp. 331 - 337&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/bbpc.19910950318">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Klosek1991" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Abstract 10.1002/bbpc.19910950318.abs We consider the problem of activated escape of a Brownian particle from a potential well. We find the stochastic separatrix S (the locus of starting points of the phase space trajectories which have equal probabilities of ending up inside or outside the well) for (i) the extremely anisotropic overdamped motion of a two-dimensional Brownian particle in a bistable potential, and (ii) the damped and underdamped motion of a one-dimensional Brownian particle in a single metastable state. The significance of S is that (1) it defines the reactant and product wells in a natural though not necessarily intuitive way, and (2) it reduces the calculation of the escape rate to the solution of the stationary Fokker-Planck equation inside S, with absorbing boundary conditions on S. Finally, employing this approach we derive an expression for the Kramers escape rate which bridges uniformly between Kramers weak damping regime and transition state theory.</td>
</tr>
<tr id="bib_Klosek1991" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Klosek1991,
  author = {Klosek, MM and Matkowsky, BJ and Schuss, Z},
  title = {The Kramers Problem in the Turnover Regime: The Role of the Stochastic Separatrix},
  journal = {Berichte der Bunsengesellschaft f&uuml;r physikalische Chemie},
  year = {1991},
  volume = {95},
  number = {3},
  pages = {331 - 337},
  doi = {https://doi.org/10.1002/bbpc.19910950318}
}
</pre></td>
</tr>
<tr id="Knapp2002" class="entry">
	<td>Knapp, M.J., Rickert, K. and Klinman, J.P.</td>
	<td>Temperature-dependent isotope effects in soybean Lipoxygenase-1: Correlating hydrogen tunneling with protein dynamics <p class="infolinks">[<a href="javascript:toggleInfo('Knapp2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Knapp2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Journal of the American Chemical Society<br/>Vol. 124(15), pp. 3865 - 3874&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ja012205t">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Knapp2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The hydrogen-atom transfer in soybean lipoxygenase-1 (SLO) exhibits a large kinetic isotope effect on kcat (KIE = 81) near room temperature and a very weak temperature dependence (Eact = 2.1 kcal/mol). These properties are consistent with H transfer that occurs entirely by a tunneling event. Mutants of SLO were prepared, and the temperature dependence of the KIE was measured, to test for alterations in the tunneling behavior. All mutants studied exhibit KIEs of similar, large magnitude at 30 C, despite an up to 3 orders of magnitude change in kcat. Eact for two of the mutants (Leu754 Ala, Leu546 Ala) is larger than for wild-type (WT), and the KIE becomes slightly more temperature dependent. In contrast, Ile553 Ala exhibits kcat and Eact parameters similar to wild-type soybean lipoxygenase-1 (WT-SLO) for protiated substrate; however, the KIE is markedly temperature dependent. The behavior of the former two mutants could reflect increased reorganization energies , but the behavior of the latter mutant is inconsistent with this description. We have invoked a full H tunneling model (Kuznetsov, A. M.; Ulstrup, J. Can. J. Chem. 1999, 77, 10851096) to explain the temperature dependence of the KIE, which is indicative of the extent to which distance sampling (gating) modulates hydrogen transfer. WT-SLO exhibits a very small Eact and a nearly temperature-independent KIE, which was modeled as arising from a compressed hydrogen transfer distance with little modulation of the hydrogen transfer distance. The observations on the Leu754 Ala and Leu546 Ala mutants were modeled as arising from a slightly less compressed active site with greater modulation of the hydrogen transfer distance by environmental dynamics. Finally, the observed behavior of the Ile553 Ala mutant indicates a relaxed active site with extensive involvement of gating to facilitate hydrogen transfer. We conclude that WT-SLO has an active site structure that is well organized to support hydrogen tunneling and that mutations perturb structural elements that support hydrogen tunneling. Modest alterations in active site residues increase and/or increase the hydrogen transfer distance, thereby affecting the probability that tunneling can occur. These studies allow the detection and characterization of a protein-gating mode in catalysis.</td>
</tr>
<tr id="bib_Knapp2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Knapp2002,
  author = {Knapp, Michael J and Rickert, Keith and Klinman, Judith P},
  title = {Temperature-dependent isotope effects in soybean Lipoxygenase-1: Correlating hydrogen tunneling with protein dynamics},
  journal = {Journal of the American Chemical Society},
  year = {2002},
  volume = {124},
  number = {15},
  pages = {3865 - 3874},
  doi = {https://doi.org/10.1021/ja012205t}
}
</pre></td>
</tr>
<tr id="Knuehl2006" class="entry">
	<td>Knuehl, C., Chen, C., Manalo, V., Hwang, P.K., Ota, N. and Brodsky, F.M.</td>
	<td>Novel binding sites on Clathrin and adaptors regulate distinct aspects of coat assembly <p class="infolinks">[<a href="javascript:toggleInfo('Knuehl2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Knuehl2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Traffic<br/>Vol. 7(12), pp. 1688 - 1700&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1111/j.1600-0854.2006.00499.x">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Knuehl2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-coated vesicles (CCVs) sort proteins at the plasma membrane, endosomes and trans Golgi network for multiple membrane traffic pathways. Clathrin recruitment to membranes and its self-assembly into a polyhedral coat depends on adaptor molecules, which interact with membrane-associated vesicle cargo. To determine how adaptors induce clathrin recruitment and assembly, we mapped novel interaction sites between these coat components. A site in the ankle domain of the clathrin triskelion leg was identified that binds a common site on the appendages of tetrameric [AP1 and AP2] and monomeric (GGA1) adaptors. Mutagenesis and modeling studies suggested that the clathrin-GGA1 appendage interface is nonlinear, unlike other peptide-appendage interactions, but overlaps with a sandwich domain binding site for accessory protein peptides, allowing for competitive regulation of coated vesicle formation. A novel clathrin box in the GGA1 hinge region was also identified and shown to mediate membrane recruitment of clathrin, while disruption of the clathrin-GGA1 appendage interaction did not affect recruitment. Thus, the distinct sites for clathrin-adaptor interactions perform distinct functions, revealing new aspects to regulation of CCV formation.</td>
</tr>
<tr id="bib_Knuehl2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Knuehl2006,
  author = {Knuehl, Christine and Chen, Chih and Manalo, Venus and Hwang, Peter K and Ota, Nobuyuki and Brodsky, Frances M},
  title = {Novel binding sites on Clathrin and adaptors regulate distinct aspects of coat assembly},
  journal = {Traffic},
  year = {2006},
  volume = {7},
  number = {12},
  pages = {1688 - 1700},
  note = {3T3 cells (mouse) domain information. ap2b1 - chc},
  doi = {https://doi.org/10.1111/j.1600-0854.2006.00499.x}
}
</pre></td>
</tr>
<tr id="Ko2010" class="entry">
	<td>Ko, G., Paradise, S., Chen, H., Graham, M., Vecchi, M., Bianchi, F., Cremona, O., Fiore, P. and Camilli, P.</td>
	<td>Selective high-level expression of epsin 3 in gastric parietal cells, where it is localized at endocytic sites of apical canaliculi. <p class="infolinks">[<a href="javascript:toggleInfo('Ko2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ko2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Proceedings of the National Academy of Sciences of the United States of America<br/>Vol. 107(50), pp. 21511 - 6&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.1016390107">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ko2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Epsin is a ubiquitin-binding endocytic adaptor, which is highly concentrated at clathrin-coated pits and coordinates acquisition of bilayer curvature with coat recruitment and cargo selection. Epsin is encoded by three distinct genes in mammals. Epsin 1 and 2 have broad tissue distribution with high-level expression in the brain. In contrast, epsin 3 was reported to be expressed primarily in immature keratinocytes. Here, we show that epsin 3 is selectively expressed at high levels in the stomach (including the majority of gastric cancers), where it is concentrated in parietal cells. In these cells, epsin 3 is enriched and colocalized with clathrin around apical canaliculi, the sites that control acidification of the stomach lumen via the exo-endocytosis of vesicles containing the H/K ATPase. Deletion of the epsin 3 gene in mice did not result in obvious pathological phenotypes in either the stomach or other organs, possibly because of overlapping functions of the other two epsins. However, levels of EHD1 and EHD2, two membrane tubulating proteins with a role in endocytic recycling, were elevated in epsin 3 knock-out stomachs, pointing to a functional interplay of epsin 3 with EHD proteins in the endocytic pathway of parietal cells. We suggest that epsin 3 cooperates with other bilayer binding proteins with curvature sensing/generating properties in the specialized traffic and membrane remodeling processes typical of gastric parietal cells.</td>
</tr>
<tr id="bib_Ko2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ko2010,
  author = {Ko, Genevieve and Paradise, Summer and Chen, Hong and Graham, Morven and Vecchi, Manuela and Bianchi, Fabrizio and Cremona, Ottavio and Fiore, Pier and Camilli, Pietro},
  title = {Selective high-level expression of epsin 3 in gastric parietal cells, where it is localized at endocytic sites of apical canaliculi.},
  journal = {Proceedings of the National Academy of Sciences of the United States of America},
  year = {2010},
  volume = {107},
  number = {50},
  pages = {21511 - 6},
  doi = {https://doi.org/10.1073/pnas.1016390107}
}
</pre></td>
</tr>
<tr id="Koh2007" class="entry">
	<td>Koh, T.-W., Korolchuk, V.I., Wairkar, Y.P., Jiao, W., Evergren, E., Pan, H., Zhou, Y., Venken, K.J.T., Shupliakov, O., Robinson, I.M., O'Kane, C.J. and Bellen, H.J.</td>
	<td>Eps15 and Dap160 control synaptic vesicle membrane retrieval and synapse development. <p class="infolinks">[<a href="javascript:toggleInfo('Koh2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Koh2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>The Journal of cell biology<br/>Vol. 178, pp. 309-322&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1083/jcb.200701030">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Koh2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Epidermal growth factor receptor pathway substrate clone 15 (Eps15) is a protein implicated in endocytosis, endosomal protein sorting, and cytoskeletal organization. Its role is, however, still unclear, because of reasons including limitations of dominant-negative experiments and apparent redundancy with other endocytic proteins. We generated Drosophila eps15-null mutants and show that Eps15 is required for proper synaptic bouton development and normal levels of synaptic vesicle (SV) endocytosis. Consistent with a role in SV endocytosis, Eps15 moves from the center of synaptic boutons to the periphery in response to synaptic activity. The endocytic protein, Dap160/intersectin, is a major binding partner of Eps15, and eps15 mutants phenotypically resemble dap160 mutants. Analyses of eps15 dap160 double mutants suggest that Eps15 functions in concert with Dap160 during SV endocytosis. Based on these data, we hypothesize that Eps15 and Dap160 promote the efficiency of endocytosis from the plasma membrane by maintaining high concentrations of multiple endocytic proteins, including dynamin, at synapses.</td>
</tr>
<tr id="bib_Koh2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Koh2007,
  author = {Koh, Tong-Wey and Korolchuk, Viktor I and Wairkar, Yogesh P and Jiao, Wei and Evergren, Emma and Pan, Hongling and Zhou, Yi and Venken, Koen J T and Shupliakov, Oleg and Robinson, Iain M and O'Kane, Cahir J and Bellen, Hugo J},
  title = {Eps15 and Dap160 control synaptic vesicle membrane retrieval and synapse development.},
  journal = {The Journal of cell biology},
  year = {2007},
  volume = {178},
  pages = {309--322},
  doi = {https://doi.org/10.1083/jcb.200701030}
}
</pre></td>
</tr>
<tr id="Koshland1959" class="entry">
	<td>Koshland, D.</td>
	<td>Enzyme flexibility and enzyme action <p class="infolinks">[<a href="javascript:toggleInfo('Koshland1959','bibtex')">BibTeX</a>]</p></td>
	<td>1959</td>
	<td>Journal of cellular and comparative physiology<br/>Vol. 54(S1), pp. 245 - 258&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/jcp.1030540420">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Koshland1959" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Koshland1959,
  author = {Koshland, DE},
  title = {Enzyme flexibility and enzyme action},
  journal = {Journal of cellular and comparative physiology},
  year = {1959},
  volume = {54},
  number = {S1},
  pages = {245 - 258},
  doi = {https://doi.org/10.1002/jcp.1030540420}
}
</pre></td>
</tr>
<tr id="Kozic2018" class="entry">
	<td>Kozic, M., Fox, S.J., Thomas, J.M., Verma, C.S. and Rigden, D.J.</td>
	<td>Large scale ab initio modelling of structurally uncharacterized antimicrobial peptides reveals known and novel folds <p class="infolinks">[<a href="javascript:toggleInfo('Kozic2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Proteins: Structure, Function, and Bioinformatics&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/prot.25473">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Kozic2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kozic2018,
  author = {Mara Kozic and Stephen J. Fox and Jens M. Thomas and Chandra S. Verma and Daniel J. Rigden},
  title = {Large scale ab initio modelling of structurally uncharacterized antimicrobial peptides reveals known and novel folds},
  journal = {Proteins: Structure, Function, and Bioinformatics},
  publisher = {Wiley-Blackwell},
  year = {2018},
  doi = {https://doi.org/10.1002/prot.25473}
}
</pre></td>
</tr>
<tr id="Kramers1940" class="entry">
	<td>Kramers, H.A.</td>
	<td>Brownian motion in a field of force and the diffusion model of chemical reactions <p class="infolinks">[<a href="javascript:toggleInfo('Kramers1940','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kramers1940','bibtex')">BibTeX</a>]</p></td>
	<td>1940</td>
	<td>Physica<br/>Vol. 7(4), pp. 284 - 304&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S0031-8914(40)90098-2">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kramers1940" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A particle which is caught in a potential hole and which, through the shuttling action of B r o w n i a n motion, can escape over a potential barrier yields a suitable model for elucidating the applicability of the transition state method for calculating the rate of chemical reactions.</td>
</tr>
<tr id="bib_Kramers1940" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kramers1940,
  author = {Kramers, Hendrik A},
  title = {Brownian motion in a field of force and the diffusion model of chemical reactions},
  journal = {Physica},
  year = {1940},
  volume = {7},
  number = {4},
  pages = {284 - 304},
  doi = {https://doi.org/10.1016/S0031-8914(40)90098-2}
}
</pre></td>
</tr>
<tr id="Kristensen2012" class="entry">
	<td>Kristensen, A.R., Gsponer, J. and Foster, L.J.</td>
	<td>A high-throughput approach for measuring temporal changes in the interactome <p class="infolinks">[<a href="javascript:toggleInfo('Kristensen2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kristensen2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td><br/>Vol. 9(9), pp. 907-909&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Kristensen2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Interactomes are often measured using affinity purification--mass spectrometry (AP-MS) or yeast two-hybrid approaches, but these methods do not provide stoichiometric or temporal information. We combine quantitative proteomics and size-exclusion chromatography to map 291 coeluting complexes. This method allows mapping of an interactome to the same depth and accuracy as AP-MS with less work and without overexpression or tagging. The  ...</td>
</tr>
<tr id="bib_Kristensen2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kristensen2012,
  author = {Kristensen, Anders R and Gsponer, Joerg and Foster, Leonard J},
  title = {A high-throughput approach for measuring temporal changes in the interactome},
  year = {2012},
  volume = {9},
  number = {9},
  pages = {907-909}
}
</pre></td>
</tr>
<tr id="Kulak2014" class="entry">
	<td>Kulak, N.A., Pichler, G., Paron, I., Nagaraj, N. and Mann, M.</td>
	<td>Minimal, encapsulated proteomic-sample processing applied to copy-number estimation in eukaryotic cells. <p class="infolinks">[<a href="javascript:toggleInfo('Kulak2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kulak2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Nature methods<br/>Vol. 11(3), pp. 319 - 24&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nmeth.2834">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kulak2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Mass spectrometry (MS)-based proteomics typically employs multistep sample-preparation workflows that are subject to sample contamination and loss. We report an in-StageTip method for performing sample processing, from cell lysis through elution of purified peptides, in a single, enclosed volume. This robust and scalable method largely eliminates contamination or loss. Peptides can be eluted in several fractions or in one step for single-run proteome analysis. In one day, we obtained the largest proteome coverage to date for budding and fission yeast, and found that protein copy numbers in these cells were highly correlated (R(2) = 0.78). Applying the in-StageTip method to quadruplicate measurements of a human cell line, we obtained copy-number estimates for 9,667 human proteins and observed excellent quantitative reproducibility between replicates (R(2) = 0.97). The in-StageTip method is straightforward and generally applicable in biological or clinical applications.</td>
</tr>
<tr id="bib_Kulak2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kulak2014,
  author = {Kulak, Nils A and Pichler, Garwin and Paron, Igor and Nagaraj, Nagarjuna and Mann, Matthias},
  title = {Minimal, encapsulated proteomic-sample processing applied to copy-number estimation in eukaryotic cells.},
  journal = {Nature methods},
  year = {2014},
  volume = {11},
  number = {3},
  pages = {319 - 24},
  doi = {https://doi.org/10.1038/nmeth.2834}
}
</pre></td>
</tr>
<tr id="Kumar2015" class="entry">
	<td>Kumar, A., Butler, B.M., Kumar, S. and Ozkan, S.B.</td>
	<td>Integration of structural dynamics and molecular evolution via protein interaction networks: a new era in genomic medicine. <p class="infolinks">[<a href="javascript:toggleInfo('Kumar2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kumar2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Current opinion in structural biology<br/>Vol. 35, pp. 135-142&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.sbi.2015.11.002">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kumar2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Sequencing technologies are revealing many new non-synonymous single nucleotide variants (nsSNVs) in each personal exome. To assess their functional impacts, comparative genomics is frequently employed to predict if they are benign or not. However, evolutionary analysis alone is insufficient, because it misdiagnoses many disease-associated nsSNVs, such as those at positions involved in protein interfaces, and because evolutionary predictions do not provide mechanistic insights into functional change or loss. Structural analyses can aid in overcoming both of these problems by incorporating conformational dynamics and allostery in nSNV diagnosis. Finally, protein-protein interaction networks using systems-level methodologies shed light onto disease etiology and pathogenesis. Bridging these network approaches with structurally resolved protein interactions and dynamics will advance genomic medicine.</td>
</tr>
<tr id="bib_Kumar2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kumar2015,
  author = {Kumar, Avishek and Butler, Brandon M and Kumar, Sudhir and Ozkan, S Banu},
  title = {Integration of structural dynamics and molecular evolution via protein interaction networks: a new era in genomic medicine.},
  journal = {Current opinion in structural biology},
  year = {2015},
  volume = {35},
  pages = {135--142},
  doi = {https://doi.org/10.1016/j.sbi.2015.11.002}
}
</pre></td>
</tr>
<tr id="Kumar1992" class="entry">
	<td>Kumar, S., Rosenberg, J.M., Bouzida, D., Swendsen, R.H. and Kollman, P.A.</td>
	<td>The weighted histogram analysis method for free energy calculations on biomolecules. I. The method <p class="infolinks">[<a href="javascript:toggleInfo('Kumar1992','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kumar1992','bibtex')">BibTeX</a>]</p></td>
	<td>1992</td>
	<td>Journal of Computational Chemistry<br/>Vol. 13(8), pp. 1011 - 1021&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/jcc.540130812">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kumar1992" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The Weighted Histogram Analysis Method (WHAM), an extension of Ferrenberg and Swendsen's Multiple Histogram Technique, has been applied for the first time on complex biomolecular Hamiltonians. The method is presented here as an extension of the Umbrella Sampling method for free-energy and Potential of Mean Force calculations. This algorithm possesses the following advantages over methods that are currently employed: (1) It provides a built-in estimate of sampling errors thereby yielding objective estimates of the optimal location and length of additional simulations needed to achieve a desired level of precision; (2) it yields the ``best'' value of free energies by taking into account all the simulations so as to minimize the statistical errors; (3) in addition to optimizing the links between simulations, it also allows multiple overlaps of probability distributions for obtaining better estimates of the free-energy differences. By recasting the Ferrenberg--Swendsen Multiple Histogram equations in a form suitable for molecular mechanics type Hamiltonians, we have demonstrated the feasibility and robustness of this method by applying it to a test problem of the generation of the Potential of Mean Force profile of the pseudorotation phase angle of the sugar ring in deoxyadenosine.</td>
</tr>
<tr id="bib_Kumar1992" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kumar1992,
  author = {Kumar, Shankar and Rosenberg, John M and Bouzida, Djamal and Swendsen, Robert H and Kollman, Peter A},
  title = {The weighted histogram analysis method for free energy calculations on biomolecules. I. The method},
  journal = {Journal of Computational Chemistry},
  year = {1992},
  volume = {13},
  number = {8},
  pages = {1011 - 1021},
  doi = {https://doi.org/10.1002/jcc.540130812}
}
</pre></td>
</tr>
<tr id="Kurokawa1998" class="entry">
	<td>Kurokawa, Y., Watanabe, A., Yoshimura, T., Esaki, N. and Soda, K.</td>
	<td>Transamination as a Side-Reaction Catalyzed by Alanine Racemase of Bacillus stearothermophilus <p class="infolinks">[<a href="javascript:toggleInfo('Kurokawa1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Journal of Biochemistry (Tokyo, Japan)<br/>Vol. 124, pp. 1163 - 1169&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Kurokawa1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kurokawa1998,
  author = {Kurokawa, Yoichi and Watanabe, Akira and Yoshimura, Tohru and Esaki, Nobuyoshi and Soda, Kenji},
  title = {Transamination as a Side-Reaction Catalyzed by Alanine Racemase of Bacillus stearothermophilus},
  journal = {Journal of Biochemistry (Tokyo, Japan)},
  year = {1998},
  volume = {124},
  pages = {1163 - 1169}
}
</pre></td>
</tr>
<tr id="Lafer2002" class="entry">
	<td>Lafer, E.M.</td>
	<td>Clathrin-protein interactions <p class="infolinks">[<a href="javascript:toggleInfo('Lafer2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Lafer2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Traffic<br/>Vol. 3, pp. 513-520&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Lafer2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: There is a complex network of protein-protein and protein-lipid interactions that underlie clathrin-mediated vesicular traffic in all compartmentalized cells from yeast to man. Major progress has been made in the determination of the three-dimensional structures of many of the components. Recently, there has been an explosion in the identification and characterization of clathrin binding partners. This review integrates the structural and biochemical information that is currently available to present a unified view of how many clathrin binding partners interact with clathrin.</td>
</tr>
<tr id="bib_Lafer2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Lafer2002,
  author = {Lafer, Eileen M},
  title = {Clathrin-protein interactions},
  journal = {Traffic},
  year = {2002},
  volume = {3},
  pages = {513--520}
}
</pre></td>
</tr>
<tr id="LeRoy2005" class="entry">
	<td>Le Roy, C. and Wrana, J.L.</td>
	<td>Clathrin-and non-clathrin-mediated endocytic regulation of cell signalling <p class="infolinks">[<a href="javascript:toggleInfo('LeRoy2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Nature reviews Molecular cell biology<br/>Vol. 6(2), pp. 112-126&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_LeRoy2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{LeRoy2005,
  author = {Le Roy, Christine and Wrana, Jeffrey L},
  title = {Clathrin-and non-clathrin-mediated endocytic regulation of cell signalling},
  journal = {Nature reviews Molecular cell biology},
  publisher = {Nature Publishing Group},
  year = {2005},
  volume = {6},
  number = {2},
  pages = {112--126}
}
</pre></td>
</tr>
<tr id="Leach1996" class="entry">
	<td>Leach, A.R.</td>
	<td>Molecular Modelling: Principles and Applications <p class="infolinks">[<a href="javascript:toggleInfo('Leach1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>&nbsp;</td>
	<td>book</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Leach1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{Leach1996,
  author = {Leach, Andrew R},
  title = {Molecular Modelling: Principles and Applications},
  publisher = {Longman},
  year = {1996}
}
</pre></td>
</tr>
<tr id="Lee2006" class="entry">
	<td>Lee, D.-W., Wu, X., Eisenberg, E. and Greene, L.E.</td>
	<td>Recruitment dynamics of GAK and auxilin to clathrin-coated pits during endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Lee2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Lee2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Journal of cell science<br/>Vol. 119, pp. 3502-3512&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1242/jcs.03092">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Lee2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Cyclin G-associated kinase (GAK), the ubiquitous form of the neuronal-specific protein auxilin 1, is an essential cofactor for Hsc70-dependent uncoating of clathrin-coated vesicles. Total internal reflectance microscopy was used to determine the timing of GAK binding relative to dynamin and clathrin binding during invagination of clathrin-coated pits. Following transient recruitment of dynamin to the clathrin puncta, large amounts of GAK are transiently recruited. GAK and clathrin then disappear from the evanescent field as the pit invaginates from the plasma membrane and finally these proteins disappear from the epifluorescence field, probably as the clathrin is uncoated from the budded vesicles by Hsc70. The recruitment of GAK is dependent on its PTEN-like domain, which we found binds to phospholipids. This suggests that interaction with phospholipids is essential for recruitment of GAK and, in turn, Hsc70, but Hsc70 recruitment alone might not be sufficient to induce irreversible clathrin uncoating. When budding of clathrin-coated pits is inhibited by actin depolymerization, there is repeated flashing of GAK on the clathrin-coated pit but neither scission nor irreversible uncoating occur. Therefore, budding as well as synchronous recruitment of GAK might be required for irreversible clathrin uncoating.</td>
</tr>
<tr id="bib_Lee2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Lee2006,
  author = {Lee, Dong-Won and Wu, Xufeng and Eisenberg, Evan and Greene, Lois E},
  title = {Recruitment dynamics of GAK and auxilin to clathrin-coated pits during endocytosis.},
  journal = {Journal of cell science},
  year = {2006},
  volume = {119},
  pages = {3502--3512},
  doi = {https://doi.org/10.1242/jcs.03092}
}
</pre></td>
</tr>
<tr id="Legendre-Guillemin2002" class="entry">
	<td>Legendre-Guillemin, V., Metzler, M., Charbonneau, M., Gan, L., Chopra, V., Philie, J., Hayden, M.R. and McPherson, P.</td>
	<td>HIP1 and HIP12 display differential binding to F-actin, AP2, and clathrin. Identification of a novel interaction with clathrin light chain <p class="infolinks">[<a href="javascript:toggleInfo('Legendre-Guillemin2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Legendre-Guillemin2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Journal of Biological Chemistry<br/>Vol. 277(22), pp. 19897 - 19904&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M112310200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Legendre-Guillemin2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Huntingtin-interacting protein 1 (HIP1) and HIP12 are orthologues of Sla2p, a yeast protein with essential functions in endocytosis and regulation of the actin cytoskeleton. We now report that HIP1 and HIP12 are major components of the clathrin coat that interact but differ in their ability to bind clathrin and the clathrin adaptor AP2. HIP1 contains a clathrin-box and AP2 consensus-binding sites that display high affinity binding to the terminal domain of the clathrin heavy chain and the ear domain of the AP2 alpha subunit, respectively. These consensus sites are poorly conserved in HIP12 and correspondingly, HIP12 does not bind to AP2 nor does it demonstrate high affinity clathrin binding. Moreover, HIP12 co-sediments with F-actin in contrast to HIP1, which exhibits no interaction with actin in vitro. Despite these differences, both proteins efficiently stimulate clathrin assembly through their central helical domain. Interestingly, in both HIP1 and HIP12, this domain binds directly to the clathrin light chain. Our data suggest that HIP1 and HIP12 play related yet distinct functional roles in clathrin-mediated endocytosis.</td>
</tr>
<tr id="bib_Legendre-Guillemin2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Legendre-Guillemin2002,
  author = {Legendre-Guillemin, Valerie and Metzler, Martina and Charbonneau, Martine and Gan, Lu and Chopra, Vikramjit and Philie, Jacynthe and Hayden, Michael R and McPherson, Peter},
  title = {HIP1 and HIP12 display differential binding to F-actin, AP2, and clathrin. Identification of a novel interaction with clathrin light chain},
  journal = {Journal of Biological Chemistry},
  year = {2002},
  volume = {277},
  number = {22},
  pages = {19897 - 19904},
  note = {HIP1/R - CLTB. difference between HIP1 and HIP1R.},
  doi = {https://doi.org/10.1074/jbc.M112310200}
}
</pre></td>
</tr>
<tr id="Legendre-Guillemin2005" class="entry">
	<td>Legendre-Guillemin, V., Metzler, M., Lemaire, J.-F., Philie, J., Gan, L., Hayden, M.R. and McPherson, P.S.</td>
	<td>Huntingtin interacting protein 1 (HIP1) regulates clathrin assembly through direct binding to the regulatory region of the clathrin light chain. <p class="infolinks">[<a href="javascript:toggleInfo('Legendre-Guillemin2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Legendre-Guillemin2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>The Journal of biological chemistry<br/>Vol. 280, pp. 6101-6108&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M408430200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Legendre-Guillemin2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Huntingtin interacting protein 1 (HIP1) is a component of clathrin coats. We previously demonstrated that HIP1 promotes clathrin assembly through its central helical domain, which binds directly to clathrin light chains (CLCs). To better understand the relationship between CLC binding and clathrin assembly we sought to dissect this interaction. Using C-terminal deletion constructs of the HIP1 helical domain, we identified a region between residues 450 and 456 that is required for CLC binding. Within this region, point mutations showed the importance of residues Leu-451, Leu-452, and Arg-453. Mutants that fail to bind CLC are unable to promote clathrin assembly in vitro but still mediate HIP1 homodimerization and heterodimerization with the family member HIP12/HIP1R. Moreover, HIP1 binding to CLC is necessary for HIP1 targeting to clathrin-coated pits and clathrin-coated vesicles. Interestingly, HIP1 binds to a highly conserved region of CLC previously demonstrated to regulate clathrin assembly. These results suggest a role for HIP1/CLC interactions in the regulation of clathrin assembly.</td>
</tr>
<tr id="bib_Legendre-Guillemin2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Legendre-Guillemin2005,
  author = {Legendre-Guillemin, Valerie and Metzler, Martina and Lemaire, Jean-Francois and Philie, Jacynthe and Gan, Lu and Hayden, Michael R and McPherson, Peter S},
  title = {Huntingtin interacting protein 1 (HIP1) regulates clathrin assembly through direct binding to the regulatory region of the clathrin light chain.},
  journal = {The Journal of biological chemistry},
  year = {2005},
  volume = {280},
  pages = {6101--6108},
  doi = {https://doi.org/10.1074/jbc.M408430200}
}
</pre></td>
</tr>
<tr id="Leskovac1998" class="entry">
	<td>Leskovac, V., Trivic, S. and Anderson, B.M.</td>
	<td>Use of competitive dead-end inhibitors to determine the chemical mechanism of action of yeast alcohol dehydrogenase <p class="infolinks">[<a href="javascript:toggleInfo('Leskovac1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Leskovac1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Molecular and Cellular Biochemistry<br/>Vol. 178(1-2), pp. 219 - 227&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Leskovac1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In this work, we have postulated a comprehensive and unified chemical mechanism of action for yeast alcohol dehydroge- nase (EC 1.1.1.1, constitutive, cytoplasmic), isolated from Saccharomyces cerevisiae. The chemical mechanism of yeast enzyme is based on the integrity of the proton relay system: His-51....NAD+ ....Thr-48....R.CH2 OH(H2 O)....Zn++ , stretching from His-51 on the surface of enzyme to the active site zinc atom in the substrate-binding site of enzyme. Further, it is based on extensive studies of steady-state kinetic properties of enzyme which were published recently. In this study, we have re- ported the pH-dependence of dissociation constants for several competitive dead-end inhibitors of yeast enzyme from their binary complexes with enzyme, or their ternary complexes with enzyme and NAD+ or NADH; inhibitors include: pyrazole, acetamide, sodium azide, 2-fluoroethanol, and 2,2,2-trifluorethanol. The unified mechanism describes the structures of four dissociation forms of apoenzyme, two forms of the binary complex E.NAD+ , three forms of the ternary complex E.NAD+ .alcohol, two forms of the ternary complex E.NADH.aldehyde and three binary complexes E.NADH. Appropriate pKa values have been ascribed to protonation forms of most of the above mentioned complexes of yeast enzyme with coenzymes and substrates.</td>
</tr>
<tr id="bib_Leskovac1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Leskovac1998,
  author = {Leskovac, Vladimir and Trivic, Svetlana and Anderson, Bruce M},
  title = {Use of competitive dead-end inhibitors to determine the chemical mechanism of action of yeast alcohol dehydrogenase},
  journal = {Molecular and Cellular Biochemistry},
  year = {1998},
  volume = {178},
  number = {1-2},
  pages = {219 - 227}
}
</pre></td>
</tr>
<tr id="Leskovac2002" class="entry">
	<td>Leskovac, V., Trivic, S. and Pericin, D.</td>
	<td>The three zinc-containing alcohol dehydrogenases from baker's yeast, Saccharomyces cerevisiae <p class="infolinks">[<a href="javascript:toggleInfo('Leskovac2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Leskovac2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>FEMS Yeast Research<br/>Vol. 2(4), pp. 481 - 494&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1111/j.1567-1364.2002.tb00116.x">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Leskovac2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: This review is a summary of our current knowledge of the structure, function and mechanism of action of the three zinc-containing alcohol dehydrogenases, YADH-1, YADH-2 and YADH-3, in baker's yeast, Saccharomyces cerevisiae. The opening section deals with the substrate specificity of the enzymes, covering the steady-state kinetic data for its most known substrates. In the following sections, the kinetic mechanism for this enzyme is reported, along with the values of all rate constants in the mechanism. The complete primary structures of the three isoenzymes of YADH are given, and the model of the 3D structure of the active site is presented. All known artificial mutations in the primary structure of the YADH are covered in full and described in detail. Further, the chemical mechanism of action for YADH is presented along with the complement of steady-state and ligand-binding data supporting this mechanism. Finally, the bio-organic chemistry of the hydride-transfer reactions catalyzed by the enzyme is covered: this chemistry explains the narrow substrate specificity and the enantioselectivity of the yeast enzyme.</td>
</tr>
<tr id="bib_Leskovac2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Leskovac2002,
  author = {Leskovac, Vladimir and Trivic, Svetlana and Pericin, Draginja},
  title = {The three zinc-containing alcohol dehydrogenases from baker's yeast, Saccharomyces cerevisiae},
  journal = {FEMS Yeast Research},
  year = {2002},
  volume = {2},
  number = {4},
  pages = {481 - 494},
  note = {THe paper that that mechanism figure is from},
  doi = {https://doi.org/10.1111/j.1567-1364.2002.tb00116.x}
}
</pre></td>
</tr>
<tr id="Levitt1983" class="entry">
	<td>Levitt, M.</td>
	<td>Molecular Dynamics of Native Protein II: Analysis and Nature of Motion <p class="infolinks">[<a href="javascript:toggleInfo('Levitt1983','bibtex')">BibTeX</a>]</p></td>
	<td>1983</td>
	<td>Journal of Molecular Biology<br/>Vol. 168, pp. 621 - 657&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Levitt1983" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Levitt1983,
  author = {Levitt, Michael},
  title = {Molecular Dynamics of Native Protein II: Analysis and Nature of Motion},
  journal = {Journal of Molecular Biology},
  year = {1983},
  volume = {168},
  pages = {621 - 657}
}
</pre></td>
</tr>
<tr id="Levitt1975" class="entry">
	<td>Levitt, M. and Warshel, A.</td>
	<td>Computer simulation of protein folding. <p class="infolinks">[<a href="javascript:toggleInfo('Levitt1975','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Levitt1975','bibtex')">BibTeX</a>]</p></td>
	<td>1975</td>
	<td>Nature<br/>Vol. 253(5494), pp. 694 - 698&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/253694a0">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Levitt1975" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A new and very simple representation of protein conformations has been used together with energy minimisation and thermalisation to simulate protein folding. Under certain conditions, the method succeeds in "renaturing" bovine pancreatic trypsin inhibitor from an open-chain conformation into a folded conformation close to that of the native molecule.</td>
</tr>
<tr id="bib_Levitt1975" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Levitt1975,
  author = {Levitt, Michael and Warshel, Arieh},
  title = {Computer simulation of protein folding.},
  journal = {Nature},
  year = {1975},
  volume = {253},
  number = {5494},
  pages = {694 - 698},
  doi = {https://doi.org/10.1038/253694a0}
}
</pre></td>
</tr>
<tr id="Lewandowicz2004" class="entry">
	<td>Lewandowicz, A. and Schramm, V.L.</td>
	<td>Transition state analysis for human and Plasmodium falciparum purine nucleoside phosphorylases. <p class="infolinks">[<a href="javascript:toggleInfo('Lewandowicz2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Lewandowicz2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Biochemistry<br/>Vol. 43(6), pp. 1458 - 1468&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi0359123">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Lewandowicz2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Recent studies have shown that Plasmodium falciparum is sensitive to a purine salvage block at purine nucleoside phosphorylase (PNP) and that human PNP is a target for T-cell proliferative diseases. Specific tight-binding inhibitors might be designed on the basis of specific PNP transition state structures. Kinetic isotope effects (KIEs) were measured for arsenolysis of inosine catalyzed by P. falciparum and human purine nucleoside phosphorylases. Intrinsic KIEs from [1'-(3)H]-, [2'-(3)H]-, [1'-(14)C]-, [9-(15)N]-, and [5'-(3)H]inosines were 1.184 +/- 0.004, 1.031 +/- 0.004, 1.002 +/- 0.006, 1.029 +/- 0.006, and 1.062 +/- 0.002 for the human enzyme and 1.116 +/- 0.007, 1.036 +/- 0.003, 0.996 +/- 0.006, 1.019 +/- 0.005, and 1.064 +/- 0.003 for P. falciparum PNPs, respectively. Analysis of KIEs indicated a highly dissociative D(N)A(N) (S(N)1) stepwise mechanism with very little leaving group involvement. The near-unity 1'-(14)C KIEs for both human and P. falciparum PNP agree with the theoretical value for a 1'-(14)C equilibrium isotope effect for oxacarbenium ion formation when computed at the B1LYP/6-31G(d) level of theory. The 9-(15)N KIE for human PNP is also in agreement with theory for equilibrium formation of hypoxanthine and oxacarbenium ion at this level of theory. The 9-(15)N KIE for P. falciparum PNP shows a constrained vibrational environment around N9 at the transition state. A relatively small beta-secondary 2'-(3)H KIE for both enzymes indicates a 3'-endo conformation for ribose and relatively weak hyperconjugation at the transition state. The large 5'-(3)H KIE reveals substantial distortion at the 5'-hydroxymethyl group which causes loosening of the C5'-H5' bonds during the reaction coordinate.</td>
</tr>
<tr id="bib_Lewandowicz2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Lewandowicz2004,
  author = {Lewandowicz, Andrzej and Schramm, Vern L},
  title = {Transition state analysis for human and Plasmodium falciparum purine nucleoside phosphorylases.},
  journal = {Biochemistry},
  year = {2004},
  volume = {43},
  number = {6},
  pages = {1458 - 1468},
  note = {Mentions KIE to first irreversable step},
  doi = {https://doi.org/10.1021/bi0359123}
}
</pre></td>
</tr>
<tr id="Lewin1998" class="entry">
	<td>Lewin, D.A. and Mellman, I.</td>
	<td>Sorting out adaptors <p class="infolinks">[<a href="javascript:toggleInfo('Lewin1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Biochimica et Biophysica Acta - Molecular Cell Research<br/>Vol. 1401(2), pp. 129 - 145&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S0167-4889(97)00130-4">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Lewin1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Lewin1998,
  author = {Lewin, David A and Mellman, Ira},
  title = {Sorting out adaptors},
  journal = {Biochimica et Biophysica Acta - Molecular Cell Research},
  year = {1998},
  volume = {1401},
  number = {2},
  pages = {129 - 145},
  doi = {https://doi.org/10.1016/S0167-4889(97)00130-4}
}
</pre></td>
</tr>
<tr id="Li2010" class="entry">
	<td>Li, C., Li, Y., Fu, X. and Deng, C.</td>
	<td>Anterior visceral endoderm SMAD4 signaling specifies anterior embryonic patterning and head induction in mice <p class="infolinks">[<a href="javascript:toggleInfo('Li2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Li2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>International Journal of Biological Sciences<br/>Vol. 6(6), pp. 569 - 583&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1091/mbc.E06">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Li2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: SMAD4 serves as a common mediator for signaling of TGF-β superfamily. Previous studies illustrated that SMAD4-null mice die at embryonic day 6.5 (E6.5) due to failure of mesoderm induction and extraembryonic defects; however, functions of SMAD4 in each germ layer remain elusive. To investigate this, we disrupted SMAD4 in the visceral endoderm and epiblast, respectively, using a Cre-loxP mediated approach. We showed that mutant embryos lack of SMAD4 in the visceral endoderm (Smad4(Co/Co);TTR-Cre) died at E7.5-E9.5 without head-fold and anterior embryonic structures. We demonstrated that TGF-β regulates expression of several genes, such as Hex1, Cer1, and Lim1, in the anterior visceral endoderm (AVE), and the failure of anterior embryonic development in Smad4(Co/Co);TTR-Cre embryos is accompanied by diminished expression of these genes. Consistent with this finding, SMAD4-deficient embryoid bodies showed impaired responsiveness to TGF-β-induced gene expression and morphological changes. On the other hand, embryos carrying Cre-loxP mediated disruption of SMAD4 in the epiblasts exhibited relatively normal mesoderm and head-fold induction although they all displayed profound patterning defects in the later stages of gastrulation. Cumulatively, our data indicate that SMAD4 signaling in the epiblasts is dispensable for mesoderm induction although it remains critical for head patterning, which is significantly different from SMAD4 signaling in the AVE, where it specifies anterior embryonic patterning and head induction.</td>
</tr>
<tr id="bib_Li2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Li2010,
  author = {Li, Cuiling and Li, Yi and Fu, Xin and Deng, Chu},
  title = {Anterior visceral endoderm SMAD4 signaling specifies anterior embryonic patterning and head induction in mice},
  journal = {International Journal of Biological Sciences},
  year = {2010},
  volume = {6},
  number = {6},
  pages = {569 - 583},
  doi = {https://doi.org/10.1091/mbc.E06}
}
</pre></td>
</tr>
<tr id="Li2013" class="entry">
	<td>Li, J., Bennett, K., Stukalov, A., Fang, B., Zhang, G., Yoshida, T., Okamoto, I., Jae-Young, K., Song, L., Bai, Y., Qian, X., Rawal, B., Schell, M., Grebien, F., Winter, G., Rix, U., Eschrich, S., Colinge, J., Koomen, J., Superti-Furga, G. and Haura, E.B.</td>
	<td>Perturbation of the mutated EGFR interactome identifies vulnerabilities and resistance mechanisms. <p class="infolinks">[<a href="javascript:toggleInfo('Li2013','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Li2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Molecular systems biology<br/>Vol. 9(705), pp. 705 - 705&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/msb.2013.61">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Li2013" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We hypothesized that elucidating the interactome of epidermal growth factor receptor (EGFR) forms that are mutated in lung cancer, via global analysis of protein-protein interactions, phosphorylation, and systematically perturbing the ensuing network nodes, should offer a new, more systems-level perspective of the molecular etiology. Here, we describe an EGFR interactome of 263 proteins and offer a 14-protein core network critical to the viability of multiple EGFR-mutated lung cancer cells. Cells with acquired resistance to EGFR tyrosine kinase inhibitors (TKIs) had differential dependence of the core network proteins based on the underlying molecular mechanisms of resistance. Of the 14 proteins, 9 are shown to be specifically associated with survival of EGFR-mutated lung cancer cell lines. This included EGFR, GRB2, MK12, SHC1, ARAF, CD11B, ARHG5, GLU2B, and CD11A. With the use of a drug network associated with the core network proteins, we identified two compounds, midostaurin and lestaurtinib, that could overcome drug resistance through direct EGFR inhibition when combined with erlotinib. Our results, enabled by interactome mapping, suggest new targets and combination therapies that could circumvent EGFR TKI resistance.</td>
</tr>
<tr id="bib_Li2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Li2013,
  author = {Li, Jiannong and Bennett, Keiryn and Stukalov, Alexey and Fang, Bin and Zhang, Guolin and Yoshida, Takeshi and Okamoto, Isamu and Jae-Young, Kim and Song, Lanxi and Bai, Yun and Qian, Xiaoning and Rawal, Bhupendra and Schell, Michael and Grebien, Florian and Winter, Georg and Rix, Uwe and Eschrich, Steven and Colinge, Jacques and Koomen, John and Superti-Furga, Giulio and Haura, Eric B},
  title = {Perturbation of the mutated EGFR interactome identifies vulnerabilities and resistance mechanisms.},
  journal = {Molecular systems biology},
  year = {2013},
  volume = {9},
  number = {705},
  pages = {705 - 705},
  note = {interactions - centered around AP2M1 - AP2A1, AP2A2, AP2B1, AP2M1, AP2S1, EPS15, FCHO2. bait, prey high-throughput.},
  doi = {https://doi.org/10.1038/msb.2013.61}
}
</pre></td>
</tr>
<tr id="Li2017" class="entry">
	<td>Li, L., Liu, J.-G. and Lu, J.</td>
	<td>Fractional Stochastic Differential Equations Satisfying Fluctuation-Dissipation Theorem <p class="infolinks">[<a href="javascript:toggleInfo('Li2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Li2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of Statistical Physics&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1007/s10955-017-1866-z">DOI</a> <a href="https://doi.org/10.1007/s10955-017-1866-z">URL</a>&nbsp;</td>
</tr>
<tr id="abs_Li2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We propose in this work a fractional stochastic differential equation (FSDE) model consistent with the over-damped limit of the generalized Langevin equation model. As a result of the `fluctuation-dissipation theorem', the differential equations driven by fractional Brownian noise to model memory effects should be paired with Caputo derivatives, and this FSDE model should be understood in an integral form. We establish the existence of strong solutions for such equations and discuss the ergodicity and convergence to Gibbs measure. In the linear forcing regime, we show rigorously the algebraic convergence to Gibbs measure when the `fluctuation-dissipation theorem' is satisfied, and this verifies that satisfying `fluctuation-dissipation theorem' indeed leads to the correct physical behavior. We further discuss possible approaches to analyze the ergodicity and convergence to Gibbs measure in the nonlinear forcing regime, while leave the rigorous analysis for future works. The FSDE model proposed is suitable for systems in contact with heat bath with power-law kernel and subdiffusion behaviors.</td>
</tr>
<tr id="bib_Li2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Li2017,
  author = {Li, Lei and Liu, Jian-Guo and Lu, Jianfeng},
  title = {Fractional Stochastic Differential Equations Satisfying Fluctuation-Dissipation Theorem},
  journal = {Journal of Statistical Physics},
  year = {2017},
  url = {https://doi.org/10.1007/s10955-017-1866-z},
  doi = {https://doi.org/10.1007/s10955-017-1866-z}
}
</pre></td>
</tr>
<tr id="Li2018" class="entry">
	<td>Li, P., Soudackov, A.V. and Hammes-Schiffer, S.</td>
	<td>Fundamental Insights into Proton-Coupled Electron Transfer in Soybean Lipoxygenase from Quantum Mechanical/Molecular Mechanical Free Energy Simulations <p class="infolinks">[<a href="javascript:toggleInfo('Li2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Journal of the American Chemical Society&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Li2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Li2018,
  author = {Li, Pengfei and Soudackov, Alexander V and Hammes-Schiffer, Sharon},
  title = {Fundamental Insights into Proton-Coupled Electron Transfer in Soybean Lipoxygenase from Quantum Mechanical/Molecular Mechanical Free Energy Simulations},
  journal = {Journal of the American Chemical Society},
  publisher = {ACS Publications},
  year = {2018}
}
</pre></td>
</tr>
<tr id="Li2018a" class="entry">
	<td>Li, W.</td>
	<td>Equipartition terms in transition path ensemble: Insights from molecular dynamics simulations of alanine dipeptide <p class="infolinks">[<a href="javascript:toggleInfo('Li2018a','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>The Journal of Chemical Physics<br/>Vol. 148(8), pp. 084105&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Li2018a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Li2018a,
  author = {Li, Wenjin},
  title = {Equipartition terms in transition path ensemble: Insights from molecular dynamics simulations of alanine dipeptide},
  journal = {The Journal of Chemical Physics},
  publisher = {AIP Publishing},
  year = {2018},
  volume = {148},
  number = {8},
  pages = {084105}
}
</pre></td>
</tr>
<tr id="Li2014" class="entry">
	<td>Li, W. and Ma, A.</td>
	<td>Recent developments in methods for identifying reaction coordinates. <p class="infolinks">[<a href="javascript:toggleInfo('Li2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Li2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Molecular simulation<br/>Vol. 40(10-11), pp. 784 - 793&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1080/08927022.2014.907898">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Li2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In the study of rare events in complex systems with many degrees of freedom, a key element is to identify the reaction coordinates of a given process. Over recent years, a number of methods and protocols have been developed to extract the reaction coordinates based on limited information from molecular dynamics simulations. In this review, we provide a brief survey over a number of major methods developed in the past decade, some of which are discussed in greater detail, to provide an overview of the problems that are partially solved and challenges that still remain. A particular emphasis has been placed on methods for identifying reaction coordinates that are related to the committor.</td>
</tr>
<tr id="bib_Li2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Li2014,
  author = {Li, Wenjin and Ma, Ao},
  title = {Recent developments in methods for identifying reaction coordinates.},
  journal = {Molecular simulation},
  year = {2014},
  volume = {40},
  number = {10-11},
  pages = {784 - 793},
  doi = {https://doi.org/10.1080/08927022.2014.907898}
}
</pre></td>
</tr>
<tr id="Lin2017" class="entry">
	<td>Lin, Y.-H. and Chan, H.</td>
	<td>Phase Separation and Single-Chain Compactness of Charged Disordered Proteins Are Strongly Correlated <p class="infolinks">[<a href="javascript:toggleInfo('Lin2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Lin2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Biophysical Journal<br/>Vol. 112(10), pp. 2043-2046&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.bpj.2017.04.021">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Lin2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Liquid-liquid phase separation of intrinsically disordered proteins (IDPs) is a major undergirding factor in the regu-lated formation of membraneless organelles in the cell. The phase behavior of an IDP is sensitive to its amino acid sequence. Here we apply a recent random-phase-approximation polymer theory to investigate how the tendency for multiple chains of a protein to phase-separate, as characterized by the critical temperature T* cr , is related to the protein's single-chain average radius of gyration hR g i. For a set of sequences containing different permutations of an equal number of positively and negatively charged residues, we found a striking correlation T* cr &dollar; hR g i &Agrave;g with g as large as &dollar;6.0, indicating that electrostatic effects have similarly significant impact on promoting single-chain conformational compactness and phase separation. Moreover, T* cr f &Agrave;SCD, where SCD is a recently proposed ''sequence charge decoration'' parameter determined solely by sequence information. Ramifications of our findings for deciphering the sequence dependence of IDP phase separation are discussed.</td>
</tr>
<tr id="bib_Lin2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Lin2017,
  author = {Lin, Yi-Hsuan and Chan, Hue},
  title = {Phase Separation and Single-Chain Compactness of Charged Disordered Proteins Are Strongly Correlated},
  journal = {Biophysical Journal},
  year = {2017},
  volume = {112},
  number = {10},
  pages = {2043-2046},
  doi = {https://doi.org/10.1016/j.bpj.2017.04.021}
}
</pre></td>
</tr>
<tr id="Lin2011" class="entry">
	<td>Lin, Y.-L., Gao, J., Rubinstein, A. and Major, D.</td>
	<td>Molecular dynamics simulations of the intramolecular proton transfer and carbanion stabilization in the pyridoxal 5'-phosphate dependent enzymes L-dopa decarboxylase and alanine racemase. <p class="infolinks">[<a href="javascript:toggleInfo('Lin2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Lin2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Biochimica et biophysica acta<br/>Vol. 1814(11), pp. 1438 - 46&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.bbapap.2011.05.002">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Lin2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Molecular dynamics simulations using a combined quantum mechanical and molecular mechanical (QM/MM) potential have been carried out to investigate the internal proton transfer equilibrium of the external aldimine species in l-dopa decarboxylase, and carbanion stabilization by the enzyme cofactor in the active site of alanine racemase. Solvent effects lower the free energy of the O-protonated PLP tautomer both in aqueous solution and in the active site, resulting a free energy difference of about −1kcal/mol relative to the N-protonated Schiff base in the enzyme. The external aldimine provides the dominant contribution to lowering the free energy barrier for the spontaneous decarboxylation of l-dopa in water, by a remarkable 16kcal/mol, while the enzyme l-dopa decarboxylase further lowers the barrier by 8kcal/mol. Kinetic isotope effects were also determined using a path integral free energy perturbation theory on the primary 13C and the secondary 2H substitutions. In the case of alanine racemase, if the pyridine ring is unprotonated as that in the active site, there is destabilizing contribution to the formation of the α-carbanion in the gas phase, although when the pyridine ring is protonated the contribution is stabilizing. In aqueous solution and in alanine racemase, the α-carbanion is stabilized both when the pyridine ring is protonated and unprotonated. The computational studies illustrated in this article show that combined QM/MM simulations can help provide a deeper understanding of the mechanisms of PLP-dependent enzymes. This article is part of a Special Issue entitled: Pyridoxal Phosphate Enzymology.</td>
</tr>
<tr id="bib_Lin2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Lin2011,
  author = {Lin, Yen-Lin and Gao, Jiali and Rubinstein, Amir and Major, Dan},
  title = {Molecular dynamics simulations of the intramolecular proton transfer and carbanion stabilization in the pyridoxal 5'-phosphate dependent enzymes L-dopa decarboxylase and alanine racemase.},
  journal = {Biochimica et biophysica acta},
  year = {2011},
  volume = {1814},
  number = {11},
  pages = {1438 - 46},
  doi = {https://doi.org/10.1016/j.bbapap.2011.05.002}
}
</pre></td>
</tr>
<tr id="Lippert2007" class="entry">
	<td>Lippert, R.A., Bowers, K.J., Dror, R.O., Eastwood, M.P., Gregersen, B.A., Klepeis, J.L., Kolossvary, I. and Shaw, D.E.</td>
	<td>A common, avoidable source of error in molecular dynamics integrators. <p class="infolinks">[<a href="javascript:toggleInfo('Lippert2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Lippert2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>The Journal of chemical physics<br/>Vol. 126(4), pp. 046101 - 046101&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.2431176">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Lippert2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In constrained molecular dynamics simulations using some of the most popular molecular dynamics codes, calculation of the velocities of constrained particles is based solely on the differences in particle positions during two successive time steps. This creates a numerical instability that the authors' show to be signicant in a typical single-precision floating-point simulation. They describe a simple modification that eliminates this source of instability and demonstrate that this change substantially reduces the energy drift of a sample single-precision N V E simulation.</td>
</tr>
<tr id="bib_Lippert2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Lippert2007,
  author = {Lippert, Ross A and Bowers, Kevin J and Dror, Ron O and Eastwood, Michael P and Gregersen, Brent A and Klepeis, John L and Kolossvary, Istvan and Shaw, David E},
  title = {A common, avoidable source of error in molecular dynamics integrators.},
  journal = {The Journal of chemical physics},
  year = {2007},
  volume = {126},
  number = {4},
  pages = {046101 - 046101},
  doi = {https://doi.org/10.1063/1.2431176}
}
</pre></td>
</tr>
<tr id="Liu2018" class="entry">
	<td>Liu, F., Choi, D., Xie, L. and Roeder, K.</td>
	<td>Global spectral clustering in dynamic networks <p class="infolinks">[<a href="javascript:toggleInfo('Liu2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Proceedings of the National Academy of Sciences, pp. 201718449&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Liu2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Liu2018,
  author = {Liu, Fuchen and Choi, David and Xie, Lu and Roeder, Kathryn},
  title = {Global spectral clustering in dynamic networks},
  journal = {Proceedings of the National Academy of Sciences},
  publisher = {National Acad Sciences},
  year = {2018},
  pages = {201718449}
}
</pre></td>
</tr>
<tr id="Liu2017" class="entry">
	<td>Liu, H., Dong, P., Ioannou, M.S., Li, L., Shea, J., Pasolli, H.A., Grimm, J.B., Rivlin, P.K., Lavis, L.D., Koyama, M. and others</td>
	<td>Visualizing long-term single-molecule dynamics in vivo by stochastic protein labeling <p class="infolinks">[<a href="javascript:toggleInfo('Liu2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Proceedings of the National Academy of Sciences, pp. 201713895&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Liu2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Liu2017,
  author = {Liu, Hui and Dong, Peng and Ioannou, Maria S and Li, Li and Shea, Jamien and Pasolli, H Amalia and Grimm, Jonathan B and Rivlin, Patricia K and Lavis, Luke D and Koyama, Minoru and others},
  title = {Visualizing long-term single-molecule dynamics in vivo by stochastic protein labeling},
  journal = {Proceedings of the National Academy of Sciences},
  publisher = {National Acad Sciences},
  year = {2017},
  pages = {201713895}
}
</pre></td>
</tr>
<tr id="Liu2011" class="entry">
	<td>Liu, Yang-Yu., Slotine, Jean-Jacques. and Barab&aacute;si, Albert-L.</td>
	<td>Controllability of complex networks. <p class="infolinks">[<a href="javascript:toggleInfo('Liu2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Liu2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Nature<br/>Vol. 473(7346), pp. 167 - 73&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nature10011">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Liu2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The ultimate proof of our understanding of natural or technological systems is reflected in our ability to control them. Although control theory offers mathematical tools for steering engineered and natural systems towards a desired state, a framework to control complex self-organized systems is lacking. Here we develop analytical tools to study the controllability of an arbitrary complex directed network, identifying the set of driver nodes with time-dependent control that can guide the system's entire dynamics. We apply these tools to several real networks, finding that the number of driver nodes is determined mainly by the network's degree distribution. We show that sparse inhomogeneous networks, which emerge in many real complex systems, are the most difficult to control, but that dense and homogeneous networks can be controlled using a few driver nodes. Counterintuitively, we find that in both model and real systems the driver nodes tend to avoid the high-degree nodes.</td>
</tr>
<tr id="bib_Liu2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Liu2011,
  author = {Liu, Yang-Yu and Slotine, Jean-Jacques and Barab&aacute;si, Albert-L&aacute;szl&oacute;},
  title = {Controllability of complex networks.},
  journal = {Nature},
  year = {2011},
  volume = {473},
  number = {7346},
  pages = {167 - 73},
  doi = {https://doi.org/10.1038/nature10011}
}
</pre></td>
</tr>
<tr id="Lobaugh1996" class="entry">
	<td>Lobaugh, J. and Voth, G.A.</td>
	<td>The quantum dynamics of an excess proton in water <p class="infolinks">[<a href="javascript:toggleInfo('Lobaugh1996','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Lobaugh1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>The Journal of Chemical Physics<br/>Vol. 104(5), pp. 2056 - 2056&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.470962">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Lobaugh1996" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The quantum dynamics and energetics of an excess proton in water have been studied computationally. Comparison of a quantum mechanical treatment of the transferring proton and the water solvent is made with a classical treatment of the same system. The exchange of the proton between two water molecules is found to be an activationless quantum process. Analysis of the microscopic structure of the solvent around the proton transfer complex is also carried out, and the quantum infrared spectrum of the transferring proton is calculated and analyzed in terms of Zundel polarization. The Gr&ouml;tthus mechanism for proton migration in water is also examined within the context of the model. Gr&ouml;tthus behavior is suggested to depend critically on the dynamics of water molecules in the second solvation shell of the H5O+ 2 complex, as well as the inward fluctuations of the oxygen--oxygen distance of water molecules that hydrogen bond to the H5O+ 2 complex in the first solvation shell. The quantum effects on the nuclear dynamics are found to be significant.</td>
</tr>
<tr id="bib_Lobaugh1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Lobaugh1996,
  author = {Lobaugh, J and Voth, Gregory A},
  title = {The quantum dynamics of an excess proton in water},
  journal = {The Journal of Chemical Physics},
  year = {1996},
  volume = {104},
  number = {5},
  pages = {2056 - 2056},
  doi = {https://doi.org/10.1063/1.470962}
}
</pre></td>
</tr>
<tr id="Loerke2009" class="entry">
	<td>Loerke, D., Mettlen, M., Yarar, D., Jaqaman, K., Jaqaman, H., Danuser, G. and Schmid, S.L.</td>
	<td>Cargo and dynamin regulate clathrin-coated pit maturation <p class="infolinks">[<a href="javascript:toggleInfo('Loerke2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>PLoS Biology<br/>Vol. 7(3), pp. e1000057&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Loerke2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Loerke2009,
  author = {Loerke, Dinah and Mettlen, Marcel and Yarar, Defne and Jaqaman, Khuloud and Jaqaman, Henry and Danuser, Gaudenz and Schmid, Sandra L},
  title = {Cargo and dynamin regulate clathrin-coated pit maturation},
  journal = {PLoS Biology},
  publisher = {Public Library of Science},
  year = {2009},
  volume = {7},
  number = {3},
  pages = {e1000057}
}
</pre></td>
</tr>
<tr id="Lonsdale2010" class="entry">
	<td>Lonsdale, R., Ranaghan, K.E. and Mulholland, A.J.</td>
	<td>Computational enzymology. <p class="infolinks">[<a href="javascript:toggleInfo('Lonsdale2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Lonsdale2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Chemical communications (Cambridge, England)<br/>Vol. 46(14), pp. 2354 - 72&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1039/b925647d">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Lonsdale2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Molecular simulations and modelling are changing the science of enzymology. Calculations can provide detailed, atomic-level insight into the fundamental mechanisms of biological catalysts. Computational enzymology is a rapidly developing area, and is testing theories of catalysis, challenging 'textbook' mechanisms, and identifying novel catalytic mechanisms. Increasingly, modelling is contributing directly to experimental studies of enzyme-catalysed reactions. Potential practical applications include interpretation of experimental data, catalyst design and drug development.</td>
</tr>
<tr id="bib_Lonsdale2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Lonsdale2010,
  author = {Lonsdale, Richard and Ranaghan, Kara E and Mulholland, Adrian J},
  title = {Computational enzymology.},
  journal = {Chemical communications (Cambridge, England)},
  year = {2010},
  volume = {46},
  number = {14},
  pages = {2354 - 72},
  doi = {https://doi.org/10.1039/b925647d}
}
</pre></td>
</tr>
<tr id="Lu2007" class="entry">
	<td>Lu, N. and Woolf, T.B.</td>
	<td>Understanding and Improving Free Energy Calculations in Molecular Simulations: Error Analysis and Reduction Methods <p class="infolinks">[<a href="javascript:toggleInfo('Lu2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td><br/>Vol. 86Free Energy Calculations, pp. 199&nbsp;</td>
	<td>inbook</td>
	<td><a href="https://doi.org/10.1007/978-3-540-38448-9_6">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Lu2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inbook{Lu2007,
  author = {Lu, Nandou and Woolf, Thomas B},
  title = {Understanding and Improving Free Energy Calculations in Molecular Simulations: Error Analysis and Reduction Methods},
  booktitle = {Free Energy Calculations},
  publisher = {Springer Berlin Heidelberg},
  year = {2007},
  volume = {86},
  pages = {199},
  doi = {https://doi.org/10.1007/978-3-540-38448-9_6}
}
</pre></td>
</tr>
<tr id="Lu2016" class="entry">
	<td>Lu, R., Drubin, D. and Sun, Y.</td>
	<td>Clathrin-mediated endocytosis in budding yeast at a glance <p class="infolinks">[<a href="javascript:toggleInfo('Lu2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Lu2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>J Cell Sci&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Lu2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: ABSTRACT Clathrin-mediated endocytosis is an essential cellular process that involves the concerted assembly and disassembly of many different proteins at the plasma membrane. In yeast, live-cell imaging has shown that the spatiotemporal dynamics of these proteins is highly stereotypical. Recent work has focused on determining how the timing and functions of endocytic proteins are regulated. In this Cell Science at a Glance article and  ...</td>
</tr>
<tr id="bib_Lu2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Lu2016,
  author = {Lu, R and Drubin, DG and Sun, Y},
  title = {Clathrin-mediated endocytosis in budding yeast at a glance},
  journal = {J Cell Sci},
  year = {2016}
}
</pre></td>
</tr>
<tr id="Lv2018" class="entry">
	<td>Lv, J., Ren, K. and Chen, Y.</td>
	<td>CO2 Diffusion in Various Carbonated Beverages: A Molecular Dynamics Study <p class="infolinks">[<a href="javascript:toggleInfo('Lv2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>The Journal of Physical Chemistry B&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Lv2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Lv2018,
  author = {Lv, Ji and Ren, Kaixin and Chen, Yakun},
  title = {CO2 Diffusion in Various Carbonated Beverages: A Molecular Dynamics Study},
  journal = {The Journal of Physical Chemistry B},
  publisher = {ACS Publications},
  year = {2018}
}
</pre></td>
</tr>
<tr id="Ma2016" class="entry">
	<td>Ma, L., Umasankar, P.K., Wrobel, A.G., Lymar, A., McCoy, Airlie, J, Holkar, S.S., Jha, A., Tirthadipa, Pradhan-Sundd,., Watkins, S.C., Owen, D.J. and Traub, L.M.</td>
	<td>Transient Fcho1/2*Eps15/R*AP-2 Nanoclusters Prime the AP-2 Clathrin Adaptor for Cargo Binding <p class="infolinks">[<a href="javascript:toggleInfo('Ma2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ma2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Developmental Cell<br/>Vol. 37(5), pp. 428 - 443&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.devcel.2016.05.003">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ma2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-coated vesicles form by rapid assembly of discrete coat constituents into a cargo-sorting lattice. How the sequential phases of coat construction are choreographed is unclear, but transient protein-protein interactions mediated by short interaction motifs are pivotal. We show that arrayed Asp-Pro-Phe (DPF) motifs within the early-arriving endocytic pioneers Eps15/R are differentially decoded by other endocytic pioneers Fcho1/2 and AP-2. The structure of an Eps15/R???Fcho1????-homology domain complex reveals a spacing-dependent DPF triad, bound in a mechanistically distinct way from the mode of single DPF binding to AP-2. Using cells lacking FCHO1/2 and with Eps15 sequestered from the plasma membrane, we establish that without these two endocytic pioneers, AP-2 assemblies are??fleeting and endocytosis stalls. Thus, distinct DPF-based codes within the unstructured Eps15/R C terminus direct the assembly of temporary Fcho1/2???Eps15/R???AP-2 ternary complexes to facilitate conformational activation of AP-2 by the Fcho1/2 interdomain linker to promote AP-2 cargo engagement.</td>
</tr>
<tr id="bib_Ma2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ma2016,
  author = {Ma, Li and Umasankar, Perunthottathu K and Wrobel, Antoni G and Lymar, Anastasia and J, McCoy, Airlie and Holkar, Sachin S and Jha, Anupma and Tirthadipa, Pradhan-Sundd, and Watkins, Simon C and Owen, David J and Traub, Linton M},
  title = {Transient Fcho1/2*Eps15/R*AP-2 Nanoclusters Prime the AP-2 Clathrin Adaptor for Cargo Binding},
  journal = {Developmental Cell},
  year = {2016},
  volume = {37},
  number = {5},
  pages = {428 - 443},
  note = {good citations for domain information. dissociation constants.},
  doi = {https://doi.org/10.1016/j.devcel.2016.05.003}
}
</pre></td>
</tr>
<tr id="MacGregor2014" class="entry">
	<td>MacGregor, K.A., Robertson, M.J., Young, K.A., von Kleist, L., Stahlschmidt, W., Whiting, A., Chau, N., Robinson, P.J., Haucke, V. and McCluskey, A.</td>
	<td>Development of 1,8-naphthalimides as clathrin inhibitors. <p class="infolinks">[<a href="javascript:toggleInfo('MacGregor2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('MacGregor2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Journal of medicinal chemistry<br/>Vol. 57, pp. 131-143&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jm4015263">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_MacGregor2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We reported the first small molecule inhibitors of the interaction between the clathrin N-terminal domain (TD) and endocyctic accessory proteins (i.e., clathrin inhibition1). Initial screening of a ∼17 000 small molecule ChemBioNet library identified 1. Screening of an existing in-house propriety library identified four substituted 1,8-napthalimides as ∼80-120 μM clathrin inhibitors. Focused library development gave 3-sulfo-N-(4-aminobenzyl)-1,8-naphthalimide, potassium salt (18, IC50 ≈ 18 μM). A second library targeting the 4-aminobenzyl moiety was developed, and four analogues displayed comparable activity (26, 27, 28, 34 with IC50 values of 22, 16, 15, and 15 μM respectively) with a further four (24, 25, 32, 33) more active than 18 with IC50 values of 10, 6.9, 12, and 10 μM, respectively. Docking studies rationalized the structure-activity relationship (SAR) with the biological data. 3-Sulfo-N-benzyl-1,8-naphthalimide, potassium salt (25) with an IC50 ≈ 6.9 μM, is the most potent clathrin terminal domain-amphiphysin inhibitor reported to date.</td>
</tr>
<tr id="bib_MacGregor2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{MacGregor2014,
  author = {MacGregor, Kylie A and Robertson, Mark J and Young, Kelly A and von Kleist, Lisa and Stahlschmidt, Wiebke and Whiting, Ainslie and Chau, Ngoc and Robinson, Phillip J and Haucke, Volker and McCluskey, Adam},
  title = {Development of 1,8-naphthalimides as clathrin inhibitors.},
  journal = {Journal of medicinal chemistry},
  year = {2014},
  volume = {57},
  pages = {131--143},
  note = {"small" molecule clathrin inhibitors},
  doi = {https://doi.org/10.1021/jm4015263}
}
</pre></td>
</tr>
<tr id="Mackerell2004" class="entry">
	<td>Mackerell, A.D., Feig, M. and Brooks, C.L.</td>
	<td>Extending the treatment of backbone energetics in protein force fields: Limitations of gas-phase quantum mechanics in reproducing protein conformational distributions in molecular dynamics simulations <p class="infolinks">[<a href="javascript:toggleInfo('Mackerell2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Mackerell2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of Computational Chemistry<br/>Vol. 25(11), pp. 1400 - 1415&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/jcc.20065">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Mackerell2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Computational studies of proteins based on empirical force fields represent a powerful tool to obtain structure--function relationships at an atomic level, and are central in current efforts to solve the protein folding problem. The results from studies applying these tools are, however, dependent on the quality of the force fields used. In particular, accurate treatment of the peptide backbone is crucial to achieve representative conformational distributions in simulation studies. To improve the treatment of the peptide backbone, quantum mechanical (QM) and molecular mechanical (MM) calculations were undertaken on the alanine, glycine, and proline dipeptides, and the results from these calculations were combined with molecular dynamics (MD) simulations of proteins in crystal and aqueous environments. QM potential energy maps of the alanine and glycine dipeptides at the LMP2/cc-pVxZ//MP2/6-31G* levels, where x = D, T, and Q, were determined, and are compared to available QM studies on these molecules. The LMP2/cc-pVQZ//MP2/6-31G* energy surfaces for all three dipeptides were then used to improve the MM treatment of the dipeptides. These improvements included additional parameter optimization via Monte Carlo simulated annealing and extension of the potential energy function to contain peptide backbone ϕ, ψ dihedral crossterms or a ϕ, ψ grid-based energy correction term. Simultaneously, MD simulations of up to seven proteins in their crystalline environments were used to validate the force field enhancements. Comparison with QM and crystallographic data showed that an additional optimization of the ϕ, ψ dihedral parameters along with the grid-based energy correction were required to yield significant improvements over the CHARMM22 force field. However, systematic deviations in the treatment of ϕ and ψ in the helical and sheet regions were evident. Accordingly, empirical adjustments were made to the grid-based energy correction for alanine and glycine to account for these systematic differences. These adjustments lead to greater deviations from QM data for the two dipeptides but also yielded improved agreement with experimental crystallographic data. These improvements enhance the quality of the CHARMM force field in treating proteins. This extension of the potential energy function is anticipated to facilitate improved treatment of biological macromolecules via MM approaches in general. textcopyright 2004 Wiley Periodicals, Inc. J Comput Chem 25: 1400--1415, 2004</td>
</tr>
<tr id="bib_Mackerell2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mackerell2004,
  author = {Mackerell, Alexander D and Feig, Michael and Brooks, Charles L},
  title = {Extending the treatment of backbone energetics in protein force fields: Limitations of gas-phase quantum mechanics in reproducing protein conformational distributions in molecular dynamics simulations},
  journal = {Journal of Computational Chemistry},
  year = {2004},
  volume = {25},
  number = {11},
  pages = {1400 - 1415},
  note = {NULL},
  doi = {https://doi.org/10.1002/jcc.20065}
}
</pre></td>
</tr>
<tr id="Madern2002" class="entry">
	<td>Madern, D.</td>
	<td>Molecular evolution within the L-malate and L-lactate dehydrogenase super-family <p class="infolinks">[<a href="javascript:toggleInfo('Madern2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Madern2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Journal of Molecular Evolution<br/>Vol. 54(6), pp. 825 - 840&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1007/s00239-001-0088-8">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Madern2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The NAD(P)-dependent malate (L-MalDH) and NAD-dependent lactate (L-LDH) form a large super-family that has been characterized in organisms belonging to the three domains of life. In the first part of this study, the group of [LDH-like] L-MalDH, which are malate dehydrogenases resembling lactate dehydrogenase, were analyzed and clearly defined with respect to the other enzymes. In the second part, the phylogenetic relationships of the whole super-family were presented by taking into account the [LDH-like] L-MalDH. The inferred tree unambiguously shows that two ancestral genes duplications, and not one as generally thought, are needed to explain both the distribution into two enzymatic functions and the observation of three main groups within the super-family: L-LDH, [LDH-like] L-MalDH, and dimeric L-MalDH. In addition, various cases of functional changes within each group were observed and analyzed. The direction of evolution was found to always be polarized: from enzymes with a high stringency of substrate recognition to enzymes with a broad substrate specificity. A specific phyletic distribution of the L-LDH, [LDH-like] L-MalDH, and dimeric L-MalDH over the Archaeal, Bacterial, and Eukaryal domains was observed. This was analyzed in the light of biochemical, structural, and genomic data available for the L-LDH, [LDH-like] L-MalDH, and dimeric L-MalDH. This analysis led to the elaboration of a refined evolutionary scenario of the super-family, in which the selection of L-LDH and the fate of L-MalDH during mitochrondrial genesis are presented.</td>
</tr>
<tr id="bib_Madern2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Madern2002,
  author = {Madern, Dominique},
  title = {Molecular evolution within the L-malate and L-lactate dehydrogenase super-family},
  journal = {Journal of Molecular Evolution},
  year = {2002},
  volume = {54},
  number = {6},
  pages = {825 - 840},
  note = {NULL},
  doi = {https://doi.org/10.1007/s00239-001-0088-8}
}
</pre></td>
</tr>
<tr id="Madern2004" class="entry">
	<td>Madern, D., Cai, X., Abrahamsen, M.S. and Zhu, G.</td>
	<td>Evolution of Cryptosporidium parvum lactate dehydrogenase from malate dehydrogenase by a very recent event of gene duplication. <p class="infolinks">[<a href="javascript:toggleInfo('Madern2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Madern2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Molecular Biology and Evolution<br/>Vol. 21(3), pp. 489 - 497&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/molbev/msh042">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Madern2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We have expressed the L-lactate dehydrogenase (LDH) and L-malate dehydrogenase (malDH) genes from the apicomplexan Cryptosporidium parvum (CpLDH1 and CpMalDH1) as maltose-binding protein (MBP) fusion proteins in Escherichia coli. The substrate specificities, enzymatic kinetics, and oligomeric states of these two parasite enzymes have been characterized. By taking advantage of recently completed and ongoing apicomplexan genome sequencing projects, we identified additional MalDH genes from Plasmodium spp., Toxoplasma gondii, and Eimeria tenella that were previously unavailable. All apicomplexan MalDHs appeared to be cytosolic and no organellar homologs were identified from the completely sequenced P. falciparum genome and other ongoing apicomplexan genome-sequencing projects. Using these expanded apicomplexan LDH and MalDH sequence databases, we reexamined their phylogenetic relationships and reconfirmed their relationship to alpha-proteobacterial MalDHs. All LDH and MalDH enzymes from apicomplexans were monophyletic within the LDH-like MalDH group (i.e., MalDH resembling LDH) as a sister to alpha-proteobacterial MalDHs. All apicomplexan LDHs, with the exception of CpLDH1, formed a separate clade from their MalDH counterparts, indicating that these LDHs were evolved from an ancestral apicomplexan MalDH by a gene duplication coupled with functional conversion before the expansion of apicomplexans. Finally, CpLDH1 was consistently placed together with CpMalDH1 within the apicomplexan MalDH cluster, confirming an early working hypothesis that CpLDH1 was probably evolved from the same ancestor of CpMalDH1 by a very recent gene duplication that occurred after C. parvum diverged from other apicomplexans.</td>
</tr>
<tr id="bib_Madern2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Madern2004,
  author = {Madern, Dominique and Cai, Xiaomin and Abrahamsen, Mitchell S and Zhu, Guan},
  title = {Evolution of Cryptosporidium parvum lactate dehydrogenase from malate dehydrogenase by a very recent event of gene duplication.},
  journal = {Molecular Biology and Evolution},
  year = {2004},
  volume = {21},
  number = {3},
  pages = {489 - 497},
  note = {Evolution of cpldh recently from cpmdh},
  doi = {https://doi.org/10.1093/molbev/msh042}
}
</pre></td>
</tr>
<tr id="Magonet1992" class="entry">
	<td>Magonet, E., Hayen, P., Delforge, D., Delaive, E. and Remacle, J.</td>
	<td>Importance of the structural zinc atom for the stability of yeast alcohol dehydrogenase <p class="infolinks">[<a href="javascript:toggleInfo('Magonet1992','bibtex')">BibTeX</a>]</p></td>
	<td>1992</td>
	<td>Biochemical Journal<br/>Vol. 287, pp. 361 - 365&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Magonet1992" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Magonet1992,
  author = {Magonet, Etienne and Hayen, Pierre and Delforge, Dominique and Delaive, Edouard and Remacle, Jose},
  title = {Importance of the structural zinc atom for the stability of yeast alcohol dehydrogenase},
  journal = {Biochemical Journal},
  year = {1992},
  volume = {287},
  pages = {361 - 365}
}
</pre></td>
</tr>
<tr id="Mahan1974" class="entry">
	<td>Mahan, B.H.</td>
	<td>Activated complex theory of bimolecular reactions <p class="infolinks">[<a href="javascript:toggleInfo('Mahan1974','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Mahan1974','bibtex')">BibTeX</a>]</p></td>
	<td>1974</td>
	<td>Journal of Chemical Education<br/>Vol. 51(11), pp. 709 - 709&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ed051p709">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Mahan1974" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: It is the purpose of this paper to present a detailed but simple derivation of the ART\ expression for a bimolecular reaction which avoids certain of the difficulties in the common treatments, and exposes the assumptions involved in a clear fashion.</td>
</tr>
<tr id="bib_Mahan1974" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mahan1974,
  author = {Mahan, Bruce H},
  title = {Activated complex theory of bimolecular reactions},
  journal = {Journal of Chemical Education},
  year = {1974},
  volume = {51},
  number = {11},
  pages = {709 - 709},
  doi = {https://doi.org/10.1021/ed051p709}
}
</pre></td>
</tr>
<tr id="Major2005" class="entry">
	<td>Major, D. and Gao, J.</td>
	<td>Implementation of the bisection sampling method in path integral simulations. <p class="infolinks">[<a href="javascript:toggleInfo('Major2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Major2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Journal of molecular graphics &amp; modelling<br/>Vol. 24(2), pp. 121 - 7&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.jmgm.2005.04.002">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Major2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A bisection sampling method is implemented in the framework of a quantized classical path algorithm to include nuclear quantum effects in path integral simulations. The present study examines the convergence of these calculations on two model systems with respect to the number of beads used in the polymer chain and the number of configurations both in free-particle sampling and in classical configuration sampling. The results will be useful for future studies of kinetic isotope effects in enzymatic reactions.</td>
</tr>
<tr id="bib_Major2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Major2005,
  author = {Major, Dan and Gao, Jiali},
  title = {Implementation of the bisection sampling method in path integral simulations.},
  journal = {Journal of molecular graphics &amp; modelling},
  year = {2005},
  volume = {24},
  number = {2},
  pages = {121 - 7},
  doi = {https://doi.org/10.1016/j.jmgm.2005.04.002}
}
</pre></td>
</tr>
<tr id="Major2006" class="entry">
	<td>Major, D. and Gao, J.</td>
	<td>A combined quantum mechanical and molecular mechanical study of the reaction mechanism and α-amino acidity in alanine racemase <p class="infolinks">[<a href="javascript:toggleInfo('Major2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Journal of the American Chemical Society<br/>Vol. 128, pp. 16345 - 16357&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Major2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Major2006,
  author = {Major, Dan and Gao, Jiali},
  title = {A combined quantum mechanical and molecular mechanical study of the reaction mechanism and α-amino acidity in alanine racemase},
  journal = {Journal of the American Chemical Society},
  year = {2006},
  volume = {128},
  pages = {16345 - 16357}
}
</pre></td>
</tr>
<tr id="Major2007" class="entry">
	<td>Major, D. and Gao, J.</td>
	<td>An integrated path integral and free-energy perturbation - umbrella sampling method for computing kinetic isotope effects of chemical reactions in solution and in enzymes <p class="infolinks">[<a href="javascript:toggleInfo('Major2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Major2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Journal of Chemical Theory and Computation<br/>Vol. 3(3), pp. 949 - 960&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ct600371k">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Major2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: An integrated centroid path integral and free-energy perturbation-umbrella sampling (PI-FEP/UM) method for computing kinetic isotope effects (KIEs) for chemical reactions in solution and in enzymes is presented. The method is based on the bisection sampling in centroid path integral simulations to include nuclear quantum effects to the classical potential of mean force. The required accuracy for computing kinetic isotope effects is achieved by coupled free-energy perturbation and umbrella sampling for reactions involving different isotopes. The use of FEP with respect to different masses results in relatively small statistical uncertainties, whereas if KIEs are computed directly by the difference in free energies obtained from the quantum mechanical potentials of mean force for different isotopes, the statistical errors are significantly greater. The PI-FEP/UM method is illustrated in two applications. The first reaction is the decarboxylation of N-methyl picolinate in water, for which the primary (13)C and secondary (15)N KIEs have been determined. The second reaction is the proton-transfer reaction between nitroethane and an acetate ion in water. In both cases, the computational results are in accord with experimental data, and the findings provide further insight into the mechanism of these reactions in water.</td>
</tr>
<tr id="bib_Major2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Major2007,
  author = {Major, Dan and Gao, Jiali},
  title = {An integrated path integral and free-energy perturbation - umbrella sampling method for computing kinetic isotope effects of chemical reactions in solution and in enzymes},
  journal = {Journal of Chemical Theory and Computation},
  year = {2007},
  volume = {3},
  number = {3},
  pages = {949 - 960},
  doi = {https://doi.org/10.1021/ct600371k}
}
</pre></td>
</tr>
<tr id="Major2006a" class="entry">
	<td>Major, D.T., Nam, K. and Gao, J.</td>
	<td>Transition state stabilization and alpha-amino carbon acidity in alanine racemase. <p class="infolinks">[<a href="javascript:toggleInfo('Major2006a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Major2006a','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Journal of the American Chemical Society<br/>Vol. 128(25), pp. 8114 - 5&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ja062272t">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Major2006a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Combined QM/MM simulations have been carried out to investigate the origin of the carbon acidity enhancement in the alanine racemization reaction catalyzed by alanine racemase (AlaR). The present studies show that enhancement of carbon acidity of alpha-amino acids by cofactor pyridoxal 5'-phosphate, PLP, with an unusual, unprotonated pyridine is due to solvation effects, in contrast to the intrinsic electron-withdrawing stabilization by the pyridinium ion to form a quinonoid intermediate. Alanine racemase further lowers the alpha-proton acidity and provides an overall 14-17 kcal/mol transition state stabilization. Our computational results are consistent with the hypothesis that the use of the unusual form of PLP cofactor in AlaR is to raise the free energy of the intermediate, thereby increasing the reprotonation rate and enhancing the enzyme selectivity for racemization.</td>
</tr>
<tr id="bib_Major2006a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Major2006a,
  author = {Major, Dan T and Nam, Kwangho and Gao, Jiali},
  title = {Transition state stabilization and alpha-amino carbon acidity in alanine racemase.},
  journal = {Journal of the American Chemical Society},
  year = {2006},
  volume = {128},
  number = {25},
  pages = {8114 - 5},
  note = {Solvent effects stabilize the carbanion in alanine racemase.},
  doi = {https://doi.org/10.1021/ja062272t}
}
</pre></td>
</tr>
<tr id="Mangel1979" class="entry">
	<td>Mangel, M.</td>
	<td>Small fluctuations in systems with multiple steady states <p class="infolinks">[<a href="javascript:toggleInfo('Mangel1979','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Mangel1979','bibtex')">BibTeX</a>]</p></td>
	<td>1979</td>
	<td>SIAM Journal of Applied Mathematics<br/>Vol. 36(3), pp. 544 - 572&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Mangel1979" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: JSTOR is a not-for-profit service that helps scholars, researchers, and students discover, use, and build upon a wide range of content in a trusted digital archive. We use information technology and tools to increase productivity and facilitate new forms of scholarship. For more information about JSTOR, please contact support@jstor.org. Abstract. We consider the effect of small random perturbations on deterministic syste steady states. The deterministic systems of interest may have two stable nodes and a saddle point. As parameters vary, two or three of the steady states coalesce. This work is concerned with the long time behavior of the system, when it starts near the deterministic separatrix. The separatrix is surrounded by a tube that may contain two stable steady states. The quantity of basic interest is the conditional probability of first exit from the tube through a specified boundary, conditioned on initial position. We use the diffusion approximation, so that the conditional probability satisfies the Kolmogorov backward equation. Formal asymptotic solutions of the backward equation are constructed. The solutions are obtained by a generalized "ray method" and are given in terms of various incomplete special functions. In the appendices, we construct asymptotic solutions of the equation satisfied by the mean value of the first exit time.</td>
</tr>
<tr id="bib_Mangel1979" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mangel1979,
  author = {Mangel, Marc},
  title = {Small fluctuations in systems with multiple steady states},
  journal = {SIAM Journal of Applied Mathematics},
  year = {1979},
  volume = {36},
  number = {3},
  pages = {544 - 572}
}
</pre></td>
</tr>
<tr id="Mangel1977" class="entry">
	<td>Mangel, M. and Ludwig, D.</td>
	<td>Probability of Extinction in a Stochastic Competition <p class="infolinks">[<a href="javascript:toggleInfo('Mangel1977','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Mangel1977','bibtex')">BibTeX</a>]</p></td>
	<td>1977</td>
	<td>SIAM Journal on Applied Mathematics<br/>Vol. 33(2), pp. 256 - 266&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1137/0133015">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Mangel1977" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: When noise is present, the outcome of the competition between two species is no longer a deterministic function of the initial population sizes. An approximate value is given for the probability that a specified species will become extinct, conditioned on the initial population sizes. The calculation involves the asymptotic solution of the Kolmogorov backward equation. The theory is compared with Monte Carlo experiments for the case of flour beetle competition. It is also shown that the theory can be used to describe threshold fluctuations in nerves.</td>
</tr>
<tr id="bib_Mangel1977" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mangel1977,
  author = {Mangel, Marc and Ludwig, Donald},
  title = {Probability of Extinction in a Stochastic Competition},
  journal = {SIAM Journal on Applied Mathematics},
  year = {1977},
  volume = {33},
  number = {2},
  pages = {256 - 266},
  doi = {https://doi.org/10.1137/0133015}
}
</pre></td>
</tr>
<tr id="Mani1970" class="entry">
	<td>Mani, J., Pietruszko, R. and Theorell, H.</td>
	<td>Methanol activity of alcohol dehydrogenases from human liver, horse liver, and yeast <p class="infolinks">[<a href="javascript:toggleInfo('Mani1970','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Mani1970','bibtex')">BibTeX</a>]</p></td>
	<td>1970</td>
	<td>Archives of Biochemistry and Biophysics<br/>Vol. 140(1), pp. 52 - 59&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/0003-9861(70)90009-3">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Mani1970" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Formaldehyde and NADH are formed from methanol and NAD in the reaction catalyzed by horse liver ADH and yeast ADH. In this reaction the above enzymes resemble closely human liver ADH, which has been known for some time to catalyze methanol dehydrogenation. The formation of formaldehyde from melhanol has been followed by chromatography of 2:4-DNPH derivatives of the reaction products and quantitation by the highly specific chromotropic acid method. Kinetic results have shown that turnover numbers ?? sec-1 ?? active site-1 are very similar for all three enzymes. The specificity of all three enzymes with respect to methanol is therefore identical. Human liver ADH, however, has the lowest Km for methanol, and on this basis only may be considered to be the best catalyst of this reaction. ?? 1970.</td>
</tr>
<tr id="bib_Mani1970" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mani1970,
  author = {Mani, Jean and Pietruszko, Regina and Theorell, Hugo},
  title = {Methanol activity of alcohol dehydrogenases from human liver, horse liver, and yeast},
  journal = {Archives of Biochemistry and Biophysics},
  year = {1970},
  volume = {140},
  number = {1},
  pages = {52 - 59},
  doi = {https://doi.org/10.1016/0003-9861(70)90009-3}
}
</pre></td>
</tr>
<tr id="Manuel2015" class="entry">
	<td>Manuel, A.P., Lambert, J. and Woodside, M.T.</td>
	<td>Reconstructing folding energy landscapes from splitting probability analysis of single-molecule trajectories <p class="infolinks">[<a href="javascript:toggleInfo('Manuel2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Proceedings of the National Academy of Sciences<br/>Vol. 112(23), pp. 7183-7188&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Manuel2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Manuel2015,
  author = {Manuel, Ajay P and Lambert, John and Woodside, Michael T},
  title = {Reconstructing folding energy landscapes from splitting probability analysis of single-molecule trajectories},
  journal = {Proceedings of the National Academy of Sciences},
  publisher = {National Acad Sciences},
  year = {2015},
  volume = {112},
  number = {23},
  pages = {7183--7188}
}
</pre></td>
</tr>
<tr id="Maritzen2012" class="entry">
	<td>Maritzen, T., Koo, S.J. and Haucke, V.</td>
	<td>Turning CALM into excitement: AP180 and CALM in endocytosis and disease <p class="infolinks">[<a href="javascript:toggleInfo('Maritzen2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Biology of the Cell<br/>Vol. 104(10), pp. 588-602&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Maritzen2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Maritzen2012,
  author = {Maritzen, Tanja and Koo, Seong Joo and Haucke, Volker},
  title = {Turning CALM into excitement: AP180 and CALM in endocytosis and disease},
  journal = {Biology of the Cell},
  publisher = {Wiley Online Library},
  year = {2012},
  volume = {104},
  number = {10},
  pages = {588--602}
}
</pre></td>
</tr>
<tr id="Marsh2006" class="entry">
	<td>Marsh, M. and Helenius, A.</td>
	<td>Virus entry: Open sesame <p class="infolinks">[<a href="javascript:toggleInfo('Marsh2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Marsh2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Cell<br/>Vol. 124(4), pp. 729 - 740&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cell.2006.02.007">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Marsh2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Detailed information about the replication cycle of viruses and their interactions with host organisms is required to develop strategies to stop them. Cell biology studies, live-cell imaging, and systems biology have started to illuminate the multiple and subtly different pathways that animal viruses use to enter host cells. These insights are revolutionizing our understanding of endocytosis and the movement of vesicles within cells. In addition, such insights reveal new targets for attacking viruses before they can usurp the host-cell machinery for replication. textcopyright2006 Elsevier Inc.</td>
</tr>
<tr id="bib_Marsh2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Marsh2006,
  author = {Marsh, Mark and Helenius, Ari},
  title = {Virus entry: Open sesame},
  journal = {Cell},
  year = {2006},
  volume = {124},
  number = {4},
  pages = {729 - 740},
  doi = {https://doi.org/10.1016/j.cell.2006.02.007}
}
</pre></td>
</tr>
<tr id="Marshall1999" class="entry">
	<td>Marshall, J. and Oberwinkler, J.</td>
	<td>Ultraviolet vision The colourful world of the mantis shrimp <p class="infolinks">[<a href="javascript:toggleInfo('Marshall1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Nature<br/>Vol. 401, pp. 873 - 874&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Marshall1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Marshall1999,
  author = {Marshall, Justin and Oberwinkler, Johannes},
  title = {Ultraviolet vision The colourful world of the mantis shrimp},
  journal = {Nature},
  year = {1999},
  volume = {401},
  pages = {873 - 874}
}
</pre></td>
</tr>
<tr id="Martyna1992" class="entry">
	<td>Martyna, G.J., Klein, M.L. and Tuckerman, M.E.</td>
	<td>Nose--Hoover chains: The canonical ensemble via continuous dynamics <p class="infolinks">[<a href="javascript:toggleInfo('Martyna1992','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Martyna1992','bibtex')">BibTeX</a>]</p></td>
	<td>1992</td>
	<td>J. Chem. Phys.<br/>Vol. 97(1992), pp. 2635 - 2643&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.463940">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Martyna1992" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Nos&eacute; has derived a set of dynamical equations that can be shown to give canonically distributed positions and momenta provided the phase space average can be taken into the trajectory average, i.e., the system is ergodic [S. Nos&eacute;, J. Chem. Phys. 81, 511 (1984), W. G. Hoover, Phys. Rev. A 31, 1695 (1985)]. Unfortunately, the Nos&eacute;--Hoover dynamics is not ergodic for small or stiff systems. Here a modification of the dynamics is proposed which includes not a single thermostat variable but a chain of variables, Nos&eacute;--Hoover chains. The ``new'' dynamics gives the canonical distribution where the simple formalism fails. In addition, the new method is easier to use than an extension [D. Kusnezov, A. Bulgac, and W. Bauer, Ann. Phys. 204, 155 (1990)] which also gives the canonical distribution for stiff cases.</td>
</tr>
<tr id="bib_Martyna1992" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Martyna1992,
  author = {Martyna, Glenn J and Klein, Michael L and Tuckerman, Mark E},
  title = {Nose--Hoover chains: The canonical ensemble via continuous dynamics},
  journal = {J. Chem. Phys.},
  year = {1992},
  volume = {97},
  number = {1992},
  pages = {2635 - 2643},
  doi = {https://doi.org/10.1063/1.463940}
}
</pre></td>
</tr>
<tr id="Martyna1994" class="entry">
	<td>Martyna, G.J., Tobias, D.J. and Klein, M.L.</td>
	<td>Constant pressure molecular dynamics algorithms <p class="infolinks">[<a href="javascript:toggleInfo('Martyna1994','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Martyna1994','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td>The Journal of Chemical Physics<br/>Vol. 101(5), pp. 4177 - 4177&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.467468">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Martyna1994" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Modularly invariant equations of motion are derived that generate the isothermal--isobaric ensemble as their phase space averages. Isotropic volume fluctuations and fully flexible simulation cells as well as a hybrid scheme that naturally combines the two motions are considered. The resulting methods are tested on two problems, a particle in a one‐dimensional periodic potential and a spherical model of C60 in the solid/fluid phase.</td>
</tr>
<tr id="bib_Martyna1994" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Martyna1994,
  author = {Martyna, Glenn J and Tobias, Douglas J and Klein, Michael L},
  title = {Constant pressure molecular dynamics algorithms},
  journal = {The Journal of Chemical Physics},
  year = {1994},
  volume = {101},
  number = {5},
  pages = {4177 - 4177},
  note = {MTK method for NPT systems},
  doi = {https://doi.org/10.1063/1.467468}
}
</pre></td>
</tr>
<tr id="Martyna1995" class="entry">
	<td>Martyna, G.J. and Tuckerman, M.E.</td>
	<td>Symplectic reversible integrators: Predictor--corrector methods <p class="infolinks">[<a href="javascript:toggleInfo('Martyna1995','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Martyna1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>J. Chem. Phys. ‏<br/>Vol. 102(20), pp. 8071 - 8071&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.469006">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Martyna1995" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A new fourth order predictor--corrector integration scheme is presented.\textbackslash\nThe unique feature of the\textbackslash\n\textbackslash\nnew algorithm and what distinguishes it from a Gear predictor--corrector\textbackslash\nis that the method is\textbackslash\n\textbackslash\nderived from the Trotter decomposition of a specially formulated evolution\textbackslash\noperator and as such, is\textbackslash\n\textbackslash\nboth symplectic and reversible. In addition, the method retains the\textbackslash\nuseful property of Gear methods\textbackslash\n\textbackslash\nthat only one force evaluation per time step is required. The new\textbackslash\nintegrator is tested on a harmonic\textbackslash\n\textbackslash\nplus quartic oscillator and the Henon--Heiles system. Comparisons are\textbackslash\nmade to the second order\textbackslash\n\textbackslash\nvelocity Verlet integrator, the true fourth order Yoshida/Suzuki schemes\textbackslash\nand fourth order Gear. In all\textbackslash\n\textbackslash\ncases, the new method works well, giving energy conservation and trajectories\textbackslash\nof much better\textbackslash\n\textbackslash\nquality than velocity Verlet and of comparable quality to the results\textbackslash\nof the true fourth order schemes\textbackslash\n\textbackslash\nfor the same computational cost as velocity Verlet.</td>
</tr>
<tr id="bib_Martyna1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Martyna1995,
  author = {Martyna, Glenn J and Tuckerman, Mark E},
  title = {Symplectic reversible integrators: Predictor--corrector methods},
  journal = {J. Chem. Phys. ‏},
  year = {1995},
  volume = {102},
  number = {20},
  pages = {8071 - 8071},
  doi = {https://doi.org/10.1063/1.469006}
}
</pre></td>
</tr>
<tr id="Martyna1996" class="entry">
	<td>Martyna, G.J., Tuckerman, M.E., Tobias, D.J. and Klein, M.L.</td>
	<td>Explicit reversible integrators for extended systems dynamics <p class="infolinks">[<a href="javascript:toggleInfo('Martyna1996','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Martyna1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>Molecular Physics<br/>Vol. 87(5), pp. 1117 - 1157&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1080/00268979600100761">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Martyna1996" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Explicit reversible integrators, suitable for use in large-scale computer simulations, are derived for extended systems generating the canonical and isothermal-isobaric ensembles. The new methods are compared with the standard implicit (iterative) integrators on some illustrative example problems. In addition, modification of the proposed algorithms for multiple time step integration is outlined.</td>
</tr>
<tr id="bib_Martyna1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Martyna1996,
  author = {Martyna, Glenn J and Tuckerman, Mark E and Tobias, Douglas J and Klein, Michael L},
  title = {Explicit reversible integrators for extended systems dynamics},
  journal = {Molecular Physics},
  year = {1996},
  volume = {87},
  number = {5},
  pages = {1117 - 1157},
  note = {NULL},
  doi = {https://doi.org/10.1080/00268979600100761}
}
</pre></td>
</tr>
<tr id="Mascarenhas2018" class="entry">
	<td>Mascarenhas, N.M., Terse, V.L. and Gosavi, S.</td>
	<td>Intrinsic Disorder in a Well-Folded Globular Protein <p class="infolinks">[<a href="javascript:toggleInfo('Mascarenhas2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>The Journal of Physical Chemistry B&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Mascarenhas2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mascarenhas2018,
  author = {Mascarenhas, Nahren Manuel and Terse, Vishram L and Gosavi, Shachi},
  title = {Intrinsic Disorder in a Well-Folded Globular Protein},
  journal = {The Journal of Physical Chemistry B},
  publisher = {ACS Publications},
  year = {2018}
}
</pre></td>
</tr>
<tr id="Maslov2002" class="entry">
	<td>Maslov, S. and Sneppen, K.</td>
	<td>Specificity and Stability in Topology of Protein Networks <p class="infolinks">[<a href="javascript:toggleInfo('Maslov2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Maslov2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Science<br/>Vol. 296(5569), pp. 910 - 913&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1126/science.1065103">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Maslov2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Molecular networks guide the biochemistry of a living cell on multiple levels: Its metabolic and signaling pathways are shaped by the network of interacting proteins, whose production, in turn, is controlled by the genetic regulatory network. To address topological properties of these two networks, we quantified correlations between connectivities of interacting nodes and compared them to a null model of a network, in which all links were randomly rewired. We found that for both interaction and regulatory networks, links between highly connected proteins are systematically suppressed, whereas those between a highly connected and low-connected pairs of proteins are favored. This effect decreases the likelihood of cross talk between different functional modules of the cell and increases the overall robustness of a network by localizing effects of deleterious perturbations.</td>
</tr>
<tr id="bib_Maslov2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Maslov2002,
  author = {Maslov, Sergei and Sneppen, Kim},
  title = {Specificity and Stability in Topology of Protein Networks},
  journal = {Science},
  year = {2002},
  volume = {296},
  number = {5569},
  pages = {910 - 913},
  doi = {https://doi.org/10.1126/science.1065103}
}
</pre></td>
</tr>
<tr id="Masterson2013" class="entry">
	<td>Masterson, J.E. and Schwartz, S.D.</td>
	<td>Changes in protein architecture and subpicosecond protein dynamics impact the reaction catalyzed by lactate dehydrogenase. <p class="infolinks">[<a href="javascript:toggleInfo('Masterson2013','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Masterson2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Journal of Physical Chemistry A<br/>Vol. 117(32), pp. 7107 - 7113&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp400376h">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Masterson2013" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We have previously established the importance of a promoting vibration, a subpicosecond protein motion that propagates through a specific axis of residues, in the reaction coordinate of lactate dehydrogenase (LDH). To test the effect that perturbation of this motion would have on the enzymatic reaction, we employ transition path sampling to obtain transition path ensembles for four independent LDH enzymatic systems: the wild type enzyme, a version of the enzyme expressing heavy isotopic substitution, and two enzymes with mutations in the promoting vibration axis. We show that even slight changes in the promoting vibration of LDH result in dramatic changes in enzymatic chemistry. In the "heavy" version of the enzyme, we find that the dampening of the subpicosecond dynamics from heavy isotopic substitution leads to a drastic increase in the time of barrier crossing. Furthermore, we see that mutation of the promoting vibration axis causes a decrease in the variability of transition paths available to the enzymatic reaction. The combined results reveal the importance of the protein architecture of LDH in enzymatic catalysis by establishing how the promoting vibration is finely tuned to facilitate chemistry.</td>
</tr>
<tr id="bib_Masterson2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Masterson2013,
  author = {Masterson, Jean E and Schwartz, Steven D},
  title = {Changes in protein architecture and subpicosecond protein dynamics impact the reaction catalyzed by lactate dehydrogenase.},
  journal = {Journal of Physical Chemistry A},
  year = {2013},
  volume = {117},
  number = {32},
  pages = {7107 - 7113},
  note = {NULL},
  doi = {https://doi.org/10.1021/jp400376h}
}
</pre></td>
</tr>
<tr id="Masterson2014" class="entry">
	<td>Masterson, J.E. and Schwartz, S.D.</td>
	<td>The enzymatic reaction catalyzed by lactate dehydrogenase exhibits one dominant reaction path. <p class="infolinks">[<a href="javascript:toggleInfo('Masterson2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Masterson2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Chemical Physics<br/>Vol. 442(17), pp. 132 - 136&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.chemphys.2014.02.018">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Masterson2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Enzymes are the most efficient chemical catalysts known, but the exact nature of chemical barrier crossing in enzymes is not fully understood. Application of transition state theory to enzymatic reactions indicates that the rates of all possible reaction paths, weighted by their relative probabilities, must be considered in order to achieve an accurate calculation of the overall rate. Previous studies in our group have shown a single mechanism for enzymatic barrier passage in human heart lactate dehydrogenase (LDH). To ensure that this result was not due to our methodology insufficiently sampling reactive phase space, we implement high-perturbation transition path sampling in both microcanonical and canonical regimes for the reaction catalyzed by human heart LDH. We find that, although multiple, distinct paths through reactive phase space are possible for this enzymatic reaction, one specific reaction path is dominant. Since the frequency of these paths in a canonical ensemble is inversely proportional to the free energy barriers separating them from other regions of phase space, we conclude that the rarer reaction paths are likely to have a negligible contribution. Furthermore, the non-dominate reaction paths correspond to altered reactive conformations and only occur after multiple steps of high perturbation, suggesting that these paths may be the result of non-biologically significant changes to the structure of the enzymatic active site.</td>
</tr>
<tr id="bib_Masterson2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Masterson2014,
  author = {Masterson, Jean E and Schwartz, Steven D},
  title = {The enzymatic reaction catalyzed by lactate dehydrogenase exhibits one dominant reaction path.},
  journal = {Chemical Physics},
  year = {2014},
  volume = {442},
  number = {17},
  pages = {132 - 136},
  note = {NULL},
  doi = {https://doi.org/10.1016/j.chemphys.2014.02.018}
}
</pre></td>
</tr>
<tr id="Masterson2015" class="entry">
	<td>Masterson, J.E. and Schwartz, S.D.</td>
	<td>Evolution alters the enzymatic reaction coordinate of dihydrofolate reductase <p class="infolinks">[<a href="javascript:toggleInfo('Masterson2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Masterson2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 119(3), pp. 989 - 996&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp506373q">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Masterson2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: How evolution has affected enzyme function is a topic of great interest in the field of biophysical chemistry. Evolutionary changes from Escherichia coli dihydrofolate reductase (ecDHFR) to human dihydrofolate reductase (hsDHFR) have resulted in increased catalytic efficiency and an altered dynamic landscape in the human enzyme. Here, we show that a subpicosecond protein motion is dynamically coupled to hydride transfer catalyzed by hsDHFR but not ecDHFR. This motion propagates through residues that correspond to mutational events along the evolutionary path from ecDHFR to hsDHFR. We observe an increase in the variability of the transition states, reactive conformations, and times of barrier crossing in the human system. In the hsDHFR active site, we detect structural changes that have enabled the coupling of fast protein dynamics to the reaction coordinate. These results indicate a shift in the DHFR family to a form of catalysis that incorporates rapid protein dynamics and a concomitant shift to a more flexible path through reactive phase space.</td>
</tr>
<tr id="bib_Masterson2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Masterson2015,
  author = {Masterson, Jean E and Schwartz, Steven D},
  title = {Evolution alters the enzymatic reaction coordinate of dihydrofolate reductase},
  journal = {Journal of Physical Chemistry B},
  year = {2015},
  volume = {119},
  number = {3},
  pages = {989 - 996},
  note = {NULL},
  doi = {https://doi.org/10.1021/jp506373q}
}
</pre></td>
</tr>
<tr id="Mayers2013" class="entry">
	<td>Mayers, J.R., Wang, L., Pramanik, J., Johnson, A., Sarkeshik, A., Wang, Y., Saengsawang, W., Yates, J.R. and Audhya, A.</td>
	<td>Regulation of ubiquitin-dependent cargo sorting by multiple endocytic adaptors at the plasma membrane. <p class="infolinks">[<a href="javascript:toggleInfo('Mayers2013','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Mayers2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Proceedings of the National Academy of Sciences of the United States of America<br/>Vol. 110, pp. 11857-11862&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.1302918110">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Mayers2013" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Endocytic protein trafficking is directed by sorting signals on cargo molecules that are recognized by cytosolic adaptor proteins. However, the steps necessary to segregate the variety of cargoes during endocytosis remain poorly defined. Using Caenorhabditis elegans, we demonstrate that multiple plasma membrane endocytic adaptors function redundantly to regulate clathrin-mediated endocytosis and to recruit components of the endosomal sorting complex required for transport (ESCRT) machinery to the cell surface to direct the sorting of ubiquitin-modified substrates. Moreover, our data suggest that preassembly of cargoes with the ESCRT-0 complex at the plasma membrane enhances the efficiency of downstream sorting events in the endolysosomal system. In the absence of a heterooligomeric adaptor complex composed of FCHO, Eps15, and intersectin, ESCRT-0 accumulation at the cell surface is diminished, and the degradation of a ubiquitin-modified cargo slows significantly without affecting the rate of its clathrin-mediated internalization. Consistent with a role for the ESCRT machinery during cargo endocytosis, we further show that the ESCRT-0 complex accumulates at a subset of clathrin-coated pits on the surface of human cells. Our findings suggest a unique mechanism by which ubiquitin-modified cargoes are sequestered into the endolysosomal pathway.</td>
</tr>
<tr id="bib_Mayers2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mayers2013,
  author = {Mayers, Jonathan R and Wang, Lei and Pramanik, Jhuma and Johnson, Adam and Sarkeshik, Ali and Wang, Yueju and Saengsawang, Witchuda and Yates, John R and Audhya, Anjon},
  title = {Regulation of ubiquitin-dependent cargo sorting by multiple endocytic adaptors at the plasma membrane.},
  journal = {Proceedings of the National Academy of Sciences of the United States of America},
  year = {2013},
  volume = {110},
  pages = {11857--11862},
  doi = {https://doi.org/10.1073/pnas.1302918110}
}
</pre></td>
</tr>
<tr id="McAdams1997" class="entry">
	<td>McAdams, H. and Arkin, A.P.</td>
	<td>Stochastic mechanisms in gene expression <p class="infolinks">[<a href="javascript:toggleInfo('McAdams1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('McAdams1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Proceedings of the National Acadamy of Sciences<br/>Vol. 94(3), pp. 814 - 819&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.94.3.814">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_McAdams1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In cellular regulatory networks, genetic activity is controlled by molecular signals that determine when and how often a given gene is transcribed. In genetically controlled pathways, the protein product encoded by one gene often regulates expression of other genes. The time delay, after activation of the first promoter, to reach an effective level to control the next promoter depends on the rate of protein accumulation. We have analyzed the chemical reactions controlling transcript initiation and translation termination in a single such "genetically coupled" link as a precursor to modeling networks constructed from many such links. Simulation of the processes of gene expression shows that proteins are produced from an activated promoter in short bursts of variable numbers of proteins that occur at random time intervals. As a result, there can be large differences in the time between successive events in regulatory cascades across a cell population. In addition, the random pattern of expression of competitive effectors can produce probabilistic outcomes in switching mechanisms that select between alternative regulatory paths. The result can be a partitioning of the cell population into different phenotypes as the cells follow different paths. There are numerous unexplained examples of phenotypic variations in isogenic populations of both prokaryotic and eukaryotic cells that may be the result of these stochastic gene expression mechanisms.</td>
</tr>
<tr id="bib_McAdams1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{McAdams1997,
  author = {McAdams, Harley and Arkin, Adam P},
  title = {Stochastic mechanisms in gene expression},
  journal = {Proceedings of the National Acadamy of Sciences},
  year = {1997},
  volume = {94},
  number = {3},
  pages = {814 - 819},
  doi = {https://doi.org/10.1073/pnas.94.3.814}
}
</pre></td>
</tr>
<tr id="McDonald1967" class="entry">
	<td>McDonald, I. and Singer, K.</td>
	<td>Machine Calculation of Thermodynamic Properties of a Simple Fluid at Supercritical Temperatures <p class="infolinks">[<a href="javascript:toggleInfo('McDonald1967','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('McDonald1967','bibtex')">BibTeX</a>]</p></td>
	<td>1967</td>
	<td>Journal of Chemical Physics<br/>Vol. 47(11), pp. 4766 - 4766&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.1701695">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_McDonald1967" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A Monte Carlo method for the evaluation of equilibrium thermodynamic properties of a system of interacting particles is described. The classical configurational partition function at a given density is calculated from the estimated value of the weight function 'Y(\textless\I&raquo;, up to a factor that is independent of temperature; 'Y (\textless\I&raquo; d\textless\l\textgreater\ is the fraction of all accessible configurations of N particles confined to a fixed volume for which the total potential energy has a value between \textless\I\textgreater\ and \textless\I\textgreater\+d\textless\l\textgreater\. The weight function is found to vary approximatelyas (\textless\I\textgreater\\textless\I\textgreater\o)n, where \textless\1\textgreater\0 and n are constants; n increases roughly linearly with density. The method is applied to the calculation of thermodynamic properties of gaseous argon at four densities in the temperature range between -100textdegree and 150textdegreeC. A Lennard-Jones potential function is used and though the agreement with experiment is good there is evidence of inadequacy in the Leonnard-Jones model. All calculations are made for N =32.</td>
</tr>
<tr id="bib_McDonald1967" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{McDonald1967,
  author = {McDonald, IR and Singer, K},
  title = {Machine Calculation of Thermodynamic Properties of a Simple Fluid at Supercritical Temperatures},
  journal = {Journal of Chemical Physics},
  year = {1967},
  volume = {47},
  number = {11},
  pages = {4766 - 4766},
  doi = {https://doi.org/10.1063/1.1701695}
}
</pre></td>
</tr>
<tr id="McMahon2011" class="entry">
	<td>McMahon, H. and Boucrot, E.</td>
	<td>Molecular mechanism and physiological functions of clathrin-mediated endocytosis <p class="infolinks">[<a href="javascript:toggleInfo('McMahon2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('McMahon2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Nature Reviews. Molecular Cell Biology<br/>Vol. 12(8), pp. 517 - 533&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nrm3151">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_McMahon2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis is the endocytic portal into cells through which cargo is packaged into vesicles with the aid of a clathrin coat. It is fundamental to neurotransmission, signal transduction and the regulation of many plasma membrane activities and is thus essential to higher eukaryotic life. Morphological stages of vesicle formation are mirrored by progression through various protein modules (complexes). The process involves the formation of a putative FCH domain only (FCHO) initiation complex, which matures through adaptor protein 2 (AP2)-dependent cargo selection, and subsequent coat building, dynamin-mediated scission and finally auxilin- and heat shock cognate 70 (HSC70)-dependent uncoating. Some modules can be used in other pathways, and additions or substitutions confer cell specificity and adaptability.</td>
</tr>
<tr id="bib_McMahon2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{McMahon2011,
  author = {McMahon, HT and Boucrot, E},
  title = {Molecular mechanism and physiological functions of clathrin-mediated endocytosis},
  journal = {Nature Reviews. Molecular Cell Biology},
  year = {2011},
  volume = {12},
  number = {8},
  pages = {517 - 533},
  note = {- interesting information about over- and underexpression of proteins in human tumours (HIP1R is overexpressed, but not in cervical cancer, e.g. HeLa)},
  doi = {https://doi.org/10.1038/nrm3151}
}
</pre></td>
</tr>
<tr id="McMahon1997" class="entry">
	<td>McMahon, H., Wigge, P. and Smith, C.</td>
	<td>Clathrin interacts specifically with amphiphysin and is displaced by dynamin <p class="infolinks">[<a href="javascript:toggleInfo('McMahon1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('McMahon1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>FEBS Letters<br/>Vol. 4134(2), pp. 319 - 322&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S0014-5793(97)00928-9">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_McMahon1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Amphiphysin is an SH3 domain protein that has been implicated in synaptic vesicle endocytosis. We have recently cloned a second amphiphysin isoform, Amph2 (sequence submitted to GenBank, Y13380). Proteins capable of forming a complex with amphiphysin were isolated from rat brain by using recombinant GST-Amph2 for binding experiments. As well as interacting with dynamin I, the full-length protein bound to a weaker 180-kDa band. Immunoblotting demonstrated this protein to be clathrin. To address whether this is a direct interaction, the clathrin binding to amphiphysin was reconstituted in vitro with purified proteins. The N-terminal domain of Amph2 is sufficient for clathrin binding. Dynamin, which interacts with the SH3 domain of Amph2, displaces clathrin from the N-terminus. We propose a model that may explain how clathrin and dynamin are recruited to non-overlapping sites of the coated pit.</td>
</tr>
<tr id="bib_McMahon1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{McMahon1997,
  author = {McMahon, Harvey and Wigge, Patrick and Smith, Corrin},
  title = {Clathrin interacts specifically with amphiphysin and is displaced by dynamin},
  journal = {FEBS Letters},
  year = {1997},
  volume = {4134},
  number = {2},
  pages = {319 - 322},
  note = {rat brain. amph2 interactions some domain information.},
  doi = {https://doi.org/10.1016/S0014-5793(97)00928-9}
}
</pre></td>
</tr>
<tr id="McMahon1999" class="entry">
	<td>McMahon, H.T.</td>
	<td>Endocytosis: an assembly protein for clathrin cages. <p class="infolinks">[<a href="javascript:toggleInfo('McMahon1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('McMahon1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Current biology : CB<br/>Vol. 9, pp. R332-R335&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_McMahon1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The protein AP180 is known to have clathrin-assembly activity in vitro. AP180 has now been found to be crucial for synaptic vesicle endocytosis and the maintenance of a uniform-size vesicle population in vivo. These results significantly advance our understanding of clathrin-mediated endocytosis in the synapse and elsewhere.</td>
</tr>
<tr id="bib_McMahon1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{McMahon1999,
  author = {McMahon, H T},
  title = {Endocytosis: an assembly protein for clathrin cages.},
  journal = {Current biology : CB},
  year = {1999},
  volume = {9},
  pages = {R332--R335}
}
</pre></td>
</tr>
<tr id="McQuarrie1967" class="entry">
	<td>McQuarrie, D.A.</td>
	<td>Stochastic Approach to Chemical Kinetics <p class="infolinks">[<a href="javascript:toggleInfo('McQuarrie1967','bibtex')">BibTeX</a>]</p></td>
	<td>1967</td>
	<td>Journal of Applied Probability<br/>Vol. 4(3), pp. 413 - 478&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_McQuarrie1967" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{McQuarrie1967,
  author = {McQuarrie, Donald A},
  title = {Stochastic Approach to Chemical Kinetics},
  journal = {Journal of Applied Probability},
  year = {1967},
  volume = {4},
  number = {3},
  pages = {413 - 478}
}
</pre></td>
</tr>
<tr id="Mercadante2017" class="entry">
	<td>Mercadante, D., Wagner, J.A., Aramburu, I.V., Lemke, E.A. and Grater, F.</td>
	<td>Sampling Long- versus Short-Range Interactions Defines the Ability of Force Fields To Reproduce the Dynamics of Intrinsically Disordered Proteins <p class="infolinks">[<a href="javascript:toggleInfo('Mercadante2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of Chemical Theory and Computation&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jctc.7b00143">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Mercadante2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mercadante2017,
  author = {Mercadante, Davide and Wagner, Johannes A and Aramburu, Iker V and Lemke, Edward A and Grater, Frauke},
  title = {Sampling Long- versus Short-Range Interactions Defines the Ability of Force Fields To Reproduce the Dynamics of Intrinsically Disordered Proteins},
  journal = {Journal of Chemical Theory and Computation},
  year = {2017},
  doi = {https://doi.org/10.1021/acs.jctc.7b00143}
}
</pre></td>
</tr>
<tr id="Merrifield2014" class="entry">
	<td>Merrifield, C.J. and Kaksonen, M.</td>
	<td>Endocytic accessory factors and regulation of clathrin-mediated endocytosis <p class="infolinks">[<a href="javascript:toggleInfo('Merrifield2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Merrifield2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Cold Spring Harbor Perspectives in Biology<br/>Vol. 6(11), pp. a016733 - a016733&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1101/cshperspect.a016733">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Merrifield2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Up to 60 different proteins are recruited to the site of clathrin-mediated endocytosis in an ordered sequence. These accessory proteins have roles during all the different stages of clathrin-mediated endocytosis. First, they participate in the initiation of the endocytic event, thereby determining when and where endocytic vesicles are made; later they are involved in the maturation of the clathrin coat, recruitment of specific cargo molecules, bending of the membrane, and finally in scission and uncoating of the nascent vesicle. In addition, many of the accessory components are involved in regulating and coupling the actin cytoskeleton to the endocytic membrane. We will discuss the different accessory components and their various roles. Most of the data comes from studies performed with cultured mammalian cells or yeast cells. The process of endocytosis is well conserved between these different organisms, but there are also many interesting differences that may shed light on the mechanistic principles of endocytosis.</td>
</tr>
<tr id="bib_Merrifield2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Merrifield2014,
  author = {Merrifield, Christien J and Kaksonen, Marko},
  title = {Endocytic accessory factors and regulation of clathrin-mediated endocytosis},
  journal = {Cold Spring Harbor Perspectives in Biology},
  year = {2014},
  volume = {6},
  number = {11},
  pages = {a016733 - a016733},
  doi = {https://doi.org/10.1101/cshperspect.a016733}
}
</pre></td>
</tr>
<tr id="Metropolis1953" class="entry">
	<td>Metropolis, N., Rosenbluth, A.W., Rosenbluth, M.N., Teller, A.H. and Teller, E.</td>
	<td>Equation of State by Fast Computing Machines <p class="infolinks">[<a href="javascript:toggleInfo('Metropolis1953','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Metropolis1953','bibtex')">BibTeX</a>]</p></td>
	<td>1953</td>
	<td>The Journal of Chemical Physics<br/>Vol. 21(6), pp. 1087 - 1092&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.1699114">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Metropolis1953" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A general method, suitable for fast computing machines, for investigating such properties as equations of state for substances consisting of interacting individual molecules is described. The method consists of a modified Monte Carlo integration over configuration space. Results for the two‐dimensional rigid‐sphere system have been obtained on the Los Alamos MANIAC and are presented here. These results are compared to the free volume equation of state and to a four‐term virial coefficient expansion.</td>
</tr>
<tr id="bib_Metropolis1953" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Metropolis1953,
  author = {Metropolis, Nicholas and Rosenbluth, Arianna W and Rosenbluth, Marshall N and Teller, Augusta H and Teller, Edward},
  title = {Equation of State by Fast Computing Machines},
  journal = {The Journal of Chemical Physics},
  year = {1953},
  volume = {21},
  number = {6},
  pages = {1087 - 1092},
  doi = {https://doi.org/10.1063/1.1699114}
}
</pre></td>
</tr>
<tr id="Mettlen2009" class="entry">
	<td>Mettlen, M., Stoeber, M., Loerke, D., Antonescu, C.N., Danuser, G. and Schmid, S.L.</td>
	<td>Endocytic accessory proteins are functionally distinguished by their differential effects on the maturation of clathrin-coated pits. <p class="infolinks">[<a href="javascript:toggleInfo('Mettlen2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Mettlen2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Molecular biology of the cell<br/>Vol. 20, pp. 3251-3260&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1091/mbc.E09-03-0256">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Mettlen2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Diverse cargo molecules (i.e., receptors and ligand/receptor complexes) are taken into the cell by clathrin-mediated endocytosis (CME) utilizing a core machinery consisting of cargo-specific adaptors, clathrin and the GTPase dynamin. Numerous endocytic accessory proteins are also required, but their differential roles and functional hierarchy during CME are not yet understood. Here, we used a combination of quantitative live-cell imaging by total internal reflection fluorescence microscopy (TIR-FM), and decomposition of the lifetime distributions of clathrin-coated pits (CCPs) to measure independent aspects of CCP dynamics, including the turnover of abortive and productive CCP species and their relative contributions. Capitalizing on the sensitivity of this assay, we have examined the effects of specific siRNA-mediated depletion of endocytic accessory proteins on CME progression. Of the 12 endocytic accessory proteins examined, we observed seven qualitatively different phenotypes upon protein depletion. From this data we derive a temporal hierarchy of protein function during early steps of CME. Our results support the idea that a subset of accessory proteins, which mediate coat assembly, membrane curvature, and cargo selection, can provide input into an endocytic restriction point/checkpoint mechanism that monitors CCP maturation.</td>
</tr>
<tr id="bib_Mettlen2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mettlen2009,
  author = {Mettlen, Marcel and Stoeber, Miriam and Loerke, Dinah and Antonescu, Costin N and Danuser, Gaudenz and Schmid, Sandra L},
  title = {Endocytic accessory proteins are functionally distinguished by their differential effects on the maturation of clathrin-coated pits.},
  journal = {Molecular biology of the cell},
  year = {2009},
  volume = {20},
  pages = {3251--3260},
  doi = {https://doi.org/10.1091/mbc.E09-03-0256}
}
</pre></td>
</tr>
<tr id="Metzler2001" class="entry">
	<td>Metzler, M., Legendre-Guillemin, V., Gan, L., Chopra, V., Kwok, A., McPherson, P.S. and Hayden, M.R.</td>
	<td>HIP1 functions in clathrin-mediated endocytosis through binding to clathrin and adaptor protein 2 <p class="infolinks">[<a href="javascript:toggleInfo('Metzler2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Journal of Biological Chemistry<br/>Vol. 276(42), pp. 39271-39276&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Metzler2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Metzler2001,
  author = {Metzler, Martina and Legendre-Guillemin, Val&eacute;rie and Gan, Lu and Chopra, Vikramjit and Kwok, Anita and McPherson, Peter S and Hayden, Michael R},
  title = {HIP1 functions in clathrin-mediated endocytosis through binding to clathrin and adaptor protein 2},
  journal = {Journal of Biological Chemistry},
  publisher = {ASBMB},
  year = {2001},
  volume = {276},
  number = {42},
  pages = {39271--39276},
  note = {specific to HIP1, not HIP1R}
}
</pre></td>
</tr>
<tr id="Metzner2009" class="entry">
	<td>Metzner, P., Schutte, C. and Vanden-Eijnden, E.</td>
	<td>Transition Path Theory for Markov Jump Processes <p class="infolinks">[<a href="javascript:toggleInfo('Metzner2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Metzner2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Multiscale Modeling &amp; Simulation<br/>Vol. 7(3), pp. 1192 - 1219&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1137/070699500">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Metzner2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The framework of transition path theory (TPT) is developed in the context of continuous-time Markov chains on discrete state-spaces. Under assumption of ergodicity, TPT singles out any two subsets in the state-space and analyzes the statistical properties of the associated reactive trajectories, i.e., those trajectories by which the random walker transits from one subset to another. TPT gives properties such as the probability distribution of the reactive trajectories, their probability current and flux, and their rate of occurrence and the dominant reaction pathways. In this paper the framework of TPT for Markov chains is developed in detail, and the relation of the theory to electric resistor network theory and data analysis tools such as Laplacian eigenmaps and diffusion maps is discussed as well. Various algorithms for the numerical calculation of the various objects in TPT are also introduced. Finally, the theory and the algorithms are illustrated in several examples.</td>
</tr>
<tr id="bib_Metzner2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Metzner2009,
  author = {Metzner, Philipp and Schutte, Christof and Vanden-Eijnden, Eric},
  title = {Transition Path Theory for Markov Jump Processes},
  journal = {Multiscale Modeling &amp; Simulation},
  year = {2009},
  volume = {7},
  number = {3},
  pages = {1192 - 1219},
  doi = {https://doi.org/10.1137/070699500}
}
</pre></td>
</tr>
<tr id="Meyerholz2005" class="entry">
	<td>Meyerholz, A., Hinrichsen, L., Groos, S., Esk, P., Brandes, G. and Ungewickell, E.J.</td>
	<td>Effect of clathrin assembly lymphoid myeloid leukemia protein depletion on clathrin coat formation <p class="infolinks">[<a href="javascript:toggleInfo('Meyerholz2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Meyerholz2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Traffic<br/>Vol. 6(12), pp. 1225 - 1234&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1111/j.1600-0854.2005.00355.x">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Meyerholz2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The endocytic accessory clathrin assembly lymphoid myeloid leukemia protein (CALM) is the ubiquitously expressed homolog of the neuron-specific protein AP180 that has been implicated in the retrieval of synaptic vesicle. Here, we show that CALM associates with the alpha-appendage domain of the AP2 adaptor via the three peptide motifs 420DPF, 375DIF and 489FESVF and to a lesser extent with the amino-terminal domain of the clathrin heavy chain. Reducing clathrin levels by RNA interference did not significantly affect CALM localization, but depletion of AP2 weakens its association with the plasma membrane. In cells, where CALM levels were reduced by RNA interference, AP2 and clathrin remained organized in somewhat enlarged bright fluorescent puncta. Electron microscopy showed that the depletion of CALM drastically affected the clathrin lattice structure. Round-coated buds, which are the predominant features in control cells, were replaced by irregularly shaped buds and long clathrin-coated tubules. Moreover, we noted an increase in the number of very small cages that formed on flat lattices. Furthermore, we noticed a redistribution of endosomal markers and AP1 in cells that were CALM depleted. Taken together, our findings indicate a critical role for CALM in the regulation and orderly progression of coated bud formation at the plasma membrane.</td>
</tr>
<tr id="bib_Meyerholz2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Meyerholz2005,
  author = {Meyerholz, Anika and Hinrichsen, Lars and Groos, Stephanie and Esk, Peter and Brandes, Gudrun and Ungewickell, Ernst J},
  title = {Effect of clathrin assembly lymphoid myeloid leukemia protein depletion on clathrin coat formation},
  journal = {Traffic},
  year = {2005},
  volume = {6},
  number = {12},
  pages = {1225 - 1234},
  doi = {https://doi.org/10.1111/j.1600-0854.2005.00355.x}
}
</pre></td>
</tr>
<tr id="Miao2014" class="entry">
	<td>Miao, Y., Nichols, S.E. and McCammon, A.</td>
	<td>Free energy landscape of G-protein coupled receptors, explored by accelerated molecular dynamics. <p class="infolinks">[<a href="javascript:toggleInfo('Miao2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Miao2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Physical chemistry chemical physics : PCCP<br/>Vol. 16(14), pp. 6398 - 406&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1039/c3cp53962h">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Miao2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: G-protein coupled receptors (GPCRs) mediate cellular responses to various hormones and neurotransmitters and are important targets for treating a wide spectrum of diseases. They are known to adopt multiple conformational states (e.g., inactive, intermediate and active) during their modulation of various cell signaling pathways. Here, the free energy landscape of GPCRs is explored using accelerated molecular dynamics (aMD) simulations as demonstrated on the M2 muscarinic receptor, a key GPCR that regulates human heart rate and contractile forces of cardiomyocytes. Free energy profiles of important structural motifs that undergo conformational transitions upon GPCR activation and allosteric signaling are analyzed in detail, including the Arg(3.50)-Glu(6.30) ionic lock, the Trp(6.48) toggle switch and the hydrogen interactions between Tyr(5.58)-Tyr(7.53).</td>
</tr>
<tr id="bib_Miao2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Miao2014,
  author = {Miao, Yinglong and Nichols, Sara E and McCammon, Andrew},
  title = {Free energy landscape of G-protein coupled receptors, explored by accelerated molecular dynamics.},
  journal = {Physical chemistry chemical physics : PCCP},
  year = {2014},
  volume = {16},
  number = {14},
  pages = {6398 - 406},
  doi = {https://doi.org/10.1039/c3cp53962h}
}
</pre></td>
</tr>
<tr id="Miao2014a" class="entry">
	<td>Miao, Y., Sinko, W., Pierce, L., Bucher, D., Walker, R.C. and McCammon, A.</td>
	<td>Improved Reweighting of Accelerated Molecular Dynamics Simulations for Free Energy Calculation. <p class="infolinks">[<a href="javascript:toggleInfo('Miao2014a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Miao2014a','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Journal of chemical theory and computation<br/>Vol. 10(7), pp. 2677 - 2689&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ct500090q">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Miao2014a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Accelerated molecular dynamics (aMD) simulations greatly improve the efficiency of conventional molecular dynamics (cMD) for sampling biomolecular conformations, but they require proper reweighting for free energy calculation. In this work, we systematically compare the accuracy of different reweighting algorithms including the exponential average, Maclaurin series, and cumulant expansion on three model systems: alanine dipeptide, chignolin, and Trp-cage. Exponential average reweighting can recover the original free energy profiles easily only when the distribution of the boost potential is narrow (e.g., the range ≤20k BT) as found in dihedral-boost aMD simulation of alanine dipeptide. In dual-boost aMD simulations of the studied systems, exponential average generally leads to high energetic fluctuations, largely due to the fact that the Boltzmann reweighting factors are dominated by a very few high boost potential frames. In comparison, reweighting based on Maclaurin series expansion (equivalent to cumulant expansion on the first order) greatly suppresses the energetic noise but often gives incorrect energy minimum positions and significant errors at the energy barriers (∼2-3k BT). Finally, reweighting using cumulant expansion to the second order is able to recover the most accurate free energy profiles within statistical errors of ∼k BT, particularly when the distribution of the boost potential exhibits low anharmonicity (i.e., near-Gaussian distribution), and should be of wide applicability. A toolkit of Python scripts for aMD reweighting "PyReweighting" is distributed free of charge at http://mccammon.ucsd.edu/computing/amdReweighting/.</td>
</tr>
<tr id="bib_Miao2014a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Miao2014a,
  author = {Miao, Yinglong and Sinko, William and Pierce, Levi and Bucher, Denis and Walker, Ross C and McCammon, Andrew},
  title = {Improved Reweighting of Accelerated Molecular Dynamics Simulations for Free Energy Calculation.},
  journal = {Journal of chemical theory and computation},
  year = {2014},
  volume = {10},
  number = {7},
  pages = {2677 - 2689},
  doi = {https://doi.org/10.1021/ct500090q}
}
</pre></td>
</tr>
<tr id="Michaelis1913" class="entry">
	<td>Michaelis, L. and Menten, M.</td>
	<td>Die Kinetik der Invertinwirkung <p class="infolinks">[<a href="javascript:toggleInfo('Michaelis1913','bibtex')">BibTeX</a>]</p></td>
	<td>1913</td>
	<td>Biochemische Zeitschrift<br/>Vol. 49, pp. 333 - 369&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Michaelis1913" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Michaelis1913,
  author = {Michaelis, L and Menten, M.L},
  title = {Die Kinetik der Invertinwirkung},
  journal = {Biochemische Zeitschrift},
  year = {1913},
  volume = {49},
  pages = {333 - 369}
}
</pre></td>
</tr>
<tr id="Miele2004" class="entry">
	<td>Miele, A.E., Watson, P.J., Evans, P.R., Traub, L.M. and Owen, D.J.</td>
	<td>Two distinct interaction motifs in amphiphysin bind two independent sites on the clathrin terminal domain β-propeller <p class="infolinks">[<a href="javascript:toggleInfo('Miele2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Nature Structural and Molecular Biology<br/>Vol. 11(3), pp. 242-248&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nsmb736">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Miele2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Miele2004,
  author = {Miele, Adriana E and Watson, Peter J and Evans, Philip R and Traub, Linton M and Owen, David J},
  title = {Two distinct interaction motifs in amphiphysin bind two independent sites on the clathrin terminal domain β-propeller},
  journal = {Nature Structural and Molecular Biology},
  year = {2004},
  volume = {11},
  number = {3},
  pages = {242-248},
  note = {dissociation constants. domain information.},
  doi = {https://doi.org/10.1038/nsmb736}
}
</pre></td>
</tr>
<tr id="Miller2015" class="entry">
	<td>Miller, S.E., Mathiasen, S., Bright, N.A., Pierre, F., Kelly, B.T., Kladt, N., Schauss, A., Merrifield, C.J., Stamou, D., Höning, S. and Owen, D.J.</td>
	<td>CALM regulates clathrin-coated vesicle size and maturation by directly sensing and driving membrane curvature. <p class="infolinks">[<a href="javascript:toggleInfo('Miller2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Miller2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Developmental cell<br/>Vol. 33, pp. 163-175&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.devcel.2015.03.002">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Miller2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The size of endocytic clathrin-coated vesicles (CCVs) is remarkably uniform, suggesting that it is optimized to achieve the appropriate levels of cargo and lipid internalization. The three most abundant proteins in mammalian endocytic CCVs are clathrin and the two cargo-selecting, clathrin adaptors, CALM and AP2. Here we demonstrate that depletion of CALM causes a substantial increase in the ratio of "open" clathrin-coated pits (CCPs) to "necked"/"closed" CCVs and a doubling of CCP/CCV diameter, whereas AP2 depletion has opposite effects. Depletion of either adaptor, however, significantly inhibits endocytosis of transferrin and epidermal growth factor. The phenotypic effects of CALM depletion can be rescued by re-expression of wild-type CALM, but not with CALM that lacks a functional N-terminal, membrane-inserting, curvature-sensing/driving amphipathic helix, the existence and properties of which are demonstrated. CALM is thus a major factor in controlling CCV size and maturation and hence in determining the rates of endocytic cargo uptake.</td>
</tr>
<tr id="bib_Miller2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Miller2015,
  author = {Miller, Sharon E and Mathiasen, Signe and Bright, Nicholas A and Pierre, Fabienne and Kelly, Bernard T and Kladt, Nikolay and Schauss, Astrid and Merrifield, Christien J and Stamou, Dimitrios and Höning, Stefan and Owen, David J},
  title = {CALM regulates clathrin-coated vesicle size and maturation by directly sensing and driving membrane curvature.},
  journal = {Developmental cell},
  year = {2015},
  volume = {33},
  pages = {163--175},
  doi = {https://doi.org/10.1016/j.devcel.2015.03.002}
}
</pre></td>
</tr>
<tr id="Miller1974" class="entry">
	<td>Miller, W.</td>
	<td>Quantum mechanical transition state theory and a new semiclassical model for reaction rate constants <p class="infolinks">[<a href="javascript:toggleInfo('Miller1974','bibtex')">BibTeX</a>]</p></td>
	<td>1974</td>
	<td>The Journal of Chemical Physics<br/>Vol. 61(5), pp. 1823 - 1823&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Miller1974" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Miller1974,
  author = {Miller, W.H.},
  title = {Quantum mechanical transition state theory and a new semiclassical model for reaction rate constants},
  journal = {The Journal of Chemical Physics},
  year = {1974},
  volume = {61},
  number = {5},
  pages = {1823 - 1823}
}
</pre></td>
</tr>
<tr id="Miller1975" class="entry">
	<td>Miller, W.H.</td>
	<td>Semiclassical limit of quantum mechanical transition state theory for nonseparable systems <p class="infolinks">[<a href="javascript:toggleInfo('Miller1975','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Miller1975','bibtex')">BibTeX</a>]</p></td>
	<td>1975</td>
	<td>The Journal of Chemical Physics<br/>Vol. 62(5), pp. 1899 - 1906&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.430676">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Miller1975" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The semiclassical limit of quantum mechanical transition state theory is derived by invoking the classical path approximation for the Boltzmann density operator and making use of the stationary phase approximation; separability of motion along a reaction coordinate is not assumed. The resulting expression for the rate constant bears an interesting similarity to that of conventional transition state theory, although all quantities in it refer to the full classical dynamics on the potential energy surface. In place of the vibrational frequencies of the ''activated complex'' which appear in the conventional theory, for example, the semiclassical expression contains characteristic frequencies related to the stability properties of a periodic classical trajectory. Conservation of total angular momentum is easily accounted for in a rigorous manner so that the semiclassical model can be applied to three−dimensional dynamical systems.</td>
</tr>
<tr id="bib_Miller1975" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Miller1975,
  author = {Miller, William H},
  title = {Semiclassical limit of quantum mechanical transition state theory for nonseparable systems},
  journal = {The Journal of Chemical Physics},
  year = {1975},
  volume = {62},
  number = {5},
  pages = {1899 - 1906},
  doi = {https://doi.org/10.1063/1.430676}
}
</pre></td>
</tr>
<tr id="Miller1976" class="entry">
	<td>Miller, W.H.</td>
	<td>Importance of Nonseparability in Quantum Mechanical Transition-State Theory <p class="infolinks">[<a href="javascript:toggleInfo('Miller1976','bibtex')">BibTeX</a>]</p></td>
	<td>1976</td>
	<td>Acc. Chem. Res.<br/>Vol. 9(8), pp. 306 - 312&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ar50104a005">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Miller1976" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Miller1976,
  author = {Miller, William H},
  title = {Importance of Nonseparability in Quantum Mechanical Transition-State Theory},
  journal = {Acc. Chem. Res.},
  year = {1976},
  volume = {9},
  number = {8},
  pages = {306 - 312},
  doi = {https://doi.org/10.1021/ar50104a005}
}
</pre></td>
</tr>
<tr id="Miller1993" class="entry">
	<td>Miller, W.H.</td>
	<td>Beyond transition-state theory: a rigorous quantum theory of chemical reaction rates <p class="infolinks">[<a href="javascript:toggleInfo('Miller1993','bibtex')">BibTeX</a>]</p></td>
	<td>1993</td>
	<td>Accounts of Chemical Research<br/>Vol. 26(4), pp. 174 - 181&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ar00028a007">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Miller1993" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Miller1993,
  author = {Miller, William H},
  title = {Beyond transition-state theory: a rigorous quantum theory of chemical reaction rates},
  journal = {Accounts of Chemical Research},
  year = {1993},
  volume = {26},
  number = {4},
  pages = {174 - 181},
  doi = {https://doi.org/10.1021/ar00028a007}
}
</pre></td>
</tr>
<tr id="Mincer2004" class="entry">
	<td>Mincer, J.S., Nunez, S. and Schwartz, S.D.</td>
	<td>Coupling protein dynamics to reaction center electron density in enzymes: An electronic protein promoting vibration in human purine nucleoside phosphorylase <p class="infolinks">[<a href="javascript:toggleInfo('Mincer2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Mincer2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of Theoretical &amp; Computational Chemistry<br/>Vol. 3(4), pp. 501 - 509&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1142/S0219633604001215">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Mincer2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The notable three oxygen stacking that occurs upon binding of ribonucleoside substrate and phosphate nucleophile by human purine nucleoside phosphorylase (hPNP) enables the coupling of protein dynamic modes to compress this stack, squeezing the ribosyl O4' between ribosyl O5' and the nuclophilic O-P. Created primarily by the motion of active site residue H257, this compression dynamically lowers the barrier height for N9-C1' ribosidic bond cleavage by as much as 20%. As such, this compression constitutes a protein promoting vibration (PPV) (S. Nunez et al.). Presently, we demonstrate charge fluctuations in the ribose and purine components of the ribonucleoside substrate, as well as specifically across the N9-C1' ribosidic bond, that are correlated with the PPV and can explain the decrease in reaction barrier height due to their facilitating cleavage of the ribosidic bond. hPNP apparently employs protein dynamics to push electrons, a finding that suggests that this coupling may be found more generally in enzymes that catalyze substitution and elimination reactions</td>
</tr>
<tr id="bib_Mincer2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mincer2004,
  author = {Mincer, Joshua S and Nunez, S and Schwartz, Steven D},
  title = {Coupling protein dynamics to reaction center electron density in enzymes: An electronic protein promoting vibration in human purine nucleoside phosphorylase},
  journal = {Journal of Theoretical &amp; Computational Chemistry},
  year = {2004},
  volume = {3},
  number = {4},
  pages = {501 - 509},
  doi = {https://doi.org/10.1142/S0219633604001215}
}
</pre></td>
</tr>
<tr id="Mishra2001" class="entry">
	<td>Mishra, S.K., Agostinelli, N.R., Brett, T.J., Mizukami, I., Ross, T.S. and Traub, L.M.</td>
	<td>Clathrin- and AP-2-binding sites in HIP1 uncover a general assembly role for endocytic accessory proteins. <p class="infolinks">[<a href="javascript:toggleInfo('Mishra2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Mishra2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>The Journal of biological chemistry<br/>Vol. 276, pp. 46230-46236&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M108177200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Mishra2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis is a major pathway for the internalization of macromolecules into the cytoplasm of eukaryotic cells. The principle coat components, clathrin and the AP-2 adaptor complex, assemble a polyhedral lattice at plasma membrane bud sites with the aid of several endocytic accessory proteins. Here, we show that huntingtin-interacting protein 1 (HIP1), a binding partner of huntingtin, copurifies with brain clathrin-coated vesicles and associates directly with both AP-2 and clathrin. The discrete interaction sequences within HIP1 that facilitate binding are analogous to motifs present in other accessory proteins, including AP180, amphiphysin, and epsin. Bound to a phosphoinositide-containing membrane surface via an epsin N-terminal homology (ENTH) domain, HIP1 associates with AP-2 to provide coincident clathrin-binding sites that together efficiently recruit clathrin to the bilayer. Our data implicate HIP1 in endocytosis, and the similar modular architecture and function of HIP1, epsin, and AP180 suggest a common role in lipid-regulated clathrin lattice biogenesis.</td>
</tr>
<tr id="bib_Mishra2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mishra2001,
  author = {Mishra, S K and Agostinelli, N R and Brett, T J and Mizukami, I and Ross, T S and Traub, L M},
  title = {Clathrin- and AP-2-binding sites in HIP1 uncover a general assembly role for endocytic accessory proteins.},
  journal = {The Journal of biological chemistry},
  year = {2001},
  volume = {276},
  pages = {46230--46236},
  doi = {https://doi.org/10.1074/jbc.M108177200}
}
</pre></td>
</tr>
<tr id="Mishra2004" class="entry">
	<td>Mishra, S.K., Hawryluk, M.J., Brett, T.J., Keyel, P.A., Dupin, A.L., Jha, A., Heuser, J.E., Fremont, D.H. and Traub, L.M.</td>
	<td>Dual engagement regulation of protein interactions with the AP-2 adaptor alpha appendage. <p class="infolinks">[<a href="javascript:toggleInfo('Mishra2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Mishra2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>The Journal of Biological Chemistry<br/>Vol. 279, pp. 46191-46203&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M408095200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Mishra2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis depends upon the coordinated assembly of a large number of discrete clathrin coat components to couple cargo selection with rapid internalization from the cell surface. Accordingly, the heterotetrameric AP-2 adaptor complex binds not only to clathrin and select cargo molecules, but also to an extensive family of endocytic accessory factors and alternate sorting adaptors. Physical associations between accessory proteins and AP-2 occur primarily through DP(F/W) or FXDXF motifs, which engage an interaction surface positioned on the C-terminal platform subdomain of the independently folded alpha subunit appendage. Here, we find that the WXX(F/W)X(D/E) interaction motif found in several endocytic proteins, including synaptojanin 1, stonin 2, AAK1, GAK, and NECAP1, binds a second interaction site on the bilobal alpha appendage, located on the N-terminal beta sandwich subdomain. Both alpha appendage binding sites can be engaged synchronously, and our data reveal that varied assemblies of interaction motifs with different affinities for two sites upon the alpha appendage can provide a mechanism for temporal ordering of endocytic accessory proteins during clathrin-mediated endocytosis.</td>
</tr>
<tr id="bib_Mishra2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mishra2004,
  author = {Mishra, Sanjay K and Hawryluk, Matthew J and Brett, Tom J and Keyel, Peter A and Dupin, Amie L and Jha, Anupma and Heuser, John E and Fremont, Daved H and Traub, Linton M},
  title = {Dual engagement regulation of protein interactions with the AP-2 adaptor alpha appendage.},
  journal = {The Journal of Biological Chemistry},
  year = {2004},
  volume = {279},
  pages = {46191--46203},
  note = {bindong of synaptojanin 1, stonin 2, AAK1, GAK, and NECAP1 to beta-sandwich of AP2A?},
  doi = {https://doi.org/10.1074/jbc.M408095200}
}
</pre></td>
</tr>
<tr id="Miyauchi2009" class="entry">
	<td>Miyauchi, K., Kim, Y., Latinovic, O., Morozov, V. and Melikyan, G.B.</td>
	<td>HIV enters cells via endocytosis and dynamin-dependent fusion with endosomes. <p class="infolinks">[<a href="javascript:toggleInfo('Miyauchi2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Miyauchi2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Cell<br/>Vol. 137, pp. 433-444&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cell.2009.02.046">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Miyauchi2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Enveloped viruses that rely on a low pH-dependent step for entry initiate infection by fusing with acidic endosomes, whereas the entry sites for pH-independent viruses, such as HIV-1, have not been defined. These viruses have long been assumed to fuse directly with the plasma membrane. Here we used population-based measurements of the viral content delivery into the cytosol and time-resolved imaging of single viruses to demonstrate that complete HIV-1 fusion occurred in endosomes. In contrast, viral fusion with the plasma membrane did not progress beyond the lipid mixing step. HIV-1 underwent receptor-mediated internalization long before endosomal fusion, thus minimizing the surface exposure of conserved viral epitopes during fusion and reducing the efficacy of inhibitors targeting these epitopes. We also show that, strikingly, endosomal fusion is sensitive to a dynamin inhibitor, dynasore. These findings imply that HIV-1 infects cells via endocytosis and envelope glycoprotein- and dynamin-dependent fusion with intracellular compartments.</td>
</tr>
<tr id="bib_Miyauchi2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Miyauchi2009,
  author = {Miyauchi, Kosuke and Kim, Yuri and Latinovic, Olga and Morozov, Vladimir and Melikyan, Gregory B},
  title = {HIV enters cells via endocytosis and dynamin-dependent fusion with endosomes.},
  journal = {Cell},
  year = {2009},
  volume = {137},
  pages = {433--444},
  doi = {https://doi.org/10.1016/j.cell.2009.02.046}
}
</pre></td>
</tr>
<tr id="Monti2013" class="entry">
	<td>Monti, S., Corozzi, A., Fristrup, P., Joshi, K.L., Shin, K., Oelschlaeger, P., Duin, A.CT., Barone, V., Shin, Y., Oelschlaeger, P., van Duin, A.CT., Barone, V., Shin, K., Oelschlaeger, P., Duin, A.CT., Barone, V., Shin, Y., Oelschlaeger, P., van Duin, A.CT. and Barone, V.</td>
	<td>Exploring the conformational and reactive dynamics of biomolecules in solution using an extended version of the glycine reactive force field <p class="infolinks">[<a href="javascript:toggleInfo('Monti2013','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Monti2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Phys. Chem. Chem. Phys.<br/>Vol. 15(36), pp. 15062 - 77&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1039/c3cp51931g">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Monti2013" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In order to describe possible reaction mechanisms involving amino acids, and the evolution of the protonation state of amino acid side chains in solution, a reactive force field (ReaxFF-based description) for peptide and protein simulations has been developed as an expansion of the previously reported glycine parameters. This expansion consists of adding to the training set more than five hundred molecular systems, including all the amino acids and some short peptide structures, which have been investigated by means of quantum mechanical calculations. The performance of this ReaxFF protein force field on a relatively short time scale (500 ps) is validated by comparison with classical non-reactive simulations and experimental data of well characterized test cases, comprising capped amino acids, peptides, and small proteins, and reaction mechanisms connected to the pharmaceutical sector. A good agreement of ReaxFF predicted conformations and kinetics with reference data is obtained.</td>
</tr>
<tr id="bib_Monti2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Monti2013,
  author = {Monti, Susanna and Corozzi, Alessandro and Fristrup, Peter and Joshi, Kaushik L and Shin, Kyung and Oelschlaeger, Peter and Duin, Adri CT and Barone, Vincenzo and Shin, Yun and Oelschlaeger, Peter and van Duin, Adri CT and Barone, Vincenzo and Shin, Kyung and Oelschlaeger, Peter and Duin, Adri CT and Barone, Vincenzo and Shin, Yun and Oelschlaeger, Peter and van Duin, Adri CT and Barone, Vincenzo},
  title = {Exploring the conformational and reactive dynamics of biomolecules in solution using an extended version of the glycine reactive force field},
  journal = {Phys. Chem. Chem. Phys.},
  year = {2013},
  volume = {15},
  number = {36},
  pages = {15062 - 77},
  doi = {https://doi.org/10.1039/c3cp51931g}
}
</pre></td>
</tr>
<tr id="Morgan2000" class="entry">
	<td>Morgan, J.R., Prasad, K., Hao, W., Augustine, G.J. and Lafer, E.M.</td>
	<td>A conserved clathrin assembly motif essential for synaptic vesicle endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Morgan2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Morgan2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>The Journal of neuroscience : the official journal of the Society for Neuroscience<br/>Vol. 20, pp. 8667-8676&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Morgan2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Although clathrin assembly by adaptor proteins (APs) plays a major role in the recycling of synaptic vesicles, the molecular mechanism that allows APs to assemble clathrin is poorly understood. Here we demonstrate that AP180, like AP-2 and AP-3, binds to the N-terminal domain of clathrin. Sequence analysis reveals a motif, containing the sequence DLL, that exists in multiple copies in many clathrin APs. Progressive deletion of these motifs caused a gradual reduction in the ability of AP180 to assemble clathrin in vitro. Peptides from AP180 or AP-2 containing this motif also competitively inhibited clathrin assembly by either protein. Microinjection of these peptides into squid giant presynaptic terminals reversibly blocked synaptic transmission and inhibited synaptic vesicle endocytosis by preventing coated pit formation at the plasma membrane. These results indicate that the DLL motif confers clathrin assembly properties to AP180 and AP-2 and, perhaps, to other APs. We propose that APs promote clathrin assembly by cross-linking clathrin triskelia via multivalent interactions between repeated DLL motifs in the APs and complementary binding sites on the N-terminal domain of clathrin. These results reveal the structural basis for clathrin assembly and provide novel insights into the molecular mechanism of clathrin-mediated synaptic vesicle endocytosis.</td>
</tr>
<tr id="bib_Morgan2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Morgan2000,
  author = {Morgan, J R and Prasad, K and Hao, W and Augustine, G J and Lafer, E M},
  title = {A conserved clathrin assembly motif essential for synaptic vesicle endocytosis.},
  journal = {The Journal of neuroscience : the official journal of the Society for Neuroscience},
  year = {2000},
  volume = {20},
  pages = {8667--8676}
}
</pre></td>
</tr>
<tr id="Morgan2003" class="entry">
	<td>Morgan, J.R., Prasad, K., Jin, S., Augustine, G.J. and Lafer, E.M.</td>
	<td>Eps15 homology domain-NPF motif interactions regulate clathrin coat assembly during synaptic vesicle recycling. <p class="infolinks">[<a href="javascript:toggleInfo('Morgan2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Morgan2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>The Journal of biological chemistry<br/>Vol. 278, pp. 33583-33592&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M304346200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Morgan2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Although genetic and biochemical studies suggest a role for Eps15 homology domain containing proteins in clathrin-mediated endocytosis, the specific functions of these proteins have been elusive. Eps15 is found at the growing edges of clathrin-coated pits, leading to the hypothesis that it participates in the formation of coated vesicles. We have evaluated this hypothesis by examining the effect of Eps15 on clathrin assembly. We found that although Eps15 has no intrinsic ability to assemble clathrin, it potently stimulates the ability of the clathrin adaptor protein, AP180, to assemble clathrin at physiological pH. We have also defined the binding sites for Eps15 on squid AP180. These sites contain an NPF motif, and peptides derived from these binding sites inhibit the ability of Eps15 to stimulate clathrin assembly in vitro. Furthermore, when injected into squid giant presynaptic nerve terminals, these peptides inhibit the formation of clathrin-coated pits and coated vesicles during synaptic vesicle endocytosis. This is consistent with the hypothesis that Eps15 regulates clathrin coat assembly in vivo, and indicates that interactions between Eps15 homology domains and NPF motifs are involved in clathrin-coated vesicle formation during synaptic vesicle recycling.</td>
</tr>
<tr id="bib_Morgan2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Morgan2003,
  author = {Morgan, Jennifer R and Prasad, Kondury and Jin, Suping and Augustine, George J and Lafer, Eileen M},
  title = {Eps15 homology domain-NPF motif interactions regulate clathrin coat assembly during synaptic vesicle recycling.},
  journal = {The Journal of biological chemistry},
  year = {2003},
  volume = {278},
  pages = {33583--33592},
  note = {squid ap180, human eps15. some domain information.},
  doi = {https://doi.org/10.1074/jbc.M304346200}
}
</pre></td>
</tr>
<tr id="Morgan1999" class="entry">
	<td>Morgan, J.R., Zhao, X., Womack, M., Prasad, K., Augustine, G.J. and Lafer, E.M.</td>
	<td>A role for the clathrin assembly domain of AP180 in synaptic vesicle endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Morgan1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Morgan1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>The Journal of neuroscience : the official journal of the Society for Neuroscience<br/>Vol. 19, pp. 10201-10212&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Morgan1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We have used the squid giant synapse to determine whether clathrin assembly by AP180 is important for synaptic vesicle endocytosis. The squid homolog of AP180 encodes a 751 amino acid protein with 40% sequence identity to mouse AP180. Alignment of squid AP180 with other AP180 homologs shows that amino acid identity was highest in the N-terminal inositide-binding domain of the protein and weakest in the C-terminal clathrin assembly domain. Recombinant squid AP180 was able to assemble clathrin in vitro, suggesting a conserved three-dimensional structure that mediates clathrin assembly despite the divergent primary sequence of the C-terminal domain. Microinjection of the C-terminal domains of either mouse or squid AP180 into the giant presynaptic terminal of squid enhanced synaptic transmission. Conversely, a peptide from the C-terminal domain of squid AP180 that inhibited clathrin assembly in vitro completely blocked synaptic transmission when it was injected into the giant presynaptic terminal. This inhibitory effect occurred over a time scale of minutes when the synapse was stimulated at low (0.03 Hz), physiological rates. Electron microscopic analysis revealed several structural changes consistent with the inhibition of synaptic vesicle endocytosis; peptide-injected terminals had far fewer synaptic vesicles, were depleted of coated vesicles, and had a larger plasma membrane perimeter than terminals injected with control solutions. In addition, the remaining synaptic vesicles were significantly larger in diameter. We conclude that the clathrin assembly domain of AP180 is important for synaptic vesicle recycling at physiological rates of activity and that assembly of clathrin by AP180 is necessary for maintaining a pool of releasable synaptic vesicles.</td>
</tr>
<tr id="bib_Morgan1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Morgan1999,
  author = {Morgan, J R and Zhao, X and Womack, M and Prasad, K and Augustine, G J and Lafer, E M},
  title = {A role for the clathrin assembly domain of AP180 in synaptic vesicle endocytosis.},
  journal = {The Journal of neuroscience : the official journal of the Society for Neuroscience},
  year = {1999},
  volume = {19},
  pages = {10201--10212},
  note = {squid ap180}
}
</pre></td>
</tr>
<tr id="Morinaka1999" class="entry">
	<td>Morinaka, K., Koyama, S., Nakashima, S., Hinoi, T., Okawa, K., Iwamatsu, A. and Kikuchi, A.</td>
	<td>Epsin binds to the EH domain of POB1 and regulates receptor-mediated endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Morinaka1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Morinaka1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Oncogene<br/>Vol. 18, pp. 5915-5922&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/sj.onc.1202974">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Morinaka1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: POB1 has been identified as a RalBP1-binding protein and has the Eps15 homology (EH) domain. The EH domain-containing proteins have been suggested to be involved in clathrin-dependent endocytosis. To clarify the function of POB1, we purified a protein which binds to the EH domain of POB1 from bovine brain cytosol and identified it as Epsin, which is known to bind to the EH domain of Eps15. Epsin has three Asn-Pro-Phe (NPF) motifs in the C-terminal region, which are known to form the core sequence for the binding to the EH domain. The EH domain of POB1 interacted directly with the region containing the NPF motifs of Epsin. Expression of Epsin in CHO-IR cells inhibited internalization of insulin although it affected neither insulin-binding nor autophosphorylation activities of the insulin receptor. Taken together with the observations that Epsin is involved in internalization of the receptors for epidermal growth factor and transferrin, these results suggest that Epsin is a binding partner of POB1 and their binding regulates receptor-mediated endocytosis.</td>
</tr>
<tr id="bib_Morinaka1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Morinaka1999,
  author = {Morinaka, K and Koyama, S and Nakashima, S and Hinoi, T and Okawa, K and Iwamatsu, A and Kikuchi, A},
  title = {Epsin binds to the EH domain of POB1 and regulates receptor-mediated endocytosis.},
  journal = {Oncogene},
  year = {1999},
  volume = {18},
  pages = {5915--5922},
  doi = {https://doi.org/10.1038/sj.onc.1202974}
}
</pre></td>
</tr>
<tr id="Moshkanbaryans2014" class="entry">
	<td>Moshkanbaryans, L., Chan, L.-S. and Graham, M.E.</td>
	<td>The biochemical properties and functions of CALM and AP180 in clathrin mediated endocytosis <p class="infolinks">[<a href="javascript:toggleInfo('Moshkanbaryans2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Moshkanbaryans2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td><br/>Vol. 4(3), pp. 388-413&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Moshkanbaryans2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Abstract: Clathrin-mediated endocytosis (CME) is a fundamental process for the regulated internalization of transmembrane cargo and ligands via the formation of vesicles using a clathrin coat. A vesicle coat is initially created at the plasma membrane by clathrin assembly into a lattice, while a specific cargo sorting process selects and concentrates proteins for inclusion in the new vesicle. Vesicles formed via CME traffic to different parts of the cell  ...</td>
</tr>
<tr id="bib_Moshkanbaryans2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Moshkanbaryans2014,
  author = {Moshkanbaryans, Lia and Chan, Ling-Shan and Graham, Mark E},
  title = {The biochemical properties and functions of CALM and AP180 in clathrin mediated endocytosis},
  year = {2014},
  volume = {4},
  number = {3},
  pages = {388-413}
}
</pre></td>
</tr>
<tr id="Moshkanbaryans2016" class="entry">
	<td>Moshkanbaryans, L., Xue, J., Wark, J.R., Robinson, P.J. and Graham, M.E.</td>
	<td>A Novel Sequence in AP180 and CALM Promotes Efficient Clathrin Binding and Assembly. <p class="infolinks">[<a href="javascript:toggleInfo('Moshkanbaryans2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Moshkanbaryans2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>PloS one<br/>Vol. 11, pp. e0162050&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pone.0162050">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Moshkanbaryans2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The clathrin heavy chain N-terminal domain interacts with endocytic adapter proteins via clathrin binding motifs to assemble clathrin triskelia into cages. However, the precise mechanism of clathrin assembly is not yet known. Clathrin assembly protein AP180 has more clathrin binding motifs than any other endocytic protein and has a major role in the assembly of the clathrin coat during synaptic vesicle biogenesis. We now demonstrate that some of the previously identified binding motifs in AP180 may be non-functional and that a non-conventional clathrin binding sequence has a major influence on AP180 function. The related protein, clathrin assembly lymphoid myeloid leukemia protein (CALM), has fewer clathrin binding motifs and functions ubiquitously in clathrin-mediated endocytosis. The C-terminal &nbsp;16 kDa sub-domain in AP180, which has relatively high similarity with CALM, was shown in earlier work to have an unexplained role in clathrin binding. We identified the specific sequences in this sub-domain that bind to clathrin. Evidence for a role for these sequences in promoting clathrin binding was examined using in vitro and ex vivo experiments that compared the clathrin binding ability of site mutants with the wild type sequence. A sequence conserved in both AP180 and CALM (LDSSLA[S/N]LVGNLGI) was found to be the major interaction site and mutation caused a deficit in clathrin assembly, which is the first example of a mutation having this effect. In contrast, single or double mutation of DL(L/F) motifs in full length AP180 had no significant effect on clathrin binding, despite higher clathrin affinity for isolated peptides containing these motifs. We conclude that the novel clathrin interaction sites identified here in CALM and AP180 have a major role in how these proteins interface with clathrin. This work advances the case that AP180 and CALM are required to use a combination of standard clathrin N-terminal domain binding motifs and the sequence identified here for optimal binding and assembling clathrin.</td>
</tr>
<tr id="bib_Moshkanbaryans2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Moshkanbaryans2016,
  author = {Moshkanbaryans, Lia and Xue, Jing and Wark, Jesse Ray and Robinson, Phillip James and Graham, Mark Evan},
  title = {A Novel Sequence in AP180 and CALM Promotes Efficient Clathrin Binding and Assembly.},
  journal = {PloS one},
  year = {2016},
  volume = {11},
  pages = {e0162050},
  doi = {https://doi.org/10.1371/journal.pone.0162050}
}
</pre></td>
</tr>
<tr id="Muenzner2017" class="entry">
	<td>Muenzner, J., Traub, L.M., Kelly, B.T. and Graham, S.C.</td>
	<td>Cellular and viral peptides bind multiple sites on the N-terminal domain of clathrin. <p class="infolinks">[<a href="javascript:toggleInfo('Muenzner2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Muenzner2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Traffic (Copenhagen, Denmark)<br/>Vol. 18, pp. 44-57&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1111/tra.12457">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Muenzner2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Short peptide motifs in unstructured regions of clathrin-adaptor proteins recruit clathrin to membranes to facilitate post-Golgi membrane transport. Three consensus clathrin-binding peptide sequences have been identified and structural studies show that each binds distinct sites on the clathrin heavy chain N-terminal domain (NTD). A fourth binding site for adaptors on NTD has been functionally identified but not structurally characterised. We have solved high resolution structures of NTD bound to peptide motifs from the cellular clathrin adaptors β2 adaptin and amphiphysin plus a putative viral clathrin adaptor, hepatitis D virus large antigen (HDAg-L). Surprisingly, with each peptide we observe simultaneous peptide binding at multiple sites on NTD and viral peptides binding to the same sites as cellular peptides. Peptides containing clathrin-box motifs (CBMs) with the consensus sequence LΦxΦ[DE] bind at the 'arrestin box' on NTD, between β-propeller blades 4 and 5, which had previously been thought to bind a distinct consensus sequence. Further, we structurally define the fourth peptide binding site on NTD, which we term the Royle box. In vitro binding assays show that clathrin is more readily captured by cellular CBMs than by HDAg-L, and site-directed mutagenesis confirms that multiple binding sites on NTD contribute to efficient capture by CBM peptides.</td>
</tr>
<tr id="bib_Muenzner2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Muenzner2017,
  author = {Muenzner, Julia and Traub, Linton M and Kelly, Bernard T and Graham, Stephen C},
  title = {Cellular and viral peptides bind multiple sites on the N-terminal domain of clathrin.},
  journal = {Traffic (Copenhagen, Denmark)},
  year = {2017},
  volume = {18},
  pages = {44--57},
  doi = {https://doi.org/10.1111/tra.12457}
}
</pre></td>
</tr>
<tr id="Mulkearns2012" class="entry">
	<td>Mulkearns, E.E. and Cooper, J.A.</td>
	<td>FCH domain only-2 organizes clathrin-coated structures and interacts with Disabled-2 for low-density lipoprotein receptor endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Mulkearns2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Mulkearns2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Molecular biology of the cell<br/>Vol. 23, pp. 1330-1342&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1091/mbc.E11-09-0812">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Mulkearns2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis regulates the internalization of many nutrient and signaling receptors. Clathrin and endocytic accessory proteins are recruited to receptors by specific adaptors. The adaptor Disabled-2 (Dab2) recruits its cargoes, including the low-density lipoprotein receptor (LDLR), and mediates endocytosis, even when the major adaptor protein AP2 is depleted. We hypothesized that the accessory proteins normally recruited by AP2 may be recruited by Dab2 if AP2 is absent. We identified one such accessory protein, the F-BAR protein FCH domain only-2 (FCHO2), as a major Dab2-interacting protein. The μ-homology domain (μHD) of FCHO2 binds directly to DPF sequences in Dab2 that also bind AP2. Disrupting the Dab2-FCHO2 interaction inhibited Dab2-mediated LDLR endocytosis in AP2-depleted cells. Depleting FCHO2 reduced the number but increased the size of clathrin structures on the adherent surface of HeLa cells and inhibited LDLR and transferrin receptor clustering. However, LDLR was internalized efficiently by FCHO2-deficient cells when additional time was provided for LDLR to enter the enlarged structures before budding, suggesting that later steps of endocytosis are normal under these conditions. These results indicate FCHO2 regulates the size of clathrin structures, and its interaction with Dab2 is needed for LDLR endocytosis under conditions of low AP2.</td>
</tr>
<tr id="bib_Mulkearns2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mulkearns2012,
  author = {Mulkearns, Erin E and Cooper, Jonathan A},
  title = {FCH domain only-2 organizes clathrin-coated structures and interacts with Disabled-2 for low-density lipoprotein receptor endocytosis.},
  journal = {Molecular biology of the cell},
  year = {2012},
  volume = {23},
  pages = {1330--1342},
  doi = {https://doi.org/10.1091/mbc.E11-09-0812}
}
</pre></td>
</tr>
<tr id="Musco2017" class="entry">
	<td>Musco, C., Su, H.-H. and Lynch, N.A.</td>
	<td>Ant-inspired density estimation via random walks <p class="infolinks">[<a href="javascript:toggleInfo('Musco2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Proceedings of the National Academy of Sciences<br/>Vol. 114(40), pp. 10534-10541&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Musco2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Musco2017,
  author = {Musco, Cameron and Su, Hsin-Hao and Lynch, Nancy A},
  title = {Ant-inspired density estimation via random walks},
  journal = {Proceedings of the National Academy of Sciences},
  publisher = {National Acad Sciences},
  year = {2017},
  volume = {114},
  number = {40},
  pages = {10534--10541}
}
</pre></td>
</tr>
<tr id="Mustata2002" class="entry">
	<td>Mustata, G. and Briggs, J.M.</td>
	<td>A structure-based design approach for the identification of novel inhibitors: application to an alanine racemase <p class="infolinks">[<a href="javascript:toggleInfo('Mustata2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Journal of Computer-Aided Molecular Design<br/>Vol. 16(12), pp. 935 - 953&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Mustata2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mustata2002,
  author = {Mustata, Gabriela and Briggs, James M},
  title = {A structure-based design approach for the identification of novel inhibitors: application to an alanine racemase},
  journal = {Journal of Computer-Aided Molecular Design},
  year = {2002},
  volume = {16},
  number = {12},
  pages = {935 - 953}
}
</pre></td>
</tr>
<tr id="Mustata2003" class="entry">
	<td>Mustata, G., Soares, T.A. and Briggs, J.M.</td>
	<td>Molecular Dynamics Studies of Alanine Racemase: A Structural Model for Drug Design <p class="infolinks">[<a href="javascript:toggleInfo('Mustata2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Biopolymers<br/>Vol. 70(2), pp. 186 - 200&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Mustata2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mustata2003,
  author = {Mustata, Gabriela and Soares, Thereza A and Briggs, James M},
  title = {Molecular Dynamics Studies of Alanine Racemase: A Structural Model for Drug Design},
  journal = {Biopolymers},
  year = {2003},
  volume = {70},
  number = {2},
  pages = {186 - 200}
}
</pre></td>
</tr>
<tr id="Nagaraj2011" class="entry">
	<td>Nagaraj, N., Wisniewski, J., Geiger, T., Cox, J., Kircher, M., Kelso, J., Paabo, S. and Mann, M.</td>
	<td>Deep proteome and transcriptome mapping of a human cancer cell line <p class="infolinks">[<a href="javascript:toggleInfo('Nagaraj2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Nagaraj2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Molecular Systems Biology<br/>Vol. 7(548), pp. 548 - 548&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/msb.2011.81">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Nagaraj2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: While the number and identity of proteins expressed in a single human cell type is currently unknown, this fundamental question can be addressed by advanced mass spectrometry (MS)-based proteomics. Online liquid chromatography coupled to high-resolution MS and MS/MS yielded 166 420 peptides with unique amino-acid sequence from HeLa cells. These peptides identified 10 255 different human proteins encoded by 9207 human genes, providing a lower limit on the proteome in this cancer cell line. Deep transcriptome sequencing revealed transcripts for nearly all detected proteins. We calculate copy numbers for the expressed proteins and show that the abundances of \textgreater\ 90&#37; of them are within a factor 60 of the median protein expression level. Comparisons of the proteome and the transcriptome, and analysis of protein complex databases and GO categories, suggest that we achieved deep coverage of the functional transcriptome and the proteome of a single cell type.</td>
</tr>
<tr id="bib_Nagaraj2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Nagaraj2011,
  author = {Nagaraj, N and Wisniewski, JR and Geiger, T and Cox, J and Kircher, M and Kelso, J and Paabo, S and Mann, M},
  title = {Deep proteome and transcriptome mapping of a human cancer cell line},
  journal = {Molecular Systems Biology},
  year = {2011},
  volume = {7},
  number = {548},
  pages = {548 - 548},
  doi = {https://doi.org/10.1038/msb.2011.81}
}
</pre></td>
</tr>
<tr id="Nagel2006" class="entry">
	<td>Nagel, Z.D. and Klinman, J.P.</td>
	<td>Tunneling and dynamics in enzymatic hydride transfer <p class="infolinks">[<a href="javascript:toggleInfo('Nagel2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Chemical Reviews<br/>Vol. 106(8), pp. 3095 - 3118&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/cr050301x">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Nagel2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Nagel2006,
  author = {Nagel, Zachary D and Klinman, Judith P},
  title = {Tunneling and dynamics in enzymatic hydride transfer},
  journal = {Chemical Reviews},
  year = {2006},
  volume = {106},
  number = {8},
  pages = {3095 - 3118},
  doi = {https://doi.org/10.1021/cr050301x}
}
</pre></td>
</tr>
<tr id="Nakashima1999" class="entry">
	<td>Nakashima, S., Morinaka, K., Koyama, S., Ikeda, M., Kishida, M., Okawa, K., Iwamatsu, A., Kishida, S. and Kikuchi, A.</td>
	<td>Small G protein Ral and its downstream molecules regulate endocytosis of EGF and insulin receptors. <p class="infolinks">[<a href="javascript:toggleInfo('Nakashima1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Nakashima1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>The EMBO journal<br/>Vol. 18, pp. 3629-3642&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/emboj/18.13.3629">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Nakashima1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The involvement of Ral and its downstream molecules in receptor-mediated endocytosis was examined. Expression of either RalG23V or RalS28N, which are known to be constitutively active and dominantnegative forms, respectively, in A431 cells blocked internalization of epidermal growth factor (EGF). Stable expression of RalG23V or RalS28N in CHO-IR cells also inhibited internalization of insulin. Internalization of EGF and insulin was not affected by full-length RalBP1 which is an effector protein of Ral, but was inhibited by its C-terminal region which binds directly to Ral and POB1. POB1 is a binding protein of RalBP1 and has the Eps15 homology (EH) domain. Deletion mutants of POB1 inhibited internalization of EGF and insulin. However, internalization of transferrin was unaffected by Ral, RalBP1, POB1 and their mutants. Epsin and Eps15 have been reported to be involved in the regulation of endocytosis of the receptors for EGF and transferrin. The EH domain of POB1 bound directly to Epsin and Eps15. Taken together with the observation that EGF and insulin activate Ral, these results suggest that Ral, RalBP1 and POB1 transmit the signal from the receptors to Epsin and Eps15, thereby regulating ligand-dependent receptor-mediated endocytosis.</td>
</tr>
<tr id="bib_Nakashima1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Nakashima1999,
  author = {Nakashima, S and Morinaka, K and Koyama, S and Ikeda, M and Kishida, M and Okawa, K and Iwamatsu, A and Kishida, S and Kikuchi, A},
  title = {Small G protein Ral and its downstream molecules regulate endocytosis of EGF and insulin receptors.},
  journal = {The EMBO journal},
  year = {1999},
  volume = {18},
  pages = {3629--3642},
  doi = {https://doi.org/10.1093/emboj/18.13.3629}
}
</pre></td>
</tr>
<tr id="Naslavsky2007" class="entry">
	<td>Naslavsky, N., Rahajeng, J., Chenavas, S., Sorgen, P.L. and Caplan, S.</td>
	<td>EHD1 and Eps15 interact with phosphatidylinositols via their Eps15 homology domains. <p class="infolinks">[<a href="javascript:toggleInfo('Naslavsky2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Naslavsky2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>The Journal of biological chemistry<br/>Vol. 282, pp. 16612-16622&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M609493200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Naslavsky2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The C-terminal Eps15 homology domain-containing protein, EHD1, regulates the recycling of receptors from the endocytic recycling compartment to the plasma membrane. In cells, EHD1 localizes to tubular and spherical recycling endosomes. To date, the mode by which EHD1 associates with endosomal membranes remains unknown, and it has not been determined whether this interaction is direct or via interacting proteins. Here, we provide evidence demonstrating that EHD1 has the ability to bind directly and preferentially to an array of phospholipids, preferring phosphatidylinositols with a phosphate at position 3. Previous studies have demonstrated that EH domains coordinate calcium binding and interact with proteins containing the tripeptide asparagine-proline-phenylalanine (NPF). Using two-dimensional nuclear magnetic resonance analysis, we now describe a new function for the Eps15 homology (EH) domain of EHD1 and show that it is capable of directly binding phosphatidylinositol moieties. Moreover, we have expanded our studies to include the C-terminal EH domain of EHD4 and the second of the three N-terminal EH domains of Eps15 and demonstrated that phosphatidylinositol binding may be a more general property shared by certain other EH domains. Further studies identified a positively charged lysine residue (Lys-483) localized within the third helix of the EH domain, on the opposite face of the NPF-binding pocket, as being critical for the interaction with the phosphatidylinositols.</td>
</tr>
<tr id="bib_Naslavsky2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Naslavsky2007,
  author = {Naslavsky, Naava and Rahajeng, Juliati and Chenavas, Sylvie and Sorgen, Paul L and Caplan, Steve},
  title = {EHD1 and Eps15 interact with phosphatidylinositols via their Eps15 homology domains.},
  journal = {The Journal of biological chemistry},
  year = {2007},
  volume = {282},
  pages = {16612--16622},
  doi = {https://doi.org/10.1074/jbc.M609493200}
}
</pre></td>
</tr>
<tr id="Naethke1992" class="entry">
	<td>Näthke, I.S., Heuser, J., Lupas, A., Stock, J., Turck, C.W. and Brodsky, F.M.</td>
	<td>Folding and trimerization of clathrin subunits at the triskelion hub <p class="infolinks">[<a href="javascript:toggleInfo('Naethke1992','bibtex')">BibTeX</a>]</p></td>
	<td>1992</td>
	<td>Cell<br/>Vol. 68(5), pp. 899 - 910&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/0092-8674(92)90033-9">DOI</a> <a href="http://www.sciencedirect.com/science/article/pii/0092867492900339">URL</a>&nbsp;</td>
</tr>
<tr id="bib_Naethke1992" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Naethke1992,
  author = {Inke S. Näthke and John Heuser and Andrei Lupas and Jeff Stock and Christoph W. Turck and Frances M. Brodsky},
  title = {Folding and trimerization of clathrin subunits at the triskelion hub},
  journal = {Cell},
  year = {1992},
  volume = {68},
  number = {5},
  pages = {899 - 910},
  note = {clathrin trimerizes using residues around 1488-1587},
  url = {http://www.sciencedirect.com/science/article/pii/0092867492900339},
  doi = {https://doi.org/10.1016/0092-8674(92)90033-9}
}
</pre></td>
</tr>
<tr id="Negelein1937" class="entry">
	<td>Negelein, E. and Wulff, H.-J.</td>
	<td>Diphosphopyridinproteid: Alkohol, Acedaldehyd <p class="infolinks">[<a href="javascript:toggleInfo('Negelein1937','bibtex')">BibTeX</a>]</p></td>
	<td>1937</td>
	<td>Biochemische Zeitschrift<br/>Vol. 293, pp. 351 - 389&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Negelein1937" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Negelein1937,
  author = {Negelein, Erwin and Wulff, Hans-Joachim},
  title = {Diphosphopyridinproteid: Alkohol, Acedaldehyd},
  journal = {Biochemische Zeitschrift},
  year = {1937},
  volume = {293},
  pages = {351 - 389}
}
</pre></td>
</tr>
<tr id="Nelson2005" class="entry">
	<td>Nelson, D.L. and Cox, M.M.</td>
	<td>Lehninger Principles of Biochemistry <p class="infolinks">[<a href="javascript:toggleInfo('Nelson2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>&nbsp;</td>
	<td>book</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Nelson2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{Nelson2005,
  author = {Nelson, David L and Cox, Michael M},
  title = {Lehninger Principles of Biochemistry},
  publisher = {W.H. Freeman and Company},
  year = {2005},
  edition = {4th}
}
</pre></td>
</tr>
<tr id="Nicholls1992" class="entry">
	<td>Nicholls, D.J., Miller, J., Scawen, M.D., Clarke, A.R., Holbrook, J.J., Atkinson, T. and Goward, C.R.</td>
	<td>The importance of arginine 102 for the substrate specificity of Escherichia coli malate dehydrogenase <p class="infolinks">[<a href="javascript:toggleInfo('Nicholls1992','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Nicholls1992','bibtex')">BibTeX</a>]</p></td>
	<td>1992</td>
	<td>Biochemical and Biophysical Research Communications<br/>Vol. 189(2), pp. 1057 - 1062&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/0006-291X(92)92311-K">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Nicholls1992" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The malate dehydrogenase from Escherichia coli has been specifically altered at a single amino acid residue by using site-directed mutagenesis. The conserved Arg residue at amino acid position 102 in the putative substrate binding site was replaced with a Gln residue. The result was the loss of the high degree of specificity for oxaloacetate. The difference in relative binding energy for oxaloacetate amounted to about 7 kcal/mol and a difference in specificity between oxaloacetate and pyruvate of 8 orders of magnitude between the wild-type and mutant enzymes. These differences may be explained by the large hydration potential of Arg and the formation of a salt bridge with a carboxylate group of oxaloacetate. ?? 1992.</td>
</tr>
<tr id="bib_Nicholls1992" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Nicholls1992,
  author = {Nicholls, David J and Miller, Julie and Scawen, Michael D and Clarke, Anthony R and Holbrook, John J and Atkinson, Tony and Goward, Christopher R},
  title = {The importance of arginine 102 for the substrate specificity of Escherichia coli malate dehydrogenase},
  journal = {Biochemical and Biophysical Research Communications},
  year = {1992},
  volume = {189},
  number = {2},
  pages = {1057 - 1062},
  note = {NULL},
  doi = {https://doi.org/10.1016/0006-291X(92)92311-K}
}
</pre></td>
</tr>
<tr id="Nielsen2016" class="entry">
	<td>Nielsen, J., Hedeholm, R.B., Heinemeier, J., Bushnell, P.G., Christiansen, J.S., Olsen, J., Ramsey, C., Brill, R.W., Simon, M., Steffensen, K.F. and Steffensen, J.F.</td>
	<td>Eye lens radiocarbon reveals centuries of longevity in the Greenland shark (Somniosus microcephalus) <p class="infolinks">[<a href="javascript:toggleInfo('Nielsen2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Science<br/>Vol. 353(6300)&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Nielsen2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Nielsen2016,
  author = {Nielsen, Julius and Hedeholm, Rasmus B and Heinemeier, Jan and Bushnell, Peter G and Christiansen, J&oslash;rgen S and Olsen, Jesper and Ramsey, Christopher and Brill, Richard W and Simon, Malene and Steffensen, Kirstine F and Steffensen, John F},
  title = {Eye lens radiocarbon reveals centuries of longevity in the Greenland shark (Somniosus microcephalus)},
  journal = {Science},
  year = {2016},
  volume = {353},
  number = {6300}
}
</pre></td>
</tr>
<tr id="Niethammer2007" class="entry">
	<td>Niethammer, P., Kronja, I., Stefanie, Kandels-Lewis,., Rybina, S., Bastiaens, P. and Karsenti, E.</td>
	<td>Discrete states of a protein interaction network govern interphase and mitotic microtubule dynamics <p class="infolinks">[<a href="javascript:toggleInfo('Niethammer2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Niethammer2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>PLoS Biology<br/>Vol. 5(2), pp. 0190 - 0202&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pbio.0050029">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Niethammer2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The cytoplasm of eukaryotic cells is thought to adopt discrete "states" corresponding to different steady states of protein networks that govern changes in subcellular organization. For example, in Xenopus eggs, the interphase to mitosis transition is induced solely by activation of cyclin-dependent kinase 1 (CDK1) that phosphorylates many proteins leading to a reorganization of the nucleus and assembly of the mitotic spindle. Among these changes, the large array of stable microtubules that exists in interphase is replaced by short, highly dynamic microtubules in metaphase. Using a new visual immunoprecipitation assay that quantifies pairwise protein interactions in a non-perturbing manner in Xenopus egg extracts, we reveal the existence of a network of interactions between a series of microtubule-associated proteins (MAPs). In interphase, tubulin interacts with XMAP215, which is itself interacting with XKCM1, which connects to APC, EB1, and CLIP170. In mitosis, tubulin interacts with XMAP215, which is connected to EB1. We show that in interphase, microtubules are stable because the catastrophe-promoting activity of XKCM1 is inhibited by its interactions with the other MAPs. In mitosis, microtubules are short and dynamic because XKCM1 is free and has a strong destabilizing activity. In this case, the interaction of XMAP215 with EB1 is required to counteract the strong activity of XKCM1. This provides the beginning of a biochemical description of the notion of "cytoplasmic states" regarding the microtubule system.</td>
</tr>
<tr id="bib_Niethammer2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Niethammer2007,
  author = {Niethammer, Philipp and Kronja, Iva and Stefanie, Kandels-Lewis, and Rybina, Sonja and Bastiaens, Philippe and Karsenti, Eric},
  title = {Discrete states of a protein interaction network govern interphase and mitotic microtubule dynamics},
  journal = {PLoS Biology},
  year = {2007},
  volume = {5},
  number = {2},
  pages = {0190 - 0202},
  doi = {https://doi.org/10.1371/journal.pbio.0050029}
}
</pre></td>
</tr>
<tr id="Nonhebel1927" class="entry">
	<td>Nonhebel, G., Colvin, J., Patterson, H. and Whytlaw-Gray, R.</td>
	<td>The Coagulation of Smokes and the Theory of Smoluchowski <p class="infolinks">[<a href="javascript:toggleInfo('Nonhebel1927','bibtex')">BibTeX</a>]</p></td>
	<td>1927</td>
	<td>Proceedings of the Royal Society A: Mathematical, Physical and Engineering Sciences<br/>Vol. 116(775), pp. 540 - 553&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1098/rspa.1927.0150">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Nonhebel1927" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Nonhebel1927,
  author = {Nonhebel, G and Colvin, J and Patterson, HS and Whytlaw-Gray, R},
  title = {The Coagulation of Smokes and the Theory of Smoluchowski},
  journal = {Proceedings of the Royal Society A: Mathematical, Physical and Engineering Sciences},
  year = {1927},
  volume = {116},
  number = {775},
  pages = {540 - 553},
  doi = {https://doi.org/10.1098/rspa.1927.0150}
}
</pre></td>
</tr>
<tr id="Nooren2003" class="entry">
	<td>Nooren, I. and Thornton, J.M.</td>
	<td>Diversity of protein-protein interactions <p class="infolinks">[<a href="javascript:toggleInfo('Nooren2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Nooren2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>EMBO Journal<br/>Vol. 22(14), pp. 3486 - 3492&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/emboj/cdg359">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Nooren2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In this review, we discuss the structural and functional diversity of protein-protein interactions (PPIs) based primarily on protein families for which three-dimensional structural data are available. PPIs play diverse roles in biology and differ based on the composition, affinity and whether the association is permanent or transient. In vivo, the protomer's localization, concentration and local environment can affect the interaction between protomers and are vital to control the composition and oligomeric state of protein complexes. Since a change in quaternary state is often coupled with biological function or activity, transient PPIs are important biological regulators. Structural characteristics of different types of PPIs are discussed and related to their physiological function, specificity and evolution.</td>
</tr>
<tr id="bib_Nooren2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Nooren2003,
  author = {Nooren, IMA and Thornton, Janet M},
  title = {Diversity of protein-protein interactions},
  journal = {EMBO Journal},
  year = {2003},
  volume = {22},
  number = {14},
  pages = {3486 - 3492},
  doi = {https://doi.org/10.1093/emboj/cdg359}
}
</pre></td>
</tr>
<tr id="Nordbotten2018" class="entry">
	<td>Nordbotten, J.M., Levin, S.A., Szathm&aacute;ry, E. and Stenseth, N.C.</td>
	<td>Ecological and evolutionary dynamics of interconnectedness and modularity <p class="infolinks">[<a href="javascript:toggleInfo('Nordbotten2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Proceedings of the National Academy of Sciences, pp. 201716078&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Nordbotten2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Nordbotten2018,
  author = {Nordbotten, Jan M and Levin, Simon A and Szathm&aacute;ry, E&ouml;rs and Stenseth, Nils C},
  title = {Ecological and evolutionary dynamics of interconnectedness and modularity},
  journal = {Proceedings of the National Academy of Sciences},
  publisher = {National Acad Sciences},
  year = {2018},
  pages = {201716078}
}
</pre></td>
</tr>
<tr id="Northrop1975" class="entry">
	<td>Northrop, D.B.</td>
	<td>Steady-State Analysis of Kinetic Isotope Effects in Enzymic Reactions <p class="infolinks">[<a href="javascript:toggleInfo('Northrop1975','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Northrop1975','bibtex')">BibTeX</a>]</p></td>
	<td>1975</td>
	<td>Biochemistry<br/>Vol. 14(12), pp. 2644 - 2651&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi00683a013">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Northrop1975" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The decrease in the rate of bond cleavage re-sulting from the presence of heavy isotopes is expressed to a greater or lesser extent as an isotope effect on steady-state kinetic parameters of enzyme-catalyzed reactions, depend-ing upon complex relationships between individual rate con-stants. This paper describes these relationships and derives general kinetic expressions which allow the determination of the limits of the relative contribution of different reaction components to V,,, and Vmax/Km. The value of the true isotope effect on a carbon-hydrogen bond breaking step, T h e major reason for determining kinetic isotope effects in enzymology has been to determine whether the maximal velocity is a measure of the rate of the step in which cova-lent change takes place (Jencks, 1969). Such a step is nor-mally identified as the " rate-determining step " of the enzy-matic reaction. The results to date and the steady-state con-cepts of current kinetic theory suggest, however, that the domination of maximal velocities of enzymatic reactions by a single covalent step is a rare event. It now appears likely that the maximal velocity of most enzymes is dependent upon several " rate-contributing'' or " partially rate-limit-ing " steps. In this latter situation, apparent isotope effects have meaning only in terms of a comparison to the true iso-tope effects exerted on the catalytic step of covalent change. However, since true isotope effects could not previously be determined, the usual procedure has been to evaluate ap-parent effects on an absolute scale, based upon an implied comparison with apparent effects for other enzymes and re-sults obtained in chemical reactions. Such an approach ig-nores the possible wide variation of the true isotope effects in enzyme-catalyzed reactions, which are, in addition, prob-ably much greater than the current standards of compari-son.</td>
</tr>
<tr id="bib_Northrop1975" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Northrop1975,
  author = {Northrop, Dexter B},
  title = {Steady-State Analysis of Kinetic Isotope Effects in Enzymic Reactions},
  journal = {Biochemistry},
  year = {1975},
  volume = {14},
  number = {12},
  pages = {2644 - 2651},
  doi = {https://doi.org/10.1021/bi00683a013}
}
</pre></td>
</tr>
<tr id="Northrop1981" class="entry">
	<td>Northrop, D.B.</td>
	<td>Minimal kinetic mechanism and general equation for deuterium isotope effects on enzymic reactions: uncertainty of detecting a rate-limiting step <p class="infolinks">[<a href="javascript:toggleInfo('Northrop1981','bibtex')">BibTeX</a>]</p></td>
	<td>1981</td>
	<td>Biochemistry<br/>Vol. 20(14), pp. 4056 - 4061&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi00517a017">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Northrop1981" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Northrop1981,
  author = {Northrop, Dexter B},
  title = {Minimal kinetic mechanism and general equation for deuterium isotope effects on enzymic reactions: uncertainty of detecting a rate-limiting step},
  journal = {Biochemistry},
  year = {1981},
  volume = {20},
  number = {14},
  pages = {4056 - 4061},
  note = {Kinetic complexity masks KIEs},
  doi = {https://doi.org/10.1021/bi00517a017}
}
</pre></td>
</tr>
<tr id="Nossal2001" class="entry">
	<td>Nossal, R.</td>
	<td>Energetics of clathrin basket assembly. <p class="infolinks">[<a href="javascript:toggleInfo('Nossal2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Nossal2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Traffic (Copenhagen, Denmark)<br/>Vol. 2, pp. 138-147&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Nossal2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A minimal thermodynamic model is used to study the in vitro equilibrium assembly of reconstituted clathrin baskets. The model contains parameters accounting for i) the combined bending and flexing rigidities of triskelion legs and hubs, ii) the intrinsic curvature of an isolated triskelion, and iii) the free energy changes associated with interactions between legs of neighboring triskelions. Analytical expressions for basket size distributions are derived, and published size distribution data (Zaremba S, Keen JH. J Cell Biol 1983;97: 1339-1347) are then used to provide estimates for net total basket assembly energies. Results suggest that energies involved in adding triskelions to partially formed clathrin lattices are small (of the order of kBT), in accord with the notion that lattice remodeling during basket formation occurs as a result of thermodynamic fluctuations. In addition, analysis of data showing the effects of assembly proteins (APs) on basket size indicates that the binding of APs increases the intrinsic curvature of an elemental triskelial subunit, the stabilizing energy of leg interactions, and the effective leg/hub rigidity. Values of effective triskelial rigidity determined in this investigation are similar to those estimated by previous analysis of shape fluctuations of isolated triskelia.</td>
</tr>
<tr id="bib_Nossal2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Nossal2001,
  author = {Nossal, R},
  title = {Energetics of clathrin basket assembly.},
  journal = {Traffic (Copenhagen, Denmark)},
  year = {2001},
  volume = {2},
  pages = {138--147},
  note = {interesting thermo study of clathrin assembly}
}
</pre></td>
</tr>
<tr id="Nunez2004" class="entry">
	<td>Nunez, S., Antoniou, D., Schramm, V.L. and Schwartz, S.D.</td>
	<td>Promoting vibrations in human purine nucleoside phosphorylase. A molecular dynamics and hybrid quantum mechanical/molecular mechanical study. <p class="infolinks">[<a href="javascript:toggleInfo('Nunez2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Nunez2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of the American Chemical Society<br/>Vol. 126(2), pp. 15720 - 15729&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ja0457563">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Nunez2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Crystallographic studies of human purine nucleoside phosphorylase (hPNP) with several transition-state (TS) analogues in the immucillin family showed an unusual geometric arrangement of the atoms O-5', O-4', and O(P), the nucleophilic phosphate oxygen, lying in a close three-oxygen stack. These observations were corroborated by extensive experimental kinetic isotope effect analysis. We propose that protein-facilitated dynamic modes in hPNP cause this stack, centered on the ribosyl O-4' oxygen, to squeeze together and push electrons toward the purine ring, stabilizing the oxacarbenium character of the TS. As the N-ribosidic bond is cleaved during the reaction, the pK(a) values of N-7 and O-6 increase by the electron density expelled by the oxygen-stack compression toward the purine ring. Increased electron density in the purine ring improves electrostatic interactions with nearby residues and facilitates the abstraction of a proton from a solvent proton or an unidentified general acid, making the purine a better leaving group, and accelerating catalysis. Classical and mixed quantum/classical molecular dynamics (MD) simulations of the Michaelis complex of hPNP with the substrates guanosine and phosphate were performed to assess the existence of protein-promoting vibrations (PPVs). Analogous simulations were performed for the substrates in aqueous solution. In the catalytic site, the O-5', O-4', and O(P) oxygens vibrate at frequencies of ca. 125 and 465 cm(-1), as opposed to 285 cm(-1) in the absence of hPNP. The hybrid quantum mechanical/molecular mechanical method was used to assess whether this enzymatic vibration pushing the oxygens together is coupled to the reaction coordinate, and thus has a direct positive impact on catalysis. The potential energy surface for the phosphorolysis reaction for several snapshots taken from the classical MD simulation showed substantial differences in oxygen compression. Our calculations showed the existence of PPVs coupled to the reaction coordinate, which effect electronic alterations in the active site by pushing the three oxygen centers together in proximity, and accelerate substrate turnover in the phosphorolysis reaction catalyzed by hPNP.</td>
</tr>
<tr id="bib_Nunez2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Nunez2004,
  author = {Nunez, Sara and Antoniou, Dimitri and Schramm, Vern L and Schwartz, Steven D},
  title = {Promoting vibrations in human purine nucleoside phosphorylase. A molecular dynamics and hybrid quantum mechanical/molecular mechanical study.},
  journal = {Journal of the American Chemical Society},
  year = {2004},
  volume = {126},
  number = {2},
  pages = {15720 - 15729},
  doi = {https://doi.org/10.1021/ja0457563}
}
</pre></td>
</tr>
<tr id="OBryan2010" class="entry">
	<td>O'Bryan, J.P.</td>
	<td>Intersecting pathways in cell biology <p class="infolinks">[<a href="javascript:toggleInfo('OBryan2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Science signaling<br/>Vol. 3(152), pp. re10&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_OBryan2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{OBryan2010,
  author = {O'Bryan, John P},
  title = {Intersecting pathways in cell biology},
  journal = {Science signaling},
  publisher = {NIH Public Access},
  year = {2010},
  volume = {3},
  number = {152},
  pages = {re10}
}
</pre></td>
</tr>
<tr id="Offenbacher2017" class="entry">
	<td>Offenbacher, A.R., Hu, S., Poss, E.M., Carr, C.AM., Scouras, A.D., Prigozhin, D.M., Iavarone, A.T., Palla, A., Alber, T., Fraser, J.S. and Klinman, J.P.</td>
	<td>Hydrogen--Deuterium Exchange of Lipoxygenase Uncovers a Relationship between Distal, Solvent Exposed Protein Motions and the Thermal Activation Barrier for Catalytic Proton-Coupled Electron Tunneling <p class="infolinks">[<a href="javascript:toggleInfo('Offenbacher2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Offenbacher2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>ACS Central Science<br/>Vol. 3(6), pp. 570 - 579&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acscentsci.7b00142">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Offenbacher2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Defining specific pathways for efficient heat transfer from protein--solvent interfaces to their active sites represents one of the compelling and timely challenges in our quest for a physical description of the origins of enzyme catalysis. Enzymatic hydrogen tunneling reactions constitute excellent systems in which to validate experimental approaches to this important question, given the inherent temperature independence of quantum mechanical wave function overlap. Herein, we present the application of hydrogen--deuterium exchange coupled to mass spectrometry toward the spatial resolution of protein motions that can be related to an enzyme's catalytic parameters. Employing the proton-coupled electron transfer reaction of soybean lipoxygenase as proof of principle, we first corroborate the impact of active site mutations on increased local flexibility and, second, uncover a solvent-exposed loop, 15--34 &Aring; from the reactive ferric center whose temperature-dependent motions are demonstrated to mirror the enthal...</td>
</tr>
<tr id="bib_Offenbacher2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Offenbacher2017,
  author = {Offenbacher, Adam R and Hu, Shenshen and Poss, Erin M and Carr, Cody AM and Scouras, Alexander D and Prigozhin, Daniil M and Iavarone, Anthony T and Palla, Ali and Alber, Tom and Fraser, James S and Klinman, Judith P},
  title = {Hydrogen--Deuterium Exchange of Lipoxygenase Uncovers a Relationship between Distal, Solvent Exposed Protein Motions and the Thermal Activation Barrier for Catalytic Proton-Coupled Electron Tunneling},
  journal = {ACS Central Science},
  year = {2017},
  volume = {3},
  number = {6},
  pages = {570 - 579},
  doi = {https://doi.org/10.1021/acscentsci.7b00142}
}
</pre></td>
</tr>
<tr id="Ohmae2008" class="entry">
	<td>Ohmae, E., Tatsuta, M., Abe, F., Kato, C., Tanaka, N., Kunugi, S. and Gekko, K.</td>
	<td>Effects of pressure on enzyme function of Escherichia coli dihydrofolate reductase <p class="infolinks">[<a href="javascript:toggleInfo('Ohmae2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Biochimica et Biophysica Acta (BBA)-Proteins and Proteomics<br/>Vol. 1784(7-8), pp. 1115-1121&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Ohmae2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ohmae2008,
  author = {Ohmae, Eiji and Tatsuta, Mineyuki and Abe, Fumiyoshi and Kato, Chiaki and Tanaka, Naoki and Kunugi, Shigeru and Gekko, Kunihiko},
  title = {Effects of pressure on enzyme function of Escherichia coli dihydrofolate reductase},
  journal = {Biochimica et Biophysica Acta (BBA)-Proteins and Proteomics},
  publisher = {Elsevier},
  year = {2008},
  volume = {1784},
  number = {7-8},
  pages = {1115--1121}
}
</pre></td>
</tr>
<tr id="Olesen2008" class="entry">
	<td>Olesen, L.E., Ford, M.G.J., Schmid, E.M., Vallis, Y., Babu, M.M., Li, P.H., Mills, I.G., McMahon, H.T. and Praefcke, G.J.K.</td>
	<td>Solitary and repetitive binding motifs for the AP2 complex alpha-appendage in amphiphysin and other accessory proteins. <p class="infolinks">[<a href="javascript:toggleInfo('Olesen2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Olesen2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>The Journal of biological chemistry<br/>Vol. 283, pp. 5099-5109&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M708621200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Olesen2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Adaptor protein (AP) complexes bind to transmembrane proteins destined for internalization and to membrane lipids, so linking cargo to the accessory internalization machinery. This machinery interacts with the appendage domains of APs, which have platform and beta-sandwich subdomains, forming the binding surfaces for interacting proteins. Proteins that interact with the subdomains do so via short motifs, usually found in regions of low structural complexity of the interacting proteins. So far, up to four motifs have been identified that bind to and partially compete for at least two sites on each of the appendage domains of the AP2 complex. Motifs in individual accessory proteins, their sequential arrangement into motif domains, and partial competition for binding sites on the appendage domains coordinate the formation of endocytic complexes in a temporal and spatial manner. In this work, we examine the dominant interaction sequence in amphiphysin, a synapse-enriched accessory protein, which generates membrane curvature and recruits the scission protein dynamin to the necks of coated pits, for the platform subdomain of the alpha-appendage. The motif domain of amphiphysin1 contains one copy of each of a DX(F/W) and FXDXF motif. We find that the FXDXF motif is the main determinant for the high affinity interaction with the alpha-adaptin appendage. We describe the optimal sequence of the FXDXF motif using thermodynamic and structural data and show how sequence variation controls the affinities of these motifs for the alpha-appendage.</td>
</tr>
<tr id="bib_Olesen2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Olesen2008,
  author = {Olesen, Lene E and Ford, Marijn G J and Schmid, Eva M and Vallis, Yvonne and Babu, M Madan and Li, Peter H and Mills, Ian G and McMahon, Harvey T and Praefcke, Gerrit J K},
  title = {Solitary and repetitive binding motifs for the AP2 complex alpha-appendage in amphiphysin and other accessory proteins.},
  journal = {The Journal of biological chemistry},
  year = {2008},
  volume = {283},
  pages = {5099--5109},
  note = {dissociation constants for single DPF binding motifs},
  doi = {https://doi.org/10.1074/jbc.M708621200}
}
</pre></td>
</tr>
<tr id="Oliveira2008" class="entry">
	<td>de Oliveira, C.F., Hamelberg, D. and McCammon, Andrew, J</td>
	<td>Coupling Accelerated Molecular Dynamics Methods with Thermodynamic Integration Simulations. <p class="infolinks">[<a href="javascript:toggleInfo('Oliveira2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Oliveira2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Journal of chemical theory and computation<br/>Vol. 4(9), pp. 1516 - 1525&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ct800160q">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Oliveira2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In this work we propose a straightforward and efficient approach to improve accuracy and convergence of free energy simulations in condensed-phase systems. We also introduce a new accelerated Molecular Dynamics (MD) approach in which molecular conformational transitions are accelerated by lowering the energy barriers while the potential surfaces near the minima are left unchanged. All free energy calculations were performed on the propane-to-propane model system. The accuracy of free energy simulations was significantly improved when sampling of internal degrees of freedom of solute was enhanced. However, accurate and converged results were only achieved when the solvent interactions were taken into account in the accelerated MD approaches. The analysis of the distribution of boost potential along the free energy simulations showed that the new accelerated MD approach samples efficiently both low- and high-energy regions of the potential surface. Since this approach also maintains substantial populations in regions near the minima, the statistics are not compromised in the thermodynamic integration calculations, and, as a result, the ensemble average can be recovered.</td>
</tr>
<tr id="bib_Oliveira2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Oliveira2008,
  author = {de Oliveira, C&eacute;sar F and Hamelberg, Donald and J, McCammon, Andrew},
  title = {Coupling Accelerated Molecular Dynamics Methods with Thermodynamic Integration Simulations.},
  journal = {Journal of chemical theory and computation},
  year = {2008},
  volume = {4},
  number = {9},
  pages = {1516 - 1525},
  note = {Shows that statistical errors arise with reweighting of the modified potential to the real potential.},
  doi = {https://doi.org/10.1021/ct800160q}
}
</pre></td>
</tr>
<tr id="Olsson2006" class="entry">
	<td>Olsson, M., Mavri, J. and Warshel, A.</td>
	<td>Transition state theory can be used in studies of enzyme catalysis: lessons from simulations of tunnelling and dynamical effects in lipoxygenase and other systems <p class="infolinks">[<a href="javascript:toggleInfo('Olsson2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Olsson2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Philosophical Transactions of the Royal Society B: Biological Sciences<br/>Vol. 361(1472), pp. 1417 - 1432&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1098/rstb.2006.1880">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Olsson2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The idea that enzyme catalysis involves special factors such as coherent fluctuations, quantum mechanical tunnelling and non-equilibrium solvation (NES) effects has gained popularity in recent years. It has also been suggested that transition state theory (TST) cannot be used in studies of enzyme catalysis. The present work uses reliable state of the art simulation approaches to examine the above ideas. We start by demonstrating that we are able to simulate any of the present catalytic proposals using the empirical valence bond (EVB) potential energy surfaces, the dispersed polaron model and the quantized classical path (QCP) approach, as well as the approximate vibronic method. These approaches do not treat the catalytic effects by phenomenological treatments and thus can be considered as first principles approaches (at least their ability to compare enzymatic reaction to the corresponding solution reactions). This work will consider the lipoxygenase reaction, and to lesser extent other enzymes, for specific demonstration. It will be pointed out that our study of the lipoxygenase reaction reproduces the very large observed isotope effect and the observed rate constant while obtaining no catalytic contribution from nuclear quantum mechanical (NQM) effects. Furthermore, it will be clarified that our studies established that the NQM effect decreases rather than increases when the donor--acceptor distance is compressed. The consequences of these findings in terms of the temperature dependence of the kinetic isotope effect and in terms of different catalytic proposals will be discussed.This paper will also consider briefly the dynamical effects and conclude that such effects do not contribute in a significant way to enzyme catalysis. Furthermore, it will be pointed out that, in contrast to recent suggestions, NES effects are not dynamical effects and should therefore be part of the activation free energy rather than the transmission factor. In view of findings of the present work and our earlier works, it seems that TST provides a quantitative tool for studies of enzyme catalysis and that the key open questions are related to the nature of the factors that lead to transition state stabilization.</td>
</tr>
<tr id="bib_Olsson2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Olsson2006,
  author = {Olsson, Mats and Mavri, Janez and Warshel, Arieh},
  title = {Transition state theory can be used in studies of enzyme catalysis: lessons from simulations of tunnelling and dynamical effects in lipoxygenase and other systems},
  journal = {Philosophical Transactions of the Royal Society B: Biological Sciences},
  year = {2006},
  volume = {361},
  number = {1472},
  pages = {1417 - 1432},
  doi = {https://doi.org/10.1098/rstb.2006.1880}
}
</pre></td>
</tr>
<tr id="Olusanya2001" class="entry">
	<td>Olusanya, O., Andrews, P.D., Swedlow, J.R. and Smythe, E.</td>
	<td>Phosphorylation of threonine 156 of the mu2 subunit of the AP2 complex is essential for endocytosis in vitro and in vivo. <p class="infolinks">[<a href="javascript:toggleInfo('Olusanya2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Olusanya2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Current biology : CB<br/>Vol. 11, pp. 896-900&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Olusanya2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The clathrin-coated pit is the major port of entry for many receptors and pathogens and is the paradigm for membrane-based sorting events in higher cells [1]. Recently, it has been possible to reconstitute in vitro the events leading to assembly, invagination, and budding off of clathrin-coated vesicles, allowing dissection of the machinery required for sequestration of receptors into these structures [2-6]. The AP2 adaptor complex is a key element of this machinery linking receptors to the coat lattice, and it has previously been reported that AP2 can be phosphorylated both in vitro and in vivo [7-10]. However, the physiological significance of this has never been established. Here, we show that phosphorylation of a single threonine residue (Thr156) of the mu2 subunit of the AP2 complex is essential for efficient endocytosis of transferrin both in an in vitro coated-pit budding assay and in living cells.</td>
</tr>
<tr id="bib_Olusanya2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Olusanya2001,
  author = {Olusanya, O and Andrews, P D and Swedlow, J R and Smythe, E},
  title = {Phosphorylation of threonine 156 of the mu2 subunit of the AP2 complex is essential for endocytosis in vitro and in vivo.},
  journal = {Current biology : CB},
  year = {2001},
  volume = {11},
  pages = {896--900}
}
</pre></td>
</tr>
<tr id="Onsager1938" class="entry">
	<td>Onsager, L.</td>
	<td>Initial recombination of ions <p class="infolinks">[<a href="javascript:toggleInfo('Onsager1938','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Onsager1938','bibtex')">BibTeX</a>]</p></td>
	<td>1938</td>
	<td>Physical Review<br/>Vol. 54(8), pp. 554 - 557&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1103/PhysRev.54.554">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Onsager1938" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The probability that a pair of ions of given initial separation will recombine with each other is computed from the laws of Brownian motion, which is the proper procedure whenever the Langevin factor equals unity, as in gases at high pressures. In the absence of forces other than the Coulomb attraction, the probability of escape equals the reciprocal of the Boltzmann factor. This result includes the correlation between temperature and pressure coefficients of the ionization by light particles previously predicted by Compton, Bennett and Stearns, if one allows their basic hypothesis about the laws which govern the initial separation of the ions. The effect of an electric field is to increase the fraction of escaping ions by a factor which in the incipient stage of the effect is proportional to the field intensity and independent of the initial distance, although it depends on the orientation of an ion pair. The predicted increase of the ionization current is a little more than one percent for every 100 volts/cm, which accounts for the observed effects of fields exceeding 1500 volts/cm. A reasonable amount of columnar recombination would help to explain the proportionately greater effects of weak fields. The inferred initial separations of the ions are apparently compatible with present knowledge of electron scattering and attachment.</td>
</tr>
<tr id="bib_Onsager1938" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Onsager1938,
  author = {Onsager, L},
  title = {Initial recombination of ions},
  journal = {Physical Review},
  year = {1938},
  volume = {54},
  number = {8},
  pages = {554 - 557},
  doi = {https://doi.org/10.1103/PhysRev.54.554}
}
</pre></td>
</tr>
<tr id="Orengo2001" class="entry">
	<td>Orengo, C.A., Sillitoe, I., Reeves, G. and Pearl, FM.</td>
	<td>Review: what can structural classifications reveal about protein evolution? <p class="infolinks">[<a href="javascript:toggleInfo('Orengo2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Orengo2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Journal of Structural Biology<br/>Vol. 134(2-3), pp. 145 - 65&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1006/jsbi.2001.4398">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Orengo2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In this article we present a review of the methods used for comparing and classifying protein structures. We discuss the hierarchies and populations of fold groups and evolutionary families in some of the major classifications and we consider some of the problems confronting any general analyses of structural evolution in protein families. We also review some more recent analyses that have expanded these classifications by identifying sequence relatives in the genomes and thereby reveal interesting trends in fold usage and recurrence.</td>
</tr>
<tr id="bib_Orengo2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Orengo2001,
  author = {Orengo, Christine A and Sillitoe, Ian and Reeves, Gabrielle and Pearl, FM},
  title = {Review: what can structural classifications reveal about protein evolution?},
  journal = {Journal of Structural Biology},
  year = {2001},
  volume = {134},
  number = {2-3},
  pages = {145 - 65},
  doi = {https://doi.org/10.1006/jsbi.2001.4398}
}
</pre></td>
</tr>
<tr id="Overstreet2004" class="entry">
	<td>Overstreet, E., Fitch, E. and Fischer, J.A.</td>
	<td>Fat facets and Liquid facets promote Delta endocytosis and Delta signaling in the signaling cells. <p class="infolinks">[<a href="javascript:toggleInfo('Overstreet2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Overstreet2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Development (Cambridge, England)<br/>Vol. 131, pp. 5355-5366&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1242/dev.01434">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Overstreet2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Endocytosis modulates the Notch signaling pathway in both the signaling and receiving cells. One recent hypothesis is that endocytosis of the ligand Delta by the signaling cells is essential for Notch activation in the receiving cells. Here, we present evidence in strong support of this model. We show that in the developing Drosophila eye Fat facets (Faf), a deubiquitinating enzyme, and its substrate Liquid facets (Lqf), an endocytic epsin, promote Delta internalization and Delta signaling in the signaling cells. We demonstrate that while Lqf is necessary for three different Notch/Delta signaling events at the morphogenetic furrow, Faf is essential only for one: Delta signaling by photoreceptor precluster cells, which prevents recruitment of ectopic neurons. In addition, we show that the ubiquitin-ligase Neuralized (Neur), which ubiquitinates Delta, functions in the signaling cells with Faf and Lqf. The results presented bolster one model for Neur function in which Neur enhances Delta signaling by stimulating Delta internalization in the signaling cells. We propose that Faf plays a role similar to that of Neur in the Delta signaling cells. By deubiquitinating Lqf, which enhances the efficiency of Delta internalization, Faf stimulates Delta signaling.</td>
</tr>
<tr id="bib_Overstreet2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Overstreet2004,
  author = {Overstreet, Erin and Fitch, Erin and Fischer, Janice A},
  title = {Fat facets and Liquid facets promote Delta endocytosis and Delta signaling in the signaling cells.},
  journal = {Development (Cambridge, England)},
  year = {2004},
  volume = {131},
  pages = {5355--5366},
  doi = {https://doi.org/10.1242/dev.01434}
}
</pre></td>
</tr>
<tr id="Owen2000" class="entry">
	<td>Owen, D.J.., Vallis, Y., Pearse, B.M.F.., H.T., McMahon,. and Evans, P.R..</td>
	<td>The structure and function of the β2‐adaptin appendage domain <p class="infolinks">[<a href="javascript:toggleInfo('Owen2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Owen2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>EMBO Journal<br/>Vol. 19(16), pp. 4216-4227&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/emboj/19.16.4216">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Owen2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The heterotetrameric AP2 adaptor (α, β2, μ2 and σ2 subunits) plays a central role in clathrin-mediated endocytosis. We present the protein recruitment function and 1.7 &Aring; resolution structure of its β2-appendage domain to complement those previously determined for the μ2 subunit and α appendage. Using structure-directed mutagenesis, we demonstrate the ability of the β2 appendage alone to bind directly to clathrin and the accessory proteins AP180, epsin and eps15 at the same site. Clathrin polymerization is promoted by binding of clathrin simultaneously to the β2-appendage site and to a second site on the adjacent β2 hinge. This results in the displacement of the other ligands from the β2 appendage. Thus clathrin binding to an AP2--accessory protein complex would cause the controlled release of accessory proteins at sites of vesicle formation.</td>
</tr>
<tr id="bib_Owen2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Owen2000,
  author = {Owen, D.J. and Vallis, Y and Pearse, B.M.F. and H.T., McMahon, and Evans, P.R.},
  title = {The structure and function of the β2‐adaptin appendage domain},
  journal = {EMBO Journal},
  year = {2000},
  volume = {19},
  number = {16},
  pages = {4216-4227},
  note = {human beta 2 subunit. INTERACTIONS: AP2B2 - EPS15, EPN1, AP180. specifically not AMPH.},
  doi = {https://doi.org/10.1093/emboj/19.16.4216}
}
</pre></td>
</tr>
<tr id="Owen2004" class="entry">
	<td>Owen, D.J., Collins, B.M. and Evans, P.R.</td>
	<td>Adaptors for clathrin coats: structure and function. <p class="infolinks">[<a href="javascript:toggleInfo('Owen2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Owen2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Annual review of cell and developmental biology<br/>Vol. 20, pp. 153-191&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1146/annurev.cellbio.20.010403.104543">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Owen2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-coated vesicles (CCVs) are responsible for the transport of proteins between various compartments of the secretory and endocytic systems. Clathrin forms a scaffold around these vesicles that is linked to membranes by clathrin adaptors. The adaptors simultaneously bind to clathrin and to transmembrane proteins and/or phospholipids and can also interact with each other and with other components of the CCV formation machinery. The result is a collection of proteins that can make multiple, moderate strength (microM Kd) interactions and thereby establish the dynamic regulatable networks to drive vesicle genesis at the correct time and place in the cell. This review focuses on the structure of clathrin adaptors and how these structures provide functional information on the mechanism of CCV formation.</td>
</tr>
<tr id="bib_Owen2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Owen2004,
  author = {Owen, David J and Collins, Brett M and Evans, Philip R},
  title = {Adaptors for clathrin coats: structure and function.},
  journal = {Annual review of cell and developmental biology},
  year = {2004},
  volume = {20},
  pages = {153--191},
  doi = {https://doi.org/10.1146/annurev.cellbio.20.010403.104543}
}
</pre></td>
</tr>
<tr id="Owen2001" class="entry">
	<td>Owen, D.J., Setiadi, H., Evans, P.R., McEver, R.P. and Green, S.A.</td>
	<td>A Third Specificity-Determining Site in &mu;2 Adaptin for Sequences Upstream of Yxx&Phi; Sorting Motifs <p class="infolinks">[<a href="javascript:toggleInfo('Owen2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Traffic<br/>Vol. 2(2), pp. 105-110&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Owen2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Owen2001,
  author = {Owen, David J and Setiadi, Hendra and Evans, Philip R and McEver, Rodger P and Green, Samuel A},
  title = {A Third Specificity-Determining Site in &mu;2 Adaptin for Sequences Upstream of Yxx&Phi; Sorting Motifs},
  journal = {Traffic},
  publisher = {Wiley Online Library},
  year = {2001},
  volume = {2},
  number = {2},
  pages = {105--110}
}
</pre></td>
</tr>
<tr id="Owen1999" class="entry">
	<td>Owen, D.J., Vallis, Y., Noble, M.E., Hunter, J.B., Dafforn, T.R., Evans, P.R. and McMahon, H.T.</td>
	<td>A structural explanation for the binding of multiple ligands by the &alpha;-adaptin appendage domain <p class="infolinks">[<a href="javascript:toggleInfo('Owen1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Cell<br/>Vol. 97(6), pp. 805-815&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/s0092-8674(00)80791-6">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Owen1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Owen1999,
  author = {Owen, David J and Vallis, Yvonne and Noble, Martin EM and Hunter, Jack B and Dafforn, Tim R and Evans, Philip R and McMahon, Harvey T},
  title = {A structural explanation for the binding of multiple ligands by the &alpha;-adaptin appendage domain},
  journal = {Cell},
  publisher = {Elsevier},
  year = {1999},
  volume = {97},
  number = {6},
  pages = {805--815},
  note = {Competitive binding of accessory proteins to AP2 alpha subunit. estimated Kd of EPS15.EH, EPS15.DPF.},
  doi = {https://doi.org/10.1016/s0092-8674(00)80791-6}
}
</pre></td>
</tr>
<tr id="Pal2006" class="entry">
	<td>Pal, C., Papp, B. and Lercher, M.J.</td>
	<td>An integrated view of protein evolution <p class="infolinks">[<a href="javascript:toggleInfo('Pal2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Pal2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Nature Reviews Genetics<br/>Vol. 7(5), pp. 337 - 348&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nrg1838">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Pal2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Why do proteins evolve at different rates? Advances in \textbackslash\nsystems biology and genomics have facilitated a move from \textbackslash\nstudying individual proteins to characterizing global \textbackslash\ncellular factors. Systematic surveys indicate that protein \textbackslash\nevolution is not determined exclusively by selection on \textbackslash\nprotein structure and function, but is also affected by the \textbackslash\ngenomic position of the encoding genes, their expression \textbackslash\npatterns, their position in biological networks and possibly \textbackslash\ntheir robustness to mistranslation. Recent work has allowed \textbackslash\ninsights into the relative importance of these factors. We \textbackslash\ndiscuss the status of a much-needed coherent view that \textbackslash\nintegrates studies on protein evolution with biochemistry \textbackslash\nand functional and structural genomics.</td>
</tr>
<tr id="bib_Pal2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Pal2006,
  author = {Pal, Csaba and Papp, Balazs and Lercher, Martin J},
  title = {An integrated view of protein evolution},
  journal = {Nature Reviews Genetics},
  year = {2006},
  volume = {7},
  number = {5},
  pages = {337 - 348},
  doi = {https://doi.org/10.1038/nrg1838}
}
</pre></td>
</tr>
<tr id="Pal2009" class="entry">
	<td>Pal, S., Park, Doo-Hong. and Plapp, B.V.</td>
	<td>Activity of yeast alcohol dehydrogenases on benzyl alcohols and benzaldehydes: characterization of ADH1 from Saccharomyces carlsbergensis and transition state analysis. <p class="infolinks">[<a href="javascript:toggleInfo('Pal2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Pal2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Chemico-biological interactions<br/>Vol. 178(1-3), pp. 16 - 23&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cbi.2008.10.037">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Pal2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The substrate specificities of yeast alcohol dehydrogenases I and II from Saccharomyces cerevisiae (SceADH1 and SceADH2) and Saccharomyces carlsbergensis (ScbADH1) were studied. For this work, the gene for the S. carlsbergensis ADH1 was cloned, sequenced and expressed. The amino acid sequence of ScbADH1 differs at four positions as compared to SceADH1, including substitutions of two glutamine residues with glutamic acid residues, and has the same sequence as the commercial yeast enzyme, which apparently is prepared from S. carlsbergensis. The electrophoretic mobilities of ScbADH1, SceADH2 and commercial ADH are similar. The kinetics and specificities of ScbADH1 and SceADH1 acting on branched, long-chain and benzyl alcohols are very similar, but the catalytic efficiency of SceADH2 is about 10-100-fold higher on these substrates. A three-dimensional structure of SceADH1 shows that the substrate binding pocket has Met-270, whereas SceADH2 has Leu-270, which allows larger substrates to bind. The reduction of a series of p-substituted benzaldehydes catalyzed by SceADH2 is significantly enhanced by electron-withdrawing groups, whereas the oxidation of p-substituted aromatic alcohols may be only slightly affected by the substituents. The substituent effects on catalysis generally reflect the effects on the equilibrium constant for the reaction, where electron-withdrawing substituents favor alcohol. The results are consistent with a transition state that is electronically similar to the alcohol, supporting previous results obtained with commercial yeast ADH.</td>
</tr>
<tr id="bib_Pal2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Pal2009,
  author = {Pal, Suresh and Park, Doo-Hong and Plapp, Bryce V},
  title = {Activity of yeast alcohol dehydrogenases on benzyl alcohols and benzaldehydes: characterization of ADH1 from Saccharomyces carlsbergensis and transition state analysis.},
  journal = {Chemico-biological interactions},
  year = {2009},
  volume = {178},
  number = {1-3},
  pages = {16 - 23},
  note = {Experimental rate constant information for YADH-1},
  doi = {https://doi.org/10.1016/j.cbi.2008.10.037}
}
</pre></td>
</tr>
<tr id="Palazzesi2017" class="entry">
	<td>Palazzesi, F., Valsson, O. and Parrinello, M.</td>
	<td>Conformational Entropy as Collective Variable for Proteins <p class="infolinks">[<a href="javascript:toggleInfo('Palazzesi2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Palazzesi2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>The journal of physical chemistry letters, pp. 4752-4756&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jpclett.7b01770">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Palazzesi2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Many enhanced sampling methods rely on the identification of appropriate collective variables. For proteins, even small ones, finding appropriate descriptors has proven challenging. Here we suggest that the NMR S(2) order parameter can be used to this effect. We trace the validity of this statement to the suggested relation between S(2) and conformational entropy. Using the S(2) order parameter and a surrogate for the protein enthalpy in conjunction with metadynamics or variationally enhanced sampling, we are able to reversibly fold and unfold a small protein and draw its free energy at a fraction of the time that is needed in unbiased simulations. We also use S(2) in combination with the free energy flooding method to compute the unfolding rate of this peptide. We repeat this calculation at different temperatures to obtain the unfolding activation energy.</td>
</tr>
<tr id="bib_Palazzesi2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Palazzesi2017,
  author = {Palazzesi, Ferruccio and Valsson, Omar and Parrinello, Michele},
  title = {Conformational Entropy as Collective Variable for Proteins},
  journal = {The journal of physical chemistry letters},
  year = {2017},
  pages = {4752--4756},
  doi = {https://doi.org/10.1021/acs.jpclett.7b01770}
}
</pre></td>
</tr>
<tr id="Palla2005" class="entry">
	<td>Palla, G., Derenyi, I., Farkas, I. and Vicsek, T.</td>
	<td>Uncovering the overlapping community structure of complex networks in nature and society <p class="infolinks">[<a href="javascript:toggleInfo('Palla2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Palla2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Nature<br/>Vol. 435(7043), pp. 1 - 10&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nature03607">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Palla2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Many complex systems in nature and society can be described in terms of networks capturing the intricate web of connections among the units they are made of. A key question is how to interpret the global organization of such networks as the coexistence of their structural subunits (communities) associated with more highly interconnected parts. Identifying these a priori unknown building blocks (such as functionally related proteins, industrial sectors and groups of people) is crucial to the understanding of the structural and functional properties of networks. The existing deterministic methods used for large networks find separated communities, whereas most of the actual networks are made of highly overlapping cohesive groups of nodes. Here we introduce an approach to analysing the main statistical features of the interwoven sets of overlapping communities that makes a step towards uncovering the modular structure of complex systems. After defining a set of new characteristic quantities for the statistics of communities, we apply an efficient technique for exploring overlapping communities on a large scale. We find that overlaps are significant, and the distributions we introduce reveal universal features of networks. Our studies of collaboration, word-association and protein interaction graphs show that the web of communities has non-trivial correlations and specific scaling properties.</td>
</tr>
<tr id="bib_Palla2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Palla2005,
  author = {Palla, Gergely and Derenyi, Imre and Farkas, Illes and Vicsek, Tamas},
  title = {Uncovering the overlapping community structure of complex networks in nature and society},
  journal = {Nature},
  year = {2005},
  volume = {435},
  number = {7043},
  pages = {1 - 10},
  doi = {https://doi.org/10.1038/nature03607}
}
</pre></td>
</tr>
<tr id="Pan2015" class="entry">
	<td>Pan, X. and Schwartz, S.D.</td>
	<td>Free Energy Surface of the Michaelis Complex of Lactate Dehydrogenase: A Network Analysis of Microsecond Simulations <p class="infolinks">[<a href="javascript:toggleInfo('Pan2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Pan2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 119(17), pp. 5430 - 5436&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jpcb.5b01840">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Pan2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: It has long been recognized that the structure of a protein creates a hierarchy of conformations interconverting on multiple time scales. The conformational hetero- geneity of the Michaelis complex is of particular interest in the context of enzymatic catalysis in which the reactant is usually represented by a single conformation of the en- zyme/substrate complex. Lactate dehydrogenase (LDH) catalyzes the interconversion of pyruvate and lactate with concomitant interconversion of two forms of the cofactor nicotinamide adenine dinucleotide (NADH and NAD+). Recent experimental results suggest that multiple substates exist within the Michaelis complex of LDH, and they show a strong variance in their propensity toward the on-enzyme chemical step. In this study, microsecond-scale all-atom molecular dynamics simulations were performed on LDH to explore the free energy landscape of the Michaelis complex, and network analysis was used to characterize the distribution of the conformations. Our results provide a detailed view of the kinetic network of the Michaelis complex and the struc- tures of the substates at atomistic scales. They also shed light on the complete picture of the catalytic mechanism of LDH.</td>
</tr>
<tr id="bib_Pan2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Pan2015,
  author = {Pan, Xiaoliang and Schwartz, Steven D},
  title = {Free Energy Surface of the Michaelis Complex of Lactate Dehydrogenase: A Network Analysis of Microsecond Simulations},
  journal = {Journal of Physical Chemistry B},
  year = {2015},
  volume = {119},
  number = {17},
  pages = {5430 - 5436},
  doi = {https://doi.org/10.1021/acs.jpcb.5b01840}
}
</pre></td>
</tr>
<tr id="Park2015" class="entry">
	<td>Park, B.-C., Yim, Y.-I., Zhao, X., Olszewski, M.B., Eisenberg, E. and Greene, L.E.</td>
	<td>The clathrin-binding and J-domains of GAK support the uncoating and chaperoning of clathrin by Hsc70 in the brain. <p class="infolinks">[<a href="javascript:toggleInfo('Park2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Park2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Journal of cell science<br/>Vol. 128, pp. 3811-3821&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1242/jcs.171058">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Park2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Cyclin-G-associated kinase (GAK), the ubiquitously expressed J-domain protein, is essential for the chaperoning and uncoating of clathrin that is mediated by Hsc70 (also known as HSPA8). Adjacent to the C-terminal J-domain that binds to Hsc70, GAK has a clathrin-binding domain that is linked to an N-terminal kinase domain through a PTEN-like domain. Knocking out GAK in fibroblasts caused inhibition of clathrin-dependent trafficking, which was rescued by expressing a 62-kDa fragment of GAK, comprising just the clathrin-binding and J-domains. Expressing this fragment as a transgene in mice rescued the lethality and the histological defects caused by knocking out GAK in the liver or in the brain. Furthermore, when both GAK and auxilin (also known as DNAJC6), the neuronal-specific homolog of GAK, were knocked out in the brain, mice expressing the 62-kDa GAK fragment were viable, lived a normal life-span and had no major behavior abnormalities. However, these mice were about half the size of wild-type mice. Therefore, the PTEN-like domains of GAK and auxilin are not essential for Hsc70-dependent chaperoning and uncoating of clathrin, but depending on the tissue, these domains appear to increase the efficiency of these co-chaperones.</td>
</tr>
<tr id="bib_Park2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Park2015,
  author = {Park, Bum-Chan and Yim, Yang-In and Zhao, Xiaohong and Olszewski, Maciej B and Eisenberg, Evan and Greene, Lois E},
  title = {The clathrin-binding and J-domains of GAK support the uncoating and chaperoning of clathrin by Hsc70 in the brain.},
  journal = {Journal of cell science},
  year = {2015},
  volume = {128},
  pages = {3811--3821},
  doi = {https://doi.org/10.1242/jcs.171058}
}
</pre></td>
</tr>
<tr id="Patel2017" class="entry">
	<td>Patel, J.S. and Ytreberg, F.M.</td>
	<td>Fast Calculation of Protein-Protein Binding Free Energies using Umbrella Sampling with a Coarse-Grained Model <p class="infolinks">[<a href="javascript:toggleInfo('Patel2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of chemical theory and computation&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Patel2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Patel2017,
  author = {Patel, Jagdish Suresh and Ytreberg, F Marty},
  title = {Fast Calculation of Protein-Protein Binding Free Energies using Umbrella Sampling with a Coarse-Grained Model},
  journal = {Journal of chemical theory and computation},
  publisher = {ACS Publications},
  year = {2017}
}
</pre></td>
</tr>
<tr id="Patton2009" class="entry">
	<td>Patton, L., Wittgenstein, E., Carnap, R., Von, H. and Logic, H.</td>
	<td>Signs, toy models, and the a priori: from Helmholtz to Wittgenstein <p class="infolinks">[<a href="javascript:toggleInfo('Patton2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Patton2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Studies in History and Philosophy of Science<br/>Vol. 40(3), pp. 281 - 289&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.shpsa.2009.07.004">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Patton2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: a b s t r a c t The Marburg neo-Kantians argue that Hermann von Helmholtz's empiricist account of the a priori does not account for certain knowledge, since it is based on a psychological phenomenon, trust in the regular-ities of nature. They argue that Helmholtz's account raises the 'problem of validity' (G&uuml;ltigkeitsproblem): how to establish a warranted claim that observed regularities are based on actual relations. I reconstruct Heinrich Hertz's and Ludwig Wittgenstein's Bild theoretic answer to the problem of validity: that scien-tists and philosophers can depict the necessary a priori constraints on states of affairs in a given system, and can establish whether these relations are actual relations in nature. The analysis of necessity within a system is a lasting contribution of the Bild theory. However, Hertz and Wittgenstein argue that the logical and mathematical sentences of a Bild are rules, tools for constructing relations, and the rules themselves are meaningless outside the theory. Carnap revises the argument for validity by attempting to give semantic rules for translation between frameworks. Russell and Quine object that pragmatics better accounts for the role of a priori reasoning in translating between frameworks. The conclusion of the tale, then, is a partial vindication of Helmholtz's original account.</td>
</tr>
<tr id="bib_Patton2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Patton2009,
  author = {Patton, Lydia and Wittgenstein, Empiricism and Carnap, Rudolf and Von, Hermann and Logic, Helmholtz},
  title = {Signs, toy models, and the a priori: from Helmholtz to Wittgenstein},
  journal = {Studies in History and Philosophy of Science},
  year = {2009},
  volume = {40},
  number = {3},
  pages = {281 - 289},
  doi = {https://doi.org/10.1016/j.shpsa.2009.07.004}
}
</pre></td>
</tr>
<tr id="Pauling1946" class="entry">
	<td>Pauling, L.</td>
	<td>Molecular Architecture and Biological Reactions <p class="infolinks">[<a href="javascript:toggleInfo('Pauling1946','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Pauling1946','bibtex')">BibTeX</a>]</p></td>
	<td>1946</td>
	<td>Chemical and Engineering News<br/>Vol. 10(10), pp. 1375 - 1376&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/cen-v024n010.p1375">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Pauling1946" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: no</td>
</tr>
<tr id="bib_Pauling1946" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Pauling1946,
  author = {Pauling, Linus},
  title = {Molecular Architecture and Biological Reactions},
  journal = {Chemical and Engineering News},
  year = {1946},
  volume = {10},
  number = {10},
  pages = {1375 - 1376},
  doi = {https://doi.org/10.1021/cen-v024n010.p1375}
}
</pre></td>
</tr>
<tr id="Pauling1948" class="entry">
	<td>Pauling, L.</td>
	<td>Nature of forces between large molecules of biological interest. <p class="infolinks">[<a href="javascript:toggleInfo('Pauling1948','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Pauling1948','bibtex')">BibTeX</a>]</p></td>
	<td>1948</td>
	<td>Nature<br/>Vol. 161(4097), pp. 707 - 709&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/161707a0">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Pauling1948" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: As I look at a living organism, I see reminders of many questions that need to be answered. Not all these questions are obviously important, nor would their answers be useful&acirc;€''but we want them answered. Thomas Wright in 1601 said, "Nothing is so curious and thirsty after knowledge of dark and obscure matters as the nature of man"&acirc;€''of scientific men especially, he might have said.</td>
</tr>
<tr id="bib_Pauling1948" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Pauling1948,
  author = {Pauling, Linus},
  title = {Nature of forces between large molecules of biological interest.},
  journal = {Nature},
  year = {1948},
  volume = {161},
  number = {4097},
  pages = {707 - 709},
  doi = {https://doi.org/10.1038/161707a0}
}
</pre></td>
</tr>
<tr id="Pearse2000" class="entry">
	<td>Pearse, B.M., Smith, C.J. and Owen, D.J.</td>
	<td>Clathrin coat construction in endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Pearse2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Pearse2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Current opinion in structural biology<br/>Vol. 10, pp. 220-228&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Pearse2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Electron cryomicroscopy of the clathrin coat and X-ray crystallography of parts of the clathrin heavy chain combine to give a detailed picture of the clathrin molecule, assembled as a cage. Recently determined domain structures of other components of the endocytic machinery, particularly the mu2 subunit and the alpha-appendage domain of the AP2 adaptor complex, provide important information on the sequence of recognition events involved in the dynamic process of clathrin coat assembly.</td>
</tr>
<tr id="bib_Pearse2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Pearse2000,
  author = {Pearse, B M and Smith, C J and Owen, D J},
  title = {Clathrin coat construction in endocytosis.},
  journal = {Current opinion in structural biology},
  year = {2000},
  volume = {10},
  pages = {220--228}
}
</pre></td>
</tr>
<tr id="Pechstein2010" class="entry">
	<td>Pechstein, A., Bacetic, J., Vahedi-Faridi, A., Gromova, K., Sundborger, A., Tomlin, N., Krainer, G., Vorontsova, O., Schäfer, J.G., Owe, S.G., Cousin, M.A., Saenger, W., Shupliakov, O. and Haucke, V.</td>
	<td>Regulation of synaptic vesicle recycling by complex formation between intersectin 1 and the clathrin adaptor complex AP2. <p class="infolinks">[<a href="javascript:toggleInfo('Pechstein2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Pechstein2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Proceedings of the National Academy of Sciences of the United States of America<br/>Vol. 107, pp. 4206-4211&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.0911073107">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Pechstein2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated synaptic vesicle (SV) recycling involves the spatiotemporally controlled assembly of clathrin coat components at phosphatidylinositiol (4, 5)-bisphosphate [PI(4,5)P(2)]-enriched membrane sites within the periactive zone. Such spatiotemporal control is needed to coordinate SV cargo sorting with clathrin/AP2 recruitment and to restrain membrane fission and synaptojanin-mediated uncoating until membrane deformation and clathrin coat assembly are completed. The molecular events underlying these control mechanisms are unknown. Here we show that the endocytic SH3 domain-containing accessory protein intersectin 1 scaffolds the endocytic process by directly associating with the clathrin adaptor AP2. Acute perturbation of the intersectin 1-AP2 interaction in lamprey synapses in situ inhibits the onset of SV recycling. Structurally, complex formation can be attributed to the direct association of hydrophobic peptides within the intersectin 1 SH3A-B linker region with the "side sites" of the AP2 alpha- and beta-appendage domains. AP2 appendage association of the SH3A-B linker region inhibits binding of the inositol phosphatase synaptojanin 1 to intersectin 1. These data identify the intersectin-AP2 complex as an important regulator of clathrin-mediated SV recycling in synapses.</td>
</tr>
<tr id="bib_Pechstein2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Pechstein2010,
  author = {Pechstein, Arndt and Bacetic, Jelena and Vahedi-Faridi, Ardeschir and Gromova, Kira and Sundborger, Anna and Tomlin, Nikolay and Krainer, Georg and Vorontsova, Olga and Schäfer, Johannes G and Owe, Simen G and Cousin, Michael A and Saenger, Wolfram and Shupliakov, Oleg and Haucke, Volker},
  title = {Regulation of synaptic vesicle recycling by complex formation between intersectin 1 and the clathrin adaptor complex AP2.},
  journal = {Proceedings of the National Academy of Sciences of the United States of America},
  year = {2010},
  volume = {107},
  pages = {4206--4211},
  note = {ITSN1 - AP2A2, AP2B1. domain information. rat, lamprey brain.},
  doi = {https://doi.org/10.1073/pnas.0911073107}
}
</pre></td>
</tr>
<tr id="Pechstein2010a" class="entry">
	<td>Pechstein, A., Shupliakov, O. and Haucke, V.</td>
	<td>Intersectin 1: a versatile actor in the synaptic vesicle cycle. <p class="infolinks">[<a href="javascript:toggleInfo('Pechstein2010a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Pechstein2010a','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Biochemical Society transactions<br/>Vol. 38, pp. 181-186&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1042/BST0380181">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Pechstein2010a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: During neurotransmitter release, SVs (synaptic vesicles) fuse at the active zone and are recovered predominantly via clathrin-mediated endocytosis at the presynaptic compartment surrounding the site of release, referred to as the periactive zone. Exo- and endo-cytosis in synapses are tightly temporarily and spatially coupled to sustain synaptic transmission. The molecular mechanisms linking these two cellular events, which take place in separate compartments of the nerve terminal, remain largely enigmatic. Several lines of evidence indicate that multiple factors may be involved in exocytic-endocytic coupling including SV integral membrane proteins, SV membrane lipids and the membrane-associated actin cytoskeleton. A number of recent studies also indicate that multimodular adaptor proteins shuttling between the active and periactive zones aid the dynamic assembly of macromolecular protein complexes that execute the exo- and endo-cytic limbs of the SV cycle. Here, we discuss recent evidence implicating the multidomain scaffolding and adaptor protein ITSN1 (intersectin 1) as a central regulator of SV cycling.</td>
</tr>
<tr id="bib_Pechstein2010a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Pechstein2010a,
  author = {Pechstein, Arndt and Shupliakov, Oleg and Haucke, Volker},
  title = {Intersectin 1: a versatile actor in the synaptic vesicle cycle.},
  journal = {Biochemical Society transactions},
  year = {2010},
  volume = {38},
  pages = {181--186},
  note = {good figure of domain interactions with itsn1},
  doi = {https://doi.org/10.1042/BST0380181}
}
</pre></td>
</tr>
<tr id="Pechukas1976" class="entry">
	<td>Pechukas, P.</td>
	<td>Statistical Approximations in Collision Theory <p class="infolinks">[<a href="javascript:toggleInfo('Pechukas1976','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Pechukas1976','bibtex')">BibTeX</a>]</p></td>
	<td>1976</td>
	<td>Dynamics of Molecular Collisions, pp. 269&nbsp;</td>
	<td>inbook</td>
	<td><a href="https://doi.org/10.1007/978-1-4757-0644-4_6">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Pechukas1976" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: This chapter is an introduction to statistical approximation in the theory of reactive collisions. The theme of the chapter is the transition state,* and the statistics in statistical theory is in essence just the counting of the various ways a system can pass through a transition state.</td>
</tr>
<tr id="bib_Pechukas1976" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inbook{Pechukas1976,
  author = {Pechukas, Philip},
  title = {Statistical Approximations in Collision Theory},
  booktitle = {Dynamics of Molecular Collisions},
  publisher = {Springer US},
  year = {1976},
  pages = {269},
  doi = {https://doi.org/10.1007/978-1-4757-0644-4_6}
}
</pre></td>
</tr>
<tr id="Pechukas1973" class="entry">
	<td>Pechukas, P. and McLafferty, F.J.</td>
	<td>On transition-state theory and the classical mechanics of collinear collisions <p class="infolinks">[<a href="javascript:toggleInfo('Pechukas1973','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Pechukas1973','bibtex')">BibTeX</a>]</p></td>
	<td>1973</td>
	<td>J. Chem. Phys.<br/>Vol. 58(4), pp. 1622 - 1625&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.1679404">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Pechukas1973" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We consider classical transition-state theory of collinear atom-diatom reactions. There are (many) potential surfaces for which this theory is exact provided all trajectories with energy above some fixed value are disregarded; the energy cutoff of course depends on the surface. We develop a simple criterion for recognizing or constructing such surfaces, and discuss examples. We also show that if a surface has a proper transition state, as defined below, then transition-state theory is exact unless collision complexes exist.</td>
</tr>
<tr id="bib_Pechukas1973" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Pechukas1973,
  author = {Pechukas, Philip and McLafferty, F. J.},
  title = {On transition-state theory and the classical mechanics of collinear collisions},
  journal = {J. Chem. Phys.},
  year = {1973},
  volume = {58},
  number = {4},
  pages = {1622 - 1625},
  doi = {https://doi.org/10.1063/1.1679404}
}
</pre></td>
</tr>
<tr id="Perthold2017" class="entry">
	<td>Perthold, J.W. and Oostenbrink, C.</td>
	<td>Simulation of Reversible Protein-Protein Binding and Calculation of Binding Free Energies using Perturbed Distance Restraints <p class="infolinks">[<a href="javascript:toggleInfo('Perthold2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of Chemical Theory and Computation&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Perthold2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Perthold2017,
  author = {Perthold, Jan Walther and Oostenbrink, Chris},
  title = {Simulation of Reversible Protein-Protein Binding and Calculation of Binding Free Energies using Perturbed Distance Restraints},
  journal = {Journal of Chemical Theory and Computation},
  publisher = {ACS Publications},
  year = {2017}
}
</pre></td>
</tr>
<tr id="Peter2004" class="entry">
	<td>Peter, B.J., Kent, H.M., Mills, I.G., Vallis, Y., Butler, P.J.G., Evans, P.R. and McMahon, H.T.</td>
	<td>BAR domains as sensors of membrane curvature: the amphiphysin BAR structure <p class="infolinks">[<a href="javascript:toggleInfo('Peter2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Science<br/>Vol. 303(5657), pp. 495-499&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Peter2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Peter2004,
  author = {Peter, Brian J and Kent, Helen M and Mills, Ian G and Vallis, Yvonne and Butler, P Jonathan G and Evans, Philip R and McMahon, Harvey T},
  title = {BAR domains as sensors of membrane curvature: the amphiphysin BAR structure},
  journal = {Science},
  publisher = {American Association for the Advancement of Science},
  year = {2004},
  volume = {303},
  number = {5657},
  pages = {495--499},
  note = {amph dimer. dissociation constant. drosophilia amph.}
}
</pre></td>
</tr>
<tr id="Peters2010" class="entry">
	<td>Peters, B.</td>
	<td>Transition-State Theory, Dynamics, and Narrow Time Scale Separation in the Rate-Promoting Vibrations Model of Enzyme Catalysis <p class="infolinks">[<a href="javascript:toggleInfo('Peters2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Peters2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Journal of Chemical Theory and Computation<br/>Vol. 6(5), pp. 1447 - 1454&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ct100051a">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Peters2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The power of transition-state theory (TST) for understanding enzymes is evidenced by its recent use in the design and synthesis of highly active de novo enzymes. However, dynamics can influence reaction kinetics, and some studies of rate-promoting vibrations even claim that dynamical theories instead of TST are needed to understand enzymatic reaction mechanisms. For the rate-promoting vibration (RPV) model of enzyme catalysis [Antoniou et al., J. Chem. Phys. 2004, 121, 6442], a reactive flux correlation function analysis shows that dynamical effects do slow the kinetics. However, the RPV model also shows extremely long-lived correlations because the RPV and the bath are not directly coupled. Additionally, earlier studies of the RPV model show a narrow time scale separation due to a small 5kT barrier. Thus earlier findings based on the RPV model may have little bearing on the properties of real enzymes. The intrinsic reaction coordinate (IRC) reveals that the RPV is an important component of the reaction coordinate at early and late stages of the pathway, but the RPV is not an important component of the reaction coordinate direction at the transition state. The unstable eigenmode from harmonic TST (which coincides with the IRC at the saddle point) gives a larger transmission coefficient than the coordinate used in the correlation functions of Antoniou et al. Thus while TST cannot predict the transmission coefficient, the RPV model suggests that TST can provide mechanistic insights on elementary steps in enzyme catalysis. Finally, we propose a method for using the transition-state ensemble as predicted from harmonic TST to distinguish promoting vibrations from other more mundane bath variables.<br>The power of transition-state theory (TST) for understanding enzymes is evidenced by its recent use in the design and synthesis of highly active de novo enzymes. However, dynamics can influence reaction kinetics, and some studies of rate-promoting vibrations even claim that dynamical theories instead of TST are needed to understand enzymatic reaction mechanisms. For the rate-promoting vibration (RPV) model of enzyme catalysis [Antoniou et al., J. Chem. Phys. 2004, 121, 6442], a reactive flux correlation function analysis shows that dynamical effects do slow the kinetics. However, the RPV model also shows extremely long-lived correlations because the RPV and the bath are not directly coupled. Additionally, earlier studies of the RPV model show a narrow time scale separation due to a small 5kT barrier. Thus earlier findings based on the RPV model may have little bearing on the properties of real enzymes. The intrinsic reaction coordinate (IRC) reveals that the RPV is an important component of the reaction coordinate at early and late stages of the pathway, but the RPV is not an important component of the reaction coordinate direction at the transition state. The unstable eigenmode from harmonic TST (which coincides with the IRC at the saddle point) gives a larger transmission coefficient than the coordinate used in the correlation functions of Antoniou et al. Thus while TST cannot predict the transmission coefficient, the RPV model suggests that TST can provide mechanistic insights on elementary steps in enzyme catalysis. Finally, we propose a method for using the transition-state ensemble as predicted from harmonic TST to distinguish promoting vibrations from other more mundane bath variables.</td>
</tr>
<tr id="bib_Peters2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Peters2010,
  author = {Peters, Baron},
  title = {Transition-State Theory, Dynamics, and Narrow Time Scale Separation in the Rate-Promoting Vibrations Model of Enzyme Catalysis},
  journal = {Journal of Chemical Theory and Computation},
  year = {2010},
  volume = {6},
  number = {5},
  pages = {1447 - 1454},
  doi = {https://doi.org/10.1021/ct100051a}
}
</pre></td>
</tr>
<tr id="Peulen2017" class="entry">
	<td>Peulen, Thomas-Otavio., Opanasyuk, O. and Seidel, C.AM.</td>
	<td>Combining Graphical and Analytical Methods with Molecular Simulations To Analyze Time-Resolved FRET Measurements of Labeled Macromolecules Accurately <p class="infolinks">[<a href="javascript:toggleInfo('Peulen2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jpcb.7b03441">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Peulen2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Peulen2017,
  author = {Peulen, Thomas-Otavio and Opanasyuk, Oleg and Seidel, Claus AM},
  title = {Combining Graphical and Analytical Methods with Molecular Simulations To Analyze Time-Resolved FRET Measurements of Labeled Macromolecules Accurately},
  year = {2017},
  doi = {https://doi.org/10.1021/acs.jpcb.7b03441}
}
</pre></td>
</tr>
<tr id="Phanse2016" class="entry">
	<td>Phanse, S., Wan, C., Borgeson, B., Tu, F., Drew, K., Clark, G., Xiong, X., Kagan, O., Kwan, J., Bezginov, A., Chessman, K., Pal, S., Cromar, G., Papoulas, O., Ni, Z., Boutz, D.R., Stoilova, S., Havugimana, P.C., Guo, X., Malty, R.H., Sarov, M., Greenblatt, J., Babu, M., Derry, WB., Tillier, E.R., Wallingford, J.B., Parkinson, J., Marcotte, E.M. and Emili, A.</td>
	<td>Proteome-wide dataset supporting the study of ancient metazoan macromolecular complexes. <p class="infolinks">[<a href="javascript:toggleInfo('Phanse2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Phanse2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Data Brief<br/>Vol. 6, pp. 715-21&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.dib.2015.11.062">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Phanse2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Our analysis examines the conservation of multiprotein complexes among metazoa through use of high resolution biochemical fractionation and precision mass spectrometry applied to soluble cell extracts from 5 representative model organisms Caenorhabditis elegans, Drosophila melanogaster, Mus musculus, Strongylocentrotus purpuratus, and Homo sapiens. The interaction network obtained from the data was validated globally in 4 distant species (Xenopus laevis, Nematostella vectensis, Dictyostelium discoideum, Saccharomyces cerevisiae) and locally by targeted affinity-purification experiments. Here we provide details of our massive set of supporting biochemical fractionation data available via ProteomeXchange (PXD002319-PXD002328), PPIs via BioGRID (185267); and interaction network projections via (http://metazoa.med.utoronto.ca) made fully accessible to allow further exploration. The datasets here are related to the research article on metazoan macromolecular complexes in Nature [1].</td>
</tr>
<tr id="bib_Phanse2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Phanse2016,
  author = {Phanse, Sadhna and Wan, Cuihong and Borgeson, Blake and Tu, Fan and Drew, Kevin and Clark, Greg and Xiong, Xuejian and Kagan, Olga and Kwan, Julian and Bezginov, Alexandr and Chessman, Kyle and Pal, Swati and Cromar, Graham and Papoulas, Ophelia and Ni, Zuyao and Boutz, Daniel R and Stoilova, Snejana and Havugimana, Pierre C and Guo, Xinghua and Malty, Ramy H and Sarov, Mihail and Greenblatt, Jack and Babu, Mohan and Derry, WB and Tillier, Elisabeth R and Wallingford, John B and Parkinson, John and Marcotte, Edward M and Emili, Andrew},
  title = {Proteome-wide dataset supporting the study of ancient metazoan macromolecular complexes.},
  journal = {Data Brief},
  year = {2016},
  volume = {6},
  pages = {715-21},
  doi = {https://doi.org/10.1016/j.dib.2015.11.062}
}
</pre></td>
</tr>
<tr id="Pineda2010" class="entry">
	<td>Pineda, E.JRT., Antoniou, D. and Schwartz, S.D.</td>
	<td>Slow conformational motions that favor sub-picosecond motions important for catalysis <p class="infolinks">[<a href="javascript:toggleInfo('Pineda2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Pineda2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 114(48), pp. 15985 - 15990&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp1071296">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Pineda2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: It has been accepted for many years that functionally important motions are crucial to binding properties of ligands in such molecules as hemoglobin and myoglobin. In enzymatic reactions, theory and now experiment are beginning to confirm the importance of motions on a fast (ps) time scale in the chemical step of the catalytic process. What is missing is a clear physical picture of how slow conformational fluctuations are related to the fast motions that have been identified as crucial. This paper presents a theoretical analysis of this issue for human heart lactate dehydrogenase. We will examine how slow conformational motions bring the system to conformations that are distinguished as catalytically competent because they favor specific fast motions.</td>
</tr>
<tr id="bib_Pineda2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Pineda2010,
  author = {Pineda, Exequiel JRT and Antoniou, Dimitri and Schwartz, Steven D},
  title = {Slow conformational motions that favor sub-picosecond motions important for catalysis},
  journal = {Journal of Physical Chemistry B},
  year = {2010},
  volume = {114},
  number = {48},
  pages = {15985 - 15990},
  doi = {https://doi.org/10.1021/jp1071296}
}
</pre></td>
</tr>
<tr id="Piserchia2017" class="entry">
	<td>Piserchia, A., Banerjee, S. and Barone, V.</td>
	<td>General Approach to Coupled Reactive Smoluchowski Equations: Integration and Application of DVR and Generalized Coordinate Methods to Diffusive Problems. <p class="infolinks">[<a href="javascript:toggleInfo('Piserchia2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of Chemical Theory and Computation&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Piserchia2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Piserchia2017,
  author = {Piserchia, Andrea and Banerjee, Shiladitya and Barone, Vincenzo},
  title = {General Approach to Coupled Reactive Smoluchowski Equations: Integration and Application of DVR and Generalized Coordinate Methods to Diffusive Problems.},
  journal = {Journal of Chemical Theory and Computation},
  publisher = {ACS Publications},
  year = {2017}
}
</pre></td>
</tr>
<tr id="Plapp2010" class="entry">
	<td>Plapp, B.V.</td>
	<td>Conformational changes and catalysis by alcohol dehydrogenase. <p class="infolinks">[<a href="javascript:toggleInfo('Plapp2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Plapp2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Archives of biochemistry and biophysics<br/>Vol. 493(1), pp. 3 - 12&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.abb.2009.07.001">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Plapp2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: As shown by X-ray crystallography, horse liver alcohol dehydrogenase undergoes a global conformational change upon binding of NAD(+) or NADH, involving a rotation of the catalytic domain relative to the coenzyme binding domain and the closing up of the active site to produce a catalytically efficient enzyme. The conformational change requires a complete coenzyme and is affected by various chemical or mutational substitutions that can increase the catalytic turnover by altering the kinetics of the isomerization and rate of dissociation of coenzymes. The binding of NAD(+) is kinetically limited by a unimolecular isomerization (corresponding to the conformational change) that is controlled by deprotonation of the catalytic zinc-water to produce a negatively-charged zinc-hydroxide, which can attract the positively-charged nicotinamide ring. The deprotonation is facilitated by His-51 acting through a hydrogen-bonded network to relay the proton to solvent. Binding of NADH also involves a conformational change, but the rate is very fast. After the enzyme binds NAD(+) and closes up, the substrate displaces the hydroxide bound to the catalytic zinc; this exchange may involve a double displacement reaction where the carboxylate group of a glutamate residue first displaces the hydroxide (inverting the tetrahedral coordination of the zinc), and then the exogenous ligand displaces the glutamate. The resulting enzyme-NAD(+)-alcoholate complex is poised for hydrogen transfer, and small conformational fluctuations may bring the reactants together so that the hydride ion is transferred by quantum mechanical tunneling. In the process, the nicotinamide ring may become puckered, as seen in structures of complexes of the enzyme with NADH. The conformational changes of alcohol dehydrogenase demonstrate the importance of protein dynamics in catalysis.</td>
</tr>
<tr id="bib_Plapp2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Plapp2010,
  author = {Plapp, Bryce V},
  title = {Conformational changes and catalysis by alcohol dehydrogenase.},
  journal = {Archives of biochemistry and biophysics},
  year = {2010},
  volume = {493},
  number = {1},
  pages = {3 - 12},
  note = {Flexibility of YADH},
  doi = {https://doi.org/10.1016/j.abb.2009.07.001}
}
</pre></td>
</tr>
<tr id="Polo2003" class="entry">
	<td>Polo, S., Confalonieri, S., Salcini, A.E. and Di Fiore, P.P.</td>
	<td>EH and UIM: endocytosis and more. <p class="infolinks">[<a href="javascript:toggleInfo('Polo2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Polo2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Science's STKE : signal transduction knowledge environment<br/>Vol. 2003, pp. re17&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1126/stke.2132003re17">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Polo2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Exogenously and endogenously originated signals are propagated within the cell by functional and physical networks of proteins, leading to numerous biological outcomes. Many protein-protein interactions take place between binding domains and short peptide motifs. Frequently, these interactions are inducible by upstream signaling events, in which case one of the two binding surfaces may be created by a posttranslational modification. Here, we discuss two protein networks. One, the EH-network, is based on the Eps15 homology (EH) domain, which binds to peptides containing the sequence Asp-Pro-Phe (NPF). The other, which we define as the monoubiquitin (mUb) network, relies on monoubiquitination, which is emerging as an important posttranslational modification that regulates protein function. Both networks were initially implicated in the control of plasma membrane receptor endocytosis and in the regulation of intracellular trafficking routes. The ramifications of these two networks, however, appear to extend into many other aspects of cell physiology as well, such as transcriptional regulation, actin cytoskeleton remodeling, and DNA repair. The focus of this review is to integrate available knowledge of the EH- and mUb networks with predictions of genetic and physical interactions stemming from functional genomics approaches.</td>
</tr>
<tr id="bib_Polo2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Polo2003,
  author = {Polo, Simona and Confalonieri, Stefano and Salcini, Anna Elisabetta and Di Fiore, Pier Paolo},
  title = {EH and UIM: endocytosis and more.},
  journal = {Science's STKE : signal transduction knowledge environment},
  year = {2003},
  volume = {2003},
  pages = {re17},
  note = {review, some domain information.},
  doi = {https://doi.org/10.1126/stke.2132003re17}
}
</pre></td>
</tr>
<tr id="Poovathingal2010" class="entry">
	<td>Poovathingal, SK. and Gunawan, R.</td>
	<td>Global parameter estimation methods for stochastic biochemical systems <p class="infolinks">[<a href="javascript:toggleInfo('Poovathingal2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Poovathingal2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Bioinformatics<br/>Vol. 11(1), pp. 414 - 414&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1186/1471-2105-11-414">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Poovathingal2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Background: The importance of stochasticity in cellular processes having low number of molecules has resulted in the development of stochastic models such as chemical master equation. As in other modelling frameworks, the accompanying rate constants are important for the end-applications like analyzing system properties (e. g. robustness) or predicting the effects of genetic perturbations. Prior knowledge of kinetic constants is usually limited and the model identification routine typically includes parameter estimation from experimental data. Although the subject of parameter estimation is well-established for deterministic models, it is not yet routine for the chemical master equation. In addition, recent advances in measurement technology have made the quantification of genetic substrates possible to single molecular levels. Thus, the purpose of this work is to develop practical and effective methods for estimating kinetic model parameters in the chemical master equation and other stochastic models from single cell and cell population experimental data.\textbackslash\n\textbackslash\nResults: Three parameter estimation methods are proposed based on the maximum likelihood and density function distance, including probability and cumulative density functions. Since stochastic models such as chemical master equations are typically solved using a Monte Carlo approach in which only a finite number of Monte Carlo realizations are computationally practical, specific considerations are given to account for the effect of finite sampling in the histogram binning of the state density functions. Applications to three practical case studies showed that while maximum likelihood method can effectively handle low replicate measurements, the density function distance methods, particularly the cumulative density function distance estimation, are more robust in estimating the parameters with consistently higher accuracy, even for systems showing multimodality.\textbackslash\n\textbackslash\nConclusions: The parameter estimation methodologies described in this work have provided an effective and practical approach in the estimation of kinetic parameters of stochastic systems from either sparse or dense cell population data. Nevertheless, similar to kinetic parameter estimation in other modelling frameworks, not all parameters can be estimated accurately, which is a common problem arising from the lack of complete parameter identifiability from the available data.</td>
</tr>
<tr id="bib_Poovathingal2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Poovathingal2010,
  author = {Poovathingal, SK and Gunawan, R},
  title = {Global parameter estimation methods for stochastic biochemical systems},
  journal = {Bioinformatics},
  year = {2010},
  volume = {11},
  number = {1},
  pages = {414 - 414},
  doi = {https://doi.org/10.1186/1471-2105-11-414}
}
</pre></td>
</tr>
<tr id="Popova2013" class="entry">
	<td>Popova, N.V., Deyev, I.E. and Petrenko, A.G.</td>
	<td>Clathrin-mediated endocytosis and adaptor proteins. <p class="infolinks">[<a href="javascript:toggleInfo('Popova2013','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Popova2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Acta naturae<br/>Vol. 5, pp. 62-73&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Popova2013" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Macromolecules gain access to the cytoplasm of eukaryotic cells using one of several ways of which clathrin-dependent endocytosis is the most researched. Although the mechanism of clathrin-mediated endocytosis is well understood in general, novel adaptor proteins that play various roles in ensuring specific regulation of the mentioned process are being discovered all the time. This review provides a detailed account of the mechanism of clathrin-mediated internalization of activated G protein-coupled receptors, as well as a description of the major proteins involved in this process.</td>
</tr>
<tr id="bib_Popova2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Popova2013,
  author = {Popova, N V and Deyev, I E and Petrenko, A G},
  title = {Clathrin-mediated endocytosis and adaptor proteins.},
  journal = {Acta naturae},
  year = {2013},
  volume = {5},
  pages = {62--73}
}
</pre></td>
</tr>
<tr id="Posor2015" class="entry">
	<td>Posor, Y., Eichhorn-Grünig, M. and Haucke, V.</td>
	<td>Phosphoinositides in endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Posor2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Posor2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Biochimica et biophysica acta<br/>Vol. 1851, pp. 794-804&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.bbalip.2014.09.014">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Posor2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The internalization and subsequent endosomal trafficking of proteins and membrane along the endocytic pathway is a fundamental cellular process. Over the last two decades, this pathway has emerged to be subject to extensive regulation by phosphoinositides (PIs), phosphorylated derivatives of the minor membrane phospholipid phosphatidylinositol. Clathrin-mediated endocytosis (CME) is the endocytic mechanism characterized in most detail. It now represents a prime example of a process spatiotemporally organized by the interplay of PI metabolizing enzymes. The most abundant PI, phosphatidylinositol-4,5-bisphosphate [PI(4,5)P₂], serves as a denominator of plasma membrane identity and together with cargo proteins is instrumental for the initiation of clathrin-coated pit (CCP) formation. During later stages of the process, the generation of phosphatidylinositol-3,4-bisphosphate [PI(3,4)P₂] and the dephosphorylation of PI(4,5)P₂regulate CCP maturation and vesicle uncoating. Here we provide an overview of the mechanisms by which PIs are made and consumed to regulate CME and other endocytic pathways and how conversion of PIs en route to endosomes may be accomplished. Mutations in PI converting enzymes are linked to multiple diseases ranging from mental retardation and neurodegeneration, to inherited muscle and kidney disorders suggesting that the tight control of PI levels along the endocytic pathway plays a critical role in cell physiology. This article is part of a Special Issue entitled Phosphoinositides.</td>
</tr>
<tr id="bib_Posor2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Posor2015,
  author = {Posor, York and Eichhorn-Grünig, Marielle and Haucke, Volker},
  title = {Phosphoinositides in endocytosis.},
  journal = {Biochimica et biophysica acta},
  year = {2015},
  volume = {1851},
  pages = {794--804},
  doi = {https://doi.org/10.1016/j.bbalip.2014.09.014}
}
</pre></td>
</tr>
<tr id="Praefcke2004" class="entry">
	<td>Praefcke, G.JK., Ford, M.GJ., Schmid, E.M., Olesen, L.E., Gallop, J.L., Sew-Yeu, Peak-Chew,., Vallis, Y., Babu, M.M., Mills, I.G. and McMahon, Harvey, T</td>
	<td>Evolving nature of the AP2 alpha-appendage hub during clathrin-coated vesicle endocytosis <p class="infolinks">[<a href="javascript:toggleInfo('Praefcke2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Praefcke2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>The EMBO Journal<br/>Vol. 23(22), pp. 4371 - 4383&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/sj.emboj.7600445">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Praefcke2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis involves the assembly of a network of proteins that select cargo, modify membrane shape and drive invagination, vesicle scission and uncoating. This network is initially assembled around adaptor protein (AP) appendage domains, which are protein interaction hubs. Using crystallography, we show that FxDxF and WVxF peptide motifs from synaptojanin bind to distinct subdomains on alpha-appendages, called 'top' and 'side' sites. Appendages use both these sites to interact with their binding partners in vitro and in vivo. Occupation of both sites simultaneously results in high-affinity reversible interactions with lone appendages (e.g. eps15 and epsin1). Proteins with multiple copies of only one type of motif bind multiple appendages and so will aid adaptor clustering. These clustered alpha(appendage)-hubs have altered properties where they can sample many different binding partners, which in turn can interact with each other and indirectly with clathrin. In the final coated vesicle, most appendage binding partners are absent and thus the functional status of the appendage domain as an interaction hub is temporal and transitory giving directionality to vesicle assembly.</td>
</tr>
<tr id="bib_Praefcke2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Praefcke2004,
  author = {Praefcke, Gerrit JK and Ford, Marijn GJ and Schmid, Eva M and Olesen, Lene E and Gallop, Jennifer L and Sew-Yeu, Peak-Chew, and Vallis, Yvonne and Babu, Madan M and Mills, Ian G and T, McMahon, Harvey},
  title = {Evolving nature of the AP2 alpha-appendage hub during clathrin-coated vesicle endocytosis},
  journal = {The EMBO Journal},
  year = {2004},
  volume = {23},
  number = {22},
  pages = {4371 - 4383},
  note = {Also contains discussion on stonin-2. Excellent figure of domains and interactions with AP2 alpha subunit. Contains Kd for AP2A2 (but only of peptide sequences of specific proteins). eps15, epn1, ap180 bind to multiple ap2a?s.},
  doi = {https://doi.org/10.1038/sj.emboj.7600445}
}
</pre></td>
</tr>
<tr id="Pratt1986" class="entry">
	<td>Pratt, L.R.</td>
	<td>A statistical method for identifying transition states in high dimensional problems <p class="infolinks">[<a href="javascript:toggleInfo('Pratt1986','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Pratt1986','bibtex')">BibTeX</a>]</p></td>
	<td>1986</td>
	<td>Journal of Chemical Physics<br/>Vol. 85(1986), pp. 5045 - 5045&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.451695">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Pratt1986" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A statistical method based on Monte Carlo sampling of paths associated with Markovianstochastic processes is proposed for the location of transition states in molecularly complex reactions. It is discussed in what respects the proposed method would identify transition state regions, and an algorithm for Metropolis sampling of Metropolis paths (based on the Metropolis random walk process) is devised. Physical aspects of this new algorithm are examined, and the relation of this method to simulated annealing algorithms for physical design of circuits is noted.</td>
</tr>
<tr id="bib_Pratt1986" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Pratt1986,
  author = {Pratt, Lawrence R},
  title = {A statistical method for identifying transition states in high dimensional problems},
  journal = {Journal of Chemical Physics},
  year = {1986},
  volume = {85},
  number = {1986},
  pages = {5045 - 5045},
  doi = {https://doi.org/10.1063/1.451695}
}
</pre></td>
</tr>
<tr id="Prokop2013" class="entry">
	<td></td>
	<td>Systems Biology: Integrative Biology and Simulation Tools <p class="infolinks">[<a href="javascript:toggleInfo('Prokop2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>&nbsp;</td>
	<td>book</td>
	<td><a href="https://doi.org/10.1007/SpringerReference_36933">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Prokop2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{Prokop2013,,
  title = {Systems Biology: Integrative Biology and Simulation Tools},
  publisher = {Springer},
  year = {2013},
  doi = {https://doi.org/10.1007/SpringerReference_36933}
}
</pre></td>
</tr>
<tr id="Pu2004" class="entry">
	<td>Pu, J., Gao, J. and Truhlar, D.G.</td>
	<td>Generalized Hybrid Orbital (GHO) Method for Combining Ab Initio Hartree−Fock Wave Functions with Molecular Mechanics <p class="infolinks">[<a href="javascript:toggleInfo('Pu2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Pu2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of Physical Chemistry A<br/>Vol. 108(4), pp. 632 - 650&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp036755k">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Pu2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The generalized hybrid orbital (GHO) method provides a way to combine quantum mechanical (QM) and molecular mechanical (MM) calculations on a single molecular system or supramolecular assembly by providing an electrostatically stable connection between the QM portion and the MM portion. The GHO method has previously been developed for semiempirical molecular orbital calculations, on the basis of neglect of diatomic differential overlap (GHO?NDDO); in the present work, it is extended to the ab initio Hartree?Fock (HF) level (GHO?AIHF). First, the theoretical foundation for the GHO?AIHF extension is discussed, and four different approaches are proposed to overcome the nonorthogonality between active molecular orbitals (MOs) and auxiliary MOs. In the first scheme, the auxiliary hybrid basis functions are projected out of the active QM basis. The second scheme neglects the diatomic differential overlap between the auxiliary basis and the active QM basis. In the third scheme, hybrid orbitals are constructed from L&ouml;wdin-type symmetric orthogonalized atomic orbitals on the basis of global L&ouml;wdin orthogonalization. The fourth procedure involves local L&ouml;wdin orthogonalization. The procedures for implementing the four GHO?AIHF schemes are described, and analytical gradient expressions are derived. The unparametrized GHO?AIHF method is tested for hydrocarbons with various basis sets, in particular, the geometries and charges are compared with pure QM calculations for ethane, ethyl radical, and n-octane, and the method is tested for the torsion potential around the central bond in n-butane. Furthermore, a parametrization of the GHO?AIHF method for the MIDI! basis is presented and tested for 16 molecules and ions with various functional groups near the QM/MM boundary. The results show the robustness of the algorithm and illustrate the significant improvement made by introducing several one-electron integral-scaling parameters. Finally, the energetic performance of the method is tested by comparing the proton affinities for a set of small model compounds (alcohols, amines, thiols, and acids) to results obtained from fully QM calculations. We conclude that the GHO?AIHF scheme provides a reasonable fundamental solution to the problem of combining an ab initio quantum mechanical electronic structure calculation with molecular mechanics. The generalized hybrid orbital (GHO) method provides a way to combine quantum mechanical (QM) and molecular mechanical (MM) calculations on a single molecular system or supramolecular assembly by providing an electrostatically stable connection between the QM portion and the MM portion. The GHO method has previously been developed for semiempirical molecular orbital calculations, on the basis of neglect of diatomic differential overlap (GHO?NDDO); in the present work, it is extended to the ab initio Hartree?Fock (HF) level (GHO?AIHF). First, the theoretical foundation for the GHO?AIHF extension is discussed, and four different approaches are proposed to overcome the nonorthogonality between active molecular orbitals (MOs) and auxiliary MOs. In the first scheme, the auxiliary hybrid basis functions are projected out of the active QM basis. The second scheme neglects the diatomic differential overlap between the auxiliary basis and the active QM basis. In the third scheme, hybrid orbitals are constructed from L&ouml;wdin-type symmetric orthogonalized atomic orbitals on the basis of global L&ouml;wdin orthogonalization. The fourth procedure involves local L&ouml;wdin orthogonalization. The procedures for implementing the four GHO?AIHF schemes are described, and analytical gradient expressions are derived. The unparametrized GHO?AIHF method is tested for hydrocarbons with various basis sets, in particular, the geometries and charges are compared with pure QM calculations for ethane, ethyl radical, and n-octane, and the method is tested for the torsion potential around the central bond in n-butane. Furthermore, a parametrization of the GHO?AIHF method for the MIDI! basis is presented and tested for 16 molecules and ions with various functional groups near the QM/MM boundary. The results show the robustness of the algorithm and illustrate the significant improvement made by introducing several one-electron integral-scaling parameters. Finally, the energetic performance of the method is tested by comparing the proton affinities for a set of small model compounds (alcohols, amines, thiols, and acids) to results obtained from fully QM calculations. We conclude that the GHO?AIHF scheme provides a reasonable fundamental solution to the problem of combining an ab initio quantum mechanical electronic structure calculation with molecular mechanics.</td>
</tr>
<tr id="bib_Pu2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Pu2004,
  author = {Pu, Jingzhi and Gao, Jiali and Truhlar, Donald G},
  title = {Generalized Hybrid Orbital (GHO) Method for Combining Ab Initio Hartree−Fock Wave Functions with Molecular Mechanics},
  journal = {Journal of Physical Chemistry A},
  year = {2004},
  volume = {108},
  number = {4},
  pages = {632 - 650},
  doi = {https://doi.org/10.1021/jp036755k}
}
</pre></td>
</tr>
<tr id="Pucharcos2001" class="entry">
	<td>Pucharcos, C., Casas, C., Nadal, M., Estivill, X. and de la Luna, S.</td>
	<td>The human intersectin genes and their spliced variants are differentially expressed. <p class="infolinks">[<a href="javascript:toggleInfo('Pucharcos2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Pucharcos2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Biochimica et biophysica acta<br/>Vol. 1521, pp. 1-11&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Pucharcos2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Human intersectins (ITSN1 and ITSN2) are members of a conserved family of proteins involved in clathrin-mediated endocytosis. A short and a long isoform with different protein domain compositions have been described for both human intersectins. Here, we have resolved the exon/intron structure of the ITSN2 gene to explain the genomic origin of its alternatively spliced transcripts. Comparison of the two ITSN human genes shows a high level of conservation in their genomic organization, including the main alternative splicing events. An extensive tissue expression analysis of the two predominant transcripts as well as other minor variants shows that ITSN expression is under tissue and developmental controls. Their differential expression is made more evident when the expression of both intersectins is studied by in situ hybridization in mouse brain.</td>
</tr>
<tr id="bib_Pucharcos2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Pucharcos2001,
  author = {Pucharcos, C and Casas, C and Nadal, M and Estivill, X and de la Luna, S},
  title = {The human intersectin genes and their spliced variants are differentially expressed.},
  journal = {Biochimica et biophysica acta},
  year = {2001},
  volume = {1521},
  pages = {1--11},
  note = {long form of itsn1 is found exclusively in brain.}
}
</pre></td>
</tr>
<tr id="Punta2012" class="entry">
	<td>Punta, M., Coggill, P., Eberhardt, R., Mistry, J., Tate, J., Boursnell, C., Pang, N., Forslund, K., Ceric, G., Clements, J., Heger, A., Holm, L., Sonnhammer, E., Eddy, S., Bateman, A. and Finn, R.</td>
	<td>The Pfam protein families databases. <p class="infolinks">[<a href="javascript:toggleInfo('Punta2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Punta2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Nucleic Acids Research<br/>Vol. 30(1), pp. 1 - 12&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/nar/gkp985">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Punta2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Pfam is a large collection of protein multiple sequence alignments and profile hidden Markov models. Pfam is available on the World Wide Web in the UK at http://www.sanger.ac.uk/Software/Pfam/, in Sweden at http://www.cgb.ki.se/Pfam/, in France at http://pfam.jouy.inra.fr/ and in the US at http://pfam.wustl.edu/. The latest version (6.6) of Pfam contains 3071 families, which match 69&#37; of proteins in SWISS-PROT 39 and TrEMBL 14. Structural data, where available, have been utilised to ensure that Pfam families correspond with structural domains, and to improve domain-based annotation. Predictions of non-domain regions are now also included. In addition to secondary structure, Pfam multiple sequence alignments now contain active site residue mark-up. New search tools, including taxonomy search and domain query, greatly add to the functionality and usability of the Pfam resource.</td>
</tr>
<tr id="bib_Punta2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Punta2012,
  author = {Punta, M and Coggill, Pc and Eberhardt, Ry and Mistry, Jaina and Tate, John and Boursnell, C and Pang, N and Forslund, Kristoffer and Ceric, Goran and Clements, J and Heger, Andreas and Holm, Liisa and Sonnhammer, Ell and Eddy, Sr and Bateman, Alex and Finn, Rd},
  title = {The Pfam protein families databases.},
  journal = {Nucleic Acids Research},
  year = {2012},
  volume = {30},
  number = {1},
  pages = {1 - 12},
  doi = {https://doi.org/10.1093/nar/gkp985}
}
</pre></td>
</tr>
<tr id="Pusuluk2017" class="entry">
	<td>Pusuluk, O., Farrow, T., Deliduman, C., Burnett, K. and Vedral, V.</td>
	<td>Proton tunneling in hydrogen bonds and its possible implications in an induced-fit model of enzyme catalysis <p class="infolinks">[<a href="javascript:toggleInfo('Pusuluk2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Pusuluk2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>preprint&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Pusuluk2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The role of proton tunneling in biological catalysis remains an open question usually addressed with the tools of biochemistry. Here, we map the proton motion in a hydrogen-bonded system into a problem of pseudo-spins to allow us to approach the problem using quantum information theory and thermodynamics. We investigate the dynamics of the quantum correlations generated through two hydrogen bonds between a prototypical enzyme and a substrate, and discuss the possibility of utilizing these correlations as a resource in the catalytic power of the enzyme. In particular, we show that classical changes induced in the binding site of the enzyme spreads the quantum correlations among all of the four hydrogen-bonded atoms. If the enzyme suddenly returns to its initial state after the binding stage, the substrate ends in a quantum superposition state. Environmental effects can then naturally drive the reaction in the forward direction from the substrate to the product without needing the additional catalytic stage that is usually assumed to follow the binding stage. We find that in this scenario the enzyme lowers the activation energy to a much lower value than expected in biochemical reactions.</td>
</tr>
<tr id="bib_Pusuluk2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Pusuluk2017,
  author = {Pusuluk, Onur and Farrow, Tristan and Deliduman, Cemsinan and Burnett, Keith and Vedral, Vlatko},
  title = {Proton tunneling in hydrogen bonds and its possible implications in an induced-fit model of enzyme catalysis},
  journal = {preprint},
  year = {2017}
}
</pre></td>
</tr>
<tr id="Quaytman2009" class="entry">
	<td>Quaytman, S. and Schwartz, S.D.</td>
	<td>Comparison studies of the human heart and bacillus stearothermophilus LDH by transition path sampling <p class="infolinks">[<a href="javascript:toggleInfo('Quaytman2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>The Journal of Physical Chemistry A<br/>Vol. 113, pp. 1892 - 1897&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Quaytman2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Quaytman2009,
  author = {Quaytman, S and Schwartz, Steven D},
  title = {Comparison studies of the human heart and bacillus stearothermophilus LDH by transition path sampling},
  journal = {The Journal of Physical Chemistry A},
  year = {2009},
  volume = {113},
  pages = {1892 - 1897},
  note = {NULL}
}
</pre></td>
</tr>
<tr id="Quigley2004" class="entry">
	<td>Quigley, D. and Probert, MIJ.</td>
	<td>Langevin dynamics in constant pressure extended systems. <p class="infolinks">[<a href="javascript:toggleInfo('Quigley2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Quigley2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>The Journal of chemical physics<br/>Vol. 120(24), pp. 11432 - 41&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.1755657">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Quigley2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The advantages of performing Langevin dynamics in extended systems are discussed. A simple Langevin dynamics scheme for producing the canonical ensemble is reviewed, and is then extended to the Hoover ensemble. We show that the resulting equations of motion generate the isobaric-isothermal ensemble. The Parrinello-Rahman ensemble is then discussed and we show that despite the presence of intrinsic probability gradients in this system, a Langevin dynamics approach samples the extended phase space in the correct fashion. The implementation of these methods in the ab initio plane wave density functional theory code CASTEP [M. D. Segall, P. L. D. Lindan, M. J. Probert, C. J. Pickard, P. J. Hasnip, S. J. Clarke, and M. C. Payne, J. Phys.: Condens. Matter 14, 2717 (2003)] is demonstrated.</td>
</tr>
<tr id="bib_Quigley2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Quigley2004,
  author = {Quigley, D and Probert, MIJ},
  title = {Langevin dynamics in constant pressure extended systems.},
  journal = {The Journal of chemical physics},
  year = {2004},
  volume = {120},
  number = {24},
  pages = {11432 - 41},
  note = {Equations of motion and integrator for NPT ensemble (langevin-hoover dynamics)},
  doi = {https://doi.org/10.1063/1.1755657}
}
</pre></td>
</tr>
<tr id="Raj2014" class="entry">
	<td>Raj, S., Ramaswamy, S. and Plapp, B.V.</td>
	<td>Yeast alcohol dehydrogenase structure and catalysis. <p class="infolinks">[<a href="javascript:toggleInfo('Raj2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Raj2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Biochemistry<br/>Vol. 53(36), pp. 5791 - 803&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi5006442">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Raj2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Yeast (Saccharomyces cerevisiae) alcohol dehydrogenase I (ADH1) is the constitutive enzyme that reduces acetaldehyde to ethanol during the fermentation of glucose. ADH1 is a homotetramer of subunits with 347 amino acid residues. A structure for ADH1 was determined by X-ray crystallography at 2.4 &Aring; resolution. The asymmetric unit contains four different subunits, arranged as similar dimers named AB and CD. The unit cell contains two different tetramers made up of "back-to-back" dimers, AB:AB and CD:CD. The A and C subunits in each dimer are structurally similar, with a closed conformation, bound coenzyme, and the oxygen of 2,2,2-trifluoroethanol ligated to the catalytic zinc in the classical tetrahedral coordination with Cys-43, Cys-153, and His-66. In contrast, the B and D subunits have an open conformation with no bound coenzyme, and the catalytic zinc has an alternative, inverted coordination with Cys-43, Cys-153, His-66, and the carboxylate of Glu-67. The asymmetry in the dimeric subunits of the tetramer provides two structures that appear to be relevant for the catalytic mechanism. The alternative coordination of the zinc may represent an intermediate in the mechanism of displacement of the zinc-bound water with alcohol or aldehyde substrates. Substitution of Glu-67 with Gln-67 decreases the catalytic efficiency by 100-fold. Previous studies of structural modeling, evolutionary relationships, substrate specificity, chemical modification, and site-directed mutagenesis are interpreted more fully with the three-dimensional structure.</td>
</tr>
<tr id="bib_Raj2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Raj2014,
  author = {Raj, Savarimuthu and Ramaswamy, S and Plapp, Bryce V},
  title = {Yeast alcohol dehydrogenase structure and catalysis.},
  journal = {Biochemistry},
  year = {2014},
  volume = {53},
  number = {36},
  pages = {5791 - 803},
  note = {X-Ray crystal structure of YADH used in our studies},
  doi = {https://doi.org/10.1021/bi5006442}
}
</pre></td>
</tr>
<tr id="Rajasekaran2017" class="entry">
	<td>Rajasekaran, N., Suresh, S., Gopi, S., Raman, K. and Naganathan, A.N.</td>
	<td>A General Mechanism for the Propagation of Mutational Effects in Proteins <p class="infolinks">[<a href="javascript:toggleInfo('Rajasekaran2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Rajasekaran2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Biochemistry<br/>Vol. 56(1), pp. 294 - 305&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.biochem.6b00798">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Rajasekaran2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Mutations in the hydrophobic interior of proteins are generally thought to weaken the interactions only in their immediate neighborhood. This forms the basis of protein engineering-based studies of folding mechanism and function. However, mutational work on diverse proteins has shown that distant residues are thermodynamically coupled, with the network of interactions within the protein acting as signal conduits, thus raising an intriguing paradox. Are mutational effects localized, and if not, is there a general rule for the extent of percolation and the functional form of this propagation? We explore these questions from multiple perspectives in this work. Perturbation analysis of interaction networks within proteins and microsecond long molecular dynamics simulations of several aliphatic mutants of ubiquitin reveal strong evidence of the distinct alteration of distal residue--residue communication networks. We find that mutational effects consistently propagate into the second shell of the altered site (...</td>
</tr>
<tr id="bib_Rajasekaran2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Rajasekaran2017,
  author = {Rajasekaran, Nandakumar and Suresh, Swaathiratna and Gopi, Soundhararajan and Raman, Karthik and Naganathan, Athi N},
  title = {A General Mechanism for the Propagation of Mutational Effects in Proteins},
  journal = {Biochemistry},
  year = {2017},
  volume = {56},
  number = {1},
  pages = {294 - 305},
  doi = {https://doi.org/10.1021/acs.biochem.6b00798}
}
</pre></td>
</tr>
<tr id="Raman2017" class="entry">
	<td>Raman, S.</td>
	<td>Systems Approaches to Understanding and Designing Allosteric Proteins <p class="infolinks">[<a href="javascript:toggleInfo('Raman2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Biochemistry&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Raman2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Raman2017,
  author = {Raman, Srivatsan},
  title = {Systems Approaches to Understanding and Designing Allosteric Proteins},
  journal = {Biochemistry},
  publisher = {ACS Publications},
  year = {2017}
}
</pre></td>
</tr>
<tr id="Ramanan2011" class="entry">
	<td>Ramanan, V., Agrawal, N.J., Liu, J., Engles, S., Toy, R. and Radhakrishnan, R.</td>
	<td>Systems biology and physical biology of clathrin-mediated endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Ramanan2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ramanan2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Integrative biology : quantitative biosciences from nano to macro<br/>Vol. 3, pp. 803-815&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1039/c1ib00036e">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ramanan2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In this review, we describe the application of experimental data and modeling of intracellular endocytic trafficking mechanisms with a focus on the process of clathrin-mediated endocytosis. A detailed parts-list for the protein-protein interactions in clathrin-mediated endocytosis has been available for some time. However, recent experimental, theoretical, and computational tools have proved to be critical in establishing a sequence of events, cooperative dynamics, and energetics of the intracellular process. On the experimental front, total internal reflection fluorescence microscopy, photo-activated localization microscopy, and spinning-disk confocal microscopy have focused on assembly and patterning of endocytic proteins at the membrane, while on the theory front, minimal theoretical models for clathrin nucleation, biophysical models for membrane curvature and bending elasticity, as well as methods from computational structural and systems biology, have proved insightful in describing membrane topologies, curvature mechanisms, and energetics.</td>
</tr>
<tr id="bib_Ramanan2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ramanan2011,
  author = {Ramanan, Vyas and Agrawal, Neeraj J and Liu, Jin and Engles, Sean and Toy, Randall and Radhakrishnan, Ravi},
  title = {Systems biology and physical biology of clathrin-mediated endocytosis.},
  journal = {Integrative biology : quantitative biosciences from nano to macro},
  year = {2011},
  volume = {3},
  pages = {803--815},
  doi = {https://doi.org/10.1039/c1ib00036e}
}
</pre></td>
</tr>
<tr id="Ramanathan2014" class="entry">
	<td>Ramanathan, A., Savol, A., Burger, V., Chennubhotla, C.S. and Agarwal, P.K.</td>
	<td>Protein Conformational Populations and Functionally Relevant Substrates <p class="infolinks">[<a href="javascript:toggleInfo('Ramanathan2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Accounts of Chemical Research<br/>Vol. 47(1), pp. 149 - 156&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Ramanathan2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ramanathan2014,
  author = {Ramanathan, Arvind and Savol, Andrej and Burger, Virginia and Chennubhotla, Chakra S and Agarwal, Pratul K},
  title = {Protein Conformational Populations and Functionally Relevant Substrates},
  journal = {Accounts of Chemical Research},
  year = {2014},
  volume = {47},
  number = {1},
  pages = {149 - 156}
}
</pre></td>
</tr>
<tr id="Ramaswamy1994" class="entry">
	<td>Ramaswamy, S., Kratzer, D., Hershey, A.D., Rogers, P.H., Arnone, A., Eklund, H. and Plapp, B.V.</td>
	<td>Crystallization and preliminary crystallographic studies of Saccharomyces cerevisiae alcohol dehydrogenase I <p class="infolinks">[<a href="javascript:toggleInfo('Ramaswamy1994','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ramaswamy1994','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td>Journal of Molecular Biology, pp. 777&nbsp;</td>
	<td>misc</td>
	<td><a href="https://doi.org/10.1006/jmbi.1994.1031">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ramaswamy1994" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The cytoplasmic yeast alcohol dehydrogenase I crystallized at 5 degrees C as hexagonal plates or short columns in the presence of NAD+ and 2,2,2-trifluoroethanol, in sodium N-tris(hydroxymethyl)methyl-3-aminopropanesulfonate buffer at pH 8.2 to 8.6, using polyethylene glycol 4000 as precipitant. X-ray diffraction data to 3.2 A resolution show that the crystals are hexagonal in space group P622 with unit cell dimensions a = b = 147.9 A, c = 69.1 A. There is one subunit of the tetrameric enzyme per asymmetric unit, giving a packing density of 2.9 A3/Da.</td>
</tr>
<tr id="bib_Ramaswamy1994" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@misc{Ramaswamy1994,
  author = {Ramaswamy, S and Kratzer, Darla and Hershey, Andrew D and Rogers, Paul H and Arnone, Arthur and Eklund, Hans and Plapp, Bryce V},
  title = {Crystallization and preliminary crystallographic studies of Saccharomyces cerevisiae alcohol dehydrogenase I},
  journal = {Journal of Molecular Biology},
  publisher = {sciencedirect},
  year = {1994},
  pages = {777},
  doi = {https://doi.org/10.1006/jmbi.1994.1031}
}
</pre></td>
</tr>
<tr id="Ramjaun1998" class="entry">
	<td>Ramjaun, A. and McPherson, P.</td>
	<td>Multiple amphiphysin II splice variants display differential clathrin binding: identification of two distinct clathrin-binding sites. <p class="infolinks">[<a href="javascript:toggleInfo('Ramjaun1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ramjaun1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Journal of Neurochemistry<br/>Vol. 70(6), pp. 2369-76&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Ramjaun1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Amphiphysin I and II are nerve terminal-enriched proteins that display src homology 3 domain-mediated interactions with dynamin and synaptojanin. It has been demonstrated that the amphiphysins also bind to clathrin, and we have proposed that this interaction may help to target synaptojanin and dynamin to sites of synaptic vesicle endocytosis. To understand better this potential functional role, we have begun to characterize clathrin-amphiphysin interactions. Using PCR from adult human cortex cDNA, we have cloned a number of amphiphysin II splice variants. In in vitro binding assays, the amphiphysin II splice variants display differential clathrin binding and define a 44-amino acid region mediating the interaction. Amphiphysin II truncation and deletion mutants identify two distinct clathrin-binding domains within this region: one with the sequence LLDLDFDP, the second with the sequence PWDLW. Both domains are conserved in amphiphysin I, and saturation binding analysis demonstrates that both sites bind clathrin with approximately equal affinity. The elucidation of clathrin as a splice-specific binding partner for amphiphysin II begins to address the potential functional role(s) for the multiple amphiphysin II splice variants and further supports an important function for clathrin-amphiphysin interactions in protein targeting during endocytosis.</td>
</tr>
<tr id="bib_Ramjaun1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ramjaun1998,
  author = {Ramjaun, AR and McPherson, PS},
  title = {Multiple amphiphysin II splice variants display differential clathrin binding: identification of two distinct clathrin-binding sites.},
  journal = {Journal of Neurochemistry},
  year = {1998},
  volume = {70},
  number = {6},
  pages = {2369-76},
  note = {amph1, 2 - cltc. some domain information for amph2, but is homologous to amph1. dissociation constant.}
}
</pre></td>
</tr>
<tr id="Ramjaun1997" class="entry">
	<td>Ramjaun, A.R., Micheva, K.D., Bouchelet, I. and McPherson, P.S.</td>
	<td>Identification and characterization of a nerve terminal-enriched amphiphysin isoform. <p class="infolinks">[<a href="javascript:toggleInfo('Ramjaun1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ramjaun1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>The Journal of Biological Chemistry<br/>Vol. 272, pp. 16700-16706&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Ramjaun1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Amphiphysin is a nerve terminal-enriched protein thought to function in synaptic vesicle endocytosis, in part through Src homology 3 (SH3) domain-mediated interactions with dynamin and synaptojanin. Here, we report the characterization of a novel amphiphysin isoform (termed amphiphysin II) that was identified through a homology search of the data base of expressed sequence tags. Antibodies specific to amphiphysin II recognize a 90-kDa protein on Western blot that is brain-specific and highly enriched in nerve terminals. Like amphiphysin (now referred to as amphiphysin I), amphiphysin II binds to dynamin and synaptojanin through its SH3 domain. Further, both proteins bind directly to clathrin in an SH3 domain-independent manner. Taken together, these data suggest that amphiphysin II may participate with amphiphysin I in the regulation of synaptic vesicle endocytosis.</td>
</tr>
<tr id="bib_Ramjaun1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ramjaun1997,
  author = {Ramjaun, A R and Micheva, K D and Bouchelet, I and McPherson, P S},
  title = {Identification and characterization of a nerve terminal-enriched amphiphysin isoform.},
  journal = {The Journal of Biological Chemistry},
  year = {1997},
  volume = {272},
  pages = {16700--16706}
}
</pre></td>
</tr>
<tr id="Ranganathan1997" class="entry">
	<td>Ranganathan, S. and Gready, J.E.</td>
	<td>Hybrid quantum and molecular mechanical (QM/MM) studies on the pyruvate to L-lactate interconversion in L-lactate dehydrogenase <p class="infolinks">[<a href="javascript:toggleInfo('Ranganathan1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ranganathan1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 5647(97), pp. 5614 - 5618&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/JP971051U">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ranganathan1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Hybrid quantum mechanical (QM) and molecular mechanical (MM) calculations were undertaken to study the catalytic mechanism for the interconversion of pyruvate to l-lactate by the enzyme l-lactate dehydrogenase, in the presence of the cofactor nicotinamide adenine dinucleotide (NAD). The QM system comprised molecular species or fragments involved in the chemical bond-making and -breaking events: the substrate pyruvate, trans-1-methyldihydronicotinamide (a fragment of the cofactor), and His-195. The remainder of the enzyme, cofactor, and water molecules made up the MM system. The QM/MM potential energy surface was calculated as a grid of points for two reaction coordinates representing the transfers of a proton and of a hydride ion. From this surface, two transition states for the two transfers were identified, with the hydride-ion transfer step indicated as being rate-limiting and preceding the proton transfer. The intermediate is deprotonated l-lactate. This result disagrees with our earlier all-QM resul...</td>
</tr>
<tr id="bib_Ranganathan1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ranganathan1997,
  author = {Ranganathan, Shoba and Gready, Jill E},
  title = {Hybrid quantum and molecular mechanical (QM/MM) studies on the pyruvate to L-lactate interconversion in L-lactate dehydrogenase},
  journal = {Journal of Physical Chemistry B},
  year = {1997},
  volume = {5647},
  number = {97},
  pages = {5614 - 5618},
  note = {NULL},
  doi = {https://doi.org/10.1021/JP971051U}
}
</pre></td>
</tr>
<tr id="Rao2001" class="entry">
	<td>Rao, D.S., Chang, J.C., Kumar, P.D., Mizukami, I., Smithson, G.M., Bradley, S.V., Parlow, A.F. and Ross, T.S.</td>
	<td>Huntingtin interacting protein 1 Is a clathrin coat binding protein required for differentiation of late spermatogenic progenitors. <p class="infolinks">[<a href="javascript:toggleInfo('Rao2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Rao2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Molecular and cellular biology<br/>Vol. 21, pp. 7796-7806&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1128/MCB.21.22.7796-7806.2001">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Rao2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Huntingtin-interacting protein 1 (HIP1) interacts with huntingtin, the protein whose gene is mutated in Huntington's disease. In addition, a fusion between HIP1 and platelet-derived growth factor beta receptor causes chronic myelomonocytic leukemia. The HIP1 proteins, including HIP1 and HIP1-related (HIP1r), have an N-terminal polyphosphoinositide-interacting epsin N-terminal homology, domain, which is found in proteins involved in clathrin-mediated endocytosis. HIP1 and HIP1r also share a central leucine zipper and an actin binding TALIN homology domain. Here we show that HIP1, like HIP1r, colocalizes with clathrin coat components. We also show that HIP1 physically associates with clathrin and AP-2, the major components of the clathrin coat. To further understand the putative biological role(s) of HIP1, we have generated a targeted deletion of murine HIP1. HIP1(-/-) mice developed into adulthood, did not develop overt neurologic symptoms in the first year of life, and had normal peripheral blood counts. However, HIP1-deficient mice exhibited testicular degeneration with increased apoptosis of postmeiotic spermatids. Postmeiotic spermatids are the only cells of the seminiferous tubules that express HIP1. These findings indicate that HIP1 is required for differentiation, proliferation, and/or survival of spermatogenic progenitors. The association of HIP1 with clathrin coats and the requirement of HIP1 for progenitor survival suggest a role for HIP1 in the regulation of endocytosis.</td>
</tr>
<tr id="bib_Rao2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Rao2001,
  author = {Rao, D S and Chang, J C and Kumar, P D and Mizukami, I and Smithson, G M and Bradley, S V and Parlow, A F and Ross, T S},
  title = {Huntingtin interacting protein 1 Is a clathrin coat binding protein required for differentiation of late spermatogenic progenitors.},
  journal = {Molecular and cellular biology},
  year = {2001},
  volume = {21},
  pages = {7796--7806},
  doi = {https://doi.org/10.1128/MCB.21.22.7796-7806.2001}
}
</pre></td>
</tr>
<tr id="Rao1973" class="entry">
	<td>Rao, ST. and Rossmann, M.G.</td>
	<td>Comparison of super-secondary structures in proteins <p class="infolinks">[<a href="javascript:toggleInfo('Rao1973','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Rao1973','bibtex')">BibTeX</a>]</p></td>
	<td>1973</td>
	<td>Journal of Molecular Biology<br/>Vol. 76(2), pp. 241 - 256&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/0022-2836(73)90388-4">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Rao1973" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A method of comparing the conformations of different, but structurally related proteins is described. Local variations, such as the systematic translation of a helix, or the position of deletions and insertions can be detected, and the correspondence of only marginally similar structures can be measured. The occurrence of larger continuous folds ("super-secondary structures") has been detected in the comparison of lactate dehydrogenase with itself and with other protein structures. ?? 1973.</td>
</tr>
<tr id="bib_Rao1973" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Rao1973,
  author = {Rao, ST and Rossmann, Michael G},
  title = {Comparison of super-secondary structures in proteins},
  journal = {Journal of Molecular Biology},
  year = {1973},
  volume = {76},
  number = {2},
  pages = {241 - 256},
  note = {NULL},
  doi = {https://doi.org/10.1016/0022-2836(73)90388-4}
}
</pre></td>
</tr>
<tr id="Ray2018" class="entry">
	<td>Ray, E.L. and Reich, N.G.</td>
	<td>Prediction of infectious disease epidemics via weighted density ensembles <p class="infolinks">[<a href="javascript:toggleInfo('Ray2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>PLoS Computational Biology&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Ray2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ray2018,
  author = {Ray, Evan L and Reich, Nicholas G},
  title = {Prediction of infectious disease epidemics via weighted density ensembles},
  journal = {PLoS Computational Biology},
  year = {2018}
}
</pre></td>
</tr>
<tr id="Razzaq2001" class="entry">
	<td>Razzaq, A., Robinson, I.M., McMahon, H.T., Skepper, J.N., Su, Y., Zelhof, A.C., Jackson, A.P., Gay, N.J. and O'Kane, C.J.</td>
	<td>Amphiphysin is necessary for organization of the excitation-contraction coupling machinery of muscles, but not for synaptic vesicle endocytosis in Drosophila. <p class="infolinks">[<a href="javascript:toggleInfo('Razzaq2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Razzaq2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Genes &amp; development<br/>Vol. 15, pp. 2967-2979&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1101/gad.207801">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Razzaq2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Amphiphysins 1 and 2 are enriched in the mammalian brain and are proposed to recruit dynamin to sites of endocytosis. Shorter amphiphysin 2 splice variants are also found ubiquitously, with an enrichment in skeletal muscle. At the Drosophila larval neuromuscular junction, amphiphysin is localized postsynaptically and amphiphysin mutants have no major defects in neurotransmission; they are also viable, but flightless. Like mammalian amphiphysin 2 in muscles, Drosophila amphiphysin does not bind clathrin, but can tubulate lipids and is localized on T-tubules. Amphiphysin mutants have a novel phenotype, a severely disorganized T-tubule/sarcoplasmic reticulum system. We therefore propose that muscle amphiphysin is not involved in clathrin-mediated endocytosis, but in the structural organization of the membrane-bound compartments of the excitation-contraction coupling machinery of muscles.</td>
</tr>
<tr id="bib_Razzaq2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Razzaq2001,
  author = {Razzaq, A and Robinson, I M and McMahon, H T and Skepper, J N and Su, Y and Zelhof, A C and Jackson, A P and Gay, N J and O'Kane, C J},
  title = {Amphiphysin is necessary for organization of the excitation-contraction coupling machinery of muscles, but not for synaptic vesicle endocytosis in Drosophila.},
  journal = {Genes &amp; development},
  year = {2001},
  volume = {15},
  pages = {2967--2979},
  note = {amph doesn't participate in CME in Dros. -- no clathrin box},
  doi = {https://doi.org/10.1101/gad.207801}
}
</pre></td>
</tr>
<tr id="Read1999" class="entry">
	<td>Read, J.A.., Wilkinson, KW., Tranter, R., Sessions, RB. and Brady, RL.</td>
	<td>Chloroquine binds in the cofactor binding site of Plasmodium falciparum lactate dehydrogenase. <p class="infolinks">[<a href="javascript:toggleInfo('Read1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Read1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Journal of Biological Chemistry<br/>Vol. 274(15), pp. 10213 - 8&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/JBC.274.15.10213">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Read1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Although the molecular mechanism by which chloroquine exerts its effects on the malarial parasite Plasmodium falciparum remains unclear, the drug has previously been found to interact specifically with the glycolytic enzyme lactate dehydrogenase from the parasite. In this study we have determined the crystal structure of the complex between chloroquine and P. falciparum lactate dehydrogenase. The bound chloroquine is clearly seen within the NADH binding pocket of the enzyme, occupying a position similar to that of the adenyl ring of the cofactor. Chloroquine hence competes with NADH for binding to the enzyme, acting as a competitive inhibitor for this critical glycolytic enzyme. Specific interactions between the drug and amino acids unique to the malarial form of the enzyme suggest this binding is selective. Inhibition studies confirm that chloroquine acts as a weak inhibitor of lactate dehydrogenase, with mild selectivity for the parasite enzyme. As chloroquine has been shown to accumulate to millimolar concentrations within the food vacuole in the gut of the parasite, even low levels of inhibition may contribute to the biological efficacy of the drug. The structure of this enzyme-inhibitor complex provides a template from which the quinoline moiety might be modified to develop more efficient inhibitors of the enzyme.</td>
</tr>
<tr id="bib_Read1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Read1999,
  author = {Read, J.A. and Wilkinson, KW and Tranter, R and Sessions, RB and Brady, RL},
  title = {Chloroquine binds in the cofactor binding site of Plasmodium falciparum lactate dehydrogenase.},
  journal = {Journal of Biological Chemistry},
  year = {1999},
  volume = {274},
  number = {15},
  pages = {10213 - 8},
  doi = {https://doi.org/10.1074/JBC.274.15.10213}
}
</pre></td>
</tr>
<tr id="Read2001" class="entry">
	<td>Read, J.A.., Winter, V.J.., Eszes, C.M.., Sessions, R.B.. and Brady, R.L..</td>
	<td>Structural basis for altered activity of M- and H-isozyme forms of human lactate dehydrogenase <p class="infolinks">[<a href="javascript:toggleInfo('Read2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Proteins: Structure, Function, and Bioinformatics<br/>Vol. 43(2), pp. 175 - 185&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Read2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Read2001,
  author = {Read, J.A. and Winter, V.J. and Eszes, C.M. and Sessions, R.B. and Brady, R.L.},
  title = {Structural basis for altered activity of M- and H-isozyme forms of human lactate dehydrogenase},
  journal = {Proteins: Structure, Function, and Bioinformatics},
  year = {2001},
  volume = {43},
  number = {2},
  pages = {175 - 185},
  note = {NULL}
}
</pre></td>
</tr>
<tr id="Rebholz-Schuhmann2005" class="entry">
	<td>Rebholz-Schuhmann, D., Kirsch, H. and Couto, F.</td>
	<td>Facts from Text -- Is Text Mining Ready to Deliver? <p class="infolinks">[<a href="javascript:toggleInfo('Rebholz-Schuhmann2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>PLoS Biology<br/>Vol. 3(2), pp. e65&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pbio.0030065">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Rebholz-Schuhmann2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Rebholz-Schuhmann2005,
  author = {Rebholz-Schuhmann, Dietrich and Kirsch, Harald and Couto, Francisco},
  title = {Facts from Text -- Is Text Mining Ready to Deliver?},
  journal = {PLoS Biology},
  year = {2005},
  volume = {3},
  number = {2},
  pages = {e65},
  doi = {https://doi.org/10.1371/journal.pbio.0030065}
}
</pre></td>
</tr>
<tr id="Reddish2017" class="entry">
	<td>Reddish, M.J., Callender, R. and Dyer, B.R.</td>
	<td>Resolution of Submillisecond Kinetics of Multiple Reaction Pathways for Lactate Dehydrogenase <p class="infolinks">[<a href="javascript:toggleInfo('Reddish2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Reddish2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Biophysical Journal<br/>Vol. 112(9), pp. 1852-1862&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.bpj.2017.03.031">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Reddish2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Enzymes are known to exhibit conformational flexibility. An important consequence of this flexibility is that the same enzyme reaction can occur via multiple reaction pathways on a reaction landscape. A model enzyme for the study of reaction landscapes is lactate dehydrogenase. We have previously used temperature-jump (T-jump) methods to demonstrate that the reaction landscape of lactate dehydrogenase branches at multiple points creating pathways with varied reactivity. A limitation of this previous work is that the T-jump method makes only small pertur-bations to equilibrium and may not report conclusively on all steps in a reaction. Therefore, interpreting T-jump results of lactate dehydrogenase kinetics has required extensive computational modeling work. Rapid mixing methods offer a complementary approach that can access large perturbations from equilibrium; however, traditional enzyme mixing methods like stopped-flow do not allow for the observation of fast protein dynamics. In this report, we apply a microfluidic rapid mixing device with a mixing time of \textless\100 ms that allows us to study these fast dynamics and the catalytic redox step of the enzyme reaction. Additionally, we report UV absorbance and emission T-jump results with improved signal-to-noise ratio at fast times. The combination of mixing and T-jump results yields an unprecedented view of lactate dehydrogenase enzymology, confirming the timescale of substrate-induced conformational change and presence of multiple reaction pathways.</td>
</tr>
<tr id="bib_Reddish2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Reddish2017,
  author = {Reddish, Michael J and Callender, Robert and Dyer, Brian R},
  title = {Resolution of Submillisecond Kinetics of Multiple Reaction Pathways for Lactate Dehydrogenase},
  journal = {Biophysical Journal},
  year = {2017},
  volume = {112},
  number = {9},
  pages = {1852-1862},
  doi = {https://doi.org/10.1016/j.bpj.2017.03.031}
}
</pre></td>
</tr>
<tr id="Reider2009" class="entry">
	<td>Reider, A., Barker, S.L., Mishra, S.K., Im, Y., Lymarie, Maldonado-B., Hurley, J.H., Traub, L.M. and Wendland, B.</td>
	<td>Syp1 is a conserved endocytic adaptor that contains domains involved in cargo selection and membrane tubulation. <p class="infolinks">[<a href="javascript:toggleInfo('Reider2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Reider2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>The EMBO Journal<br/>Vol. 28(20), pp. 3103 - 16&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/emboj.2009.248">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Reider2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Internalization of diverse transmembrane cargos from the plasma membrane requires a similarly diverse array of specialized adaptors, yet only a few adaptors have been characterized. We report the identification of the muniscin family of endocytic adaptors that is conserved from yeast to human beings. Solving the structures of yeast muniscin domains confirmed the unique combination of an N-terminal domain homologous to the crescent-shaped membrane-tubulating EFC/F-BAR domains and a C-terminal domain homologous to cargo-binding mu homology domains (muHDs). In vitro and in vivo assays confirmed membrane-tubulation activity for muniscin EFC/F-BAR domains. The muHD domain has conserved interactions with the endocytic adaptor/scaffold Ede1/eps15, which influences muniscin localization. The transmembrane protein Mid2, earlier implicated in polarized Rho1 signalling, was identified as a cargo of the yeast adaptor protein. These and other data suggest a model in which the muniscins provide a combined adaptor/membrane-tubulation activity that is important for regulating endocytosis.</td>
</tr>
<tr id="bib_Reider2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Reider2009,
  author = {Reider, Amanda and Barker, Sarah L and Mishra, Sanjay K and Im, Young and Lymarie, Maldonado-B&aacute;ez, and Hurley, James H and Traub, Linton M and Wendland, Beverly},
  title = {Syp1 is a conserved endocytic adaptor that contains domains involved in cargo selection and membrane tubulation.},
  journal = {The EMBO Journal},
  year = {2009},
  volume = {28},
  number = {20},
  pages = {3103 - 16},
  note = {ede1 c-terminal domain is too different from itsn1-L to use these results},
  doi = {https://doi.org/10.1038/emboj.2009.248}
}
</pre></td>
</tr>
<tr id="Reider2011" class="entry">
	<td>Reider, A. and Wendland, B.</td>
	<td>Endocytic adaptors--social networking at the plasma membrane. <p class="infolinks">[<a href="javascript:toggleInfo('Reider2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Reider2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Journal of cell science<br/>Vol. 124, pp. 1613-1622&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1242/jcs.073395">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Reider2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Receptor-mediated endocytosis is a dynamic process that is crucial for maintaining plasma membrane composition and controlling cell-signaling pathways. A variety of entry routes have evolved to ensure that the vast array of molecules on the cell surface can be differentially internalized by endocytosis. This diversity has extended to include a growing list of endocytic adaptor proteins, which are thought to initiate the internalization process. The key function of adaptors is to select the proteins that should be removed from the cell surface. Thus, they have a central role in defining the physiology of a cell. This has made the study of adaptor proteins a very active area of research that is ripe for exciting future discoveries. Here, we review recent work on how adaptors mediate endocytosis and address the following questions: what characteristics define an endocytic adaptor protein? What roles do these proteins fulfill in addition to selecting cargo and how might adaptors function in clathrin-independent endocytic pathways? Through the findings discussed in this Commentary, we hope to stimulate further characterization of known adaptors and expansion of the known repertoire by identification of new adaptors.</td>
</tr>
<tr id="bib_Reider2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Reider2011,
  author = {Reider, Amanda and Wendland, Beverly},
  title = {Endocytic adaptors--social networking at the plasma membrane.},
  journal = {Journal of cell science},
  year = {2011},
  volume = {124},
  pages = {1613--1622},
  note = {table of accessory proteins and their interactions.},
  doi = {https://doi.org/10.1242/jcs.073395}
}
</pre></td>
</tr>
<tr id="Ren2006" class="entry">
	<td>Ren, G., Vajjhala, P., Lee, J.S., Winsor, B. and Munn, A.L.</td>
	<td>The BAR domain proteins: molding membranes in fission, fusion, and phagy. <p class="infolinks">[<a href="javascript:toggleInfo('Ren2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ren2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Microbiology and molecular biology reviews : MMBR<br/>Vol. 70, pp. 37-120&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1128/MMBR.70.1.37-120.2006">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ren2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The Bin1/amphiphysin/Rvs167 (BAR) domain proteins are a ubiquitous protein family. Genes encoding members of this family have not yet been found in the genomes of prokaryotes, but within eukaryotes, BAR domain proteins are found universally from unicellular eukaryotes such as yeast through to plants, insects, and vertebrates. BAR domain proteins share an N-terminal BAR domain with a high propensity to adopt alpha-helical structure and engage in coiled-coil interactions with other proteins. BAR domain proteins are implicated in processes as fundamental and diverse as fission of synaptic vesicles, cell polarity, endocytosis, regulation of the actin cytoskeleton, transcriptional repression, cell-cell fusion, signal transduction, apoptosis, secretory vesicle fusion, excitation-contraction coupling, learning and memory, tissue differentiation, ion flux across membranes, and tumor suppression. What has been lacking is a molecular understanding of the role of the BAR domain protein in each process. The three-dimensional structure of the BAR domain has now been determined and valuable insight has been gained in understanding the interactions of BAR domains with membranes. The cellular roles of BAR domain proteins, characterized over the past decade in cells as distinct as yeasts, neurons, and myocytes, can now be understood in terms of a fundamental molecular function of all BAR domain proteins: to sense membrane curvature, to bind GTPases, and to mold a diversity of cellular membranes.</td>
</tr>
<tr id="bib_Ren2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ren2006,
  author = {Ren, Gang and Vajjhala, Parimala and Lee, Janet S and Winsor, Barbara and Munn, Alan L},
  title = {The BAR domain proteins: molding membranes in fission, fusion, and phagy.},
  journal = {Microbiology and molecular biology reviews : MMBR},
  year = {2006},
  volume = {70},
  pages = {37--120},
  doi = {https://doi.org/10.1128/MMBR.70.1.37-120.2006}
}
</pre></td>
</tr>
<tr id="Riccardi2006" class="entry">
	<td>Riccardi, D., Schaefer, P., Yang, Y., Yu, H., Ghosh, N., Xavier, Prat-Resina,., K&ouml;nig, P., Li, G., Xu, D., Guo, H., Elstner, M. and Cui, Q.</td>
	<td>Development of effective quantum mechanical/molecular mechanical (QM/MM) methods for complex biological processes. <p class="infolinks">[<a href="javascript:toggleInfo('Riccardi2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Riccardi2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 110(13), pp. 6458 - 69&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp056361o">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Riccardi2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Motivated by the long-term goal of understanding vectorial biological processes such as proton transport (PT) in biomolecular ion pumps, a number of developments were made to establish combined quantum mechanical/molecular mechanical (QM/MM) methods suitable for studying chemical reactions involving significant charge separation in the condensed phase. These developments were summarized and discussed with representative problems. Specifically, free energy perturbation and boundary potential methods for treating long-range electrostatics were implemented to test the robustness of QM/MM results for protein systems. It was shown that consistent models with sufficient sampling were able to produce quantitatively satisfactory results, such as pKa for titritable groups in the interior of T4-lysozyme, while an inconsistent treatment of electrostatics or lack of sufficient sampling may produce incorrect results. Modifications were made to an approximate density functional theory (SCC-DFTB) to improve the description of proton affinity and hydrogen-bonding, which are crucial for the treatment of PT in polar systems. Test calculations on water autoionization showed clearly that both improvements are necessary for quantitatively reliable results. Finally, the newly established SCC-DFTB/MM-GSBP protocol was used to explore mechanistic issues in carbonic anhydrase (CA). Preliminary results suggest that PT in CA occurs mainly through short water wires containing two water molecules in a thermally activated fashion. Although longer water wires occur with similar frequencies, PT along those pathways, on average, has substantially higher barriers, a result not expected based on previous studies. The fluctuations of water molecules peripheral to the water wire were found to make a larger impact on the PT energetics compared to polar protein residues in the active site, which are largely pre-organized and therefore have less tendency to reorganize during the reaction.</td>
</tr>
<tr id="bib_Riccardi2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Riccardi2006,
  author = {Riccardi, Demian and Schaefer, Patricia and Yang, Yang and Yu, Haibo and Ghosh, Nilanjan and Xavier, Prat-Resina, and K&ouml;nig, Peter and Li, Guohui and Xu, Dingguo and Guo, Hua and Elstner, Marcus and Cui, Qiang},
  title = {Development of effective quantum mechanical/molecular mechanical (QM/MM) methods for complex biological processes.},
  journal = {Journal of Physical Chemistry B},
  year = {2006},
  volume = {110},
  number = {13},
  pages = {6458 - 69},
  doi = {https://doi.org/10.1021/jp056361o}
}
</pre></td>
</tr>
<tr id="Rickert1999" class="entry">
	<td>Rickert, K.W. and Klinman, J.P.</td>
	<td>Nature of hydrogen transfer in soybean lipoxygenase 1: Separation of primary and secondary isotope effects <p class="infolinks">[<a href="javascript:toggleInfo('Rickert1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Rickert1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Biochemistry<br/>Vol. 38(38), pp. 12218 - 12228&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi990834y">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Rickert1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Previous measurements of the kinetics of oxidation of linoleic acid by soybean lipoxygenase 1 have indicated very large deuterium isotope effects, but have not been able to distinguish the primary isotope effect from the alpha-secondary effect. To address this question, singly deuterated linoleic acid was prepared, and enantiomerically resolved using the enzyme itself. Noncompetitive measurements of the primary deuterium isotope effect give a value of ca. 40 which is temperature-independent. The enthalpy of activation is low and isotope-independent, and there is a large isotope effect on the Arrhenius prefactor. A very large apparent secondary isotope effect (ca. 2.1) is measured with deuterium in the primary position, but a greatly reduced value (1.1) is observed with protium in the primary position. Mutagenesis of the active site leads to a significant reduction in k(cat) and perturbed isotope effects, in particular, a secondary effect of 5.6 when deuterium is in the primary position. The anomalous secondary isotope effects are shown to arise from imperfect stereoselectivity of hydrogen abstraction which, for the mutant, is attributed to a combination of inverse substrate binding and increased flexibility at the reactive carbon. After correction, a very large primary (76-84) and small secondary (1.1-1.2) kinetic isotope effects are calculated for both mutant and wild-type enzymes. The weight of the evidence is taken to favor hydrogen tunneling as the primary mechanism of hydrogen transfer.</td>
</tr>
<tr id="bib_Rickert1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Rickert1999,
  author = {Rickert, Keith W and Klinman, Judith P},
  title = {Nature of hydrogen transfer in soybean lipoxygenase 1: Separation of primary and secondary isotope effects},
  journal = {Biochemistry},
  year = {1999},
  volume = {38},
  number = {38},
  pages = {12218 - 12228},
  doi = {https://doi.org/10.1021/bi990834y}
}
</pre></td>
</tr>
<tr id="Ricotta2002" class="entry">
	<td>Ricotta, D., Conner, S.D., Schmid, S.L., von Figura, K. and H&ouml;ning, S.</td>
	<td>Phosphorylation of the AP2 &mu; subunit by AAK1 mediates high affinity binding to membrane protein sorting signals <p class="infolinks">[<a href="javascript:toggleInfo('Ricotta2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>The Journal of cell biology<br/>Vol. 156(5), pp. 791-795&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Ricotta2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ricotta2002,
  author = {Ricotta, Doris and Conner, Sean D and Schmid, Sandra L and von Figura, Kurt and H&ouml;ning, Stefan},
  title = {Phosphorylation of the AP2 &mu; subunit by AAK1 mediates high affinity binding to membrane protein sorting signals},
  journal = {The Journal of cell biology},
  publisher = {Rockefeller University Press},
  year = {2002},
  volume = {156},
  number = {5},
  pages = {791--795}
}
</pre></td>
</tr>
<tr id="Ritter2013" class="entry">
	<td>Ritter, B., Murphy, S., Dokainish, H., Girard, M., Gudheti, M.V., Kozlov, G., Halin, M., Philie, J., Jorgensen, E.M., Gehring, K. and others</td>
	<td>NECAP 1 regulates AP-2 interactions to control vesicle size, number, and cargo during clathrin-mediated endocytosis <p class="infolinks">[<a href="javascript:toggleInfo('Ritter2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>PLoS biology<br/>Vol. 11(10), pp. e1001670&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Ritter2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ritter2013,
  author = {Ritter, Brigitte and Murphy, Sebastian and Dokainish, Hatem and Girard, Martine and Gudheti, Manasa V and Kozlov, Guennadi and Halin, Marilene and Philie, Jacynthe and Jorgensen, Erik M and Gehring, Kalle and others},
  title = {NECAP 1 regulates AP-2 interactions to control vesicle size, number, and cargo during clathrin-mediated endocytosis},
  journal = {PLoS biology},
  publisher = {Public Library of Science},
  year = {2013},
  volume = {11},
  number = {10},
  pages = {e1001670}
}
</pre></td>
</tr>
<tr id="Ritter2003" class="entry">
	<td>Ritter, B., Philie, J., Girard, M., Tung, E.C., Blondeau, F. and McPherson, P.S.</td>
	<td>Identification of a family of endocytic proteins that define a new alpha-adaptin ear-binding motif. <p class="infolinks">[<a href="javascript:toggleInfo('Ritter2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ritter2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>EMBO reports<br/>Vol. 4, pp. 1089-1095&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/sj.embor.embor7400004">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ritter2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Endocytosis by clathrin-coated vesicles (CCVs) is an important mechanism mediating protein internalization. Here, we show that two proteins identified through a proteomics analysis of CCVs are new components of the endocytic machinery. The proteins, named NECAP (adaptin-ear-binding coat-associated protein) 1 and 2, are paralogues that display no sequence similarity or common domains with any known protein. Both are enriched in CCV coats, and further analysis of the brain-enriched isoform, NECAP 1, shows its partial localization to clathrin-coated pits and direct binding to the globular ear domain of the alpha-adaptin subunit (alpha-ear) of the adaptor protein 2 (AP-2) complex. Intriguingly, this interaction is mediated by a new motif, WVQF, that uses a distinct alpha-ear interface relative to known alpha-ear-binding partners. Disruption of this interaction blocks clathrin-mediated endocytosis. Together, our studies identify a new family of endocytic proteins that define a unique AP-2-binding motif.</td>
</tr>
<tr id="bib_Ritter2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ritter2003,
  author = {Ritter, Brigitte and Philie, Jacynthe and Girard, Martine and Tung, Elaine C and Blondeau, Francois and McPherson, Peter S},
  title = {Identification of a family of endocytic proteins that define a new alpha-adaptin ear-binding motif.},
  journal = {EMBO reports},
  year = {2003},
  volume = {4},
  pages = {1089--1095},
  doi = {https://doi.org/10.1038/sj.embor.embor7400004}
}
</pre></td>
</tr>
<tr id="Robert2014" class="entry">
	<td>Robert, X. and Gouet, P.</td>
	<td>Deciphering key features in protein structures with the new ENDscript server <p class="infolinks">[<a href="javascript:toggleInfo('Robert2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Robert2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Nucleic Acids Research<br/>Vol. 42(W1), pp. W320 - W324&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/nar/gku316">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Robert2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: ENDscript 2 is a friendly Web server for extracting and rendering a comprehensive analysis of primary to quaternary protein structure information in an automated way. This major upgrade has been fully re-engineered to enhance speed, accuracy and usability with interactive 3D visualization. It takes advantage of the new version 3 of ESPript, our well-known sequence alignment renderer, improved to handle a large number of data with reduced computation time. From a single PDB entry or file, ENDscript produces high quality figures displaying multiple sequence alignment of proteins homologous to the query, colored according to residue conservation. Furthermore, the experimental secondary structure elements and a detailed set of relevant biophysical and structural data are depicted. All this information and more are now mapped on interactive 3D PyMOL representations. Thanks to its adaptive and rigorous algorithm, beginner to expert users can modify settings to fine-tune ENDscript to their needs. ENDscript has also been upgraded as an open platform for the visualization of multiple biochemical and structural data coming from external biotool Web servers, with both 2D and 3D representations. ENDscript 2 and ESPript 3 are freely available at http://endscript.ibcp.fr and http://espript.ibcp.fr, respectively.</td>
</tr>
<tr id="bib_Robert2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Robert2014,
  author = {Robert, Xavier and Gouet, Patrice},
  title = {Deciphering key features in protein structures with the new ENDscript server},
  journal = {Nucleic Acids Research},
  year = {2014},
  volume = {42},
  number = {W1},
  pages = {W320 - W324},
  doi = {https://doi.org/10.1093/nar/gku316}
}
</pre></td>
</tr>
<tr id="Roberts2006" class="entry">
	<td>Roberts, P.M.</td>
	<td>Mining literature for systems biology <p class="infolinks">[<a href="javascript:toggleInfo('Roberts2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Roberts2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Briefings in Bioinformatics<br/>Vol. 7(4), pp. 399 - 406&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/bib/bbl037">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Roberts2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Currently, literature is integrated in systems biology studies in three ways. Hand-curated pathways have been sufficient for assembling models in numerous studies. Second, literature is frequently accessed in a derived form, such as the concepts represented by the Medical Subject Headings (MeSH) and Gene Ontologies (GO), or functional relationships captured in protein-protein interaction (PPI) databases; both of these are convenient, consistent reductions of more complex concepts expressed as free text in the literature. Moreover, their contents are easily integrated into computational processes required for dealing with large data sets. Last, mining text directly for specific types of information is on the rise as text analytics methods become more accurate and accessible. These uses of literature, specifically manual curation, derived concepts captured in ontologies and databases, and indirect and direct application of text mining, will be discussed as they pertain to systems biology.</td>
</tr>
<tr id="bib_Roberts2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Roberts2006,
  author = {Roberts, Phoebe M},
  title = {Mining literature for systems biology},
  journal = {Briefings in Bioinformatics},
  year = {2006},
  volume = {7},
  number = {4},
  pages = {399 - 406},
  doi = {https://doi.org/10.1093/bib/bbl037}
}
</pre></td>
</tr>
<tr id="Robinson2001" class="entry">
	<td>Robinson, MS. and Bonifacino, JS.</td>
	<td>Adaptor-related proteins <p class="infolinks">[<a href="javascript:toggleInfo('Robinson2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Robinson2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Current Opinion in Cell Biology<br/>Vol. 13(4), pp. 444 - 453&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S0955-0674(00)00235-0">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Robinson2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Two new adaptor-related protein complexes, AP-3 and AP-4, have recently been identified, and both have been implicated in protein sorting at the trans-Golgi network (TGN) and/or endosomes. In addition, two families of monomeric proteins with adaptor-related domains, the GGAs and the stoned B family, have also been identified and shown to act at the TGN and plasma membrane, respectively. Together with the two conventional adaptors, AP-1 and AP-2, these proteins may act to direct different types of cargo proteins to different post-Golgi membrane compartments.</td>
</tr>
<tr id="bib_Robinson2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Robinson2001,
  author = {Robinson, MS and Bonifacino, JS},
  title = {Adaptor-related proteins},
  journal = {Current Opinion in Cell Biology},
  year = {2001},
  volume = {13},
  number = {4},
  pages = {444 - 453},
  doi = {https://doi.org/10.1016/S0955-0674(00)00235-0}
}
</pre></td>
</tr>
<tr id="Robinson2004" class="entry">
	<td>Robinson, M.S.</td>
	<td>Adaptable adaptors for coated vesicles <p class="infolinks">[<a href="javascript:toggleInfo('Robinson2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Robinson2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Trends in Cell Biology<br/>Vol. 14, pp. 167-174&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.tcb.2004.02.002">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Robinson2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Adaptors select cargo for inclusion into coated vesicles in the late secretory and endocytic pathways. Although originally there were thought to be just two adaptors, AP-1 and AP-2, it is now clear that there are many more: two additional adaptor complexes, AP-3 and AP-4, which might function independently of clathrin; a family of monomeric adaptors, the GGAs; and an ever-growing number of cargo-specific adaptors. The adaptors are targeted to the appropriate membrane at least in part by interacting with phosphoinositides, and, once on the membrane, they form interconnected networks to get different types of cargo into the same vesicle. Adaptors participate in trafficking pathways shared by all cells, and they are also used to generate specialized organelles and to influence cell fate during development.</td>
</tr>
<tr id="bib_Robinson2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Robinson2004,
  author = {Robinson, Margaret S},
  title = {Adaptable adaptors for coated vesicles},
  journal = {Trends in Cell Biology},
  year = {2004},
  volume = {14},
  pages = {167--174},
  doi = {https://doi.org/10.1016/j.tcb.2004.02.002}
}
</pre></td>
</tr>
<tr id="Robinson2015" class="entry">
	<td>Robinson, M.S.</td>
	<td>Forty Years of Clathrin-coated Vesicles <p class="infolinks">[<a href="javascript:toggleInfo('Robinson2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Traffic<br/>Vol. 16(12), pp. 1210-1238&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Robinson2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Robinson2015,
  author = {Robinson, Margaret S},
  title = {Forty Years of Clathrin-coated Vesicles},
  journal = {Traffic},
  publisher = {Wiley Online Library},
  year = {2015},
  volume = {16},
  number = {12},
  pages = {1210--1238},
  note = {disease information}
}
</pre></td>
</tr>
<tr id="Rodemer2008" class="entry">
	<td>Rodemer, C. and Haucke, V.</td>
	<td>Clathrin/AP-2-Dependent Endocytosis: A Novel Playground for the Pharmacological Toolbox? <p class="infolinks">[<a href="javascript:toggleInfo('Rodemer2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Rodemer2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Protein-Protein Interactions as New Drug Targets, pp. 105-122&nbsp;</td>
	<td>inbook</td>
	<td><a href="https://doi.org/10.1007/978-3-540-72843-6_5">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Rodemer2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Endocytosis is a vital process for mammalian cells by which they communicate with their environment, internalize nutrients, hormones, or growth factors, or take up extracellular fluids and particles. The best studied among the various pathways to ingest material from the extracellular side is clathrin/AP-2-mediated endocytosis. The past several years have allowed us to gain unprecedented molecular insights into the role of the heterotetrameric AP-2 adaptor complex as a central protein-protein and protein-lipid interaction hub at the plasmalemma. During the initial stages of clathrin-coated pit formation, AP-2 interacts with phosphoinositides and cargo membrane proteins as well as with a variety of accessory proteins and clathrin to coordinate clathrin coat polymerization with membrane deformation and cargo recruitment. In addition, a growing list of alternative adaptors provides opportunity for clathrin-dependent cargo selective pathways of internalization and endosomal sorting. Many of these interactions are now understood in structural detail and are thus amenable to pharmacological interference. In this review we will summarize our present state of knowledge about AP-2 and its partners in endocytosis and delineate potential strategies for pharmacological manipulations.</td>
</tr>
<tr id="bib_Rodemer2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inbook{Rodemer2008,
  author = {Rodemer, C. and Haucke, V.},
  title = {Clathrin/AP-2-Dependent Endocytosis: A Novel Playground for the Pharmacological Toolbox?},
  booktitle = {Protein-Protein Interactions as New Drug Targets},
  publisher = {Springer Berlin Heidelberg},
  year = {2008},
  pages = {105--122},
  doi = {https://doi.org/10.1007/978-3-540-72843-6_5}
}
</pre></td>
</tr>
<tr id="Rodriguez2018" class="entry">
	<td>Rodriguez, A., d'Errico , M., Facco, E. and Laio, A.</td>
	<td>Computing the Free Energy without Collective Variables <p class="infolinks">[<a href="javascript:toggleInfo('Rodriguez2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Journal of Chemical Theory and Computation&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jctc.7b00916">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Rodriguez2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Rodriguez2018,
  author = {Alex Rodriguez and Maria d'Errico and Elena Facco and Alessandro Laio},
  title = {Computing the Free Energy without Collective Variables},
  journal = {Journal of Chemical Theory and Computation},
  publisher = {American Chemical Society (ACS)},
  year = {2018},
  doi = {https://doi.org/10.1021/acs.jctc.7b00916}
}
</pre></td>
</tr>
<tr id="Rohde2002" class="entry">
	<td>Rohde, G., Wenzel, D. and Haucke, V.</td>
	<td>A phosphatidylinositol (4,5)-bisphosphate binding site within mu2-adaptin regulates clathrin-mediated endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Rohde2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Rohde2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>The Journal of cell biology<br/>Vol. 158, pp. 209-214&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1083/jcb.200203103">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Rohde2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The clathrin adaptor complex AP-2 serves to coordinate clathrin-coated pit assembly with the sorting of transmembrane cargo proteins at the plasmalemma. How precisely AP-2 assembly and cargo protein recognition at sites of endocytosis are regulated has remained unclear, but recent evidence implicates phosphoinositides, in particular phosphatidylinositol (4,5)-bisphosphate (PI[4,5]P2), in these processes. Here we have identified and functionally characterized a conserved binding site for PI(4,5)P2 within mu2-adaptin, the medium chain of the clathrin adaptor complex AP-2. Mutant mu2 lacking a cluster of conserved lysine residues fails to bind PI(4,5)P2 and to compete the recruitment of native clathrin/AP-2 to PI(4,5)P2-containing liposomes or to presynaptic membranes. Moreover, we show that expression of mutant mu2 inhibits receptor-mediated endocytosis in living cells. We suggest that PI(4,5)P2 binding to mu2-adaptin regulates clathrin-mediated endocytosis and thereby may contribute to structurally linking cargo recognition to coat formation.</td>
</tr>
<tr id="bib_Rohde2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Rohde2002,
  author = {Rohde, Gundula and Wenzel, Dirk and Haucke, Volker},
  title = {A phosphatidylinositol (4,5)-bisphosphate binding site within mu2-adaptin regulates clathrin-mediated endocytosis.},
  journal = {The Journal of cell biology},
  year = {2002},
  volume = {158},
  pages = {209--214},
  doi = {https://doi.org/10.1083/jcb.200203103}
}
</pre></td>
</tr>
<tr id="Rosenthal1999" class="entry">
	<td>Rosenthal, J.A., Chen, H., Slepnev, V.I., Pellegrini, L., Salcini, A.E., Di Fiore, P.P. and De Camilli, P.</td>
	<td>The epsins define a family of proteins that interact with components of the clathrin coat and contain a new protein module <p class="infolinks">[<a href="javascript:toggleInfo('Rosenthal1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Journal of Biological Chemistry<br/>Vol. 274(48), pp. 33959-33965&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Rosenthal1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Rosenthal1999,
  author = {Rosenthal, Julie A and Chen, Hong and Slepnev, Vladimir I and Pellegrini, Lorenzo and Salcini, Anna Elisabetta and Di Fiore, Pier Paolo and De Camilli, Pietro},
  title = {The epsins define a family of proteins that interact with components of the clathrin coat and contain a new protein module},
  journal = {Journal of Biological Chemistry},
  publisher = {ASBMB},
  year = {1999},
  volume = {274},
  number = {48},
  pages = {33959--33965}
}
</pre></td>
</tr>
<tr id="Roston2014" class="entry">
	<td>Roston, D., Islam, Z. and Kohen, A.</td>
	<td>Kinetic isotope effects as a probe of hydrogen transfers to and from common enzymatic cofactors. <p class="infolinks">[<a href="javascript:toggleInfo('Roston2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Roston2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Archives of biochemistry and biophysics<br/>Vol. 544, pp. 96 - 104&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.abb.2013.10.010">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Roston2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Enzymes use a number of common cofactors as sources of hydrogen to drive biological processes, but the physics of the hydrogen transfers to and from these cofactors is not fully understood. Researchers study the mechanistically important contributions from quantum tunneling and enzyme dynamics and connect those processes to the catalytic power of enzymes that use these cofactors. Here we describe some progress that has been made in studying these reactions, particularly through the use of kinetic isotope effects (KIEs). We first discuss the general theoretical framework necessary to interpret experimental KIEs, and then describe practical uses for KIEs in the context of two case studies. The first example is alcohol dehydrogenase, which uses a nicotinamide cofactor to catalyze a hydride transfer, and the second example is thymidylate synthase, which uses a folate cofactor to catalyze both a hydride and a proton transfer.</td>
</tr>
<tr id="bib_Roston2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Roston2014,
  author = {Roston, Daniel and Islam, Zahidul and Kohen, Amnon},
  title = {Kinetic isotope effects as a probe of hydrogen transfers to and from common enzymatic cofactors.},
  journal = {Archives of biochemistry and biophysics},
  year = {2014},
  volume = {544},
  pages = {96 - 104},
  doi = {https://doi.org/10.1016/j.abb.2013.10.010}
}
</pre></td>
</tr>
<tr id="Roston2010" class="entry">
	<td>Roston, D. and Kohen, A.</td>
	<td>Elusive transition state of alcohol dehydrogenase unveiled. <p class="infolinks">[<a href="javascript:toggleInfo('Roston2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Roston2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Proceedings of the National Academy of Sciences of the United States of America<br/>Vol. 107(21), pp. 9572 - 7&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.1000931107">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Roston2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: For several decades the hydride transfer catalyzed by alcohol dehydrogenase has been difficult to understand. Here we add to the large corpus of anomalous and paradoxical data collected for this reaction by measuring a normal (\textgreater\ 1) 2 degrees kinetic isotope effect (KIE) for the reduction of benzaldehyde. Because the relevant equilibrium effect is inverse (\textless\ 1), this KIE eludes the traditional interpretation of 2 degrees KIEs. It does, however, enable the development of a comprehensive model for the "tunneling ready state" (TRS) of the reaction that fits into the general scheme of Marcus-like models of hydrogen tunneling. The TRS is the ensemble of states along the intricate reorganization coordinate, where H tunneling between the donor and acceptor occurs (the crossing point in Marcus theory). It is comparable to the effective transition state implied by ensemble-averaged variational transition state theory. Properties of the TRS are approximated as an average of the individual properties of the donor and acceptor states. The model is consistent with experimental findings that previously appeared contradictory; specifically, it resolves the long-standing ambiguity regarding the location of the TRS (aldehyde-like vs. alcohol-like). The new picture of the TRS for this reaction identifies the principal components of the collective reaction coordinate and the average structure of the saddle point along that coordinate.</td>
</tr>
<tr id="bib_Roston2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Roston2010,
  author = {Roston, Daniel and Kohen, Amnon},
  title = {Elusive transition state of alcohol dehydrogenase unveiled.},
  journal = {Proceedings of the National Academy of Sciences of the United States of America},
  year = {2010},
  volume = {107},
  number = {21},
  pages = {9572 - 7},
  doi = {https://doi.org/10.1073/pnas.1000931107}
}
</pre></td>
</tr>
<tr id="Roston2013" class="entry">
	<td>Roston, D. and Kohen, A.</td>
	<td>A critical test of the "tunneling and coupled motion" concept in enzymatic alcohol oxidation. <p class="infolinks">[<a href="javascript:toggleInfo('Roston2013','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Roston2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Journal of the American Chemical Society<br/>Vol. 135(37), pp. 13624 - 7&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ja405917m">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Roston2013" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The physical mechanism of C-H bond activation by enzymes is the subject of intense study, and we have tested the predictions of two competing models for C-H activation in the context of alcohol dehydrogenase. The kinetic isotope effects (KIEs) in this enzyme have previously suggested a model of quantum mechanical tunneling and coupled motion of primary (1textdegree) and secondary (2textdegree) hydrogens. Here we measure the 2textdegree H/T KIEs with both H and D at the 1textdegree position and find that the 2textdegree KIE is significantly deflated with D-transfer, consistent with the predictions of recent Marcus-like models of H-transfer. The results suggest that the fast dynamics of H-tunneling result in a 1textdegree isotope effect on the structure of the tunneling ready state: the trajectory of D-transfer goes through a shorter donor-acceptor distance than that of H-transfer.</td>
</tr>
<tr id="bib_Roston2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Roston2013,
  author = {Roston, Daniel and Kohen, Amnon},
  title = {A critical test of the "tunneling and coupled motion" concept in enzymatic alcohol oxidation.},
  journal = {Journal of the American Chemical Society},
  year = {2013},
  volume = {135},
  number = {37},
  pages = {13624 - 7},
  doi = {https://doi.org/10.1021/ja405917m}
}
</pre></td>
</tr>
<tr id="Roux1995" class="entry">
	<td>Roux, B. and Karplus, M.</td>
	<td>Potential Energy Function for Cation-Peptide Interactions: An Ab Initio Study <p class="infolinks">[<a href="javascript:toggleInfo('Roux1995','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Roux1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>Journal of Computational Chemistry<br/>Vol. 16(6), pp. 690 - 704&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/jcc.540160605">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Roux1995" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A potential energy function is developed to represent the interaction of small monovalent cations, Li', Na', and K' , with the backbone of polypeptides. The results are based on ab initio calculations up to the 6-31G* level of the interactions of the ions with acetamide and N-methylacetamide. Basis set superposition errors are corrected with the counterpoise method. A systematic overestimate of the bond polarities is taken into account by an empirical scaling procedure that uses the ratio of the experimental to ab initio dipole moment. The calculated binding energies obtained with this procedure show consistent convergence with different basis sets and are in good agreement with experimental data on cation-water and cation-dimethylformamide systems. Investigations of the calculated ab initio potential energy surface indicate that the cation-peptide interaction is dominated by electrostatics and includes a nonnegligible contribution from polarization of the peptide group by the ion. The induced polarization results in a steeper-than-Coulombic interaction and cannot be described by fixed ion-peptide partial charges electrostatics. Atomic polarizabilities located on the atoms of the ligand molecule are introduced to account for the induced polarization in the empirical energy function. A -l/r4 attractive interaction appears in the potential function. The resulting radial and angular dependence of the potential energy surface is well reproduced.</td>
</tr>
<tr id="bib_Roux1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Roux1995,
  author = {Roux, Beno\textasciitilde\t and Karplus, Martin},
  title = {Potential Energy Function for Cation-Peptide Interactions: An Ab Initio Study},
  journal = {Journal of Computational Chemistry},
  year = {1995},
  volume = {16},
  number = {6},
  pages = {690 - 704},
  doi = {https://doi.org/10.1002/jcc.540160605}
}
</pre></td>
</tr>
<tr id="Roxrud2008" class="entry">
	<td>Roxrud, I., Raiborg, C., Pedersen, N., Stang, E. and Stenmark, H.</td>
	<td>An endosomally localized isoform of Eps15 interacts with Hrs to mediate degradation of epidermal growth factor receptor <p class="infolinks">[<a href="javascript:toggleInfo('Roxrud2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Roxrud2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Journal of Cell Biology<br/>Vol. 180(6), pp. 1205 - 1218&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1083/jcb.200708115">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Roxrud2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Down-regulation of activated and ubiquitinated growth factor (GF) receptors by endocytosis and subsequent lysosomal degradation ensures attenuation of GF signaling. The ubiquitin-binding adaptor protein Eps15 (epidermal growth factor receptor [EGFR] pathway substrate 15) functions in endocytosis of such receptors. Here, we identify an Eps15 isoform, Eps15b, and demonstrate its expression in human cells and conservation across vertebrate species. Although both Eps15 and Eps15b interact with the endosomal sorting protein Hrs (hepatocyte growth factor-regulated tyrosine kinase substrate) in vitro, we find that Hrs specifically binds Eps15b in vivo (whereas adaptor protein 2 preferentially interacts with Eps15). Although Eps15 mainly localizes to clathrin-coated pits at the plasma membrane, Eps15b localizes to Hrs-positive microdomains on endosomes. Eps15b overexpression, similarly to Hrs overexpression, inhibits ligand-mediated degradation of EGFR, whereas Eps15 is without effect. Similarly, depletion of Eps15b but not Eps15 delays degradation and promotes recycling of EGFR. These results indicate that Eps15b is an endosomally localized isoform of Eps15 that is present in the Hrs complex via direct Hrs interaction and important for the sorting function of this complex.</td>
</tr>
<tr id="bib_Roxrud2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Roxrud2008,
  author = {Roxrud, Ingrid and Raiborg, Camilla and Pedersen, Nina and Stang, Espen and Stenmark, Harald},
  title = {An endosomally localized isoform of Eps15 interacts with Hrs to mediate degradation of epidermal growth factor receptor},
  journal = {Journal of Cell Biology},
  year = {2008},
  volume = {180},
  number = {6},
  pages = {1205 - 1218},
  note = {Hep-2 cells. interaction of EPS15 and alpha-adaptin. doesn't say alpha-a or alpha-c.},
  doi = {https://doi.org/10.1083/jcb.200708115}
}
</pre></td>
</tr>
<tr id="Royle2006" class="entry">
	<td>Royle, S.J.</td>
	<td>The cellular functions of clathrin. <p class="infolinks">[<a href="javascript:toggleInfo('Royle2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Royle2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Cellular and molecular life sciences : CMLS<br/>Vol. 63, pp. 1823-1832&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1007/s00018-005-5587-0">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Royle2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Membranes and proteins are moved around the cell in small vesicles. A protein coat aids the budding of such vesicles from donor membranes. The major type of coat used by the cell is composed of clathrin, a three-legged protein that can form lattice-like coats on membranes destined for trafficking. In this review, I outline what we know about clathrin and discuss some recent advances in understanding the basic biology of this fascinating molecule, which include building a molecular model of a clathrin lattice and discovery of a new function for clathrin that occurs during mitosis.</td>
</tr>
<tr id="bib_Royle2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Royle2006,
  author = {Royle, S J},
  title = {The cellular functions of clathrin.},
  journal = {Cellular and molecular life sciences : CMLS},
  year = {2006},
  volume = {63},
  pages = {1823--1832},
  note = {dissociation constant citations.},
  doi = {https://doi.org/10.1007/s00018-005-5587-0}
}
</pre></td>
</tr>
<tr id="Rozanov1982" class="entry">
	<td>Rozanov, Y.A.</td>
	<td>Markov Random Fields <p class="infolinks">[<a href="javascript:toggleInfo('Rozanov1982','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Rozanov1982','bibtex')">BibTeX</a>]</p></td>
	<td>1982</td>
	<td>, pp. 201&nbsp;</td>
	<td>book</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Rozanov1982" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In this book we study Markov random functions of several variables. What is traditionally meant by the Markov property for a random process (a random function of one time variable) is connected to the concept of the phase state of the process and refers to the independence of the behavior of the process in the future from its behavior in the past, given knowledge of its state at the present moment. Extension to a generalized random process immediately raises nontrivial questions about the definition of a suitable" phase state," so that given the state, future behavior does not depend on past behavior. Attempts to translate the Markov property to random functions of multi-dimensional "time," where the role of "past" and "future" are taken by arbitrary complementary regions in an appro priate multi-dimensional time domain have, until comparatively recently, been carried out only in the framework of isolated examples. How the Markov property should be formulated for generalized random functions of several variables is the principal question in this book. We think that it has been substantially answered by recent results establishing the Markov property for a whole collection of different classes of random functions. These results are interesting for their applications as well as for the theory. In establishing them, we found it useful to introduce a general probability model which we have called a random field. In this book we investigate random fields on continuous time domains. Contents CHAPTER 1 General Facts About Probability Distributions sect\1. 1 General Facts About Probability Distributions -- ʹ1. Probability Spaces -- ʹ2. Conditional Distributions -- ʹ3. Zero-One Laws. Regularity -- ʹ4. Consistent Conditional Distributions -- ʹ5. Gaussian Probability Distributions -- 2 Markov Random Fields -- ʹ1. Basic Definitions and Useful Propositions -- ʹ2. Stopping?-algebras. Random Sets and the Strong Markov Property -- ʹ3. Gaussian Fields. Markov Behavior in the Wide Sense -- 3 The Markov Property for Generalized Random Functions -- ʹ1. Biorthogonal Generalized Functions and the Duality Property -- ʹ2. Stationary Generalized Functions -- ʹ3. Biorthogonal Generalized Functions Given by a Differential Form -- ʹ4. Markov Random Functions Generated by Elliptic Differential Forms -- ʹ5. Stochastic Differential Equations -- 4 Vector-Valued Stationary Functions -- ʹ1. Conditions for Existence of the Dual Field -- ʹ2. The Markov Property for Stationary Functions -- ʹ3. Markov Extensions of Random Processes -- Notes.</td>
</tr>
<tr id="bib_Rozanov1982" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{Rozanov1982,
  author = {Rozanov, Yu. A},
  title = {Markov Random Fields},
  publisher = {Springer New York},
  year = {1982},
  pages = {201}
}
</pre></td>
</tr>
<tr id="Rual2005" class="entry">
	<td>Rual, J.F..</td>
	<td>Towards a proteome-scale map of the human protein--protein interaction network <p class="infolinks">[<a href="javascript:toggleInfo('Rual2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Rual2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Nature<br/>Vol. 437(October), pp. 1173 - 1178&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nature04209">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Rual2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Systematic mapping of protein--protein interactions, or `interactome' mapping, was initiated in model organisms, starting with defined biological processes1, 2 and then expanding to the scale of the proteome3, 4, 5, 6, 7. Although far from complete, such maps have revealed global topological and dynamic features of interactome networks that relate to known biological properties8, 9, suggesting that a human interactome map will provide insight into development and disease mechanisms at a systems level. Here we describe an initial version of a proteome-scale map of human binary protein--protein interactions. Using a stringent, high-throughput yeast two-hybrid system, we tested pairwise interactions among the products of \textasciitilde\8,100 currently available Gateway-cloned open reading frames and detected \textasciitilde\2,800 interactions. This data set, called CCSB-HI1, has a verification rate of \textasciitilde\78&#37; as revealed by an independent co-affinity purification assay, and correlates significantly with other biological attributes. The CCSB-HI1 data set increases by \textasciitilde\70&#37; the set of available binary interactions within the tested space and reveals more than 300 new connections to over 100 disease-associated proteins. This work represents an important step towards a systematic and comprehensive human interactome project.</td>
</tr>
<tr id="bib_Rual2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Rual2005,
  author = {Rual, J.F.},
  title = {Towards a proteome-scale map of the human protein--protein interaction network},
  journal = {Nature},
  year = {2005},
  volume = {437},
  number = {October},
  pages = {1173 - 1178},
  note = {literature curated + Y2H study. only Y2H result = AP2B1 - AP2M1},
  doi = {https://doi.org/10.1038/nature04209}
}
</pre></td>
</tr>
<tr id="Rubinstein2010" class="entry">
	<td>Rubinstein, A. and Major, D.</td>
	<td>Understanding Catalytic Specificity in Alanine Racemase from Quantum Mechanical and Molecular Mechanical Simulations of the Arginine 219 Mutant <p class="infolinks">[<a href="javascript:toggleInfo('Rubinstein2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Biochemistry<br/>Vol. 49, pp. 3957 - 3964&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Rubinstein2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Rubinstein2010,
  author = {Rubinstein, Amir and Major, Dan},
  title = {Understanding Catalytic Specificity in Alanine Racemase from Quantum Mechanical and Molecular Mechanical Simulations of the Arginine 219 Mutant},
  journal = {Biochemistry},
  year = {2010},
  volume = {49},
  pages = {3957 - 3964}
}
</pre></td>
</tr>
<tr id="Rucker1999" class="entry">
	<td>Rucker, J. and Klinman, J.P.</td>
	<td>Computational study of tunneling and coupled motion in alcohol dehydrogenase-catalyzed reactions: Implication for measured hydrogen and carbon isotope effects <p class="infolinks">[<a href="javascript:toggleInfo('Rucker1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Rucker1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Journal of the American Chemical Society<br/>Vol. 121(10), pp. 1997 - 2006&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ja9824425">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Rucker1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The relationship between the two hydrogen isotope effects, k(H)/k(T) and k(D)/k(T), can provide a probe for the role of tunneling and coupled motion in enzyme-catalyzed reactions. Using vibrational analysis and the Bigeleisen-Mayer equation, we have developed a simple computational model to explain the unusual exponential relationships that have been experimentally observed in the yeast alcohol dehydrogenase (YADH)-catalyzed oxidation of benzyl alcohol. The experimental results are fitted by a model that has both substantial hydrogen tunneling and coupling between the reaction coordinate and a large number of vibrational modes. We show that the secondary k(D)/k(T) isotope effect is expected to be the most sensitive parameter to changes in reaction coordinate properties. A high degree of coupled motion leads to an unexpected suppression of the semiclassical secondary isotope effects, resulting in secondary isotope effects which are primarily manifestations of tunneling. This has implications for the use of secondary hydrogen isotope effects as probes of transition state position. During the course of these computational studies, primary carbon isotope effects were shown to undergo a consistent increase in magnitude upon substitution of the transferring protium with deuterium. We suggest that this effect can be explained using simple semiclassical principles and will apply to a broad range of reactions. Inference of reaction mechanism from the observation of a change in a heavy atom isotope effects upon substrate deuteration should be interpreted with caution when the position of heavy atom and hydrogen isotope substitution are the same.</td>
</tr>
<tr id="bib_Rucker1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Rucker1999,
  author = {Rucker, Joseph and Klinman, Judith P},
  title = {Computational study of tunneling and coupled motion in alcohol dehydrogenase-catalyzed reactions: Implication for measured hydrogen and carbon isotope effects},
  journal = {Journal of the American Chemical Society},
  year = {1999},
  volume = {121},
  number = {10},
  pages = {1997 - 2006},
  doi = {https://doi.org/10.1021/ja9824425}
}
</pre></td>
</tr>
<tr id="Rumpf2008" class="entry">
	<td>Rumpf, J., Simon, B., Jung, N., Maritzen, T., Haucke, V., Sattler, M. and Groemping, Y.</td>
	<td>Structure of the Eps15-stonin2 complex provides a molecular explanation for EH-domain ligand specificity. <p class="infolinks">[<a href="javascript:toggleInfo('Rumpf2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Rumpf2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>The EMBO journal<br/>Vol. 27, pp. 558-569&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/sj.emboj.7601980">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Rumpf2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Eps15 homology (EH) domain-containing proteins play a key regulatory role in intracellular membrane trafficking and cell signalling. EH domains serve as interaction platforms for short peptide motifs comprising the residues NPF within natively unstructured regions of accessory proteins. The EH-NPF interactions described thus far are of very low affinity and specificity. Here, we identify the presynaptic endocytic sorting adaptor stonin2 as a high-affinity ligand for the second EH domain (EH2) of the clathrin accessory protein Eps15. Calorimetric data indicate that both NPF motifs within stonin2 interact with EH2 simultaneously and with sub-micromolar affinity. The solution structure of this complex reveals that the first NPF motif binds to the conserved site on the EH domain, whereas the second motif inserts into a novel hydrophobic pocket. Our data show how combination of two EH-attachment sites provides a means for modulating specificity and allows discrimination from a large pool of potential binding partners containing NPF motifs.</td>
</tr>
<tr id="bib_Rumpf2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Rumpf2008,
  author = {Rumpf, Julia and Simon, Bernd and Jung, Nadja and Maritzen, Tanja and Haucke, Volker and Sattler, Michael and Groemping, Yvonne},
  title = {Structure of the Eps15-stonin2 complex provides a molecular explanation for EH-domain ligand specificity.},
  journal = {The EMBO journal},
  year = {2008},
  volume = {27},
  pages = {558--569},
  doi = {https://doi.org/10.1038/sj.emboj.7601980}
}
</pre></td>
</tr>
<tr id="Ryckaert1977" class="entry">
	<td>Ryckaert, Jean-Paul., Ciccotti, G. and Berendsen, H.</td>
	<td>Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes <p class="infolinks">[<a href="javascript:toggleInfo('Ryckaert1977','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ryckaert1977','bibtex')">BibTeX</a>]</p></td>
	<td>1977</td>
	<td>Journal of Computational Physics<br/>Vol. 23(3), pp. 327 - 341&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/0021-9991(77)90098-5">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ryckaert1977" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A numerical algorithm integrating the 3N Cartesian equations of motion of a system of N points subject to holonomic constraints is formulated. The relations of constraint remain perfectly fulfilled at each step of the trajectory despite the approximate character of numerical integration. The method is applied to a molecular dynamics simulation of a liquid of 64 n-butane molecules and compared to a simulation using generalized coordinates. The method should be useful for molecular dynamics calculations on large molecules with internal degrees of freedom.</td>
</tr>
<tr id="bib_Ryckaert1977" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ryckaert1977,
  author = {Ryckaert, Jean-Paul and Ciccotti, Giovanni and Berendsen, Herman},
  title = {Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes},
  journal = {Journal of Computational Physics},
  year = {1977},
  volume = {23},
  number = {3},
  pages = {327 - 341},
  note = {Original SHAKE paper},
  doi = {https://doi.org/10.1016/0021-9991(77)90098-5}
}
</pre></td>
</tr>
<tr id="Ryter1987" class="entry">
	<td>Ryter, D.</td>
	<td>Noise-induced transitions in a double-well potential at low friction <p class="infolinks">[<a href="javascript:toggleInfo('Ryter1987','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ryter1987','bibtex')">BibTeX</a>]</p></td>
	<td>1987</td>
	<td>Journal of Statistical Physics<br/>Vol. 49(3-4), pp. 751 - 765&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1007/BF01009355">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ryter1987" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A new integral representation of the transition rate holds for any friction and is shown to allow a feasible evaluation in a wide friction range. Analytic approximations include the (high-friction) Kramers result with the leading correction, as well as a low-friction case. The method is complementary to a recent one of Melnikov and Meshkov.</td>
</tr>
<tr id="bib_Ryter1987" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ryter1987,
  author = {Ryter, D},
  title = {Noise-induced transitions in a double-well potential at low friction},
  journal = {Journal of Statistical Physics},
  year = {1987},
  volume = {49},
  number = {3-4},
  pages = {751 - 765},
  doi = {https://doi.org/10.1007/BF01009355}
}
</pre></td>
</tr>
<tr id="Ryter1987a" class="entry">
	<td>Ryter, D.</td>
	<td>On the eigenfunctions of the Fokker-Planck operator and of its adjoint <p class="infolinks">[<a href="javascript:toggleInfo('Ryter1987a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ryter1987a','bibtex')">BibTeX</a>]</p></td>
	<td>1987</td>
	<td>Physica A: Statistical Mechanics and its Applications<br/>Vol. 142(1-3), pp. 103 - 121&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/0378-4371(87)90019-7">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ryter1987a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The eigenfunctions of the forward and backward operator are linked by means of an "associate s system", for which the stationary distribution and the eigenvalues are the same. For systems with M (???2) stable states a Feynman-Kac result provides an efficient approximation of the first M backward eigenfunctions when the noise is "moderate"; the corresponding form of the first M forward eigenfunctions follows by the above relation. At weak noise the associate system becomes more explicit; moreover, it leads to a new understanding and to a generalization of the Kramers method. ?? 1987.</td>
</tr>
<tr id="bib_Ryter1987a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ryter1987a,
  author = {Ryter, D},
  title = {On the eigenfunctions of the Fokker-Planck operator and of its adjoint},
  journal = {Physica A: Statistical Mechanics and its Applications},
  year = {1987},
  volume = {142},
  number = {1-3},
  pages = {103 - 121},
  doi = {https://doi.org/10.1016/0378-4371(87)90019-7}
}
</pre></td>
</tr>
<tr id="Saen-Oon2008" class="entry">
	<td>Saen-Oon, S., Schramm, V.L. and Schwartz, S.D.</td>
	<td>Transition Path Sampling Study of the Reaction Catalyzed by Purine Nucleoside Phosphorylase. <p class="infolinks">[<a href="javascript:toggleInfo('Saen-Oon2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Saen-Oon2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Zeitschrift fur physikalische Chemie<br/>Vol. 222(8-9), pp. 1359 - 1374&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1524/zpch.2008.5395">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Saen-Oon2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The Transition Path Sampling (TPS) method is a powerful technique for studying rare events in complex systems, that allows description of reactive events in atomic detail without prior knowledge of reaction coordinates and transition states. We have applied TPS in combination with a hybrid Quantum Mechanical/Molecular Mechanical (QM/MM) method to study the enzyme human purine nucleoside phosphorylase (hPNP). This enzyme catalyzes the reversible phosphorolysis of 6-oxypurine (deoxy)nucleosides to generate the corresponding purine base and (deoxy)ribose 1-phosphate. Hundreds of reactive trajectories were generated. Analysis of this transition path ensembles provides insight into the detailed mechanistic dynamics of reaction in the enzyme. Our studies have indicated a reaction mechanism involving the cleavage of the N-ribosidic bond to form transition states with substantial ribooxacarbenium ion character, that is then followed by conformational changes in the enzyme and the ribosyl group leading to migration of the anomeric carbon of the ribosyl group toward phosphate to form the product ribose 1-phosphate. This latter process is crucial in PNP, because several strong H-bonds form between active site residues in order to capture and align the phosphate nucleophile. Calculations of the commitment probability along reactive paths demonstrated the presence of a broad energy barrier at the transition state. Analysis of these transition state structures showed that bond-breaking and bond-forming distances are not a good choice for the reaction coordinate, but that the pseudorotational phase of the ribose ring is also a significant variable.</td>
</tr>
<tr id="bib_Saen-Oon2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Saen-Oon2008,
  author = {Saen-Oon, Suwipa and Schramm, Vern L and Schwartz, Steven D},
  title = {Transition Path Sampling Study of the Reaction Catalyzed by Purine Nucleoside Phosphorylase.},
  journal = {Zeitschrift fur physikalische Chemie},
  year = {2008},
  volume = {222},
  number = {8-9},
  pages = {1359 - 1374},
  doi = {https://doi.org/10.1524/zpch.2008.5395}
}
</pre></td>
</tr>
<tr id="Sankar2018" class="entry">
	<td>Sankar, K., Mishra, S.K. and Jernigan, R.L.</td>
	<td>Comparisons of Protein Dynamics from Experimental Structure Ensembles, Molecular Dynamics Ensembles, and Coarse-Grained Elastic Network Models <p class="infolinks">[<a href="javascript:toggleInfo('Sankar2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>The Journal of Physical Chemistry B&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Sankar2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Sankar2018,
  author = {Sankar, Kannan and Mishra, Sambit K and Jernigan, Robert L},
  title = {Comparisons of Protein Dynamics from Experimental Structure Ensembles, Molecular Dynamics Ensembles, and Coarse-Grained Elastic Network Models},
  journal = {The Journal of Physical Chemistry B},
  publisher = {ACS Publications},
  year = {2018}
}
</pre></td>
</tr>
<tr id="Sarvenaz2017" class="entry">
	<td>Sarabipour, S. and Mac Gabhann, F.</td>
	<td>Computational Systems Biochemistry: Beyond the Static Interactome. <p class="infolinks">[<a href="javascript:toggleInfo('Sarvenaz2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Biochemistry&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Sarvenaz2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Sarvenaz2017,
  author = {Sarabipour, Sarvenaz and Mac Gabhann, Feilim},
  title = {Computational Systems Biochemistry: Beyond the Static Interactome.},
  journal = {Biochemistry},
  year = {2017}
}
</pre></td>
</tr>
<tr id="Saunders1985" class="entry">
	<td>Saunders, W.H.</td>
	<td>Calculations of Isotope Effects in Elimination Reactions. New Experimental Criteria for Tunneling in Slow Proton Transfers. <p class="infolinks">[<a href="javascript:toggleInfo('Saunders1985','bibtex')">BibTeX</a>]</p></td>
	<td>1985</td>
	<td>J. Am. Chem. Soc.<br/>Vol. 107(8), pp. 164 - 164&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ja00287a029">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Saunders1985" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Saunders1985,
  author = {Saunders, William H},
  title = {Calculations of Isotope Effects in Elimination Reactions. New Experimental Criteria for Tunneling in Slow Proton Transfers.},
  journal = {J. Am. Chem. Soc.},
  year = {1985},
  volume = {107},
  number = {8},
  pages = {164 - 164},
  doi = {https://doi.org/10.1021/ja00287a029}
}
</pre></td>
</tr>
<tr id="Sawada1994" class="entry">
	<td>Sawada, S., Tanaka, Y., Hayashi, S., Ryu, M., Hasegawa, T., Yamamoto, Y., Esaki, N., Soda, K. and Takahashi, S.</td>
	<td>Kinetics of Thermostable Alanine Racemase of Bacillus stearothermophilus <p class="infolinks">[<a href="javascript:toggleInfo('Sawada1994','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Sawada1994','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td>Bioscience, Biotechnology and Biochemistry<br/>Vol. 58(5), pp. 807 - 811&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1271/bbb.58.807">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Sawada1994" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Unlabeled D- and L-alanine were racemized in deuterium oxide with an alanine racemase of Bacillus stearothermophilus at saturated concentration of substrate, and various p2H and temperature. Samples of the solution were taken at intervals, and all alanine isomers in the samples were transformed into a mixture of diastereomeric derivatives of methyl N-(--)-camphanylalaninate. Their ratio was measured on a GC-Mass, and the relative rate was calculated at the initial stage of the reaction. There was little difference in the decrease rate of the optical rotation between the enantiomers. Internal proton-transfer to the antipode was almost zero for either substrate. The α-hydrogen was abstracted 1.2--2.3 times faster from D-alanine than from L-alanine. D-Alanine gave an almost even mixture of deuterium labeled D- and L-alanine, while L-alanine gave a mixture of labeled D- and L-alanine at a ratio of 3:1. These results suggest the racemase builds two different bases in the active site. The base for D-alanine may be closer to the enzyme surface, and that for L-alanine inside.</td>
</tr>
<tr id="bib_Sawada1994" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Sawada1994,
  author = {Sawada, Seiji and Tanaka, Yoshinari and Hayashi, Sayoko and Ryu, Manami and Hasegawa, Takeshi and Yamamoto, Yoshiro and Esaki, Nobuyoshi and Soda, Kenji and Takahashi, Sho},
  title = {Kinetics of Thermostable Alanine Racemase of Bacillus stearothermophilus},
  journal = {Bioscience, Biotechnology and Biochemistry},
  year = {1994},
  volume = {58},
  number = {5},
  pages = {807 - 811},
  note = {Proton abstraction is rate limiting step},
  doi = {https://doi.org/10.1271/bbb.58.807}
}
</pre></td>
</tr>
<tr id="Scaletti2012" class="entry">
	<td>Scaletti, E.R., Luckner, S.R. and Krause, K.L.</td>
	<td>Structural features and kinetic characterization of alanine racemase from Staphylococcus aureus (Mu50). <p class="infolinks">[<a href="javascript:toggleInfo('Scaletti2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Scaletti2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Acta crystallographica. Section D, Biological crystallography<br/>Vol. 68(Pt 1), pp. 82 - 92&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1107/S0907444911050682">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Scaletti2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Staphylococcus aureus is an opportunistic Gram-positive bacterium which causes a wide variety of diseases ranging from minor skin infections to potentially fatal conditions such as pneumonia, meningitis and septicaemia. The pathogen is a leading cause of nosocomial acquired infections, a problem that is exacerbated by the existence of methicillin- and glycopeptide antibiotic-resistant strains which can be challenging to treat. Alanine racemase (Alr) is a pyridoxal-5'-phosphate-dependent enzyme which catalyzes reversible racemization between enantiomers of alanine. As D-alanine is an essential component of the bacterial cell-wall peptidoglycan, inhibition of Alr is lethal to prokaryotes. Additionally, while ubiquitous amongst bacteria, this enzyme is absent in humans and most eukaryotes, making it an excellent antibiotic drug target. The crystal structure of S. aureus alanine racemase (Alr(Sas)), the sequence of which corresponds to that from the highly antibiotic-resistant Mu50 strain, has been solved to 2.15 &Aring; resolution. Comparison of the Alr(Sas) structure with those of various alanine racemases demonstrates a conserved overall fold, with the enzyme sharing most similarity to those from other Gram-positive bacteria. Structural examination indicates that the active-site binding pocket, dimer interface and active-site entryway of the enzyme are potential targets for structure-aided inhibitor design. Kinetic constants were calculated in this study and are reported here. The potential for a disulfide bond in this structure is noted. This structural and biochemical information provides a template for future structure-based drug-development efforts targeting Alr(Sas).</td>
</tr>
<tr id="bib_Scaletti2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Scaletti2012,
  author = {Scaletti, Emma R and Luckner, Sylvia R and Krause, Kurt L},
  title = {Structural features and kinetic characterization of alanine racemase from Staphylococcus aureus (Mu50).},
  journal = {Acta crystallographica. Section D, Biological crystallography},
  year = {2012},
  volume = {68},
  number = {Pt 1},
  pages = {82 - 92},
  doi = {https://doi.org/10.1107/S0907444911050682}
}
</pre></td>
</tr>
<tr id="Schink2016" class="entry">
	<td>Schink, K.O., Tan, K.-W. and Stenmark, H.</td>
	<td>Phosphoinositides in Control of Membrane Dynamics. <p class="infolinks">[<a href="javascript:toggleInfo('Schink2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schink2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Annual review of cell and developmental biology<br/>Vol. 32, pp. 143-171&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1146/annurev-cellbio-111315-125349">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Schink2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Most functions of eukaryotic cells are controlled by cellular membranes, which are not static entities but undergo frequent budding, fission, fusion, and sculpting reactions collectively referred to as membrane dynamics. Consequently, regulation of membrane dynamics is crucial for cellular functions. A key mechanism in such regulation is the reversible recruitment of cytosolic proteins or protein complexes to specific membranes at specific time points. To a large extent this recruitment is orchestrated by phosphorylated derivatives of the membrane lipid phosphatidylinositol, known as phosphoinositides. The seven phosphoinositides found in nature localize to distinct membrane domains and recruit distinct effectors, thereby contributing strongly to the maintenance of membrane identity. Many of the phosphoinositide effectors are proteins that control membrane dynamics, and in this review we discuss the functions of phosphoinositides in membrane dynamics during exocytosis, endocytosis, autophagy, cell division, cell migration, and epithelial cell polarity, with emphasis on protein effectors that are recruited by specific phosphoinositides during these processes.</td>
</tr>
<tr id="bib_Schink2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Schink2016,
  author = {Schink, Kay O and Tan, Kia-Wee and Stenmark, Harald},
  title = {Phosphoinositides in Control of Membrane Dynamics.},
  journal = {Annual review of cell and developmental biology},
  year = {2016},
  volume = {32},
  pages = {143--171},
  doi = {https://doi.org/10.1146/annurev-cellbio-111315-125349}
}
</pre></td>
</tr>
<tr id="Schmid2006" class="entry">
	<td>Schmid, E.M., Ford, M.GJ., Burtey, A., Praefcke, G.JK., Sew, Peak-Chew,., Mills, I.G., Benmerah, A. and McMahon, Harvey, T</td>
	<td>Role of the AP2 beta-appendage hub in recruiting partners for clathrin-coated vesicle assembly <p class="infolinks">[<a href="javascript:toggleInfo('Schmid2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schmid2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>PLoS Biology<br/>Vol. 4(9), pp. 1532 - 1548&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pbio.0040262">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Schmid2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Adaptor protein complex 2 alpha and beta-appendage domains act as hubs for the assembly of accessory protein networks involved in clathrin-coated vesicle formation. We identify a large repertoire of beta-appendage interactors by mass spectrometry. These interact with two distinct ligand interaction sites on the beta-appendage (the "top" and "side" sites) that bind motifs distinct from those previously identified on the alpha-appendage. We solved the structure of the beta-appendage with a peptide from the accessory protein Eps15 bound to the side site and with a peptide from the accessory cargo adaptor beta-arrestin bound to the top site. We show that accessory proteins can bind simultaneously to multiple appendages, allowing these to cooperate in enhancing ligand avidities that appear to be irreversible in vitro. We now propose that clathrin, which interacts with the beta-appendage, achieves ligand displacement in vivo by self-polymerisation as the coated pit matures. This changes the interaction environment from liquid-phase, affinity-driven interactions, to interactions driven by solid-phase stability ("matricity"). Accessory proteins that interact solely with the appendages are thereby displaced to areas of the coated pit where clathrin has not yet polymerised. However, proteins such as beta-arrestin (non-visual arrestin) and autosomal recessive hypercholesterolemia protein, which have direct clathrin interactions, will remain in the coated pits with their interacting receptors.</td>
</tr>
<tr id="bib_Schmid2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Schmid2006,
  author = {Schmid, Eva M and Ford, Marijn GJ and Burtey, Anne and Praefcke, Gerrit JK and Sew, Peak-Chew, and Mills, Ian G and Benmerah, Alexandre and T, McMahon, Harvey},
  title = {Role of the AP2 beta-appendage hub in recruiting partners for clathrin-coated vesicle assembly},
  journal = {PLoS Biology},
  year = {2006},
  volume = {4},
  number = {9},
  pages = {1532 - 1548},
  note = {beta2 binding a lot of things. AMPH only found in brain. some domain and dissociation constant information.},
  doi = {https://doi.org/10.1371/journal.pbio.0040262}
}
</pre></td>
</tr>
<tr id="Schmid2007" class="entry">
	<td>Schmid, E.M. and McMahon, Harvey, T</td>
	<td>Integrating molecular and network biology to decode endocytosis <p class="infolinks">[<a href="javascript:toggleInfo('Schmid2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schmid2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Nature<br/>Vol. 448(7156), pp. 883 - 888&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nature06031">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Schmid2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The strength of network biology lies in its ability to derive cell biological information without a priori mechanistic or molecular knowledge. It is shown here how a careful understanding of a given biological pathway can refine an interactome approach. This permits the elucidation of additional design principles and of spatio-temporal dynamics behind pathways, and aids in experimental design and interpretation.</td>
</tr>
<tr id="bib_Schmid2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Schmid2007,
  author = {Schmid, Eva M and T, McMahon, Harvey},
  title = {Integrating molecular and network biology to decode endocytosis},
  journal = {Nature},
  year = {2007},
  volume = {448},
  number = {7156},
  pages = {883 - 888},
  note = {EXCELLENT SI},
  doi = {https://doi.org/10.1038/nature06031}
}
</pre></td>
</tr>
<tr id="Schmid1998" class="entry">
	<td>Schmid, S.L., McNiven, M.A. and De Camilli, P.</td>
	<td>Dynamin and its partners: a progress report <p class="infolinks">[<a href="javascript:toggleInfo('Schmid1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Current opinion in cell biology<br/>Vol. 10(4), pp. 504-512&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Schmid1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Schmid1998,
  author = {Schmid, Sandra L and McNiven, Mark A and De Camilli, Pietro},
  title = {Dynamin and its partners: a progress report},
  journal = {Current opinion in cell biology},
  publisher = {Elsevier},
  year = {1998},
  volume = {10},
  number = {4},
  pages = {504--512}
}
</pre></td>
</tr>
<tr id="Schmidt1999" class="entry">
	<td>Schmidt, R.K. and Gready, J.E.</td>
	<td>Molecular Dynamics Simulations of L-Lactate Dehydrogenase: Conformation of a Mobile Loop and Influence of the Tetrameric Protein Environment <p class="infolinks">[<a href="javascript:toggleInfo('Schmidt1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schmidt1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Journal of Molecular Modeling<br/>Vol. 5(9), pp. 153 - 168&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1007/s008940050115">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Schmidt1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: L-lactate dehydrogenase (LDH) catalyses the interconversion of pyruvate and L-lactate in the presence of the coenzyme NADH. Molecular dynamics (MD) simulations have been performed for LDH complexed with NADH and the pyruvate-analogue inhibitor oxamate with the aim of characterising important influences on maintaining the geometry and hydrogen bond network of the active site. Two features in particular were found to dominate. First, the tetrameric protein environment is found to play a significant role in maintaining the active-site geometry. Simulations of the monomer alone reproduce the crystallographic structure poorly, and at least part of the neighboring subunit is necessary to prevent water penetration into the active site and to provide rigidity to the α1G-α2G helix immediately adjacent to the active site. These results offer one explanation for the observation that the monomer is not biologically active. Second, the conformation of Arg109 (part of the mobile loop which closes over the active site) is shown to play a key role in maintaining the active-site geometry. In some simulations, a torsional rotation in the side chain of Arg109 results in the breaking of crystallographic hydrogen bonds which are important for polarising the carbonyl bond of the substrate. This conformational change appears to be a trigger for the opening of the mobile loop. Long-range nonbonded interactions are found to be influential in maintaining the proper crystallographic conformation of Arg109. Thus, we conclude that to adequately model LDH, at least part of the neighboring subunit must be included in the MD simulations and nonbonded interactions must be properly represented to ensure that Arg109 remains in the crystallographic conformation. Out of a set of simulation protocols tested here, one meets both these criteria and will be used for the generation of starting structures for future reaction-mechanism calculations.</td>
</tr>
<tr id="bib_Schmidt1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Schmidt1999,
  author = {Schmidt, Rebecca K and Gready, Jill E},
  title = {Molecular Dynamics Simulations of L-Lactate Dehydrogenase: Conformation of a Mobile Loop and Influence of the Tetrameric Protein Environment},
  journal = {Journal of Molecular Modeling},
  year = {1999},
  volume = {5},
  number = {9},
  pages = {153 - 168},
  doi = {https://doi.org/10.1007/s008940050115}
}
</pre></td>
</tr>
<tr id="Scholfield2017" class="entry">
	<td>Scholfield, M.R., Ford, M., Carlsson, Anna-Carin., Butta, H., Mehl, R.A. and Ho, P.</td>
	<td>Structure-Energy Relationships of Halogen Bonds in Proteins <p class="infolinks">[<a href="javascript:toggleInfo('Scholfield2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Scholfield2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Biochemistry, pp. acs.biochem.7b00022 - acs.biochem.7b00022&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.biochem.7b00022">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Scholfield2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The structures and stabilities of proteins are defined by a series of weak noncovalent electrostatic, van der Waals, and hydrogen bond (HB) interactions. In this study, we have designed and engineered halogen bonds (XBs) site-specifically to study their structure--energy relationship in a model protein, T4 lysozyme. The evidence for XBs is the displacement of the aromatic side chain toward an oxygen acceptor, at distances that are equal to or less than the sums of their respective van der Waals radii, when the hydroxyl substituent of the wild-type tyrosine is replaced by a halogen. In addition, thermal melting studies show that the iodine XB rescues the stabilization energy from an otherwise destabilizing substitution (at an equivalent noninteracting site), indicating that the interaction is also present in solution. Quantum chemical calculations show that the XB complements an HB at this site and that solvent structure must also be considered in trying to design molecular interactions such as XBs into bio...</td>
</tr>
<tr id="bib_Scholfield2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Scholfield2017,
  author = {Scholfield, Matthew R and Ford, Melissa and Carlsson, Anna-Carin and Butta, Hawera and Mehl, Ryan A and Ho, Pui},
  title = {Structure-Energy Relationships of Halogen Bonds in Proteins},
  journal = {Biochemistry},
  year = {2017},
  pages = {acs.biochem.7b00022 - acs.biochem.7b00022},
  doi = {https://doi.org/10.1021/acs.biochem.7b00022}
}
</pre></td>
</tr>
<tr id="Schramm2011" class="entry">
	<td>Schramm, V.L.</td>
	<td>Enzymatic transition States, transition-state analogs, dynamics, thermodynamics, and lifetimes. <p class="infolinks">[<a href="javascript:toggleInfo('Schramm2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schramm2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Annual review of biochemistry<br/>Vol. 80(1), pp. 703 - 732&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1146/annurev-biochem-061809-100742">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Schramm2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Experimental analysis of enzymatic transition-state structures uses kinetic isotope effects (KIEs) to report on bonding and geometry differences between reactants and the transition state. Computational correlation of experimental values with chemical models permits three-dimensional geometric and electrostatic assignment of transition states formed at enzymatic catalytic sites. The combination of experimental and computational access to transition-state information permits (a) the design of transition-state analogs as powerful enzymatic inhibitors, (b) exploration of protein features linked to transition-state structure, (c) analysis of ensemble atomic motions involved in achieving the transition state, (d) transition-state lifetimes, and (e) separation of ground-state (Michaelis complexes) from transition-state effects. Transition-state analogs with picomolar dissociation constants have been achieved for several enzymatic targets. Transition states of closely related isozymes indicate that the protein's dynamic architecture is linked to transition-state structure. Fast dynamic motions in catalytic sites are linked to transition-state generation. Enzymatic transition states have lifetimes of femtoseconds, the lifetime of bond vibrations. Binding isotope effects (BIEs) reveal relative reactant and transition-state analog binding distortion for comparison with actual transition states.</td>
</tr>
<tr id="bib_Schramm2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Schramm2011,
  author = {Schramm, Vern L},
  title = {Enzymatic transition States, transition-state analogs, dynamics, thermodynamics, and lifetimes.},
  journal = {Annual review of biochemistry},
  year = {2011},
  volume = {80},
  number = {1},
  pages = {703 - 732},
  doi = {https://doi.org/10.1146/annurev-biochem-061809-100742}
}
</pre></td>
</tr>
<tr id="Schuss2013" class="entry">
	<td>Schuss, Z.</td>
	<td>Brownian Dynamics at Boundaries and Interfaces: In Physics, Chemistry, and Biology <p class="infolinks">[<a href="javascript:toggleInfo('Schuss2013','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schuss2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>, pp. 322&nbsp;</td>
	<td>book</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Schuss2013" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Brownian dynamics serve as mathematical models for the diffusive motion of microscopic particles of various shapes in gaseous, liquid, or solid environments. The renewed interest in Brownian dynamics is due primarily to their key role in molecular and cellular biophysics: diffusion of ions and molecules is the driver of all life. Brownian dynamics simulations are the numerical realizations of stochastic differential equations that model the functions of biological micro devices such as protein ionic channels of biological membranes, cardiac myocytes, neuronal synapses, and many more. Stochastic differential equations are ubiquitous models in computational physics, chemistry, biophysics, computer science, communications theory, mathematical finance theory, and many other disciplines. Brownian dynamics simulations of the random motion of particles, be it molecules or stock prices, give rise to mathematical problems that neither the kinetic theory of Maxwell and Boltzmann, nor Einstein's and Langevin's theories of Brownian motion could predict.This book takes the readers on a journey that starts with the rigorous definition of mathematical Brownian motion, and ends with the explicit solution of a series of complex problems that have immediate applications. It is aimed at applied mathematicians, physicists, theoretical chemists, and physiologists who are interested in modeling, analysis, and simulation of micro devices of microbiology. The book contains exercises and worked out examples throughout.</td>
</tr>
<tr id="bib_Schuss2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{Schuss2013,
  author = {Schuss, Zeev},
  title = {Brownian Dynamics at Boundaries and Interfaces: In Physics, Chemistry, and Biology},
  publisher = {Springer},
  year = {2013},
  pages = {322}
}
</pre></td>
</tr>
<tr id="Schuss1979" class="entry">
	<td>Schuss, Z. and Matkowsky, BJ.</td>
	<td>The Exit Problem : A New Approach to Diffusion Across Potential Barriers <p class="infolinks">[<a href="javascript:toggleInfo('Schuss1979','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schuss1979','bibtex')">BibTeX</a>]</p></td>
	<td>1979</td>
	<td>SIAM Journal on Applied Mathematics<br/>Vol. 36(3), pp. 604 - 623&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Schuss1979" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We consider the problem of a Brownian particle confined in a potential well of forces, which escapes the potential barrrier as the result of white noise forces acting on it. The problem is characterized by a diffusion process in a force field and is described by Langevin's stochastic differential equation. We consider potential wells with many transition states and compute the expected exit time of the particle from the well as well as the probability distribution of the exit points. Our method relates these quantities to the solutions of certain singularly perturbed elliptic boundary value problems which are solved asymptotically. Our results are then applied to the calculation of chemical reaction rates by considering the breaking of chemical bonds caused by random molecular collisions, and to the calculation of the diffusion matrix in crystals by considering random atomic migration in the periodic force field of the crystal lattice, caused by thermal vibrations of the lattice.</td>
</tr>
<tr id="bib_Schuss1979" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Schuss1979,
  author = {Schuss, Z and Matkowsky, BJ},
  title = {The Exit Problem : A New Approach to Diffusion Across Potential Barriers},
  journal = {SIAM Journal on Applied Mathematics},
  year = {1979},
  volume = {36},
  number = {3},
  pages = {604 - 623}
}
</pre></td>
</tr>
<tr id="Schwartz2010" class="entry">
	<td>Schwartz, C.M., Cheng, A., Mughal, M.R., Mattson, M.P. and Yao, P.J.</td>
	<td>Clathrin assembly proteins AP180 and CALM in the embryonic rat brain. <p class="infolinks">[<a href="javascript:toggleInfo('Schwartz2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schwartz2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>The Journal of comparative neurology<br/>Vol. 518, pp. 3803-3818&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/cne.22425">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Schwartz2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-coated vesicles are known to play diverse and pivotal roles in cells. The proper formation of clathrin-coated vesicles is dependent on, and highly regulated by, a large number of clathrin assembly proteins. These assembly proteins likely determine the functional specificity of clathrin-coated vesicles, and together they control a multitude of intracellular trafficking pathways, including those involved in embryonic development. In this study, we focus on two closely related clathrin assembly proteins, AP180 and CALM (clathrin assembly lymphoid myeloid leukemia protein), in the developing embryonic rat brain. We find that AP180 begins to be expressed at embryonic day 14 (E14), but only in postmitotic cells that have acquired a neuronal fate. CALM, on the other hand, is expressed as early as E12, by both neural stem cells and postmitotic neurons. In vitro loss-of-function studies using RNA interference (RNAi) indicate that AP180 and CALM are dispensable for some aspects of embryonic neurogenesis but are required for the growth of postmitotic neurons. These results identify the developmental stage of AP180 and CALM expression and suggest that each protein has distinct functions in neural development.</td>
</tr>
<tr id="bib_Schwartz2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Schwartz2010,
  author = {Schwartz, Catherine M and Cheng, Aiwu and Mughal, Mohamed R and Mattson, Mark P and Yao, Pamela J},
  title = {Clathrin assembly proteins AP180 and CALM in the embryonic rat brain.},
  journal = {The Journal of comparative neurology},
  year = {2010},
  volume = {518},
  pages = {3803--3818},
  doi = {https://doi.org/10.1002/cne.22425}
}
</pre></td>
</tr>
<tr id="Schwartz1996" class="entry">
	<td>Schwartz, S.D.</td>
	<td>Quantum activated rates - an evolution operator approach <p class="infolinks">[<a href="javascript:toggleInfo('Schwartz1996','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schwartz1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>The Journal of Chemical Physics<br/>Vol. 105(16), pp. 6871 - 6879&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.471981">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Schwartz1996" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: This article presents a derivation of the rate of reaction in the quantum activated rate problem. In this problem, one studies the rate of a chemical reaction when the reaction is placed in a dissipative bath. Our derivation defines the rate in terms of the flux autocorrelation function and proceeds via the recently developed interaction representation for nonadiabatic corrections to adiabatic evolution operators. This methodology is an infinite order resummation of nonadiabatic corrections to evolution operators. The approach produces an analytic expression which yields accurate results over a range of temperatures, viscosities and system parameters through the Kramers turnover region.</td>
</tr>
<tr id="bib_Schwartz1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Schwartz1996,
  author = {Schwartz, Steven D},
  title = {Quantum activated rates - an evolution operator approach},
  journal = {The Journal of Chemical Physics},
  year = {1996},
  volume = {105},
  number = {16},
  pages = {6871 - 6879},
  note = {NULL},
  doi = {https://doi.org/10.1063/1.471981}
}
</pre></td>
</tr>
<tr id="Schwartz2005" class="entry">
	<td>Schwartz, S.D.</td>
	<td>Vibrationally Enhanced Tunneling and Kinetic Isotope Effects in Enzymatic Reactions <p class="infolinks">[<a href="javascript:toggleInfo('Schwartz2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Isotope Effects In Chemistry and Biology, pp. 475&nbsp;</td>
	<td>inbook</td>
	<td><a href="https://doi.org/10.1201/9781420028027.ch18">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Schwartz2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inbook{Schwartz2005,
  author = {Schwartz, Steven D},
  title = {Vibrationally Enhanced Tunneling and Kinetic Isotope Effects in Enzymatic Reactions},
  booktitle = {Isotope Effects In Chemistry and Biology},
  publisher = {taylorfrancis},
  year = {2005},
  pages = {475},
  note = {NULL},
  doi = {https://doi.org/10.1201/9781420028027.ch18}
}
</pre></td>
</tr>
<tr id="Schwartz2013" class="entry">
	<td>Schwartz, S.D.</td>
	<td>Protein dynamics and the enzymatic reaction <p class="infolinks">[<a href="javascript:toggleInfo('Schwartz2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Top. Curr. Chem.<br/>Vol. 337, pp. 189 - 208&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1007/128">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Schwartz2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Schwartz2013,
  author = {Schwartz, Steven D},
  title = {Protein dynamics and the enzymatic reaction},
  journal = {Top. Curr. Chem.},
  year = {2013},
  volume = {337},
  pages = {189 - 208},
  doi = {https://doi.org/10.1007/128}
}
</pre></td>
</tr>
<tr id="Schwarz1989" class="entry">
	<td>Schwarz, F.P., Steer, C.J. and Kirchhoff, W.H.</td>
	<td>A differential scanning calorimetric study of brain clathrin. <p class="infolinks">[<a href="javascript:toggleInfo('Schwarz1989','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schwarz1989','bibtex')">BibTeX</a>]</p></td>
	<td>1989</td>
	<td>Archives of biochemistry and biophysics<br/>Vol. 273, pp. 433-439&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Schwarz1989" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The thermal denaturation of clathrin-coated vesicles isolated from bovine brain tissue has been studied by differential scanning calorimetry and has been compared to basket structures reformed from isolated triskelion trimers of clathrin and to isolated triskelions. The coated vesicles and reformed baskets displayed similar, yet distinct, thermal behavior. Calorimetric data of the coated vesicles exhibited a single denaturation transition peak at 55.9 +/- 0.1 degrees C, skewed to low temperatures whereas the thermograms for the reformed baskets exhibited a broad transition peak at 53.1 +/- 0.1 degrees C and a peak at 56.3 +/- 0.1 degrees C. Neither transition was reversible. The specific transition enthalpy was 11.5 +/- 1.0 J g-1 for the coated vesicles and the total transition enthalpy was 9.1 +/- 0.3 J g-1 for the reformed baskets. In contrast, isolated triskelions showed no thermal transition between 15 and 90 degrees C. Although the coated vesicles and the reformed baskets have similar stability reflecting their similar structures, the coated vesicles appear to be marginally more stable than the reformed baskets. The complexity of the transition profiles and their lack of symmetry suggest the existence of several, somewhat independent, domains unique to the cage-like structure of the coated vesicles and reformed baskets.</td>
</tr>
<tr id="bib_Schwarz1989" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Schwarz1989,
  author = {Schwarz, F P and Steer, C J and Kirchhoff, W H},
  title = {A differential scanning calorimetric study of brain clathrin.},
  journal = {Archives of biochemistry and biophysics},
  year = {1989},
  volume = {273},
  pages = {433--439},
  note = {clathrin polymerization appears to go through an intermediate of 4-7 trimers}
}
</pre></td>
</tr>
<tr id="Senftle2016" class="entry">
	<td>Senftle, T.P., Hong, S., Islam, M., Kylasa, S.B., Zheng, Y., Shin, Y., Junkermeier, C., Roman, Engel-Herbert,., Janik, M.J., Aktulga, H., Verstraelen, T., Grama, A. and van Duin, A.CT.</td>
	<td>The ReaxFF reactive force-field: development, applications and future directions <p class="infolinks">[<a href="javascript:toggleInfo('Senftle2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Senftle2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>npj Computational Materials<br/>Vol. 2(November 2015), pp. 15011 - 15011&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/npjcompumats.2015.11">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Senftle2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The reactive force-field (ReaxFF) interatomic potential is a powerful computational tool for exploring, developing and optimizing material properties. Methods based on the principles of quantum mechanics (QM), while offering valuable theoretical guidance at the electronic level, are often too computationally intense for simulations that consider the full dynamic evolution of a system. Alternatively, empirical interatomic potentials that are based on classical principles require significantly fewer computational resources, which enables simulations to better describe dynamic processes over longer timeframes and on larger scales. Such methods, however, typically require a predefined connectivity between atoms, precluding simulations that involve reactive events. The ReaxFF method was developed to help bridge this gap. Approaching the gap from the classical side, ReaxFF casts the empirical interatomic potential within a bond-order formalism, thus implicitly describing chemical bonding without expensive QM calculations. This article provides an overview of the development, application, and future directions of the ReaxFF method.</td>
</tr>
<tr id="bib_Senftle2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Senftle2016,
  author = {Senftle, Thomas P and Hong, Sungwook and Islam, Md and Kylasa, Sudhir B and Zheng, Yuanxia and Shin, Yun and Junkermeier, Chad and Roman, Engel-Herbert, and Janik, Michael J and Aktulga, Hasan and Verstraelen, Toon and Grama, Ananth and van Duin, Adri CT},
  title = {The ReaxFF reactive force-field: development, applications and future directions},
  journal = {npj Computational Materials},
  year = {2016},
  volume = {2},
  number = {November 2015},
  pages = {15011 - 15011},
  doi = {https://doi.org/10.1038/npjcompumats.2015.11}
}
</pre></td>
</tr>
<tr id="Sengar1999" class="entry">
	<td>Sengar, AS., Wang, W., Bishay, J., Cohen, S. and Egan, SE.</td>
	<td>The EH and SH3 domain Ese proteins regulate endocytosis by linking to dynamin and Eps15. <p class="infolinks">[<a href="javascript:toggleInfo('Sengar1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Sengar1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>The EMBO journal<br/>Vol. 18(5), pp. 1159 - 71&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/emboj/18.5.1159">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Sengar1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis is a multistep process which requires interaction between a number of conserved proteins. We have cloned two mammalian genes which code for a number of endocytic adaptor proteins. Two of these proteins, termed Ese1 and Ese2, contain two N-terminal EH domains, a central coiled-coil domain and five C-terminal SH3 domains. Ese1 is constitutively associated with Eps15 proteins to form a complex with at least 14 protein-protein interaction surfaces. Yeast two-hybrid assays have revealed that Ese1 EH and SH3 domains bind epsin family proteins and dynamin, respectively. Overexpression of Ese1 is sufficient to block clathrin-mediated endocytosis in cultured cells, presumably through disruption of higher order protein complexes, which are assembled on the endogenous Ese1-Eps15 scaffold. The Ese1-Eps15 scaffold therefore links dynamin, epsin and other endocytic pathway components.</td>
</tr>
<tr id="bib_Sengar1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Sengar1999,
  author = {Sengar, AS and Wang, W and Bishay, J and Cohen, S and Egan, SE},
  title = {The EH and SH3 domain Ese proteins regulate endocytosis by linking to dynamin and Eps15.},
  journal = {The EMBO journal},
  year = {1999},
  volume = {18},
  number = {5},
  pages = {1159 - 71},
  note = {ESE1 - EPS15. ESE1 is the mouse homolog of ITSN1. seems to have the same binding sites},
  doi = {https://doi.org/10.1093/emboj/18.5.1159}
}
</pre></td>
</tr>
<tr id="Shao2015" class="entry">
	<td>Shao, Y., Gan, Z., Epifanovsky, E., Gilbert, A., Wormit, M., Kussmann, J., Lange, A.W., Behn, A., Deng, J., Feng, X., Ghosh, D., Goldey, M., Horn, P.R., Jacobson, L.D., Kaliman, I., Khaliullin, R.Z., Ku&#347;, T., Landau, A., Liu, J., Proynov, E.I., Rhee, Y., Richard, R.M., Rohrdanz, M.A., Steele, R.P., Sundstrom, E.J., Woodcock, L.H., Zimmerman, P.M., Zuev, D., Albrecht, B., Alguire, E., Austin, B., Beran, G.JO., Bernard, Y.A., Berquist, E., Brandhorst, K., Bravaya, K.B., Brown, S.T., Casanova, D., Chang, Chun-Min., Chen, Y., Chien, S., Closser, K.D., Crittenden, D.L., Diedenhofen, M., DiStasio, Robert, A, Do, H., Dutoi, A.D., Edgar, R.G., Fatehi, S., Laszlo, Fusti-Molnar,., Ghysels, A., Anna, Golubeva-Zadorozhnaya,., Gomes, J., Magnus, Hanson-Heine,., Harbach, P., Hauser, A.W., Hohenstein, E.G., Holden, Z.C., Jagau, Thomas-C.., Ji, H., Kaduk, B., Khistyaev, K., Kim, J., Kim, J., King, R.A., Klunzinger, P., Kosenkov, D., Kowalczyk, T., Krauter, C.M., Lao, K., Laurent, A.D., Lawler, K.V., Levchenko, S.V., Lin, C., Liu, F., Livshits, E., Lochan, R.C., Luenser, A., Manohar, P., Manzer, S.F., Mao, Shan-Ping., Mardirossian, N., Marenich, A.V., Maurer, S.A., Mayhall, N.J., Neuscamman, E., Oana, M.C., Roberto, Olivares-Amaya,., O'Neill, Darragh, P, Parkhill, J.A., Perrine, T.M., Peverati, R., Prociuk, A., Rehn, D.R., Rosta, E., Russ, N.J., Sharada, S.M., Sharma, S., Small, D.W., Sodt, A., Stein, T., St&uuml;ck, D., Su, Yu-Chuan., Thom, A., Tsuchimochi, T., Vanovschi, V., Vogt, L., Vydrov, O., Wang, T., Watson, M.A., Wenzel, J., White, A., Williams, C.F., Yang, J., Yeganeh, S., Yost, S.R., You, Zhi-Qiang., Zhang, I., Zhang, X., Zhao, Y., Brooks, B.R., Chan, G., Chipman, D.M., Cramer, C.J., Goddard, W.A., Gordon, M.S., Hehre, W.J., Klamt, A., Schaefer, H.F., Schmidt, M.W., Sherrill, D.C., Truhlar, D.G., Warshel, A., Xu, X., Al&aacute;n, Aspuru-Guzik,., Baer, R., Bell, A.T., Besley, N.A., Chai, Jeng-Da., Dreuw, A., Dunietz, B.D., Furlani, T.R., Gwaltney, S.R., Hsu, Chao-Ping., Jung, Y., Kong, J., Lambrecht, D.S., Liang, WanZhen., Ochsenfeld, C., Rassolov, V.A., Slipchenko, L.V., Subotnik, J.E., Voorhis, T., Herbert, J.M., Krylov, A.I., Gill, P. and Martin, Head-Gordon,.</td>
	<td>Advances in molecular quantum chemistry contained in the Q-Chem 4 program package <p class="infolinks">[<a href="javascript:toggleInfo('Shao2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Shao2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Molecular Physics<br/>Vol. 113(2), pp. 184 - 215&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1080/00268976.2014.952696">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Shao2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A summary of the technical advances that are incorporated in the fourth major release of the Q-CHEM quantum chemistry program is provided, covering approximately the last seven years. These include developments in density functional theory methods and algorithms, nuclear magnetic resonance (NMR) property evaluation, coupled cluster and perturbation theories, methods for electronically excited and open-shell species, tools for treating extended environments, algorithms for walking on potential surfaces, analysis tools, energy and electron transfer modelling, parallel computing capabilities, and graphical user interfaces. In addition, a selection of example case studies that illustrate these capabilities is given. These include extensive benchmarks of the comparative accuracy of modern density functionals for bonded and non-bonded interactions, tests of attenuated second order M&oslash;ller--Plesset (MP2) methods for intermolecular interactions, a variety of parallel performance benchmarks, and tests of the accuracy of implicit solvation models. Some specific chemical examples include calculations on the strongly correlated Cr 2 dimer, exploring zeolite-catalysed ethane dehydrogenation, energy decomposition analysis of a charged ter-molecular complex arising from glycerol photoionisation, and natural transition orbitals for a Frenkel exciton state in a nine-unit model of a self-assembling nanotube.</td>
</tr>
<tr id="bib_Shao2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Shao2015,
  author = {Shao, Yihan and Gan, Zhengting and Epifanovsky, Evgeny and Gilbert, Andrew and Wormit, Michael and Kussmann, Joerg and Lange, Adrian W and Behn, Andrew and Deng, Jia and Feng, Xintian and Ghosh, Debashree and Goldey, Matthew and Horn, Paul R and Jacobson, Leif D and Kaliman, Ilya and Khaliullin, Rustam Z and Ku&#347;, Tomasz and Landau, Arie and Liu, Jie and Proynov, Emil I and Rhee, Young and Richard, Ryan M and Rohrdanz, Mary A and Steele, Ryan P and Sundstrom, Eric J and Woodcock, Lee H and Zimmerman, Paul M and Zuev, Dmitry and Albrecht, Ben and Alguire, Ethan and Austin, Brian and Beran, Gregory JO and Bernard, Yves A and Berquist, Eric and Brandhorst, Kai and Bravaya, Ksenia B and Brown, Shawn T and Casanova, David and Chang, Chun-Min and Chen, Yunqing and Chien, Siu and Closser, Kristina D and Crittenden, Deborah L and Diedenhofen, Michael and A, DiStasio, Robert and Do, Hainam and Dutoi, Anthony D and Edgar, Richard G and Fatehi, Shervin and Laszlo, Fusti-Molnar, and Ghysels, An and Anna, Golubeva-Zadorozhnaya, and Gomes, Joseph and Magnus, Hanson-Heine, and Harbach, Philipp and Hauser, Andreas W and Hohenstein, Edward G and Holden, Zachary C and Jagau, Thomas-C. and Ji, Hyunjun and Kaduk, Benjamin and Khistyaev, Kirill and Kim, Jaehoon and Kim, Jihan and King, Rollin A and Klunzinger, Phil and Kosenkov, Dmytro and Kowalczyk, Tim and Krauter, Caroline M and Lao, Ka and Laurent, Ad&egrave;le D and Lawler, Keith V and Levchenko, Sergey V and Lin, Ching and Liu, Fenglai and Livshits, Ester and Lochan, Rohini C and Luenser, Arne and Manohar, Prashant and Manzer, Samuel F and Mao, Shan-Ping and Mardirossian, Narbe and Marenich, Aleksandr V and Maurer, Simon A and Mayhall, Nicholas J and Neuscamman, Eric and Oana, Melania C and Roberto, Olivares-Amaya, and P, O'Neill, Darragh and Parkhill, John A and Perrine, Trilisa M and Peverati, Roberto and Prociuk, Alexander and Rehn, Dirk R and Rosta, Edina and Russ, Nicholas J and Sharada, Shaama M and Sharma, Sandeep and Small, David W and Sodt, Alexander and Stein, Tamar and St&uuml;ck, David and Su, Yu-Chuan and Thom, Alex and Tsuchimochi, Takashi and Vanovschi, Vitalii and Vogt, Leslie and Vydrov, Oleg and Wang, Tao and Watson, Mark A and Wenzel, Jan and White, Alec and Williams, Christopher F and Yang, Jun and Yeganeh, Sina and Yost, Shane R and You, Zhi-Qiang and Zhang, Igor and Zhang, Xing and Zhao, Yan and Brooks, Bernard R and Chan, Garnet and Chipman, Daniel M and Cramer, Christopher J and Goddard, William A and Gordon, Mark S and Hehre, Warren J and Klamt, Andreas and Schaefer, Henry F and Schmidt, Michael W and Sherrill, David C and Truhlar, Donald G and Warshel, Arieh and Xu, Xin and Al&aacute;n, Aspuru-Guzik, and Baer, Roi and Bell, Alexis T and Besley, Nicholas A and Chai, Jeng-Da and Dreuw, Andreas and Dunietz, Barry D and Furlani, Thomas R and Gwaltney, Steven R and Hsu, Chao-Ping and Jung, Yousung and Kong, Jing and Lambrecht, Daniel S and Liang, WanZhen and Ochsenfeld, Christian and Rassolov, Vitaly A and Slipchenko, Lyudmila V and Subotnik, Joseph E and Voorhis, Troy and Herbert, John M and Krylov, Anna I and Gill, Peter and Martin, Head-Gordon,},
  title = {Advances in molecular quantum chemistry contained in the Q-Chem 4 program package},
  journal = {Molecular Physics},
  year = {2015},
  volume = {113},
  number = {2},
  pages = {184 - 215},
  doi = {https://doi.org/10.1080/00268976.2014.952696}
}
</pre></td>
</tr>
<tr id="Shaw1997" class="entry">
	<td>Shaw, J.P., Petsko, G.A. and Ringe, D.</td>
	<td>Determination of Structure of Alanine Racemase from Bacillus stearothermophilus at 1.9-&Aring; Resolution <p class="infolinks">[<a href="javascript:toggleInfo('Shaw1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Biochemistry<br/>Vol. 36, pp. 1329 - 1342&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Shaw1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Shaw1997,
  author = {Shaw, Jeffrey P and Petsko, Gregory A and Ringe, Dagmar},
  title = {Determination of Structure of Alanine Racemase from Bacillus stearothermophilus at 1.9-&Aring; Resolution},
  journal = {Biochemistry},
  year = {1997},
  volume = {36},
  pages = {1329 - 1342}
}
</pre></td>
</tr>
<tr id="Shih1995" class="entry">
	<td>Shih, W., Gallusser, A. and Kirchhausen, T.</td>
	<td>A clathrin-binding site in the hinge of the beta 2 chain of mammalian AP-2 complexes. <p class="infolinks">[<a href="javascript:toggleInfo('Shih1995','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Shih1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>The Journal of biological chemistry<br/>Vol. 270, pp. 31083-31090&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Shih1995" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The assembly of cytosolic clathrin into the cytoplasmic face of coated pits and coated vesicles appears to be driven by the clathrin-associated protein (AP) complexes. We have previously shown that one of the large chains of the AP complexes, the beta chain, is sufficient to drive coat assembly in vitro. This chain consists of two domains, the amino-terminal trunk and the carboxyl-terminal ear, linked by a "hinge." We report here that presence of the hinge in recombinant beta trunk or in recombinant beta ear fragments is essential for driving in vitro assembly of clathrin into coats. We have also used a binding assay to map the clathrin-binding site by nested deletion of hinge sequences to a 50-residue region in the center of the hinge. This sequence is conserved in all known beta sequences from multicellular organisms. The interaction of a single beta hinge with a clathrin triskelion is weak, and we propose that recruitment of cytosolic clathrin to a forming coated pit involves simultaneous contacts between the legs of single clathrin trimers and the beta hinges of two or three membrane-bound AP complexes. Uncoating is likely to require interruption of these contacts.</td>
</tr>
<tr id="bib_Shih1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Shih1995,
  author = {Shih, W and Gallusser, A and Kirchhausen, T},
  title = {A clathrin-binding site in the hinge of the beta 2 chain of mammalian AP-2 complexes.},
  journal = {The Journal of biological chemistry},
  year = {1995},
  volume = {270},
  pages = {31083--31090}
}
</pre></td>
</tr>
<tr id="Shimada2007" class="entry">
	<td>Shimada, A., Niwa, H., Tsujita, K., Suetsugu, S., Nitta, K., Hanawa-Suetsugu, K., Akasaka, R., Nishino, Y., Toyama, M., Chen, L., Liu, Z.-J., Wang, B.-C., Yamamoto, M., Terada, T., Miyazawa, A., Tanaka, A., Sugano, S., Shirouzu, M., Nagayama, K., Takenawa, T. and Yokoyama, S.</td>
	<td>Curved EFC/F-BAR-domain dimers are joined end to end into a filament for membrane invagination in endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Shimada2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Shimada2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Cell<br/>Vol. 129, pp. 761-772&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cell.2007.03.040">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Shimada2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Pombe Cdc15 homology (PCH) proteins play an important role in a variety of actin-based processes, including clathrin-mediated endocytosis (CME). The defining feature of the PCH proteins is an evolutionarily conserved EFC/F-BAR domain for membrane association and tubulation. In the present study, we solved the crystal structures of the EFC domains of human FBP17 and CIP4. The structures revealed a gently curved helical-bundle dimer of approximately 220 A in length, which forms filaments through end-to-end interactions in the crystals. The curved EFC dimer fits a tubular membrane with an approximately 600 A diameter. We subsequently proposed a model in which the curved EFC filament drives tubulation. In fact, striation of tubular membranes was observed by phase-contrast cryo-transmission electron microscopy, and mutations that impaired filament formation also impaired membrane tubulation and cell membrane invagination. Furthermore, FBP17 is recruited to clathrin-coated pits in the late stage of CME, indicating its physiological role.</td>
</tr>
<tr id="bib_Shimada2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Shimada2007,
  author = {Shimada, Atsushi and Niwa, Hideaki and Tsujita, Kazuya and Suetsugu, Shiro and Nitta, Koji and Hanawa-Suetsugu, Kyoko and Akasaka, Ryogo and Nishino, Yuri and Toyama, Mitsutoshi and Chen, Lirong and Liu, Zhi-Jie and Wang, Bi-Cheng and Yamamoto, Masaki and Terada, Takaho and Miyazawa, Atsuo and Tanaka, Akiko and Sugano, Sumio and Shirouzu, Mikako and Nagayama, Kuniaki and Takenawa, Tadaomi and Yokoyama, Shigeyuki},
  title = {Curved EFC/F-BAR-domain dimers are joined end to end into a filament for membrane invagination in endocytosis.},
  journal = {Cell},
  year = {2007},
  volume = {129},
  pages = {761--772},
  doi = {https://doi.org/10.1016/j.cell.2007.03.040}
}
</pre></td>
</tr>
<tr id="Shimada2016" class="entry">
	<td>Shimada, A., Yamaguchi, A. and Kohda, D.</td>
	<td>Structural basis for the recognition of two consecutive mutually interacting DPF motifs by the SGIP1 μ homology domain <p class="infolinks">[<a href="javascript:toggleInfo('Shimada2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Shimada2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Scientific Reports<br/>Vol. 6(September 2015), pp. 19565 - 19565&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/srep19565">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Shimada2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: FCHo1, FCHo2, and SGIP1 are key regulators of clathrin-mediated endocytosis. Their μ homology domains (μHDs) interact with the C-terminal region of an endocytic scaffold protein, Eps15, containing fifteen Asp-Pro-Phe (DPF) motifs. Here, we show that the high-affinity μHD-binding site in Eps15 is a region encompassing six consecutive DPF motifs, while the minimal μHD-binding unit is two consecutive DPF motifs. We present the crystal structures of the SGIP1 μHD in complex with peptides containing two DPF motifs. The peptides bind to a novel ligand-binding site of the μHD, which is distinct from those of other distantly related μHD-containing proteins. The two DPF motifs, which adopt three-dimensional structures stabilized by sequence-specific intramotif and intermotif interactions, are extensively recognized by the μHD and are both required for binding. Thus, consecutive and singly scattered DPF motifs play distinct roles in μHD binding.</td>
</tr>
<tr id="bib_Shimada2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Shimada2016,
  author = {Shimada, Atsushi and Yamaguchi, Atsuko and Kohda, Daisuke},
  title = {Structural basis for the recognition of two consecutive mutually interacting DPF motifs by the SGIP1 μ homology domain},
  journal = {Scientific Reports},
  year = {2016},
  volume = {6},
  number = {September 2015},
  pages = {19565 - 19565},
  note = {human cells. n FCHO1/2 and SGIP binding with EPS15 and alpha-adaptin. n Some domain information. n Some dissociation constant information. n Suggests that FCHO1/2/SGIP - EPS15 - AP2 (alpha) form a ternary complex.},
  doi = {https://doi.org/10.1038/srep19565}
}
</pre></td>
</tr>
<tr id="Shimizu2018" class="entry">
	<td>Shimizu, M. and Takada, S.</td>
	<td>Reconstruction of atomistic structures from coarse-grained models for protein-DNA complexes <p class="infolinks">[<a href="javascript:toggleInfo('Shimizu2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Journal of Chemical Theory and Computation&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Shimizu2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Shimizu2018,
  author = {Shimizu, Masahiro and Takada, Shoji},
  title = {Reconstruction of atomistic structures from coarse-grained models for protein-DNA complexes},
  journal = {Journal of Chemical Theory and Computation},
  publisher = {Cold Spring Harbor Laboratory},
  year = {2018}
}
</pre></td>
</tr>
<tr id="Shoemark2007" class="entry">
	<td>Shoemark, D.K., Cliff, M.J., Sessions, R.B. and Clarke, A.R.</td>
	<td>Enzymatic properties of the lactate dehydrogenase enzyme from Plasmodium falciparum <p class="infolinks">[<a href="javascript:toggleInfo('Shoemark2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Shoemark2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>FEBS Journal<br/>Vol. 274(11), pp. 2738 - 2748&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1111/j.1742-4658.2007.05808.x">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Shoemark2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The lactate dehydrogenase enzyme from Plasmodium falciparum (PfLDH) is a target for antimalarial compounds owing to structural and functional differences from the human isozymes. The plasmodial enzyme possesses a five-residue insertion in the substrate-specificity loop and exhibits less marked substrate inhibition than its mammalian counterparts. Here we provide a comprehensive kinetic analysis of the enzyme by steady-state and transient kinetic methods. The mechanism deduced by product inhibition studies proves that PfLDH shares a common mechanism with the human LDHs, that of an ordered sequential bireactant system with coenzyme binding first. Transient kinetic analysis reveals that the major rate-limiting step is the closure of the substrate-specificity loop prior to hydride transfer, in line with other LDHs. The five-residue insertion in this loop markedly increases substrate specificity compared with the human muscle and heart isoforms.</td>
</tr>
<tr id="bib_Shoemark2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Shoemark2007,
  author = {Shoemark, Deborah K and Cliff, Matthew J and Sessions, Richard B and Clarke, Anthony R},
  title = {Enzymatic properties of the lactate dehydrogenase enzyme from Plasmodium falciparum},
  journal = {FEBS Journal},
  year = {2007},
  volume = {274},
  number = {11},
  pages = {2738 - 2748},
  note = {NULL},
  doi = {https://doi.org/10.1111/j.1742-4658.2007.05808.x}
}
</pre></td>
</tr>
<tr id="Silva2011" class="entry">
	<td>Silva, R.G., Murkin, A.S. and Schramm, V.L.</td>
	<td>Femtosecond dynamics coupled to chemical barrier crossing in a Born-Oppenheimer enzyme <p class="infolinks">[<a href="javascript:toggleInfo('Silva2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Silva2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Proceedings of the National Academy of Science<br/>Vol. 108(46), pp. 18661 - 18665&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.1114900108">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Silva2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Contributions of fast (femtosecond) dynamic motion to barrier crossing at enzyme catalytic sites is in dispute. Human purine nucleoside phosphorylase (PNP) forms a ribocation-like transition state in the phosphorolysis of purine nucleosides and fast protein motions have been proposed to participate in barrier crossing. In the present study, (13)C-, (15)N-, (2)H-labeled human PNP (heavy PNP) was expressed, purified to homogeneity, and shown to exhibit a 9.9&#37; increase in molecular mass relative to its unlabeled counterpart (light PNP). Kinetic isotope effects and steady-state kinetic parameters were indistinguishable for both enzymes, indicating that transition-state structure, equilibrium binding steps, and the rate of product release were not affected by increased protein mass. Single-turnover rate constants were slowed for heavy PNP, demonstrating reduced probability of chemical barrier crossing from enzyme-bound substrates to enzyme-bound products. In a second, independent method to probe barrier crossing, heavy PNP exhibited decreased forward commitment factors, also revealing mass-dependent decreased probability for barrier crossing. Increased atomic mass in human PNP alters bond vibrational modes on the femtosecond time scale and reduces on-enzyme chemical barrier crossing. This study demonstrates coupling of enzymatic bond vibrations on the femtosecond time scale to barrier crossing.</td>
</tr>
<tr id="bib_Silva2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Silva2011,
  author = {Silva, Rafael G and Murkin, Andrew S and Schramm, Vern L},
  title = {Femtosecond dynamics coupled to chemical barrier crossing in a Born-Oppenheimer enzyme},
  journal = {Proceedings of the National Academy of Science},
  year = {2011},
  volume = {108},
  number = {46},
  pages = {18661 - 18665},
  note = {From Duplicate 2 (Femtosecond dynamics coupled to chemical barrier crossing in a Born-Oppenheimer enzyme - Silva, Rafael G.; Murkin, Andrew S.; Schramm, Vern L.) NULL},
  doi = {https://doi.org/10.1073/pnas.1114900108}
}
</pre></td>
</tr>
<tr id="Simon1966" class="entry">
	<td>Simon, H. and Palm, D.</td>
	<td>Isotope effects in organic chemistry and biochemistry <p class="infolinks">[<a href="javascript:toggleInfo('Simon1966','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Simon1966','bibtex')">BibTeX</a>]</p></td>
	<td>1966</td>
	<td>Angewandte Chemie International Edition<br/>Vol. 5(11), pp. 920 - 933&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/anie.196609201">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Simon1966" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The present article shows the extent to which isotope effects are likely to be encountered in the use of isotope techniques and what problems are studied with primary and secondary isotope effects. By way of example, the results of studies on E2 reactions, particularly in the Hofmann degradation, are discussed, followed by a discussion of some ``analytical isotope effects''. Finally, the problems encountered and the information that can be obtained from isotope effects studies on enzyme reactions, and the advantages and disadvantages of competitive and non-competitive techniques are described. In addition to a survey of isotope effects in dehydrogenase reactions, new isotope effects encountered in the dehydrogenation of T-labeled alcohols are reported.</td>
</tr>
<tr id="bib_Simon1966" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Simon1966,
  author = {Simon, H and Palm, D},
  title = {Isotope effects in organic chemistry and biochemistry},
  journal = {Angewandte Chemie International Edition},
  year = {1966},
  volume = {5},
  number = {11},
  pages = {920 - 933},
  doi = {https://doi.org/10.1002/anie.196609201}
}
</pre></td>
</tr>
<tr id="Simonsen2001" class="entry">
	<td>Simonsen, A., Wurmser, A.E., Emr, S.D. and Stenmark, H.</td>
	<td>The role of phosphoinositides in membrane transport <p class="infolinks">[<a href="javascript:toggleInfo('Simonsen2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Current opinion in cell biology<br/>Vol. 13(4), pp. 485-492&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Simonsen2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Simonsen2001,
  author = {Simonsen, Anne and Wurmser, Andrew E and Emr, Scott D and Stenmark, Harald},
  title = {The role of phosphoinositides in membrane transport},
  journal = {Current opinion in cell biology},
  publisher = {Elsevier},
  year = {2001},
  volume = {13},
  number = {4},
  pages = {485--492}
}
</pre></td>
</tr>
<tr id="Simonson2007" class="entry">
	<td>Simonson, T.</td>
	<td>Free Energy Calculations: Approximate Methods for Biological Macromolecules <p class="infolinks">[<a href="javascript:toggleInfo('Simonson2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Free Energy Calculations, pp. 423&nbsp;</td>
	<td>inbook</td>
	<td><a href="https://doi.org/10.1007/978-3-540-38448-9_12">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Simonson2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inbook{Simonson2007,
  author = {Simonson, Thomas},
  title = {Free Energy Calculations: Approximate Methods for Biological Macromolecules},
  booktitle = {Free Energy Calculations},
  publisher = {Springer Berlin Heidelberg},
  year = {2007},
  pages = {423},
  doi = {https://doi.org/10.1007/978-3-540-38448-9_12}
}
</pre></td>
</tr>
<tr id="Simpson1966" class="entry">
	<td>Simpson, HJ. and Marsh, RE.</td>
	<td>The crystal structure of L -alanine <p class="infolinks">[<a href="javascript:toggleInfo('Simpson1966','bibtex')">BibTeX</a>]</p></td>
	<td>1966</td>
	<td>Acta Crystallographica<br/>Vol. 20(4), pp. 550 - 555&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1107/S0365110X66001221">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Simpson1966" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Simpson1966,
  author = {Simpson, HJ and Marsh, RE},
  title = {The crystal structure of L -alanine},
  journal = {Acta Crystallographica},
  year = {1966},
  volume = {20},
  number = {4},
  pages = {550 - 555},
  note = {Crystal structure of L-alanine},
  doi = {https://doi.org/10.1107/S0365110X66001221}
}
</pre></td>
</tr>
<tr id="Singh1986" class="entry">
	<td>Singh, C.U. and Kollman, P.A.</td>
	<td>A combined ab initio quantum mechanical and molecular mechanical method for carrying out simulations on complex molecular systems: Applications to the CH3Cl + Cl− exchange reaction and gas phase protonation of polyethers <p class="infolinks">[<a href="javascript:toggleInfo('Singh1986','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Singh1986','bibtex')">BibTeX</a>]</p></td>
	<td>1986</td>
	<td>Journal of Computational Chemistry<br/>Vol. 7(6), pp. 718 - 730&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/jcc.540070604">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Singh1986" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present an approach to couple ab initio quantum mechanical geometry optimiuzations with molecular mechanical optimizations, with the added capability to carry out molecular dynamics simulations of the systems to earch for new local minima. The approach is applied to the aqueous solution CH3Cl + Cl− exchange reaction and the gas phase protonation of polyethers.</td>
</tr>
<tr id="bib_Singh1986" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Singh1986,
  author = {Singh, Chandra U and Kollman, Peter A},
  title = {A combined ab initio quantum mechanical and molecular mechanical method for carrying out simulations on complex molecular systems: Applications to the CH3Cl + Cl− exchange reaction and gas phase protonation of polyethers},
  journal = {Journal of Computational Chemistry},
  year = {1986},
  volume = {7},
  number = {6},
  pages = {718 - 730},
  doi = {https://doi.org/10.1002/jcc.540070604}
}
</pre></td>
</tr>
<tr id="Sinko2011" class="entry">
	<td>Sinko, W., de Oliveira, C.F., Pierce, L.CT. and McCammon, Andrew, J</td>
	<td>Protecting High Energy Barriers: A New Equation to Regulate Boost Energy in Accelerated Moecular Dynamics Simulations <p class="infolinks">[<a href="javascript:toggleInfo('Sinko2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Journal of Chemical Theory and Computation<br/>Vol. 8, pp. 17 - 23&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Sinko2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Sinko2011,
  author = {Sinko, William and de Oliveira, Cesar F and Pierce, Levi CT and J, McCammon, Andrew},
  title = {Protecting High Energy Barriers: A New Equation to Regulate Boost Energy in Accelerated Moecular Dynamics Simulations},
  journal = {Journal of Chemical Theory and Computation},
  year = {2011},
  volume = {8},
  pages = {17 - 23}
}
</pre></td>
</tr>
<tr id="Skruzny2012" class="entry">
	<td>Skruzny, M., Brach, T., Ciuffa, R., Rybina, S., Wachsmuth, M. and Kaksonen, M.</td>
	<td>Molecular basis for coupling the plasma membrane to the actin cytoskeleton during clathrin-mediated endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Skruzny2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Skruzny2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Proceedings of the National Academy of Sciences of the United States of America<br/>Vol. 109, pp. E2533-E2542&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.1207011109">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Skruzny2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Dynamic actin filaments are a crucial component of clathrin-mediated endocytosis when endocytic proteins cannot supply enough energy for vesicle budding. Actin cytoskeleton is thought to provide force for membrane invagination or vesicle scission, but how this force is transmitted to the plasma membrane is not understood. Here we describe the molecular mechanism of plasma membrane-actin cytoskeleton coupling mediated by cooperative action of epsin Ent1 and the HIP1R homolog Sla2 in yeast Saccharomyces cerevisiae. Sla2 anchors Ent1 to a stable endocytic coat by an unforeseen interaction between Sla2's ANTH and Ent1's ENTH lipid-binding domains. The ANTH and ENTH domains bind each other in a ligand-dependent manner to provide critical anchoring of both proteins to the membrane. The C-terminal parts of Ent1 and Sla2 bind redundantly to actin filaments via a previously unknown phospho-regulated actin-binding domain in Ent1 and the THATCH domain in Sla2. By the synergistic binding to the membrane and redundant interaction with actin, Ent1 and Sla2 form an essential molecular linker that transmits the force generated by the actin cytoskeleton to the plasma membrane, leading to membrane invagination and vesicle budding.</td>
</tr>
<tr id="bib_Skruzny2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Skruzny2012,
  author = {Skruzny, Michal and Brach, Thorsten and Ciuffa, Rodolfo and Rybina, Sofia and Wachsmuth, Malte and Kaksonen, Marko},
  title = {Molecular basis for coupling the plasma membrane to the actin cytoskeleton during clathrin-mediated endocytosis.},
  journal = {Proceedings of the National Academy of Sciences of the United States of America},
  year = {2012},
  volume = {109},
  pages = {E2533--E2542},
  doi = {https://doi.org/10.1073/pnas.1207011109}
}
</pre></td>
</tr>
<tr id="Slattery1995" class="entry">
	<td>Slattery, P.J. and O'Brien,, SJ</td>
	<td>Molecular Phylogeny of the Red Panda (Ailurus fulgens) <p class="infolinks">[<a href="javascript:toggleInfo('Slattery1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>Journal of Heredity<br/>Vol. 86(6), pp. 413 - 422&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/oxfordjournals.jhered.a111615">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Slattery1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Slattery1995,
  author = {Slattery, Pecon J and SJ, O'Brien,},
  title = {Molecular Phylogeny of the Red Panda (Ailurus fulgens)},
  journal = {Journal of Heredity},
  year = {1995},
  volume = {86},
  number = {6},
  pages = {413 - 422},
  doi = {https://doi.org/10.1093/oxfordjournals.jhered.a111615}
}
</pre></td>
</tr>
<tr id="Slepnev2000" class="entry">
	<td>Slepnev, V.I., Ochoa, G., Butler, M.H. and Camilli, P.</td>
	<td>Tandem arrangement of the clathrin and AP-2 binding domains in amphiphysin 1 and disruption of clathrin coat function by amphiphysin fragments comprising these sites <p class="infolinks">[<a href="javascript:toggleInfo('Slepnev2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Slepnev2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Journal of Biological Chemistry<br/>Vol. 275(23), pp. 17583 - 17589&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M910430199">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Slepnev2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Amphiphysin 1 and 2 are proteins implicated in the recycling of synaptic vesicles in nerve terminals. They interact with dynamin and synaptojanin via their COOH-terminal SH3 domain, whereas their central regions contain binding sites for clathrin and for the clathrin adaptor AP-2. We have defined here amino acids of amphiphysin 1 crucial for binding to AP-2 and clathrin. Overexpression in Chinese hamster ovary cells of an amphiphysin 1 fragment that binds both AP-2 and clathrin resulted in a segregation of clathrin, which acquired a diffuse distribution, from AP-2, which accumulated at patches also positive for Eps15. These effects correlated with a block in clathrin-mediated endocytosis. A fragment selectively interacting with clathrin produced a similar effect. These results can be explained by the binding of amphiphysin to the NH(2)-terminal domain of clathrin and by a competition with the binding of this domain to the beta-subunit of AP-2 and AP180. The interaction of amphiphysin 1 with either clathrin or AP-2 did not prevent its interaction with dynamin, supporting the existence of tertiary complexes between these proteins. Together with previous evidence indicating a direct interaction between amphiphysin and membrane lipids, these findings support a model in which amphiphysin acts as a multifunctional adaptor linking the membrane to coat proteins and coat proteins to dynamin and synaptojanin.</td>
</tr>
<tr id="bib_Slepnev2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Slepnev2000,
  author = {Slepnev, Vladimir I and Ochoa, Gian and Butler, Margaret H and Camilli, Pietro},
  title = {Tandem arrangement of the clathrin and AP-2 binding domains in amphiphysin 1 and disruption of clathrin coat function by amphiphysin fragments comprising these sites},
  journal = {Journal of Biological Chemistry},
  year = {2000},
  volume = {275},
  number = {23},
  pages = {17583 - 17589},
  note = {chinese hamster. domain information competition between accessory proteins for binding to alpha-ear},
  doi = {https://doi.org/10.1074/jbc.M910430199}
}
</pre></td>
</tr>
<tr id="Slepnev1998" class="entry">
	<td>Slepnev, V.I., Ochoa, G.-C., Butler, M.H., Grabs, D. and De Camilli, P.</td>
	<td>Role of phosphorylation in regulation of the assembly of endocytic coat complexes <p class="infolinks">[<a href="javascript:toggleInfo('Slepnev1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Science<br/>Vol. 281(5378), pp. 821-824&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Slepnev1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Slepnev1998,
  author = {Slepnev, Vladimir I and Ochoa, Gian-Carlo and Butler, Margaret H and Grabs, Detlev and De Camilli, Pietro},
  title = {Role of phosphorylation in regulation of the assembly of endocytic coat complexes},
  journal = {Science},
  publisher = {American Association for the Advancement of Science},
  year = {1998},
  volume = {281},
  number = {5378},
  pages = {821--824}
}
</pre></td>
</tr>
<tr id="Smidt2008" class="entry">
	<td>de Smidt, O., du Preez, J.C. and Albertyn, J.</td>
	<td>The alcohol dehydrogenases of Saccharomyces cerevisiae: a comprehensive review. <p class="infolinks">[<a href="javascript:toggleInfo('Smidt2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Smidt2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>FEMS yeast research<br/>Vol. 8(7), pp. 967 - 78&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1111/j.1567-1364.2008.00387.x">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Smidt2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Alcohol dehydrogenases (ADHs) constitute a large family of enzymes responsible for the reversible oxidation of alcohols to aldehydes with the concomitant reduction of NAD(+) or NADP(+). These enzymes have been identified not only in yeasts, but also in several other eukaryotes and even prokaryotes. The ADHs of Saccharomyces cerevisiae have been studied intensively for over half a century. With the ever-evolving techniques available for scientific analysis and since the completion of the Yeast Genome Project, a vast amount of new information has been generated during the past 10 years. This review attempts to provide a brief summary of the wealth of knowledge gained from earlier studies as well as more recent work. Relevant aspects regarding the primary and secondary structure, kinetic characteristics, function and molecular regulation of the ADHs in S. cerevisiae are discussed in detail. A brief outlook also contemplates possible future research opportunities.</td>
</tr>
<tr id="bib_Smidt2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Smidt2008,
  author = {de Smidt, Olga and du Preez, James C and Albertyn, Jacobus},
  title = {The alcohol dehydrogenases of Saccharomyces cerevisiae: a comprehensive review.},
  journal = {FEMS yeast research},
  year = {2008},
  volume = {8},
  number = {7},
  pages = {967 - 78},
  note = {General information on YADH},
  doi = {https://doi.org/10.1111/j.1567-1364.2008.00387.x}
}
</pre></td>
</tr>
<tr id="Smith2012" class="entry">
	<td>Smith, A.M., Xu, W., Sun, Y., Faeder, J.R. and Marai, E.G.</td>
	<td>RuleBender: integrated modeling, simulation and visualization for rule-based intracellular biochemistry <p class="infolinks">[<a href="javascript:toggleInfo('Smith2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Smith2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>BMC Bioinformatics<br/>Vol. 13(Suppl {8file:///D:/Downloads/Brief\%20Bioinform-2009-Raman-435-49.pdf}), pp. S3 - S3&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1186/1471-2105-13-S8-S3">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Smith2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Background\textbackslash\nRule-based modeling (RBM) is a powerful and increasingly popular approach to modeling cell signaling networks. However, novel visual tools are needed in order to make RBM accessible to a broad range of users, to make specification of models less error prone, and to improve workflows.\textbackslash\n\textbackslash\nResults\textbackslash\nWe introduce RuleBender, a novel visualization system for the integrated visualization, modeling and simulation of rule-based intracellular biochemistry. We present the user requirements, visual paradigms, algorithms and design decisions behind RuleBender, with emphasis on visual global/local model exploration and integrated execution of simulations. The support of RBM creation, debugging, and interactive visualization expedites the RBM learning process and reduces model construction time; while built-in model simulation and results with multiple linked views streamline the execution and analysis of newly created models and generated networks.\textbackslash\n\textbackslash\nConclusion\textbackslash\nRuleBender has been adopted as both an educational and a research tool and is available as a free open source tool at http://www.rulebender.org. A development cycle that includes close interaction with expert users allows RuleBender to better serve the needs of the systems biology community.</td>
</tr>
<tr id="bib_Smith2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Smith2012,
  author = {Smith, Adam M and Xu, Wen and Sun, Yao and Faeder, James R and Marai, Elisabeta G},
  title = {RuleBender: integrated modeling, simulation and visualization for rule-based intracellular biochemistry},
  journal = {BMC Bioinformatics},
  year = {2012},
  volume = {13},
  number = {Suppl 8file:///D:/Downloads/Brief0Bioinform-2009-Raman-435-49.pdf},
  pages = {S3 - S3},
  doi = {https://doi.org/10.1186/1471-2105-13-S8-S3}
}
</pre></td>
</tr>
<tr id="Sneddon2011" class="entry">
	<td>Sneddon, M.W., Faeder, J.R. and Emonet, T.</td>
	<td>Efficient modeling, simulation and coarse-graining of biological complexity with NFsim <p class="infolinks">[<a href="javascript:toggleInfo('Sneddon2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Sneddon2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Nature Methods<br/>Vol. 8(2), pp. 177 - 183&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nmeth.1546">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Sneddon2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: textcopyright 2011 Nature America, Inc. All rights reserved. Managing the overwhelming numbers of molecular states and interactions is a fundamental obstacle to building predictive models of biological systems. Here we introduce the Network- Free Stochastic Simulator (NFsim), a general-purpose modeling platform that overcomes the combinatorial nature of molecular interactions. Unlike standard simulators that represent molecular species as variables in equations, NFsim uses a biologically intuitive representation: objects with binding and modification sites acted on by reaction rules. During simulations, rules operate directly on molecular objects to produce exact stochastic results with performance that scales independently of the reaction network size. Reaction rates can be defined as arbitrary functions of molecular states to provide powerful coarse-graining capabilities, for example to merge Boolean and kinetic representations of biological networks. NFsim enables researchers to simulate many biological systems that were previously inaccessible to general-purpose software, as we illustrate with models of immune system signaling, microbial signaling, cytoskeletal assembly and oscillating gene expression.</td>
</tr>
<tr id="bib_Sneddon2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Sneddon2011,
  author = {Sneddon, Michael W and Faeder, James R and Emonet, Thierry},
  title = {Efficient modeling, simulation and coarse-graining of biological complexity with NFsim},
  journal = {Nature Methods},
  year = {2011},
  volume = {8},
  number = {2},
  pages = {177 - 183},
  doi = {https://doi.org/10.1038/nmeth.1546}
}
</pre></td>
</tr>
<tr id="So2003" class="entry">
	<td>So, C., Lin, M., Ayton, P.M., Chen, E.H. and Cleary, M.L.</td>
	<td>Dimerization contributes to oncogenic activation of MLL chimeras in acute leukemias <p class="infolinks">[<a href="javascript:toggleInfo('So2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('So2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Cancer Cell<br/>Vol. 4(2), pp. 99 - 110&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S1535-6108(03)00188-0">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_So2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: MLL is a histone methyltransferase that can be converted into an oncoprotein by acquisition of transcriptional effector domains following heterologous protein fusions with a variety of nuclear transcription factors, cofactors, or chromatin remodeling proteins in acute leukemias. Here we demonstrate an alternative mechanism for activation of MLL following fusions with proteins (AF1p/Eps15 and GAS7) that normally reside in the cytoplasm. The coiled-coil oligomerization domains of these proteins are necessary and sufficient for leukemogenic transformation induced by the respective MLL fusion proteins. Furthermore, homodimerization of MLL by synthetic dimerization modules mimics bona fide MLL fusion proteins resulting in Hox gene activation and enhanced self-renewal of hematopoietic progenitors. Our studies support an oligomerization-dependent mechanism for oncogenic conversion of MLL, presumably in part by recruitment of accessory factors through the dimerized MLL moiety of the chimeric protein.</td>
</tr>
<tr id="bib_So2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{So2003,
  author = {So, Chi and Lin, Min and Ayton, Paul M and Chen, Everett H and Cleary, Michael L},
  title = {Dimerization contributes to oncogenic activation of MLL chimeras in acute leukemias},
  journal = {Cancer Cell},
  year = {2003},
  volume = {4},
  number = {2},
  pages = {99 - 110},
  note = {rat cells. dimerization of EPS15 through the coiled coiled region. AF1p = EPS15},
  doi = {https://doi.org/10.1016/S1535-6108(03)00188-0}
}
</pre></td>
</tr>
<tr id="Sochacki2017" class="entry">
	<td>Sochacki, K.A., Dickey, A.M., Strub, M.-P. and Taraska, J.W.</td>
	<td>Endocytic proteins are partitioned at the edge of the clathrin lattice in mammalian cells. <p class="infolinks">[<a href="javascript:toggleInfo('Sochacki2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Sochacki2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Nature cell biology<br/>Vol. 19, pp. 352-361&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/ncb3498">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Sochacki2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Dozens of proteins capture, polymerize and reshape the clathrin lattice during clathrin-mediated endocytosis (CME). How or if this ensemble of proteins is organized in relation to the clathrin coat is unknown. Here, we map key molecules involved in CME at the nanoscale using correlative super-resolution light and transmission electron microscopy. We localize 19 different endocytic proteins (amphiphysin1, AP2, β2-arrestin, CALM, clathrin, DAB2, dynamin2, EPS15, epsin1, epsin2, FCHO2, HIP1R, intersectin, NECAP, SNX9, stonin2, syndapin2, transferrin receptor, VAMP2) on thousands of individual clathrin structures, generating a comprehensive molecular architecture of endocytosis with nanoscale precision. We discover that endocytic proteins distribute into distinct spatial zones in relation to the edge of the clathrin lattice. The presence or concentrations of proteins within these zones vary at distinct stages of organelle development. We propose that endocytosis is driven by the recruitment, reorganization and loss of proteins within these partitioned nanoscale zones.</td>
</tr>
<tr id="bib_Sochacki2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Sochacki2017,
  author = {Sochacki, Kem A and Dickey, Andrea M and Strub, Marie-Paule and Taraska, Justin W},
  title = {Endocytic proteins are partitioned at the edge of the clathrin lattice in mammalian cells.},
  journal = {Nature cell biology},
  year = {2017},
  volume = {19},
  pages = {352--361},
  doi = {https://doi.org/10.1038/ncb3498}
}
</pre></td>
</tr>
<tr id="Sokkar2015" class="entry">
	<td>Sokkar, P., Boulanger, E., Thiel, W. and Elsa, Sanchez-Garcia,.</td>
	<td>Hybrid Quantum Mechanics/Molecular Mechanics/Coarse Grained Modeling: A Triple-Resolution Approach for Biomolecular Systems <p class="infolinks">[<a href="javascript:toggleInfo('Sokkar2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Sokkar2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Journal of Chemical Theory and Computation, pp. 150312082900003 - 150312082900003&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ct500956u">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Sokkar2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present a hybrid quantum mechanics/molecular mechanics/coarse-grained (QM/MM/CG) multiresolution approach for solvated biomolecular systems. The chemically important active-site region is treated at the QM level. The biomolecular environment is described by an atomistic MM force field, and the solvent is modeled with the CG Martini force field using standard or polarizable (pol-CG) water. Interactions within the QM, MM, and CG regions, and between the QM and MM regions, are treated in the usual manner, whereas the CG?MM and CG?QM interactions are evaluated using the virtual sites approach. The accuracy and efficiency of our implementation is tested for two enzymes, chorismate mutase (CM) and p-hydroxybenzoate hydroxylase (PHBH). In CM, the QM/MM/CG potential energy scans along the reaction coordinate yield reaction energies that are too large, both for the standard and polarizable Martini CG water models, which can be attributed to adverse effects of using large CG water beads. The inclusion of an atomistic MM water layer (10 &Aring; for uncharged CG water and 5 &Aring; for polarizable CG water) around the QM region improves the energy profiles compared to the reference QM/MM calculations. In analogous QM/MM/CG calculations on PHBH, the use of the pol-CG description for the outer water does not affect the stabilization of the highly charged FADHOOH-pOHB transition state compared to the fully atomistic QM/MM calculations. Detailed performance analysis in a glycine?water model system indicates that computation times for QM energy and gradient evaluations at the density functional level are typically reduced by 40?70&#37; for QM/MM/CG relative to fully atomistic QM/MM calculations.<br>We present a hybrid quantum mechanics/molecular mechanics/coarse-grained (QM/MM/CG) multiresolution approach for solvated biomolecular systems. The chemically important active-site region is treated at the QM level. The biomolecular environment is described by an atomistic MM force field, and the solvent is modeled with the CG Martini force field using standard or polarizable (pol-CG) water. Interactions within the QM, MM, and CG regions, and between the QM and MM regions, are treated in the usual manner, whereas the CG?MM and CG?QM interactions are evaluated using the virtual sites approach. The accuracy and efficiency of our implementation is tested for two enzymes, chorismate mutase (CM) and p-hydroxybenzoate hydroxylase (PHBH). In CM, the QM/MM/CG potential energy scans along the reaction coordinate yield reaction energies that are too large, both for the standard and polarizable Martini CG water models, which can be attributed to adverse effects of using large CG water beads. The inclusion of an atomistic MM water layer (10 &Aring; for uncharged CG water and 5 &Aring; for polarizable CG water) around the QM region improves the energy profiles compared to the reference QM/MM calculations. In analogous QM/MM/CG calculations on PHBH, the use of the pol-CG description for the outer water does not affect the stabilization of the highly charged FADHOOH-pOHB transition state compared to the fully atomistic QM/MM calculations. Detailed performance analysis in a glycine?water model system indicates that computation times for QM energy and gradient evaluations at the density functional level are typically reduced by 40?70&#37; for QM/MM/CG relative to fully atomistic QM/MM calculations.</td>
</tr>
<tr id="bib_Sokkar2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Sokkar2015,
  author = {Sokkar, Pandian and Boulanger, Eliot and Thiel, Walter and Elsa, Sanchez-Garcia,},
  title = {Hybrid Quantum Mechanics/Molecular Mechanics/Coarse Grained Modeling: A Triple-Resolution Approach for Biomolecular Systems},
  journal = {Journal of Chemical Theory and Computation},
  year = {2015},
  pages = {150312082900003 - 150312082900003},
  doi = {https://doi.org/10.1021/ct500956u}
}
</pre></td>
</tr>
<tr id="Soohoo2013" class="entry">
	<td>Soohoo, A.L. and Puthenveedu, M.A.</td>
	<td>Divergent modes for cargo-mediated control of clathrin-coated pit dynamics. <p class="infolinks">[<a href="javascript:toggleInfo('Soohoo2013','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Soohoo2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Molecular biology of the cell<br/>Vol. 24, pp. 1725-34, S1-12&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1091/mbc.E12-07-0550">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Soohoo2013" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis has long been viewed as a process driven by core endocytic proteins, with internalized cargo proteins being passive. In contrast, an emerging view suggests that signaling receptor cargo may actively control its fate by regulating the dynamics of clathrin-coated pits (CCPs) that mediate their internalization. Despite its physiological implications, very little is known about such "cargo-mediated regulation" of CCPs by signaling receptors. Here, using multicolor total internal reflection fluorescence microscopy imaging and quantitative analysis in live cells, we show that the μ-opioid receptor, a physiologically relevant G protein-coupled signaling receptor, delays the dynamics of CCPs in which it is localized. This delay is mediated by the interactions of two critical leucines on the receptor cytoplasmic tail. Unlike the previously known mechanism of cargo-mediated regulation, these residues regulate the lifetimes of dynamin, a key component of CCP scission. These results identify a novel means for selectively controlling the endocytosis of distinct cargo that share common trafficking components and indicate that CCP regulation by signaling receptors can operate via divergent modes.</td>
</tr>
<tr id="bib_Soohoo2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Soohoo2013,
  author = {Soohoo, Amanda L and Puthenveedu, Manojkumar A},
  title = {Divergent modes for cargo-mediated control of clathrin-coated pit dynamics.},
  journal = {Molecular biology of the cell},
  year = {2013},
  volume = {24},
  pages = {1725--34, S1-12},
  doi = {https://doi.org/10.1091/mbc.E12-07-0550}
}
</pre></td>
</tr>
<tr id="Spath1980" class="entry">
	<td>Spath, H.</td>
	<td>Cluster analysis algorithms for data reduction and classification of objects <p class="infolinks">[<a href="javascript:toggleInfo('Spath1980','bibtex')">BibTeX</a>]</p></td>
	<td>1980</td>
	<td>&nbsp;</td>
	<td>book</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Spath1980" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{Spath1980,
  author = {Spath, Helmuth},
  title = {Cluster analysis algorithms for data reduction and classification of objects},
  publisher = {Halsted Press},
  year = {1980}
}
</pre></td>
</tr>
<tr id="Spies2003" class="entry">
	<td>Spies, A.M. and Toney, M.D.</td>
	<td>Multiple hydrogen kinetic isotope effects for enzymes catalyzing exchange with solvent: application to alanine racemase. <p class="infolinks">[<a href="javascript:toggleInfo('Spies2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Biochemistry<br/>Vol. 42(17), pp. 5099 - 107&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi0274064">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Spies2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Spies2003,
  author = {Spies, Ashley M and Toney, Michael D},
  title = {Multiple hydrogen kinetic isotope effects for enzymes catalyzing exchange with solvent: application to alanine racemase.},
  journal = {Biochemistry},
  year = {2003},
  volume = {42},
  number = {17},
  pages = {5099 - 107},
  note = {KIE for AlaR},
  doi = {https://doi.org/10.1021/bi0274064}
}
</pre></td>
</tr>
<tr id="Spies2004" class="entry">
	<td>Spies, A.M., Woodward, J.J., Watnik, M.R. and Toney, M.D.</td>
	<td>Alanine Racemase Free Energy Profiles from Global Analyses of Progress Curves <p class="infolinks">[<a href="javascript:toggleInfo('Spies2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of the American Chemical Society<br/>Vol. 126, pp. 7464 - 7475&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Spies2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Spies2004,
  author = {Spies, Ashley M and Woodward, Joshua J and Watnik, Mitchell R and Toney, Michael D},
  title = {Alanine Racemase Free Energy Profiles from Global Analyses of Progress Curves},
  journal = {Journal of the American Chemical Society},
  year = {2004},
  volume = {126},
  pages = {7464 - 7475}
}
</pre></td>
</tr>
<tr id="Spirin2003" class="entry">
	<td>Spirin, V. and Mirny, L.A.</td>
	<td>Protein Complexes and Functional Modules in Molecular Networks <p class="infolinks">[<a href="javascript:toggleInfo('Spirin2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Proceedings of the National Acadamy of Sciences<br/>Vol. 100(21), pp. 12123 - 12128&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Spirin2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Spirin2003,
  author = {Spirin, Victor and Mirny, Leonid A},
  title = {Protein Complexes and Functional Modules in Molecular Networks},
  journal = {Proceedings of the National Acadamy of Sciences},
  year = {2003},
  volume = {100},
  number = {21},
  pages = {12123 - 12128}
}
</pre></td>
</tr>
<tr id="Sprik1985" class="entry">
	<td>Sprik, M., Klein, M.L. and Chandler, D.</td>
	<td>Staging: A sampling technique for the Monte Carlo evaluation of path integrals <p class="infolinks">[<a href="javascript:toggleInfo('Sprik1985','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Sprik1985','bibtex')">BibTeX</a>]</p></td>
	<td>1985</td>
	<td>Physical Review B<br/>Vol. 31(7), pp. 4234 - 4244&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1103/PhysRevB.31.4234">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Sprik1985" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We introduce a new sampling technique, called staging, which is likely to have applicability in the Monte Carlo evaluation of path integrals. In order to circumvent the probelm of high attrition rates in the isomorphic classical system, allowed configurations are constructed with the use of a staging technique. To illustrate the utility of this approach, the method is applied to a single quantum particle in rigid bcc hard-sphere lattice and in a disordered system of fixed hard spheres.</td>
</tr>
<tr id="bib_Sprik1985" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Sprik1985,
  author = {Sprik, Michiel and Klein, Michael L and Chandler, David},
  title = {Staging: A sampling technique for the Monte Carlo evaluation of path integrals},
  journal = {Physical Review B},
  year = {1985},
  volume = {31},
  number = {7},
  pages = {4234 - 4244},
  doi = {https://doi.org/10.1103/PhysRevB.31.4234}
}
</pre></td>
</tr>
<tr id="Stahelin2003" class="entry">
	<td>Stahelin, RV., Long, F., Peter, BJ. and Murray, D.</td>
	<td>Contrasting membrane interaction mechanisms of AP180 N-terminal homology (ANTH) and epsin N-terminal homology (ENTH) domains <p class="infolinks">[<a href="javascript:toggleInfo('Stahelin2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Stahelin2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Journal of Biological &hellip;&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M302865200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Stahelin2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Abstract Epsin and AP180/CALM are endocytotic accessory proteins that have been implicated in the formation of clathrin-coated pits. Both proteins have phosphatidylinositol 4, 5-bisphosphate (PtdIns (4, 5) P 2)-binding domains in their N termini, but these domains are structurally and functionally different. To understand the basis of their distinct properties, we measured the PtdIns (4, 5) P 2-dependent membrane binding of the epsin N-terminal  ...</td>
</tr>
<tr id="bib_Stahelin2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Stahelin2003,
  author = {Stahelin, RV and Long, F and Peter, BJ and Murray, D},
  title = {Contrasting membrane interaction mechanisms of AP180 N-terminal homology (ANTH) and epsin N-terminal homology (ENTH) domains},
  journal = {Journal of Biological &hellip;},
  year = {2003},
  note = {PIP2 interactions with AP180 and EPN1. Probable dissociation constant information.},
  doi = {https://doi.org/10.1074/jbc.M302865200}
}
</pre></td>
</tr>
<tr id="Stare2017" class="entry">
	<td>Stare, J.</td>
	<td>Complete sampling of an enzyme reaction pathway: a lesson from gas phase simulations <p class="infolinks">[<a href="javascript:toggleInfo('Stare2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Stare2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>RSC Adv.<br/>Vol. 7(15), pp. 8740 - 8754&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1039/C6RA27894A">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Stare2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Jernej Stare This work addresses the sampling issue commonly accompanying the simulation of chemical reactions. Very often the sampling is severely limited by complexity of the phase space, possibly leading to poorly converged or inaccurate free energy profiles. We explored the factors governing the completeness of reaction path sampling for the rate limiting step of phenylethylamine oxidation by lumiflavin in the gas phase, a reaction important for the pharmacology of the central nervous system. The simulations utilize the free energy perturbation sampling technique together with the empirical valence bond methodology for the free energy calculations. The simplicity of the system allows for the acquisition of fully converged free energy profiles, even for simulation free of restraints. The bottleneck for convergence is in the noticeably poorer sampling statistics in the transition state region, which is resolved by performing sufficiently long simulation to ensure reversibility of all processes accompanying the reaction. In the present case, convergence is attained in microseconds of simulation, but the required simulation time generally depends on the complexity of the potential energy surface pertinent to the reaction. Accordingly, the use of restraints reduces the complexity of the phase space, decreasing the required time by about an order of magnitude. In the case of elementary nucleophilic substitution with even simpler potential energy surface convergence is reached already at a timescale of few nanoseconds. For related biomolecular reactions embedded in an enzyme, significantly longer simulation times may be needed, rendering the sampling problem exceedingly difficult and representing a challenge for advanced sampling techniques. Accordingly, suggestions are given for optimal simulation of biomolecular reactions based on the presently employed techniques and under the aforementioned limitations.</td>
</tr>
<tr id="bib_Stare2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Stare2017,
  author = {Stare, Jernej},
  title = {Complete sampling of an enzyme reaction pathway: a lesson from gas phase simulations},
  journal = {RSC Adv.},
  year = {2017},
  volume = {7},
  number = {15},
  pages = {8740 - 8754},
  doi = {https://doi.org/10.1039/C6RA27894A}
}
</pre></td>
</tr>
<tr id="Stelzl2017" class="entry">
	<td>Stelzl, L.S. and Hummer, G.</td>
	<td>Kinetics from Replica Exchange Molecular Dynamics Simulations <p class="infolinks">[<a href="javascript:toggleInfo('Stelzl2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Stelzl2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of Chemical Theory and Computation, pp. acs.jctc.7b00372 - acs.jctc.7b00372&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jctc.7b00372">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Stelzl2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Transitions between metastable states govern many fundamental processes in physics, chemistry, and biology, from nucleation events in phase transitions to the folding of proteins. The free energy surfaces underlying these processes can be obtained from simulations using enhanced sampling methods. However, their altered dynamics makes kinetic and mechanistic information difficult or impossible to extract. Here we show that with replica-exchange molecular dynamics (REMD), one can not only sample equilibrium properties, but also extract kinetic information. For systems that strictly obey first-order kinetics, the procedure to extract rates is rigorous. For actual molecular systems whose long-time dynamics is captured by kinetic rate models, accurate rate coefficients can be determined from the statistics of the transitions between the metastable states at each replica temperatures. We demonstrate the practical applicability of the procedure by constructing master equation (Markov state) models of peptide and...</td>
</tr>
<tr id="bib_Stelzl2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Stelzl2017,
  author = {Stelzl, Lukas S and Hummer, Gerhard},
  title = {Kinetics from Replica Exchange Molecular Dynamics Simulations},
  journal = {Journal of Chemical Theory and Computation},
  year = {2017},
  pages = {acs.jctc.7b00372 - acs.jctc.7b00372},
  doi = {https://doi.org/10.1021/acs.jctc.7b00372}
}
</pre></td>
</tr>
<tr id="Stewart1995" class="entry">
	<td>Stewart, J.D. and Benkovic, S.J.</td>
	<td>Transition-state stabilization as a measure of the efficiency of antibody catalysis <p class="infolinks">[<a href="javascript:toggleInfo('Stewart1995','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Stewart1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>Nature<br/>Vol. 375(6530), pp. 388 - 391&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/375388a0">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Stewart1995" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: There are now about 60 examples of reactions that have been successfully catalysed by monoclonal antibodies. Not surprisingly, many of the early examples involved reactions that were already favoured kinetically (such as carbonate and ester hydrolysis). But it has since been shown that antibodies can also accelerate reaction pathways that are normally disfavoured kinetically (by at least a few kcal mol-1). Here we use transition-state theory to provide a quantitative analysis of the scope and limitations of antibody catalysis. We show that the observed rate accelerations can be predicted from the ratio of equilibrium binding constants of the reaction substrate and the transition-state analogue used to raise the antibody. This scheme allows us to rationalize the product selectivity displayed in antibody catalysis of disfavoured reactions, to predict the degree of rate acceleration that catalytic antibodies may ultimately afford, and to highlight some differences between the way that they and enzymes catalyse reactions.</td>
</tr>
<tr id="bib_Stewart1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Stewart1995,
  author = {Stewart, Jon D and Benkovic, Stephen J},
  title = {Transition-state stabilization as a measure of the efficiency of antibody catalysis},
  journal = {Nature},
  year = {1995},
  volume = {375},
  number = {6530},
  pages = {388 - 391},
  doi = {https://doi.org/10.1038/375388a0}
}
</pre></td>
</tr>
<tr id="Stewart1989" class="entry">
	<td>Stewart, J.JP.</td>
	<td>Optimization of parameters for semiempirical methods II. Applications <p class="infolinks">[<a href="javascript:toggleInfo('Stewart1989','bibtex')">BibTeX</a>]</p></td>
	<td>1989</td>
	<td>Journal of Computational Chemistry<br/>Vol. 10(2), pp. 221 - 264&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/jcc.540100209">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Stewart1989" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Stewart1989,
  author = {Stewart, James JP},
  title = {Optimization of parameters for semiempirical methods II. Applications},
  journal = {Journal of Computational Chemistry},
  year = {1989},
  volume = {10},
  number = {2},
  pages = {221 - 264},
  note = {PM3 paper 2},
  doi = {https://doi.org/10.1002/jcc.540100209}
}
</pre></td>
</tr>
<tr id="Stewart1989a" class="entry">
	<td>Stewart, J.JP.</td>
	<td>Optimization of parameters for semiempirical methods I. Method <p class="infolinks">[<a href="javascript:toggleInfo('Stewart1989a','bibtex')">BibTeX</a>]</p></td>
	<td>1989</td>
	<td>Journal of Computational Chemistry<br/>Vol. 10(2), pp. 209 - 220&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/jcc.540100208">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Stewart1989a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Stewart1989a,
  author = {Stewart, James JP},
  title = {Optimization of parameters for semiempirical methods I. Method},
  journal = {Journal of Computational Chemistry},
  year = {1989},
  volume = {10},
  number = {2},
  pages = {209 - 220},
  note = {PM3 paper 1},
  doi = {https://doi.org/10.1002/jcc.540100208}
}
</pre></td>
</tr>
<tr id="Stirnemann2015" class="entry">
	<td>Stirnemann, G. and Sterpone, F.</td>
	<td>Recovering Protein Thermal Stability Using All-Atom Hamiltonian Replica-Exchange Simulations in Explicit Solvent <p class="infolinks">[<a href="javascript:toggleInfo('Stirnemann2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Journal of Chemical Theory and Computation<br/>Vol. 11(12), pp. acs.jctc.5b00954 - acs.jctc.5b00954&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jctc.5b00954">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Stirnemann2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Stirnemann2015,
  author = {Stirnemann, Guillaume and Sterpone, Fabio},
  title = {Recovering Protein Thermal Stability Using All-Atom Hamiltonian Replica-Exchange Simulations in Explicit Solvent},
  journal = {Journal of Chemical Theory and Computation},
  year = {2015},
  volume = {11},
  number = {12},
  pages = {acs.jctc.5b00954 - acs.jctc.5b00954},
  doi = {https://doi.org/10.1021/acs.jctc.5b00954}
}
</pre></td>
</tr>
<tr id="Strych2001" class="entry">
	<td>Strych, U.</td>
	<td>Characterization of the alanine racemases from two Mycobacteria <p class="infolinks">[<a href="javascript:toggleInfo('Strych2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Strych2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>FEMS Microbiology Letters<br/>Vol. 196(2), pp. 93 - 98&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S0378-1097(01)00045-3">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Strych2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: D-Alanine is a necessary precursor in the biosynthesis of the bacterial peptidoglycan. The naturally occurring L-alanine isomer is racemized to its D-form through the action of a class of enzymes called alanine racemases. These enzymes are ubiquitous among prokaryotes, and with very few exceptions are absent in eukaryotes, making them a logical target for the development of novel antibiotics. The alanine racemase gene from both Mycobacterium tuberculosis and M. avium was amplified by PCR and cloned in Escherichia coli. Overexpression of the proteins in the E. coli BL21 system, both as native and as His-tagged recombinant products, has been achieved. The proteins have been purified to electrophoretic homogeneity and analyzed biochemically. A D-alanine requiring double knock-out mutant of E. coli (alr, dadX) was constructed and the cloned genes were able to complement its deficiencies.</td>
</tr>
<tr id="bib_Strych2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Strych2001,
  author = {Strych, U},
  title = {Characterization of the alanine racemases from two Mycobacteria},
  journal = {FEMS Microbiology Letters},
  year = {2001},
  volume = {196},
  number = {2},
  pages = {93 - 98},
  note = {Vmax of AlaR(Mtb)},
  doi = {https://doi.org/10.1016/S0378-1097(01)00045-3}
}
</pre></td>
</tr>
<tr id="Strych2002" class="entry">
	<td>Strych, U. and Benedik, MJ.</td>
	<td>Mutant Analysis Shows that Alanine Racemases from Pseudomonas aeruginosa and Escherichia coli Are Dimeric <p class="infolinks">[<a href="javascript:toggleInfo('Strych2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Strych2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Journal of Bacteriology<br/>Vol. 184(15), pp. 4321 - 4325&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1128/JB.184.15.4321-4325.2002">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Strych2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Alanine racemases are ubiquitous prokaryotic enzymes providing the essential peptidoglycan precursor D-alanine. We present evidence that the enzymes from Pseudomonas aeruginosa and Escherichia coli function exclusively as homodimers. Moreover, we demonstrate that expression of a K35A Y235A double mutation of dadX in E. coli suppresses bacterial growth in a dominant negative fashion.</td>
</tr>
<tr id="bib_Strych2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Strych2002,
  author = {Strych, U and Benedik, MJ},
  title = {Mutant Analysis Shows that Alanine Racemases from Pseudomonas aeruginosa and Escherichia coli Are Dimeric},
  journal = {Journal of Bacteriology},
  year = {2002},
  volume = {184},
  number = {15},
  pages = {4321 - 4325},
  note = {Only homodimeric AlaR is catalytically active.},
  doi = {https://doi.org/10.1128/JB.184.15.4321-4325.2002}
}
</pre></td>
</tr>
<tr id="Suarez2015" class="entry">
	<td>Suarez, J. and Schramm, V.L.</td>
	<td>Isotope-specific and amino acid-specific heavy atom substitutions alter barrier crossing in human purine nucleoside phosphorylase <p class="infolinks">[<a href="javascript:toggleInfo('Suarez2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Suarez2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Proceedings of the National Academy of Sciences<br/>Vol. 112(36), pp. 11247 - 11251&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.1513956112">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Suarez2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Computational chemistry predicts that atomic motions on the femtosecond timescale are coupled to transition-state formation (barrier-crossing) in human purine nucleoside phosphorylase (PNP). The prediction is experimentally supported by slowed catalytic site chemistry in isotopically labeled PNP (13C, 15N, and 2H). However, other explanations are possible, including altered volume or bond polarization from carbon-deuterium bonds or propagation of the femtosecond bond motions into slower (nanoseconds to milliseconds) motions of the larger protein architecture to alter catalytic site chemistry. We address these possibilities by analysis of chemistry rates in isotope-specific labeled PNPs. Catalytic site chemistry was slowed for both [2H]PNP and [13C, 15N]PNP in proportion to their altered protein masses. Secondary effects emanating from carbon--deuterium bond properties can therefore be eliminated. Heavy-enzyme mass effects were probed for local or global contributions to catalytic site chemistry by generating [15N, 2H]His8-PNP. Of the eight His per subunit, three participate in contacts to the bound reactants and five are remote from the catalytic sites. [15N, 2H]His8-PNP had reduced catalytic site chemistry larger than proportional to the enzymatic mass difference. Altered barrier crossing when only His are heavy supports local catalytic site femtosecond perturbations coupled to transition-state formation. Isotope-specific and amino acid specific labels extend the use of heavy enzyme methods to distinguish global from local isotope effects.</td>
</tr>
<tr id="bib_Suarez2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Suarez2015,
  author = {Suarez, Javier and Schramm, Vern L},
  title = {Isotope-specific and amino acid-specific heavy atom substitutions alter barrier crossing in human purine nucleoside phosphorylase},
  journal = {Proceedings of the National Academy of Sciences},
  year = {2015},
  volume = {112},
  number = {36},
  pages = {11247 - 11251},
  doi = {https://doi.org/10.1073/pnas.1513956112}
}
</pre></td>
</tr>
<tr id="Sugiyama2005" class="entry">
	<td>Sugiyama, S., Kishida, S., Chayama, K., Koyama, S. and Kikuchi, A.</td>
	<td>Ubiquitin-interacting motifs of epsin are involved in the regulation of insulin-dependent endocytosis <p class="infolinks">[<a href="javascript:toggleInfo('Sugiyama2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Sugiyama2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Journal of Biochemistry<br/>Vol. 137(3), pp. 355 - 364&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/jb/mvi044">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Sugiyama2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Epsin is a key molecule in receptor-mediated endocytosis. Epsin is phosphorylated and ubiquitinated, and these post-translational modifications are necessary for the regulation of endocytosis. Since human Epsin (hEpsin) has two ubiquitin-interacting motifs (UIMs), we investigated the roles of these UIMs in endocytosis. hEpsin formed a complex with ubiquitinated proteins but did not bind to monoubiquitin. Neither of the two UIMs of hEpsin alone was sufficient to form a complex with ubiquitinated proteins: both UIMs were necessary. Mutations of Asp209 and Asp210 to Ala in UIM (hEpsinDA) abolished the binding activity of hEpsin to ubiquitinated proteins. However, hEpsinDA interacted with Eps15, POB1, and AP-2, which are involved in receptor-mediated endocytosis, as efficiently as wild-type hEpsin. Expression of hEpsinDA in CHO-IR cells affected neither the binding of insulin to nor insulin-dependent autophosphorylation of its receptor. Expression of wild-type hEpsin inhibited the internalization of insulin, whereas that of hEpsinDA did not. These results suggest that the UIM motifs of hEpsin interact with proteins modified with ubiquitin, and that the complex formation is involved in insulin-dependent receptor endocytosis.</td>
</tr>
<tr id="bib_Sugiyama2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Sugiyama2005,
  author = {Sugiyama, Shinichiro and Kishida, Shosei and Chayama, Kazuaki and Koyama, Shinya and Kikuchi, Akira},
  title = {Ubiquitin-interacting motifs of epsin are involved in the regulation of insulin-dependent endocytosis},
  journal = {Journal of Biochemistry},
  year = {2005},
  volume = {137},
  number = {3},
  pages = {355 - 364},
  note = {hamster ovary, but expressing human EPN1. information on ubiquitination of EPN1. EPN1 - AP2A2 (added), EPS15},
  doi = {https://doi.org/10.1093/jb/mvi044}
}
</pre></td>
</tr>
<tr id="Sundborger2011" class="entry">
	<td>Sundborger, A., Soderblom, C., Vorontsova, O., Evergren, E., Hinshaw, J.E. and Shupliakov, O.</td>
	<td>An endophilin-dynamin complex promotes budding of clathrin-coated vesicles during synaptic vesicle recycling. <p class="infolinks">[<a href="javascript:toggleInfo('Sundborger2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Sundborger2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Journal of cell science<br/>Vol. 124, pp. 133-143&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1242/jcs.072686">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Sundborger2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated vesicle recycling in synapses is maintained by a unique set of endocytic proteins and interactions. We show that endophilin localizes in the vesicle pool at rest and in spirals at the necks of clathrin-coated pits (CCPs) during activity in lamprey synapses. Endophilin and dynamin colocalize at the base of the clathrin coat. Protein spirals composed of these proteins on lipid tubes in vitro have a pitch similar to the one observed at necks of CCPs in living synapses, and lipid tubules are thinner than those formed by dynamin alone. Tubulation efficiency and the amount of dynamin recruited to lipid tubes are dramatically increased in the presence of endophilin. Blocking the interactions of the endophilin SH3 domain in situ reduces dynamin accumulation at the neck and prevents the formation of elongated necks observed in the presence of GTPγS. Therefore, endophilin recruits dynamin to a restricted part of the CCP neck, forming a complex, which promotes budding of new synaptic vesicles.</td>
</tr>
<tr id="bib_Sundborger2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Sundborger2011,
  author = {Sundborger, Anna and Soderblom, Cynthia and Vorontsova, Olga and Evergren, Emma and Hinshaw, Jenny E and Shupliakov, Oleg},
  title = {An endophilin-dynamin complex promotes budding of clathrin-coated vesicles during synaptic vesicle recycling.},
  journal = {Journal of cell science},
  year = {2011},
  volume = {124},
  pages = {133--143},
  doi = {https://doi.org/10.1242/jcs.072686}
}
</pre></td>
</tr>
<tr id="Swain1958" class="entry">
	<td>Swain, G.C., Stivers, E.C., Jr., J.F. and Schaad, L.J.</td>
	<td>Use of hydrogen isotope effects to identify the attacking nucleophile in the enolization of ketones catalyzed by acetic acid <p class="infolinks">[<a href="javascript:toggleInfo('Swain1958','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Swain1958','bibtex')">BibTeX</a>]</p></td>
	<td>1958</td>
	<td>Journal of the American Chemical Society<br/>Vol. 80(1930), pp. 5885 - 5893&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ja01554a077">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Swain1958" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: cf. C.A. 45, 4647a. That catalysis of enolization by HOAc is due to an H+ acting on the ketonic O and AcO- removing the proton from C in the case of α-phenylisocaprophenone (a ketone with one α-H) in aq. soln. with 8.4M dioxane at 100textdegree is detd. by comparing the H-T isotope effect for the system (11.4) with systems catalyzed by H+, AcO-, and OH-. The magnitude of kH/kT increases as the basicity and reactivity of the nucleophile increases. A relation between H-T and H-D isotope effects is derived and compared with exptl. data. [on SciFinder(R)]</td>
</tr>
<tr id="bib_Swain1958" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Swain1958,
  author = {Swain, Gardner C and Stivers, Edward C and Jr., Joseph F and Schaad, Lawrence J},
  title = {Use of hydrogen isotope effects to identify the attacking nucleophile in the enolization of ketones catalyzed by acetic acid},
  journal = {Journal of the American Chemical Society},
  year = {1958},
  volume = {80},
  number = {1930},
  pages = {5885 - 5893},
  doi = {https://doi.org/10.1021/ja01554a077}
}
</pre></td>
</tr>
<tr id="Swiderek2015" class="entry">
	<td>&#346;widerek, K., Tu&ntilde;&oacute;n, I., Mart\i, S. and Moliner, V.</td>
	<td>Protein Conformational Landscapes and Catalysis. Influence of Active Site Conformations in the Reaction Catalyzed by L-Lactate Dehydrogenase <p class="infolinks">[<a href="javascript:toggleInfo('Swiderek2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Swiderek2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>ACS Catalysis<br/>Vol. 5(2), pp. 1172 - 1185&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/cs501704f">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Swiderek2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In the past decade, L-Lactate Dehydrogenase (LDH) has become an extremely useful marker in both clinical diagnosis and in monitoring the course of many human diseases. It has been assumed since the 1980s that the full catalytic process of LDH starts with the binding of the cofactor and the substrate followed by the enclosure of the active site by a mobile loop of the protein before the reaction takes place. In this paper, we show that the chemical step of the LDH-catalyzed reaction can proceed within the open loop conformation, and the different reactivity of the different protein conformations would be in agreement with the broad range of rate constants measured in single-molecule spectrometry studies. Starting from a recently solved X-ray diffraction structure that presented an open loop conformation in two of the four chains of the tetramer, QM/MM free energy surfaces have been obtained at different levels of theory. Depending on the level of theory used to describe the electronic structure, the free e...</td>
</tr>
<tr id="bib_Swiderek2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Swiderek2015,
  author = {&#346;widerek, Katarzyna and Tu&ntilde;&oacute;n, I&ntilde;aki and Mart\i, Sergio and Moliner, Vicent},
  title = {Protein Conformational Landscapes and Catalysis. Influence of Active Site Conformations in the Reaction Catalyzed by L-Lactate Dehydrogenase},
  journal = {ACS Catalysis},
  year = {2015},
  volume = {5},
  number = {2},
  pages = {1172 - 1185},
  note = {NULL},
  doi = {https://doi.org/10.1021/cs501704f}
}
</pre></td>
</tr>
<tr id="Swope1982" class="entry">
	<td>Swope, W.C.</td>
	<td>A computer simulation method for the calculation of equilibrium constants for the formation of physical clusters of molecules: Application to small water clusters <p class="infolinks">[<a href="javascript:toggleInfo('Swope1982','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Swope1982','bibtex')">BibTeX</a>]</p></td>
	<td>1982</td>
	<td>J. Chem. Phys.<br/>Vol. 76(1), pp. 637 - 637&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.442716">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Swope1982" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present a molecular dynamics computer simulation method for calculating equilibrium constants for the formation of physical clusters of molecules. The method is based on Hill's formal theory of physical clusters. In the method, a molecular dynamics calculation is used to calculate the average potential energy of a cluster of molecules as a function of temperature, and the equilibrium constants are calculated from the integral of the energy with respect to reciprocal temperature. The method is illustrated by calculations of the equilibrium constants for the formation of clusters of two to five water molecules that interact with each other by an intermolecular potential devised by Watts. The method is compared with other procedures for calculating the thermodynamic properties of clusters.</td>
</tr>
<tr id="bib_Swope1982" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Swope1982,
  author = {Swope, William C},
  title = {A computer simulation method for the calculation of equilibrium constants for the formation of physical clusters of molecules: Application to small water clusters},
  journal = {J. Chem. Phys.},
  year = {1982},
  volume = {76},
  number = {1},
  pages = {637 - 637},
  doi = {https://doi.org/10.1063/1.442716}
}
</pre></td>
</tr>
<tr id="Takahashi2010" class="entry">
	<td>Takahashi, K., Sorin, T. and Rein, P.</td>
	<td>Spatio-temporal correlations can drastically change the response of a MAPK pathway <p class="infolinks">[<a href="javascript:toggleInfo('Takahashi2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Takahashi2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Proceedings of the National Acadamy of Sciences<br/>Vol. 107(6), pp. 2473 - 2478&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.0906885107">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Takahashi2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Multisite covalent modification of proteins is omnipresent in eukaryotic cells. A well-known example is the mitogen-activated protein kinase (MAPK) cascade where, in each layer of the cascade, a protein is phosphorylated at two sites. It has long been known that the response of a MAPK pathway strongly depends on whether the enzymes that modify the protein act processively or distributively. A distributive mechanism, in which the enzyme molecules have to release the substrate molecules in between the modification of the two sites, can generate an ultrasensitive response and lead to hysteresis and bistability. We study by Green's Function Reaction Dynamics (GFRD), a stochastic scheme that makes it possible to simulate biochemical networks at the particle level in time and space, a dual phosphorylation cycle in which the enzymes act according to a distributive mechanism. We find that the response of this network can differ dramatically from that predicted by a mean-field analysis based on the chemical rate equations. In particular, rapid rebindings of the enzyme molecules to the substrate molecules after modification of the first site can markedly speed up the response and lead to loss of ultrasensitivity and bistability. In essence, rapid enzyme-substrate rebindings can turn a distributive mechanism into a processive mechanism. We argue that slow ADP release by the enzymes can protect the system against these rapid rebindings, thus enabling ultrasensitivity and bistability.</td>
</tr>
<tr id="bib_Takahashi2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Takahashi2010,
  author = {Takahashi, Koichi and Sorin, T and Rein, Pieter},
  title = {Spatio-temporal correlations can drastically change the response of a MAPK pathway},
  journal = {Proceedings of the National Acadamy of Sciences},
  year = {2010},
  volume = {107},
  number = {6},
  pages = {2473 - 2478},
  doi = {https://doi.org/10.1073/pnas.0906885107}
}
</pre></td>
</tr>
<tr id="Takei1999" class="entry">
	<td>Takei, K., Slepnev, V.I., Haucke, V. and De Camilli, P.</td>
	<td>Functional partnership between amphiphysin and dynamin in clathrin-mediated endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Takei1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Takei1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Nature cell biology<br/>Vol. 1, pp. 33-39&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/9004">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Takei1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Amphiphysin, a protein that is highly concentrated in nerve terminals, has been proposed to function as a linker between the clathrin coat and dynamin in the endocytosis of synaptic vesicles. Here, using a cell-free system, we provide direct morphological evidence in support of this hypothesis. Unexpectedly, we also find that amphiphysin-1, like dynamin-1, can transform spherical liposomes into narrow tubules. Moreover, amphiphysin-1 assembles with dynamin-1 into ring-like structures around the tubules and enhances the liposome-fragmenting activity of dynamin-1 in the presence of GTP. These results show that amphiphysin binds lipid bilayers, indicate a potential function for amphiphysin in the changes in bilayer curvature that accompany vesicle budding, and imply a close functional partnership between amphiphysin and dynamin in endocytosis.</td>
</tr>
<tr id="bib_Takei1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Takei1999,
  author = {Takei, K and Slepnev, V I and Haucke, V and De Camilli, P},
  title = {Functional partnership between amphiphysin and dynamin in clathrin-mediated endocytosis.},
  journal = {Nature cell biology},
  year = {1999},
  volume = {1},
  pages = {33--39},
  note = {dynamin and amphiphysin copolymerize to create thicker rings than they would alone.},
  doi = {https://doi.org/10.1038/9004}
}
</pre></td>
</tr>
<tr id="Taniguchi2011" class="entry">
	<td>Taniguchi, Y., Choi, P.J., Li, G.-w., Chen, H., Babu, M., Hearn, J., Emili, A. and Xie, S.X.</td>
	<td>Quantifying E. coli Proteome and Transcriptome with Single-Molecule Sensitivity in Single Cells <p class="infolinks">[<a href="javascript:toggleInfo('Taniguchi2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Taniguchi2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Science<br/>Vol. 329(5991), pp. 533 - 539&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1126/science.1188308">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Taniguchi2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Protein and messenger RNA (mRNA) copy numbers vary from cell to cell in isogenic bacterial populations. However, these molecules often exist in low copy numbers and are difficult to detect in single cells. We carried out quantitative system-wide analyses of protein and mRNA expression in individual cells with single-molecule sensitivity using a newly constructed yellow fluorescent protein fusion library for Escherichia coli. We found that almost all protein number distributions can be described by the gamma distribution with two fitting parameters which, at low expression levels, have clear physical interpretations as the transcription rate and protein burst size. At high expression levels, the distributions are dominated by extrinsic noise. We found that a single cell's protein and mRNA copy numbers for any given gene are uncorrelated.</td>
</tr>
<tr id="bib_Taniguchi2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Taniguchi2011,
  author = {Taniguchi, Yuichi and Choi, Paul J and Li, Gene-wei and Chen, Huiyi and Babu, Mohan and Hearn, Jeremy and Emili, Andrew and Xie, Sunney X},
  title = {Quantifying E. coli Proteome and Transcriptome with Single-Molecule Sensitivity in Single Cells},
  journal = {Science},
  year = {2011},
  volume = {329},
  number = {5991},
  pages = {533 - 539},
  doi = {https://doi.org/10.1126/science.1188308}
}
</pre></td>
</tr>
<tr id="Taylor2011" class="entry">
	<td>Taylor, M.J., Perrais, D. and Merrifield, C.J.</td>
	<td>A high precision survey of the molecular dynamics of mammalian clathrin-mediated endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Taylor2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Taylor2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>PLoS biology<br/>Vol. 9, pp. e1000604&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pbio.1000604">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Taylor2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Dual colour total internal reflection fluorescence microscopy is a powerful tool for decoding the molecular dynamics of clathrin-mediated endocytosis (CME). Typically, the recruitment of a fluorescent protein-tagged endocytic protein was referenced to the disappearance of spot-like clathrin-coated structure (CCS), but the precision of spot-like CCS disappearance as a marker for canonical CME remained unknown. Here we have used an imaging assay based on total internal reflection fluorescence microscopy to detect scission events with a resolution of ∼ 2 s. We found that scission events engulfed comparable amounts of transferrin receptor cargo at CCSs of different sizes and CCS did not always disappear following scission. We measured the recruitment dynamics of 34 types of endocytic protein to scission events: Abp1, ACK1, amphiphysin1, APPL1, Arp3, BIN1, CALM, CIP4, clathrin light chain (Clc), cofilin, coronin1B, cortactin, dynamin1/2, endophilin2, Eps15, Eps8, epsin2, FBP17, FCHo1/2, GAK, Hip1R, lifeAct, mu2 subunit of the AP2 complex, myosin1E, myosin6, NECAP, N-WASP, OCRL1, Rab5, SNX9, synaptojanin2β1, and syndapin2. For each protein we aligned ∼ 1,000 recruitment profiles to their respective scission events and constructed characteristic "recruitment signatures" that were grouped, as for yeast, to reveal the modular organization of mammalian CME. A detailed analysis revealed the unanticipated recruitment dynamics of SNX9, FBP17, and CIP4 and showed that the same set of proteins was recruited, in the same order, to scission events at CCSs of different sizes and lifetimes. Collectively these data reveal the fine-grained temporal structure of CME and suggest a simplified canonical model of mammalian CME in which the same core mechanism of CME, involving actin, operates at CCSs of diverse sizes and lifetimes.</td>
</tr>
<tr id="bib_Taylor2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Taylor2011,
  author = {Taylor, Marcus J and Perrais, David and Merrifield, Christien J},
  title = {A high precision survey of the molecular dynamics of mammalian clathrin-mediated endocytosis.},
  journal = {PLoS biology},
  year = {2011},
  volume = {9},
  pages = {e1000604},
  doi = {https://doi.org/10.1371/journal.pbio.1000604}
}
</pre></td>
</tr>
<tr id="Tebar1999" class="entry">
	<td>Tebar, F., Bohlander, SK. and Sorkin, A.</td>
	<td>Clathrin assembly lymphoid myeloid leukemia (CALM) protein: localization in endocytic-coated pits, interactions with clathrin, and the impact of overexpression on clathrin-mediated traffic. <p class="infolinks">[<a href="javascript:toggleInfo('Tebar1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Tebar1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Molecular biology of the cell<br/>Vol. 10(8), pp. 2687 - 702&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1091/mbc.10.8.2687">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Tebar1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The clathrin assembly lymphoid myeloid leukemia (CALM) gene encodes a putative homologue of the clathrin assembly synaptic protein AP180. Hence the biochemical properties, the subcellular localization, and the role in endocytosis of a CALM protein were studied. In vitro binding and coimmunoprecipitation demonstrated that the clathrin heavy chain is the major binding partner of CALM. The bulk of cellular CALM was associated with the membrane fractions of the cell and localized to clathrin-coated areas of the plasma membrane. In the membrane fraction, CALM was present at near stoichiometric amounts relative to clathrin. To perform structure-function analysis of CALM, we engineered chimeric fusion proteins of CALM and its fragments with the green fluorescent protein (GFP). GFP-CALM was targeted to the plasma membrane-coated pits and also found colocalized with clathrin in the Golgi area. High levels of expression of GFP-CALM or its fragments with clathrin-binding activity inhibited the endocytosis of transferrin and epidermal growth factor receptors and altered the steady-state distribution of the mannose-6-phosphate receptor in the cell. In addition, GFP-CALM overexpression caused the loss of clathrin accumulation in the trans-Golgi network area, whereas the localization of the clathrin adaptor protein complex 1 in the trans-Golgi network remained unaffected. The ability of the GFP-tagged fragments of CALM to affect clathrin-mediated processes correlated with the targeting of the fragments to clathrin-coated areas and their clathrin-binding capacities. Clathrin-CALM interaction seems to be regulated by multiple contact interfaces. The C-terminal part of CALM binds clathrin heavy chain, although the full-length protein exhibited maximal ability for interaction. Altogether, the data suggest that CALM is an important component of coated pit internalization machinery, possibly involved in the regulation of clathrin recruitment to the membrane and/or the formation of the coated pit.</td>
</tr>
<tr id="bib_Tebar1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Tebar1999,
  author = {Tebar, F and Bohlander, SK and Sorkin, A},
  title = {Clathrin assembly lymphoid myeloid leukemia (CALM) protein: localization in endocytic-coated pits, interactions with clathrin, and the impact of overexpression on clathrin-mediated traffic.},
  journal = {Molecular biology of the cell},
  year = {1999},
  volume = {10},
  number = {8},
  pages = {2687 - 702},
  note = {HeLa cells. CLTC - CALM},
  doi = {https://doi.org/10.1091/mbc.10.8.2687}
}
</pre></td>
</tr>
<tr id="Tebar1997" class="entry">
	<td>Tebar, F., Confalonieri, S., Carter, R.E., Di Fiore, P.P. and Sorkin, A.</td>
	<td>Eps15 is constitutively oligomerized due to homophilic interaction of its coiled-coil region. <p class="infolinks">[<a href="javascript:toggleInfo('Tebar1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Tebar1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>The Journal of Biological Chemistry<br/>Vol. 272, pp. 15413-15418&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Tebar1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Eps15 is a member of an emerging family of proteins containing a novel protein/protein interaction domain, the EH domain, of as yet unknown function. Recent findings of Eps15 association with clathrin adaptor complex AP-2 and its localization in clathrin-coated pits have implicated Eps15 in the regulation of vesicle trafficking. Here we show that Eps15 exists in several multimeric states in vivo. When purified recombinant Eps15 or lysates of NIH 3T3 cells were treated with cross-linking reagents, covalent dimers of Eps15 and larger covalent multimers were detected in high yield. Large Eps15 oligomers co-immunoprecipitated with AP-2 at an efficiency higher than that of Eps15 dimers. Furthermore, cross-linking of the membrane-bound fraction of Eps15 in mildly permeabilized cells was as efficient as that of the cytosolic fraction. Size-exclusion column chromatography of recombinantly produced Eps15 and of total cell lysates was performed to examine the equilibrium ratio of the monomers versus the aggregated forms of Eps15. These experiments showed that essentially all the Eps15 was aggregated, whereas monomers of Eps15 could be obtained only under strong denaturing conditions. To map the region of Eps15 responsible for dimerization, fusion proteins corresponding to the three structural domains of Eps15 were prepared. Cross-linking analysis revealed that the central portion of Eps15, which possesses a coiled-coil region (residues 321-520), serves as the interacting interface. The possibility that hetero-oligomeric complexes of Eps15 dimers and AP-2 function during the recruitment of proteins into coated pits is discussed.</td>
</tr>
<tr id="bib_Tebar1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Tebar1997,
  author = {Tebar, F and Confalonieri, S and Carter, R E and Di Fiore, P P and Sorkin, A},
  title = {Eps15 is constitutively oligomerized due to homophilic interaction of its coiled-coil region.},
  journal = {The Journal of Biological Chemistry},
  year = {1997},
  volume = {272},
  pages = {15413--15418},
  note = {Mouse 3T3 cells expressing human eps15. oligomerization of EPS15. domain information.}
}
</pre></td>
</tr>
<tr id="Thalappilly2008" class="entry">
	<td>Thalappilly, S., Suliman, M., Gayet, O., Soubeyran, P., Hermant, A., Lecine, P., Iovanna, J.L. and Dusetti, N.J.</td>
	<td>Identification of multi-SH3 domain-containing protein interactome in pancreatic cancer: A yeast two-hybrid approach <p class="infolinks">[<a href="javascript:toggleInfo('Thalappilly2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Thalappilly2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Proteomics<br/>Vol. 8(15), pp. 3071 - 3081&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/pmic.200701157">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Thalappilly2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Pancreatic ductal adenocarcinoma (PDAC) is a fatal disease that shows minimal response to chemotherapy. Genetic changes involved in the progression of PDAC concern genes that encode proteins related to signal transduction networks. This fact reveals the importance in identifying the role and the relations between multiple signaling cascades in PDAC. One of the major factors that modulate signaling events is multidomain scaffold proteins that function by binding several proteins simultaneously, inducing their proximity and influencing the outcome of signaling. A particular group among them, containing multiple Src homology 3 (SH3) domains that can bind proteins containing proline-rich motifs, was associated to different aspects of cancer cell homeostasis. In this work, using a microarray-based analysis, we have shown that 13 multiple SH3 domain containing scaffold proteins are expressed in PDAC cells. Using a yeast two-hybrid approach, we have identified proteins that interact with these adaptor proteins. Among them we have found several molecules that modulate cell proliferation and survival (CIZ1, BIRC6, RBBP6), signaling (LTBP4, Notch2, TOM1L1, STK24) and membrane dynamics (PLSCR1, DDEF2, VCP). Our results indicate that interactions mediated by multi-SH3 domain-containing proteins could lead to the formation of dynamic protein complexes that function in pancreatic cancer cell signaling. The identification of such protein complexes is of paramount importance in deciphering pancreatic cancer biology and designing novel therapeutic approaches.</td>
</tr>
<tr id="bib_Thalappilly2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Thalappilly2008,
  author = {Thalappilly, Subhash and Suliman, Muhtadi and Gayet, Odile and Soubeyran, Philippe and Hermant, Aur&eacute;lie and Lecine, Patrick and Iovanna, Juan L and Dusetti, Nelson J},
  title = {Identification of multi-SH3 domain-containing protein interactome in pancreatic cancer: A yeast two-hybrid approach},
  journal = {Proteomics},
  year = {2008},
  volume = {8},
  number = {15},
  pages = {3071 - 3081},
  note = {pancreatic cancer cells (multiple lines, human). ITSN1 - AP2B1. some domain information},
  doi = {https://doi.org/10.1002/pmic.200701157}
}
</pre></td>
</tr>
<tr id="Theobald2012" class="entry">
	<td>Theobald, D.L. and Steindel, P.A.</td>
	<td>Optimal simultaneous superpositioning of multiple structures with missing data <p class="infolinks">[<a href="javascript:toggleInfo('Theobald2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Theobald2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Bioinformatics<br/>Vol. 28(15), pp. 1972 - 1979&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/bioinformatics/bts243">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Theobald2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: MOTIVATION: Superpositioning is an essential technique in structural biology that facilitates the comparison and analysis of conformational differences among topologically similar structures. Performing a superposition requires a one-to-one correspondence, or alignment, of the point sets in the different structures. However, in practice, some points are usually 'missing' from several structures, for example, when the alignment contains gaps. Current superposition methods deal with missing data simply by superpositioning a subset of points that are shared among all the structures. This practice is inefficient, as it ignores important data, and it fails to satisfy the common least-squares criterion. In the extreme, disregarding missing positions prohibits the calculation of a superposition altogether. RESULTS: Here, we present a general solution for determining an optimal superposition when some of the data are missing. We use the expectation-maximization algorithm, a classic statistical technique for dealing with incomplete data, to find both maximum-likelihood solutions and the optimal least-squares solution as a special case. AVAILABILITY AND IMPLEMENTATION: The methods presented here are implemented in THESEUS 2.0, a program for superpositioning macromolecular structures. ANSI C source code and selected compiled binaries for various computing platforms are freely available under the GNU open source license from http://www.theseus3d.org. CONTACT: dtheobald@brandeis.edu SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.</td>
</tr>
<tr id="bib_Theobald2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Theobald2012,
  author = {Theobald, Douglas L and Steindel, Phillip A},
  title = {Optimal simultaneous superpositioning of multiple structures with missing data},
  journal = {Bioinformatics},
  year = {2012},
  volume = {28},
  number = {15},
  pages = {1972 - 1979},
  doi = {https://doi.org/10.1093/bioinformatics/bts243}
}
</pre></td>
</tr>
<tr id="Theobald2008" class="entry">
	<td>Theobald, D.L. and Wuttke, D.S.</td>
	<td>Accurate structural correlations from maximum likelihood superpositions <p class="infolinks">[<a href="javascript:toggleInfo('Theobald2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Theobald2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>PLoS Computational Biology<br/>Vol. 4(2), pp. e43 - e43&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pcbi.0040043">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Theobald2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The cores of globular proteins are densely packed, resulting in complicated networks of structural interactions. These interactions in turn give rise to dynamic structural correlations over a wide range of time scales. Accurate analysis of these complex correlations is crucial for understanding biomolecular mechanisms and for relating structure to function. Here we report a highly accurate technique for inferring the major modes of structural correlation in macromolecules using likelihood-based statistical analysis of sets of structures. This method is generally applicable to any ensemble of related molecules, including families of nuclear magnetic resonance (NMR) models, different crystal forms of a protein, and structural alignments of homologous proteins, as well as molecular dynamics trajectories. Dominant modes of structural correlation are determined using principal components analysis (PCA) of the maximum likelihood estimate of the correlation matrix. The correlations we identify are inherently independent of the statistical uncertainty and dynamic heterogeneity associated with the structural coordinates. We additionally present an easily interpretable method ("PCA plots") for displaying these positional correlations by color-coding them onto a macromolecular structure. Maximum likelihood PCA of structural superpositions, and the structural PCA plots that illustrate the results, will facilitate the accurate determination of dynamic structural correlations analyzed in diverse fields of structural biology.</td>
</tr>
<tr id="bib_Theobald2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Theobald2008,
  author = {Theobald, Douglas L and Wuttke, Deborah S},
  title = {Accurate structural correlations from maximum likelihood superpositions},
  journal = {PLoS Computational Biology},
  year = {2008},
  volume = {4},
  number = {2},
  pages = {e43 - e43},
  doi = {https://doi.org/10.1371/journal.pcbi.0040043}
}
</pre></td>
</tr>
<tr id="Thery1994" class="entry">
	<td>Thery, V., Rinaldi, D., Rivail, Jean-Louis., Maigret, B. and Ferenczy, G.G.</td>
	<td>Quantum mechanical computations on very large molecular systems: The local self-consistent field method <p class="infolinks">[<a href="javascript:toggleInfo('Thery1994','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Thery1994','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td>Journal of Computational Chemistry<br/>Vol. 15(3), pp. 269 - 282&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/jcc.540150303">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Thery1994" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Quantum chemical computations on a subset of a large molecule can be performed, at the neglect of diatomic differential overlap (NDDO) level, without further approximation provided that the atomic orbitals of the frontier atoms are replaced by parametrized orthogonal hybrid orbitals. The electrostatic interaction with the rest of the molecule, treated classically by the usual molecular mechanical approximations, is included into the self-consistent field (SCF) equations. The first and second derivatives of energy are obtained analytically, allowing the search for energy minima and transition states as well as the resolution of Newton equations in molecular dynamics simulations. The local self-consistent field (LSCF) method based on these approximations is tested by studying the intramolecular proton transfer in a Gly-Arg-Glu-Gly model tetrapeptide, which reveals an excellent agreement between a computation performed on the whole molecule and the results obtained by the present method, especially if the quantum subsystem includes the side chains and the peptidic unit in between. The merits of the LSCF method are examplified by a study of proton transfer in the Asp69---Arg71 salt bridge in dihydrofolate reductase. Simulations of large systems, involving local changes of electronic structure, are therefore possible at a good degree of approximation by introducing a quantum chemical part in molecular dynamics studies. This methodology is expected to be very useful for reactivity studies in biomolecules or at the surface of covalent solids. textcopyright 1994 by John Wiley &amp; Sons, Inc.</td>
</tr>
<tr id="bib_Thery1994" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Thery1994,
  author = {Thery, Vincent and Rinaldi, Daniel and Rivail, Jean-Louis and Maigret, Bernard and Ferenczy, Gy&ouml;rgy G},
  title = {Quantum mechanical computations on very large molecular systems: The local self-consistent field method},
  journal = {Journal of Computational Chemistry},
  year = {1994},
  volume = {15},
  number = {3},
  pages = {269 - 282},
  doi = {https://doi.org/10.1002/jcc.540150303}
}
</pre></td>
</tr>
<tr id="Toney2013" class="entry">
	<td>Toney, M.D., Castro, J. and Addington, T.A.</td>
	<td>Heavy-enzyme kinetic isotope effects on proton transfer in alanine racemase <p class="infolinks">[<a href="javascript:toggleInfo('Toney2013','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Toney2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Journal of the American Chemical Society<br/>Vol. 135(7), pp. 2509 - 2511&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ja3101243">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Toney2013" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The catalytic effects of perdeuterating the pyridoxal phosphate-dependent enzyme alanine racemase from Geobacillus stearothermophilus are reported. The mass of the heavy perdeuterated form is \textasciitilde\5.5&#37; greater than that of the protiated form, causing kinetic isotope effects (KIEs) of \textasciitilde\1.3 on k(cat) and k(cat)/K(M) for both L- and D-alanine. These values increase when Cα-deuterated alanine is used as the substrate. The heavy-enzyme KIEs of \textasciitilde\3 on k(cat)/K(M) with deuterated substrates are greater than the product of the individual heavy-enzyme and primary substrate KIEs. This breakdown of the rule of the geometric mean is likely due to coupled motion between the protein and the proton-transfer reaction coordinate in the rate-limiting step. These data implicate a direct role for protein vibrational motions in barrier crossing for proton-transfer steps in alanine racemase.</td>
</tr>
<tr id="bib_Toney2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Toney2013,
  author = {Toney, Michael D and Castro, Joan and Addington, Trevor A},
  title = {Heavy-enzyme kinetic isotope effects on proton transfer in alanine racemase},
  journal = {Journal of the American Chemical Society},
  year = {2013},
  volume = {135},
  number = {7},
  pages = {2509 - 2511},
  note = {NULL},
  doi = {https://doi.org/10.1021/ja3101243}
}
</pre></td>
</tr>
<tr id="Tonikian2009" class="entry">
	<td>Tonikian, R., Xin, X., Toret, C.P., Gfeller, D., Landgraf, C., Panni, S., Paoluzi, S., Castagnoli, L., Currell, B., Seshagiri, S., Yu, H., Winsor, B., Vidal, M., Gerstein, M.B., Bader, G.D., Volkmer, R., Cesareni, G., Drubin, D.G., Kim, P.M., Sidhu, S.S. and Boone, C.</td>
	<td>Bayesian modeling of the yeast SH3 domain interactome predicts spatiotemporal dynamics of endocytosis proteins <p class="infolinks">[<a href="javascript:toggleInfo('Tonikian2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Tonikian2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>PLoS Biology<br/>Vol. 7(10), pp. e1000218 - e1000218&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pbio.1000218">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Tonikian2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: SH3 domains are peptide recognition modules that mediate the assembly of diverse biological complexes. We scanned billions of phage-displayed peptides to map the binding specificities of the SH3 domain family in the budding yeast, Saccharomyces cerevisiae. Although most of the SH3 domains fall into the canonical classes I and II, each domain utilizes distinct features of its cognate ligands to achieve binding selectivity. Furthermore, we uncovered several SH3 domains with specificity profiles that clearly deviate from the two canonical classes. In conjunction with phage display, we used yeast two-hybrid and peptide array screening to independently identify SH3 domain binding partners. The results from the three complementary techniques were integrated using a Bayesian algorithm to generate a high-confidence yeast SH3 domain interaction map. The interaction map was enriched for proteins involved in endocytosis, revealing a set of SH3-mediated interactions that underlie formation of protein complexes essential to this biological pathway. We used the SH3 domain interaction network to predict the dynamic localization of several previously uncharacterized endocytic proteins, and our analysis suggests a novel role for the SH3 domains of Lsb3p and Lsb4p as hubs that recruit and assemble several endocytic complexes.</td>
</tr>
<tr id="bib_Tonikian2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Tonikian2009,
  author = {Tonikian, Raffi and Xin, Xiaofeng and Toret, Christopher P and Gfeller, David and Landgraf, Christiane and Panni, Simona and Paoluzi, Serena and Castagnoli, Luisa and Currell, Bridget and Seshagiri, Somasekar and Yu, Haiyuan and Winsor, Barbara and Vidal, Marc and Gerstein, Mark B and Bader, Gary D and Volkmer, Rudolf and Cesareni, Gianni and Drubin, David G and Kim, Philip M and Sidhu, Sachdev S and Boone, Charles},
  title = {Bayesian modeling of the yeast SH3 domain interactome predicts spatiotemporal dynamics of endocytosis proteins},
  journal = {PLoS Biology},
  year = {2009},
  volume = {7},
  number = {10},
  pages = {e1000218 - e1000218},
  doi = {https://doi.org/10.1371/journal.pbio.1000218}
}
</pre></td>
</tr>
<tr id="Torrie1977" class="entry">
	<td>Torrie, GM. and Valleau, JP.</td>
	<td>Nonphysical sampling distributions in Monte Carlo free-energy estimation: Umbrella sampling <p class="infolinks">[<a href="javascript:toggleInfo('Torrie1977','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Torrie1977','bibtex')">BibTeX</a>]</p></td>
	<td>1977</td>
	<td>Journal of Computational Physics<br/>Vol. 23(2), pp. 187 - 199&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/0021-9991(77)90121-8">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Torrie1977" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The free energy difference between a model system and some reference system can easily be written as an ensemble average, but the conventional Monte Carlo methods of obtaining such averages are inadequate for the free-energy case. That is because the Boltzmann-weighted sampling distribution ordinarily used is extremely inefficient for the purpose. This paper describes the use of arbitrary sampling distributions chosen to facilitate such estimates. The methods have been tested successfully on the Lennard-Jones system over a wide range of temperature and density, including the gas-liquid coexistence region, and are found to be extremely powerful and economical. ?? 1977.</td>
</tr>
<tr id="bib_Torrie1977" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Torrie1977,
  author = {Torrie, GM and Valleau, JP},
  title = {Nonphysical sampling distributions in Monte Carlo free-energy estimation: Umbrella sampling},
  journal = {Journal of Computational Physics},
  year = {1977},
  volume = {23},
  number = {2},
  pages = {187 - 199},
  doi = {https://doi.org/10.1016/0021-9991(77)90121-8}
}
</pre></td>
</tr>
<tr id="Touboul2018" class="entry">
	<td>Touboul, J.D., Staver, A.C. and Levin, S.A.</td>
	<td>On the complex dynamics of savanna landscapes <p class="infolinks">[<a href="javascript:toggleInfo('Touboul2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Proceedings of the National Academy of Sciences, pp. 201712356&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Touboul2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Touboul2018,
  author = {Touboul, Jonathan David and Staver, Ann Carla and Levin, Simon Asher},
  title = {On the complex dynamics of savanna landscapes},
  journal = {Proceedings of the National Academy of Sciences},
  publisher = {National Acad Sciences},
  year = {2018},
  pages = {201712356}
}
</pre></td>
</tr>
<tr id="Toure2018" class="entry">
	<td>Tour&eacute;, V., Nov&egrave;re, N.L., Waltemath, D. and Wolkenhauer, O.</td>
	<td>Quick tips for creating effective and impactful biological pathways using the Systems Biology Graphical Notation <p class="infolinks">[<a href="javascript:toggleInfo('Toure2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>PLOS Computational Biology<br/>Vol. 14(2), pp. e1005740&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pcbi.1005740">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Toure2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Toure2018,
  author = {Vasundra Tour&eacute; and Nicolas Le Nov&egrave;re and Dagmar Waltemath and Olaf Wolkenhauer},
  title = {Quick tips for creating effective and impactful biological pathways using the Systems Biology Graphical Notation},
  journal = {PLOS Computational Biology},
  publisher = {Public Library of Science (PLoS)},
  year = {2018},
  volume = {14},
  number = {2},
  pages = {e1005740},
  doi = {https://doi.org/10.1371/journal.pcbi.1005740}
}
</pre></td>
</tr>
<tr id="Towse2017" class="entry">
	<td>Towse, Clare-Louise., Akke, M. and Dagget, V.</td>
	<td>The Dynameomics Entropy Dictionary: A Large-Scale Assessment of Conformational Entropy Across Protein Fold Space <p class="infolinks">[<a href="javascript:toggleInfo('Towse2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Towse2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 121, pp. 3933 - 3945&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jpcb.7b00577">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Towse2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Molecular dynamics (MD) simulations contain considerable information with regard to the mo-tions and fluctuations of a protein, the magnitude of which can be used to estimate conforma-tional entropy. Here we survey conformational entropy across protein fold space using the Dy-nameomics database, which represents the largest existing dataset of protein MD simulations for representatives of essentially all known protein folds. We provide an overview of MD-derived entropies accounting for all possible degrees of dihedral freedom on an unprecedented scale. Although different side chains might be expected to impose varying restrictions on the confor-mational space that the backbone can sample, we found that the backbone entropy and side chain size are not strictly coupled. An outcome of these analyses is the Dynameomics Entropy Dic-tionary, the contents of which have been compared with entropies derived by other theoretical approaches and experiment. As might be expected, the conformational entropies scale linearly with the number of residues, demonstrating that conformational entropy is an extensive property of proteins. The calculated conformational entropies of folding agree well with previous esti-mates. Detailed analysis of specific cases identify deviations in conformational entropy from the average values that highlight how conformational entropy varies with sequence, secondary struc-ture, and tertiary fold. Notably, α-helices have lower entropy on average than do β-sheets, and both are lower than coil regions.</td>
</tr>
<tr id="bib_Towse2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Towse2017,
  author = {Towse, Clare-Louise and Akke, Mikael and Dagget, Valerie},
  title = {The Dynameomics Entropy Dictionary: A Large-Scale Assessment of Conformational Entropy Across Protein Fold Space},
  journal = {Journal of Physical Chemistry B},
  year = {2017},
  volume = {121},
  pages = {3933 - 3945},
  doi = {https://doi.org/10.1021/acs.jpcb.7b00577}
}
</pre></td>
</tr>
<tr id="Traub2003" class="entry">
	<td>Traub, L.M.</td>
	<td>Sorting it out: AP-2 and alternate clathrin adaptors in endocytic cargo selection. <p class="infolinks">[<a href="javascript:toggleInfo('Traub2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Traub2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>The Journal of cell biology<br/>Vol. 163, pp. 203-208&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1083/jcb.200309175">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Traub2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The AP-2 adaptor complex is widely viewed as a linchpin molecule in clathrin-mediated endocytosis, simultaneously binding both clathrin and receptors. This dual interaction couples cargo capture with clathrin coat assembly, but it has now been discovered that the association with cargo is tightly regulated. Remarkably, AP-2 is not obligatory for all clathrin-mediated uptake, and several alternate adaptors appear to perform similar sorting and assembly functions at the clathrin bud site.</td>
</tr>
<tr id="bib_Traub2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Traub2003,
  author = {Traub, Linton M},
  title = {Sorting it out: AP-2 and alternate clathrin adaptors in endocytic cargo selection.},
  journal = {The Journal of cell biology},
  year = {2003},
  volume = {163},
  pages = {203--208},
  doi = {https://doi.org/10.1083/jcb.200309175}
}
</pre></td>
</tr>
<tr id="Traub2009" class="entry">
	<td>Traub, L.M.</td>
	<td>Tickets to ride: selecting cargo for clathrin-regulated internalization. <p class="infolinks">[<a href="javascript:toggleInfo('Traub2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Traub2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Nature reviews. Molecular cell biology<br/>Vol. 10, pp. 583-596&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nrm2751">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Traub2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis oversees the constitutive packaging of selected membrane cargoes into transport vesicles that fuse with early endosomes. The process is responsive to activation of signalling receptors and ion channels, promptly clearing post-translationally tagged forms of cargo off the plasma membrane. To accommodate the diverse array of transmembrane proteins that are variably gathered into forming vesicles, a dedicated sorting machinery cooperates to ensure that non-competitive uptake from the cell surface occurs within minutes. Recent structural and functional data reveal remarkable plasticity in how disparate sorting signals are recognized by cargo-selective clathrin adaptors, such as AP-2. Cargo loading also seems to govern whether coats ultimately bud or dismantle abortively at the cell surface.</td>
</tr>
<tr id="bib_Traub2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Traub2009,
  author = {Traub, Linton M},
  title = {Tickets to ride: selecting cargo for clathrin-regulated internalization.},
  journal = {Nature reviews. Molecular cell biology},
  year = {2009},
  volume = {10},
  pages = {583--596},
  note = {Fig 4 has some domain and interaction information},
  doi = {https://doi.org/10.1038/nrm2751}
}
</pre></td>
</tr>
<tr id="Traub1999" class="entry">
	<td>Traub, L.M., Downs, M.A., Westrich, J.L. and Fremont, D.H.</td>
	<td>Crystal structure of the α appendage of AP-2 reveals a recruitment platform for clathrin-coat assembly <p class="infolinks">[<a href="javascript:toggleInfo('Traub1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Traub1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Proceedings of the National Academy of Sciences<br/>Vol. 96(16)&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.96.16.8907">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Traub1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: AP-2 adaptors regulate clathrin-bud formation at the cell surface by recruiting clathrin trimers to the plasma membrane and by selecting certain membrane proteins for inclusion within the developing clathrin-coat structure. These functions are performed by discrete subunits of the adaptor heterotetramer. The carboxyl-terminal appendage of the AP-2 α subunit appears to regulate the translocation of several endocytic accessory proteins to the bud site. We have determined the crystal structure of the α appendage at 1.4-&Aring; resolution by multiwavelength anomalous diffraction phasing. It is composed of two distinct structural modules, a β-sandwich domain and a mixed α--β platform domain. Structure-based mutagenesis shows that alterations to the molecular surface of a highly conserved region on the platform domain differentially affect associations of the appendage with amphiphysin, eps15, epsin, and AP180, revealing a common protein-binding interface.</td>
</tr>
<tr id="bib_Traub1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Traub1999,
  author = {Traub, Linton M. and Downs, Maureen A. and Westrich, Jennifer L. and Fremont, Daved H.},
  title = {Crystal structure of the α appendage of AP-2 reveals a recruitment platform for clathrin-coat assembly},
  journal = {Proceedings of the National Academy of Sciences},
  year = {1999},
  volume = {96},
  number = {16},
  note = {Competitive binding of accessory proteins to AP2 alpha subunit},
  doi = {https://doi.org/10.1073/pnas.96.16.8907}
}
</pre></td>
</tr>
<tr id="Truhlar1984" class="entry">
	<td>Truhlar, DG. and Garrett, BC.</td>
	<td>Variational Transition State Theory <p class="infolinks">[<a href="javascript:toggleInfo('Truhlar1984','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Truhlar1984','bibtex')">BibTeX</a>]</p></td>
	<td>1984</td>
	<td>Annual Review of Physical Chemistry<br/>Vol. 35(1), pp. 159 - 189&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1146/annurev.pc.35.100184.001111">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Truhlar1984" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Transition state theory (TST)l is the most widely used theory for calculating rates of bimolecular reactions occurring in the gas phase and in condensed phases. TST is also incorporated into the widely used RRKM theory for unimolecular reactions. The popularity of TST is largely due to its simplicity and its usefulness for correlating trends in reaction rates in terms of easily interpreted quantities. Several forms of variational transition state theory (VTST) have been proposed, one as early as 1937 ; however, until recently, most applications of TST have been limited to the conventional (nonvariational) formulation.</td>
</tr>
<tr id="bib_Truhlar1984" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Truhlar1984,
  author = {Truhlar, DG and Garrett, BC},
  title = {Variational Transition State Theory},
  journal = {Annual Review of Physical Chemistry},
  year = {1984},
  volume = {35},
  number = {1},
  pages = {159 - 189},
  doi = {https://doi.org/10.1146/annurev.pc.35.100184.001111}
}
</pre></td>
</tr>
<tr id="Truhlar1996" class="entry">
	<td>Truhlar, DG., Garrett, B.C. and Klippenstein, S.J.</td>
	<td>Current Status of Transition-State Theory <p class="infolinks">[<a href="javascript:toggleInfo('Truhlar1996','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Truhlar1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>J. Phys. Chem.<br/>Vol. 100(31), pp. 12771 - 12800&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp953748q">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Truhlar1996" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present an overview of the current status of transition-state theory and its generalizations. We emphasize (i) recent improvements in available methodology for calculations on complex systems, including the interface with electronic structure theory, (ii) progress in the theory and application of transition-state theory to condensed-phase reactions, and (iii) insight into the relation of transition-state theory to accurate quantum dynamics and tests of its accuracy via comparisons with both experimental and other theoretical dynamical approximations.</td>
</tr>
<tr id="bib_Truhlar1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Truhlar1996,
  author = {Truhlar, DG and Garrett, Bruce C and Klippenstein, Stephen J},
  title = {Current Status of Transition-State Theory},
  journal = {J. Phys. Chem.},
  year = {1996},
  volume = {100},
  number = {31},
  pages = {12771 - 12800},
  doi = {https://doi.org/10.1021/jp953748q}
}
</pre></td>
</tr>
<tr id="Truhlar1983" class="entry">
	<td>Truhlar, D.G.</td>
	<td>Current Status of Transltion-State Theory <p class="infolinks">[<a href="javascript:toggleInfo('Truhlar1983','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Truhlar1983','bibtex')">BibTeX</a>]</p></td>
	<td>1983</td>
	<td>Journal of Physical Chemistry<br/>Vol. 87(15), pp. 2664 - 2682&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/j100238a003">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Truhlar1983" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: accurate, there is always hope for empirical or semi-empirical correlations, especially when these are carried out in the convenient transition-state-theory formalism of enthalpy of activation, entropy of activation, and heat capacity of activation.' Furthermore, in some applications, as for kinetic isotope effects and high-temperature rate constants, the sensitivity of the calculated results to the barrier height of the potential energy surface is reduced. For other cases, as in choosing between postulated mech-anisms for some reactions of complicated species, definite conclusions can be drawn even in the face of large un-certainties in the absolute rate constants. In still other cases, e.g., reactions of mechanistic intermediates or re-actions under extreme conditions, there may not be a good way to obtain information on rate coefficients by labora-tory experiments, so theory may be the only recourse. Thus, absolute rate calculations can often be very useful, and we may expect them to become more and more useful as techniques for both the structural and dynamic parts of the calculation improve. Transition-state theory, despite its age, is still the method of choice for most attempts at absolute rate calculations. Furthermore, it appears to be almost without competition for more qualitative discus-sions of a wide variety of important questions in chemical kinetics, such as solvent effects. The reduction of the dynamics problem to the consideration of a single structure provides unique opportunities for qualitative considera-tions; e.g., can this fast a rate constant or these relative rates possibly be consistent with this transition-state structure? As discussed very clearly in the preceding article by Laidler and King: the calculation of absolute reaction rates from potential energy surfaces was cast in a particularly appealing form by Eyring in 193L3 Because of the sim-plicity and elegance of the resulting equations, they became widely used and are still widely used in their original 1935 form. Transition-state theory is based on a quasiequili-brium hypothesis with a simple physical interpretation. Most importantly from a practical standpoint, this qua-siequilibrium hypothesis greatly reduces the computational requirements of the theory. The reduced computational requirements plus the ability of the theory to explain general trends in preexponential factors and kinetic isotope effects for bimolecular reactions and the energy depen-dence of rate constants for unimolecular reactions have been the major reasons for its general popularity. Modern work, however, is not so constrained by the requirement to minimize computing, and a major goal of much current research on transition-state theory is to re-fine and improve or correct the theory, even at increased computational expense. This is the general goal of the rapidly widening field of generalized transition-state the-ories. Most attempts to improve the theory are based on incorporating more detailed dynamics into it. Concomi-tantly there is an effort to test transition-state theory, in both its conventional and generalized formulations, against more accurate dynamical theories and against experiment. The improved understanding of the dynamic basis of transition-state theory, the new developments in gener-alized transition-state theory, progress in learning how to make applications to complicated solution phenomena more realistic, and the successes, failures, and ambiguities (1) S. W. Benson, "Thermochemical Kinetics", 2nd ed., Wiley, New (2) K. J. Laidler and M. C. King, J. Phys. Chem., preceding paper in York, 1976. this issue. (3) H. Eyring, J. Chem. Phys., 3,107 (1935); S. Glasstone, K. Laidler, and H. Eyring, "The Theory of Rate Processes", McGraw-Hill, New York, 1941. of the various testa constitute the current status of tran-sition-state theory in physical chemistry, and these are the areas with which the present article is concerned. Transition-state concepts are so widely used in chemistry that complete coverage of all current approaches and ap-plications in one article seems impossible. This fact itself may be the greatest tribute to the theory and ita founders. Recent extensions of the theory exhibit very encouraging vitality, however, so we expect that future successes of transition-state theory and closely related models de-scended from it will continue to make the task of complete coverage harder and harder and make our debt to the early</td>
</tr>
<tr id="bib_Truhlar1983" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Truhlar1983,
  author = {Truhlar, Donald G},
  title = {Current Status of Transltion-State Theory},
  journal = {Journal of Physical Chemistry},
  year = {1983},
  volume = {87},
  number = {15},
  pages = {2664 - 2682},
  doi = {https://doi.org/10.1021/j100238a003}
}
</pre></td>
</tr>
<tr id="Truhlar2015" class="entry">
	<td>Truhlar, D.G.</td>
	<td>Transition state theory for enzyme kinetics <p class="infolinks">[<a href="javascript:toggleInfo('Truhlar2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Truhlar2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Archives of Biochemistry and Biophysics<br/>Vol. 582, pp. 10 - 17&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.abb.2015.05.004">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Truhlar2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: This article is an essay that discusses the concepts underlying the application of modern transition state theory to reactions in enzymes. Issues covered include the potential of mean force, the quantization of vibrations, the free energy of activation, and transmission coefficients to account for nonequilibrium effect, recrossing, and tunneling.</td>
</tr>
<tr id="bib_Truhlar2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Truhlar2015,
  author = {Truhlar, Donald G},
  title = {Transition state theory for enzyme kinetics},
  journal = {Archives of Biochemistry and Biophysics},
  year = {2015},
  volume = {582},
  pages = {10 - 17},
  doi = {https://doi.org/10.1016/j.abb.2015.05.004}
}
</pre></td>
</tr>
<tr id="Truhlar2004" class="entry">
	<td>Truhlar, D.G., Gao, J., Mireia, Garcia-Viloca,., Alhambra, C., Corchado, J., Sanchez, M. and Poulsen, T.D.</td>
	<td>Ensemble-averaged variational transition state theory with optimized multidimensional tunneling for enzyme kinetics and other condensed-phase reactions <p class="infolinks">[<a href="javascript:toggleInfo('Truhlar2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Truhlar2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>International Journal of Quantum Chemistry<br/>Vol. 100(6), pp. 1136 - 1152&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/qua.20205">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Truhlar2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: This paper provides an overview of a new method developed to include quantum mechanical effects and free energy sampling in calculations of reaction rates in enzymes. The paper includes an overview of variational transition state theory with optimized multidimensional tunneling for simple gas-phase reactions and then shows how this is extended to incorporate free energy effects and to include protein motions in the reaction coordinate by ensemble averaging. Finally we summarize recent comparisons to experiment for primary and secondary kinetic isotope effects for proton and hydride transfer reactions catalyzed by enzymes. (C) 2004 Wiley Periodicals, Inc.</td>
</tr>
<tr id="bib_Truhlar2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Truhlar2004,
  author = {Truhlar, Donald G and Gao, Jiali and Mireia, Garcia-Viloca, and Alhambra, Cristobal and Corchado, Jose and Sanchez, Maria and Poulsen, Tina D},
  title = {Ensemble-averaged variational transition state theory with optimized multidimensional tunneling for enzyme kinetics and other condensed-phase reactions},
  journal = {International Journal of Quantum Chemistry},
  year = {2004},
  volume = {100},
  number = {6},
  pages = {1136 - 1152},
  doi = {https://doi.org/10.1002/qua.20205}
}
</pre></td>
</tr>
<tr id="Tsyba2011" class="entry">
	<td>Tsyba, L., Nikolaienko, O., Dergai, O., Dergai, M., Novokhatska, O., Skrypkina, I. and Rynditch, A.</td>
	<td>Intersectin multidomain adaptor proteins: regulation of functional diversity. <p class="infolinks">[<a href="javascript:toggleInfo('Tsyba2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Tsyba2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Gene<br/>Vol. 473, pp. 67-75&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.gene.2010.11.016">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Tsyba2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Adaptor/scaffold proteins serve as platforms for the assembly of multiprotein complexes and regulate the efficiency and specificity of signalling cascades. Intersectins (ITSNs) are an evolutionarily conserved adaptor protein family engaged in endo- and exocytosis, actin cytoskeleton rearrangement and signal transduction. This review summarizes recent advances in the function of ITSNs in neuronal and non-neuronal cells, the role of alternative splicing and alternative transcription in regulating the structural and functional diversity of ITSNs, their expression patterns in different tissues and during development, their interactions with proteins, as well as the potential relevance of ITSNs for neurodegenerative diseases and cancer. The diversity of mechanisms in the regulation of ITSN expression and specificity in different cells emphasizes the important role of ITSN proteins in vesicle trafficking and signalling.</td>
</tr>
<tr id="bib_Tsyba2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Tsyba2011,
  author = {Tsyba, Liudmyla and Nikolaienko, Oleksii and Dergai, Oleksandr and Dergai, Mykola and Novokhatska, Olga and Skrypkina, Inessa and Rynditch, Alla},
  title = {Intersectin multidomain adaptor proteins: regulation of functional diversity.},
  journal = {Gene},
  year = {2011},
  volume = {473},
  pages = {67--75},
  note = {excellent figure showing domain interactions of ITSN1.},
  doi = {https://doi.org/10.1016/j.gene.2010.11.016}
}
</pre></td>
</tr>
<tr id="Tuckerman1993" class="entry">
	<td>Tuckerman, M.E., Berne, B.J., Martyna, G.J. and Klein, M.L.</td>
	<td>Efficient molecular dynamics and hybrid Monte Carlo algorithms for path integrals <p class="infolinks">[<a href="javascript:toggleInfo('Tuckerman1993','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Tuckerman1993','bibtex')">BibTeX</a>]</p></td>
	<td>1993</td>
	<td>Journal of Chemical Physics<br/>Vol. 99(4), pp. 2796 - 2796&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.465188">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Tuckerman1993" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: New path integral molecular dynamics (PIMD) and path integral hybrid Monte Carlo (PIHMC) algorithms are developed. It is shown that the use of a simple noncanonical change of variables that naturally divides the quadratic part of the action into long and short wavelength modes and multiple time scale integration techniques results in very efficient algorithms. The PIMD method also employs a constant temperature MD technique that has been shown to give canonical averages even for stiff systems. The new methods are applied to the simple quantum mechanical harmonic oscillator and to electron solvation in fluid helium and xenon. Comparisons are made with PIMC and the more basic PIMD and PIHMC methods.</td>
</tr>
<tr id="bib_Tuckerman1993" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Tuckerman1993,
  author = {Tuckerman, Mark E and Berne, Bruce J and Martyna, Glenn J and Klein, Michael L},
  title = {Efficient molecular dynamics and hybrid Monte Carlo algorithms for path integrals},
  journal = {Journal of Chemical Physics},
  year = {1993},
  volume = {99},
  number = {4},
  pages = {2796 - 2796},
  doi = {https://doi.org/10.1063/1.465188}
}
</pre></td>
</tr>
<tr id="Tuckerman2001" class="entry">
	<td>Tuckerman, M.E. and Marx, D.</td>
	<td>Heavy-atom skeleton quantization and proton tunneling in "intermediate-barrier" hydrogen bonds. <p class="infolinks">[<a href="javascript:toggleInfo('Tuckerman2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Tuckerman2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Physical Review Letters<br/>Vol. 86(21), pp. 4946 - 9&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1103/PhysRevLett.86.4946">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Tuckerman2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Quantum effects on proton transfer through barriers of several kcal/mol in hydrogen bonds are investigated theoretically in malonaldehyde. Such "intermediate-barrier" proton transfer processes play a key role in the catalytic activity of some enzymes. Tunneling is shown to be significant in this reaction even at room temperature. More importantly, the quantum nature of the heavy molecular frame atoms is found to substantially enhance proton tunneling. These findings have far-reaching implications for common modeling strategies of proton transfer in complex systems such as biomolecules.</td>
</tr>
<tr id="bib_Tuckerman2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Tuckerman2001,
  author = {Tuckerman, Mark E and Marx, D},
  title = {Heavy-atom skeleton quantization and proton tunneling in "intermediate-barrier" hydrogen bonds.},
  journal = {Physical Review Letters},
  year = {2001},
  volume = {86},
  number = {21},
  pages = {4946 - 9},
  doi = {https://doi.org/10.1103/PhysRevLett.86.4946}
}
</pre></td>
</tr>
<tr id="Tuckerman1996" class="entry">
	<td>Tuckerman, M.E., Marx, D., Klein, M.L. and Parrinello, M.</td>
	<td>Efficient and general algorithms for path integral Car--Parrinello molecular dynamics <p class="infolinks">[<a href="javascript:toggleInfo('Tuckerman1996','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Tuckerman1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>Journal of Chemical Physics<br/>Vol. 104(14), pp. 5579 - 5579&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.471771">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Tuckerman1996" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Trace amines are endogenous compounds, typified by 2-phenylethylamine (PE) and p-tyramine (TA), found in the vertebrate central nervous system. Although synthesized in pre-synaptic terminals, trace amines do not appear to act as neurotransmitters, but rather modulate responsivity to co-existing neurotransmitters. Trace amines are neither actively accumulated in synaptic vesicles, nor released in an activity-dependent manner. Further, Trace Amine-Associated Receptor 1 (TAAR1), which is selectively activated by PE and TA, is intracellular. As such, PE and TA need to cross cell membranes in order to exert their effects. This has been assumed to occur by simple diffusion, but has not previously been systematically examined. Experimental data were obtained using Fluorosome(&reg;) technology. A permeability coefficient of 25.3 textpm 3.8 &Aring;/s (n = 6) was obtained for TA which was not significantly different from that obtained for the monoamine neurotransmitter noradrenaline (20.3 textpm 3.8 &Aring;/s, n = 8). PE was unsuitable for use with this system. We have also used molecular dynamics computer simulation techniques to determine the potential of mean force (PMF) associated with trace amine passage across lipid bilayers. A PMF peak barrier of 25 textpm 6 kcal/mol (protonated) and 13 textpm 1 kcal/mol (deprotonated) was obtained for PE. Protonated TA peak energy barriers were even greater at 31 textpm 1 kcal/mol. Application of a homogeneous solubility-diffusion model combining the measured permeability coefficients and simulated PMF allows fitting of the diffusion coefficient for trace amines in the hydrophobic region of the lipid bilayer. The diffusion coefficients in other regions of the membrane were found to make negligible contributions to the permeability coefficient for the calculated PMF. The fit obtained a value for the diffusion coefficient of (163 textpm 25) texttimes 10(-10) m(2)/s for TA(+) in the hydrophobic core region. The diffusion coefficient for TA(+) in the aqueous compartment was also calculated directly by simulation yielding a value of (0.62 textpm 0.26) texttimes 10(-10) m(2)/s. The adopted simulation methods failed to yield diffusion coefficients in the core region indicating that further work will be required to accurately predict permeability coefficients for trace amines passing through membranes.</td>
</tr>
<tr id="bib_Tuckerman1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Tuckerman1996,
  author = {Tuckerman, Mark E and Marx, Dominik and Klein, Michael L and Parrinello, Michele},
  title = {Efficient and general algorithms for path integral Car--Parrinello molecular dynamics},
  journal = {Journal of Chemical Physics},
  year = {1996},
  volume = {104},
  number = {14},
  pages = {5579 - 5579},
  doi = {https://doi.org/10.1063/1.471771}
}
</pre></td>
</tr>
<tr id="Tunon2015" class="entry">
	<td>Tu&ntilde;&oacute;n, I., Laage, D. and Hynes, J.T.</td>
	<td>Are there dynamical effects in enzyme catalysis? Some thoughts concerning the enzymatic chemical step <p class="infolinks">[<a href="javascript:toggleInfo('Tunon2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Tunon2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Archives of Biochemistry and Biophysics<br/>Vol. 582, pp. 42 - 55&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.abb.2015.06.004">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Tunon2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We offer some thoughts on the much debated issue of dynamical effects in enzyme catalysis, and more specifically on their potential role in the acceleration of the chemical step. Since the term 'dynamics' has been used with different meanings, we find it useful to first return to the Transition State Theory rate constant, its assumptions and the choices it involves, and detail the various sources of deviations from it due to dynamics (or not). We suggest that much can be learned about the key current questions for enzyme catalysis from prior extensive studies of dynamical and other effects in the case of reactions in solution. We analyze dynamical effects both in the neighborhood of the transition state and far from it, together with the situation when quantum nuclear motion is central to the reaction, and we illustrate our discussion with various examples of enzymatic reactions.</td>
</tr>
<tr id="bib_Tunon2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Tunon2015,
  author = {Tu&ntilde;&oacute;n, I&ntilde;aki and Laage, Damien and Hynes, James T},
  title = {Are there dynamical effects in enzyme catalysis? Some thoughts concerning the enzymatic chemical step},
  journal = {Archives of Biochemistry and Biophysics},
  year = {2015},
  volume = {582},
  pages = {42 - 55},
  doi = {https://doi.org/10.1016/j.abb.2015.06.004}
}
</pre></td>
</tr>
<tr id="Uezu2011" class="entry">
	<td>Uezu, A., Umeda, K., Tsujita, K., Suetsugu, S., Takenawa, T. and Nakanishi, H.</td>
	<td>Characterization of the EFC/F-BAR domain protein, FCHO2 <p class="infolinks">[<a href="javascript:toggleInfo('Uezu2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Uezu2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Genes to Cells<br/>Vol. 16(8), pp. 868 - 878&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1111/j.1365-2443.2011.01536.x">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Uezu2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We have previously shown that SGIP1α is an endocytic protein specifically expressed in neural tissues. SGIP1α has a lipid-binding domain called the MP domain, which shows no significant homology to any other domains. In this study, we characterized FCHO2, a protein with a high level of homology to SGIP1α. FCHO2 lacks the MP domain but has another lipid-binding domain, the EFC/F-BAR domain. FCHO2 was ubiquitously expressed. The FCHO2 EFC domain bound to phosphatidylserine and phosphoinositides and deformed the plasma membrane and liposomes into narrow tubes. FCHO2 localized to clathrin-coated pits at the plasma membrane and bound to Eps15, an important adaptor protein in clathrin-mediated endocytosis. FCHO2 knockdown reduced transferrin endocytosis. These results suggest that FCHO2 regulates clathrin-mediated endocytosis through its interactions with membranes and Eps15. These properties of FCHO2 are similar to those of SGIP1α. FCHO2 is likely to be a ubiquitous homologue of SGIP1α. We furthermore found that FCHO2 was subjected to monoubiquitination, and gel filtration analysis showed that FCHO2 formed an oligomer. These new properties might also contribute to the role of FCHO2 in clathrin-mediated endocytosis.</td>
</tr>
<tr id="bib_Uezu2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Uezu2011,
  author = {Uezu, Akiyoshi and Umeda, Kazuaki and Tsujita, Kazuya and Suetsugu, Shiro and Takenawa, Tadaomi and Nakanishi, Hiroyuki},
  title = {Characterization of the EFC/F-BAR domain protein, FCHO2},
  journal = {Genes to Cells},
  year = {2011},
  volume = {16},
  number = {8},
  pages = {868 - 878},
  note = {mouse cells. suggests FCHO2 might oligomerize. domain and dissociation constant information. FCHO2 - PIP2, EPS15. Says FHCO2 co-precipitates with alpha-adaptin, but its not clear that its a direct interaction.},
  doi = {https://doi.org/10.1111/j.1365-2443.2011.01536.x}
}
</pre></td>
</tr>
<tr id="Umasankar2012" class="entry">
	<td>Umasankar, PK., Sanker, S., Thieman, J.R., Chakraborty, S., Wendland, B., Tsang, M. and Traub, L.M.</td>
	<td>Distinct and separable activities of the endocytic clathrin-coat components Fcho1/2 and AP-2 in developmental patterning <p class="infolinks">[<a href="javascript:toggleInfo('Umasankar2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Umasankar2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Nature Cell Biology<br/>Vol. 14(5), pp. 488 - 501&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/ncb2473">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Umasankar2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis occurs at multiple independent import sites on the plasma membrane, but how these positions are selected and how different cargo is simultaneously recognized is obscure. FCHO1 and FCHO2 are early-arriving proteins at surface clathrin assemblies and are speculated to act as compulsory coat nucleators, preceding the core clathrin adaptor AP-2. Here, we show that the μ-homology domain of FCHO1/2 represents an endocytic interaction hub. Translational silencing of fcho1 in zebrafish embryos causes strong dorsoventral patterning defects analogous to Bmp signal failure. The Fcho1 μ-homology domain interacts with the Bmp receptor Alk8, uncovering an endocytic component that positively modulates Bmp signal transmission. Still, the fcho1 morphant phenotype is distinct from severe embryonic defects apparent when AP-2 is depleted. Our data thus challenge the primacy of FCHO1/2 in coat initiation.</td>
</tr>
<tr id="bib_Umasankar2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Umasankar2012,
  author = {Umasankar, PK and Sanker, Subramaniam and Thieman, James R and Chakraborty, Souvik and Wendland, Beverly and Tsang, Michael and Traub, Linton M},
  title = {Distinct and separable activities of the endocytic clathrin-coat components Fcho1/2 and AP-2 in developmental patterning},
  journal = {Nature Cell Biology},
  year = {2012},
  volume = {14},
  number = {5},
  pages = {488 - 501},
  note = {FCHO1/2.MUH interactions. domain interactions. zebrafish FCHO1/2.},
  doi = {https://doi.org/10.1038/ncb2473}
}
</pre></td>
</tr>
<tr id="Umasankar2014" class="entry">
	<td>Umasankar, P.K., Ma, L., Thieman, J.R., Jha, A., Doray, B., Watkins, S.C. and Traub, L.M.</td>
	<td>A clathrin coat assembly role for the muniscin protein central linker revealed by TALEN-mediated gene editing. <p class="infolinks">[<a href="javascript:toggleInfo('Umasankar2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Umasankar2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>eLife<br/>Vol. 3&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.7554/eLife.04137">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Umasankar2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis is an evolutionarily ancient membrane transport system regulating cellular receptivity and responsiveness. Plasmalemma clathrin-coated structures range from unitary domed assemblies to expansive planar constructions with internal or flanking invaginated buds. Precisely how these morphologically-distinct coats are formed, and whether all are functionally equivalent for selective cargo internalization is still disputed. We have disrupted the genes encoding a set of early arriving clathrin-coat constituents, FCHO1 and FCHO2, in HeLa cells. Endocytic coats do not disappear in this genetic background; rather clustered planar lattices predominate and endocytosis slows, but does not cease. The central linker of FCHO proteins acts as an allosteric regulator of the prime endocytic adaptor, AP-2. By loading AP-2 onto the plasma membrane, FCHO proteins provide a parallel pathway for AP-2 activation and clathrin-coat fabrication. Further, the steady-state morphology of clathrin-coated structures appears to be a manifestation of the availability of the muniscin linker during lattice polymerization.</td>
</tr>
<tr id="bib_Umasankar2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Umasankar2014,
  author = {Umasankar, Perunthottathu K and Ma, Li and Thieman, James R and Jha, Anupma and Doray, Balraj and Watkins, Simon C and Traub, Linton M},
  title = {A clathrin coat assembly role for the muniscin protein central linker revealed by TALEN-mediated gene editing.},
  journal = {eLife},
  year = {2014},
  volume = {3},
  doi = {https://doi.org/10.7554/eLife.04137}
}
</pre></td>
</tr>
<tr id="Ungewickell" class="entry">
	<td>Ungewickell, E.</td>
	<td>Clathrin: A good view of a shapely leg <p class="infolinks">[<a href="javascript:toggleInfo('Ungewickell','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Current Biology<br/>Vol. 9(1), pp. R32-R35&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/s0960-9822(99)80040-2">DOI</a> <a href="http://dx.doi.org/10.1016/S0960-9822(99)80040-2">URL</a>&nbsp;</td>
</tr>
<tr id="bib_Ungewickell" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ungewickell,
  author = {Ungewickell, Ernst},
  title = {Clathrin: A good view of a shapely leg},
  journal = {Current Biology},
  publisher = {Elsevier},
  year = {1999},
  volume = {9},
  number = {1},
  pages = {R32--R35},
  url = {http://dx.doi.org/10.1016/S0960-9822(99)80040-2},
  doi = {https://doi.org/10.1016/s0960-9822(99)80040-2}
}
</pre></td>
</tr>
<tr id="Vanden-Eijnden2005" class="entry">
	<td>Vanden-Eijnden, E. and Tal, F.A.</td>
	<td>Transition state theory: variational formulation, dynamical corrections, and error estimates <p class="infolinks">[<a href="javascript:toggleInfo('Vanden-Eijnden2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Vanden-Eijnden2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Journal of Chemical Physics<br/>Vol. 123(18), pp. 184103 - 184103&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.2102898">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Vanden-Eijnden2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Transition state theory (TST) is revisited, as well as evolutions upon TST such as variational TST in which the TST dividing surface is optimized so as to minimize the rate of recrossing through this surface and methods which aim at computing dynamical corrections to the TST transition rate constant. The theory is discussed from an original viewpoint. It is shown how to compute exactly the mean frequency of transition between two predefined sets which either partition phase space (as in TST) or are taken to be well-separated metastable sets corresponding to long-lived conformation states (as necessary to obtain the actual transition rate constants between these states). Exact and approximate criterions for the optimal TST dividing surface with minimum recrossing rate are derived. Some issues about the definition and meaning of the free energy in the context of TST are also discussed. Finally precise error estimates for the numerical procedure to evaluate the transmission coefficient kappaS of the TST dividing surface are given, and it is shown that the relative error on kappaS scales as 1/square root(kappaS) when kappaS is small. This implies that dynamical corrections to the TST rate constant can be computed efficiently if and only if the TST dividing surface has a transmission coefficient kappaS which is not too small. In particular, the TST dividing surface must be optimized upon (for otherwise kappaS is generally very small), but this may not be sufficient to make the procedure numerically efficient (because the optimal dividing surface has maximum kappaS, but this coefficient may still be very small).</td>
</tr>
<tr id="bib_Vanden-Eijnden2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Vanden-Eijnden2005,
  author = {Vanden-Eijnden, Eric and Tal, Fabio A},
  title = {Transition state theory: variational formulation, dynamical corrections, and error estimates},
  journal = {Journal of Chemical Physics},
  year = {2005},
  volume = {123},
  number = {18},
  pages = {184103 - 184103},
  doi = {https://doi.org/10.1063/1.2102898}
}
</pre></td>
</tr>
<tr id="Vanommeslaeghe2010" class="entry">
	<td>Vanommeslaeghe, K., Hatcher, E., Acharya, C., Kundu, S., Zhong, S., Shim, J., Darian, E., Guvench, O., Lopes, P., Vorobyov, I. and Mackerell, AD.</td>
	<td>CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields. <p class="infolinks">[<a href="javascript:toggleInfo('Vanommeslaeghe2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Journal of computational chemistry<br/>Vol. 31(4), pp. 671 - 90&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/jcc.21367">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Vanommeslaeghe2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Vanommeslaeghe2010,
  author = {Vanommeslaeghe, K and Hatcher, E and Acharya, C and Kundu, S and Zhong, S and Shim, J and Darian, E and Guvench, O and Lopes, P and Vorobyov, I and Mackerell, AD},
  title = {CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields.},
  journal = {Journal of computational chemistry},
  year = {2010},
  volume = {31},
  number = {4},
  pages = {671 - 90},
  doi = {https://doi.org/10.1002/jcc.21367}
}
</pre></td>
</tr>
<tr id="Vanommeslaeghe2012a" class="entry">
	<td>Vanommeslaeghe, K., Raman, P.E. and MacKerell,, AD</td>
	<td>Automation of the CHARMM General Force Field CGenFF I: bond perception and atom typing <p class="infolinks">[<a href="javascript:toggleInfo('Vanommeslaeghe2012a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Vanommeslaeghe2012a','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Journal of Chemical Information and Modeling<br/>Vol. 52(12), pp. 3155 - 68&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ci3003649">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Vanommeslaeghe2012a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Molecular mechanics force fields are widely used in computer-aided drug design for the study of drug candidates interacting with biological systems. In these simulations, the biological part is typically represented by a specialized biomolecular force field, while the drug is represented by a matching general (organic) force field. In order to apply these general force fields to an arbitrary drug-like molecule, functionality for assignment of atom types, parameters, and partial atomic charges is required. In the present article, algorithms for the assignment of parameters and charges for the CHARMM General Force Field (CGenFF) are presented. These algorithms rely on the existing parameters and charges that were determined as part of the parametrization of the force field. Bonded parameters are assigned based on the similarity between the atom types that define said parameters, while charges are determined using an extended bond-charge increment scheme. Charge increments were optimized to reproduce the charges on model compounds that were part of the parametrization of the force field. A "penalty score" is returned for every bonded parameter and charge, allowing the user to quickly and conveniently assess the quality of the force field representation of different parts of the compound of interest. Case studies are presented to clarify the functioning of the algorithms and the significance of their output data.</td>
</tr>
<tr id="bib_Vanommeslaeghe2012a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Vanommeslaeghe2012a,
  author = {Vanommeslaeghe, K and Raman, Prabhu E and AD, MacKerell,},
  title = {Automation of the CHARMM General Force Field CGenFF I: bond perception and atom typing},
  journal = {Journal of Chemical Information and Modeling},
  year = {2012},
  volume = {52},
  number = {12},
  pages = {3155 - 68},
  note = {NULL},
  doi = {https://doi.org/10.1021/ci3003649}
}
</pre></td>
</tr>
<tr id="Vanommeslaeghe2012b" class="entry">
	<td>Vanommeslaeghe, K., Raman, P.E. and MacKerell,, AD</td>
	<td>Automation of the CHARMM General Force Field (CGenFF) II: assignment of bonded parameters and partial atomic charges. <p class="infolinks">[<a href="javascript:toggleInfo('Vanommeslaeghe2012b','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Vanommeslaeghe2012b','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Journal of Chemical Information and Modeling<br/>Vol. 52(12), pp. 3155 - 68&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ci3003649">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Vanommeslaeghe2012b" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Molecular mechanics force fields are widely used in computer-aided drug design for the study of drug candidates interacting with biological systems. In these simulations, the biological part is typically represented by a specialized biomolecular force field, while the drug is represented by a matching general (organic) force field. In order to apply these general force fields to an arbitrary drug-like molecule, functionality for assignment of atom types, parameters, and partial atomic charges is required. In the present article, algorithms for the assignment of parameters and charges for the CHARMM General Force Field (CGenFF) are presented. These algorithms rely on the existing parameters and charges that were determined as part of the parametrization of the force field. Bonded parameters are assigned based on the similarity between the atom types that define said parameters, while charges are determined using an extended bond-charge increment scheme. Charge increments were optimized to reproduce the charges on model compounds that were part of the parametrization of the force field. A "penalty score" is returned for every bonded parameter and charge, allowing the user to quickly and conveniently assess the quality of the force field representation of different parts of the compound of interest. Case studies are presented to clarify the functioning of the algorithms and the significance of their output data.</td>
</tr>
<tr id="bib_Vanommeslaeghe2012b" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Vanommeslaeghe2012b,
  author = {Vanommeslaeghe, K and Raman, Prabhu E and AD, MacKerell,},
  title = {Automation of the CHARMM General Force Field (CGenFF) II: assignment of bonded parameters and partial atomic charges.},
  journal = {Journal of Chemical Information and Modeling},
  year = {2012},
  volume = {52},
  number = {12},
  pages = {3155 - 68},
  note = {NULL},
  doi = {https://doi.org/10.1021/ci3003649}
}
</pre></td>
</tr>
<tr id="Varga2017" class="entry">
	<td>Varga, M.J., Dzierlenga, M.W. and Schwartz, S.D.</td>
	<td>Structurally linked dynamics in lactate dehydrogenases of evolutionarily distinct species <p class="infolinks">[<a href="javascript:toggleInfo('Varga2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Biochemistry<br/>Vol. 56, pp. 2488 - 2496&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Varga2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Varga2017,
  author = {Varga, Matthew J and Dzierlenga, Michael W and Schwartz, Steven D},
  title = {Structurally linked dynamics in lactate dehydrogenases of evolutionarily distinct species},
  journal = {Biochemistry},
  year = {2017},
  volume = {56},
  pages = {2488 - 2496}
}
</pre></td>
</tr>
<tr id="Varga2016" class="entry">
	<td>Varga, M.J. and Schwartz, S.D.</td>
	<td>Enzymatic kinetic isotope effects from first principles path sampling calculations <p class="infolinks">[<a href="javascript:toggleInfo('Varga2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Varga2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Journal of Chemical Theory and Computation<br/>Vol. 12(4), pp. 2047 - 2054&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.jctc.5b01169">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Varga2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In this study, we develop and test a method to determine the rate of particle transfer and kinetic isotope effects in enzymatic reactions, specifically yeast alcohol dehydrogenase (YADH), from first-principles. Transition path sampling (TPS) and normal mode centroid dynamics (CMD) are used to simulate these enzymatic reactions without knowledge of their reaction coordinates and with the inclusion of quantum effects, such as zero-point energy and tunneling, on the transferring particle. Though previous studies have used TPS to calculate reaction rate constants in various model and real systems, it has not been applied to a system as large as YADH. The calculated primary H/D kinetic isotope effect agrees with previously reported experimental results, within experimental error. The kinetic isotope effects calculated with this method correspond to the kinetic isotope effect of the transfer event itself. The results reported here show that the kinetic isotope effects calculated from first-principles, purely fo...</td>
</tr>
<tr id="bib_Varga2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Varga2016,
  author = {Varga, Matthew J and Schwartz, Steven D},
  title = {Enzymatic kinetic isotope effects from first principles path sampling calculations},
  journal = {Journal of Chemical Theory and Computation},
  year = {2016},
  volume = {12},
  number = {4},
  pages = {2047 - 2054},
  note = {NULL},
  doi = {https://doi.org/10.1021/acs.jctc.5b01169}
}
</pre></td>
</tr>
<tr id="Vasilyev1994" class="entry">
	<td>Vasilyev, V.V..</td>
	<td>Tetrahedral intermediate formation in the acylation step of acetylcholinesterases. A combined quantum chemical and molecular mechanical model <p class="infolinks">[<a href="javascript:toggleInfo('Vasilyev1994','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Vasilyev1994','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td>Journal of Molecular Structure: THEOCHEM<br/>Vol. 304(2), pp. 129 - 141&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S0166-1280(96)80005-4">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Vasilyev1994" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A combined quantum chemical (PM3) and molecular mechanical (OPLS) study of the formation of the tetrahedral intermediate in the active site of acetylcholinesterase enzyme is presented. The influence of the protein environment on the reaction flow was investigated. It is shown that there are strong hydrogen bonds between the elements of the catalytic triad (Ser200-His440-Glu327) in the free enzyme. This fact, together with the unique form of the electrostatic potential in the active site of the enzyme, favour the activation of the serine residue for the subsequent attack of substrate. The interaction of the substrate with the ``oxyanion hole'' of the enzyme was demonstrated to lower the barrier height of the tetrahedral intermediate formation and to stabilize it substantially. The inhibition of the enzyme by the phospho-compound P(F)(O)(NH2)(CH3) was also investigated. The origin of the enzyme catalysis is discussed. The electrostatic interactions were concluded to be the principal cause of enzyme power.</td>
</tr>
<tr id="bib_Vasilyev1994" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Vasilyev1994,
  author = {Vasilyev, V.V.},
  title = {Tetrahedral intermediate formation in the acylation step of acetylcholinesterases. A combined quantum chemical and molecular mechanical model},
  journal = {Journal of Molecular Structure: THEOCHEM},
  year = {1994},
  volume = {304},
  number = {2},
  pages = {129 - 141},
  doi = {https://doi.org/10.1016/S0166-1280(96)80005-4}
}
</pre></td>
</tr>
<tr id="Verlet1967" class="entry">
	<td>Verlet, L.</td>
	<td>Computer "experiments" on classical fluids. I. Thermodynamical properties of Lennard-Jones molecules <p class="infolinks">[<a href="javascript:toggleInfo('Verlet1967','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Verlet1967','bibtex')">BibTeX</a>]</p></td>
	<td>1967</td>
	<td>Physical Review<br/>Vol. 159(1), pp. 98 - 103&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1103/PhysRev.159.98">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Verlet1967" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The equation of motion of a system of 864 particles interacting through a Lennard-Jones potential has been integrated for various values of the temperature and density, relative, generally, to a fluid state. The equilibrium properties have been calculated and are shown to agree very well with the corresponding properties of argon. It is concluded that, to a good approximation, the equilibrium state of argon can be described through a two-body potential.</td>
</tr>
<tr id="bib_Verlet1967" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Verlet1967,
  author = {Verlet, Loup},
  title = {Computer "experiments" on classical fluids. I. Thermodynamical properties of Lennard-Jones molecules},
  journal = {Physical Review},
  year = {1967},
  volume = {159},
  number = {1},
  pages = {98 - 103},
  doi = {https://doi.org/10.1103/PhysRev.159.98}
}
</pre></td>
</tr>
<tr id="Voelter-Mahlknecht2003" class="entry">
	<td>Voelter-Mahlknecht, S. and Mahlknecht, U.</td>
	<td>Cloning and structural characterization of the human histone deacetylase 6 gene. <p class="infolinks">[<a href="javascript:toggleInfo('Voelter-Mahlknecht2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Voelter-Mahlknecht2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>International journal of molecular medicine<br/>Vol. 12, pp. 87-93&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.3892/ijmm.12.1.87">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Voelter-Mahlknecht2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Histone deacetylases (HDACs) play a central role in the modification of chromatin structure and thus in the regulation of transcription and cellular differentiation. Based on structural and functional similarities, mammalian histone deacetylases may be grouped into three categories, class I HDACs, which are homologs of the yeast histone deacetylase RPD3, class II HDACs, which share a significant degree of homology with the yeast histone deacetylase HDA1 and class III HDACs which are closely related to the yeast SIR2 protein. We have isolated and characterized the human HDAC6 genomic sequence, which spans a region of 21,923 bp and which has one single genomic locus. Determination of the exon-intron splice junctions established that HDAC6 is encoded by 28 exons ranging in size from 41 bp (exon 5) to 677 bp (exon 24). Characterization of the 5' flanking genomic region, which precedes the HDAC6 open reading frame, revealed a TATA- and CCAAT-boxless promoter that contains a 1 kb CpG island. The 3,648 bp human HDAC6 mRNA encodes a 1,215 aa protein with a predictive molecular weight of 131.4 kDa. Fluorescence in situ hybridization analysis localized the human HDAC6 gene to the subband border of chromosome Xp11.22-23, a region which is characterized by frequent gains and losses of chromosomal material in several types of cancer and neurological disorders.</td>
</tr>
<tr id="bib_Voelter-Mahlknecht2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Voelter-Mahlknecht2003,
  author = {Voelter-Mahlknecht, Susanne and Mahlknecht, Ulrich},
  title = {Cloning and structural characterization of the human histone deacetylase 6 gene.},
  journal = {International journal of molecular medicine},
  year = {2003},
  volume = {12},
  pages = {87--93},
  note = {AP2A1 = alpha a},
  doi = {https://doi.org/10.3892/ijmm.12.1.87}
}
</pre></td>
</tr>
<tr id="Voter2007" class="entry">
	<td>Voter, AF.</td>
	<td>Introduction to the kinetic Monte Carlo method <p class="infolinks">[<a href="javascript:toggleInfo('Voter2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Voter2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td><br/>Vol. 235Radiation Effects in Solids, pp. 1&nbsp;</td>
	<td>inbook</td>
	<td><a href="https://doi.org/10.1007/978-1-4020-5295-8_1">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Voter2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Monte Carlo refers to a broad class of algorithms that solve problems through the use of random numbers. They first emerged in the late 1940's and 1950's as electronic computers came into use [1], and the name means just what it sounds like, whimsically referring to the random</td>
</tr>
<tr id="bib_Voter2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inbook{Voter2007,
  author = {Voter, AF},
  title = {Introduction to the kinetic Monte Carlo method},
  booktitle = {Radiation Effects in Solids},
  publisher = {springer},
  year = {2007},
  volume = {235},
  pages = {1},
  doi = {https://doi.org/10.1007/978-1-4020-5295-8_1}
}
</pre></td>
</tr>
<tr id="Wakeham2003" class="entry">
	<td>Wakeham, D.E., Chen, C.-Y., Greene, B., Hwang, P.K. and Brodsky, F.M.</td>
	<td>Clathrin self-assembly involves coordinated weak interactions favorable for cellular regulation. <p class="infolinks">[<a href="javascript:toggleInfo('Wakeham2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wakeham2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>The EMBO journal<br/>Vol. 22, pp. 4980-4990&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/emboj/cdg511">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wakeham2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The clathrin triskelion self-assembles into a polyhedral coat surrounding membrane vesicles that sort receptor cargo to the endocytic pathway. A triskelion comprises three clathrin heavy chains joined at their C-termini, extending into proximal and distal leg segments ending in a globular N-terminal domain. In the clathrin coat, leg segments entwine into parallel and anti-parallel interactions. Here we define the contributions of segmental interactions to the clathrin assembly reaction and measure the strength of their interactions. Proximal and distal leg segments were found to lack sufficient affinity to form stable homo- or heterodimers under assembly conditions. However, chimeric constructs of proximal or distal leg segments, trimerized by replacement of the clathrin trimerization domain with that of the invariant chain protein, were able to self-assemble in reversible reactions. Thus clathrin assembly occurs because weak leg segment affinities are coordinated through trimerization, sharing a dependence on multiple weak interactions with other biopolymers. Such polymerization is sensitive to small environmental changes and is therefore compatible with cellular regulation of assembly, disassembly and curvature during formation of clathrin-coated vesicles.</td>
</tr>
<tr id="bib_Wakeham2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wakeham2003,
  author = {Wakeham, Diane E and Chen, Chih-Ying and Greene, Barrie and Hwang, Peter K and Brodsky, Frances M},
  title = {Clathrin self-assembly involves coordinated weak interactions favorable for cellular regulation.},
  journal = {The EMBO journal},
  year = {2003},
  volume = {22},
  pages = {4980--4990},
  doi = {https://doi.org/10.1093/emboj/cdg511}
}
</pre></td>
</tr>
<tr id="Wakeham2005" class="entry">
	<td>Wakeham, D.E., Laurent, Abi-Rached,., Towler, M.C., Wilbur, J.D., Parham, P. and Brodsky, F.M.</td>
	<td>Clathrin heavy and light chain isoforms originated by independent mechanisms of gene duplication during chordate evolution. <p class="infolinks">[<a href="javascript:toggleInfo('Wakeham2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wakeham2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Proceedings of the National Academy of Sciences<br/>Vol. 102(20), pp. 7209 - 7214&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.0502058102">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wakeham2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In humans, there are two isoforms each of clathrin heavy chain (CHC17 and CHC22) and light chain (LCa and LCb) subunits, all encoded by separate genes. CHC17 forms the ubiquitous clathrin-coated vesicles that mediate membrane traffic. CHC22 is implicated in specialized membrane organization in skeletal muscle. CHC17 is bound and regulated by LCa and LCb, whereas CHC22 does not functionally interact with either light chain. The imbalanced interactions between clathrin subunit isoforms suggest a distinct evolutionary history for each isoform pair. Phylogenetic and sequence analysis placed both heavy and light chain gene duplications during chordate evolution, 510-600 million years ago. Genes encoding CHC22 orthologues were found in several vertebrate species, with only a pseudogene present in mice. Multiple paralogons surrounding the CHC genes (CLTC and CLTD) were identified, evidence that genomic or large-scale gene duplication produced the two CHC isoforms. In contrast, clathrin light chain genes (CLTA and CLTB) apparently arose by localized duplication, within 1-11 million years of CHC gene duplication. Analysis of sequence divergence patterns suggested that structural features of the CHCs were maintained after gene duplication, but new interactions with regulatory proteins evolved for the CHC22 isoform. Thus, independent mechanisms of gene duplication expanded clathrin functions, concomitant with development of neuromuscular sophistication in chordates.</td>
</tr>
<tr id="bib_Wakeham2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wakeham2005,
  author = {Wakeham, Diane E and Laurent, Abi-Rached, and Towler, Mhairi C and Wilbur, Jeremy D and Parham, Peter and Brodsky, Frances M},
  title = {Clathrin heavy and light chain isoforms originated by independent mechanisms of gene duplication during chordate evolution.},
  journal = {Proceedings of the National Academy of Sciences},
  year = {2005},
  volume = {102},
  number = {20},
  pages = {7209 - 7214},
  doi = {https://doi.org/10.1073/pnas.0502058102}
}
</pre></td>
</tr>
<tr id="Walsh0" class="entry">
	<td>Walsh, C.T..</td>
	<td>Enzymes in the D-Alanine Branch of Bacterial Wall Peptidoglycan Assembly <p class="infolinks">[<a href="javascript:toggleInfo('Walsh0','bibtex')">BibTeX</a>]</p></td>
	<td></td>
	<td>Journal of Biological Chemistry<br/>Vol. 264, pp. 2393 - 2396&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Walsh0" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Walsh0,
  author = {Walsh, C.T.},
  title = {Enzymes in the D-Alanine Branch of Bacterial Wall Peptidoglycan Assembly},
  journal = {Journal of Biological Chemistry},
  volume = {264},
  pages = {2393 - 2396}
}
</pre></td>
</tr>
<tr id="Walsh1989" class="entry">
	<td>Walsh, CT.</td>
	<td>Enzymes in the D-alanine branch of bacterial cell wall peptidoglycan assembly. <p class="infolinks">[<a href="javascript:toggleInfo('Walsh1989','bibtex')">BibTeX</a>]</p></td>
	<td>1989</td>
	<td>Journal of Biological Chemistry<br/>Vol. 264(5), pp. 2393 - 6&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Walsh1989" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Walsh1989,
  author = {Walsh, CT},
  title = {Enzymes in the D-alanine branch of bacterial cell wall peptidoglycan assembly.},
  journal = {Journal of Biological Chemistry},
  year = {1989},
  volume = {264},
  number = {5},
  pages = {2393 - 6}
}
</pre></td>
</tr>
<tr id="Walsh2003" class="entry">
	<td>Walsh, C.</td>
	<td>Antibiotics: Actions, Origins, Resistance <p class="infolinks">[<a href="javascript:toggleInfo('Walsh2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>&nbsp;</td>
	<td>book</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Walsh2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{Walsh2003,
  author = {Walsh, Christopher},
  title = {Antibiotics: Actions, Origins, Resistance},
  publisher = {ASM Press},
  year = {2003}
}
</pre></td>
</tr>
<tr id="Wan2015" class="entry">
	<td>Wan, C., Borgeson, B., Phanse, S., Tu, F., Drew, K., Clark, G., Xiong, X., Kagan, O., Kwan, J., Bezginov, A., Chessman, K., Pal, S., Cromar, G., Papoulas, O., Ni, Z., Boutz, D.R., Stoilova, S., Havugimana, P.C., Guo, X., Malty, R.H., Sarov, M., Greenblatt, J., Babu, M., Derry, B.W., Tillier, E.R., Wallingford, J.B., Parkinson, J., Marcotte, E.M., Emili, A., Tillier, E.R., Wallingford, J.B., Parkinson, J., Marcotte, E.M. and Emili, A.</td>
	<td>Panorama of ancient metazoan macromolecular complexes <p class="infolinks">[<a href="javascript:toggleInfo('Wan2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wan2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Nature<br/>Vol. 525(7569), pp. 339 - 44&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nature14877">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wan2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Macromolecular complexes are essential to conserved biological processes, but their prevalence across animals is unclear. By combining extensive biochemical fractionation with quantitative mass spectrometry, here we directly examined the composition of soluble multiprotein complexes among diverse metazoan models. Using an integrative approach, we generated a draft conservation map consisting of more than one million putative high-confidence co-complex interactions for species with fully sequenced genomes that encompasses functional modules present broadly across all extant animals. Clustering reveals a spectrum of conservation, ranging from ancient eukaryotic assemblies that have probably served cellular housekeeping roles for at least one billion years, ancestral complexes that have accrued contemporary components, and rarer metazoan innovations linked to multicellularity. We validated these projections by independent co-fractionation experiments in evolutionarily distant species, affinity purification and functional analyses. The comprehensiveness, centrality and modularity of these reconstructed interactomes reflect their fundamental mechanistic importance and adaptive value to animal cell systems.</td>
</tr>
<tr id="bib_Wan2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wan2015,
  author = {Wan, Cuihong and Borgeson, Blake and Phanse, Sadhna and Tu, Fan and Drew, Kevin and Clark, Greg and Xiong, Xuejian and Kagan, Olga and Kwan, Julian and Bezginov, Alexandr and Chessman, Kyle and Pal, Swati and Cromar, Graham and Papoulas, Ophelia and Ni, Zuyao and Boutz, Daniel R and Stoilova, Snejana and Havugimana, Pierre C and Guo, Xinghua and Malty, Ramy H and Sarov, Mihail and Greenblatt, Jack and Babu, Mohan and Derry, Brent W and Tillier, Elisabeth R and Wallingford, John B and Parkinson, John and Marcotte, Edward M and Emili, Andrew and Tillier, Elisabeth R and Wallingford, John B and Parkinson, John and Marcotte, Edward M and Emili, Andrew},
  title = {Panorama of ancient metazoan macromolecular complexes},
  journal = {Nature},
  year = {2015},
  volume = {525},
  number = {7569},
  pages = {339 - 44},
  doi = {https://doi.org/10.1038/nature14877}
}
</pre></td>
</tr>
<tr id="Wand1997" class="entry">
	<td>Wand, P.M..</td>
	<td>Data-based choice of histogram bin width <p class="infolinks">[<a href="javascript:toggleInfo('Wand1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>The American Statistician<br/>Vol. 51(1), pp. 59 - 64&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Wand1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wand1997,
  author = {Wand, P.M.},
  title = {Data-based choice of histogram bin width},
  journal = {The American Statistician},
  year = {1997},
  volume = {51},
  number = {1},
  pages = {59 - 64}
}
</pre></td>
</tr>
<tr id="Wang2017" class="entry">
	<td>Wang, B., Huang, L., Zhu, Y., Kundaje, A., Batzoglou, S. and Goldenberg, A.</td>
	<td>Vicus: Exploiting local structures to improve network-based analysis of biological data. <p class="infolinks">[<a href="javascript:toggleInfo('Wang2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wang2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>PLoS computational biology<br/>Vol. 13, pp. e1005621&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pcbi.1005621">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wang2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Biological networks entail important topological features and patterns critical to understanding interactions within complicated biological systems. Despite a great progress in understanding their structure, much more can be done to improve our inference and network analysis. Spectral methods play a key role in many network-based applications. Fundamental to spectral methods is the Laplacian, a matrix that captures the global structure of the network. Unfortunately, the Laplacian does not take into account intricacies of the network's local structure and is sensitive to noise in the network. These two properties are fundamental to biological networks and cannot be ignored. We propose an alternative matrix Vicus. The Vicus matrix captures the local neighborhood structure of the network and thus is more effective at modeling biological interactions. We demonstrate the advantages of Vicus in the context of spectral methods by extensive empirical benchmarking on tasks such as single cell dimensionality reduction, protein module discovery and ranking genes for cancer subtyping. Our experiments show that using Vicus, spectral methods result in more accurate and robust performance in all of these tasks.</td>
</tr>
<tr id="bib_Wang2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wang2017,
  author = {Wang, Bo and Huang, Lin and Zhu, Yuke and Kundaje, Anshul and Batzoglou, Serafim and Goldenberg, Anna},
  title = {Vicus: Exploiting local structures to improve network-based analysis of biological data.},
  journal = {PLoS computational biology},
  year = {2017},
  volume = {13},
  pages = {e1005621},
  doi = {https://doi.org/10.1371/journal.pcbi.1005621}
}
</pre></td>
</tr>
<tr id="Wang2016a" class="entry">
	<td>Wang, L., Johnson, A., Hanna, M. and Audhya, A.</td>
	<td>Eps15 membrane-binding and -bending activity acts redundantly with Fcho1 during clathrin-mediated endocytosis. <p class="infolinks">[<a href="javascript:toggleInfo('Wang2016a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wang2016a','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Molecular biology of the cell<br/>Vol. 27, pp. 2675-2687&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1091/mbc.E16-03-0151">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wang2016a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin coat assembly on membranes requires cytosolic adaptors and accessory proteins, which bridge triskeleons with the lipid bilayer and stabilize lattice architecture throughout the process of vesicle formation. In Caenorhabditis elegans, the prototypical AP-2 adaptor complex, which is activated by the accessory factor Fcho1 at the plasma membrane, is dispensable during embryogenesis, enabling us to define alternative mechanisms that facilitate clathrin-mediated endocytosis. Here we uncover a synthetic genetic interaction between C. elegans Fcho1 (FCHO-1) and Eps15 (EHS-1), suggesting that they function in a parallel and potentially redundant manner. Consistent with this idea, we find that the FCHO-1 EFC/F-BAR domain and the EHS-1 EH domains exhibit highly similar membrane-binding and -bending characteristics in vitro. Furthermore, we demonstrate a critical role for EHS-1 when FCHO-1 membrane-binding and -bending activity is specifically eliminated in vivo. Taken together, our data highlight Eps15 as an important membrane-remodeling factor, which acts in a partially redundant manner with Fcho proteins during the earliest stages of clathrin-mediated endocytosis.</td>
</tr>
<tr id="bib_Wang2016a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wang2016a,
  author = {Wang, Lei and Johnson, Adam and Hanna, Michael and Audhya, Anjon},
  title = {Eps15 membrane-binding and -bending activity acts redundantly with Fcho1 during clathrin-mediated endocytosis.},
  journal = {Molecular biology of the cell},
  year = {2016},
  volume = {27},
  pages = {2675--2687},
  note = {eps15 and itsn1 pick up the slack of fcho1/2},
  doi = {https://doi.org/10.1091/mbc.E16-03-0151}
}
</pre></td>
</tr>
<tr id="Wang1995" class="entry">
	<td>Wang, L.-H., S&uuml;dhof, T.C. and Anderson, R.G.</td>
	<td>The appendage domain of &alpha;-adaptin is a high affinity binding site for dynamin <p class="infolinks">[<a href="javascript:toggleInfo('Wang1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>Journal of Biological Chemistry<br/>Vol. 270(17), pp. 10079-10083&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Wang1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wang1995,
  author = {Wang, Li-Hsien and S&uuml;dhof, Thomas C and Anderson, Richard GW},
  title = {The appendage domain of &alpha;-adaptin is a high affinity binding site for dynamin},
  journal = {Journal of Biological Chemistry},
  publisher = {ASBMB},
  year = {1995},
  volume = {270},
  number = {17},
  pages = {10079--10083},
  note = {Bovine brains. Indicates that amphiphysin binds to the ear of alpha-AP2, but it is unclear. Uses fusion proteins with the subunits attached. ap2a? - amph, dyna, ap180. dissociation constant for ap2a2 - dyna}
}
</pre></td>
</tr>
<tr id="Wang2006" class="entry">
	<td>Wang, Q. and Hammes-Schiffer, S.</td>
	<td>Hybrid quantum/classical path integral approach for simulation of hydrogen transfer reactions in enzymes. <p class="infolinks">[<a href="javascript:toggleInfo('Wang2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wang2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>The Journal of chemical physics<br/>Vol. 125(18), pp. 184102 - 184102&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.2362823">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wang2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A hybrid quantum/classical path integral Monte Carlo (QC-PIMC) method for calculating the quantum free energy barrier for hydrogen transfer reactions in condensed phases is presented. In this approach, the classical potential of mean force along a collective reaction coordinate is calculated using umbrella sampling techniques in conjunction with molecular dynamics trajectories propagated according to a mapping potential. The quantum contribution is determined for each configuration along the classical trajectory with path integral Monte Carlo calculations in which the beads move according to an effective mapping potential. This type of path integral calculation does not utilize the centroid constraint and can lead to more efficient sampling of the relevant region of conformational space than free-particle path integral sampling. The QC-PIMC method is computationally practical for large systems because the path integral sampling for the quantum nuclei is performed separately from the classical molecular dynamics sampling of the entire system. The utility of the QC-PIMC method is illustrated by an application to hydride transfer in the enzyme dihydrofolate reductase. A comparison of this method to the quantized classical path and grid-based methods for this system is presented.</td>
</tr>
<tr id="bib_Wang2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wang2006,
  author = {Wang, Qian and Hammes-Schiffer, Sharon},
  title = {Hybrid quantum/classical path integral approach for simulation of hydrogen transfer reactions in enzymes.},
  journal = {The Journal of chemical physics},
  year = {2006},
  volume = {125},
  number = {18},
  pages = {184102 - 184102},
  doi = {https://doi.org/10.1063/1.2362823}
}
</pre></td>
</tr>
<tr id="Wang2012" class="entry">
	<td>Wang, X., Wei, X., Thijssen, B., Das, J., Lipkin, S.M. and Yu, H.</td>
	<td>Three-dimensional reconstruction of protein networks provides insight into human genetic disease. <p class="infolinks">[<a href="javascript:toggleInfo('Wang2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wang2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Nature biotechnology<br/>Vol. 30, pp. 159-164&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nbt.2106">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wang2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: To better understand the molecular mechanisms and genetic basis of human disease, we systematically examine relationships between 3,949 genes, 62,663 mutations and 3,453 associated disorders by generating a three-dimensional, structurally resolved human interactome. This network consists of 4,222 high-quality binary protein-protein interactions with their atomic-resolution interfaces. We find that in-frame mutations (missense point mutations and in-frame insertions and deletions) are enriched on the interaction interfaces of proteins associated with the corresponding disorders, and that the disease specificity for different mutations of the same gene can be explained by their location within an interface. We also predict 292 candidate genes for 694 unknown disease-to-gene associations with proposed molecular mechanism hypotheses. This work indicates that knowledge of how in-frame disease mutations alter specific interactions is critical to understanding pathogenesis. Structurally resolved interaction networks should be valuable tools for interpreting the wealth of data being generated by large-scale structural genomics and disease association studies.</td>
</tr>
<tr id="bib_Wang2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wang2012,
  author = {Wang, Xiujuan and Wei, Xiaomu and Thijssen, Bram and Das, Jishnu and Lipkin, Steven M and Yu, Haiyuan},
  title = {Three-dimensional reconstruction of protein networks provides insight into human genetic disease.},
  journal = {Nature biotechnology},
  year = {2012},
  volume = {30},
  pages = {159--164},
  doi = {https://doi.org/10.1038/nbt.2106}
}
</pre></td>
</tr>
<tr id="Wang2011" class="entry">
	<td>Wang, Y., Harrison, CB., Schulten, K. and McCammon, Andrew, J</td>
	<td>Implementation of accelerated molecular dynamics in NAMD <p class="infolinks">[<a href="javascript:toggleInfo('Wang2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Computational Science and Discovery<br/>Vol. 4, pp. 15002 - 15002&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Wang2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wang2011,
  author = {Wang, Y and Harrison, CB and Schulten, K and J, McCammon, Andrew},
  title = {Implementation of accelerated molecular dynamics in NAMD},
  journal = {Computational Science and Discovery},
  year = {2011},
  volume = {4},
  pages = {15002 - 15002}
}
</pre></td>
</tr>
<tr id="Wang2016" class="entry">
	<td>Wang, Z., Chang, E.P. and Schramm, V.L.</td>
	<td>Triple isotope effects support concerted hydride and proton transfer and promoting vibrations in human heart lactate dehydrogenase <p class="infolinks">[<a href="javascript:toggleInfo('Wang2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wang2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Journal of the American Chemical Society<br/>Vol. 138(45), pp. 15004 - 15010&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jacs.6b09049">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wang2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Transition path sampling simulations have proposed that human heart lactate dehydrogenase (LDH) employs protein promoting vibrations (PPVs) on the femtosecond (fs) to picosecond (ps) time scale to promote crossing of the chemical barrier. This chemical barrier involves both hydride and proton transfers to pyruvate to form l-lactate, using reduced nicotinamide adenine dinucleotide (NADH) as the cofactor. Here we report experimental evidence from three types of isotope effect experiments that support coupling of the promoting vibrations to barrier crossing and the coincidence of hydride and proton transfer. We prepared the native (light) LDH and a heavy LDH labeled with 13C, 15N, and nonexchangeable 2H (D) to perturb the predicted PPVs. Heavy LDH has slowed chemistry in single turnover experiments, supporting a contribution of PPVs to transition state formation. Both the [4-2H]NADH (NADD) kinetic isotope effect and the D2O solvent isotope effect were increased in dual-label experiments combining both NADD a...</td>
</tr>
<tr id="bib_Wang2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wang2016,
  author = {Wang, Zhen and Chang, Eric P and Schramm, Vern L},
  title = {Triple isotope effects support concerted hydride and proton transfer and promoting vibrations in human heart lactate dehydrogenase},
  journal = {Journal of the American Chemical Society},
  year = {2016},
  volume = {138},
  number = {45},
  pages = {15004 - 15010},
  note = {NULL},
  doi = {https://doi.org/10.1021/jacs.6b09049}
}
</pre></td>
</tr>
<tr id="Warfield2017" class="entry">
	<td>Warfield, B.M. and Anderson, P.C.</td>
	<td>Molecular simulations and Markov state modeling reveal the structural diversity and dynamics of a theophylline-binding RNA aptamer in its unbound state <p class="infolinks">[<a href="javascript:toggleInfo('Warfield2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Warfield2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>PLoS One(4), pp. e0176229&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pone.0176229">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Warfield2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: RNA aptamers are oligonucleotides that bind with high specificity and affinity to target ligands. In the absence of bound ligand, secondary structures of RNA aptamers are generally stable, but single-stranded and loop regions, including ligand binding sites, lack defined structures and exist as ensembles of conformations. For example, the well-characterized theophylline-binding aptamer forms a highly stable binding site when bound to theophylline, but the binding site is unstable and disordered when theophylline is absent. Experimental methods have not revealed at atomic resolution the conformations that the theophylline aptamer explores in its unbound state. Consequently, in the present study we applied 21 microseconds of molecular dynamics simulations to structurally characterize the ensemble of conformations that the aptamer adopts in the absence of theophylline. Moreover, we apply Markov state modeling to predict the kinetics of transitions between unbound conformational states. Our simulation results agree with experimental observations that the theophylline binding site is found in many distinct binding-incompetent states and show that these states lack a binding pocket that can accommodate theophylline. The binding-incompetent states interconvert with binding-competent states through structural rearrangement of the binding site on the nanosecond to microsecond timescale. Moreover, we have simulated the complete theophylline binding pathway. Our binding simulations supplement prior experimental observations of slow theophylline binding kinetics by showing that the binding site must undergo a large conformational rearrangement after the aptamer and theophylline form an initial complex, most notably, a major rearrangement of the C27 base from a buried to solvent-exposed orientation. Theophylline appears to bind by a combination of conformational selection and induced fit mechanisms. Finally, our modeling indicates that when Mg2+ ions are present the population of binding-competent aptamer states increases more than twofold. This population change, rather than direct interactions between Mg2+ and theophylline, accounts for altered theophylline binding kinetics.</td>
</tr>
<tr id="bib_Warfield2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Warfield2017,
  author = {Warfield, Becka M and Anderson, Peter C},
  title = {Molecular simulations and Markov state modeling reveal the structural diversity and dynamics of a theophylline-binding RNA aptamer in its unbound state},
  journal = {PLoS One},
  year = {2017},
  number = {4},
  pages = {e0176229},
  doi = {https://doi.org/10.1371/journal.pone.0176229}
}
</pre></td>
</tr>
<tr id="Warshel1998" class="entry">
	<td>Warshel, A.</td>
	<td>Electrostatic origin of the catalytic power of enzymes and the role of preorganized active sites <p class="infolinks">[<a href="javascript:toggleInfo('Warshel1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Warshel1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Journal of Biological Chemistry<br/>Vol. 273(42), pp. 27035 - 27038&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.273.42.27035">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Warshel1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: 10.1074/jbc.273.42.27035</td>
</tr>
<tr id="bib_Warshel1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Warshel1998,
  author = {Warshel, Arieh},
  title = {Electrostatic origin of the catalytic power of enzymes and the role of preorganized active sites},
  journal = {Journal of Biological Chemistry},
  year = {1998},
  volume = {273},
  number = {42},
  pages = {27035 - 27038},
  doi = {https://doi.org/10.1074/jbc.273.42.27035}
}
</pre></td>
</tr>
<tr id="Warshel1986" class="entry">
	<td>Warshel, A. and Hwang, J.K..</td>
	<td>Simulation of the dynamics of electron transfer reactions in polar solvents: Semiclassical trajectories and dispersed polaron approaches <p class="infolinks">[<a href="javascript:toggleInfo('Warshel1986','bibtex')">BibTeX</a>]</p></td>
	<td>1986</td>
	<td>The Journal of Chemical Physics<br/>Vol. 84(9), pp. 4938 - 4957&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.449981">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Warshel1986" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Warshel1986,
  author = {Warshel, Arieh and Hwang, J.K.},
  title = {Simulation of the dynamics of electron transfer reactions in polar solvents: Semiclassical trajectories and dispersed polaron approaches},
  journal = {The Journal of Chemical Physics},
  year = {1986},
  volume = {84},
  number = {9},
  pages = {4938 - 4957},
  doi = {https://doi.org/10.1063/1.449981}
}
</pre></td>
</tr>
<tr id="Warshel1976" class="entry">
	<td>Warshel, A. and Levitt, M.</td>
	<td>Theoretical studies of enzymic reactions: dielectric, electrostatic and steric stabilization of the carbonium ion in the reaction of lysozyme. <p class="infolinks">[<a href="javascript:toggleInfo('Warshel1976','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Warshel1976','bibtex')">BibTeX</a>]</p></td>
	<td>1976</td>
	<td>Journal of Molecular Biology<br/>Vol. 103(2), pp. 227 - 249&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/0022-2836(76)90311-9">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Warshel1976" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A general method for detailed study of enzymic reactions is presented. The method considers the complete enzyme-substrate complex together with the surrounding solvent and evaluates all the different quantum mechanical and classical energy factors that can affect the reaction pathway. These factors include the quantum mechanical energies associated with bond cleavage and charge redistribution of the substrate and the classical energies of steric and electrostatic interactions between the substrate and the enzyme. The electrostatic polarization of the enzyme atoms and the orientation of the dipoles of the surrounding water molecules is simulated by a microscopic dielectric model. The solvation energy resulting from this polarization is considerable and must be included in any realistic calculation of chemical reactions involving anything more than an isolated molecule in vacuo. Without it, acidic groups can never become ionized and the charge distribution on the substrate will not be reasonable. The same dielectric model can also be used to study the reaction of the substrate in solution. In this way the reaction in solution can be compared with the enzymic reaction. In this paper we study the stability of the carbonium ion intermediate formed in the cleavage of a glycosidic bond by lysozyme. It is found that electrostatic stabilization is an important factor in increasing the rate of the reaction step that leads to the formation of the carbonium ion intermediate. Steric factors, such as the strain of the substrate on binding to lysozyme, do not seem to contribute significantly.</td>
</tr>
<tr id="bib_Warshel1976" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Warshel1976,
  author = {Warshel, A and Levitt, M},
  title = {Theoretical studies of enzymic reactions: dielectric, electrostatic and steric stabilization of the carbonium ion in the reaction of lysozyme.},
  journal = {Journal of Molecular Biology},
  year = {1976},
  volume = {103},
  number = {2},
  pages = {227 - 249},
  doi = {https://doi.org/10.1016/0022-2836(76)90311-9}
}
</pre></td>
</tr>
<tr id="Waszkowycz1991" class="entry">
	<td>Waszkowycz, B., Hillier, I.H., Gensmantel, N. and Payling, D.W.</td>
	<td>Combined quantum mechanical--molecular mechanical study of catalysis by the enzyme phospholipase A 2 : an investigation of the potential energy surface for amide hydrolysis <p class="infolinks">[<a href="javascript:toggleInfo('Waszkowycz1991','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Waszkowycz1991','bibtex')">BibTeX</a>]</p></td>
	<td>1991</td>
	<td>J. Chem. Soc., Perkin Trans. 2(12), pp. 2025 - 2032&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1039/P29910002025">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Waszkowycz1991" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A combined ab initio molecular orbital--molecular mechanical computational model has been applied to the investigation of the potential energy surface of catalysis by the enzyme phosphoiipase A2. By the integration of molecular mechanical and quantum mechanical gradients, the model allows the geometry optimisation of a substrate within the environment of the active site. The method has been applied to a comparison of the mechanisms of ester and amide hydrolysis by phospholipase A2. An estimation of the transition state for amide hydrolysis is described.</td>
</tr>
<tr id="bib_Waszkowycz1991" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Waszkowycz1991,
  author = {Waszkowycz, Bohdan and Hillier, Ian H and Gensmantel, Nigel and Payling, David W},
  title = {Combined quantum mechanical--molecular mechanical study of catalysis by the enzyme phospholipase A 2 : an investigation of the potential energy surface for amide hydrolysis},
  journal = {J. Chem. Soc., Perkin Trans. 2},
  year = {1991},
  number = {12},
  pages = {2025 - 2032},
  doi = {https://doi.org/10.1039/P29910002025}
}
</pre></td>
</tr>
<tr id="Watanabe1999" class="entry">
	<td>Watanabe, A., Kurokawa, Y., Yoshimura, T. and Esaki, N.</td>
	<td>Role of Tyrosine 265 of Alanine Racemase from Bacillus stearothermophilus <p class="infolinks">[<a href="javascript:toggleInfo('Watanabe1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Journal of Biochemistry (Tokyo, Japan)<br/>Vol. 125, pp. 987 - 990&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Watanabe1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Watanabe1999,
  author = {Watanabe, Akira and Kurokawa, Yoichi and Yoshimura, Tohru and Esaki, Nobuyoshi},
  title = {Role of Tyrosine 265 of Alanine Racemase from Bacillus stearothermophilus},
  journal = {Journal of Biochemistry (Tokyo, Japan)},
  year = {1999},
  volume = {125},
  pages = {987 - 990}
}
</pre></td>
</tr>
<tr id="Watanabe2002" class="entry">
	<td>Watanabe, A., Yoshimura, T., Mikami, B., Hayashi, H., Kagamiyama, H. and Esaki, N.</td>
	<td>Reaction mechanism of Alanine Racemase from Bacillus stearothermophilus: X-Ray Crystallographic Studies of the Enzyme Bound with N-(5'-Phosphopyridoxyl)Alanine <p class="infolinks">[<a href="javascript:toggleInfo('Watanabe2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Journal of Biological Chemistry<br/>Vol. 277(21), pp. 19166 - 19172&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Watanabe2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Watanabe2002,
  author = {Watanabe, Akira and Yoshimura, Tohru and Mikami, Brunzo and Hayashi, Hideyuki and Kagamiyama, Hiroyuki and Esaki, Nobuyoshi},
  title = {Reaction mechanism of Alanine Racemase from Bacillus stearothermophilus: X-Ray Crystallographic Studies of the Enzyme Bound with N-(5'-Phosphopyridoxyl)Alanine},
  journal = {Journal of Biological Chemistry},
  year = {2002},
  volume = {277},
  number = {21},
  pages = {19166 - 19172}
}
</pre></td>
</tr>
<tr id="Watkins2015" class="entry">
	<td>Watkins, A.M. and Arora, P.S.</td>
	<td>Structure-based inhibition of protein-protein interactions. <p class="infolinks">[<a href="javascript:toggleInfo('Watkins2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Watkins2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>European journal of medicinal chemistry<br/>Vol. 94, pp. 480-488&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.ejmech.2014.09.047">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Watkins2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Protein-protein interactions (PPIs) are emerging as attractive targets for drug design because of their central role in directing normal and aberrant cellular functions. These interactions were once considered "undruggable" because their large and dynamic interfaces make small molecule inhibitor design challenging. However, landmark advances in computational analysis, fragment screening and molecular design have enabled development of a host of promising strategies to address the fundamental molecular recognition challenge. An attractive approach for targeting PPIs involves mimicry of protein domains that are critical for complex formation. This approach recognizes that protein subdomains or protein secondary structures are often present at interfaces and serve as organized scaffolds for the presentation of side chain groups that engage the partner protein(s). Design of protein domain mimetics is in principle rather straightforward but is enabled by a host of computational strategies that provide predictions of important residues that should be mimicked. Herein we describe a workflow proceeding from interaction network analysis, to modeling a complex structure, to identifying a high-affinity sub-structure, to developing interaction inhibitors. We apply the design procedure to peptidomimetic inhibitors of Ras-mediated signaling.</td>
</tr>
<tr id="bib_Watkins2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Watkins2015,
  author = {Watkins, Andrew M and Arora, Paramjit S},
  title = {Structure-based inhibition of protein-protein interactions.},
  journal = {European journal of medicinal chemistry},
  year = {2015},
  volume = {94},
  pages = {480--488},
  doi = {https://doi.org/10.1016/j.ejmech.2014.09.047}
}
</pre></td>
</tr>
<tr id="Watt2002" class="entry">
	<td>Watt, S.A., Kular, G., Fleming, I.N., Downes, C.P. and Lucocq, J.M.</td>
	<td>Subcellular localization of phosphatidylinositol 4,5-bisphosphate using the pleckstrin homology domain of phospholipase C delta1. <p class="infolinks">[<a href="javascript:toggleInfo('Watt2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Watt2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>The Biochemical journal<br/>Vol. 363, pp. 657-666&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Watt2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Ptd(4,5)P(2) is thought to promote and organize a wide range of cellular functions, including vesicular membrane traffic and cytoskeletal dynamics, by recruiting functional protein complexes to restricted locations in cellular membranes. However, little is known about the distribution of PtdIns(4,5)P(2) in the cell at high resolution. We have used the pleckstrin homology (PH) domain of phospholipase delta(1) (PLCdelta(1)), narrowly specific for PtdIns(4,5)P(2), to map the distribution of the lipid in astrocytoma and A431 cells. We applied the glutathione S-transferase-tagged PLCdelta(1) PH domain (PLCdelta(1)PH-GST) in an on-section labelling approach which avoids transfection procedures. Here we demonstrate PtdIns(4,5)P(2) labelling in the plasma membrane, and also in intracellular membranes, including Golgi (mainly stack), endosomes and endoplasmic reticulum, as well as in electron-dense structures within the nucleus. At the plasma membrane, labelling was more concentrated over lamellipodia, but not in caveolae, which contained less than 10% of the total cell-surface labelling. A dramatic decrease in signal over labelled compartments was observed on preincubation with the cognate headgroup [Ins(1,4,5)P(3)], and plasma-membrane labelling was substantially decreased after stimulation with thrombin-receptor-activating peptide (SFLLRN in the one-letter amino acid code), a treatment which markedly diminishes PtdIns(4,5)P(2) levels. Thus we have developed a highly selective method for mapping the PtdIns(4,5)P(2) distribution within cells at high resolution, and our data provide direct evidence for this lipid at key functional locations.</td>
</tr>
<tr id="bib_Watt2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Watt2002,
  author = {Watt, Stephen A and Kular, Gursant and Fleming, Ian N and Downes, C Peter and Lucocq, John M},
  title = {Subcellular localization of phosphatidylinositol 4,5-bisphosphate using the pleckstrin homology domain of phospholipase C delta1.},
  journal = {The Biochemical journal},
  year = {2002},
  volume = {363},
  pages = {657--666},
  note = {concentration of pip2}
}
</pre></td>
</tr>
<tr id="Watts1998" class="entry">
	<td>Watts, D.J. and Strogatz, S.H.</td>
	<td>Collective dynamics of 'small-world' networks <p class="infolinks">[<a href="javascript:toggleInfo('Watts1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Watts1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Nature<br/>Vol. 393(6684), pp. 409 - 410&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/30918">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Watts1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Networks of coupled dynamical systems have been used to model biological oscillators1, 2, 3, 4, Josephson junction arrays5,6, excitable media7, neural networks8, 9, 10, spatial games11, genetic control networks12 and many other self-organizing systems. Ordinarily, the connection topology is assumed to be either completely regular or completely random. But many biological, technological and social networks lie somewhere between these two extremes. Here we explore simple models of networks that can be tuned through this middle ground: regular networks 'rewired' to introduce increasing amounts of disorder. We find that these systems can be highly clustered, like regular lattices, yet have small characteristic path lengths, like random graphs. We call them 'small-world' networks, by analogy with the small-world phenomenon13,14 (popularly known as six degrees of separation15). The neural network of the worm Caenorhabditis elegans, the power grid of the western United States, and the collaboration graph of film actors are shown to be small-world networks. Models of dynamical systems with small-world coupling display enhanced signal-propagation speed, computational power, and synchronizability. In particular, infectious diseases spread more easily in small-world networks than in regular lattices.</td>
</tr>
<tr id="bib_Watts1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Watts1998,
  author = {Watts, Duncan J and Strogatz, Steven H},
  title = {Collective dynamics of 'small-world' networks},
  journal = {Nature},
  year = {1998},
  volume = {393},
  number = {6684},
  pages = {409 - 410},
  doi = {https://doi.org/10.1038/30918}
}
</pre></td>
</tr>
<tr id="Wehrens2018" class="entry">
	<td>Wehrens, M., Buke, F., Nghe, P. and Tans, S.J.</td>
	<td>Stochasticity in cellular metabolism and growth: approaches and consequences <p class="infolinks">[<a href="javascript:toggleInfo('Wehrens2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Current Opinion in Systems Biology&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.coisb.2018.02.006">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Wehrens2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wehrens2018,
  author = {Martijn Wehrens and Ferhat Buke and Philippe Nghe and Sander J. Tans},
  title = {Stochasticity in cellular metabolism and growth: approaches and consequences},
  journal = {Current Opinion in Systems Biology},
  publisher = {Elsevier BV},
  year = {2018},
  doi = {https://doi.org/10.1016/j.coisb.2018.02.006}
}
</pre></td>
</tr>
<tr id="Weinan2005" class="entry">
	<td>Weinan, E., Ren, W. and Vanden-Eijnden, E.</td>
	<td>Transition pathways in complex systems: Reaction coordinates, isocommittor surfaces, and transition tubes <p class="infolinks">[<a href="javascript:toggleInfo('Weinan2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Weinan2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Chemical Physics Letters<br/>Vol. 413(1-3), pp. 242 - 247&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.cplett.2005.07.084">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Weinan2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The mechanism of transition (reaction coordinate) during an activated process is best described in terms of the isocommittor surfaces. These surfaces can be used to identify effective transition tubes inside which the reactive trajectories involved in the activated process stay confined. It is shown that the isocommittor surfaces can be identified directly, i.e., without ever sampling actual reactive trajectories, and some procedures to turn this observation into practical algorithms such as the finite temperature string method are discussed. ?? 2005 Elsevier B.V. All rights reserved.</td>
</tr>
<tr id="bib_Weinan2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Weinan2005,
  author = {Weinan, E and Ren, Weiqing and Vanden-Eijnden, Eric},
  title = {Transition pathways in complex systems: Reaction coordinates, isocommittor surfaces, and transition tubes},
  journal = {Chemical Physics Letters},
  year = {2005},
  volume = {413},
  number = {1-3},
  pages = {242 - 247},
  doi = {https://doi.org/10.1016/j.cplett.2005.07.084}
}
</pre></td>
</tr>
<tr id="Weinan2010" class="entry">
	<td>Weinan, E. and Vanden-Eijnden, E.</td>
	<td>Transition-path theory and path-finding algorithms for the study of rare events. <p class="infolinks">[<a href="javascript:toggleInfo('Weinan2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Weinan2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Annual Review of Physical Chemistry<br/>Vol. 61(1), pp. 391 - 420&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1146/annurev.physchem.040808.090412">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Weinan2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Transition-path theory is a theoretical framework for describing rare events in complex systems. It can also be used as a starting point for developing efficient numerical algorithms for analyzing such rare events. Here we review the basic components of transition-path theory and path-finding algorithms. We also discuss connections with the classical transition-state theory.</td>
</tr>
<tr id="bib_Weinan2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Weinan2010,
  author = {Weinan, E and Vanden-Eijnden, Eric},
  title = {Transition-path theory and path-finding algorithms for the study of rare events.},
  journal = {Annual Review of Physical Chemistry},
  year = {2010},
  volume = {61},
  number = {1},
  pages = {391 - 420},
  doi = {https://doi.org/10.1146/annurev.physchem.040808.090412}
}
</pre></td>
</tr>
<tr id="Weinberg2012" class="entry">
	<td>Weinberg, J. and Drubin, D.G.</td>
	<td>Clathrin-mediated endocytosis in budding yeast <p class="infolinks">[<a href="javascript:toggleInfo('Weinberg2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Weinberg2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Cell(1)&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.tcb.2011.09.001">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Weinberg2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clathrin-mediated endocytosis in the budding yeast Saccharomyces cerevisiae involves the ordered recruitment, activity and disassembly of nearly 60 proteins at distinct sites on the plasma membrane. Two-color live-cell fluorescence microscopy has proven to be invaluable for in vivo analysis of endocytic proteins: identifying new components, determining the order of protein arrival and dissociation, and revealing even very subtle mutant phenotypes. Yeast genetics and functional genomics facilitate identification of complex interaction networks between endocytic proteins and their regulators. Quantitative datasets produced by these various analyses have made theoretical modeling possible. Here, we discuss recent findings on budding yeast endocytosis that have advanced our knowledge of how ∼60 endocytic proteins are recruited, perform their functions, are regulated by lipid and protein modifications, and are disassembled, all with remarkable regularity.</td>
</tr>
<tr id="bib_Weinberg2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Weinberg2012,
  author = {Weinberg, Jasper and Drubin, David G},
  title = {Clathrin-mediated endocytosis in budding yeast},
  journal = {Cell},
  year = {2012},
  number = {1},
  doi = {https://doi.org/10.1016/j.tcb.2011.09.001}
}
</pre></td>
</tr>
<tr id="Welsch2016" class="entry">
	<td>Welsch, R., Song, K., Shi, Q., Althorpe, S.C. and Miller, T.F.</td>
	<td>Non-equilibrium dynamics from RPMD and CMD <p class="infolinks">[<a href="javascript:toggleInfo('Welsch2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Welsch2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>The Journal of Chemical Physics<br/>Vol. 145(20), pp. 204118 - 204118&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.4967958">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Welsch2016" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We investigate the calculation of approximate non-equilibrium quantum time correlation functions (TCFs) using two popular path-integral-based molecular dynamics methods, ring-polymer molecular dynamics (RPMD) and centroid molecular dynamics (CMD). It is shown that for the cases of a sudden vertical excitation and an initial momentum impulse, both RPMD and CMD yield non-equilibrium TCFs for linear operators that are exact for high temperatures, in the t = 0 limit, and for harmonic potentials; the subset of these conditions that are preserved for non-equilibrium TCFs of non-linear operators is also discussed. Furthermore, it is shown that for these non-equilibrium initial conditions, both methods retain the connection to Matsubara dynamics that has previously been established for equilibrium initial conditions. Comparison of non-equilibrium TCFs from RPMD and CMD to Matsubara dynamics at short times reveals the orders in time to which the methods agree. Specifically, for the position-autocorrelation function associated with sudden vertical excitation, RPMD and CMD agree with Matsubara dynamics up to O(t4) and O(t1), respectively; for the position-autocorrelation function associated with an initial momentum impulse, RPMD and CMD agree with Matsubara dynamics up to O(t5) and O(t2), respectively. Numerical tests using model potentials for a wide range of non-equilibrium initial conditions show that RPMD and CMD yield non-equilibrium TCFs with an accuracy that is comparable to that for equilibrium TCFs. RPMD is also used to investigate excited-state proton transfer in a system-bath model, and it is compared to numerically exact calculations performed using a recently developed version of the Liouville space hierarchical equation of motion approach; again, similar accuracy is observed for non-equilibrium and equilibrium initial conditions.</td>
</tr>
<tr id="bib_Welsch2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Welsch2016,
  author = {Welsch, Ralph and Song, Kai and Shi, Qiang and Althorpe, Stuart C and Miller, Thomas F},
  title = {Non-equilibrium dynamics from RPMD and CMD},
  journal = {The Journal of Chemical Physics},
  year = {2016},
  volume = {145},
  number = {20},
  pages = {204118 - 204118},
  doi = {https://doi.org/10.1063/1.4967958}
}
</pre></td>
</tr>
<tr id="Welsh1980" class="entry">
	<td>Welsh, K.M., Creighton, D.J. and Klinman, J.P.</td>
	<td>Transition-state structure in the yeast alcohol dehydrogenase reaction: The magnitude of solvent and alpha-secondary hydrogen isotope effects <p class="infolinks">[<a href="javascript:toggleInfo('Welsh1980','bibtex')">BibTeX</a>]</p></td>
	<td>1980</td>
	<td>Biochemistry<br/>Vol. 19(10), pp. 2005 - 2016&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Welsh1980" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Welsh1980,
  author = {Welsh, Katherine M and Creighton, Donald J and Klinman, Judith P},
  title = {Transition-state structure in the yeast alcohol dehydrogenase reaction: The magnitude of solvent and alpha-secondary hydrogen isotope effects},
  journal = {Biochemistry},
  year = {1980},
  volume = {19},
  number = {10},
  pages = {2005 - 2016}
}
</pre></td>
</tr>
<tr id="Wendland2002" class="entry">
	<td>Wendland, B.</td>
	<td>Epsins: adaptors in endocytosis? <p class="infolinks">[<a href="javascript:toggleInfo('Wendland2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wendland2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Nature reviews. Molecular cell biology<br/>Vol. 3, pp. 971-977&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nrm970">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wendland2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Endocytic adaptor proteins select specific cargo for internalization by endocytosis through clathrin-coated pits or vesicles. Recent studies indicate that epsins might also be classified as adaptors.</td>
</tr>
<tr id="bib_Wendland2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wendland2002,
  author = {Wendland, Beverly},
  title = {Epsins: adaptors in endocytosis?},
  journal = {Nature reviews. Molecular cell biology},
  year = {2002},
  volume = {3},
  pages = {971--977},
  doi = {https://doi.org/10.1038/nrm970}
}
</pre></td>
</tr>
<tr id="Wick2009" class="entry">
	<td>Wick, C.D.</td>
	<td>NaCl dissociation dynamics at the air−water interface <p class="infolinks">[<a href="javascript:toggleInfo('Wick2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wick2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Journal of Physical Chemistry C<br/>Vol. 113(6), pp. 2497 - 2502&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp807901j">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wick2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The rate for NaCl dissociation was compared between the air?water interface and water bulk using the transition path sampling formulism to begin understanding how an interface affects reaction dynamics. The results showed that the dissociation dynamics at the interface was much slower than in the bulk. Free energy of dissociation calculations were carried out utilizing umbrella sampling for NaCl dissociation at the interface, and a higher barrier to dissociation was observed, which is consistent with the slower interfacial dissociation dynamics. Using the dissociation free energy profile at the interface, transition-state theory was used to predict a rate constant that was lower than the one calculated using transition path sampling. This led to the conclusion that it is difficult to capture the complex nature of interfacial reactions with mean free energy profiles. Analysis of transition states harvested from the simulations found that the NaCl vector was significantly less aligned to the normal of the air?water interface at the transition state than on average and that the Cl? ion had a high probability to be found at the Gibbs dividing surface at the transition state. The rate for NaCl dissociation was compared between the air?water interface and water bulk using the transition path sampling formulism to begin understanding how an interface affects reaction dynamics. The results showed that the dissociation dynamics at the interface was much slower than in the bulk. Free energy of dissociation calculations were carried out utilizing umbrella sampling for NaCl dissociation at the interface, and a higher barrier to dissociation was observed, which is consistent with the slower interfacial dissociation dynamics. Using the dissociation free energy profile at the interface, transition-state theory was used to predict a rate constant that was lower than the one calculated using transition path sampling. This led to the conclusion that it is difficult to capture the complex nature of interfacial reactions with mean free energy profiles. Analysis of transition states harvested from the simulations found that the NaCl vector was significantly less aligned to the normal of the air?water interface at the transition state than on average and that the Cl? ion had a high probability to be found at the Gibbs dividing surface at the transition state.</td>
</tr>
<tr id="bib_Wick2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wick2009,
  author = {Wick, Collin D},
  title = {NaCl dissociation dynamics at the air−water interface},
  journal = {Journal of Physical Chemistry C},
  year = {2009},
  volume = {113},
  number = {6},
  pages = {2497 - 2502},
  note = {Other paper on the NaCl dissociation example},
  doi = {https://doi.org/10.1021/jp807901j}
}
</pre></td>
</tr>
<tr id="Wick2010" class="entry">
	<td>Wick, C.D. and Dang, L.X.</td>
	<td>Computational investigation of the influence of organic-aqueous interfaces on NaCl dissociation dynamics. <p class="infolinks">[<a href="javascript:toggleInfo('Wick2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wick2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>The Journal of Chemical Physics<br/>Vol. 132(4), pp. 044702 - 044702&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.3299279">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wick2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: NaCl pairing and dissociation was investigated at the CCl(4)-water and 1,2-dichloroethane (DCE)-water interfaces, and compared with dissociation results in the bulk and at the air-water interface utilizing polarizable potentials. The transition path sampling methodology was used to calculate the rate constant for dissociation, while umbrella sampling was used to map out a free energy profile for NaCl dissociation. The results found that ion pairing was weakest at the organic-water interfaces, even weaker than in the water bulk. This is in contrast to what has been observed previously for the air-water interface, in which NaCl ion paring is stronger than in the bulk [C. D. Wick, J. Phys. Chem. C 113, 6356 (2009)]. A consequence of the weaker binding at the organic-water interfaces was that ion dissociation was faster than in the other systems studied. Interactions of the organic phase with the ions influenced the magnitude of the Cl(-) induced dipole moment, and at the organic-water interfaces, the average Cl(-) induced dipole was found to be lower than at the air-water interface, weakening interactions with Na(+). These weaker interactions were found to be responsible for the weaker ion pairing found at the organic-water interfaces.</td>
</tr>
<tr id="bib_Wick2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wick2010,
  author = {Wick, Collin D and Dang, Liem X},
  title = {Computational investigation of the influence of organic-aqueous interfaces on NaCl dissociation dynamics.},
  journal = {The Journal of Chemical Physics},
  year = {2010},
  volume = {132},
  number = {4},
  pages = {044702 - 044702},
  note = {Example of the TPS rate calculations being used to calculate the rate of NaCl dissociation.},
  doi = {https://doi.org/10.1063/1.3299279}
}
</pre></td>
</tr>
<tr id="Wigge1997" class="entry">
	<td>Wigge, P., Vallis, Y. and McMahon, H.T.</td>
	<td>Inhibition of receptor-mediated endocytosis by the amphiphysin SH3 domain. <p class="infolinks">[<a href="javascript:toggleInfo('Wigge1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wigge1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Current Biology<br/>Vol. 7, pp. 554-560&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Wigge1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Receptor-mediated endocytosis appears to require the GTP-binding protein dynamin, but the process by which dynamin is recruited to clathrin-coated pits remains unclear. Dynamin contains several proline-rich clusters that bind to Src homology 3 (SH3) domains, which are short modules found in many signalling proteins and which mediate protein-protein interactions. Amphiphysin, a protein that is highly expressed in the brain, interacts with dynamin in vitro, as do Grb2 and many other SH3 domain-containing proteins. In this study, we examined the role of amphiphysin in receptor-mediated endocytosis in vivo. To address the importance of the amphiphysin SH3 domain in dynamin recruitment, we used a transferrin and epidermal growth factor (EGF) uptake assay in COS-7 fibroblasts. Amphiphysin is present in these cells at a low level and indeed in other peripheral tissues. Confocal immunofluorescence revealed that cells transfected with the amphiphysin SH3 domain showed a potent blockade in receptor-mediated endocytosis. To test whether the cellular target of amphiphysin is dynamin, COS-7 cells were contransfected with both dynamin and the amphiphysin SH3 domain; here, transferrin uptake was efficiently rescued. Importantly, the SH3 domains of Grb2, phospholipase C gamma and spectrin all failed to exert any effect on endocytosis. The mechanism of amphiphysin action in recruiting dynamin was additionally tested in vitro: amphiphysin could associate with both dynamin and alpha-adaptin simultaneously, further supporting a role for amphiphysin in endocytosis. Our results suggest that the SH3 domain of amphiphysin recruits dynamin to coated pits in vivo, probably via plasma membrane adaptor complexes. We propose that amphiphysin is not only required for synaptic-vesicle endocytosis, but might also be a key player in dynamin recruitment in all cells undergoing receptor-mediated endocytosis.</td>
</tr>
<tr id="bib_Wigge1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wigge1997,
  author = {Wigge, P and Vallis, Y and McMahon, H T},
  title = {Inhibition of receptor-mediated endocytosis by the amphiphysin SH3 domain.},
  journal = {Current Biology},
  year = {1997},
  volume = {7},
  pages = {554--560},
  note = {Amph - dyna still critical in nonneuronal cells. expression of AMPH is &nbsp;10 fold less in nonneuronal cells. Amph can interact with dyna and ap2 simultaneously}
}
</pre></td>
</tr>
<tr id="Wigner1937" class="entry">
	<td>Wigner, E.</td>
	<td>Calculation of the Rate of Elementary Association Reactions <p class="infolinks">[<a href="javascript:toggleInfo('Wigner1937','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wigner1937','bibtex')">BibTeX</a>]</p></td>
	<td>1937</td>
	<td>The Journal of Chemical Physics<br/>Vol. 5(9), pp. 720 - 720&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.1750107">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wigner1937" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: An upper limit for the rate of association reactions is found by determining the probability of a decrease of the relative energy of two atoms below zero energy, under the influence of a third body. The relation of this approximate calculation to the rigorous solution of the problem is discussed in Section 2. The results are applied to the recombination of J atoms, measured by Rabinowitch and Wood. Numerically, the agreement is quite good; however, the calculated values are somewhat too low, which cannot be explained by an inaccuracy of the method. Reasons for the discrepancy other than the possible nonadiabatic character of the reaction are discussed.</td>
</tr>
<tr id="bib_Wigner1937" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wigner1937,
  author = {Wigner, Eugene},
  title = {Calculation of the Rate of Elementary Association Reactions},
  journal = {The Journal of Chemical Physics},
  year = {1937},
  volume = {5},
  number = {9},
  pages = {720 - 720},
  doi = {https://doi.org/10.1063/1.1750107}
}
</pre></td>
</tr>
<tr id="Wigner1938" class="entry">
	<td>Wigner, E.</td>
	<td>The transition state method <p class="infolinks">[<a href="javascript:toggleInfo('Wigner1938','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wigner1938','bibtex')">BibTeX</a>]</p></td>
	<td>1938</td>
	<td>Trans. Faraday Soc.<br/>Vol. 34(0), pp. 29 - 29&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1039/tf9383400029">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wigner1938" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: No abstract available</td>
</tr>
<tr id="bib_Wigner1938" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wigner1938,
  author = {Wigner, E},
  title = {The transition state method},
  journal = {Trans. Faraday Soc.},
  year = {1938},
  volume = {34},
  number = {0},
  pages = {29 - 29},
  doi = {https://doi.org/10.1039/tf9383400029}
}
</pre></td>
</tr>
<tr id="Wilbur2008" class="entry">
	<td>Wilbur, J.D., Chen, C.-Y., Manalo, V., Hwang, P.K., Fletterick, R.J. and Brodsky, F.M.</td>
	<td>Actin binding by Hip1 (huntingtin-interacting protein 1) and Hip1R (Hip1-related protein) is regulated by clathrin light chain. <p class="infolinks">[<a href="javascript:toggleInfo('Wilbur2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wilbur2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>The Journal of biological chemistry<br/>Vol. 283, pp. 32870-32879&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M802863200">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wilbur2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The huntingtin-interacting protein family members (Hip1 and Hip1R in mammals and Sla2p in yeast) link clathrin-mediated membrane traffic to actin cytoskeleton dynamics. Genetic data in yeast have implicated the light chain subunit of clathrin in regulating this link. To test this hypothesis, the biophysical properties of mammalian Hip1 and Hip1R and their interaction with clathrin light chain and actin were analyzed. The coiled-coil domains (clathrin light chain-binding) of Hip1 and Hip1R were found to be stable homodimers with no propensity to heterodimerize in vitro. Homodimers were also predominant in vivo, accounting for cellular segregation of Hip1 and Hip1R functions. Coiled-coil domains of Hip1 and Hip1R differed in their stability and flexibility, correlating with slightly different affinities for clathrin light chain and more markedly with effects of clathrin light chain binding on Hip protein-actin interactions. Clathrin light chain binding induced a compact conformation of both Hip1 and Hip1R and significantly reduced actin binding by their THATCH domains. Thus, clathrin is a negative regulator of Hip-actin interactions. These observations necessarily change models proposed for Hip protein function.</td>
</tr>
<tr id="bib_Wilbur2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wilbur2008,
  author = {Wilbur, Jeremy D and Chen, Chih-Ying and Manalo, Venus and Hwang, Peter K and Fletterick, Robert J and Brodsky, Frances M},
  title = {Actin binding by Hip1 (huntingtin-interacting protein 1) and Hip1R (Hip1-related protein) is regulated by clathrin light chain.},
  journal = {The Journal of biological chemistry},
  year = {2008},
  volume = {283},
  pages = {32870--32879},
  note = {HIP1 homodimerizes and interacts with CLTB with a KD of 1.02 uM},
  doi = {https://doi.org/10.1074/jbc.M802863200}
}
</pre></td>
</tr>
<tr id="Wilbur2010" class="entry">
	<td>Wilbur, J.D., Hwang, P.K., Ybe, J.A., Lane, M., Sellers, B.D., Jacobson, M.P., Fletterick, R.J. and Brodsky, F.M.</td>
	<td>Conformation switching of clathrin light chain regulates clathrin lattice assembly <p class="infolinks">[<a href="javascript:toggleInfo('Wilbur2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Developmental Cell<br/>Vol. 18(5), pp. 854-861&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Wilbur2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wilbur2010,
  author = {Wilbur, Jeremy D and Hwang, Peter K and Ybe, Joel A and Lane, Michael and Sellers, Benjamin D and Jacobson, Matthew P and Fletterick, Robert J and Brodsky, Frances M},
  title = {Conformation switching of clathrin light chain regulates clathrin lattice assembly},
  journal = {Developmental Cell},
  publisher = {Elsevier},
  year = {2010},
  volume = {18},
  number = {5},
  pages = {854--861}
}
</pre></td>
</tr>
<tr id="Wilde1996" class="entry">
	<td>Wilde, A. and Brodsky, F.M.</td>
	<td>In vivo phosphorylation of adaptors regulates their interaction with clathrin. <p class="infolinks">[<a href="javascript:toggleInfo('Wilde1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>The Journal of Cell Biology<br/>Vol. 135(3), pp. 635-645&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Wilde1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wilde1996,
  author = {Wilde, Andrew and Brodsky, Frances M},
  title = {In vivo phosphorylation of adaptors regulates their interaction with clathrin.},
  journal = {The Journal of Cell Biology},
  publisher = {Rockefeller University Press},
  year = {1996},
  volume = {135},
  number = {3},
  pages = {635--645}
}
</pre></td>
</tr>
<tr id="Wilhelm2014" class="entry">
	<td>Wilhelm, B.G., Mandad, S., Truckenbrodt, S., Kröhnert, K., Schäfer, C., Rammner, B., Koo, S.J., Claßen, G.A., Krauss, M., Haucke, V., Urlaub, H. and Rizzoli, S.O.</td>
	<td>Composition of isolated synaptic boutons reveals the amounts of vesicle trafficking proteins. <p class="infolinks">[<a href="javascript:toggleInfo('Wilhelm2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wilhelm2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Science (New York, N.Y.)<br/>Vol. 344, pp. 1023-1028&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1126/science.1252884">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wilhelm2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Synaptic vesicle recycling has long served as a model for the general mechanisms of cellular trafficking. We used an integrative approach, combining quantitative immunoblotting and mass spectrometry to determine protein numbers; electron microscopy to measure organelle numbers, sizes, and positions; and super-resolution fluorescence microscopy to localize the proteins. Using these data, we generated a three-dimensional model of an "average" synapse, displaying 300,000 proteins in atomic detail. The copy numbers of proteins involved in the same step of synaptic vesicle recycling correlated closely. In contrast, copy numbers varied over more than three orders of magnitude between steps, from about 150 copies for the endosomal fusion proteins to more than 20,000 for the exocytotic ones.</td>
</tr>
<tr id="bib_Wilhelm2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wilhelm2014,
  author = {Wilhelm, Benjamin G and Mandad, Sunit and Truckenbrodt, Sven and Kröhnert, Katharina and Schäfer, Christina and Rammner, Burkhard and Koo, Seong Joo and Claßen, Gala A and Krauss, Michael and Haucke, Volker and Urlaub, Henning and Rizzoli, Silvio O},
  title = {Composition of isolated synaptic boutons reveals the amounts of vesicle trafficking proteins.},
  journal = {Science (New York, N.Y.)},
  year = {2014},
  volume = {344},
  pages = {1023--1028},
  note = {rat synapse copy numbers (AMPH, AP180)},
  doi = {https://doi.org/10.1126/science.1252884}
}
</pre></td>
</tr>
<tr id="Wilks1992" class="entry">
	<td>Wilks, H.M., Moreton, K.M., Halsall, D.J., Hart, K.W., Sessions, R.D., Clarke, A.R. and Holbrook, J.J.</td>
	<td>Design of a specific phenyllactate dehydrogenase by peptide loop exchange on the Bacillus stearothermophilus lactate dehydrogenase framework. <p class="infolinks">[<a href="javascript:toggleInfo('Wilks1992','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wilks1992','bibtex')">BibTeX</a>]</p></td>
	<td>1992</td>
	<td>Biochemistry<br/>Vol. 31(34), pp. 7802 - 6&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Wilks1992" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Restriction sites were introduced into the gene for Bacillus stearothermophilus lactate dehydrogenase which enabled a region of the gene to be excised which coded for a mobile surface loop of polypeptide (residues 98-110) which normally seals the active site vacuole from bulk solvent and is a major determinant of substrate specificity. Oligonucleotide-overlap extension (using the polymerase chain reaction) was used to obtain double-stranded DNA regions which coded for different length and sequence loops and which also contained the same restriction sites. The variable length and sequence loops were inserted into the cut gene and used to synthesize hydroxyacid dehydrogenases with altered substrate specificities. Loops which were longer and shorter than the original were made. The substrate specificities of enzymes with these new loops were considerably altered. For many poor enzyme-substrate pairs, the effect of fructose 1,6-bisphosphate on the steady-state kinetic parameters suggested that the substrate was mainly bound in a nonproductive mode. With one longer loop construction (BL1), activity with pyruvate was reduced one-million-fold but activity with phenylpyruvate was largely unaltered. A switch in specificity (kcat/KM) of 390,000-fold was achieved. The 1700:1 selectivity of enzyme BL1 for phenylpyruvate over pyruvate is that required in a phenyllactate dehydrogenase to be used in monitoring phenylpyruvate in the urine of patients with phenylketonuria consuming an apparently phenylalanine-free diet.</td>
</tr>
<tr id="bib_Wilks1992" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wilks1992,
  author = {Wilks, Helen M and Moreton, Kathleen M and Halsall, David J and Hart, Keith W and Sessions, Richard D and Clarke, Anthony R and Holbrook, John J},
  title = {Design of a specific phenyllactate dehydrogenase by peptide loop exchange on the Bacillus stearothermophilus lactate dehydrogenase framework.},
  journal = {Biochemistry},
  year = {1992},
  volume = {31},
  number = {34},
  pages = {7802 - 6}
}
</pre></td>
</tr>
<tr id="Willox2012" class="entry">
	<td>Willox, A.K. and Royle, S.J.</td>
	<td>Functional analysis of interaction sites on the N-terminal domain of clathrin heavy chain. <p class="infolinks">[<a href="javascript:toggleInfo('Willox2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Willox2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Traffic (Copenhagen, Denmark)<br/>Vol. 13, pp. 70-81&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1111/j.1600-0854.2011.01289.x">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Willox2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In clathrin-mediated membrane traffic, clathrin does not bind directly to cargo and instead binds to adaptors that mediate this function. For endocytosis, the main adaptor is the adaptor protein (AP)-2 complex, but it is uncertain how clathrin contacts AP-2. Here we tested in human cells the importance of the three binding sites that have been identified so far on the N-terminal domain (NTD) of clathrin. We find that mutation of each of the three sites on the NTD, alone or in combination, does not block clathrin/AP-2-mediated endocytosis in the same way as deletion of the NTD. We report here the fourth and final site on the NTD that is required for clathrin/AP-2-mediated endocytic function. Each of the four interaction sites can operate alone to mediate endocytosis. The observed functional redundancy between interaction sites on the NTD explains how productivity of clathrin-coated vesicle formation is ensured.</td>
</tr>
<tr id="bib_Willox2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Willox2012,
  author = {Willox, Anna K and Royle, Stephen J},
  title = {Functional analysis of interaction sites on the N-terminal domain of clathrin heavy chain.},
  journal = {Traffic (Copenhagen, Denmark)},
  year = {2012},
  volume = {13},
  pages = {70--81},
  doi = {https://doi.org/10.1111/j.1600-0854.2011.01289.x}
}
</pre></td>
</tr>
<tr id="Wilson1989" class="entry">
	<td>Wilson, C. and Doniach, S.</td>
	<td>A computer model to dynamically simulate protein folding: studies with crambin. <p class="infolinks">[<a href="javascript:toggleInfo('Wilson1989','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wilson1989','bibtex')">BibTeX</a>]</p></td>
	<td>1989</td>
	<td>Proteins<br/>Vol. 6(2), pp. 193 - 209&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/prot.340060208">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wilson1989" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The current work describes a simplified representation of protein structure with uses in the simulation of protein folding. The model assumes that a protein can be represented by a freely rotating rigid chain with a single atom approximating the effect of each side chain. Potentials describing the attraction or repulsion between different types of amino acids are determined directly from the distribution of amino acids in the database of known protein structures. The optimization technique of simulated annealing has been used to dynamically sample the conformations available to this simple model, allowing the protein to evolve from an extended, random coil into a compact globular structure. Many characteristics expected of true proteins, such as the sequence-dependent formation of secondary structure, the partitioning of hydrophobic residues, and specific disulfide pairing, are reproduced by the simulation, suggesting the model may accurately simulate the folding process.</td>
</tr>
<tr id="bib_Wilson1989" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wilson1989,
  author = {Wilson, C and Doniach, S},
  title = {A computer model to dynamically simulate protein folding: studies with crambin.},
  journal = {Proteins},
  year = {1989},
  volume = {6},
  number = {2},
  pages = {193 - 209},
  note = {Paper detailing the use of simulated annealing to minimize local minima conformations of crambin during folding.},
  doi = {https://doi.org/10.1002/prot.340060208}
}
</pre></td>
</tr>
<tr id="Winfree1984" class="entry">
	<td>Winfree, AT.</td>
	<td>The prehistory of the Belousov-Zhabotinsky oscillator <p class="infolinks">[<a href="javascript:toggleInfo('Winfree1984','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Winfree1984','bibtex')">BibTeX</a>]</p></td>
	<td>1984</td>
	<td>Journal of Chemical Education<br/>Vol. 61(8), pp. 661 - 661&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ed061p661">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Winfree1984" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: I t would not take long to assemble an imposing catalogue of examples illustrating the resistance of human nature to observations that do not fit into existing theory. Discoveries which eventually force on textbooks a change of perspective and emphasis are therefore commonly associated with the names of a second generation of investigators, the originator having finished his contrihutious before the spotlight turned in that direction. But it is unusual for the first meticulous investigation actually to remain unpublished for 30 years before being exhumed for national and international recog-nition. The 1980 award of the Lenin Prize to B. P. Belousov, A. M. Zhahotinsky, V. I. Krinsky, and G. R. Ivanitsky cele-brated such a discovery. The explosively growing literature of chemical oscillators can he traced hack to several independent starting points, the most recent and influential of which is Zhahotiusky's original analysis (I) of the mechanism of Belousov's citric a c i d h o -mate/cerium reaction (2). Belousov's single publication on the subject (2) contains the recipe hut only sketchy conjectures about the mechanism, with no demonstration of rhythmic activity. I t appears without citations on two small pages just before a paper by Belousov and his colleague A. P. Safronov on quite a different subject in the proceedings (in Russian) of a symposium on radiation medicine. This abstract was cited only once outside Russia before the 1970's (3). Of those au-thors who have cited the ahstract, probably few have actually seen it. I was among the first to publish on this subject in the West, hut it was not until 1975 that I received from Endre Koros in Budapest a translation of the Belousov paper, nor until 1978 a xerographic copy of the original in Russian (2) from Zhabotinsky. On a recent visit to the research city of Puschino, south of Moscow, I had an opportunity to learn more of the history of this remarkable reaction, largely by conversation with Zha-botinsky (Fig. 1). With a quiet smile, perhaps surprised by my ignorance, Zhahotinsky told me much that seems common knowledge in Russia about the origins of the ceriumlbromate oscillator, hut has never before appeared in English. Several Russian publications (4-6), a movie (7), and Zhabotinsky's</td>
</tr>
<tr id="bib_Winfree1984" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Winfree1984,
  author = {Winfree, AT},
  title = {The prehistory of the Belousov-Zhabotinsky oscillator},
  journal = {Journal of Chemical Education},
  year = {1984},
  volume = {61},
  number = {8},
  pages = {661 - 661},
  doi = {https://doi.org/10.1021/ed061p661}
}
</pre></td>
</tr>
<tr id="Winkler1983" class="entry">
	<td>Winkler, F.K. and Stanley, K.K.</td>
	<td>Clathrin heavy chain, light chain interactions. <p class="infolinks">[<a href="javascript:toggleInfo('Winkler1983','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Winkler1983','bibtex')">BibTeX</a>]</p></td>
	<td>1983</td>
	<td>The EMBO journal<br/>Vol. 2, pp. 1393-1400&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Winkler1983" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Purified pig brain clathrin can be reversibly dissociated and separated into heavy chain trimers and light chains in the presence of non-denaturing concentrations of the chaotrope thiocyanate. The isolated heavy chain trimers reassemble into regular polygonal cage structures in the absence of light chains. The light chain fraction can be further resolved into its two components L alpha and L beta which give different one-dimensional peptide maps. Radiolabelled light chains bind with high affinity (KD < 10(-10) M) to heavy chain trimers, to heavy chain cages and to a 110,000 mol. wt. tryptic fragment of the heavy chain. Both light chains compete with each other and with light chains from other sources for the same binding sites on heavy chains and c.d. spectroscopy shows that the two pig brain light chains possess very similar structures. We conclude that light chains from different sources, despite some heterogeneity, have a highly conserved, high affinity binding site on the heavy chain but are not essential for the formation of regular cage structures.</td>
</tr>
<tr id="bib_Winkler1983" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Winkler1983,
  author = {Winkler, F K and Stanley, K K},
  title = {Clathrin heavy chain, light chain interactions.},
  journal = {The EMBO journal},
  year = {1983},
  volume = {2},
  pages = {1393--1400}
}
</pre></td>
</tr>
<tr id="Winter2003" class="entry">
	<td>Winter, V.</td>
	<td>Crystal structure of Plasmodium berghei lactate dehydrogenase indicates the unique structural differences of these enzymes are shared across the Plasmodium genus <p class="infolinks">[<a href="javascript:toggleInfo('Winter2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Winter2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Molecular and Biochemical Parasitology<br/>Vol. 131(1), pp. 1 - 10&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S0166-6851(03)00170-1">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Winter2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: As Plasmodium rely extensively on homolactic fermentation for energy production, Plasmodium falciparum lactate dehydrogenase (PfLDH)---the key enzyme in this process---has previously been suggested as a novel target for antimalarials. This enzyme has distinctive kinetic and structural properties that distinguish it from its human homologues. In this study, we now describe the expression, kinetic characterisation and crystal structure determination of the LDH from Plasmodium berghei. This enzyme is seen to have a similar kinetic profile to its P. falciparum counterpart, exhibiting the characteristic lack of substrate inhibition that distinguishes plasmodial from human LDHs. The crystal structure of P. berghei lactate dehydrogenase (PbLDH) shows a very similar active site arrangement to the P. falciparum enzyme. In particular, an insertion of five amino acid residues in the active site loop creates an enlarged volume in the substrate binding site, and characteristic changes in the residues lining the NADH cofactor binding pocket result in displacement of the cofactor relative to its observed position in mammalian and all other LDH structures. These results imply the special features previously described for PfLDH may be shared across the Plasmodium genus, supporting the universal application of therapeutics targeting this enzyme.</td>
</tr>
<tr id="bib_Winter2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Winter2003,
  author = {Winter, V},
  title = {Crystal structure of Plasmodium berghei lactate dehydrogenase indicates the unique structural differences of these enzymes are shared across the Plasmodium genus},
  journal = {Molecular and Biochemical Parasitology},
  year = {2003},
  volume = {131},
  number = {1},
  pages = {1 - 10},
  note = {shit about the active site loop or whatever},
  doi = {https://doi.org/10.1016/S0166-6851(03)00170-1}
}
</pre></td>
</tr>
<tr id="Wisniewski2014" class="entry">
	<td>Wi&#347;niewski, J.R., Hein, M.Y., Cox, J. and Mann, M.</td>
	<td>A "proteomic ruler" for protein copy number and concentration estimation without spike-in standards. <p class="infolinks">[<a href="javascript:toggleInfo('Wisniewski2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wisniewski2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Molecular &amp; cellular proteomics : MCP<br/>Vol. 13(12), pp. 3497 - 506&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/mcp.M113.037309">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wisniewski2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Absolute protein quantification using mass spectrometry (MS)-based proteomics delivers protein concentrations or copy numbers per cell. Existing methodologies typically require a combination of isotope-labeled spike-in references, cell counting, and protein concentration measurements. Here we present a novel method that delivers similar quantitative results directly from deep eukaryotic proteome datasets without any additional experimental steps. We show that the MS signal of histones can be used as a "proteomic ruler" because it is proportional to the amount of DNA in the sample, which in turn depends on the number of cells. As a result, our proteomic ruler approach adds an absolute scale to the MS readout and allows estimation of the copy numbers of individual proteins per cell. We compare our protein quantifications with values derived via the use of stable isotope labeling by amino acids in cell culture and protein epitope signature tags in a method that combines spike-in protein fragment standards with precise isotope label quantification. The proteomic ruler approach yields quantitative readouts that are in remarkably good agreement with results from the precision method. We attribute this surprising result to the fact that the proteomic ruler approach omits error-prone steps such as cell counting or protein concentration measurements. The proteomic ruler approach is readily applicable to any deep eukaryotic proteome dataset-even in retrospective analysis-and we demonstrate its usefulness with a series of mouse organ proteomes.</td>
</tr>
<tr id="bib_Wisniewski2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wisniewski2014,
  author = {Wi&#347;niewski, Jacek R and Hein, Marco Y and Cox, J&uuml;rgen and Mann, Matthias},
  title = {A "proteomic ruler" for protein copy number and concentration estimation without spike-in standards.},
  journal = {Molecular &amp; cellular proteomics : MCP},
  year = {2014},
  volume = {13},
  number = {12},
  pages = {3497 - 506},
  doi = {https://doi.org/10.1074/mcp.M113.037309}
}
</pre></td>
</tr>
<tr id="Wolfenden2006" class="entry">
	<td>Wolfenden, R.</td>
	<td>Degrees of difficulty of water-consuming reactions in the absence of enzymes <p class="infolinks">[<a href="javascript:toggleInfo('Wolfenden2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Chemical Reviews<br/>Vol. 106(8), pp. 3379 - 3396&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/cr050311y">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Wolfenden2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wolfenden2006,
  author = {Wolfenden, Richard},
  title = {Degrees of difficulty of water-consuming reactions in the absence of enzymes},
  journal = {Chemical Reviews},
  year = {2006},
  volume = {106},
  number = {8},
  pages = {3379 - 3396},
  doi = {https://doi.org/10.1021/cr050311y}
}
</pre></td>
</tr>
<tr id="Wolfenden1999" class="entry">
	<td>Wolfenden, R., Snider, M., Ridgway, C. and Miller, B.</td>
	<td>The temperature dependence of enzyme rate enhancements <p class="infolinks">[<a href="javascript:toggleInfo('Wolfenden1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wolfenden1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Journal of the American Chemical Society<br/>Vol. 121(32), pp. 7419 - 7420&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ja991280p">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wolfenden1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: After the initial binding of a substrate by an enzyme, their mutual affinity in water increases by a factor commensurate with the rate enhancement that the enzyme produces in water.1 During the substrate's transformation, their Kd sometimes attains values as low as 10-23 M in the transition state.2 In seeking to analyze the sources of that attraction, it seems reasonable to inquire whether the rate enhancement produced by an enzyme, and hence the corresponding increase in affinity, tends to be mainly enthalpic or entropic in its origins.</td>
</tr>
<tr id="bib_Wolfenden1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wolfenden1999,
  author = {Wolfenden, Richard and Snider, Mark and Ridgway, Caroline and Miller, Brian},
  title = {The temperature dependence of enzyme rate enhancements},
  journal = {Journal of the American Chemical Society},
  year = {1999},
  volume = {121},
  number = {32},
  pages = {7419 - 7420},
  doi = {https://doi.org/10.1021/ja991280p}
}
</pre></td>
</tr>
<tr id="Wolfenden2001" class="entry">
	<td>Wolfenden, R. and Snider, M.J.</td>
	<td>The depth of chemical time and the power of enzymes as catalysts <p class="infolinks">[<a href="javascript:toggleInfo('Wolfenden2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wolfenden2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Accounts of Chemical Research<br/>Vol. 34(12), pp. 938 - 945&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/ar000058i">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wolfenden2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The fastest known reactions include reactions catalyzed by enzymes, but the rate enhancements that enzymes produce had not been fully appreciated until recently. In the absence of enzymes, these same reactions are among the slowest that have ever been measured, some with half-times approaching the age of the Earth. This difference provides a measure of the proficiencies of enzymes as catalysts and their relative susceptibilities to inhibition by transition-state analogue inhibitors. Thermodynamic comparisons between spontaneous and enzyme-catalyzed reactions, coupled with structural information, suggest that in addition to electrostatic and H-bonding interactions, the liberation of water molecules from an enzyme's active site into bulk solvent sometimes plays a prominent role in determining the relative binding affinities of the altered substrate in the ground state and transition state. These comparisons also indicate a high level of synergism in the action of binding determinants of both the substrate and the enzyme, that are not directly involved in the chemical transformation of the substrate but contribute to the rate of its transformation at an enzyme's active site.</td>
</tr>
<tr id="bib_Wolfenden2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wolfenden2001,
  author = {Wolfenden, R and Snider, Mark J},
  title = {The depth of chemical time and the power of enzymes as catalysts},
  journal = {Accounts of Chemical Research},
  year = {2001},
  volume = {34},
  number = {12},
  pages = {938 - 945},
  doi = {https://doi.org/10.1021/ar000058i}
}
</pre></td>
</tr>
<tr id="Wong2012" class="entry">
	<td>Wong, K.A., Wilson, J., Russo, A., Wang, L., Okur, M., Wang, X., Martin, N.P., Scappini, E., Carnegie, G.K. and O'Bryan, John, P</td>
	<td>Intersectin (ITSN) family of scaffolds function as molecular hubs in protein interaction networks <p class="infolinks">[<a href="javascript:toggleInfo('Wong2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wong2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>PLoS ONE<br/>Vol. 7(4), pp. 1 - 9&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pone.0036023">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wong2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Members of the intersectin (ITSN) family of scaffold proteins consist of multiple modular domains, each with distinct ligand preferences. Although ITSNs were initially implicated in the regulation of endocytosis, subsequent studies have revealed a more complex role for these scaffold proteins in regulation of additional biochemical pathways. In this study, we performed a high throughput yeast two-hybrid screen to identify additional pathways regulated by these scaffolds. Although several known ITSN binding partners were identified, we isolated more than 100 new targets for the two mammalian ITSN proteins, ITSN1 and ITSN2. We present the characterization of several of these new targets which implicate ITSNs in the regulation of the Rab and Arf GTPase pathways as well as regulation of the disrupted in schizophrenia 1 (DISC1) interactome. In addition, we demonstrate that ITSN proteins form homomeric and heteromeric complexes with each other revealing an added level of complexity in the function of these evolutionarily conserved scaffolds.</td>
</tr>
<tr id="bib_Wong2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wong2012,
  author = {Wong, Katy A and Wilson, Jessica and Russo, Angela and Wang, Li and Okur, Mustafa and Wang, Xuerong and Martin, Negin P and Scappini, Erica and Carnegie, Graeme K and P, O'Bryan, John},
  title = {Intersectin (ITSN) family of scaffolds function as molecular hubs in protein interaction networks},
  journal = {PLoS ONE},
  year = {2012},
  volume = {7},
  number = {4},
  pages = {1 - 9},
  note = {COS (monkey kidney) and HEK (human embryonic kidney) cells. domain information. ITSN1 - ITSN1, CALM, CLTC, EPS15, AP2B1},
  doi = {https://doi.org/10.1371/journal.pone.0036023}
}
</pre></td>
</tr>
<tr id="Wong1995" class="entry">
	<td>Wong, WT., Schumacher, C. and Salcini, AE.</td>
	<td>A protein-binding domain, EH, identified in the receptor tyrosine kinase substrate Eps15 and conserved in evolution <p class="infolinks">[<a href="javascript:toggleInfo('Wong1995','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wong1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>Proceedings of the &hellip;&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Wong1995" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Abstract In this report we structurally and functionally define a binding domain that is involved in protein association and that we have designated EH (for Eps15 homology domain). This domain was identified in the tyrosine kinase substrate Eps15 on the basis of regional conservation with several heterogeneous proteins of yeast and nematode. The EH domain spans about 70 amino acids and shows approximately 60&#37; overall amino acid  ...</td>
</tr>
<tr id="bib_Wong1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wong1995,
  author = {Wong, WT and Schumacher, C and Salcini, AE},
  title = {A protein-binding domain, EH, identified in the receptor tyrosine kinase substrate Eps15 and conserved in evolution},
  journal = {Proceedings of the &hellip;},
  year = {1995}
}
</pre></td>
</tr>
<tr id="Wratten1963" class="entry">
	<td>Wratten, C.C. and Cleland, WW.</td>
	<td>Product inhibition studies on yeast and liver alcohol dehydrogenases <p class="infolinks">[<a href="javascript:toggleInfo('Wratten1963','bibtex')">BibTeX</a>]</p></td>
	<td>1963</td>
	<td>Biochemistry<br/>Vol. 2(5), pp. 935 - 941&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/bi00905a007">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Wratten1963" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wratten1963,
  author = {Wratten, Craig C and Cleland, WW},
  title = {Product inhibition studies on yeast and liver alcohol dehydrogenases},
  journal = {Biochemistry},
  year = {1963},
  volume = {2},
  number = {5},
  pages = {935 - 941},
  note = {ternary complex interconversion is not rate-determining},
  doi = {https://doi.org/10.1021/bi00905a007}
}
</pre></td>
</tr>
<tr id="Wuchty2002" class="entry">
	<td>Wuchty, S.</td>
	<td>Interaction and domain networks of yeast <p class="infolinks">[<a href="javascript:toggleInfo('Wuchty2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wuchty2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Proteomics<br/>Vol. 2(12), pp. 1715 - 1723&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/1615-9861(200212)2">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wuchty2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Data of currently available protein-protein interaction sets and protein domain sets of yeast are used to set up protein and domain interaction and domain sequence networks. All of them are far from being random or regular networks. In fact, they turn out to be sparse and locally well clustered indicating so-called scale-free and partially small-world topology. These subtle topologies display considerable indirect properties which are measured with a newly introduced transitivity coefficient. Fairly small sets of highly connected proteins and domains shape the topologies of the underlying networks, emphasizing a kind of backbone the nets are based on. The biological nature of these particular nodes is further investigated. Since highly connected proteins and domains accumulated a significant higher number of links by their important involvement in certain cellular aspects, their mutational effect on the cell is considered by a perturbation analysis. In comparison to domains of yeast, what factors force domains to accumulate links to other domains in protein sequences of higher eukaryotes are investigated.</td>
</tr>
<tr id="bib_Wuchty2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wuchty2002,
  author = {Wuchty, Stefan},
  title = {Interaction and domain networks of yeast},
  journal = {Proteomics},
  year = {2002},
  volume = {2},
  number = {12},
  pages = {1715 - 1723},
  doi = {https://doi.org/10.1002/1615-9861(200212)2}
}
</pre></td>
</tr>
<tr id="Wuchty2004" class="entry">
	<td>Wuchty, S.</td>
	<td>Evolution and Topology in the Yeast Protein Interaction Network <p class="infolinks">[<a href="javascript:toggleInfo('Wuchty2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Wuchty2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Genome Research<br/>Vol. 14(7), pp. 1310 - 1314&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1101/gr.2300204.served">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Wuchty2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The integrity of the yeast protein--protein interaction network is maintained by a few highly connected proteins, or hubs, which hold the numerous less-connected proteins together. The structural importance and the increased essentiality of these proteins suggest that they are likely to be conserved in evolution, implying a strong relationship between the number of interactions and their evolutionary distance to its orthologs in other organisms. The existence of this coherence was recently reported to strongly depend on the quality of the protein interaction and orthologs data. Here, we introduce a novel method, the evolutionary excess retention (ER), allowing us to uncover a robust and strong correlation between the conservation, essentiality, and connectivity of a yeast protein. We conclude that the relevance of the hubs for the network integrity is simultaneously reflected by a considerable probability of simultaneously being evolutionarily conserved and essential, an observation that does not have an equivalent for nonessential proteins. Providing a thorough assessment of the impact noisy and incomplete data have on our findings, we conclude that our results are largely insensitive to the quality of the utilized data.</td>
</tr>
<tr id="bib_Wuchty2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Wuchty2004,
  author = {Wuchty, Stefan},
  title = {Evolution and Topology in the Yeast Protein Interaction Network},
  journal = {Genome Research},
  year = {2004},
  volume = {14},
  number = {7},
  pages = {1310 - 1314},
  doi = {https://doi.org/10.1101/gr.2300204.served}
}
</pre></td>
</tr>
<tr id="Xia2011" class="entry">
	<td>Xia, F., Bronowska, A.K., Cheng, S. and Gr&auml;ter, F.</td>
	<td>Base-catalyzed peptide hydrolysis is insensitive to mechanical stress <p class="infolinks">[<a href="javascript:toggleInfo('Xia2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Xia2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 115(33), pp. 10126 - 10132&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp202162r">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Xia2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Biochemical reactions can be guided by mechanical stress. An external force has been previously shown both experimentally and theoretically to act as a catalyst for the scission of a disulfide bond in thiol/disulfide exchange reactions. How the dynamics of peptide hydrolysis, one of the most prevalent biochemical reactions, is influenced by a stretching force was investigated here using combined quantum and molecular mechanical (QM/MM) simulations together with transition path sampling. Our simulations predict mechanical force to only marginally enhance the reactivity of the rate-limiting step, the nucleophilic attack of hydroxide to the peptide moiety, and not to alter the reaction mechanism, even though the peptide bond and its π-electron conjugation is weakened by force. We describe a previously unidentified hydrogen bonded intermediate state, which is likely to play a role in general in base-catalyzed and analogous enzymatic reactions. Our predictions can be directly tested by single molecule stretching experiments.</td>
</tr>
<tr id="bib_Xia2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Xia2011,
  author = {Xia, Fei and Bronowska, Agnieszka K and Cheng, Shanmei and Gr&auml;ter, Frauke},
  title = {Base-catalyzed peptide hydrolysis is insensitive to mechanical stress},
  journal = {Journal of Physical Chemistry B},
  year = {2011},
  volume = {115},
  number = {33},
  pages = {10126 - 10132},
  note = {Example of TPS rate calculations being done on peptide hydrolysis},
  doi = {https://doi.org/10.1021/jp202162r}
}
</pre></td>
</tr>
<tr id="Xu2011" class="entry">
	<td>Xu, W., Smith, A.M., Faeder, J.R. and Marai, E.G.</td>
	<td>RuleBender: A visual interface for rule-based modeling <p class="infolinks">[<a href="javascript:toggleInfo('Xu2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Xu2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Bioinformatics<br/>Vol. 27(12), pp. 1721 - 1722&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1093/bioinformatics/btr197">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Xu2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: SUMMARY: Rule-based modeling (RBM) is a powerful and increasingly popular approach to modeling intracellular biochemistry. Current interfaces for RBM are predominantly text-based and command-line driven. Better visual tools are needed to make RBM accessible to a broad range of users, to make specification of models less error prone and to improve workflows. We present RULEBENDER, an open-source visual interface that facilitates interactive debugging, simulation and analysis of RBMs. AVAILABILITY: RULEBENDER is freely available for Mac, Windows and Linux at http://rulebender.org.</td>
</tr>
<tr id="bib_Xu2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Xu2011,
  author = {Xu, Wen and Smith, Adam M and Faeder, James R and Marai, Elisabeta G},
  title = {RuleBender: A visual interface for rule-based modeling},
  journal = {Bioinformatics},
  year = {2011},
  volume = {27},
  number = {12},
  pages = {1721 - 1722},
  doi = {https://doi.org/10.1093/bioinformatics/btr197}
}
</pre></td>
</tr>
<tr id="Yamabhai1998" class="entry">
	<td>Yamabhai, M., Hoffman, N.G., Hardison, N.L., McPherson, P.S., Castagnoli, L., Cesareni, G. and Kay, B.K.</td>
	<td>Intersectin, a novel adaptor protein with two Eps15 homology and five Src homology 3 domains. <p class="infolinks">[<a href="javascript:toggleInfo('Yamabhai1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Yamabhai1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>The Journal of biological chemistry<br/>Vol. 273, pp. 31401-31407&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Yamabhai1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We screened a Xenopus laevis oocyte cDNA expression library with a Src homology 3 (SH3) class II peptide ligand and identified a 1270-amino acid-long protein containing two Eps15 homology (EH) domains, a central coiled-coil region, and five SH3 domains. We named this protein Intersectin, because it potentially brings together EH and SH3 domain-binding proteins into a macromolecular complex. The ligand preference of the EH domains were deduced to be asparajine-proline-phenylalanine (NPF) or cyclized NPF (CX1-2NPFXXC), depending on the type of phage-displayed combinatorial peptide library used. Screens of a mouse embryo cDNA library with the EH domains of Intersectin yielded clones for the Rev-associated binding/Rev-interacting protein (RAB/Rip) and two novel proteins, which we named Intersectin-binding proteins (Ibps) 1 and 2. All three proteins contain internal and C-terminal NPF peptide sequences, and Ibp1 and Ibp2 also contain putative clathrin-binding sites. Deletion of the C-terminal sequence, NPFL-COOH, from RAB/Rip eliminated EH domain binding, whereas fusion of the same peptide sequence to glutathione S-transferase generated strong binding to the EH domains of Intersectin. Several experiments support the conclusion that the free carboxylate group contributes to binding of the NPFL motif at the C terminus of RAB/Rip to the EH domains of Intersectin. Finally, affinity selection experiments with the SH3 domains of Intersectin identified two endocytic proteins, dynamin and synaptojanin, as potential interacting proteins. We propose that Intersectin is a component of the endocytic machinery.</td>
</tr>
<tr id="bib_Yamabhai1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Yamabhai1998,
  author = {Yamabhai, M and Hoffman, N G and Hardison, N L and McPherson, P S and Castagnoli, L and Cesareni, G and Kay, B K},
  title = {Intersectin, a novel adaptor protein with two Eps15 homology and five Src homology 3 domains.},
  journal = {The Journal of biological chemistry},
  year = {1998},
  volume = {273},
  pages = {31401--31407}
}
</pre></td>
</tr>
<tr id="Yang2012" class="entry">
	<td>Yang, JR., Liao, BY., Zhuang, SM. and Zhang, JZ.</td>
	<td>Protein misinteraction avoidance causes highly expressed proteins to evolve slowly <p class="infolinks">[<a href="javascript:toggleInfo('Yang2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Yang2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Proceedings of the National Academy of Sciences<br/>Vol. 109(14), pp. E831 - E840&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.1117408109">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Yang2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The tempo and mode of protein evolution have been central questions in biology. Genomic data have shown a strong influence of the expression level of a protein on its rate of sequence evolution (E-R anticorrelation), which is currently explained by the protein misfolding avoidance hypothesis. Here, we show that this hypothesis does not fully explain the E-R anticorrelation, especially for protein surface residues. We propose that natural selection against protein-protein misinteraction, which wastes functional molecules and is potentially toxic, constrains the evolution of surface residues. Because highly expressed proteins are under stronger pressures to avoid misinteraction, surface residues are expected to show an E-R anticorrelation. Our molecular-level evolutionary simulation and yeast genomic analysis confirm multiple predictions of the hypothesis. These findings show a pluralistic origin of the E-R anticorrelation and reveal the role of protein misinteraction, an inherent property of complex cellular systems, in constraining protein evolution.</td>
</tr>
<tr id="bib_Yang2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Yang2012,
  author = {Yang, JR and Liao, BY and Zhuang, SM and Zhang, JZ},
  title = {Protein misinteraction avoidance causes highly expressed proteins to evolve slowly},
  journal = {Proceedings of the National Academy of Sciences},
  year = {2012},
  volume = {109},
  number = {14},
  pages = {E831 - E840},
  doi = {https://doi.org/10.1073/pnas.1117408109}
}
</pre></td>
</tr>
<tr id="Yang2008" class="entry">
	<td>Yang, J., Monine, M.I., Faeder, J.R. and Hlavacek, W.S.</td>
	<td>Kinetic Monte Carlo method for rule-based modeling of biochemical networks <p class="infolinks">[<a href="javascript:toggleInfo('Yang2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Yang2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Physical Review E<br/>Vol. 78(3), pp. 1 - 7&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1103/PhysRevE.78.031910">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Yang2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present a kinetic Monte Carlo method for simulating chemical transformations specified by reaction rules, which can be viewed as generators of chemical reactions, or equivalently, definitions of reaction classes. A rule identifies the molecular components involved in a transformation, how these components change, conditions that affect whether a transformation occurs, and a rate law. The computational cost of the method, unlike conventional simulation approaches, is independent of the number of possible reactions, which need not be specified in advance or explicitly generated in a simulation. To demonstrate the method, we apply it to study the kinetics of multivalent ligand-receptor interactions. We expect the method will be useful for studying cellular signaling systems and other physical systems involving aggregation phenomena.</td>
</tr>
<tr id="bib_Yang2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Yang2008,
  author = {Yang, Jin and Monine, Michael I and Faeder, James R and Hlavacek, William S},
  title = {Kinetic Monte Carlo method for rule-based modeling of biochemical networks},
  journal = {Physical Review E},
  year = {2008},
  volume = {78},
  number = {3},
  pages = {1 - 7},
  doi = {https://doi.org/10.1103/PhysRevE.78.031910}
}
</pre></td>
</tr>
<tr id="Yang2017" class="entry">
	<td>Yang, P.J., Morgan, LaMarca,., Kaminski, C., Chu, D.I. and Hu, D.L.</td>
	<td>Hydrodynamics of defecation <p class="infolinks">[<a href="javascript:toggleInfo('Yang2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Yang2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Soft Matter<br/>Vol. 13(29), pp. 4960-4970&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1039/C6SM02795D">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Yang2017" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Animals discharge feces within a range of sizes and shapes. Such variation has long been used to track animals as well as to diagnose illnesses in both humans and animals. However, the physics by which feces are discharged remain poorly understood. In this combined experimental and theoretical study, we investigate the defecation of mammals from cats to elephants using the dimensions of large intestines and feces, videography at Zoo Atlanta, cone-on-plate rheological measurements of feces and mucus, and a mathematical model of defecation. The diameter of feces is comparable to that of the rectum, but the length is double that of the rectum, indicating that not only the rectum but also the colon is a storage facility for feces. Despite the length of rectum ranging from 4 to 40 cm, mammals from cats to elephants defecate within a nearly constant duration of 12 textpm 7 seconds (N = 23). We rationalize this surprising trend by our mathematical model, which shows that feces slide along the large intestine by a layer of mucus, similar to a sled sliding down a chute. Larger animals have not only more feces but also thicker mucus layers, which facilitate their ejection. Our model accounts for the shorter and longer defecation times associated with diarrhea and constipation, respectively. This study may support clinicians use of non-invasive procedures such as defecation time in the diagnoses of ailments of the digestive system.</td>
</tr>
<tr id="bib_Yang2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Yang2017,
  author = {Yang, Patricia J and Morgan, LaMarca, and Kaminski, Candice and Chu, Daniel I and Hu, David L},
  title = {Hydrodynamics of defecation},
  journal = {Soft Matter},
  year = {2017},
  volume = {13},
  number = {29},
  pages = {4960-4970},
  doi = {https://doi.org/10.1039/C6SM02795D}
}
</pre></td>
</tr>
<tr id="Yang2018" class="entry">
	<td>Yang, Y., Cui, Y., Sang, K., Dong, Y., Ni, Z., Ma, S. and Hu, H.</td>
	<td>Ketamine blocks bursting in the lateral habenula to rapidly relieve depression <p class="infolinks">[<a href="javascript:toggleInfo('Yang2018','bibtex')">BibTeX</a>]</p></td>
	<td>2018</td>
	<td>Nature<br/>Vol. 554(7692), pp. 317-322&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1038/nature25509">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Yang2018" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Yang2018,
  author = {Yan Yang and Yihui Cui and Kangning Sang and Yiyan Dong and Zheyi Ni and Shuangshuang Ma and Hailan Hu},
  title = {Ketamine blocks bursting in the lateral habenula to rapidly relieve depression},
  journal = {Nature},
  publisher = {Springer Nature},
  year = {2018},
  volume = {554},
  number = {7692},
  pages = {317--322},
  doi = {https://doi.org/10.1038/nature25509}
}
</pre></td>
</tr>
<tr id="Ybe2009" class="entry">
	<td>Ybe, J.A., Clegg, M.E., Illingworth, M., Gonzalez, C. and Niu, Q.</td>
	<td>Two Distantly Spaced Basic Patches in the Flexible Domain of Huntingtin-Interacting Protein 1 (HIP1) Are Essential for the Binding of Clathrin Light Chain. <p class="infolinks">[<a href="javascript:toggleInfo('Ybe2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ybe2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Research letters in biochemistry<br/>Vol. 2009, pp. 256124&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1155/2009/256124">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ybe2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The interaction between HIP family proteins (HIP1 and HIP12/1R) and clathrin is fundamental to endocytosis. We used circular dichroism (CD) to study the stability of an HIP1 subfragment (aa468-530) that is splayed open. CD thermal melts show HIP1 468-530 is only stable at low temperatures, but this HIP1 fragment contains a structural unit that does not melt out even at 83°C. We then created HIP1 mutants to probe our hypothesis that a short hydrophobic path in the opened region is the binding site for clathrin light chain. We found that the binding of hub/LCb was sensitive to mutating two distantly separated basic residues (K474 and K494). The basic patches marked by K474 and K494 are conserved in HIP12/1R. The lack of conservation in sla2p (S. cerevisiae), HIP1 from D. melanogaster, and HIP1 homolog ZK370.3 from C. elegans implies the binding of HIP1 and HIP1 homologs to clathrin light chain may be different in these organisms.</td>
</tr>
<tr id="bib_Ybe2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ybe2009,
  author = {Ybe, Joel A and Clegg, Mary E and Illingworth, Melissa and Gonzalez, Claire and Niu, Qian},
  title = {Two Distantly Spaced Basic Patches in the Flexible Domain of Huntingtin-Interacting Protein 1 (HIP1) Are Essential for the Binding of Clathrin Light Chain.},
  journal = {Research letters in biochemistry},
  year = {2009},
  volume = {2009},
  pages = {256124},
  doi = {https://doi.org/10.1155/2009/256124}
}
</pre></td>
</tr>
<tr id="Ybe2007" class="entry">
	<td>Ybe, J.A., Perez-Miller, S., Niu, Q., Coates, D.A., Drazer, M.W. and Clegg, M.E.</td>
	<td>Light chain C-terminal region reinforces the stability of clathrin heavy chain trimers. <p class="infolinks">[<a href="javascript:toggleInfo('Ybe2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ybe2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Traffic (Copenhagen, Denmark)<br/>Vol. 8, pp. 1101-1110&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1111/j.1600-0854.2007.00597.x">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ybe2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The self-assembly of clathrin into lattices relies on the ability of heavy chain legs to form a three-legged pinwheel structure. We investigated the role of light chains in clathrin trimerization by challenging recombinant hub (plus and minus light chain) with an anionic detergent. The binding of light chain increases the amount of detergent needed to induce detrimerization, suggesting light chains reinforced hub trimers. We also show that light chain C-terminal residues are important for enhancing the in vitro assembly of hub at low pH. We assessed how much the C-terminus of light chain contributed to the stability of the trimerization domain by adding full-length and truncated light chains to trimer-defective hub mutants, C1573S and C1573A. Adding full-length LCb to C1573S caused some retrimerization, but little activity was restored, suggesting the majority of oligomeric C1573S was nonnative. A larger percentage of monomeric C1573A could be retrimerized into an assembly-competent form by adding intact LCb. We also discovered that C-terminally deleted light chains produced a heterogeneous population of hubs that were smaller than native hubs, but were assembly active. We propose a model showing how light chains reinforce the puckered clathrin triskelion. Finally, the ability of light chains to retrimerize C1573A hub suggests that the structural role of light chain may be conserved in yeast and mammals.</td>
</tr>
<tr id="bib_Ybe2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ybe2007,
  author = {Ybe, Joel A and Perez-Miller, Samantha and Niu, Qian and Coates, David A and Drazer, Michael W and Clegg, Mary E},
  title = {Light chain C-terminal region reinforces the stability of clathrin heavy chain trimers.},
  journal = {Traffic (Copenhagen, Denmark)},
  year = {2007},
  volume = {8},
  pages = {1101--1110},
  note = {CLTB stabilizes CLTC triskelia and lock them into a specific oligomeric conformation},
  doi = {https://doi.org/10.1111/j.1600-0854.2007.00597.x}
}
</pre></td>
</tr>
<tr id="Yim2010" class="entry">
	<td>Yim, Y.-I., Sun, T., Wu, L.-G., Raimondi, A., De Camilli, P., Eisenberg, E. and Greene, L.E.</td>
	<td>Endocytosis and clathrin-uncoating defects at synapses of auxilin knockout mice. <p class="infolinks">[<a href="javascript:toggleInfo('Yim2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Yim2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Proceedings of the National Academy of Sciences of the United States of America<br/>Vol. 107, pp. 4412-4417&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.1000738107">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Yim2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Neuronally expressed auxilin and ubiquitously expressed cyclin-G-dependent kinase (GAK) are homologous proteins that act as cochaperones to support the Hsc70-dependent clathrin uncoating of clathrin-coated vesicles. GAK was previously shown to be essential in mouse during embryonic development and in the adult. We have now engineered an auxilin knockout mouse. Mutant mice had a high rate of early postnatal mortality and surviving pups generally had a lower body weight than wild-type pups, although they had a normal life span. GAK was up-regulated as much as 3-fold in the brains of both surviving neonates and adult mutant mice. An increased number of clathrin-coated vesicles and empty cages were present at knockout synapses both in situ and in primary neuronal cultures. Additionally, clathrin-mediated endocytosis of synaptic vesicles in knockout hippocampal neurons was impaired, most likely due to sequestration of coat components in assembled coats and cages. Collectively, our results demonstrate the specialized role of auxilin in the recycling of synaptic vesicles at synapses, but also show that its function can be partially compensated for by up-regulation of GAK.</td>
</tr>
<tr id="bib_Yim2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Yim2010,
  author = {Yim, Yang-In and Sun, Tao and Wu, Ling-Gang and Raimondi, Andrea and De Camilli, Pietro and Eisenberg, Evan and Greene, Lois E},
  title = {Endocytosis and clathrin-uncoating defects at synapses of auxilin knockout mice.},
  journal = {Proceedings of the National Academy of Sciences of the United States of America},
  year = {2010},
  volume = {107},
  pages = {4412--4417},
  note = {no mention of auxilin during invagination},
  doi = {https://doi.org/10.1073/pnas.1000738107}
}
</pre></td>
</tr>
<tr id="Yogurtcu2015" class="entry">
	<td>Yogurtcu, O.N. and Johnson, M.E.</td>
	<td>Theory of bi-molecular association dynamics in 2D for accurate model and experimental parameterization of binding rates <p class="infolinks">[<a href="javascript:toggleInfo('Yogurtcu2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Yogurtcu2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Journal of Chemical Physics<br/>Vol. 143(8), pp. 084117&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.4929390">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Yogurtcu2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The dynamics of association between diffusing and reacting molecular species are routinely quanti-fied using simple rate-equation kinetics that assume both well-mixed concentrations of species and a single rate constant for parameterizing the binding rate. In two-dimensions (2D), however, even when systems are well-mixed, the assumption of a single characteristic rate constant for describing association is not generally accurate, due to the properties of diffusional searching in dimensions d ≤ 2. Establishing rigorous bounds for discriminating between 2D reactive systems that will be accurately described by rate equations with a single rate constant, and those that will not, is critical for both modeling and experimentally parameterizing binding reactions restricted to surfaces such as cellular membranes. We show here that in regimes of intrinsic reaction rate (k a) and diffusion (D) parameters k a /D \textgreater\ 0.05, a single rate constant cannot be fit to the dynamics of concentrations of associating species independently of the initial conditions. Instead, a more sophisticated multi-parametric description than rate-equations is necessary to robustly characterize bimolecular reactions from experiment. Our quantitative bounds derive from our new analysis of 2D rate-behavior predicted from Smoluchowski theory. Using a recently developed single particle reaction-diffusion algorithm we extend here to 2D, we are able to test and validate the predictions of Smoluchowski theory and several other theories of reversible reaction dynamics in 2D for the first time. Finally, our results also mean that simulations of reactive systems in 2D using rate equations must be undertaken with caution when reactions have k a /D \textgreater\ 0.05, regardless of the simulation volume. We introduce here a simple formula for an adaptive concentration dependent rate constant for these chemical kinetics simulations which improves on existing formulas to better capture non-equilibrium reaction dynamics from dilute to dense systems. C 2015 AIP Publishing LLC. [http://dx.doi.org/10.1063/1.4929390]</td>
</tr>
<tr id="bib_Yogurtcu2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Yogurtcu2015,
  author = {Yogurtcu, Osman N and Johnson, Margaret E},
  title = {Theory of bi-molecular association dynamics in 2D for accurate model and experimental parameterization of binding rates},
  journal = {Journal of Chemical Physics},
  year = {2015},
  volume = {143},
  number = {8},
  pages = {084117},
  doi = {https://doi.org/10.1063/1.4929390}
}
</pre></td>
</tr>
<tr id="Yogurtcu2017" class="entry">
	<td>Yogurtcu, O.N. and Johnson, M.E.</td>
	<td>Cytosolic proteins can exploit membrane localization to trigger functional assembly <p class="infolinks">[<a href="javascript:toggleInfo('Yogurtcu2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>preprint&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Yogurtcu2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Yogurtcu2017,
  author = {Yogurtcu, Osman N and Johnson, Margaret E},
  title = {Cytosolic proteins can exploit membrane localization to trigger functional assembly},
  journal = {preprint},
  year = {2017}
}
</pre></td>
</tr>
<tr id="Yokoigawa2003" class="entry">
	<td>Yokoigawa, K., Okubo, Y. and Soda, K.</td>
	<td>Subunit interaction of monomeric alanine racemases from four Shigella species in catalytic reaction <p class="infolinks">[<a href="javascript:toggleInfo('Yokoigawa2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>FEMS Microbiology Letters<br/>Vol. 221(2), pp. 263 - 268&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/S0378-1097(03)00216-7">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Yokoigawa2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Yokoigawa2003,
  author = {Yokoigawa, Kumio and Okubo, Yoko and Soda, Kenji},
  title = {Subunit interaction of monomeric alanine racemases from four Shigella species in catalytic reaction},
  journal = {FEMS Microbiology Letters},
  year = {2003},
  volume = {221},
  number = {2},
  pages = {263 - 268},
  note = {&amp;quot;It has recently been proposed that variations in kinetic activity found in different alanine racemases mauy be correlated with their association constant for dimer formation&amp;quot;},
  doi = {https://doi.org/10.1016/S0378-1097(03)00216-7}
}
</pre></td>
</tr>
<tr id="Yook2004" class="entry">
	<td>Yook, Soon‐Hyung., Oltvai, Z.N. and Barab&aacute;si, Albert‐L.</td>
	<td>Functional and topological characterization of protein interaction networks <p class="infolinks">[<a href="javascript:toggleInfo('Yook2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Yook2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Proteomics(4)&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/pmic.200300636">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Yook2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The elucidation of the cell's large-scale organization is a primary challenge for post-genomic biology, and understanding the structure of protein interaction networks offers an important starting point for such studies. We compare four available databases that approximate the protein interaction network of the yeast, Saccharomyces cerevisiae, aiming to uncover the network's generic large-scale properties and the impact of the proteins' function and cellular localization on the network topology. We show how each database supports a scale-free, topology with hierarchical modularity, indicating that these features represent a robust and generic property of the protein interactions network. We also find strong correlations between the network's structure and the functional role and subcellular localization of its protein constituents, concluding that most functional and/or localization classes appear as relatively segregated subnetworks of the full protein interaction network. The uncovered systematic differences between the four protein interaction databases reflect their relative coverage for different functional and localization classes and provide a guide for their utility in various bioinformatics studies.</td>
</tr>
<tr id="bib_Yook2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Yook2004,
  author = {Yook, Soon‐Hyung and Oltvai, Zolt&aacute;n N and Barab&aacute;si, Albert‐L&aacute;szl&oacute;},
  title = {Functional and topological characterization of protein interaction networks},
  journal = {Proteomics},
  year = {2004},
  number = {4},
  doi = {https://doi.org/10.1002/pmic.200300636}
}
</pre></td>
</tr>
<tr id="Yoon2012" class="entry">
	<td>Yoon, Y., Zhang, X. and Cho, W.</td>
	<td>Phosphatidylinositol 4,5-bisphosphate (PtdIns(4,5)P2) specifically induces membrane penetration and deformation by Bin/amphiphysin/Rvs (BAR) domains. <p class="infolinks">[<a href="javascript:toggleInfo('Yoon2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Yoon2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>The Journal of biological chemistry<br/>Vol. 287, pp. 34078-34090&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1074/jbc.M112.372789">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Yoon2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Cellular proteins containing Bin/amphiphysin/Rvs (BAR) domains play a key role in clathrin-mediated endocytosis. Despite extensive structural and functional studies of BAR domains, it is still unknown how exactly these domains interact with the plasma membrane containing phosphatidylinositol 4,5-bisphosphate (PtdIns(4,5)P(2)) and whether they function by a universal mechanism or by different mechanisms. Here we report that PtdIns(4,5)P(2) specifically induces partial membrane penetration of the N-terminal amphiphilic α-helix (H(0)) of two representative N-BAR domains from Drosophila amphiphysin (dAmp-BAR) and rat endophilin A1 (EndoA1-BAR). Our quantitative fluorescence imaging analysis shows that PtdIns(4,5)P(2)-dependent membrane penetration of H(0) is important for self-association of membrane-bound dAmp-BAR and EndoA1-BAR and their membrane deformation activity. EndoA1-BAR behaves differently from dAmp-BAR because the former has an additional amphiphilic α-helix that penetrates the membrane in a PtdIns(4,5)P(2)-independent manner. Depletion of PtdIns(4,5)P(2) from the plasma membrane of HEK293 cells abrogated the membrane deforming activity of EndoA1-BAR and dAmp-BAR. Collectively, these studies suggest that the local PtdIns(4,5)P(2) concentration in the plasma membrane may regulate the membrane interaction and deformation by N-BAR domain-containing proteins during clathrin-mediated endocytosis.</td>
</tr>
<tr id="bib_Yoon2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Yoon2012,
  author = {Yoon, Youngdae and Zhang, Xiuqi and Cho, Wonhwa},
  title = {Phosphatidylinositol 4,5-bisphosphate (PtdIns(4,5)P2) specifically induces membrane penetration and deformation by Bin/amphiphysin/Rvs (BAR) domains.},
  journal = {The Journal of biological chemistry},
  year = {2012},
  volume = {287},
  pages = {34078--34090},
  note = {PIP2 - ENDO.BAR (30 uM, rat), AMPH.BAR (105 uM, Dros.)},
  doi = {https://doi.org/10.1074/jbc.M112.372789}
}
</pre></td>
</tr>
<tr id="Yu2014" class="entry">
	<td>Yu, A., Shibata, Y., Shah, B., Calamini, B., Lo, D.C. and Morimoto, R.I.</td>
	<td>Protein aggregation can inhibit clathrin-mediated endocytosis by chaperone competition. <p class="infolinks">[<a href="javascript:toggleInfo('Yu2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Yu2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>Proceedings of the National Academy of Sciences of the United States of America<br/>Vol. 111, pp. E1481-E1490&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1073/pnas.1321811111">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Yu2014" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Protein conformational diseases exhibit complex pathologies linked to numerous molecular defects. Aggregation of a disease-associated protein causes the misfolding and aggregation of other proteins, but how this interferes with diverse cellular pathways is unclear. Here, we show that aggregation of neurodegenerative disease-related proteins (polyglutamine, huntingtin, ataxin-1, and superoxide dismutase-1) inhibits clathrin-mediated endocytosis (CME) in mammalian cells by aggregate-driven sequestration of the major molecular chaperone heat shock cognate protein 70 (HSC70), which is required to drive multiple steps of CME. CME suppression was also phenocopied by HSC70 RNAi depletion and could be restored by conditionally increasing HSC70 abundance. Aggregation caused dysregulated AMPA receptor internalization and also inhibited CME in primary neurons expressing mutant huntingtin, showing direct relevance of our findings to the pathology in neurodegenerative diseases. We propose that aggregate-associated chaperone competition leads to both gain-of-function and loss-of-function phenotypes as chaperones become functionally depleted from multiple clients, leading to the decline of multiple cellular processes. The inherent properties of chaperones place them at risk, contributing to the complex pathologies of protein conformational diseases.</td>
</tr>
<tr id="bib_Yu2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Yu2014,
  author = {Yu, Anan and Shibata, Yoko and Shah, Bijal and Calamini, Barbara and Lo, Donald C and Morimoto, Richard I},
  title = {Protein aggregation can inhibit clathrin-mediated endocytosis by chaperone competition.},
  journal = {Proceedings of the National Academy of Sciences of the United States of America},
  year = {2014},
  volume = {111},
  pages = {E1481--E1490},
  doi = {https://doi.org/10.1073/pnas.1321811111}
}
</pre></td>
</tr>
<tr id="Zaremba1983" class="entry">
	<td>Zaremba, S. and Keen, J.H.</td>
	<td>Assembly polypeptides from coated vesicles mediate reassembly of unique clathrin coats. <p class="infolinks">[<a href="javascript:toggleInfo('Zaremba1983','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Zaremba1983','bibtex')">BibTeX</a>]</p></td>
	<td>1983</td>
	<td>The Journal of cell biology<br/>Vol. 97, pp. 1339-1347&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Zaremba1983" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A protein activity has been identified in extracts of coated vesicles that enables purified clathrin triskelions to reassemble in vitro into coat structures of uniform size. Coats formed in the presence of this preparation, regardless of the buffer system employed, are uniform in size with a mean diameter of 78 nm (+/- 5 nm SD) and a sedimentation coefficient (S20,w) of approximately 250S. Analysis of the reassembled coats on dodecyl sulfate acrylamide gels reveals that they have specifically incorporated three polypeptides from the preparation: those of Mr congruent to 52,000, 100,000, and 110,000. The 52,000-, 100,000-, and 110,000-mol-wt polypeptides are incorporated in molar ratios of 0.85, 1.11, and 0.26, respectively, per three clathrin monomers (equivalent to one triskelion). We therefore designate these as assembly polypeptides (AP). In contrast, coats formed from clathrin alone, under permissive buffer conditions, are larger (400S), more heterogeneous in size (101 nm +/- 15 nm SD), and are composed only of clathrin and its associated light chains. These biochemical and biophysical characteristics distinguish AP-reassembled coats from coats formed by triskelions alone. AP-reassembled coats can be isolated, dissociated, then reassembled in the absence of any other factors. This recycling indicates that all the information needed for reassembly is present in the coat-incorporated polypeptides themselves. Reassembly is stoichiometric and saturable with respect to both clathrin and AP concentration. In the presence of AP, significant coat reassembly occurs at clathrin concentrations as low as 0.06 mg/ml. AP-mediated reassembly proceeds at 4 degrees, 22 degrees, and 37 degrees C. Coat formation also proceeds efficiently at intracellular pH values (7.2-7.5) in the presence of AP. In its absence, reassembly does not occur at all above pH 6.7. In summary, AP promotes clathrin reassembly into coat structures of uniform size and distinctive composition under physiologically relevant salt, temperature, and pH conditions. In addition, the close similarity in size between AP-reassembled coats in vitro and coated membranes in the Golgi region in vivo raises the possibility that AP in the cell may be associated with this subpopulation of coat structures.</td>
</tr>
<tr id="bib_Zaremba1983" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Zaremba1983,
  author = {Zaremba, S and Keen, J H},
  title = {Assembly polypeptides from coated vesicles mediate reassembly of unique clathrin coats.},
  journal = {The Journal of cell biology},
  year = {1983},
  volume = {97},
  pages = {1339--1347}
}
</pre></td>
</tr>
<tr id="Zelmann2012" class="entry">
	<td>Zelmann, R., Mari, F., Jacobs, J., Zijlmans, M., Dubeau, F. and Gotman, J.</td>
	<td>A high-throughput approach for measuring temporal changes in the interactome <p class="infolinks">[<a href="javascript:toggleInfo('Zelmann2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Zelmann2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Nature Methods<br/>Vol. 9(9), pp. 907 - 909&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.clinph.2011.06.006.A">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Zelmann2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Objective---High frequency oscillations (HFOs) are a biomarker of epileptogenicity. Visual marking of HFOs is highly time-consuming and inevitably subjective, making automatic detection necessary. We compare four existing detectors on the same dataset. Methods---HFOs and baselines were identified by experienced reviewers in intracerebral EEGs from 20 patients. A new feature of our detector to deal with channels where baseline cannot be found is presented. The original and an optimal configuration are implemented. Receiver operator curves, false discovery rate, and channel ranking are used to evaluate performance. Results---All detectors improve performance with the optimal configuration. Our detector had higher sensitivity, lower false positives than the others, and similar false detections. The main difference in performance was in very active channels. Conclusions---Each detector was developed for different recordings and with different aims. Our detector performed better in this dataset, but was developed on data similar to the test data. Moreover, optimizing on a particular data type improves performance in any detector. Significance---Automatic HFO detection is crucial to propel their clinical use as biomarkers of epileptogenic tissue. Comparing detectors on a single dataset is important to analyze their performance and to emphasize the issues involved in validation.</td>
</tr>
<tr id="bib_Zelmann2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Zelmann2012,
  author = {Zelmann, R and Mari, F and Jacobs, J and Zijlmans, M and Dubeau, F and Gotman, J},
  title = {A high-throughput approach for measuring temporal changes in the interactome},
  journal = {Nature Methods},
  year = {2012},
  volume = {9},
  number = {9},
  pages = {907 - 909},
  note = {HeLa cells. AP2B1 - EPN1.},
  doi = {https://doi.org/10.1016/j.clinph.2011.06.006.A}
}
</pre></td>
</tr>
<tr id="Zeymer2017" class="entry">
	<td>Zeymer, C., Zschoche, R. and Hilvert, D.</td>
	<td>Optimization of Enzyme Mechanism along the Evolutionary Trajectory of a Computationally Designed (Retro-)Aldolase <p class="infolinks">[<a href="javascript:toggleInfo('Zeymer2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Journal of the American Chemical Society&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jacs.7b05796">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Zeymer2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Zeymer2017,
  author = {Zeymer, Cathleen and Zschoche, Reinhard and Hilvert, Donald},
  title = {Optimization of Enzyme Mechanism along the Evolutionary Trajectory of a Computationally Designed (Retro-)Aldolase},
  journal = {Journal of the American Chemical Society},
  year = {2017},
  doi = {https://doi.org/10.1021/jacs.7b05796}
}
</pre></td>
</tr>
<tr id="Zhang2017" class="entry">
	<td>Zhang, G.-Q., Liu, K.-W., Li, Z., Lohaus, R., Hsiao, Y.-Y., Niu, S.-C., Wang, J.-Y., Lin, Y.-C., Xu, Q., Chen, L.-J., Yoshida, K., Fujiwara, S., Wang, Z.-W., Zhang, Y.-Q., Mitsuda, N., Wang, M., Liu, G.-H., Pecoraro, L., Huang, H.-X., Xiao, X.-J., Lin, M., Wu, X.-Y., Wu, W.-L., Chen, Y.-Y., Chang, S.-B., Sakamoto, S., Ohme-Takagi, M., Yagi, M., Zeng, S.-J., Shen, C.-Y., Yeh, C.-M., Luo, Y.-B., Tsai, W.-C., Van de Peer, Y. and Liu, Z.-J.</td>
	<td>The Apostasia genome and the evolution of orchids <p class="infolinks">[<a href="javascript:toggleInfo('Zhang2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>Nature<br/>Vol. advance online publication&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1038/nature23897">URL</a>&nbsp;</td>
</tr>
<tr id="bib_Zhang2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Zhang2017,
  author = {Zhang, Guo-Qiang and Liu, Ke-Wei and Li, Zhen and Lohaus, Rolf and Hsiao, Yu-Yun and Niu, Shan-Ce and Wang, Jie-Yu and Lin, Yao-Cheng and Xu, Qing and Chen, Li-Jun and Yoshida, Kouki and Fujiwara, Sumire and Wang, Zhi-Wen and Zhang, Yong-Qiang and Mitsuda, Nobutaka and Wang, Meina and Liu, Guo-Hui and Pecoraro, Lorenzo and Huang, Hui-Xia and Xiao, Xin-Ju and Lin, Min and Wu, Xin-Yi and Wu, Wan-Lin and Chen, You-Yi and Chang, Song-Bin and Sakamoto, Shingo and Ohme-Takagi, Masaru and Yagi, Masafumi and Zeng, Si-Jin and Shen, Ching-Yu and Yeh, Chuan-Ming and Luo, Yi-Bo and Tsai, Wen-Chieh and Van de Peer, Yves and Liu, Zhong-Jian},
  title = {The Apostasia genome and the evolution of orchids},
  journal = {Nature},
  publisher = {Macmillan Publishers Limited, part of Springer Nature. All rights reserved.},
  year = {2017},
  volume = {advance online publication},
  url = {http://dx.doi.org/10.1038/nature23897}
}
</pre></td>
</tr>
<tr id="Zhu2012" class="entry">
	<td>Zhu, F. and Hummer, G.</td>
	<td>Convergence and error estimation in free energy calculations using the weighted histogram analysis method <p class="infolinks">[<a href="javascript:toggleInfo('Zhu2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Zhu2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>Journal of Computational Chemistry<br/>Vol. 33(4), pp. 453 - 465&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1002/jcc.21989">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Zhu2012" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The weighted histogram analysis method (WHAM) has become the standard technique for the analysis of umbrella sampling simulations. In this article, we address the challenges (1) of obtaining fast and accurate solutions of the coupled nonlinear WHAM equations, (2) of quantifying the statistical errors of the resulting free energies, (3) of diagnosing possible systematic errors, and (4) of optimally allocating of the computational resources. Traditionally, the WHAM equations are solved by a fixed-point direct iteration method, despite poor convergence and possible numerical inaccuracies in the solutions. Here, we instead solve the mathematically equivalent problem of maximizing a target likelihood function, by using superlinear numerical optimization algorithms with a significantly faster convergence rate. To estimate the statistical errors in one-dimensional free energy profiles obtained from WHAM, we note that for densely spaced umbrella windows with harmonic biasing potentials, the WHAM free energy profile can be approximated by a coarse-grained free energy obtained by integrating the mean restraining forces. The statistical errors of the coarse-grained free energies can be estimated straightforwardly and then used for the WHAM results. A generalization to multidimensional WHAM is described. We also propose two simple statistical criteria to test the consistency between the histograms of adjacent umbrella windows, which help identify inadequate sampling and hysteresis in the degrees of freedom orthogonal to the reaction coordinate. Together, the estimates of the statistical errors and the diagnostics of inconsistencies in the potentials of mean force provide a basis for the efficient allocation of computational resources in free energy simulations.</td>
</tr>
<tr id="bib_Zhu2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Zhu2012,
  author = {Zhu, Fangqiang and Hummer, Gerhard},
  title = {Convergence and error estimation in free energy calculations using the weighted histogram analysis method},
  journal = {Journal of Computational Chemistry},
  year = {2012},
  volume = {33},
  number = {4},
  pages = {453 - 465},
  doi = {https://doi.org/10.1002/jcc.21989}
}
</pre></td>
</tr>
<tr id="Zhu2002" class="entry">
	<td>Zhu, G. and Keithly, J.S.</td>
	<td>Alpha-proteobacterial relationship of apicomplexan lactate and malate dehydrogenases <p class="infolinks">[<a href="javascript:toggleInfo('Zhu2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>The Journal of Eukaryotic Microbiology<br/>Vol. 49(3), pp. 255 - 261&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1111/j.1550-7408.2002.tb00532.x">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Zhu2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Zhu2002,
  author = {Zhu, Guan and Keithly, Janet S},
  title = {Alpha-proteobacterial relationship of apicomplexan lactate and malate dehydrogenases},
  journal = {The Journal of Eukaryotic Microbiology},
  year = {2002},
  volume = {49},
  number = {3},
  pages = {255 - 261},
  note = {Apicomplexan LDH's evolved from MDHs},
  doi = {https://doi.org/10.1111/j.1550-7408.2002.tb00532.x}
}
</pre></td>
</tr>
<tr id="Zhuo2015" class="entry">
	<td>Zhuo, Y., Cano, K.E., Wang, L., Ilangovan, U., Hinck, A.P., Sousa, R. and Lafer, E.M.</td>
	<td>Nuclear Magnetic Resonance Structural Mapping Reveals Promiscuous Interactions between Clathrin-Box Motif Sequences and the N-Terminal Domain of the Clathrin Heavy Chain. <p class="infolinks">[<a href="javascript:toggleInfo('Zhuo2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Zhuo2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Biochemistry<br/>Vol. 54, pp. 2571-2580&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/acs.biochem.5b00065">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Zhuo2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The recruitment and organization of clathrin at endocytic sites first to form coated pits and then clathrin-coated vesicles depend on interactions between the clathrin N-terminal domain (TD) and multiple clathrin binding sequences on the cargo adaptor and accessory proteins that are concentrated at such sites. Up to four distinct protein binding sites have been proposed to be present on the clathrin TD, with each site proposed to interact with a distinct clathrin binding motif. However, an understanding of how such interactions contribute to clathrin coat assembly must take into account observations that any three of these four sites on clathrin TD can be mutationally ablated without causing loss of clathrin-mediated endocytosis. To take an unbiased approach to mapping binding sites for clathrin-box motifs on clathrin TD, we used isothermal titration calorimetry (ITC) and nuclear magnetic resonance spectroscopy. Our ITC experiments revealed that a canonical clathrin-box motif peptide from the AP-2 adaptor binds to clathrin TD with a stoichiometry of 3:1. Assignment of 90% of the total visible amide resonances in the TROSY-HSQC spectrum of (13)C-, (2)H-, and (15)N-labeled TD40 allowed us to map these three binding sites by analyzing the chemical shift changes as clathrin-box motif peptides were titrated into clathrin TD. We found that three different clathrin-box motif peptides can each simultaneously bind not only to the previously characterized clathrin-box site but also to the W-box site and the β-arrestin splice loop site on a single TD. The promiscuity of these binding sites can help explain why their mutation does not lead to larger effects on clathrin function and suggests a mechanism by which clathrin may be transferred between different proteins during the course of an endocytic event.</td>
</tr>
<tr id="bib_Zhuo2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Zhuo2015,
  author = {Zhuo, Yue and Cano, Kristin E and Wang, Liping and Ilangovan, Udayar and Hinck, Andrew P and Sousa, Rui and Lafer, Eileen M},
  title = {Nuclear Magnetic Resonance Structural Mapping Reveals Promiscuous Interactions between Clathrin-Box Motif Sequences and the N-Terminal Domain of the Clathrin Heavy Chain.},
  journal = {Biochemistry},
  year = {2015},
  volume = {54},
  pages = {2571--2580},
  note = {each CLTC.TD binds three AP2B2s w/ KD of 420 uM},
  doi = {https://doi.org/10.1021/acs.biochem.5b00065}
}
</pre></td>
</tr>
<tr id="Zhuo2010" class="entry">
	<td>Zhuo, Y., Ilangovan, U., Schirf, V., Demeler, B., Sousa, R., Hinck, A.P. and Lafer, E.M.</td>
	<td>Dynamic interactions between clathrin and locally structured elements in a disordered protein mediate clathrin lattice assembly. <p class="infolinks">[<a href="javascript:toggleInfo('Zhuo2010','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Zhuo2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Journal of molecular biology<br/>Vol. 404, pp. 274-290&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1016/j.jmb.2010.09.044">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Zhuo2010" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Assembly of clathrin lattices is mediated by assembly/adaptor proteins that contain domains that bind lipids or membrane-bound cargo proteins and clathrin binding domains (CBDs) that recruit clathrin. Here, we characterize the interaction between clathrin and a large fragment of the CBD of the clathrin assembly protein AP180. Mutational, NMR chemical shift, and analytical ultracentrifugation analyses allowed us to precisely define two clathrin binding sites within this fragment, each of which is found to bind weakly to the N-terminal domain of the clathrin heavy chain (TD). The locations of the two clathrin binding sites are consistent with predictions from sequence alignments of previously identified clathrin binding elements and, by extension, indicate that the complete AP180 CBD contains ∼12 degenerate repeats, each containing a single clathrin binding site. Sequence and circular dichroism analyses have indicated that the AP180 CBD is predominantly unstructured and our NMR analyses confirm that this is largely the case for the AP180 fragment characterized here. Unexpectedly, unlike the many proteins that undergo binding-coupled folding upon interaction with their binding partners, the AP180 fragment is similarly unstructured in its bound and free states. Instead, we find that this fragment exhibits localized β-turn-like structures at the two clathrin binding sites both when free and when bound to clathrin. These observations are incorporated into a model in which weak binding by multiple, pre-structured clathrin binding elements regularly dispersed throughout a largely unstructured CBD allows efficient recruitment of clathrin to endocytic sites and dynamic assembly of the clathrin lattice.</td>
</tr>
<tr id="bib_Zhuo2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Zhuo2010,
  author = {Zhuo, Yue and Ilangovan, Udayar and Schirf, Virgil and Demeler, Borries and Sousa, Rui and Hinck, Andrew P and Lafer, Eileen M},
  title = {Dynamic interactions between clathrin and locally structured elements in a disordered protein mediate clathrin lattice assembly.},
  journal = {Journal of molecular biology},
  year = {2010},
  volume = {404},
  pages = {274--290},
  note = {uses a recombinant ap180 fragment containing two clathrin binding sites. dissociation constant and rate constants. two methods (NMR and AUC) to determine dissociation constants.},
  doi = {https://doi.org/10.1016/j.jmb.2010.09.044}
}
</pre></td>
</tr>
<tr id="Zlatic2011" class="entry">
	<td>Zlatic, S.A., Tornieri, K., L'Hernault, Steven, W and Faundez, V.</td>
	<td>Clathrin-dependent mechanisms modulate the subcellular distribution of class C Vps/HOPS tether subunits in polarized and nonpolarized cells. <p class="infolinks">[<a href="javascript:toggleInfo('Zlatic2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Zlatic2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>Molecular biology of the cell<br/>Vol. 22(10), pp. 1699 - 715&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1091/mbc.E10-10-0799">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Zlatic2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Coats define the composition of carriers budding from organelles. In addition, coats interact with membrane tethers required for vesicular fusion. The yeast AP-3 (Adaptor Protein Complex 3) coat and the class C Vps/HOPS (HOmotypic fusion and Protein Sorting) tether follow this model as their interaction occurs at the carrier fusion step. Here we show that mammalian Vps class C/HOPS subunits and clathrin interact and that acute perturbation of clathrin function disrupts the endosomal distribution of Vps class C/HOPS tethers in HEK293T and polarized neuronal cells. Vps class C/HOPS subunits and clathrin exist in complex with either AP-3 or hepatocyte growth factor receptor substrate (Hrs). Moreover, Vps class C/HOPS proteins cofractionate with clathrin-coated vesicles, which are devoid of Hrs. Expression of FK506 binding protein (FKBP)-clathrin light chain chimeras, to inhibit clathrin membrane association dynamics, increased Vps class C/HOPS subunit content in rab5 endosomal compartments. Additionally, Vps class C/HOPS subunits were concentrated at tips of neuronal processes, and their delivery was impaired by expression of FKBP-clathrin chimeras and AP20187 incubation. These data support a model in which Vps class C/HOPS subunits incorporate into clathrin-coated endosomal domains and carriers in mammalian cells. We propose that vesicular (AP-3) and nonvesicular (Hrs) clathrin mechanisms segregate class C Vps/HOPS tethers to organelles and domains of mammalian cells bearing complex architectures.</td>
</tr>
<tr id="bib_Zlatic2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Zlatic2011,
  author = {Zlatic, Stephanie A and Tornieri, Karine and W, L'Hernault, Steven and Faundez, Victor},
  title = {Clathrin-dependent mechanisms modulate the subcellular distribution of class C Vps/HOPS tether subunits in polarized and nonpolarized cells.},
  journal = {Molecular biology of the cell},
  year = {2011},
  volume = {22},
  number = {10},
  pages = {1699 - 715},
  doi = {https://doi.org/10.1091/mbc.E10-10-0799}
}
</pre></td>
</tr>
<tr id="Zoi2017" class="entry">
	<td>Zoi, I., Antoniou, D. and Schwartz, S.D.</td>
	<td>Electric Fields and Fast Protein Dynamics in Enzymes <p class="infolinks">[<a href="javascript:toggleInfo('Zoi2017','bibtex')">BibTeX</a>]</p></td>
	<td>2017</td>
	<td>The Journal of Physical Chemistry Letters&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Zoi2017" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Zoi2017,
  author = {Zoi, Ioanna and Antoniou, Dimitri and Schwartz, Steven D},
  title = {Electric Fields and Fast Protein Dynamics in Enzymes},
  journal = {The Journal of Physical Chemistry Letters},
  publisher = {ACS Publications},
  year = {2017}
}
</pre></td>
</tr>
<tr id="Zoi2015" class="entry">
	<td>Zoi, I., Motley, M.W., Antoniou, D., Schramm, V.L. and Schwartz, S.D.</td>
	<td>Enzyme homologues have distinct reaction paths through their transition states <p class="infolinks">[<a href="javascript:toggleInfo('Zoi2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Zoi2015','bibtex')">BibTeX</a>]</p></td>
	<td>2015</td>
	<td>Journal of Physical Chemistry B<br/>Vol. 119(9), pp. 3662 - 3668&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1021/jp511983h">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Zoi2015" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Recent studies of the bacterial enzymes EcMTAN and VcMTAN showed that they have different binding affinities for the same transition state analogue. This was surprising given the similarity of their active sites. We performed transition path sampling simulations of both enzymes to reveal the atomic details of the catalytic chemical step, which may be the key for explaining the inhibitor affinity differences. Even though all experimental data would suggest the two enzymes are almost identical, subtle dynamic differences manifest in differences of reaction coordinate, transition state structure, and eventually significant differences in inhibitor binding. Unlike EcMTAN, VcMTAN has multiple distinct transition states, which is an indication that multiple sets of coordinated protein motions can reach a transition state. Reaction coordinate information is only accessible from transition path sampling approaches, since all experimental approaches report averages. Detailed knowledge could have a significant impact on pharmaceutical design. Recent studies of the bacterial enzymes EcMTAN and VcMTAN showed that they have different binding affinities for the same transition state analogue. This was surprising given the similarity of their active sites. We performed transition path sampling simulations of both enzymes to reveal the atomic details of the catalytic chemical step, which may be the key for explaining the inhibitor affinity differences. Even though all experimental data would suggest the two enzymes are almost identical, subtle dynamic differences manifest in differences of reaction coordinate, transition state structure, and eventually significant differences in inhibitor binding. Unlike EcMTAN, VcMTAN has multiple distinct transition states, which is an indication that multiple sets of coordinated protein motions can reach a transition state. Reaction coordinate information is only accessible from transition path sampling approaches, since all experimental approaches report averages. Detailed knowledge could have a significant impact on pharmaceutical design.</td>
</tr>
<tr id="bib_Zoi2015" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Zoi2015,
  author = {Zoi, Ioanna and Motley, Matthew W and Antoniou, Dimitri and Schramm, Vern L and Schwartz, Steven D},
  title = {Enzyme homologues have distinct reaction paths through their transition states},
  journal = {Journal of Physical Chemistry B},
  year = {2015},
  volume = {119},
  number = {9},
  pages = {3662 - 3668},
  doi = {https://doi.org/10.1021/jp511983h}
}
</pre></td>
</tr>
<tr id="Zoi2016" class="entry">
	<td>Zoi, I., Suarez, J., Antoniou, D., Cameron, S.A., Schramm, V.L. and Schwartz, S.D.</td>
	<td>Modulating enzyme catalysis through mutations designed to alter rapid protein dynamics <p class="infolinks">[<a href="javascript:toggleInfo('Zoi2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>Journal of the American Chemical Society<br/>Vol. 138, pp. 3403 - 3409&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Zoi2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Zoi2016,
  author = {Zoi, Ioanna and Suarez, Javier and Antoniou, Dimitri and Cameron, Scott A and Schramm, Vern L and Schwartz, Steven D},
  title = {Modulating enzyme catalysis through mutations designed to alter rapid protein dynamics},
  journal = {Journal of the American Chemical Society},
  year = {2016},
  volume = {138},
  pages = {3403 - 3409},
  note = {NULL}
}
</pre></td>
</tr>
<tr id="Zon2005" class="entry">
	<td>Zon, J.S. and ten Wolde, P.</td>
	<td>Green's-function reaction dynamics: A particle-based approach for simulating biochemical networks in time and space <p class="infolinks">[<a href="javascript:toggleInfo('Zon2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Journal of Chemical Physics<br/>Vol. 123(23), pp. 234910 - 234910&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.2137716">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Zon2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Zon2005,
  author = {Zon, Jeroen S and ten Wolde, Pieter},
  title = {Green's-function reaction dynamics: A particle-based approach for simulating biochemical networks in time and space},
  journal = {Journal of Chemical Physics},
  year = {2005},
  volume = {123},
  number = {23},
  pages = {234910 - 234910},
  doi = {https://doi.org/10.1063/1.2137716}
}
</pre></td>
</tr>
<tr id="Zon2005a" class="entry">
	<td>Zon, J.S. and ten Wolde, P.</td>
	<td>Simulating Biochemical Networks at the Particle Level and in Time and Space: Green's Function Reaction Dynamics <p class="infolinks">[<a href="javascript:toggleInfo('Zon2005a','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Physical Review Letters<br/>Vol. 94(12), pp. 1 - 4&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1103/PhysRevLett.94.128103">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Zon2005a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Zon2005a,
  author = {Zon, Jeroen S and ten Wolde, Pieter},
  title = {Simulating Biochemical Networks at the Particle Level and in Time and Space: Green's Function Reaction Dynamics},
  journal = {Physical Review Letters},
  year = {2005},
  volume = {94},
  number = {12},
  pages = {1 - 4},
  doi = {https://doi.org/10.1103/PhysRevLett.94.128103}
}
</pre></td>
</tr>
<tr id="Zotenko2008" class="entry">
	<td>Zotenko, E., Mestre, J., O'Leary, Dianne, P and Przytycka, T.M.</td>
	<td>Why do hubs in the yeast protein interaction network tend to be essential: Reexamining the connection between the network topology and essentiality <p class="infolinks">[<a href="javascript:toggleInfo('Zotenko2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Zotenko2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>PLoS Computational Biology<br/>Vol. 4(8), pp. e1000140&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1371/journal.pcbi.1000140">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Zotenko2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The centrality-lethality rule, which notes that high-degree nodes in a protein interaction network tend to correspond to proteins that are essential, suggests that the topological prominence of a protein in a protein interaction network may be a good predictor of its biological importance. Even though the correlation between degree and essentiality was confirmed by many independent studies, the reason for this correlation remains illusive. Several hypotheses about putative connections between essentiality of hubs and the topology of protein-protein interaction networks have been proposed, but as we demonstrate, these explanations are not supported by the properties of protein interaction networks. To identify the main topological determinant of essentiality and to provide a biological explanation for the connection between the network topology and essentiality, we performed a rigorous analysis of six variants of the genomewide protein interaction network for Saccharomyces cerevisiae obtained using different techniques. We demonstrated that the majority of hubs are essential due to their involvement in Essential Complex Biological Modules, a group of densely connected proteins with shared biological function that are enriched in essential proteins. Moreover, we rejected two previously proposed explanations for the centrality-lethality rule, one relating the essentiality of hubs to their role in the overall network connectivity and another relying on the recently published essential protein interactions model.</td>
</tr>
<tr id="bib_Zotenko2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Zotenko2008,
  author = {Zotenko, Elena and Mestre, Julian and P, O'Leary, Dianne and Przytycka, Teresa M},
  title = {Why do hubs in the yeast protein interaction network tend to be essential: Reexamining the connection between the network topology and essentiality},
  journal = {PLoS Computational Biology},
  year = {2008},
  volume = {4},
  number = {8},
  pages = {e1000140},
  doi = {https://doi.org/10.1371/journal.pcbi.1000140}
}
</pre></td>
</tr>
<tr id="Zwanzig1954" class="entry">
	<td>Zwanzig, R.W.</td>
	<td>High-Temperature Equation of State by a Perturbation Method. I. Nonpolar Gases <p class="infolinks">[<a href="javascript:toggleInfo('Zwanzig1954','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Zwanzig1954','bibtex')">BibTeX</a>]</p></td>
	<td>1954</td>
	<td>Journal of Chemical Physics<br/>Vol. 22(8), pp. 1420 - 1426&nbsp;</td>
	<td>article</td>
	<td><a href="https://doi.org/10.1063/1.1740409">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Zwanzig1954" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A theoretical study is made of the equations of state of argon and nitrogen, at high temperatures and densities. The intermolecular potential used is of the Lennard‐Jones form, with an adjustable rigid sphere cutoff. A perturbation theory is developed, by which the thermodynamic</td>
</tr>
<tr id="bib_Zwanzig1954" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Zwanzig1954,
  author = {Zwanzig, Robert W},
  title = {High-Temperature Equation of State by a Perturbation Method. I. Nonpolar Gases},
  journal = {Journal of Chemical Physics},
  year = {1954},
  volume = {22},
  number = {8},
  pages = {1420 - 1426},
  doi = {https://doi.org/10.1063/1.1740409}
}
</pre></td>
</tr>
<tr id="1976" class="entry">
	<td></td>
	<td>Advances in Enzymology and Related Areas of Molecular Biology <p class="infolinks">[<a href="javascript:toggleInfo('1976','bibtex')">BibTeX</a>]</p></td>
	<td>1976</td>
	<td>&nbsp;</td>
	<td>book</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_1976" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{1976,,
  title = {Advances in Enzymology and Related Areas of Molecular Biology},
  publisher = {John Wiley &amp; Sons, Inc.},
  year = {1976},
  note = {Biology of alanine racemase, and how it fits into the peptidoglycan formation pathway}
}
</pre></td>
</tr>
<tr id="2016" class="entry">
	<td></td>
	<td>Simulating Enzyme Reactivity <p class="infolinks">[<a href="javascript:toggleInfo('2016','bibtex')">BibTeX</a>]</p></td>
	<td>2016</td>
	<td>&nbsp;</td>
	<td>book</td>
	<td><a href="https://doi.org/10.1039/9781782626831">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_2016" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{2016,,
  title = {Simulating Enzyme Reactivity},
  publisher = {Royal Society of Chemistry},
  year = {2016},
  note = {NULL},
  doi = {https://doi.org/10.1039/9781782626831}
}
</pre></td>
</tr>
<tr id="" class="entry">
	<td></td>
	<td>pnas200911073 4206..4211 <p class="infolinks">[<a href="javascript:toggleInfo('','bibtex')">BibTeX</a>]</p></td>
	<td></td>
	<td>&nbsp;</td>
	<td>misc</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@misc{,,
  title = {pnas200911073 4206..4211}
}
</pre></td>
</tr>
<tr id="" class="entry">
	<td></td>
	<td> <p class="infolinks">[<a href="javascript:toggleInfo('','bibtex')">BibTeX</a>]</p></td>
	<td></td>
	<td>&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{,
}
</pre></td>
</tr>
</tbody>
</table>
<footer>
 <small>Created by <a href="http://jabref.sourceforge.net">JabRef</a> on 01/03/2018.</small>
</footer>
<!-- file generated by JabRef -->
</body>
</html>
